<!DOCTYPE html>
<html class="js webfonts-loaded" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <title>BMGE (Block Mapping and Gathering with Entropy): a new 
software for selection of phylogenetic informative regions from multiple
 sequence alignments | BMC Evolutionary Biology | Full Text</title>

        

    <meta name="citation_abstract" content="The quality of multiple sequence alignments plays an important role in the accuracy of phylogenetic inference. It has been shown that removing ambiguously aligned regions, but also other sources of bias such as highly variable (saturated) characters, can improve the overall performance of many phylogenetic reconstruction methods. A current scientific trend is to build phylogenetic trees from a large number of sequence datasets (semi-)automatically extracted from numerous complete genomes. Because these approaches do not allow a precise manual curation of each dataset, there exists a real need for efficient bioinformatic tools dedicated to this alignment character trimming step. Here is presented a new software, named BMGE (Block Mapping and Gathering with Entropy), that is designed to select regions in a multiple sequence alignment that are suited for phylogenetic inference. For each character, BMGE computes a score closely related to an entropy value. Calculation of these entropy-like scores is weighted with BLOSUM or PAM similarity matrices in order to distinguish among biologically expected and unexpected variability for each aligned character. Sets of contiguous characters with a score above a given threshold are considered as not suited for phylogenetic inference and then removed. Simulation analyses show that the character trimming performed by BMGE produces datasets leading to accurate trees, especially with alignments including distantly-related sequences. BMGE also implements trimming and recoding methods aimed at minimizing phylogeny reconstruction artefacts due to compositional heterogeneity. BMGE is able to perform biologically relevant trimming on a multiple alignment of DNA, codon or amino acid sequences. Java source code and executable are freely available at 
                    ftp://ftp.pasteur.fr/pub/GenSoft/projects/BMGE/
                    
                  .">

    <meta name="journal_id" content="12862">

    <meta name="dc.title" content="BMGE (Block Mapping and Gathering with Entropy): a new software for selection of phylogenetic informative regions from multiple sequence alignments">

    <meta name="dc.source" content="BMC Evolutionary Biology 2010 10:1">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="BioMed Central">

    <meta name="dc.date" content="2010-07-13">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2010 Criscuolo and Gribaldo; licensee BioMed Central Ltd.">

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="dc.description" content="The quality of multiple sequence alignments plays an important role in the accuracy of phylogenetic inference. It has been shown that removing ambiguously aligned regions, but also other sources of bias such as highly variable (saturated) characters, can improve the overall performance of many phylogenetic reconstruction methods. A current scientific trend is to build phylogenetic trees from a large number of sequence datasets (semi-)automatically extracted from numerous complete genomes. Because these approaches do not allow a precise manual curation of each dataset, there exists a real need for efficient bioinformatic tools dedicated to this alignment character trimming step. Here is presented a new software, named BMGE (Block Mapping and Gathering with Entropy), that is designed to select regions in a multiple sequence alignment that are suited for phylogenetic inference. For each character, BMGE computes a score closely related to an entropy value. Calculation of these entropy-like scores is weighted with BLOSUM or PAM similarity matrices in order to distinguish among biologically expected and unexpected variability for each aligned character. Sets of contiguous characters with a score above a given threshold are considered as not suited for phylogenetic inference and then removed. Simulation analyses show that the character trimming performed by BMGE produces datasets leading to accurate trees, especially with alignments including distantly-related sequences. BMGE also implements trimming and recoding methods aimed at minimizing phylogeny reconstruction artefacts due to compositional heterogeneity. BMGE is able to perform biologically relevant trimming on a multiple alignment of DNA, codon or amino acid sequences. Java source code and executable are freely available at 
                    ftp://ftp.pasteur.fr/pub/GenSoft/projects/BMGE/
                    
                  .">

    <meta name="prism.issn" content="1471-2148">

    <meta name="prism.publicationName" content="BMC Evolutionary Biology">

    <meta name="prism.publicationDate" content="2010-07-13">

    <meta name="prism.volume" content="10">

    <meta name="prism.number" content="1">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="1">

    <meta name="prism.endingPage" content="21">

    <meta name="prism.copyright" content="2010 Criscuolo and Gribaldo; licensee BioMed Central Ltd.">

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="prism.url" content="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210">

    <meta name="prism.doi" content="doi:10.1186/1471-2148-10-210">

    <meta name="citation_pdf_url" content="https://bmcevolbiol.biomedcentral.com/track/pdf/10.1186/1471-2148-10-210">

    <meta name="citation_fulltext_html_url" content="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210">

    <meta name="citation_journal_title" content="BMC Evolutionary Biology">

    <meta name="citation_journal_abbrev" content="BMC Evol Biol">

    <meta name="citation_publisher" content="BioMed Central">

    <meta name="citation_issn" content="1471-2148">

    <meta name="citation_title" content="BMGE (Block Mapping and Gathering with Entropy): a new software for selection of phylogenetic informative regions from multiple sequence alignments">

    <meta name="citation_volume" content="10">

    <meta name="citation_issue" content="1">

    <meta name="citation_publication_date" content="2010/12">

    <meta name="citation_online_date" content="2010/07/13">

    <meta name="citation_firstpage" content="1">

    <meta name="citation_lastpage" content="21">

    <meta name="citation_article_type" content="Software">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="citation_language" content="en">

    <meta name="dc.identifier" content="doi:10.1186/1471-2148-10-210">

    <meta name="DOI" content="10.1186/1471-2148-10-210">

    <meta name="citation_doi" content="10.1186/1471-2148-10-210">

    <meta name="description" content="The quality of multiple sequence alignments plays an important role in the accuracy of phylogenetic inference. It has been shown that removing ambiguously aligned regions, but also other sources of bias such as highly variable (saturated) characters, can improve the overall performance of many phylogenetic reconstruction methods. A current scientific trend is to build phylogenetic trees from a large number of sequence datasets (semi-)automatically extracted from numerous complete genomes. Because these approaches do not allow a precise manual curation of each dataset, there exists a real need for efficient bioinformatic tools dedicated to this alignment character trimming step. Here is presented a new software, named BMGE (Block Mapping and Gathering with Entropy), that is designed to select regions in a multiple sequence alignment that are suited for phylogenetic inference. For each character, BMGE computes a score closely related to an entropy value. Calculation of these entropy-like scores is weighted with BLOSUM or PAM similarity matrices in order to distinguish among biologically expected and unexpected variability for each aligned character. Sets of contiguous characters with a score above a given threshold are considered as not suited for phylogenetic inference and then removed. Simulation analyses show that the character trimming performed by BMGE produces datasets leading to accurate trees, especially with alignments including distantly-related sequences. BMGE also implements trimming and recoding methods aimed at minimizing phylogeny reconstruction artefacts due to compositional heterogeneity. BMGE is able to perform biologically relevant trimming on a multiple alignment of DNA, codon or amino acid sequences. Java source code and executable are freely available at 
                    ftp://ftp.pasteur.fr/pub/GenSoft/projects/BMGE/
                    
                  .">

    <meta name="dc.creator" content="Alexis Criscuolo">

    <meta name="dc.creator" content="Simonetta Gribaldo">

    <meta name="dc.subject" content="Evolutionary Biology">

    <meta name="dc.subject" content="Animal Systematics/Taxonomy/Biogeography">

    <meta name="dc.subject" content="Entomology">

    <meta name="dc.subject" content="Genetics and Population Dynamics">

    <meta name="dc.subject" content="Life Sciences, general">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=The order of sequence alignment can bias the selection of tree topology; citation_author=JA Lake; citation_volume=8; citation_publication_date=1991; citation_pages=378-385; citation_id=CR1">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Effects of nucleotide sequence alignment on phylogeny estimation: a case study on 18 S rDNAs of apicomplexa; citation_author=DA Morrison, JT Ellis; citation_volume=14; citation_publication_date=1997; citation_pages=428-441; citation_doi=10.1093/oxfordjournals.molbev.a025779; citation_id=CR2">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Multiple sequence alignment accuracy and phylogenetic inference; citation_author=TH Ogden, MS Rosenberg; citation_volume=55; citation_publication_date=2006; citation_pages=314-328; citation_doi=10.1080/10635150500541730; citation_id=CR3">

    <meta name="citation_reference" content="citation_title=The impact of multiple protein sequence alignment on phylogenetic estimation; citation_inbook_title=IEEE/ACM Trans Comput Biol Bioinf; citation_publication_date=2009; citation_id=CR4; citation_author=L-S Wang; citation_author=J Leebens-Mack; citation_author=PK Wall; citation_author=K Beckmann; citation_author=CW dePamphilis; citation_author=T Warnow">

    <meta name="citation_reference" content="citation_journal_title=Curr Opin Struct Biol; citation_title=Multiple sequence alignment; citation_author=RC Edgar, S Batzoglou; citation_volume=16; citation_publication_date=2006; citation_pages=368-373; citation_doi=10.1016/j.sbi.2006.04.004; citation_id=CR5">

    <meta name="citation_reference" content="citation_journal_title=PLoS Comput Biol; citation_title=Recent evolutions of multiple sequence alignment algorithms; citation_author=C Notredame; citation_volume=3; citation_publication_date=2007; citation_pages=e123; citation_doi=10.1371/journal.pcbi.0030123; citation_id=CR6">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis; citation_author=J Castresana; citation_volume=17; citation_publication_date=2000; citation_pages=540-552; citation_doi=10.1093/oxfordjournals.molbev.a026334; citation_id=CR7">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments; citation_author=G Talavera, J Castresana; citation_volume=56; citation_publication_date=2007; citation_pages=564-577; citation_doi=10.1080/10635150701472164; citation_id=CR8">

    <meta name="citation_reference" content="citation_journal_title=FASEB J; citation_title=Ribosomal RNA: a key to phylogeny; citation_author=GJ Olsen, CR Woese; citation_volume=7; citation_publication_date=1993; citation_pages=113-123; citation_id=CR9">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Inadequate support for an evolutionary link between the Metazoa and the Fungi; citation_author=AG Rodrigo, PR Bergquist, PL Bergquist; citation_volume=43; citation_publication_date=1994; citation_pages=578-584; citation_doi=10.1093/sysbio/43.4.578; citation_id=CR10">

    <meta name="citation_reference" content="citation_title=Phylogenetic inference; citation_inbook_title=Molecular Systematics; citation_publication_date=1996; citation_pages=407-514; citation_id=CR11; citation_author=DL Swofford; citation_author=GJ Olsen; citation_author=PJ Waddell; citation_author=DM Hillis; citation_publisher=Sinauer Associates">

    <meta name="citation_reference" content="citation_journal_title=Curr Biol; citation_title=Monophyly of primary photosynthetic eukaryotes: green plants, red algae, and glaucophytes; citation_author=N Rodríguez-Ezpeleta, H Brinkmann, SC Burey, B Roure, G Burger, W Löffelhardt, H Philippe, BF Lang; citation_volume=15; citation_publication_date=2005; citation_pages=1325-1330; citation_doi=10.1016/j.cub.2005.06.040; citation_id=CR12">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=PhylomeDB: a database for genome-wide collections of gene phylogenies; citation_author=J Huerta-Cepas, A Bueno, J Dopazo, T Gabaldón; citation_volume=36; citation_publication_date=2008; citation_pages=D491-D496; citation_doi=10.1093/nar/gkm899; citation_id=CR13">

    <meta name="citation_reference" content="citation_journal_title=Algorithms for Molecular Biology; citation_title=Noisy: Identification of problematic columns in multiple sequence alignments; citation_author=AWM Dress, C Flamm, G Fritzsch, S Grünewald, M Kruspe, SJ Prohaska, PF Stadler; citation_volume=3; citation_publication_date=2008; citation_pages=7; citation_doi=10.1186/1748-7188-3-7; citation_id=CR14">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Integrating Markov clustering and molecular phylogenetics to reconstruct the cyanobacterial species tree from conserved protein families; citation_author=WD Swingley, RE Blankenship, J Raymond; citation_volume=25; citation_publication_date=2008; citation_pages=643-654; citation_doi=10.1093/molbev/msn034; citation_id=CR15">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=trimAl: a tool for automated alignment triming in large-scale phylogenetic analyses; citation_author=S Capella-Gutiérez, JM Silla-Martínez, T Gabaldón; citation_volume=25; citation_publication_date=2009; citation_pages=1972-1973; citation_doi=10.1093/bioinformatics/btp348; citation_id=CR16">

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=A phylogenomic approach to bacterial phylogeny: evidence of a core of genes sharing a common history; citation_author=V Daubin, M Gouy, G Perrière; citation_volume=12; citation_publication_date=2002; citation_pages=1080-1090; citation_doi=10.1101/gr.187002; citation_id=CR17">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Toward automatic reconstruction of a highly resolved tree of life; citation_author=FD Ciccarelli, T Doerks, C von Mering, CJ Creevey, B Snel, P Bork; citation_volume=311; citation_publication_date=2006; citation_pages=1283-1287; citation_doi=10.1126/science.1123061; citation_id=CR18">

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Plastid genome phylogeny and a model of amino acid substitution for protein encoded by chloroplast DNA; citation_author=J Adachi, PJ Waddell, W Martin, M Hasegawa; citation_volume=50; citation_publication_date=2000; citation_pages=348-358; citation_id=CR19">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Phylogenetic supermatrix analysis of GenBank sequences from 2228 Papilionoid legumes; citation_author=MM McMahon, MJ Sanderson; citation_volume=55; citation_publication_date=2006; citation_pages=818-836; citation_doi=10.1080/10635150600999150; citation_id=CR20">

    <meta name="citation_reference" content="citation_journal_title=Nucl Acids Res; citation_title=Sequence logos: a new way to display consensus sequences; citation_author=TD Schneider, RM Stephens; citation_volume=18; citation_publication_date=1990; citation_pages=6097-6100; citation_doi=10.1093/nar/18.20.6097; citation_id=CR21">

    <meta name="citation_reference" content="citation_journal_title=Evol Bioinf Online; citation_title=Estimation of phylogeny using a general Markov model; citation_author=V Jayaswal, LS Jermiin, J Robinson; citation_volume=1; citation_publication_date=2005; citation_pages=62-80; citation_id=CR22">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Inferring pattern and process: maximum-likelihood implementation of a nonhomogeneous model of DNA sequence evolution for phylogenetic analysis; citation_author=N Galtier, M Gouy; citation_volume=15; citation_publication_date=1998; citation_pages=871-879; citation_doi=10.1093/oxfordjournals.molbev.a025991; citation_id=CR23">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=The biasing effect of compositional heterogeneity on phylogenetic estimates may be underestimated; citation_author=LS Jermiin, SYW Ho, F Ababneh, J Robinson, AWD Larkum; citation_volume=53; citation_publication_date=2004; citation_pages=638-643; citation_doi=10.1080/10635150490468648; citation_id=CR24">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Genome-scale phylogeny and the detection of systematic biases; citation_author=MJ Phillips, F Delsuc, D Penny; citation_volume=21; citation_publication_date=2004; citation_pages=1455-1458; citation_doi=10.1093/molbev/msh137; citation_id=CR25">

    <meta name="citation_reference" content="citation_journal_title=Biochem J; citation_title=Abbreviations and symbols for nucleic acids, polynucleotides and their constituents; citation_author=; citation_volume=120; citation_publication_date=1970; citation_pages=449-454; citation_doi=10.1042/bj1200449; citation_id=CR26">

    <meta name="citation_reference" content="Dayhoff MO, Schwartz RM, Orcutt BD: A model of evolutionary change in proteins. Atlas of Protein Sequence and Structure. Edited by: Dayhoff MO. 1978, Washington: National Biomedical Research Foundation, 5 (Suppl 3): 345-352.">

    <meta name="citation_reference" content="citation_journal_title=Philos Trans R Soc Lond B Biol Sci; citation_title=Mitochondria and hydrogenosomes are two forms of the same fundamental organelle; citation_author=TM Embley, M van der Giezen, DS Horner, PL Dyal, P Foster; citation_volume=358; citation_publication_date=2003; citation_pages=191-203; citation_doi=10.1098/rstb.2002.1190; citation_id=CR28">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=On reduced amino acid alphabets for phylogenetic inference; citation_author=E Susko, AJ Roger; citation_volume=24; citation_publication_date=2007; citation_pages=2139-2150; citation_doi=10.1093/molbev/msm144; citation_id=CR29">

    <meta name="citation_reference" content="citation_journal_title=Biometrika; citation_title=A test for homogeneity of the marginal distributions in a two-way classification; citation_author=A Stuart; citation_volume=42; citation_publication_date=1955; citation_pages=412-416; citation_doi=10.1093/biomet/42.3-4.412; citation_id=CR30">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Matched-pairs tests of homogeneity with applications to homologous nucleotide sequences; citation_author=F Ababneh, LS Jermiin, C Ma, J Robinson; citation_volume=22; citation_publication_date=2006; citation_pages=1225-1231; citation_doi=10.1093/bioinformatics/btl064; citation_id=CR31">

    <meta name="citation_reference" content="citation_journal_title=Pure Appl Chem; citation_title=A one-letter notation for amino acid sequences (definitive rules); citation_author=; citation_volume=31; citation_publication_date=1972; citation_pages=639-645; citation_id=CR32">

    <meta name="citation_reference" content="citation_title=Mathematische Grundlagen der Quantenmechanik; citation_publication_date=1932; citation_id=CR33; citation_author=J Von Neumann; citation_publisher=Springer">

    <meta name="citation_reference" content="citation_journal_title=Protein Sci; citation_title=Are protein-protein interfaces more conserved in sequence than the rest of the protein surface?; citation_author=DR Caffrey, S Somaroo, JD Hughes, J Mintseris, ES Huang; citation_volume=13; citation_publication_date=2004; citation_pages=190-202; citation_doi=10.1110/ps.03323604; citation_id=CR34">

    <meta name="citation_reference" content="JAMA: A Java Matrix Package. [
                    http://math.nist.gov/javanumerics/jama/
                    
                  ]">

    <meta name="citation_reference" content="citation_journal_title=Bell System Tech J; citation_title=A mathematical theory of communication; citation_author=C Shannon; citation_volume=27; citation_publication_date=1948; citation_pages=379-423; citation_doi=10.1002/j.1538-7305.1948.tb01338.x; citation_id=CR36">

    <meta name="citation_reference" content="citation_journal_title=J Theor Biol; citation_title=The classification of amino acid conservation; citation_author=WR Taylor; citation_volume=119; citation_publication_date=1986; citation_pages=205-218; citation_doi=10.1016/S0022-5193(86)80075-3; citation_id=CR37">

    <meta name="citation_reference" content="citation_journal_title=J mol Biol; citation_title=Suggestion for &quot;safe&quot; residue substitutions in site-directed mutagenesis; citation_author=D Bordo, P Argos; citation_volume=217; citation_publication_date=1991; citation_pages=721-729; citation_doi=10.1016/0022-2836(91)90528-E; citation_id=CR38">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci; citation_title=Amino acid substitution matrices from protein blocks; citation_author=S Henikoff, JG Henikoff; citation_volume=89; citation_publication_date=1992; citation_pages=10915-10919; citation_doi=10.1073/pnas.89.22.10915; citation_id=CR39">

    <meta name="citation_reference" content="BLOSUM matrices. [
                    http://blocks.fhcrc.org/blocks/uploads/blosum/
                    
                  ]">

    <meta name="citation_reference" content="citation_journal_title=Nat Biotechnol; citation_title=Where did the BLOSUM62 alignment score matrix come from?; citation_author=SR Eddy; citation_volume=22; citation_publication_date=2004; citation_pages=1035-1036; citation_doi=10.1038/nbt0804-1035; citation_id=CR41">

    <meta name="citation_reference" content="citation_journal_title=Methods: A Companion to Methods Enzymol; citation_title=Improved sensitivity of nucleic acid database searches using application-specific scoring matrices; citation_author=DJ States, W Gish, SF Altschul; citation_volume=3; citation_publication_date=1991; citation_pages=66-70; citation_doi=10.1016/S1046-2023(05)80165-3; citation_id=CR42">

    <meta name="citation_reference" content="citation_journal_title=Behaviormetrika; citation_title=Validity and applicability of several tests for comparing marginal distributions of a square table with ordered categories; citation_author=H Uesaka; citation_volume=30; citation_publication_date=1991; citation_pages=65-78; citation_doi=10.2333/bhmk.18.30_65; citation_id=CR43">

    <meta name="citation_reference" content="citation_title=Numerical Recipes in C -- The Art of Scientific Computing; citation_publication_date=1992; citation_id=CR44; citation_author=WH Press; citation_author=SA Teukolsky; citation_author=WT Vetterling; citation_author=BP Flannery; citation_publisher=Cambridge University Press">

    <meta name="citation_reference" content="Artificial datasets used in [48]. [
                    http://www.atgc-montpellier.fr/phyml/datasets.php
                    
                  ]">

    <meta name="citation_reference" content="citation_journal_title=Comput Appl Biosci; citation_title=Seq-Gen: an application for the Monte Carlo simulation of DNA sequence evolution along phylogenetic trees; citation_author=A Rambaut, NC Grassly; citation_volume=13; citation_publication_date=1997; citation_pages=235-238; citation_id=CR46">

    <meta name="citation_reference" content="citation_journal_title=Comput Appl Biosci; citation_title=The rapid generation of mutation data matrices from protein sequences; citation_author=D Jones, W Taylor, J Thornton; citation_volume=8; citation_publication_date=1992; citation_pages=275-282; citation_id=CR47">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood; citation_author=S Guindon, O Gascuel; citation_volume=52; citation_publication_date=2003; citation_pages=696-704; citation_doi=10.1080/10635150390235520; citation_id=CR48">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput; citation_author=RC Edgar; citation_volume=32; citation_publication_date=2004; citation_pages=1792-1797; citation_doi=10.1093/nar/gkh340; citation_id=CR49">

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity; citation_author=RC Edgar; citation_volume=5; citation_publication_date=2004; citation_pages=113; citation_doi=10.1186/1471-2105-5-113; citation_id=CR50">

    <meta name="citation_reference" content="citation_title=Signal detection theory and ROC analysis, series in cognition and perception; citation_publication_date=1975; citation_id=CR51; citation_author=JP Egan; citation_publisher=Academic Press">

    <meta name="citation_reference" content="citation_journal_title=Semin Nucl Med; citation_title=Basic principles of ROC analysis; citation_author=CE Metz; citation_volume=8; citation_publication_date=1978; citation_pages=283-298; citation_doi=10.1016/S0001-2998(78)80014-2; citation_id=CR52">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Measuring the accuracy of diagnostic systems; citation_author=J Swets; citation_volume=240; citation_publication_date=1988; citation_pages=1285-1293; citation_doi=10.1126/science.3287615; citation_id=CR53">

    <meta name="citation_reference" content="citation_journal_title=Pattern Recogn Lett; citation_title=An introduction to ROC analysis; citation_author=T Fawcett; citation_volume=27; citation_publication_date=2006; citation_pages=861-874; citation_doi=10.1016/j.patrec.2005.10.010; citation_id=CR54">

    <meta name="citation_reference" content="citation_journal_title=Ann Math Stat; citation_title=A note on the power of the sign test; citation_author=W McStewart; citation_volume=12; citation_publication_date=1941; citation_pages=279-303; citation_doi=10.1214/aoms/1177731710; citation_id=CR55">

    <meta name="citation_reference" content="citation_journal_title=J Am Statist Assoc; citation_title=The statistical sign test; citation_author=WJ Dixon, AM Mood; citation_volume=41; citation_publication_date=1946; citation_pages=557-566; citation_doi=10.1080/01621459.1946.10501898; citation_id=CR56">

    <meta name="citation_reference" content="citation_journal_title=Proc Nederl Akad Weten Ser A; citation_title=A theorem on the sign test when ties are present; citation_author=J Hemelrijk; citation_volume=55; citation_publication_date=1952; citation_pages=322; citation_id=CR57">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data; citation_author=O Gascuel; citation_volume=14; citation_publication_date=1997; citation_pages=685-695; citation_doi=10.1093/oxfordjournals.molbev.a025808; citation_id=CR58">

    <meta name="citation_reference" content="citation_title=TNT (Tree analysis using New Technology) ver. 1.0; citation_publication_date=2000; citation_id=CR59; citation_author=P Goloboff; citation_author=S Farris; citation_author=KC Nixon">

    <meta name="citation_reference" content="citation_journal_title=Cladistics; citation_title=The parsimony ratchet, a new method for rapid parsimony analysis; citation_author=KC Nixon; citation_volume=15; citation_publication_date=1999; citation_pages=407-414; citation_doi=10.1111/j.1096-0031.1999.tb00277.x; citation_id=CR60">

    <meta name="citation_reference" content="citation_journal_title=Syst Zool; citation_title=Comparison of undirected phylogenetic trees based on subtrees of four evolutionary units; citation_author=GF Estabrook, FR McMorris, CA Meacham; citation_volume=34; citation_publication_date=1985; citation_pages=193-200; citation_doi=10.2307/2413326; citation_id=CR61">

    <meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Confidence limits on phylogenies: an approach using the bootstrap; citation_author=J Felsenstein; citation_volume=39; citation_publication_date=1985; citation_pages=783-791; citation_doi=10.2307/2408678; citation_id=CR62">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Approximate likelihood ratio test for branches: a fast, accurate and powerful alternative; citation_author=M Anisimova, O Gascuel; citation_volume=55; citation_publication_date=2006; citation_pages=539-552; citation_doi=10.1080/10635150600755453; citation_id=CR63">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0; citation_author=S Guindon, JF Dufayard, V Lefort, M Anisimova, W Hordijk, O Gascuel; citation_volume=59; citation_publication_date=2010; citation_pages=307-321; citation_doi=10.1093/sysbio/syq010; citation_id=CR64">

    <meta name="citation_reference" content="citation_journal_title=Radiology; citation_title=The meaning and the use of the area under a receiver operating characteristic (ROC) curve; citation_author=JA Hanley, BJ McNeil; citation_volume=143; citation_publication_date=1982; citation_pages=29-36; citation_doi=10.1148/radiology.143.1.7063747; citation_id=CR65">

    <meta name="citation_reference" content="citation_title=Case studies in the use of ROC curve analysis for sensor-based estimates in human computer interaction; citation_inbook_title=Proceedings of Graphics Interface (GI 2005): 09-11 May 2005; Victoria, British Columbia; citation_publication_date=2005; citation_pages=129-136; citation_id=CR66; citation_author=J Fogarty; citation_author=RS Baker; citation_author=SE Hudson">

    <meta name="citation_reference" content="Concatenate: a software to build supermatrices of characters. [
                    http://www.supertriplets.univ-montp2.fr/PhyloTools.php
                    
                  ]">

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Model of amino acid substitution in proteins encoded by mitochondrial DNA; citation_author=J Adachi, M Hasegawa; citation_volume=42; citation_publication_date=1996; citation_pages=459-468; citation_doi=10.1007/BF02498640; citation_id=CR68">

    <meta name="citation_reference" content="citation_journal_title=Curr Biol; citation_title=Monophyly of Primary Photosynthetic Eukaryotes: Green Plants, Red Algae, and Glaucophytes; citation_author=N Rodríguez-Ezpeleta, H Brinkmann, SC Burey, B Roure, G Burger, W Löffelhardt, HJ Bohnert, H Philippe; citation_volume=15; citation_publication_date=2005; citation_pages=1325-1330; citation_doi=10.1016/j.cub.2005.06.040; citation_id=CR69">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Phylogenomic analysis supports the monophyly of Cryptophytes and Haptophytes and the association of Rhizaria with Chromalveolates; citation_author=JD Hackett, HS Yoon, S Li, A Reyes-Prieto, SE Rümmele, D Bhattacharya; citation_volume=24; citation_publication_date=2007; citation_pages=1702-1713; citation_doi=10.1093/molbev/msm089; citation_id=CR70">

    <meta name="citation_reference" content="citation_journal_title=Biol Lett; citation_title=Phylogenomics reveals a new 'metagroup' including most photosynthetic eukaryotes; citation_author=F Burki, K Shalchian-Tabrizi, J Pawlowski; citation_volume=4; citation_publication_date=2008; citation_pages=366-369; citation_doi=10.1098/rsbl.2008.0224; citation_id=CR71">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Phylogenomic analyses support the monophyly of Excavata and resolve relationships among eukaryotic &quot;supergroups&quot;; citation_author=V Hampl, L Hug, JW Leigh, JB Dacks, BF Lang, AGB Simpson, AJ Roger; citation_volume=106; citation_publication_date=2009; citation_pages=3859-3864; citation_doi=10.1073/pnas.0807880106; citation_id=CR72">

    <meta name="citation_reference" content="citation_title=Statistical methods in bioinformatics: an introduction; citation_publication_date=2005; citation_id=CR73; citation_author=W Ewens; citation_author=G Grant; citation_publisher=Springer">

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Evolutionary tree from DNA sequences: a maximum likelihood approach; citation_author=J Felsenstein; citation_volume=17; citation_publication_date=1981; citation_pages=368-376; citation_doi=10.1007/BF01734359; citation_id=CR74">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Tests of applicability of several substitution models for DNA sequence data; citation_author=A Rzhetsky, M Nei; citation_volume=12; citation_publication_date=1995; citation_pages=131-151; citation_doi=10.1093/oxfordjournals.molbev.a040182; citation_id=CR75">

    <meta name="citation_reference" content="citation_journal_title=J Am Stat Assoc; citation_title=A test for symmetry in contingency tables; citation_author=AH Bowker; citation_volume=43; citation_publication_date=1948; citation_pages=572-574; citation_doi=10.1080/01621459.1948.10483284; citation_id=CR76">

    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Genome-scale approaches to resolving incongruence in molecular phylogenies; citation_author=A Rokas, BL Williams, N King, SB Carroll; citation_volume=425; citation_publication_date=2003; citation_pages=798-804; citation_doi=10.1038/nature02053; citation_id=CR77">

    <meta name="citation_reference" content="Yeast multi-gene dataset. [
                    http://systbio.org/files/Ren_etal_Rokas2003data.txt
                    
                  ]">

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=A new method for calculating evolutionary substitution rates; citation_author=C Lanave, G Preparata, C Saccone, G Serio; citation_volume=20; citation_publication_date=1984; citation_pages=86-93; citation_doi=10.1007/BF02101990; citation_id=CR79">

    <meta name="citation_reference" content="citation_journal_title=J Theor Biol; citation_title=The general stochastic model of nucleotide substitution; citation_author=R Rodriguez, JL Oliver, A Marin, JR Medina; citation_volume=142; citation_publication_date=1990; citation_pages=485-501; citation_doi=10.1016/S0022-5193(05)80104-3; citation_id=CR80">

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Estimating the pattern of nucleotide substitution; citation_author=Z Yang; citation_volume=39; citation_publication_date=1994; citation_pages=105-111; citation_id=CR81">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Reconstructing evolutionary trees from DNA and protein sequences: paralinear distances; citation_author=JA Lake; citation_volume=91; citation_publication_date=1994; citation_pages=1455-1459; citation_doi=10.1073/pnas.91.4.1455; citation_id=CR82">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Recovering evolutionary trees under a more realistic model of sequence evolution; citation_author=PJ Lockhart, MA Steel, MD Hendy, D Penny; citation_volume=11; citation_publication_date=1994; citation_pages=605-612; citation_id=CR83">

    <meta name="citation_reference" content="citation_journal_title=Appl Math Lett; citation_title=Recovering a tree from the leaf colourations it generates under a Markov model; citation_author=MA Steel; citation_volume=7; citation_publication_date=1994; citation_pages=19-23; citation_doi=10.1016/0893-9659(94)90024-8; citation_id=CR84">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=An assessment of accuracy, error, and conflict with support values from genome-scale phylogenetic data; citation_author=DJ Taylor, WH Piel; citation_volume=21; citation_publication_date=2004; citation_pages=1534-1537; citation_doi=10.1093/molbev/msh156; citation_id=CR85">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=An empirical examination of the utility of codon-substitution models in phylogeny reconstruction; citation_author=F Ren, H Tanaka, Z Yang; citation_volume=54; citation_publication_date=2005; citation_pages=808-818; citation_doi=10.1080/10635150500354688; citation_id=CR86">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Supertree bootstrapping methods for assessing phylogenetic variation among genes in genome-scale data sets; citation_author=JG Burleigh, AC Driskell, MJ Sanderson; citation_volume=55; citation_publication_date=2006; citation_pages=426-440; citation_doi=10.1080/10635150500541722; citation_id=CR87">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Bayesian estimation of concordance among gene trees; citation_author=C Ané, B Larget, DA Baum, SD Smith, A Rokas; citation_volume=24; citation_publication_date=2007; citation_pages=412-426; citation_doi=10.1093/molbev/msl170; citation_id=CR88">

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Phylogenetic inference with weighted codon evolutionary distances; citation_author=A Criscuolo, CJ Michel; citation_volume=68; citation_publication_date=2009; citation_pages=377-392; citation_doi=10.1007/s00239-009-9212-y; citation_id=CR89">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Mobyle: a new full web bioinformatics framework; citation_author=B Néron, H Ménager, C Maufrais, N Joly, J Maupetit, S Letort, S Carrere, P Tuffery, C Letondal; citation_volume=25; citation_publication_date=2009; citation_pages=3005-3011; citation_doi=10.1093/bioinformatics/btp493; citation_id=CR90">

    <meta name="citation_author" content="Alexis Criscuolo">

    <meta name="citation_author_institution" content="Institut Pasteur, Unité de Biologie Moléculaire du Gène chez les Extrêmophiles, Département de Microbiologie, Paris, France">

    <meta name="citation_author" content="Simonetta Gribaldo">

    <meta name="citation_author_institution" content="Institut Pasteur, Unité de Biologie Moléculaire du Gène chez les Extrêmophiles, Département de Microbiologie, Paris, France">



        <meta name="format-detection" content="telephone=no">

        <link rel="shortcut icon" href="https://bmcevolbiol.biomedcentral.com/static/images/favicons/bmc/favicon-9ff1bf1161.ico">
<link rel="apple-touch-icon" sizes="114x114" href="https://bmcevolbiol.biomedcentral.com/static/images/favicons/bmc/app-icon-114x114-aa3514ee38.png">
<link rel="apple-touch-icon" sizes="144x144" href="https://bmcevolbiol.biomedcentral.com/static/images/favicons/bmc/app-icon-144x144-67f5b60136.png">
<link rel="apple-touch-icon" sizes="180x180" href="https://bmcevolbiol.biomedcentral.com/static/images/favicons/bmc/app-icon-180x180-230f795aee.png">
<link rel="icon" type="image/png" sizes="16x16" href="https://bmcevolbiol.biomedcentral.com/static/images/favicons/bmc/favicon-16x16-25a0239eec.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://bmcevolbiol.biomedcentral.com/static/images/favicons/bmc/favicon-32x32-637712e609.png">
<link rel="icon" type="image/png" sizes="96x96" href="https://bmcevolbiol.biomedcentral.com/static/images/favicons/bmc/favicon-96x96-9f5dc535c7.png">
<link rel="icon" type="image/png" sizes="194x194" href="https://bmcevolbiol.biomedcentral.com/static/images/favicons/bmc/favicon-194x194-d2a2617de2.png">
<meta name="msapplication-TileColor" content="#e6e6e6">
<meta name="msapplication-TileImage" content="/static/images/favicons/bmc/app-icon-144x144-67f5b60136.png">
<meta name="theme-color" content="#1b3051">

        <script type="text/javascript" src="https://cdn.krxd.net/userdata/get?pub=bd339c69-af54-4a21-b4f1-654bcfcd83ca&amp;technographics=1&amp;callback=Krux.ns.macmillan.kxjsonp_userdata"></script><script type="text/javascript" src="https://beacon.krxd.net/optout_check?callback=Krux.ns.macmillan.kxjsonp_optOutCheck"></script><script type="text/javascript" src="bd339c69-af54-4a21-b4f1-654bcfcd83ca_002"></script><script type="text/javascript" src="bd339c69-af54-4a21-b4f1-654bcfcd83ca"></script><script async="" src="controltag.js"></script><script type="text/javascript" async="" src="MathJax.js"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="KDqylSLE.js"></script><script async="" src="gtm.js"></script><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        
    <link rel="stylesheet" media="screen" href="core-article-3523ab3d1c.css">


<link rel="stylesheet" media="screen" href="core-a9ea426f56.css">
<link rel="stylesheet" media="print" href="print-7680b1a80b.css">

<link rel="stylesheet" id="js-mustard" href="enhanced-e3b1840436.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">


    <link rel="stylesheet" href="enhanced-article-d0bd7b81f9.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">


        

        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>

        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'bmcevolbiol.biomedcentral.com',
            siteWithPath: 'bmcevolbiol.biomedcentral.com' + window.location.pathname,
            twitterHashtag: '',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/1471-2148-10-210',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            hasFigshareInvoked: false,
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script>

        <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script>

        <!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js-->
<script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script> 
        
    <script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/1471-2148-10-210","articleType":"Software","peerReviewType":"Closed","supplement":null,"keywords":"Multiple Sequence Alignment;Phylogenetic Inference;Initial Alignment;Universal Genetic Code;False Branch"},"contentInfo":{"imprint":"BioMed Central","title":"BMGE (Block Mapping and Gathering with Entropy): a new software for selection of phylogenetic informative regions from multiple sequence alignments","publishedAt":1278979200000,"publishedAtDate":"2010-07-13","author":["Alexis Criscuolo","Simonetta Gribaldo"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":"CC BY","openAccess":true},"environment":"live"},"journal":{"siteKey":"bmcevolbiol.biomedcentral.com","volume":"10","issue":"1","title":"BMC Evolutionary Biology","type":"SERIES","journalID":12862,"gaCode":"UA-62065396-10","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Software","publishingSegment":"BMC Series - Life and Health"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"},"attributes":{"featureFlags":[{"name":"ab_test_bmc_shared_article_renderer","active":true}]}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script id="springerlink-event-tracker" src="eventTracker.asc"></script>


        <script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script>
        <script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    d.className += ' webfonts-loaded';
                }
            })(document.documentElement);
        </script>
        <script src="7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><link type="text/css" href="optanon.css" rel="stylesheet"><style>#optanon ul#optanon-menu li { background-color: #F8F8F8 !important }#optanon ul#optanon-menu li.menu-item-selected { background-color: #C5F6F9 !important }#optanon #optanon-popup-wrapper .optanon-white-button-middle { background-color: #1B3051 !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle { background-color: #1B3051 !important; border-color: #1B3051 !important; }#optanon #optanon-popup-wrapper .optanon-white-button-middle button { color: #ffffff !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle button { color: #ffffff !important }#optanon #optanon-popup-bottom { background-color: #FFFFFF !important }#optanon.modern #optanon-popup-top, #optanon.modern #optanon-popup-body-left-shading { background-color: #FFFFFF !important }.optanon-alert-box-wrapper { background-color:#EFF6FC !important }.optanon-alert-box-wrapper .optanon-alert-box-bg p { color:#333333 !important }</style><script type="text/javascript" src="jquery-3.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
        
    
        
    

        
    
        
            <!-- Google Tag Manager -->
            <script>
                (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
            </script>
            <!-- End Google Tag Manager -->
        
    


        
    
    <link rel="canonical" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210">
    

        
        
        <meta property="og:url" content="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="BMC Evolutionary Biology">
        <meta property="og:title" content="BMGE (Block Mapping and Gathering with Entropy): a new software for selection of phylogenetic informative regions from multiple sequence alignments">
        <meta property="og:description" content="The quality of multiple sequence alignments plays an important role in the accuracy of phylogenetic inference. It has been shown that removing ambiguously aligned regions, but also other sources of bias such as highly variable (saturated) characters, can improve the overall performance of many phylogenetic reconstruction methods. A current scientific trend is to build phylogenetic trees from a large number of sequence datasets (semi-)automatically extracted from numerous complete genomes. Because these approaches do not allow a precise manual curation of each dataset, there exists a real need for efficient bioinformatic tools dedicated to this alignment character trimming step. Here is presented a new software, named BMGE (Block Mapping and Gathering with Entropy), that is designed to select regions in a multiple sequence alignment that are suited for phylogenetic inference. For each character, BMGE computes a score closely related to an entropy value. Calculation of these entropy-like scores is weighted with BLOSUM or PAM similarity matrices in order to distinguish among biologically expected and unexpected variability for each aligned character. Sets of contiguous characters with a score above a given threshold are considered as not suited for phylogenetic inference and then removed. Simulation analyses show that the character trimming performed by BMGE produces datasets leading to accurate trees, especially with alignments including distantly-related sequences. BMGE also implements trimming and recoding methods aimed at minimizing phylogeny reconstruction artefacts due to compositional heterogeneity. BMGE is able to perform biologically relevant trimming on a multiple alignment of DNA, codon or amino acid sequences. Java source code and executable are freely available at ftp://ftp.pasteur.fr/pub/GenSoft/projects/BMGE/ .">
        <meta property="og:image" content="https://media.springernature.com/w110/springer-static/cover/journal/12862.jpg">
    
        
    <script async="" src="core-bmc.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019073101.js" async=""></script><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 0; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax.MathJax_FullWidth {text-align: center; display: table-cell!important; width: 10000em!important}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none; box-sizing: content-box}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_test {font-style: normal; font-weight: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-transform: none; letter-spacing: normal; word-spacing: normal; overflow: hidden; height: 1px}
.MathJax_test.mjx-test-display {display: table!important}
.MathJax_test.mjx-test-inline {display: inline!important; margin-right: -1px}
.MathJax_test.mjx-test-default {display: block!important; clear: both}
.MathJax_ex_box {display: inline-block!important; position: absolute; overflow: hidden; min-height: 0; max-height: none; padding: 0; border: 0; margin: 0; width: 1px; height: 60ex}
.MathJax_em_box {display: inline-block!important; position: absolute; overflow: hidden; min-height: 0; max-height: none; padding: 0; border: 0; margin: 0; width: 1px; height: 60em}
.mjx-test-inline .MathJax_left_box {display: inline-block; width: 0; float: left}
.mjx-test-inline .MathJax_right_box {display: inline-block; width: 0; float: right}
.mjx-test-display .MathJax_right_box {display: table-cell!important; width: 10000em!important; min-width: 0; max-width: none; padding: 0; border: 0; margin: 0}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Blank; src: url('about:blank')}
#MathJax_Message {margin: 0}
.MathJax .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>

    <body class="journal journal-fulltext"><div class="optanon-alert-box-wrapper  " role="alertdialog" aria-labelledby="alert-box-title" aria-describedby="alert-box-message" style="bottom: -144.827px;"><div class="optanon-alert-box-bottom-top"><div class="optanon-alert-box-corner-close"><button class="optanon-alert-box-close banner-close-button" aria-label="Close" title="Close" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Banner Close Button');" tabindex="4"></button></div></div><div class="optanon-alert-box-bg"><div class="optanon-alert-box-logo"> </div><div class="optanon-alert-box-body"><p class="optanon-alert-box-title legacy-banner-title sr-only" id="alert-box-title" role="heading" aria-level="2">Cookie Notice</p><p class="banner-content" id="alert-box-message">We
 use cookies to personalise content and ads, to provide social media 
features and to analyse our traffic. We also share information about 
your use of our site with our social media, advertising and analytics 
partners in accordance with our <a href="https://www.biomedcentral.com/privacy-statement" class="banner-policy-link" tabindex="1" aria-label="Privacy Statement">Privacy Statement</a>. You can manage your preferences in 'Manage Cookies'.</p></div><div class="optanon-clearfix"></div><div class="optanon-alert-box-button-container"><div class="optanon-alert-box-button optanon-button-close"><div class="optanon-alert-box-button-middle"><button class="optanon-alert-box-close" aria-label="Close">Close</button></div></div><div class="optanon-alert-box-button optanon-button-allow"><div class="optanon-alert-box-button-middle accept-cookie-container"><button class="optanon-allow-all accept-cookies-button" title="OK" aria-label="OK" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Banner Accept Cookies');" tabindex="3">OK</button></div></div><div class="optanon-alert-box-button optanon-button-more"><div class="optanon-alert-box-button-middle"><button class="optanon-toggle-display cookie-settings-button" title="Manage Cookies" aria-label="Manage Cookies" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Banner Open Preferences');" tabindex="2">Manage Cookies</button></div></div></div><div class="optanon-clearfix optanon-alert-box-bottom-padding"></div></div></div><div id="optanon" class="modern"><div id="optanon-popup-bg"></div><div id="optanon-popup-wrapper" role="dialog" aria-modal="true" tabindex="-1" lang="en-GB"><div id="optanon-popup-top"><button onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Close Button');" aria-label="Close" class="optanon-close-link optanon-close optanon-close-ui" title="Close"><div id="optanon-close" style="background: url(https://cdn.cookielaw.org/skins/4.9.0/default_flat_bottom_two_button_white/v2/images/optanon-pop-up-close.png);width:34px;height:34px;"></div></button></div><div id="optanon-popup-body"><div id="optanon-popup-body-left"><div id="optanon-popup-body-left-shading"></div><div id="optanon-branding-top-logo" style="background-image: url('https://cdn.cookielaw.org/logos/5138/5138:biomedcentral.com/bmc-logo.png') !important;"></div><ul id="optanon-menu" aria-label="Navigation Menu" role="tablist"><li class="menu-item-on menu-item-about" title="Your Privacy"><p class="preference-menu-item"><button role="tab" aria-selected="true" aria-controls="tab-0" id="Your-Privacy">Your Privacy</button></p></li><li class="menu-item-necessary menu-item-on" title="Strictly Necessary Cookies"><p class="preference-menu-item"><button role="tab" aria-selected="false" aria-controls="tab-2" id="Strictly-Necessary">Strictly Necessary Cookies</button></p></li><li class="menu-item-on menu-item-performance" title="Performance &amp; Analytics Cookies"><p class="preference-menu-item"><button role="tab" aria-selected="false" aria-controls="tab-4" id="Performance-&amp;">Performance &amp; Analytics Cookies</button></p></li><li class="menu-item-on menu-item-functional" title="Functional Cookies"><p class="preference-menu-item"><button role="tab" aria-selected="false" aria-controls="tab-8" id="Functional-Cookies">Functional Cookies</button></p></li><li class="menu-item-on menu-item-advertising" title="Targeting Cookies"><p class="preference-menu-item"><button role="tab" aria-selected="false" aria-controls="tab-10" id="Targeting-Cookies">Targeting Cookies</button></p></li><li class="menu-item-moreinfo menu-item-off" title="More Information"><p class="preference-menu-item"><a target="_blank" aria-label="More Information" href="https://www.biomedcentral.com/cookies" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Cookie Policy');">More Information</a></p></li></ul></div><div id="optanon-popup-body-right"><p role="heading" aria-level="2" class="legacy-preference-banner-title h2" aria-label="Privacy Preference Centre">Privacy Preference Centre</p><div class="vendor-header-container"><p class="header-3" role="heading" aria-level="3"></p><div id="optanon-popup-more-info-bar"><div class="optanon-status"><div class="optanon-status-editable"><form><span class="fieldset"><p><input value="check" id="chkMain" checked="checked" class="legacy-group-status optanon-status-checkbox" type="checkbox"><label for="chkMain">Active</label></p></span></form></div><div class="optanon-status-always-active optanon-status-on"><p>Always Active</p></div></div></div></div><div class="optanon-main-info-text" role="tabpanel"></div></div><div class="optanon-bottom-spacer"></div></div><div id="optanon-popup-bottom"> <a href="https://onetrust.com/poweredbyonetrust" target="_blank"><div id="optanon-popup-bottom-logo" style="background: url(https://cdn.cookielaw.org/skins/4.9.0/default_flat_bottom_two_button_white/v2/images/cookie-collective-top-bottom.png);width:155px;height:35px;" title="powered by OneTrust"></div></a><div class="optanon-button-wrapper optanon-save-settings-button optanon-close optanon-close-consent"><div class="optanon-white-button-left"></div><div class="optanon-white-button-middle"><button title="Save Settings" aria-label="Save Settings" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Save Settings');">Save Settings</button></div><div class="optanon-white-button-right"></div></div><div class="optanon-button-wrapper optanon-allow-all-button optanon-allow-all" style="display: none;"><div class="optanon-white-button-left"></div><div class="optanon-white-button-middle"><button title="Allow All" aria-label="Allow All" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Allow All');">Allow All</button></div><div class="optanon-white-button-right"></div></div></div></div></div><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="">Typesetting math: 100%</div>
        
    
        
            <!-- Google Tag Manager (noscript) -->
            <noscript>
                <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                        height="0" width="0" style="display:none;visibility:hidden"></iframe>
            </noscript>
            <!-- End Google Tag Manager (noscript) -->
        
    

        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-alert" viewBox="0 0 39 30"><path d="M1.625 23.333h22.75v-.286a1.69 1.69 0 0 0-.476-1.179l-2.774-2.845v-5.69C21.125 8.731 17.487 5 13 5s-8.125 3.731-8.125 8.333v5h4.063c.448 0 .812.373.812.834 0 .46-.364.833-.813.833H3.923L2.1 21.868a1.688 1.688 0 0 0-.476 1.179v.286zM9.75 25c0 1.84 1.455 3.333 3.25 3.333s3.25-1.492 3.25-3.333h-6.5zm-8.125 0C.728 25 0 24.254 0 23.333v-.286c0-.884.343-1.732.952-2.357l2.298-2.357v-5c0-3.572 1.858-6.874 4.875-8.66a9.54 9.54 0 0 1 9.75 0c3.017 1.786 4.875 5.088 4.875 8.66v5l2.298 2.357c.61.625.952 1.473.952 2.357v.286c0 .92-.728 1.667-1.625 1.667h-6.5c0 2.761-2.183 5-4.875 5s-4.875-2.239-4.875-5h-6.5zM14.62 3.47a1.68 1.68 0 0 0-.768-1.559 1.59 1.59 0 0 0-1.703 0 1.68 1.68 0 0 0-.767 1.559 9.56 9.56 0 0 0-1.588.417A3.369 3.369 0 0 1 11.22.546a3.182 3.182 0 0 1 3.556 0 3.369 3.369 0 0 1 1.428 3.34 9.486 9.486 0 0 0-1.585-.416z"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-article" viewBox="0 0 39 26"><path d="M23.333 22.405c0 .96.667 1.6 1.667 1.6h1.667c.833 0 1.666-.8 1.666-1.6V8.002h-5v14.403zm-1.666 0V1.6h-20v20.805c0 .8.666 1.6 1.666 1.6h18.834c-.334-.48-.5-.96-.5-1.6zM0 22.405V1.6C0 .64.833 0 1.667 0h20c1 0 1.666.8 1.666 1.6v4.802H30v16.003c0 1.76-1.5 3.201-3.333 3.201H3.333C1.5 25.606 0 24.166 0 22.406zM6.667 6.402v3.2h10v-3.2h-10zM5 4.802h13.333v6.4H5v-6.4zm0 8.001h13.333v1.6H5v-1.6zm0 3.2h13.333v1.601H5v-1.6zm0 3.202h13.333v1.6H5v-1.6z"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 6 9"><path d="M1.14.854a.515.515 0 0 1 0-.708.467.467 0 0 1 .68 0l3.847 4.008a.5.5 0 0 1 0 .692L1.82 8.854a.467.467 0 0 1-.68 0 .515.515 0 0 1 0-.708L4.643 4.5 1.141.854z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><g filter="url(#a)"><path fill="#FFF" d="M0 31.01h60.643V0H0z"></path></g></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download-rounded"><path d="M0 13c0-.556.449-1 1.002-1h9.996a.999.999 0 1 1 0 2H1.002A1.006 1.006 0 0 1 0 13zM7 1v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 0 1 0 1.417l-4.177 4.177a1.001 1.001 0 0 1-1.416 0L1.115 6.715a.991.991 0 0 1-.016-1.4 1 1 0 0 1 1.42.003L5 7.8V1c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-editor" viewBox="0 0 39 30"><path d="M17 28.167c2.167-.334 4.167-1.167 5.833-2.5-1.5-4.167-3.5-6.5-6.333-7.167v-2.167c2-.666 3.333-2.5 3.333-4.666 0-2.834-2.166-5-5-5-2.833 0-4.833 2.166-4.833 5 0 2.166 1.333 4 3.333 4.666V18.5C10.5 19.167 8.5 21.5 7 25.667c1.667 1.333 3.667 2.166 5.833 2.5.834.166 1.5.166 2.167.166.667 0 1.333 0 2-.166zm7.333-3.667c2.5-2.333 4-5.833 4-9.5 0-7.333-6-13.333-13.333-13.333S1.667 7.667 1.667 15c0 3.667 1.5 7.167 4 9.5 1.333-3.333 3-6 5.833-7.167C9.667 16.167 8.333 14 8.333 11.667 8.333 8 11.333 5 15 5s6.667 3 6.667 6.667c0 2.333-1.334 4.5-3.167 5.666 2.667 1.167 4.5 3.834 5.833 7.167zM15 30C6.667 30 0 23.333 0 15S6.667 0 15 0s15 6.667 15 15-6.667 15-15 15z"></path></symbol><symbol id="icon-ethics" viewBox="0 0 39 32"><path d="M12.182 2.272l1.038-1.038a4.014 4.014 0 0 1 5.677 0l1.038 1.038a3.21 3.21 0 0 0 2.271.941h2.684a4.014 4.014 0 0 1 4.014 4.014v2.684c0 .852.339 1.669.94 2.271l1.039 1.038a4.014 4.014 0 0 1 0 5.677l-1.038 1.038a3.211 3.211 0 0 0-.94 2.271v2.684a4.014 4.014 0 0 1-4.015 4.014h-2.684c-.852 0-1.669.339-2.27.94l-1.039 1.039a4.014 4.014 0 0 1-5.677 0l-1.038-1.038a3.211 3.211 0 0 0-2.27-.94H7.226a4.014 4.014 0 0 1-4.014-4.015v-2.684c0-.852-.338-1.669-.94-2.27l-1.039-1.039a4.014 4.014 0 0 1 0-5.677l1.038-1.038a3.208 3.208 0 0 0 .941-2.27V7.226a4.014 4.014 0 0 1 4.014-4.014H9.91c.852 0 1.669-.338 2.271-.94zm1.136 1.136a4.817 4.817 0 0 1-3.407 1.41H7.227a2.409 2.409 0 0 0-2.408 2.41V9.91a4.817 4.817 0 0 1-1.411 3.407L2.37 14.356a2.41 2.41 0 0 0 0 3.406L3.408 18.8a4.817 4.817 0 0 1 1.41 3.406v2.684a2.409 2.409 0 0 0 2.41 2.409H9.91a4.82 4.82 0 0 1 3.407 1.41l1.038 1.038c.94.941 2.465.941 3.406 0L18.8 28.71a4.817 4.817 0 0 1 3.406-1.41h2.684a2.409 2.409 0 0 0 2.409-2.409v-2.684c0-1.278.507-2.503 1.41-3.406l1.038-1.038a2.408 2.408 0 0 0 0-3.406l-1.038-1.038A4.817 4.817 0 0 1 27.3 9.91V7.227a2.409 2.409 0 0 0-2.41-2.407h-2.684A4.817 4.817 0 0 1 18.8 3.408L17.762 2.37a2.409 2.409 0 0 0-3.406 0l-1.038 1.038zM15.03 17.86l4.332-4.73a.803.803 0 1 1 1.224 1.04l-4.844 5.367a.803.803 0 0 1-1.112.076l-3.1-2.605a.803.803 0 0 1 1.027-1.233l2.473 2.085zm1.028 7.832c-.41 0-.41-1.482 0-1.482a8.152 8.152 0 0 0 6.657-12.858.741.741 0 1 1 1.21-.857 9.634 9.634 0 0 1-7.867 15.197zM21.68 8.234a.741.741 0 0 1-.866 1.203 8.152 8.152 0 0 0-12.908 6.578.741.741 0 0 1-1.483-.007A9.635 9.635 0 0 1 21.68 8.234zM6.907 19.077a.741.741 0 0 1 1.407-.464 8.155 8.155 0 0 0 7.745 5.598.741.741 0 1 1 0 1.482 9.637 9.637 0 0 1-9.152-6.616z"></path></symbol><symbol id="icon-explore" viewBox="0 0 39 30"><path d="M15 28.333c7.333 0 13.333-6 13.333-13.333S22.333 1.667 15 1.667 1.667 7.667 1.667 15s6 13.333 13.333 13.333zM15 30C6.667 30 0 23.333 0 15S6.667 0 15 0s15 6.667 15 15-6.667 15-15 15zm0-4.167c-.5 0-.833-.333-.833-.833 0-.5.333-.833.833-.833 5 0 9-4 9.167-8.834 0-.5.333-.833.833-.833.5 0 .833.333.833.833-.166 5.834-5 10.5-10.833 10.5zm-.5-21.666c.5 0 .833.333.833.833 0 .5-.333.833-.666.833C9.667 6 5.833 10.167 5.833 15c0 .5-.333.833-.833.833-.5 0-.833-.333-.833-.833 0-5.833 4.5-10.5 10.333-10.833zm1.833 13.5l-2-2a.806.806 0 0 1 0-1.167.806.806 0 0 1 1.167 0l2 2 2.667-6.667-7.5 2.834-3 7.5 6.666-2.5zm6.834-10.834L18.667 18c-.167.5-.5.833-1 1L6.5 23.5 11 12.333c.333-.666.667-1 1.167-1.166l11-4.334z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.6 10.6c2.2 0 4-1.7 4-4s-1.7-4-4-4-4 1.7-4 4 1.8 4 4 4zm4 1.4l1.5-1.5 3.7 3.6c.4.5.4 1.2 0 1.6-.4.4-1.1.4-1.6 0L10.6 12zm-4 1.2C2.9 13.2 0 10.3 0 6.6S2.9 0 6.6 0s6.6 2.9 6.6 6.6-2.9 6.6-6.6 6.6z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-alert" viewBox="0 0 39 30"><path d="M1.625 23.333h22.75v-.286a1.69 1.69 0 0 0-.476-1.179l-2.774-2.845v-5.69C21.125 8.731 17.487 5 13 5s-8.125 3.731-8.125 8.333v5h4.063c.448 0 .812.373.812.834 0 .46-.364.833-.813.833H3.923L2.1 21.868a1.688 1.688 0 0 0-.476 1.179v.286zM9.75 25c0 1.84 1.455 3.333 3.25 3.333s3.25-1.492 3.25-3.333h-6.5zm-8.125 0C.728 25 0 24.254 0 23.333v-.286c0-.884.343-1.732.952-2.357l2.298-2.357v-5c0-3.572 1.858-6.874 4.875-8.66a9.54 9.54 0 0 1 9.75 0c3.017 1.786 4.875 5.088 4.875 8.66v5l2.298 2.357c.61.625.952 1.473.952 2.357v.286c0 .92-.728 1.667-1.625 1.667h-6.5c0 2.761-2.183 5-4.875 5s-4.875-2.239-4.875-5h-6.5zM14.62 3.47a1.68 1.68 0 0 0-.768-1.559 1.59 1.59 0 0 0-1.703 0 1.68 1.68 0 0 0-.767 1.559 9.56 9.56 0 0 0-1.588.417A3.369 3.369 0 0 1 11.22.546a3.182 3.182 0 0 1 3.556 0 3.369 3.369 0 0 1 1.428 3.34 9.486 9.486 0 0 0-1.585-.416z"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-article" viewBox="0 0 39 26"><path d="M23.333 22.405c0 .96.667 1.6 1.667 1.6h1.667c.833 0 1.666-.8 1.666-1.6V8.002h-5v14.403zm-1.666 0V1.6h-20v20.805c0 .8.666 1.6 1.666 1.6h18.834c-.334-.48-.5-.96-.5-1.6zM0 22.405V1.6C0 .64.833 0 1.667 0h20c1 0 1.666.8 1.666 1.6v4.802H30v16.003c0 1.76-1.5 3.201-3.333 3.201H3.333C1.5 25.606 0 24.166 0 22.406zM6.667 6.402v3.2h10v-3.2h-10zM5 4.802h13.333v6.4H5v-6.4zm0 8.001h13.333v1.6H5v-1.6zm0 3.2h13.333v1.601H5v-1.6zm0 3.202h13.333v1.6H5v-1.6z"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 6 9"><path d="M1.14.854a.515.515 0 0 1 0-.708.467.467 0 0 1 .68 0l3.847 4.008a.5.5 0 0 1 0 .692L1.82 8.854a.467.467 0 0 1-.68 0 .515.515 0 0 1 0-.708L4.643 4.5 1.141.854z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><g filter="url(#a)"><path fill="#FFF" d="M0 31.01h60.643V0H0z"></path></g></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download-rounded"><path d="M0 13c0-.556.449-1 1.002-1h9.996a.999.999 0 1 1 0 2H1.002A1.006 1.006 0 0 1 0 13zM7 1v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 0 1 0 1.417l-4.177 4.177a1.001 1.001 0 0 1-1.416 0L1.115 6.715a.991.991 0 0 1-.016-1.4 1 1 0 0 1 1.42.003L5 7.8V1c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-editor" viewBox="0 0 39 30"><path d="M17 28.167c2.167-.334 4.167-1.167 5.833-2.5-1.5-4.167-3.5-6.5-6.333-7.167v-2.167c2-.666 3.333-2.5 3.333-4.666 0-2.834-2.166-5-5-5-2.833 0-4.833 2.166-4.833 5 0 2.166 1.333 4 3.333 4.666V18.5C10.5 19.167 8.5 21.5 7 25.667c1.667 1.333 3.667 2.166 5.833 2.5.834.166 1.5.166 2.167.166.667 0 1.333 0 2-.166zm7.333-3.667c2.5-2.333 4-5.833 4-9.5 0-7.333-6-13.333-13.333-13.333S1.667 7.667 1.667 15c0 3.667 1.5 7.167 4 9.5 1.333-3.333 3-6 5.833-7.167C9.667 16.167 8.333 14 8.333 11.667 8.333 8 11.333 5 15 5s6.667 3 6.667 6.667c0 2.333-1.334 4.5-3.167 5.666 2.667 1.167 4.5 3.834 5.833 7.167zM15 30C6.667 30 0 23.333 0 15S6.667 0 15 0s15 6.667 15 15-6.667 15-15 15z"></path></symbol><symbol id="icon-ethics" viewBox="0 0 39 32"><path d="M12.182 2.272l1.038-1.038a4.014 4.014 0 0 1 5.677 0l1.038 1.038a3.21 3.21 0 0 0 2.271.941h2.684a4.014 4.014 0 0 1 4.014 4.014v2.684c0 .852.339 1.669.94 2.271l1.039 1.038a4.014 4.014 0 0 1 0 5.677l-1.038 1.038a3.211 3.211 0 0 0-.94 2.271v2.684a4.014 4.014 0 0 1-4.015 4.014h-2.684c-.852 0-1.669.339-2.27.94l-1.039 1.039a4.014 4.014 0 0 1-5.677 0l-1.038-1.038a3.211 3.211 0 0 0-2.27-.94H7.226a4.014 4.014 0 0 1-4.014-4.015v-2.684c0-.852-.338-1.669-.94-2.27l-1.039-1.039a4.014 4.014 0 0 1 0-5.677l1.038-1.038a3.208 3.208 0 0 0 .941-2.27V7.226a4.014 4.014 0 0 1 4.014-4.014H9.91c.852 0 1.669-.338 2.271-.94zm1.136 1.136a4.817 4.817 0 0 1-3.407 1.41H7.227a2.409 2.409 0 0 0-2.408 2.41V9.91a4.817 4.817 0 0 1-1.411 3.407L2.37 14.356a2.41 2.41 0 0 0 0 3.406L3.408 18.8a4.817 4.817 0 0 1 1.41 3.406v2.684a2.409 2.409 0 0 0 2.41 2.409H9.91a4.82 4.82 0 0 1 3.407 1.41l1.038 1.038c.94.941 2.465.941 3.406 0L18.8 28.71a4.817 4.817 0 0 1 3.406-1.41h2.684a2.409 2.409 0 0 0 2.409-2.409v-2.684c0-1.278.507-2.503 1.41-3.406l1.038-1.038a2.408 2.408 0 0 0 0-3.406l-1.038-1.038A4.817 4.817 0 0 1 27.3 9.91V7.227a2.409 2.409 0 0 0-2.41-2.407h-2.684A4.817 4.817 0 0 1 18.8 3.408L17.762 2.37a2.409 2.409 0 0 0-3.406 0l-1.038 1.038zM15.03 17.86l4.332-4.73a.803.803 0 1 1 1.224 1.04l-4.844 5.367a.803.803 0 0 1-1.112.076l-3.1-2.605a.803.803 0 0 1 1.027-1.233l2.473 2.085zm1.028 7.832c-.41 0-.41-1.482 0-1.482a8.152 8.152 0 0 0 6.657-12.858.741.741 0 1 1 1.21-.857 9.634 9.634 0 0 1-7.867 15.197zM21.68 8.234a.741.741 0 0 1-.866 1.203 8.152 8.152 0 0 0-12.908 6.578.741.741 0 0 1-1.483-.007A9.635 9.635 0 0 1 21.68 8.234zM6.907 19.077a.741.741 0 0 1 1.407-.464 8.155 8.155 0 0 0 7.745 5.598.741.741 0 1 1 0 1.482 9.637 9.637 0 0 1-9.152-6.616z"></path></symbol><symbol id="icon-explore" viewBox="0 0 39 30"><path d="M15 28.333c7.333 0 13.333-6 13.333-13.333S22.333 1.667 15 1.667 1.667 7.667 1.667 15s6 13.333 13.333 13.333zM15 30C6.667 30 0 23.333 0 15S6.667 0 15 0s15 6.667 15 15-6.667 15-15 15zm0-4.167c-.5 0-.833-.333-.833-.833 0-.5.333-.833.833-.833 5 0 9-4 9.167-8.834 0-.5.333-.833.833-.833.5 0 .833.333.833.833-.166 5.834-5 10.5-10.833 10.5zm-.5-21.666c.5 0 .833.333.833.833 0 .5-.333.833-.666.833C9.667 6 5.833 10.167 5.833 15c0 .5-.333.833-.833.833-.5 0-.833-.333-.833-.833 0-5.833 4.5-10.5 10.333-10.833zm1.833 13.5l-2-2a.806.806 0 0 1 0-1.167.806.806 0 0 1 1.167 0l2 2 2.667-6.667-7.5 2.834-3 7.5 6.666-2.5zm6.834-10.834L18.667 18c-.167.5-.5.833-1 1L6.5 23.5 11 12.333c.333-.666.667-1 1.167-1.166l11-4.334z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.6 10.6c2.2 0 4-1.7 4-4s-1.7-4-4-4-4 1.7-4 4 1.8 4 4 4zm4 1.4l1.5-1.5 3.7 3.6c.4.5.4 1.2 0 1.6-.4.4-1.1.4-1.6 0L10.6 12zm-4 1.2C2.9 13.2 0 10.3 0 6.6S2.9 0 6.6 0s6.6 2.9 6.6 6.6-2.9 6.6-6.6 6.6z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner">Skip to main content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/bmcevolbiol/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/1471-2148-10-210;kwrd=Multiple Sequence Alignment,Phylogenetic Inference,Initial Alignment,Universal Genetic Code,False Branch;pmc=L21001,L2504X,L25090,M31010,L00004;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/bmcevolbiol/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/1471-2148-10-210&amp;kwrd=Multiple Sequence Alignment,Phylogenetic Inference,Initial Alignment,Universal Genetic Code,False Branch&amp;pmc=L21001,L2504X,L25090,M31010,L00004&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/bmcevolbiol/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/1471-2148-10-210&amp;kwrd=Multiple Sequence Alignment,Phylogenetic Inference,Initial Alignment,Universal Genetic Code,False Branch&amp;pmc=L21001,L2504X,L25090,M31010,L00004&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            <div id="google_ads_iframe_/270604982/bmc/bmcevolbiol/articles_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
        </div>
    </div>

        
         
    
        <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>
    
    
        <header class="c-popup-search" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="c-navbar__container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope="" itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
                <div class="c-logo">
    <a href="https://www.biomedcentral.com/" itemprop="url">
        <img alt="BMC" itemprop="logo" role="img" src="logo-bmc-white-series-1da6525f4e.svg" width="76" height="18">
        <div class="c-logo__strapline">
            <img alt="Part of Springer Nature" role="img" src="logo-bmc-white-strapline-sn-e7b1d6e67f.svg" width="173" height="16">
        </div>
    </a>
</div>

            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hide-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav c-navbar__nav--right">
                    <li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hide-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="u-flex u-flex-align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true" focusable="false">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-show">Login</span>
                            <span class="u-js-hide">My Account</span>
                        </a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label="" data-track-label="" data-test="global-search" novalidate="">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="u-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text">
            <div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true" focusable="false">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


            </div>
        </div>
        <input name="searchType" value="publisherSearch" type="hidden">
    </form>
</div>

            </div>
        </div>
    </div>
</header>

    

        
            <header class="c-journal-header c-journal-header--bmc-evolutionary-biology ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="https://bmcevolbiol.biomedcentral.com/">
    
        
    
    <span class="c-journal-title__text ">BMC Evolutionary Biology</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="c-navbar__container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hide-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcevolbiol.biomedcentral.com/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcevolbiol.biomedcentral.com/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="https://bmcevolbiol.biomedcentral.com/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcevolbiol.biomedcentral.com/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--default">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl c-article-container" id="main-content" data-component="article-container">
            <div class="c-page-layout c-page-layout--article" data-component="sticky-container">

                <main class="c-page-layout__main">
                    <article itemscope="" itemtype="http://schema.org/ScholarlyArticle" lang="en">
                        

                        <ul class="c-article-identifiers" data-test="article-identifier">
                            
    
        <li class="c-article-identifiers__item">Software</li>
        <li class="c-article-identifiers__item">
            <abbr title="Open Access">Open</abbr>
            
        </li>
        
    

                            <li class="c-article-identifiers__item"><a href="#article-info">Published: <time datetime="2010-07-13" itemprop="datePublished">13 July 2010</time></a></li>
                        </ul>

                        <h1 class="c-article-title u-h1" data-test="article-title" data-article-title="" itemprop="name headline">BMGE
 (Block Mapping and Gathering with Entropy): a new software for 
selection of phylogenetic informative regions from multiple sequence 
alignments</h1>
                        <ul class="c-author-list js-list-authors js-etal-collapsed" data-etal="25" data-etal-small="3" data-test="authors-list"><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-1" class="c-author-list__item-icon js-no-scroll" data-corresp-id="c1">Alexis Criscuolo</a></span><sup class="u-js-hide"><a href="#Aff1" class="js-no-scroll" tabindex="-1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Département de Microbiologie"><meta itemprop="address" content="grid.428999.7, 0000 0001 2353 6535, Institut Pasteur, Unité de Biologie Moléculaire du Gène chez les Extrêmophiles, Département de Microbiologie, 25 rue du Dr Roux, 75015, Paris, France"></span></sup> &amp; </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-2" class="js-no-scroll">Simonetta Gribaldo</a></span><sup class="u-js-hide"><a href="#Aff1" class="js-no-scroll" tabindex="-1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Département de Microbiologie"><meta itemprop="address" content="grid.428999.7, 0000 0001 2353 6535, Institut Pasteur, Unité de Biologie Moléculaire du Gène chez les Extrêmophiles, Département de Microbiologie, 25 rue du Dr Roux, 75015, Paris, France"></span></sup>&nbsp;</li></ul>
                        <p class="c-article-info-details" data-container-section="info"><i data-test="journal-title">BMC Evolutionary Biology</i><b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;10</b>, Article&nbsp;number:&nbsp;<span data-test="article-number">210</span> (<span data-test="article-publication-year">2010</span>)
                <span class="c-article-info-details__divider u-hide-print"> | </span><a class="c-article-info-details__download-citation u-hide-print" data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210.ris">
                Download Citation
            </a></p>

                        

                        


                        <section aria-labelledby="Abs1" lang="en"><div class="c-article-section js-article-section" id="Abs1-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content js-collapsible-section" id="Abs1-content"><h3 class="c-article__sub-heading u-h3" data-test="abstract-sub-heading">Background</h3><p>The
 quality of multiple sequence alignments plays an important role in the 
accuracy of phylogenetic inference. It has been shown that removing 
ambiguously aligned regions, but also other sources of bias such as 
highly variable (saturated) characters, can improve the overall 
performance of many phylogenetic reconstruction methods. A current 
scientific trend is to build phylogenetic trees from a large number of 
sequence datasets (semi-)automatically extracted from numerous complete 
genomes. Because these approaches do not allow a precise manual curation
 of each dataset, there exists a real need for efficient bioinformatic 
tools dedicated to this alignment character trimming step.</p><h3 class="c-article__sub-heading u-h3" data-test="abstract-sub-heading">Results</h3><p>Here
 is presented a new software, named BMGE (Block Mapping and Gathering 
with Entropy), that is designed to select regions in a multiple sequence
 alignment that are suited for phylogenetic inference. For each 
character, BMGE computes a score closely related to an entropy value. 
Calculation of these entropy-like scores is weighted with BLOSUM or PAM 
similarity matrices in order to distinguish among biologically expected 
and unexpected variability for each aligned character. Sets of 
contiguous characters with a score above a given threshold are 
considered as not suited for phylogenetic inference and then removed. 
Simulation analyses show that the character trimming performed by BMGE 
produces datasets leading to accurate trees, especially with alignments 
including distantly-related sequences. BMGE also implements trimming and
 recoding methods aimed at minimizing phylogeny reconstruction artefacts
 due to compositional heterogeneity.</p><h3 class="c-article__sub-heading u-h3" data-test="abstract-sub-heading">Conclusions</h3><p>BMGE
 is able to perform biologically relevant trimming on a multiple 
alignment of DNA, codon or amino acid sequences. Java source code and 
executable are freely available at <a href="ftp://ftp.pasteur.fr/pub/GenSoft/projects/BMGE/">ftp://ftp.pasteur.fr/pub/GenSoft/projects/BMGE/</a>.</p></div></div></section>
                        
    


                        <section aria-labelledby="Sec1"><div class="c-article-section js-article-section" id="Sec1-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec1">Background</h2><div class="c-article-section__content js-collapsible-section" id="Sec1-content"><p>Most
 phylogenetic inference approaches are based on an alignment of 
homologous sequences (e.g. DNA, RNA, amino acids). The alignment of 
sequences aims at highlighting the substitutions that have occurred 
during the evolutionary process from their common ancestral sequence. 
The quality of a multiple sequence alignment can have a strong impact on
 the accuracy of the inferred phylogenetic tree, whatever the inference 
criterion used [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Lake JA: The order of sequence alignment can bias the selection of tree topology. Mol Biol Evol. 1991, 8: 378-385." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR1" id="ref-link-section-d16311e349">1</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Wang L-S, Leebens-Mack J, Wall PK, Beckmann K, dePamphilis CW, Warnow T: The impact of multiple protein sequence alignment on phylogenetic estimation. IEEE/ACM Trans Comput Biol Bioinf. 2009." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR4" id="ref-link-section-d16311e352">4</a>]. In spite of constant improvements of the multiple sequence alignment heuristics [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Edgar RC, Batzoglou S: Multiple sequence alignment. Curr Opin Struct Biol. 2006, 16: 368-373. 10.1016/j.sbi.2006.04.004." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR5" id="ref-link-section-d16311e355">5</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Notredame C: Recent evolutions of multiple sequence alignment algorithms. PLoS Comput Biol. 2007, 3: e123-10.1371/journal.pcbi.0030123." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR6" id="ref-link-section-d16311e358">6</a>], an alignment can contain regions (i.e. sets of contiguous characters, also often called blocks [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e361">7</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Talavera G, Castresana J: Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. Syst Biol. 2007, 56: 564-577. 10.1080/10635150701472164." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR8" id="ref-link-section-d16311e365">8</a>])
 where homology is ambiguous. Moreover, too divergent regions (even when
 correctly aligned) may induce a mutational saturation effect, which is 
an important source of bias for many phylogenetic reconstruction 
methods. In order to minimize the bias introduced by these problematic 
regions, a frequent approach is to detect and remove them from the 
multiple sequence alignment prior to phylogenetic analysis (e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Olsen GJ, Woese CR: Ribosomal RNA: a key to phylogeny. FASEB J. 1993, 7: 113-123." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR9" id="ref-link-section-d16311e368">9</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Huerta-Cepas J, Bueno A, Dopazo J, Gabaldón T: PhylomeDB: a database for genome-wide collections of gene phylogenies. Nucleic Acids Res. 2008, 36: D491-D496. 10.1093/nar/gkm899." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR13" id="ref-link-section-d16311e371">13</a>]). Indeed, it has been observed that the removal of such regions allows more accurate trees to be inferred [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e374">7</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Talavera G, Castresana J: Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. Syst Biol. 2007, 56: 564-577. 10.1080/10635150701472164." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR8" id="ref-link-section-d16311e377">8</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Dress AWM, Flamm C, Fritzsch G, Grünewald S, Kruspe M, Prohaska SJ, Stadler PF: Noisy: Identification of problematic columns in multiple sequence alignments. Algorithms for Molecular Biology. 2008, 3: 7-10.1186/1748-7188-3-7." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR14" id="ref-link-section-d16311e380">14</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Capella-Gutiérez S, Silla-Martínez JM, Gabaldón T: trimAl: a tool for automated alignment triming in large-scale phylogenetic analyses. Bioinformatics. 2009, 25: 1972-1973. 10.1093/bioinformatics/btp348." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR16" id="ref-link-section-d16311e384">16</a>].</p><p>A
 current trend consists in reconstructing phylogenetic trees by using a 
large number of datasets of aligned sequences from many complete 
genomes. Phylogenetic trees are then reconstructed from these datasets 
in many contexts, such as the construction of gene tree databases [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Huerta-Cepas J, Bueno A, Dopazo J, Gabaldón T: PhylomeDB: a database for genome-wide collections of gene phylogenies. Nucleic Acids Res. 2008, 36: D491-D496. 10.1093/nar/gkm899." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR13" id="ref-link-section-d16311e390">13</a>], the inference of species trees based on a core-gene set [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Daubin V, Gouy M, Perrière G: A phylogenomic approach to bacterial phylogeny: evidence of a core of genes sharing a common history. Genome Res. 2002, 12: 1080-1090. 10.1101/gr.187002." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR17" id="ref-link-section-d16311e393">17</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Ciccarelli FD, Doerks T, von Mering C, Creevey CJ, Snel B, Bork P: Toward automatic reconstruction of a highly resolved tree of life. Science. 2006, 311: 1283-1287. 10.1126/science.1123061." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR18" id="ref-link-section-d16311e396">18</a>] or the estimation of amino acid substitution matrices [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Adachi J, Waddell PJ, Martin W, Hasegawa M: Plastid genome phylogeny and a model of amino acid substitution for protein encoded by chloroplast DNA. J Mol Evol. 2000, 50: 348-358." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR19" id="ref-link-section-d16311e399">19</a>]. These different phylogenetic explorations are often based on (semi-)automated processes (e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Swingley WD, Blankenship RE, Raymond J: Integrating Markov clustering and molecular phylogenetics to reconstruct the cyanobacterial species tree from conserved protein families. Mol Biol Evol. 2008, 25: 643-654. 10.1093/molbev/msn034." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR15" id="ref-link-section-d16311e402">15</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Ciccarelli FD, Doerks T, von Mering C, Creevey CJ, Snel B, Bork P: Toward automatic reconstruction of a highly resolved tree of life. Science. 2006, 311: 1283-1287. 10.1126/science.1123061." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR18" id="ref-link-section-d16311e406">18</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="McMahon MM, Sanderson MJ: Phylogenetic supermatrix analysis of GenBank sequences from 2228 Papilionoid legumes. Syst Biol. 2006, 55: 818-836. 10.1080/10635150600999150." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR20" id="ref-link-section-d16311e409">20</a>]),
 requiring a software solution for each step of these computer 
pipelines. Given the importance of dataset quality, the use of practical
 and accurate software dedicated to alignment trimming task has become a
 real need.</p><p>In this paper, we present a novel software, named BMGE
 (Block Mapping and Gathering with Entropy), that identifies regions 
inside multiple sequence alignments that are suited for phylogenetic 
inference. BMGE computes a score for each character (i.e. amino acid, 
nucleotide or codon column), mainly determined by the entropy induced by
 the proportion of character states. To estimate realistic scores that 
take into account biologically relevant substitution processes (e.g. 
transition rates more frequent than transversions for DNA sequences, 
highest probability of changes between amino acids with physicochemical 
similarities), BMGE weights the entropy estimation with standard 
substitution matrices (e.g. PAM or BLOSUM). Averaging score values 
across the characters of the multiple sequence alignment allows 
identifying conserved (i.e. with low entropy-like score values) and 
highly variable/uncertain regions (i.e. with large entropy-like score 
values [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Swingley WD, Blankenship RE, Raymond J: Integrating Markov clustering and molecular phylogenetics to reconstruct the cyanobacterial species tree from conserved protein families. Mol Biol Evol. 2008, 25: 643-654. 10.1093/molbev/msn034." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR15" id="ref-link-section-d16311e415">15</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Schneider TD, Stephens RM: Sequence logos: a new way to display consensus sequences. Nucl Acids Res. 1990, 18: 6097-6100. 10.1093/nar/18.20.6097." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR21" id="ref-link-section-d16311e418">21</a>]).
 By removing such high entropy regions, BMGE returns trimmed datasets 
that allow the reconstruction of more accurate phylogenetic trees than 
the initial alignment, as shown by simulation studies.</p><p>In 
addition, BMGE also provides simple solutions to alleviate systematic 
artefacts caused by compositional heterogeneity. Most probabilistic 
phylogenetic inference methods make the assumption (among other more or 
less axiomatic ones) that the studied sequences arose from a common 
ancestral sequence following a stationary evolutionary process, i.e. the
 marginal probabilities of the character states remained constant over 
all sequences (e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Swofford DL, Olsen GJ, Waddell PJ, Hillis DM: Phylogenetic inference. Molecular Systematics. Edited by: Hillis DM, Moritz C, Mable BK. 1996, Sunderland: Sinauer Associates, 407-514." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR11" id="ref-link-section-d16311e424">11</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Jayaswal V, Jermiin LS, Robinson J: Estimation of phylogeny using a general Markov model. Evol Bioinf Online. 2005, 1: 62-80." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR22" id="ref-link-section-d16311e427">22</a>]).
 Consequently, when phylogenetic trees are inferred from sequences with 
heterogeneous composition of character states, the violation of the 
stationary assumption may cause systematic errors [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Adachi J, Waddell PJ, Martin W, Hasegawa M: Plastid genome phylogeny and a model of amino acid substitution for protein encoded by chloroplast DNA. J Mol Evol. 2000, 50: 348-358." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR19" id="ref-link-section-d16311e430">19</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Galtier N, Gouy M: Inferring pattern and process: maximum-likelihood implementation of a nonhomogeneous model of DNA sequence evolution for phylogenetic analysis. Mol Biol Evol. 1998, 15: 871-879." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR23" id="ref-link-section-d16311e433">23</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Phillips MJ, Delsuc F, Penny D: Genome-scale phylogeny and the detection of systematic biases. Mol Biol Evol. 2004, 21: 1455-1458. 10.1093/molbev/msh137." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR25" id="ref-link-section-d16311e436">25</a>]. BMGE is therefore able to perform RY-coding from a DNA sequence alignment [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Phillips MJ, Delsuc F, Penny D: Genome-scale phylogeny and the detection of systematic biases. Mol Biol Evol. 2004, 21: 1455-1458. 10.1093/molbev/msh137." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR25" id="ref-link-section-d16311e440">25</a>],
 and to convert amino acid sequences into their corresponding 
degenerated codons according to the universal genetic code. These two 
recoding strategies may prove useful to minimize some biases when 
dealing with datasets with known heterogeneous composition across 
sequences. As these two recoding approaches use only the standard 
one-letter nucleotide alphabet [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="International Union of Pure and Applied Chemistery and International Union of Biochemistery (IUPAC-IUB) Commission on Biochemical Nomenclature: Abbreviations and symbols for nucleic acids, polynucleotides and their constituents. Biochem J. 1970, 120: 449-454." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR26" id="ref-link-section-d16311e443">26</a>] (see Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab1">1</a>),
 the resulting datasets can be given to all phylogeny inference 
programs, in contrast to alternative recoding techniques based on 
non-standard alphabet cardinality such as the "Dayhoff classes" 
6-residue alphabet [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Dayhoff MO, Schwartz RM, Orcutt BD: A model of evolutionary change in proteins. Atlas of Protein Sequence and Structure. Edited by: Dayhoff MO. 1978, Washington: National Biomedical Research Foundation, 5 (Suppl 3): 345-352." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR27" id="ref-link-section-d16311e449">27</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Embley TM, van der Giezen M, Horner DS, Dyal PL, Foster P: Mitochondria and hydrogenosomes are two forms of the same fundamental organelle. Philos Trans R Soc Lond B Biol Sci. 2003, 358: 191-203. 10.1098/rstb.2002.1190." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR28" id="ref-link-section-d16311e452">28</a>] (see also [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Susko E, Roger AJ: On reduced amino acid alphabets for phylogenetic inference. Mol Biol Evol. 2007, 24: 2139-2150. 10.1093/molbev/msm144." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR29" id="ref-link-section-d16311e455">29</a>]
 for discussion on other recoding schemes). Moreover, the use of 
degenerated codons allow fast inference of trees, in particular with 
Maximum Likelihood (ML) methods which are faster with nucleotide 
sequences than with amino acid ones. BMGE also implements a novel 
stationary-based trimming method that allows compositionally 
heterogeneous characters to be identified and removed. To do so, BMGE 
uses the Stuart's <i>χ</i><sup>2</sup> matched-pairs test of marginal symmetry [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Stuart A: A test for homogeneity of the marginal distributions in a two-way classification. Biometrika. 1955, 42: 412-416." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR30" id="ref-link-section-d16311e463">30</a>] that allows assessing the null hypothesis that two sequences are compositionally homogeneous [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Ababneh F, Jermiin LS, Ma C, Robinson J: Matched-pairs tests of homogeneity with applications to homologous nucleotide sequences. Bioinformatics. 2006, 22: 1225-1231. 10.1093/bioinformatics/btl064." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR31" id="ref-link-section-d16311e466">31</a>],
 and iteratively performs character removal/addition steps until the 
Stuart's test assesses that each pair of sequences presents homogeneous 
composition. As shown by computer simulations with heterogeneous 
GC-content DNA sequences, this stationary-based trimming leads to 
unbiased phylogenetic trees.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b id="Tab1" data-test="table-caption">Table 1 Character state coding used by BMGE</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-category="article body" data-track-label="button" rel="nofollow" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/tables/1"><span>Full size table</span></a></div></figure></div></div></div></section><section aria-labelledby="Sec2"><div class="c-article-section js-article-section" id="Sec2-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec2">Implementation</h2><div class="c-article-section__content js-collapsible-section" id="Sec2-content"><h3 class="c-article__sub-heading u-h3" id="Sec3">Input/output files and sequence coding conversions</h3><p>The
 input file for BMGE is a multiple sequence alignment in FASTA (or 
PHYLIP sequential) format. The user must indicate whether the sequences 
are amino acids or DNA (with standard one-letter coding [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="International Union of Pure and Applied Chemistery and International Union of Biochemistery (IUPAC-IUB) Commission on Biochemical Nomenclature: Abbreviations and symbols for nucleic acids, polynucleotides and their constituents. Biochem J. 1970, 120: 449-454." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR26" id="ref-link-section-d16311e1691">26</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="International Union of Pure and Applied Chemistery and International Union of Biochemistery (IUPAC-IUB) Commission on Biochemical Nomenclature: A one-letter notation for amino acid sequences (definitive rules). Pure Appl Chem. 1972, 31: 639-645. 10.1351/pac197231040639." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR32" id="ref-link-section-d16311e1694">32</a>]; see Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab1">1</a>).
 It is also possible to consider DNA sequences as codons, which allows 
the multiple sequence alignment to be handled with amino acid 
substitution matrices. Selected (and/or removed) regions are written in 
an output file in several formats (i.e. FASTA, PHYLIP sequential, 
NEXUS). HTML output is also available to display selected sites as well 
as graphical representation of both entropy values and gap proportions.</p><p>Several sequence conversion options are also available: from DNA or codons to RY-coding [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Phillips MJ, Delsuc F, Penny D: Genome-scale phylogeny and the detection of systematic biases. Mol Biol Evol. 2004, 21: 1455-1458. 10.1093/molbev/msh137." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR25" id="ref-link-section-d16311e1703">25</a>],
 and from codons to translated amino acids (according to the universal 
genetic code). BMGE also allows converting an amino acid alignment into a
 nucleotide alignment by considering the corresponding degenerated 
codons (see Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab1">1</a>).
 In practice, given an amino acid and its set of corresponding 
synonymous codons (following the universal genetic code), the 
degenerated codon is simply obtained, for each of the three codon 
positions <i>p</i>, by considering the nucleotides corresponding to the set of possible codons at position <i>p</i>.
 For example, isoleucine (I) can be encoded by the three codons ATA, ATC
 and ATT; therefore, the degenerated codon corresponding to this set of 
codons is ATH, knowing that the degenerated nucleotide H (i.e. A, C or 
T; see Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab1">1</a>) represents the possible nucleotides at the third threefold degenerate position.</p><h3 class="c-article__sub-heading u-h3" id="Sec4">Entropy-based character trimming</h3><p>Given a multiple sequence alignment of <i>m</i> character length, BMGE computes, for each character <i>c</i> = 1,2,...,<i>m</i> , the frequency <i>g</i>(<i>c</i>) of gaps, as well as the diagonal matrix <b>∏</b> <sup>(<i>c</i>) </sup>where each diagonal entry is the relative frequency of each of the <i>r</i> possible character states (<i>r =</i> 4 or 20, for DNA or amino acid sequences, respectively). So, BMGE computes the value <i>h</i>(<i>c</i>) for each character <i>c</i>, which is closely related to the von Neumann entropy [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Von Neumann J: Mathematische Grundlagen der Quantenmechanik. 1932, Berlin: Springer" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR33" id="ref-link-section-d16311e1767">33</a>] and is given by the following formula [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Caffrey DR, Somaroo S, Hughes JD, Mintseris J, Huang ES: Are protein-protein interfaces more conserved in sequence than the rest of the protein surface?. Protein Sci. 2004, 13: 190-202. 10.1110/ps.03323604." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR34" id="ref-link-section-d16311e1770">34</a>]:</p><div id="Equ1" class="c-article-equation"><div class="c-article-equation__content"><span class="MathJax_Preview" style=""></span><div class="MathJax_Display" style="text-align: left;"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style="text-align: left; position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;block&quot;&gt;&lt;mrow&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mo&gt;&amp;#x2212;&lt;/mo&gt;&lt;mtext&gt;trace&lt;/mtext&gt;&lt;mo stretchy=&quot;false&quot;&gt;[&lt;/mo&gt;&lt;mi&gt;&amp;#x3BC;&lt;/mi&gt;&lt;msup&gt;&lt;mi&gt;&amp;#x3A0;&lt;/mi&gt;&lt;mrow&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/msup&gt;&lt;mstyle mathvariant=&quot;bold&quot; mathsize=&quot;normal&quot;&gt;&lt;mi&gt;S&lt;/mi&gt;&lt;/mstyle&gt;&lt;mi&gt;l&lt;/mi&gt;&lt;mi&gt;o&lt;/mi&gt;&lt;msub&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;/msub&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;&amp;#x3BC;&lt;/mi&gt;&lt;msup&gt;&lt;mi&gt;&amp;#x3A0;&lt;/mi&gt;&lt;mrow&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/msup&gt;&lt;mstyle mathvariant=&quot;bold&quot; mathsize=&quot;normal&quot;&gt;&lt;mi&gt;S&lt;/mi&gt;&lt;/mstyle&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo stretchy=&quot;false&quot;&gt;]&lt;/mo&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-1" style="width: 20.028em; display: inline-block;"><span style="display: inline-block; position: relative; width: 15.504em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(1.339em, 1015.44em, 2.791em, -1000em); top: -2.412em; left: 0em;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="mi" id="MathJax-Span-4" style="font-family: MathJax_Math; font-style: italic;">h</span><span class="mo" id="MathJax-Span-5" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-6" style="font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-7" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-8" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mo" id="MathJax-Span-9" style="font-family: MathJax_Main; padding-left: 0.278em;">−</span><span class="mtext" id="MathJax-Span-10" style="font-family: MathJax_Main;">trace</span><span class="mo" id="MathJax-Span-11" style="font-family: MathJax_Main;">[</span><span class="mi" id="MathJax-Span-12" style="font-family: MathJax_Math; font-style: italic;">μ</span><span class="msup" id="MathJax-Span-13"><span style="display: inline-block; position: relative; width: 1.885em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.89em, 4.134em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-14" style="font-family: MathJax_Math; font-style: italic;">Π<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.056em;"></span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; top: -4.418em; left: 0.954em;"><span class="mrow" id="MathJax-Span-15"><span class="mo" id="MathJax-Span-16" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-17" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-18" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mstyle" id="MathJax-Span-19"><span class="mrow" id="MathJax-Span-20"><span class="mi" id="MathJax-Span-21" style="font-family: MathJax_Main; font-weight: bold;">S</span></span></span><span class="mi" id="MathJax-Span-22" style="font-family: MathJax_Math; font-style: italic;">l</span><span class="mi" id="MathJax-Span-23" style="font-family: MathJax_Math; font-style: italic;">o</span><span class="msub" id="MathJax-Span-24"><span style="display: inline-block; position: relative; width: 0.871em; height: 0px;"><span style="position: absolute; clip: rect(3.434em, 1000.48em, 4.339em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-25" style="font-family: MathJax_Math; font-style: italic;">g<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; top: -3.855em; left: 0.477em;"><span class="mi" id="MathJax-Span-26" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">r</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mo" id="MathJax-Span-27" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-28" style="font-family: MathJax_Math; font-style: italic;">μ</span><span class="msup" id="MathJax-Span-29"><span style="display: inline-block; position: relative; width: 1.885em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.89em, 4.134em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-30" style="font-family: MathJax_Math; font-style: italic;">Π<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.056em;"></span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; top: -4.418em; left: 0.954em;"><span class="mrow" id="MathJax-Span-31"><span class="mo" id="MathJax-Span-32" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-33" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-34" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mstyle" id="MathJax-Span-35"><span class="mrow" id="MathJax-Span-36"><span class="mi" id="MathJax-Span-37" style="font-family: MathJax_Main; font-weight: bold;">S</span></span></span><span class="mo" id="MathJax-Span-38" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-39" style="font-family: MathJax_Main;">]</span><span class="mo" id="MathJax-Span-40" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.412em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.378em; border-left: 0px solid; width: 0px; height: 1.65em;"></span></span></nobr><span class="MJX_Assistive_MathML MJX_Assistive_MathML_Block" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><mrow><mi>h</mi><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo><mo>=</mo><mo>−</mo><mtext>trace</mtext><mo stretchy="false">[</mo><mi>μ</mi><msup><mi>Π</mi><mrow><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow></msup><mstyle mathvariant="bold" mathsize="normal"><mi>S</mi></mstyle><mi>l</mi><mi>o</mi><msub><mi>g</mi><mi>r</mi></msub><mo stretchy="false">(</mo><mi>μ</mi><msup><mi>Π</mi><mrow><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow></msup><mstyle mathvariant="bold" mathsize="normal"><mi>S</mi></mstyle><mo stretchy="false">)</mo><mo stretchy="false">]</mo><mo>,</mo></mrow></math></span></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block"><mml:mrow>
                          <mml:mi>h</mml:mi>
                          <mml:mo stretchy="false">(</mml:mo>
                          <mml:mi>c</mml:mi>
                          <mml:mo stretchy="false">)</mml:mo>
                          <mml:mo>=</mml:mo>
                          <mml:mo>−</mml:mo>
                          <mml:mtext>trace</mml:mtext>
                          <mml:mo stretchy="false">[</mml:mo>
                          <mml:mi>μ</mml:mi>
                          <mml:msup>
                            <mml:mi>Π</mml:mi>
                            <mml:mrow>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>c</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                            </mml:mrow>
                          </mml:msup>
                          <mml:mstyle mathvariant="bold" mathsize="normal">
                            <mml:mi>S</mml:mi>
                          </mml:mstyle>
                          <mml:mi>l</mml:mi>
                          <mml:mi>o</mml:mi>
                          <mml:msub>
                            <mml:mi>g</mml:mi>
                            <mml:mi>r</mml:mi>
                          </mml:msub>
                          <mml:mo stretchy="false">(</mml:mo>
                          <mml:mi>μ</mml:mi>
                          <mml:msup>
                            <mml:mi>Π</mml:mi>
                            <mml:mrow>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>c</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                            </mml:mrow>
                          </mml:msup>
                          <mml:mstyle mathvariant="bold" mathsize="normal">
                            <mml:mi>S</mml:mi>
                          </mml:mstyle>
                          <mml:mo stretchy="false">)</mml:mo>
                          <mml:mo stretchy="false">]</mml:mo>
                          <mml:mo>,</mml:mo>
                        </mml:mrow></math></script></div><div class="c-article-equation__number">
                    (1)
                </div></div><p>where <b>S</b> is a similarity matrix (see below for more details) and <i>μ</i> = [trace (<b>∏</b><sup>(<i>c</i>)</sup><b>S</b>) ]<sup>-1</sup> is a normalizing factor so that trace (<i>μ</i> <b>∏</b><sup>(<i>c</i>) </sup><b>S</b>) = 1. To compute formula (1), BMGE uses the simpler equation</p><div id="Equ2" class="c-article-equation"><div class="c-article-equation__content"><span class="MathJax_Preview" style=""></span><div class="MathJax_Display" style="text-align: left;"><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0" style="text-align: left; position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;block&quot;&gt;&lt;mrow&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mo&gt;&amp;#x2212;&lt;/mo&gt;&lt;mstyle displaystyle=&quot;true&quot;&gt;&lt;msub&gt;&lt;mo&gt;&amp;#x2211;&lt;/mo&gt;&lt;mrow&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;&amp;#x2026;&lt;/mo&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mrow&gt;&lt;msubsup&gt;&lt;mi&gt;&amp;#x3BB;&lt;/mi&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mrow&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;/mrow&gt;&lt;/mstyle&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;log&lt;/mi&gt;&lt;/mrow&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;/msub&gt;&lt;msubsup&gt;&lt;mi&gt;&amp;#x3BB;&lt;/mi&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mrow&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-41" style="width: 16.841em; display: inline-block;"><span style="display: inline-block; position: relative; width: 13.049em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(1.231em, 1012.98em, 3.034em, -1000em); top: -2.412em; left: 0em;"><span class="mrow" id="MathJax-Span-42"><span class="mrow" id="MathJax-Span-43"><span class="mi" id="MathJax-Span-44" style="font-family: MathJax_Math; font-style: italic;">h</span><span class="mo" id="MathJax-Span-45" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-46" style="font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-47" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-48" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mo" id="MathJax-Span-49" style="font-family: MathJax_Main; padding-left: 0.278em;">−</span><span class="mstyle" id="MathJax-Span-50" style="padding-left: 0.167em;"><span class="mrow" id="MathJax-Span-51"><span class="msub" id="MathJax-Span-52"><span style="display: inline-block; position: relative; width: 3.902em; height: 0px;"><span style="position: absolute; clip: rect(2.926em, 1001.39em, 4.584em, -1000em); top: -4.005em; left: 0em;"><span class="mo" id="MathJax-Span-53" style="font-family: MathJax_Size2; vertical-align: 0em;">∑</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; top: -3.52em; left: 1.444em;"><span class="mrow" id="MathJax-Span-54"><span class="mi" id="MathJax-Span-55" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span class="mo" id="MathJax-Span-56" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-57" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-58" style="font-size: 70.7%; font-family: MathJax_Main;">…</span><span class="mi" id="MathJax-Span-59" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">r</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mrow" id="MathJax-Span-60" style="padding-left: 0.167em;"><span class="msubsup" id="MathJax-Span-61"><span style="display: inline-block; position: relative; width: 1.514em; height: 0px;"><span style="position: absolute; clip: rect(3.182em, 1000.56em, 4.146em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-62" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.346em, 1000.93em, 4.311em, -1000em); top: -4.526em; left: 0.583em;"><span class="mrow" id="MathJax-Span-63"><span class="mo" id="MathJax-Span-64" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-65" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-66" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.563em, 1000.41em, 4.141em, -1000em); top: -3.857em; left: 0.583em;"><span class="mi" id="MathJax-Span-67" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span></span></span></span><span class="msub" id="MathJax-Span-68"><span style="display: inline-block; position: relative; width: 1.672em; height: 0px;"><span style="position: absolute; clip: rect(3.182em, 1001.26em, 4.34em, -1000em); top: -4.005em; left: 0em;"><span class="mrow" id="MathJax-Span-69"><span class="mi" id="MathJax-Span-70" style="font-family: MathJax_Main;">log</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; top: -3.764em; left: 1.278em;"><span class="mi" id="MathJax-Span-71" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">r</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-72"><span style="display: inline-block; position: relative; width: 1.514em; height: 0px;"><span style="position: absolute; clip: rect(3.182em, 1000.56em, 4.146em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-73" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.346em, 1000.93em, 4.311em, -1000em); top: -4.526em; left: 0.583em;"><span class="mrow" id="MathJax-Span-74"><span class="mo" id="MathJax-Span-75" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-76" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-77" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.563em, 1000.41em, 4.141em, -1000em); top: -3.857em; left: 0.583em;"><span class="mi" id="MathJax-Span-78" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mo" id="MathJax-Span-79" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.412em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.692em; border-left: 0px solid; width: 0px; height: 2.104em;"></span></span></nobr><span class="MJX_Assistive_MathML MJX_Assistive_MathML_Block" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><mrow><mi>h</mi><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo><mo>=</mo><mo>−</mo><mstyle displaystyle="true"><msub><mo>∑</mo><mrow><mi>s</mi><mo>=</mo><mn>1</mn><mo>…</mo><mi>r</mi></mrow></msub><mrow><msubsup><mi>λ</mi><mi>s</mi><mrow><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></mstyle><msub><mrow><mi>log</mi></mrow><mi>r</mi></msub><msubsup><mi>λ</mi><mi>s</mi><mrow><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>,</mo></mrow></math></span></span></div><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block"><mml:mrow>
                          <mml:mi>h</mml:mi>
                          <mml:mo stretchy="false">(</mml:mo>
                          <mml:mi>c</mml:mi>
                          <mml:mo stretchy="false">)</mml:mo>
                          <mml:mo>=</mml:mo>
                          <mml:mo>−</mml:mo>
                          <mml:mstyle displaystyle="true">
                            <mml:msub>
                              <mml:mo>∑</mml:mo>
                              <mml:mrow>
                                <mml:mi>s</mml:mi>
                                <mml:mo>=</mml:mo>
                                <mml:mn>1</mml:mn>
                                <mml:mo>…</mml:mo>
                                <mml:mi>r</mml:mi>
                              </mml:mrow>
                            </mml:msub>
                            <mml:mrow>
                              <mml:msubsup>
                                <mml:mi>λ</mml:mi>
                                <mml:mi>s</mml:mi>
                                <mml:mrow>
                                  <mml:mo stretchy="false">(</mml:mo>
                                  <mml:mi>c</mml:mi>
                                  <mml:mo stretchy="false">)</mml:mo>
                                </mml:mrow>
                              </mml:msubsup>
                            </mml:mrow>
                          </mml:mstyle>
                          <mml:msub>
                            <mml:mrow>
                              <mml:mi>log</mml:mi>
                            </mml:mrow>
                            <mml:mi>r</mml:mi>
                          </mml:msub>
                          <mml:msubsup>
                            <mml:mi>λ</mml:mi>
                            <mml:mi>s</mml:mi>
                            <mml:mrow>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>c</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                            </mml:mrow>
                          </mml:msubsup>
                          <mml:mo>,</mml:mo>
                        </mml:mrow></math></script></div><div class="c-article-equation__number">
                    (2)
                </div></div><p>where the <span class="MathJax_Preview" style=""></span><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;inline&quot;&gt;&lt;mrow&gt;&lt;msubsup&gt;&lt;mi&gt;&amp;#x3BB;&lt;/mi&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mrow&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-80" style="width: 1.94em; display: inline-block;"><span style="display: inline-block; position: relative; width: 1.507em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(1.188em, 1001.51em, 2.653em, -1000em); top: -2.369em; left: 0em;"><span class="mrow" id="MathJax-Span-81"><span class="mrow" id="MathJax-Span-82"><span class="msubsup" id="MathJax-Span-83"><span style="display: inline-block; position: relative; width: 1.514em; height: 0px;"><span style="position: absolute; clip: rect(3.182em, 1000.56em, 4.146em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-84" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.346em, 1000.93em, 4.311em, -1000em); top: -4.526em; left: 0.583em;"><span class="mrow" id="MathJax-Span-85"><span class="mo" id="MathJax-Span-86" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-87" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-88" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.563em, 1000.41em, 4.141em, -1000em); top: -3.857em; left: 0.583em;"><span class="mi" id="MathJax-Span-89" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.369em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.256em; border-left: 0px solid; width: 0px; height: 1.668em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"><mrow><msubsup><mi>λ</mi><mi>s</mi><mrow><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></span></span><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline"><mml:mrow>
                          <mml:msubsup>
                            <mml:mi>λ</mml:mi>
                            <mml:mi>s</mml:mi>
                            <mml:mrow>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>c</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                            </mml:mrow>
                          </mml:msubsup>
                        </mml:mrow></math></script> parameters are the eigenvalues of the matrix <i>μ</i> <b>∏</b><sup>(<i>c</i>)</sup><b>S</b> estimated via the JAMA package [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="JAMA: A Java Matrix Package. [
                    http://math.nist.gov/javanumerics/jama/
                    
                  ]" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR35" id="ref-link-section-d16311e2072">35</a>]. It should be stressed that the entropy normalization condition <span class="MathJax_Preview" style=""></span><span class="MathJax" id="MathJax-Element-4-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;inline&quot;&gt;&lt;mrow&gt;&lt;mstyle displaystyle=&quot;true&quot;&gt;&lt;msub&gt;&lt;mo&gt;&amp;#x2211;&lt;/mo&gt;&lt;mrow&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;&amp;#x2026;&lt;/mo&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mrow&gt;&lt;msubsup&gt;&lt;mi&gt;&amp;#x3BB;&lt;/mi&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mrow&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;/mrow&gt;&lt;/mstyle&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-90" style="width: 9.606em; display: inline-block;"><span style="display: inline-block; position: relative; width: 7.45em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(1.188em, 1007.38em, 2.991em, -1000em); top: -2.369em; left: 0em;"><span class="mrow" id="MathJax-Span-91"><span class="mrow" id="MathJax-Span-92"><span class="mstyle" id="MathJax-Span-93"><span class="mrow" id="MathJax-Span-94"><span class="msub" id="MathJax-Span-95"><span style="display: inline-block; position: relative; width: 3.902em; height: 0px;"><span style="position: absolute; clip: rect(2.926em, 1001.39em, 4.584em, -1000em); top: -4.005em; left: 0em;"><span class="mo" id="MathJax-Span-96" style="font-family: MathJax_Size2; vertical-align: 0em;">∑</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; top: -3.52em; left: 1.444em;"><span class="mrow" id="MathJax-Span-97"><span class="mi" id="MathJax-Span-98" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span class="mo" id="MathJax-Span-99" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-100" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-101" style="font-size: 70.7%; font-family: MathJax_Main;">…</span><span class="mi" id="MathJax-Span-102" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">r</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mrow" id="MathJax-Span-103" style="padding-left: 0.167em;"><span class="msubsup" id="MathJax-Span-104"><span style="display: inline-block; position: relative; width: 1.514em; height: 0px;"><span style="position: absolute; clip: rect(3.182em, 1000.56em, 4.146em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-105" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.346em, 1000.93em, 4.311em, -1000em); top: -4.526em; left: 0.583em;"><span class="mrow" id="MathJax-Span-106"><span class="mo" id="MathJax-Span-107" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-108" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-109" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.563em, 1000.41em, 4.141em, -1000em); top: -3.857em; left: 0.583em;"><span class="mi" id="MathJax-Span-110" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span></span></span></span><span class="mo" id="MathJax-Span-111" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-112" style="font-family: MathJax_Main; padding-left: 0.278em;">1</span></span></span><span style="display: inline-block; width: 0px; height: 2.369em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.692em; border-left: 0px solid; width: 0px; height: 2.104em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"><mrow><mstyle displaystyle="true"><msub><mo>∑</mo><mrow><mi>s</mi><mo>=</mo><mn>1</mn><mo>…</mo><mi>r</mi></mrow></msub><mrow><msubsup><mi>λ</mi><mi>s</mi><mrow><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></mstyle><mo>=</mo><mn>1</mn></mrow></math></span></span><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline"><mml:mrow>
                          <mml:mstyle displaystyle="true">
                            <mml:msub>
                              <mml:mo>∑</mml:mo>
                              <mml:mrow>
                                <mml:mi>s</mml:mi>
                                <mml:mo>=</mml:mo>
                                <mml:mn>1</mml:mn>
                                <mml:mo>…</mml:mo>
                                <mml:mi>r</mml:mi>
                              </mml:mrow>
                            </mml:msub>
                            <mml:mrow>
                              <mml:msubsup>
                                <mml:mi>λ</mml:mi>
                                <mml:mi>s</mml:mi>
                                <mml:mrow>
                                  <mml:mo stretchy="false">(</mml:mo>
                                  <mml:mi>c</mml:mi>
                                  <mml:mo stretchy="false">)</mml:mo>
                                </mml:mrow>
                              </mml:msubsup>
                            </mml:mrow>
                          </mml:mstyle>
                          <mml:mo>=</mml:mo>
                          <mml:mn>1</mml:mn>
                        </mml:mrow></math></script> is verified with the normalizing factor <i>μ</i>.</p><p>Knowing that <i>h</i>(<i>c</i>) = 0 indicates that character <i>c</i> is constant and that <i>h</i>(<i>c</i>) is as close to 1 as character <i>c</i>
 is unexpectedly variable (see below), BMGE looks for conserved regions 
of the multiple sequence alignment by smoothing the different <i>h</i>(<i>c</i>) values by using a sliding window of length 2<i>w</i>+1 (with <i>w</i> = 1 by default). For each <i>c</i> = 1, 2,...,<i>m</i> , the smooth <img src="12862_2010_Article_1426_IEq3_HTML.gif"> value is estimated by the weighted average</p><div id="Equ3" class="c-article-equation"><div class="c-article-equation__content"><span class="MathJax_Preview" style=""></span><div class="MathJax_Display" style="text-align: left;"><span class="MathJax" id="MathJax-Element-5-Frame" tabindex="0" style="text-align: left; position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;block&quot;&gt;&lt;mrow&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mo&gt;&amp;#x2DC;&lt;/mo&gt;&lt;/mover&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mfrac&gt;&lt;mrow&gt;&lt;msubsup&gt;&lt;mo&gt;&amp;#x2211;&lt;/mo&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mtext&gt;start&lt;/mtext&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mtext&gt;end&lt;/mtext&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;mo stretchy=&quot;false&quot;&gt;[&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;&amp;#x2212;&lt;/mo&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo stretchy=&quot;false&quot;&gt;]&lt;/mo&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;msubsup&gt;&lt;mo&gt;&amp;#x2211;&lt;/mo&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mtext&gt;start&lt;/mtext&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mtext&gt;end&lt;/mtext&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;mo stretchy=&quot;false&quot;&gt;[&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;&amp;#x2212;&lt;/mo&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo stretchy=&quot;false&quot;&gt;]&lt;/mo&gt;&lt;/mrow&gt;&lt;/mfrac&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-113" style="width: 16.41em; display: inline-block;"><span style="display: inline-block; position: relative; width: 12.705em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(0.574em, 1012.64em, 3.749em, -1000em); top: -2.412em; left: 0em;"><span class="mrow" id="MathJax-Span-114"><span class="mrow" id="MathJax-Span-115"><span class="mover" id="MathJax-Span-116"><span style="display: inline-block; position: relative; width: 0.576em; height: 0px;"><span style="position: absolute; clip: rect(3.182em, 1000.55em, 4.145em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-117" style="font-family: MathJax_Math; font-style: italic;">h</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.208em, 1000.42em, 3.569em, -1000em); top: -4.242em; left: 0.01em;"><span class="mo" id="MathJax-Span-118" style=""><span style="font-family: MathJax_Main;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mo" id="MathJax-Span-119" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-120" style="font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-121" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-122" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mfrac" id="MathJax-Span-123" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 9.085em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(2.908em, 1008.87em, 4.428em, -1000em); top: -4.746em; left: 50%; margin-left: -4.482em;"><span class="mrow" id="MathJax-Span-124"><span class="msubsup" id="MathJax-Span-125"><span style="display: inline-block; position: relative; width: 3.385em; height: 0px;"><span style="position: absolute; clip: rect(3.126em, 1001em, 4.384em, -1000em); top: -4.005em; left: 0em;"><span class="mo" id="MathJax-Span-126" style="font-family: MathJax_Size1; vertical-align: 0em;">∑</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.385em, 1001.17em, 4.142em, -1000em); top: -4.482em; left: 1.056em;"><span class="mrow" id="MathJax-Span-127"><span class="mtext" id="MathJax-Span-128" style="font-size: 70.7%; font-family: MathJax_Main;">end</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.409em, 1002.33em, 4.142em, -1000em); top: -3.72em; left: 1.056em;"><span class="mrow" id="MathJax-Span-129"><span class="mi" id="MathJax-Span-130" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-131" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mtext" id="MathJax-Span-132" style="font-size: 70.7%; font-family: MathJax_Main;">start</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mo" id="MathJax-Span-133" style="font-family: MathJax_Main;">[</span><span class="mn" id="MathJax-Span-134" style="font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-135" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="mi" id="MathJax-Span-136" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">g<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mo" id="MathJax-Span-137" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-138" style="font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-139" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-140" style="font-family: MathJax_Main;">]</span><span class="mi" id="MathJax-Span-141" style="font-family: MathJax_Math; font-style: italic;">h</span><span class="mo" id="MathJax-Span-142" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-143" style="font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-144" style="font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(2.908em, 1007.15em, 4.428em, -1000em); top: -3.09em; left: 50%; margin-left: -3.633em;"><span class="mrow" id="MathJax-Span-145"><span class="msubsup" id="MathJax-Span-146"><span style="display: inline-block; position: relative; width: 3.385em; height: 0px;"><span style="position: absolute; clip: rect(3.126em, 1001em, 4.384em, -1000em); top: -4.005em; left: 0em;"><span class="mo" id="MathJax-Span-147" style="font-family: MathJax_Size1; vertical-align: 0em;">∑</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.385em, 1001.17em, 4.142em, -1000em); top: -4.482em; left: 1.056em;"><span class="mrow" id="MathJax-Span-148"><span class="mtext" id="MathJax-Span-149" style="font-size: 70.7%; font-family: MathJax_Main;">end</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.409em, 1002.33em, 4.142em, -1000em); top: -3.72em; left: 1.056em;"><span class="mrow" id="MathJax-Span-150"><span class="mi" id="MathJax-Span-151" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-152" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mtext" id="MathJax-Span-153" style="font-size: 70.7%; font-family: MathJax_Main;">start</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mo" id="MathJax-Span-154" style="font-family: MathJax_Main;">[</span><span class="mn" id="MathJax-Span-155" style="font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-156" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="mi" id="MathJax-Span-157" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">g<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mo" id="MathJax-Span-158" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-159" style="font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-160" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-161" style="font-family: MathJax_Main;">]</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(0.887em, 1009.09em, 1.206em, -1000em); top: -1.297em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.4px solid; width: 9.085em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.077em;"></span></span></span></span><span class="mo" id="MathJax-Span-162" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.412em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.615em; border-left: 0px solid; width: 0px; height: 3.874em;"></span></span></nobr><span class="MJX_Assistive_MathML MJX_Assistive_MathML_Block" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><mrow><mover accent="true"><mi>h</mi><mo>˜</mo></mover><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mtext>start</mtext></mrow><mrow><mtext>end</mtext></mrow></msubsup><mo stretchy="false">[</mo><mn>1</mn><mo>−</mo><mi>g</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo stretchy="false">]</mo><mi>h</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow><mrow><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mtext>start</mtext></mrow><mrow><mtext>end</mtext></mrow></msubsup><mo stretchy="false">[</mo><mn>1</mn><mo>−</mo><mi>g</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo stretchy="false">]</mo></mrow></mfrac><mo>,</mo></mrow></math></span></span></div><script type="math/mml" id="MathJax-Element-5"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block"><mml:mrow>
                          <mml:mover accent="true">
                            <mml:mi>h</mml:mi>
                            <mml:mo>˜</mml:mo>
                          </mml:mover>
                          <mml:mo stretchy="false">(</mml:mo>
                          <mml:mi>c</mml:mi>
                          <mml:mo stretchy="false">)</mml:mo>
                          <mml:mo>=</mml:mo>
                          <mml:mfrac>
                            <mml:mrow>
                              <mml:msubsup>
                                <mml:mo>∑</mml:mo>
                                <mml:mrow>
                                  <mml:mi>i</mml:mi>
                                  <mml:mo>=</mml:mo>
                                  <mml:mtext>start</mml:mtext>
                                </mml:mrow>
                                <mml:mrow>
                                  <mml:mtext>end</mml:mtext>
                                </mml:mrow>
                              </mml:msubsup>
                              <mml:mo stretchy="false">[</mml:mo>
                              <mml:mn>1</mml:mn>
                              <mml:mo>−</mml:mo>
                              <mml:mi>g</mml:mi>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>i</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                              <mml:mo stretchy="false">]</mml:mo>
                              <mml:mi>h</mml:mi>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>i</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                            </mml:mrow>
                            <mml:mrow>
                              <mml:msubsup>
                                <mml:mo>∑</mml:mo>
                                <mml:mrow>
                                  <mml:mi>i</mml:mi>
                                  <mml:mo>=</mml:mo>
                                  <mml:mtext>start</mml:mtext>
                                </mml:mrow>
                                <mml:mrow>
                                  <mml:mtext>end</mml:mtext>
                                </mml:mrow>
                              </mml:msubsup>
                              <mml:mo stretchy="false">[</mml:mo>
                              <mml:mn>1</mml:mn>
                              <mml:mo>−</mml:mo>
                              <mml:mi>g</mml:mi>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>i</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                              <mml:mo stretchy="false">]</mml:mo>
                            </mml:mrow>
                          </mml:mfrac>
                          <mml:mo>,</mml:mo>
                        </mml:mrow></math></script></div><div class="c-article-equation__number">
                    (3)
                </div></div><p>with start = max(1;<i>c</i>-<i>w</i>) and end = min(<i>m</i>;<i>c</i>+<i>w</i>), in order to give more weight to characters with few gaps, i.e. <i>g</i>(<i>c</i>) <span class="stix">≅</span> 0. After this smoothing operation, BMGE defines as conserved those characters <i>c</i> that have <img src="12862_2010_Article_1426_IEq3_HTML.gif">
 value lower than a fixed threshold (0.5 by default). A conserved region
 is then defined as a set of contiguous conserved characters.</p><p>Then, the multiple sequence alignment is partitioned into successive conserved (<i>C</i> ) and variable (i.e. non-conserved; <i>V</i> ) regions, these being either a single character or a set of contiguous ones. Let <i>V</i><sub>
                    <i>i</i>
                  </sub> be the <i>i</i><sup>th</sup> non-conserved region. By definition, <i>V</i><sub>
                    <i>i</i>
                  </sub> is flanked by the two conserved regions <i>C</i><sub>
                    <i>i</i>
                  </sub> and <i>C</i><sub><i>i</i>+1</sub>. In order to distinguish variable regions due to ambiguous alignment from those due to natural variation, the average <span class="MathJax_Preview" style=""></span><span class="MathJax" id="MathJax-Element-6-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;inline&quot;&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mo&gt;&amp;#x2DC;&lt;/mo&gt;&lt;/mover&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-163" style="width: 0.734em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.56em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(1.335em, 1000.54em, 2.509em, -1000em); top: -2.369em; left: 0em;"><span class="mrow" id="MathJax-Span-164"><span class="mover" id="MathJax-Span-165"><span style="display: inline-block; position: relative; width: 0.576em; height: 0px;"><span style="position: absolute; clip: rect(3.182em, 1000.55em, 4.145em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-166" style="font-family: MathJax_Math; font-style: italic;">h</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.208em, 1000.42em, 3.569em, -1000em); top: -4.242em; left: 0.01em;"><span class="mo" id="MathJax-Span-167" style=""><span style="font-family: MathJax_Main;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.369em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.07em; border-left: 0px solid; width: 0px; height: 1.292em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"><mover accent="true"><mi>h</mi><mo>˜</mo></mover></math></span></span><script type="math/mml" id="MathJax-Element-6"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline"><mml:mover accent="true">
                          <mml:mi>h</mml:mi>
                          <mml:mo>˜</mml:mo>
                        </mml:mover></math></script> value is computed for the region <i>C</i><sub>
                    <i>i</i>
                  </sub><span class="stix">∪</span> <i>V</i><sub>
                    <i>i</i>
                  </sub><span class="stix">∪</span> <i>C</i><sub><i>i</i>+1 </sub>by formula (3) with parameters 'start' and 'end' set as the first character of <i>C</i><sub>
                    <i>i</i>
                  </sub>and the last character of <i>C</i><sub><i>i</i>+1</sub>, respectively. The consecutive regions <i>C</i><sub>
                    <i>i</i>
                  </sub>, <i>V</i><sub>
                    <i>i</i>
                  </sub>, <i>C</i><sub><i>i</i>+1 </sub>with less than 30% of gaps and with <span class="MathJax_Preview" style=""></span><span class="MathJax" id="MathJax-Element-7-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;inline&quot;&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mo&gt;&amp;#x2DC;&lt;/mo&gt;&lt;/mover&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-168" style="width: 0.734em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.56em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(1.335em, 1000.54em, 2.509em, -1000em); top: -2.369em; left: 0em;"><span class="mrow" id="MathJax-Span-169"><span class="mover" id="MathJax-Span-170"><span style="display: inline-block; position: relative; width: 0.576em; height: 0px;"><span style="position: absolute; clip: rect(3.182em, 1000.55em, 4.145em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-171" style="font-family: MathJax_Math; font-style: italic;">h</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.208em, 1000.42em, 3.569em, -1000em); top: -4.242em; left: 0.01em;"><span class="mo" id="MathJax-Span-172" style=""><span style="font-family: MathJax_Main;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.369em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.07em; border-left: 0px solid; width: 0px; height: 1.292em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"><mover accent="true"><mi>h</mi><mo>˜</mo></mover></math></span></span><script type="math/mml" id="MathJax-Element-7"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline"><mml:mover accent="true">
                          <mml:mi>h</mml:mi>
                          <mml:mo>˜</mml:mo>
                        </mml:mover></math></script> value lower than 
the fixed 0.5 threshold are then merged into a unique conserved region. 
Finally, BMGE iteratively performs these merging operations until no 
more variable region <i>V</i><sub>
                    <i>i</i>
                  </sub>can be merged with its two flanking <i>C</i><sub>
                    <i>i</i>
                  </sub>and <i>C</i><sub><i>i</i>+1 </sub>ones.</p><h3 class="c-article__sub-heading u-h3" id="Sec5">On the use of similarity matrix</h3><p>If the similarity matrix <b>S</b> in formula (1) is the identity matrix <b>I</b><sub>
                    <i>r</i>
                  </sub> , then h(<i>c</i>) is closely related to the well-known Shannon entropy [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Shannon C: A mathematical theory of communication. Bell System Tech J. 1948, 27: 379-423. 623-656" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR36" id="ref-link-section-d16311e2562">36</a>], given by formula (2) where each <span class="MathJax_Preview" style=""></span><span class="MathJax" id="MathJax-Element-8-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;inline&quot;&gt;&lt;mrow&gt;&lt;msubsup&gt;&lt;mi&gt;&amp;#x3BB;&lt;/mi&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mrow&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;msubsup&gt;&lt;mi&gt;&amp;#x3A0;&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-173" style="width: 6.118em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.737em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(1.188em, 1004.74em, 2.653em, -1000em); top: -2.369em; left: 0em;"><span class="mrow" id="MathJax-Span-174"><span class="mrow" id="MathJax-Span-175"><span class="msubsup" id="MathJax-Span-176"><span style="display: inline-block; position: relative; width: 1.514em; height: 0px;"><span style="position: absolute; clip: rect(3.182em, 1000.56em, 4.146em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-177" style="font-family: MathJax_Math; font-style: italic;">λ</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.346em, 1000.93em, 4.311em, -1000em); top: -4.526em; left: 0.583em;"><span class="mrow" id="MathJax-Span-178"><span class="mo" id="MathJax-Span-179" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-180" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-181" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.563em, 1000.41em, 4.141em, -1000em); top: -3.857em; left: 0.583em;"><span class="mi" id="MathJax-Span-182" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mo" id="MathJax-Span-183" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msubsup" id="MathJax-Span-184" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 1.885em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.89em, 4.134em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-185" style="font-family: MathJax_Math; font-style: italic;">Π<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.056em;"></span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.346em, 1000.93em, 4.311em, -1000em); top: -4.526em; left: 0.954em;"><span class="mrow" id="MathJax-Span-186"><span class="mo" id="MathJax-Span-187" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-188" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-189" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.563em, 1000.74em, 4.141em, -1000em); top: -3.857em; left: 0.831em;"><span class="mrow" id="MathJax-Span-190"><span class="mi" id="MathJax-Span-191" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span class="mi" id="MathJax-Span-192" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.369em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.256em; border-left: 0px solid; width: 0px; height: 1.668em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"><mrow><msubsup><mi>λ</mi><mi>s</mi><mrow><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>=</mo><msubsup><mi>Π</mi><mrow><mi>s</mi><mi>s</mi></mrow><mrow><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></span></span><script type="math/mml" id="MathJax-Element-8"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline"><mml:mrow>
                          <mml:msubsup>
                            <mml:mi>λ</mml:mi>
                            <mml:mi>s</mml:mi>
                            <mml:mrow>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>c</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                            </mml:mrow>
                          </mml:msubsup>
                          <mml:mo>=</mml:mo>
                          <mml:msubsup>
                            <mml:mi>Π</mml:mi>
                            <mml:mrow>
                              <mml:mi>s</mml:mi>
                              <mml:mi>s</mml:mi>
                            </mml:mrow>
                            <mml:mrow>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>c</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                            </mml:mrow>
                          </mml:msubsup>
                        </mml:mrow></math></script> is simply the proportion of the character state <i>s</i> for character <i>c</i>. If the character <i>c</i> is constant, then <i>h</i>(<i>c</i>) = 0. On the other hand, if the character <i>c</i> is highly variable, then each of the <i>r</i> character states is present with a relatively high proportion (e.g. <span class="MathJax_Preview" style=""></span><span class="MathJax" id="MathJax-Element-9-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;inline&quot;&gt;&lt;mrow&gt;&lt;msubsup&gt;&lt;mi&gt;&amp;#x3A0;&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;mo&gt;&amp;#x2248;&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mi&gt;r&lt;/mi&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-193" style="width: 6.075em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.694em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(1.231em, 1004.67em, 2.791em, -1000em); top: -2.412em; left: 0em;"><span class="mrow" id="MathJax-Span-194"><span class="mrow" id="MathJax-Span-195"><span class="msubsup" id="MathJax-Span-196"><span style="display: inline-block; position: relative; width: 1.885em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.89em, 4.134em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-197" style="font-family: MathJax_Math; font-style: italic;">Π<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.056em;"></span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.346em, 1000.93em, 4.311em, -1000em); top: -4.526em; left: 0.954em;"><span class="mrow" id="MathJax-Span-198"><span class="mo" id="MathJax-Span-199" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-200" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-201" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.563em, 1000.74em, 4.141em, -1000em); top: -3.857em; left: 0.831em;"><span class="mrow" id="MathJax-Span-202"><span class="mi" id="MathJax-Span-203" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span class="mi" id="MathJax-Span-204" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mo" id="MathJax-Span-205" style="font-family: MathJax_Main; padding-left: 0.278em;">≈</span><span class="mn" id="MathJax-Span-206" style="font-family: MathJax_Main; padding-left: 0.278em;">1</span><span class="mo" id="MathJax-Span-207" style="font-family: MathJax_Main;">/</span><span class="mi" id="MathJax-Span-208" style="font-family: MathJax_Math; font-style: italic;">r</span></span></span><span style="display: inline-block; width: 0px; height: 2.412em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.378em; border-left: 0px solid; width: 0px; height: 1.79em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"><mrow><msubsup><mi>Π</mi><mrow><mi>s</mi><mi>s</mi></mrow><mrow><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>≈</mo><mn>1</mn><mo>/</mo><mi>r</mi></mrow></math></span></span><script type="math/mml" id="MathJax-Element-9"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline"><mml:mrow>
                          <mml:msubsup>
                            <mml:mi>Π</mml:mi>
                            <mml:mrow>
                              <mml:mi>s</mml:mi>
                              <mml:mi>s</mml:mi>
                            </mml:mrow>
                            <mml:mrow>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>c</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                            </mml:mrow>
                          </mml:msubsup>
                          <mml:mo>≈</mml:mo>
                          <mml:mn>1</mml:mn>
                          <mml:mo>/</mml:mo>
                          <mml:mi>r</mml:mi>
                        </mml:mrow></math></script>), implying that <i>h(c)</i> is close to 1, its maximal value. Therefore, <i>h</i> allows the level of variability of a character to be quantified [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Schneider TD, Stephens RM: Sequence logos: a new way to display consensus sequences. Nucl Acids Res. 1990, 18: 6097-6100. 10.1093/nar/18.20.6097." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR21" id="ref-link-section-d16311e2702">21</a>]. Unfortunately, using <i>h</i> with the identity matrix <b>I</b><sub>
                    <i>r</i>
                  </sub> (as suggested in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Swingley WD, Blankenship RE, Raymond J: Integrating Markov clustering and molecular phylogenetics to reconstruct the cyanobacterial species tree from conserved protein families. Mol Biol Evol. 2008, 25: 643-654. 10.1093/molbev/msn034." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR15" id="ref-link-section-d16311e2717">15</a>]) suffers from biases. For example, when considering amino acid sequences, if a given character <i>c</i><sub>1</sub> is only made of the four residues I, L, M and V, each with 25% proportion, and a second character <i>c</i><sub>2</sub> is made of the four residues C, Q, W and Y, also with identical proportions, then formula (1) with matrix <b>I</b><sub>
                    <i>r</i>
                  </sub> returns <i>h</i>(<i>c</i><sub>1</sub>) = <i>h</i>(<i>c</i><sub>2</sub>) = -4× 0.25 log<sub>20</sub> 0.25 ≈ 0.462, and then indicates the same level of variability for both characters <i>c</i><sub>1</sub> and <i>c</i><sub>2</sub>, while residues in character <i>c</i><sub>1</sub> are much more likely to be substituted than those in character <i>c</i><sub>2</sub> [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Dayhoff MO, Schwartz RM, Orcutt BD: A model of evolutionary change in proteins. Atlas of Protein Sequence and Structure. Edited by: Dayhoff MO. 1978, Washington: National Biomedical Research Foundation, 5 (Suppl 3): 345-352." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR27" id="ref-link-section-d16311e2771">27</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Taylor WR: The classification of amino acid conservation. J Theor Biol. 1986, 119: 205-218. 10.1016/S0022-5193(86)80075-3." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR37" id="ref-link-section-d16311e2774">37</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Bordo D, Argos P: Suggestion for &quot;safe&quot; residue substitutions in site-directed mutagenesis. J mol Biol. 1991, 217: 721-729. 10.1016/0022-2836(91)90528-E." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR38" id="ref-link-section-d16311e2777">38</a>]. In contrast, using dedicated similarity matrices <b>S</b> <b><i>≠</i></b> <b>I</b> allows relevant substitution processes to be taken into account. As suggested in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Caffrey DR, Somaroo S, Hughes JD, Mintseris J, Huang ES: Are protein-protein interfaces more conserved in sequence than the rest of the protein surface?. Protein Sci. 2004, 13: 190-202. 10.1110/ps.03323604." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR34" id="ref-link-section-d16311e2790">34</a>], when using the Henikoff and Henikoff's [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Henikoff S, Henikoff JG: Amino acid substitution matrices from protein blocks. Proc Natl Acad Sci. 1992, 89: 10915-10919. 10.1073/pnas.89.22.10915." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR39" id="ref-link-section-d16311e2793">39</a>] BLOSUM50 target frequency matrix in formula (1), one obtains <i>h</i>(<i>c</i><sub>1</sub>) ≈ 0.300 and <i>h</i>(<i>c</i><sub>2</sub>) ≈ 0.453. Therefore, the function <i>h</i> with appropriate similarity matrix <b>S</b>
 computes score values that allow distinguishing among expected (i.e. 
biologically relevant) and ambiguous (e.g. source of noise) variability.</p><p>For
 practical use with amino acid sequences, BMGE provides the complete 
range of BLOSUM target frequency matrices (i.e. BLOSUM30, 35, 40, ..., 
95 from [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="BLOSUM matrices. [
                    http://blocks.fhcrc.org/blocks/uploads/blosum/
                    
                  ]" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR40" id="ref-link-section-d16311e2820">40</a>]; see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Henikoff S, Henikoff JG: Amino acid substitution matrices from protein blocks. Proc Natl Acad Sci. 1992, 89: 10915-10919. 10.1073/pnas.89.22.10915." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR39" id="ref-link-section-d16311e2823">39</a>] for more details) in order to estimate pertinent <i>h</i>
 values depending on the level of divergence between sequences. As shown
 in simulation results (see below), our entropy-based character trimming
 method performs better when using stringent matrices (e.g. BLOSUM95) 
for closely related sequences, and, reciprocally, when using more 
relaxed matrices (e.g. BLOSUM30) for distantly related sequences. By 
default, BMGE uses the popular BLOSUM62 matrix [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Eddy SR: Where did the BLOSUM62 alignment score matrix come from?. Nat Biotechnol. 2004, 22: 1035-1036. 10.1038/nbt0804-1035." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR41" id="ref-link-section-d16311e2829">41</a>].</p><p>For DNA sequences, PAM matrices are first computed. BMGE uses a transition/transversion ratio <i>κ</i>
 (= 2.0 by default) to compute the PAM-1 4 × 4 matrix: the four diagonal
 elements are all 0.99, and the off diagonal elements are 0.01 <i>κ</i> (2 + <i>κ</i>)<sup>-1</sup> for transition and 0.01(2 + <i>κ</i>)<sup>-1</sup> for transversion (see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="States DJ, Gish W, Altschul SF: Improved sensitivity of nucleic acid database searches using application-specific scoring matrices. Methods: A Companion to Methods Enzymol. 1991, 3: 66-70. 10.1016/S1046-2023(05)80165-3." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR42" id="ref-link-section-d16311e2852">42</a>] for more details about the PAM-1 calculation for DNA). Given a prefixed integer <i>η</i> (= 100 by default), BMGE then computes the PAM-<i>η</i> matrix (= PAM-1 <sup><i>η</i></sup> [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Dayhoff MO, Schwartz RM, Orcutt BD: A model of evolutionary change in proteins. Atlas of Protein Sequence and Structure. Edited by: Dayhoff MO. 1978, Washington: National Biomedical Research Foundation, 5 (Suppl 3): 345-352." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR27" id="ref-link-section-d16311e2865">27</a>]), which is finally used by BMGE as similarity matrix <b>S</b>
 in formula (1). In a similar way as the previously described amino acid
 framework, our character trimming method is more accurate when using a 
stringent matrix (e.g. PAM-1) with closely related DNA sequences, and 
when using a relaxed matrix (e.g. PAM-250) with distantly related ones.</p><h3 class="c-article__sub-heading u-h3" id="Sec6">Stationary-based character trimming</h3><p>Given two aligned sequences (e.g. taken from the multiple sequence alignment), one can use a <i>r×r</i> divergence matrix <b>F</b>
 to represent the relative proportion of each of the possible character 
state pairs in the pairwise comparison of these two sequences (as 
schematized for DNA sequences by formula (19) in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Swofford DL, Olsen GJ, Waddell PJ, Hillis DM: Phylogenetic inference. Molecular Systematics. Edited by: Hillis DM, Moritz C, Mable BK. 1996, Sunderland: Sinauer Associates, 407-514." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR11" id="ref-link-section-d16311e2886">11</a>]). If these two sequences have similar character state composition, then, for each character state <i>s</i> = 1, 2,...,<i>r</i> , this sequence pair verifies the null hypothesis <b>F</b><sub>
                    <i>s.</i>
                  </sub>= <b>F</b><sub>
                    <i>.s</i>
                  </sub>, named the marginal homogeneity (e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Stuart A: A test for homogeneity of the marginal distributions in a two-way classification. Biometrika. 1955, 42: 412-416." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR30" id="ref-link-section-d16311e2912">30</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Uesaka H: Validity and applicability of several tests for comparing marginal distributions of a square table with ordered categories. Behaviormetrika. 1991, 30: 65-78. 10.2333/bhmk.18.30_65." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR43" id="ref-link-section-d16311e2915">43</a>]) or marginal symmetry (e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Ababneh F, Jermiin LS, Ma C, Robinson J: Matched-pairs tests of homogeneity with applications to homologous nucleotide sequences. Bioinformatics. 2006, 22: 1225-1231. 10.1093/bioinformatics/btl064." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR31" id="ref-link-section-d16311e2918">31</a>]). A not-too-low <i>p</i>-value returned by the Stuart's <i>χ</i><sup>2</sup> test [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Stuart A: A test for homogeneity of the marginal distributions in a two-way classification. Biometrika. 1955, 42: 412-416." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR30" id="ref-link-section-d16311e2929">30</a>] on <b>F</b>
 allows the marginal homogeneity/symmetry to be assessed; then, one can 
assess that two homologous sequences arise from a nonstationary 
evolutionary process if the Stuart's test returns a <i>p</i>-value close to zero (e.g. &lt; 0.1). This test being essentially based on numerical linear algebra operations (see e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Ababneh F, Jermiin LS, Ma C, Robinson J: Matched-pairs tests of homogeneity with applications to homologous nucleotide sequences. Bioinformatics. 2006, 22: 1225-1231. 10.1093/bioinformatics/btl064." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR31" id="ref-link-section-d16311e2939">31</a>]
 for more details about its computation from two sequences), BMGE 
implements it with, on the one hand, matrix operations available in the 
JAMA package [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="JAMA: A Java Matrix Package. [
                    http://math.nist.gov/javanumerics/jama/
                    
                  ]" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR35" id="ref-link-section-d16311e2942">35</a>], and, on the other hand, with fast and accurate numerical algorithms for estimating <i>χ</i><sup>2</sup> cumulative distribution functions (adapted in Java from the C code sources available p. 216-219 in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Press WH, Teukolsky SA, Vetterling WT, Flannery BP: Numerical Recipes in C -- The Art of Scientific Computing. 1992, Cambridge: Cambridge University Press, 2" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR44" id="ref-link-section-d16311e2949">44</a>]).</p><p>If
 a multiple sequence alignment seems to be compositionally 
heterogeneous, BMGE implements a stationary-based character trimming in 
order to obtain a compositionally homogeneous alignment. Given a 
multiple alignment of <i>n</i> sequences, for each possible pair of distinct sequences <i>i</i>, <i>j</i> (i.e. 1 ≤ <i>i</i> &lt; <i>j</i> ≤ <i>n</i> ), BMGE computes the Stuart's test <i>p</i>-value, denoted <i>p</i><sub>
                    <i>ij</i>
                  </sub> . If there is at least one pair of sequences for which <i>p</i><sub>
                    <i>ij</i>
                  </sub> &lt; 0.1, then BMGE progressively removes the characters <i>c</i> ranked in function of their decreasing entropy-like <i>h</i>(<i>c</i>) values --as estimated by formula (1)-- until <i>p</i><sub>
                    <i>ij</i>
                  </sub> &gt; 0.1 for every pairs of sequences <i>i</i>, <i>j</i>. This first crude character removal approach leads to a set <i>C</i> of compositionally homogeneous characters. Then, BMGE aims at integrating the set <i>C</i>
 with some of the previously removed characters, in order to obtain a 
set of compositionally homogeneous characters of maximal size.</p><p>To do so, BMGE uses an iterative add-and-remove approach of characters. Defining <i>p</i><sub>
                    <i>ij</i>
                  </sub> <sup>(<i>c</i>) </sup>as the Stuart's test <i>p</i>-value for the sequence pair <i>i</i>, <i>j</i> after adding the character <i>c</i> inside the set <i>C</i>, BMGE computes the following score for each character <i>c</i> <span class="stix">∉</span> <i>C</i> :</p><div id="Equ4" class="c-article-equation"><div class="c-article-equation__content"><span class="MathJax_Preview" style=""></span><div class="MathJax_Display" style="text-align: left;"><span class="MathJax" id="MathJax-Element-10-Frame" tabindex="0" style="text-align: left; position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;block&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3C3;&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mstyle displaystyle=&quot;true&quot;&gt;&lt;munder&gt;&lt;mo&gt;&amp;#x2211;&lt;/mo&gt;&lt;mrow&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;&amp;#x2264;&lt;/mo&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo&gt;&amp;lt;&lt;/mo&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;mo&gt;&amp;#x2264;&lt;/mo&gt;&lt;mi&gt;n&lt;/mi&gt;&lt;/mrow&gt;&lt;/munder&gt;&lt;mrow&gt;&lt;mi&gt;log&lt;/mi&gt;&lt;/mrow&gt;&lt;/mstyle&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;msubsup&gt;&lt;mi&gt;p&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;msub&gt;&lt;mi&gt;p&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;.&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-209" style="width: 15.032em; display: inline-block;"><span style="display: inline-block; position: relative; width: 11.628em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(1.231em, 1011.55em, 3.873em, -1000em); top: -2.412em; left: 0em;"><span class="mrow" id="MathJax-Span-210"><span class="mrow" id="MathJax-Span-211"><span class="mi" id="MathJax-Span-212" style="font-family: MathJax_Math; font-style: italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.001em;"></span></span><span class="mo" id="MathJax-Span-213" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-214" style="font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-215" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-216" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mstyle" id="MathJax-Span-217" style="padding-left: 0.278em;"><span class="mrow" id="MathJax-Span-218"><span class="munder" id="MathJax-Span-219"><span style="display: inline-block; position: relative; width: 2.963em; height: 0px;"><span style="position: absolute; clip: rect(2.926em, 1001.39em, 4.584em, -1000em); top: -4.005em; left: 0.76em;"><span class="mo" id="MathJax-Span-220" style="font-family: MathJax_Size2; vertical-align: 0em;">∑</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.405em, 1002.95em, 4.379em, -1000em); top: -2.918em; left: 0em;"><span class="mrow" id="MathJax-Span-221"><span class="mn" id="MathJax-Span-222" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-223" style="font-size: 70.7%; font-family: MathJax_Main;">≤</span><span class="mi" id="MathJax-Span-224" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-225" style="font-size: 70.7%; font-family: MathJax_Main;">&lt;</span><span class="mi" id="MathJax-Span-226" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span><span class="mo" id="MathJax-Span-227" style="font-size: 70.7%; font-family: MathJax_Main;">≤</span><span class="mi" id="MathJax-Span-228" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">n</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mrow" id="MathJax-Span-229" style="padding-left: 0.167em;"><span class="mi" id="MathJax-Span-230" style="font-family: MathJax_Main;">log</span></span></span></span><span class="mo" id="MathJax-Span-231" style="font-family: MathJax_Main;">(</span><span class="msubsup" id="MathJax-Span-232"><span style="display: inline-block; position: relative; width: 1.434em; height: 0px;"><span style="position: absolute; clip: rect(3.434em, 1000.5em, 4.328em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-233" style="font-family: MathJax_Math; font-style: italic;">p</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.346em, 1000.93em, 4.311em, -1000em); top: -4.526em; left: 0.503em;"><span class="mrow" id="MathJax-Span-234"><span class="mo" id="MathJax-Span-235" style="font-size: 70.7%; font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-236" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span class="mo" id="MathJax-Span-237" style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.409em, 1000.61em, 4.279em, -1000em); top: -3.702em; left: 0.503em;"><span class="mrow" id="MathJax-Span-238"><span class="mi" id="MathJax-Span-239" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-240" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mo" id="MathJax-Span-241" style="font-family: MathJax_Main;">/</span><span class="msub" id="MathJax-Span-242"><span style="display: inline-block; position: relative; width: 1.113em; height: 0px;"><span style="position: absolute; clip: rect(3.434em, 1000.5em, 4.328em, -1000em); top: -4.005em; left: 0em;"><span class="mi" id="MathJax-Span-243" style="font-family: MathJax_Math; font-style: italic;">p</span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; top: -3.855em; left: 0.503em;"><span class="mrow" id="MathJax-Span-244"><span class="mi" id="MathJax-Span-245" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-246" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mo" id="MathJax-Span-247" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-248" style="font-family: MathJax_Main;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.412em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.774em; border-left: 0px solid; width: 0px; height: 3.186em;"></span></span></nobr><span class="MJX_Assistive_MathML MJX_Assistive_MathML_Block" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><mrow><mi>σ</mi><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo><mo>=</mo><mstyle displaystyle="true"><munder><mo>∑</mo><mrow><mn>1</mn><mo>≤</mo><mi>i</mi><mo>&lt;</mo><mi>j</mi><mo>≤</mo><mi>n</mi></mrow></munder><mrow><mi>log</mi></mrow></mstyle><mo stretchy="false">(</mo><msubsup><mi>p</mi><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mo stretchy="false">(</mo><mi>c</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>/</mo><msub><mi>p</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo stretchy="false">)</mo><mo>.</mo></mrow></math></span></span></div><script type="math/mml" id="MathJax-Element-10"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block"><mml:mrow>
                          <mml:mi>σ</mml:mi>
                          <mml:mo stretchy="false">(</mml:mo>
                          <mml:mi>c</mml:mi>
                          <mml:mo stretchy="false">)</mml:mo>
                          <mml:mo>=</mml:mo>
                          <mml:mstyle displaystyle="true">
                            <mml:munder>
                              <mml:mo>∑</mml:mo>
                              <mml:mrow>
                                <mml:mn>1</mml:mn>
                                <mml:mo>≤</mml:mo>
                                <mml:mi>i</mml:mi>
                                <mml:mo>&lt;</mml:mo>
                                <mml:mi>j</mml:mi>
                                <mml:mo>≤</mml:mo>
                                <mml:mi>n</mml:mi>
                              </mml:mrow>
                            </mml:munder>
                            <mml:mrow>
                              <mml:mi>log</mml:mi>
                            </mml:mrow>
                          </mml:mstyle>
                          <mml:mo stretchy="false">(</mml:mo>
                          <mml:msubsup>
                            <mml:mi>p</mml:mi>
                            <mml:mrow>
                              <mml:mi>i</mml:mi>
                              <mml:mi>j</mml:mi>
                            </mml:mrow>
                            <mml:mrow>
                              <mml:mo stretchy="false">(</mml:mo>
                              <mml:mi>c</mml:mi>
                              <mml:mo stretchy="false">)</mml:mo>
                            </mml:mrow>
                          </mml:msubsup>
                          <mml:mo>/</mml:mo>
                          <mml:msub>
                            <mml:mi>p</mml:mi>
                            <mml:mrow>
                              <mml:mi>i</mml:mi>
                              <mml:mi>j</mml:mi>
                            </mml:mrow>
                          </mml:msub>
                          <mml:mo stretchy="false">)</mml:mo>
                          <mml:mo>.</mml:mo>
                        </mml:mrow></math></script></div></div><p>If σ (<i>c</i>) &gt; 0, then adding character <i>c</i> inside <i>C</i> leads to an increase for most of the <i>n(n-1)</i>/2 <i>p</i>-values; reciprocally, adding characters <i>c</i> with σ (<i>c</i>)&lt; 0 leads to a (not wished) overall decrease of the <i>p</i>-values. BMGE then progressively removes the characters <i>c</i> <span class="stix">∉</span> <i>C</i> from the initial multiple sequence alignment following the increasing order of their respective σ(<i>c</i>) value, i.e. from the smaller (negative) to the larger (positive) σ score value. After each character removal, every <i>p</i><sub>
                    <i>ij</i>
                  </sub> are re-estimated. When all <i>p</i><sub>
                    <i>ij</i>
                  </sub> &gt; 0.1, BMGE stops removing characters from 
the initial alignment and considers the remaining (i.e. not removed) 
characters as the new set <i>C</i>. Finally, BMGE iteratively performs these add-and-remove operations until the set <i>C</i> of compositionally homogeneous characters cannot be increased further.</p></div></div></section><section aria-labelledby="Sec7"><div class="c-article-section js-article-section" id="Sec7-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec7">Results and Discussion</h2><div class="c-article-section__content js-collapsible-section" id="Sec7-content"><p>In
 order to assess the utility of multiple alignment character trimming to
 infer more accurate phylogenetic trees and to compare the respective 
performances of BMGE (with different similarity matrices) with other 
available character trimming methods (i.e. Gblocks [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e3238">7</a>]; Noisy [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Dress AWM, Flamm C, Fritzsch G, Grünewald S, Kruspe M, Prohaska SJ, Stadler PF: Noisy: Identification of problematic columns in multiple sequence alignments. Algorithms for Molecular Biology. 2008, 3: 7-10.1186/1748-7188-3-7." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR14" id="ref-link-section-d16311e3241">14</a>]; trimAl [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Capella-Gutiérez S, Silla-Martínez JM, Gabaldón T: trimAl: a tool for automated alignment triming in large-scale phylogenetic analyses. Bioinformatics. 2009, 25: 1972-1973. 10.1093/bioinformatics/btp348." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR16" id="ref-link-section-d16311e3244">16</a>]), we carried out computer simulations and real case studies. Protocol and results are described below.</p><h3 class="c-article__sub-heading u-h3" id="Sec8">Simulation results with entropy-based character trimming</h3><p>The 200 first 40-taxon trees available in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Artificial datasets used in [48]. [
                    http://www.atgc-montpellier.fr/phyml/datasets.php
                    
                  ]" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR45" id="ref-link-section-d16311e3254">45</a>]
 were selected as model trees to generate artificial amino acid sequence
 datasets. On the one hand, in order to simulate phylogenetically 
informative characters, 10 clusters of 40 sequences each were generated 
using Seq-Gen [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Rambaut A, Grassly NC: Seq-Gen: an application for the Monte Carlo simulation of DNA sequence evolution along phylogenetic trees. Comput Appl Biosci. 1997, 13: 235-238." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR46" id="ref-link-section-d16311e3257">46</a>] under the JTT model [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Jones D, Taylor W, Thornton J: The rapid generation of mutation data matrices from protein sequences. Comput Appl Biosci. 1992, 8: 275-282." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR47" id="ref-link-section-d16311e3260">47</a>]
 from each of the 200 model trees. Sequence lengths for each of these 10
 sequence clusters were randomly drawn from 30 to 70 amino acids. On the
 other hand, in order to simulate uninformative/variable characters, 
amino acid sequences were generated in the same way from a 40-taxon star
 tree (i.e. a tree with 40 leaves and a unique internal node).</p><p>Following
 these two steps, for each of the 200 initial model trees, we generated 
20 clusters of 40 amino acid sequences of lengths 50 ± 20, ten 
containing informative phylogenetic signal, and ten containing 
uninformative/variable signal. Finally, from each of these 200 sets of 
20 sequence clusters, 10 clusters were randomly drawn and concatenated, 
in order to produce a dataset composed of 200 clusters of 40 sequences 
of 500-amino acid length on average, each containing an equal mixture of
 phylogenetically informative and uninformative regions.</p><p>This 
simulation procedure to generate 200 clusters of sequences containing 
50% (on average) of uninformative/variable characters was repeated three
 times, each with initial model tree branch lengths (see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Artificial datasets used in [48]. [
                    http://www.atgc-montpellier.fr/phyml/datasets.php
                    
                  ]" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR45" id="ref-link-section-d16311e3268">45</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Guindon S, Gascuel O: A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol. 2003, 52: 696-704. 10.1080/10635150390235520." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR48" id="ref-link-section-d16311e3271">48</a>]
 for more details) multiplied by a divergence factor of 1, 2 and 3, 
respectively, in order to mimic from closely- to distantly-related 
sequence clusters. Each of these (3 × 200=)600 clusters of simulated 
amino acid sequences were aligned with MUSCLE [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Edgar RC: MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 2004, 32: 1792-1797. 10.1093/nar/gkh340." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR49" id="ref-link-section-d16311e3274">49</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Edgar RC: MUSCLE: a multiple sequence alignment method with reduced time and space complexity. BMC Bioinformatics. 2004, 5: 113-10.1186/1471-2105-5-113." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR50" id="ref-link-section-d16311e3277">50</a>].
 The average lengths of these multiple sequence alignments are 513, 561,
 and 573 for the levels of divergence ×1, ×2, and ×3, respectively.</p><p>The
 software BMGE was applied on these multiple sequence alignments with 
three similarity matrices: BLOSUM30, BLOSUM62 and BLOSUM95. As a 
comparison, character trimming was also performed by using three 
available softwares.</p><p>Gblocks (0.91 b) was used with 'strict' and 'relaxed' parameter sets (see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e3286">7</a>]
 for a precise description of these two parameter sets). However, the 
'strict' conditions (default parameters in Gblocks) are indeed very 
stringent (e.g. no character was selected in ≈40% of the multiple 
sequence alignments with level of divergence ×3), and phylogenetic trees
 inferred from the remaining blocks (when these existed) were always 
less accurate than those inferred from the blocks returned by the 
'relaxed' conditions (results not shown). Worse results than those 
obtained with the 'relaxed' conditions were also observed with 
alternative parameter sets (e.g. those described by [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Rodríguez-Ezpeleta N, Brinkmann H, Burey SC, Roure B, Burger G, Löffelhardt W, Philippe H, Lang BF: Monophyly of primary photosynthetic eukaryotes: green plants, red algae, and glaucophytes. Curr Biol. 2005, 15: 1325-1330. 10.1016/j.cub.2005.06.040." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR12" id="ref-link-section-d16311e3289">12</a>]; results not shown). Then, results from Gblocks presented below are only those obtained with the 'relaxed' conditions.</p><p>The software Noisy was used with the --nogap options. Several other options were tested (especially the --cutoff one; see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Dress AWM, Flamm C, Fritzsch G, Grünewald S, Kruspe M, Prohaska SJ, Stadler PF: Noisy: Identification of problematic columns in multiple sequence alignments. Algorithms for Molecular Biology. 2008, 3: 7-10.1186/1748-7188-3-7." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR14" id="ref-link-section-d16311e3295">14</a>] for more details) but the Noisy default options allows better results to be observed with our simulated datasets.</p><p>The
 software trimAl (1.2rev59) allows three trimming methods (among 
numerous ones) to be used: 'gappyout', 'strictplus' and 'automated1'. 
Since the 'gappyout' method mainly focuses on highly gapped regions, 
this approach removed too few characters in our poorly gapped simulated 
datasets. Consequently, the 'gappyout' results are not shown since they 
are very close to those observed with the initial (i.e. non-trimmed) 
multiple sequence alignments.</p><p>As expected in regard to the 
sequence simulation protocol (see above), initial multiple sequence 
alignments returned by MUSCLE are composed of approximately 50% 
phylogenetically informative characters (i.e. those characters that are 
not generated by Seq-Gen from the star tree). For each trimming method, 
the character set of an initial multiple sequence alignment is then 
partitioned into four subsets: true positives and false negatives (i.e. 
phylogenetically informative characters that are selected or not by the 
trimming method, respectively), false positives (i.e. characters 
selected by the trimming method and that are not phylogenetically 
informative), and true negatives (i.e. characters that are not 
phylogenetically informative and are indeed not selected by the trimming
 method). Denoting <i>tp</i> and <i>tn</i> , the number of true positive and negative characters, respectively, and <i>fp</i> and <i>fn</i> , the number of false positive and negative characters, respectively, the true positive rate <i>tpr = tp/</i>(<i>tp</i> + <i>fn</i>) and false positive rate <i>fpr = fp/</i>(<i>fp</i> + <i>tn</i>) were computed. For each trimming method and each of the three levels of divergence (i.e. ×1, ×2, ×3), a ROC graph (e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Egan JP: Signal detection theory and ROC analysis, series in cognition and perception. 1975, New York: Academic Press" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR51" id="ref-link-section-d16311e3334">51</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Fawcett T: An introduction to ROC analysis. Pattern Recogn Lett. 2006, 27: 861-874. 10.1016/j.patrec.2005.10.010." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR54" id="ref-link-section-d16311e3338">54</a>])
 depicting the plot of the true (y-axis) and false (x-axis) positive 
rates for each of the 200 datasets is represented in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig1">1</a>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1"><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 1</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig1_HTML.jpg?as=webp"><img aria-describedby="figure-1-desc" src="12862_2010_Article_1426_Fig1_HTML.jpg" alt="figure1"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>ROC graphs plotting true (y-axis) and false (x-axis) positive rates for seven character trimming methods</b>.
 The best methods (i.e., that minimize both the number of true negative 
and false positive characters) are those with the corresponding cloud 
concentrated around the upper left point (0,1). Under each ROC graph, 
the average <i>L</i><sub>1</sub> distance between each point and the 
(0,1) point is given. For each level of divergence, the best (i.e. 
lower) distance is written in boldface characters. Average <i>L</i><sub>1</sub> distances that are not significantly different to this best value (as assessed by a sign test) are underscored.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/1" data-track-dest="link:Figure1 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>All initial (i.e. non-trimmed) multiple sequence alignments (i.e. <i>tn = fn =</i> 0) correspond to the (1,1) point (i.e. <i>fpr = fp/fp = tpr = tp/tp =</i>1); reciprocally, the removal of all characters (i.e. <i>tp = fp =</i> 0) corresponds to the (0,0) point (i.e. <i>fpr =</i> 0/<i>tn = tpr =</i> 0/<i>fn</i>
 = 0). A cloud close to the (1,1) point in the ROC graph then indicates 
that the corresponding trimming method is liberal (i.e. it keeps too 
many uninformative/variable characters). Conversely, conservative 
trimming methods (i.e. that remove too many phylogenetically informative
 characters) correspond to clouds close to the (0,0) point. Ideally, the
 best method selects only those characters that are phylogenetically 
informative (i.e. <i>fn = fp =</i> 0), and then corresponds to the (0,1) point inside the ROC graph (i.e. <i>fpr =</i> 0/<i>tn =</i> 0 and <i>tpr = tp/tp =</i> 1). For each case in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig1">1</a>, the <i>L</i><sub>1</sub> distance between each point and the (0,1) point (= 1-<i>tpr+fpr</i>
 ) was computed, averaged and written under its ROC graph. For each of 
the three levels of divergence (i.e. ×1, ×2, ×3), a sign test [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="McStewart W: A note on the power of the sign test. Ann Math Stat. 1941, 12: 279-303. 10.1214/aoms/1177731710." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR55" id="ref-link-section-d16311e3417">55</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Hemelrijk J: A theorem on the sign test when ties are present. Proc Nederl Akad Weten Ser A. 1952, 55: 322-" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR57" id="ref-link-section-d16311e3420">57</a>] was performed to assess the statistical significance between the best average <i>L</i><sub>1</sub> distance measure (i.e. the lowest) and the other ones. For each level of divergence in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig1">1</a>, an average <i>L</i><sub>1</sub> distance measure is considered as non-significantly different to the best one if the <i>p</i>-value returned by the sign test is &gt; 5%.</p><p>Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig1">1</a> shows that, in all cases, trimAl has always <i>tpr</i> ≈ 1 but often <i>fpr</i> <span class="stix">≫</span> 0, indicating that it is too liberal (i.e. <i>fp</i> <span class="stix">≫</span> 0); moreover, <i>L</i><sub>1</sub>
 distances observed for trimAl are often among the worst, similarly to 
those observed for Noisy (except for the level of divergence ×3). We can
 also see from Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig1">1</a>
 that Gblocks with relaxed conditions presents very good performance 
with closely related sequences (i.e. level of divergence ×1), but 
induces <i>tpr</i> «1 when the level of divergence increases (i.e. from ×1 to ×3), showing that it becomes too conservative (i.e. <i>fn</i> <span class="stix">≫</span>
 0). As expected, BMGE with stringent option (i.e. BLOSUM95) corresponds
 to points that are more concentrated around the y-axis as the level of 
sequence divergence increases, showing that the use of stringent 
similarity matrices such as ranging from BLOSUM80 to BLOSUM95 lead to 
conservative character trimming with distantly related sequences (e.g. 
level of divergence ×3). Reciprocally, the use of the similarity matrix 
BLOSUM30 leads to selecting too many characters (i.e. <i>fpr</i> <span class="stix">≫</span> 0) with closely related sequences (i.e. level of divergence ×1). However, BMGE with BLOSUM30 allows minimizing both <i>fn</i> and <i>fp</i> when the level of divergence increases; indeed this last BMGE usage leads to the best average <i>L</i><sub>1</sub> distances for the levels of divergence ×2 and ×3.</p><h3 class="c-article__sub-heading u-h3" id="Sec9">Simulation results on phylogenetic tree accuracy</h3><p>For
 each of the three levels of divergence and from each multiple sequence 
alignment (i.e. the initial one as well as those outputted by BMGE, 
Gblocks, Noisy and trimAl), phylogenetic trees were inferred with 
several approaches. Maximum Likelihood (ML) trees were inferred by PhyML
 [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Guindon S, Gascuel O: A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol. 2003, 52: 696-704. 10.1080/10635150390235520." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR48" id="ref-link-section-d16311e3489">48</a>] with model JTT-Γ<sub>4</sub>. BioNJ [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Gascuel O: BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data. Mol Biol Evol. 1997, 14: 685-695." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR58" id="ref-link-section-d16311e3494">58</a>] trees were also inferred by PhyML with JTT distances. Maximum Parsimony (MP) trees were inferred by TNT [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Goloboff P, Farris S, Nixon KC: TNT (Tree analysis using New Technology) ver. 1.0. 2000, Published by the authors, Tucumán, Argentina" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR59" id="ref-link-section-d16311e3497">59</a>] with TBR branch swapping and parsimony ratchet [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Nixon KC: The parsimony ratchet, a new method for rapid parsimony analysis. Cladistics. 1999, 15: 407-414. 10.1111/j.1096-0031.1999.tb00277.x." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR60" id="ref-link-section-d16311e3500">60</a>].
 Bayesian inference was not performed because of the huge running time 
required by this approach. Moreover, Bayesian trees are expected to be 
very close to ML trees when inferred from our simulated datasets [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Ogden TH, Rosenberg MS: Multiple sequence alignment accuracy and phylogenetic inference. Syst Biol. 2006, 55: 314-328. 10.1080/10635150500541730." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR3" id="ref-link-section-d16311e3504">3</a>].</p><p>For
 each of the three levels of divergence, topological accuracy of the ML,
 BioNJ and MP trees inferred from the initial and the trimmed multiple 
sequence alignments was measured by the quartet distance [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Estabrook GF, McMorris FR, Meacham CA: Comparison of undirected phylogenetic trees based on subtrees of four evolutionary units. Syst Zool. 1985, 34: 193-200. 10.2307/2413326." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR61" id="ref-link-section-d16311e3510">61</a>] between each inferred tree and its corresponding model tree. All these distance measures are reported in Tables <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab2">2</a>, <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab3">3</a> and <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab4">4</a>
 for ML, BioNJ and MP trees, respectively, normalized in order to 
restrict these values to the interval [0,1], then averaged. For each of 
the three levels of divergence (i.e. ×1, ×2, ×3) and each of the three 
tree reconstruction methods (i.e. ML, BioNJ, MP), a sign test was 
performed to assess the statistical significance between the best 
average distance measure (i.e. the lowest) and the other ones. In each 
column of the Tables <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab2">2</a>, <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab3">3</a> and <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab4">4</a>, an entry is considered as non-significantly different to the best one if the <i>p</i>-value returned by the sign test is &gt; 5%.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-2"><figure><figcaption class="c-article-table__figcaption"><b id="Tab2" data-test="table-caption">Table
 2 Average quartet distances between the model trees and the ML trees 
inferred from the different trimmed multiple sequence alignments</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-category="article body" data-track-label="button" rel="nofollow" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/tables/2"><span>Full size table</span></a></div></figure></div><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-3"><figure><figcaption class="c-article-table__figcaption"><b id="Tab3" data-test="table-caption">Table
 3 Average quartet distances between the model trees and the BioNJ trees
 inferred from the different trimmed multiple sequence alignments</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-category="article body" data-track-label="button" rel="nofollow" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/tables/3"><span>Full size table</span></a></div></figure></div><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-4"><figure><figcaption class="c-article-table__figcaption"><b id="Tab4" data-test="table-caption">Table
 4 Average quartet distances between the model trees and the MP trees 
inferred from the different trimmed multiple sequence alignments</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-category="article body" data-track-label="button" rel="nofollow" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/tables/4"><span>Full size table</span></a></div></figure></div><p>Variable/uncertain
 characters contained in the initial multiple sequence alignments cause 
strong artefacts in the resulting phylogenetic trees, especially for 
BioNJ and MP trees (see Tables <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab2">2</a>, <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab3">3</a>,and <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab4">4</a>).
 When character trimming softwares are used, almost all reconstructed 
phylogenetic trees are closer to their model tree (Tables <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab2">2</a>, <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab3">3</a>, and <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab4">4</a>),
 showing that character trimming is a useful step prior any phylogenetic
 inference. However, it should be stressed that the ML approach is very 
robust to the noise introduced by uninformative/variable characters in 
our simulated datasets, mainly thanks to the Γ parameter. Even if our 
datasets were generated with equal rates across characters (see above), 
estimation of a Γ parameter was included in ML inference because 
preliminary tries without Γ parameter led to less accurate trees, 
especially those inferred from Noisy and trimAl outputs (results not 
shown). The Γ parameter then helps to compensate part of the 
phylogenetic noise contained in our datasets. However, when considering 
distantly-related sequences (e.g. level of divergence ×3), the ML 
approach (as well as the BioNJ and MP ones) needs a preliminary trimming
 step to infer significantly accurate trees, in agreement with results 
from previous simulation-based studies (e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Talavera G, Castresana J: Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. Syst Biol. 2007, 56: 564-577. 10.1080/10635150701472164." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR8" id="ref-link-section-d16311e4519">8</a>]).</p><p>In
 general, BMGE (when used with a BLOSUM substitution matrix adequate to 
the level of sequence divergence) allows reconstructing among the most 
accurate trees for each of the three levels of divergence and every tree
 reconstruction method used (Tables <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab2">2</a>, <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab3">3</a> and <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab4">4</a>).
 trimAl and Noisy infer the worst BioNJ and MP trees, whereas Gblocks 
with relaxed conditions produces good results as long as sequences are 
not too divergent. To the minor exception of Noisy with ML trees, BMGE 
trimming with the (less stringent) BLOSUM30 matrix allows reconstructing
 the significantly best trees with level of divergence ×3.</p><h3 class="c-article__sub-heading u-h3" id="Sec10">Simulation results on phylogenetic tree branch support</h3><p>In
 order to observe the impact of character trimming methods on branch 
supports inside phylogenetic trees, we have focused on two different 
approaches to estimate confidence values on the internal branches of a 
phylogenetic tree: the bootstrap-based support [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Felsenstein J: Confidence limits on phylogenies: an approach using the bootstrap. Evolution. 1985, 39: 783-791. 10.2307/2408678." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR62" id="ref-link-section-d16311e4542">62</a>] with BioNJ trees, and the approximate likelihood ratio test (aLRT; [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Anisimova M, Gascuel O: Approximate likelihood ratio test for branches: a fast, accurate and powerful alternative. Syst Biol. 2006, 55: 539-552. 10.1080/10635150600755453." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR63" id="ref-link-section-d16311e4545">63</a>]) as implemented by default in PhyML 3.0 [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Guindon S, Dufayard JF, Lefort V, Anisimova M, Hordijk W, Gascuel O: New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0. Syst Biol. 2010, 59: 307-321. 10.1093/sysbio/syq010." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR64" id="ref-link-section-d16311e4548">64</a>].</p><p>For
 each of the three levels of divergence (i.e. ×1, ×2, ×3) and from each 
multiple sequence alignment (i.e. the initial as well as the seven 
outputted by BMGE, Gblocks, Noisy and trimAl; see above), 100 
bootstrap-based replicates were generated, and 100 BioNJ trees were 
inferred from these multiple sequence alignment replicates. From these 
100 BioNJ bootstrap-based trees, bootstrap proportions were assessed on 
the internal branches of the corresponding (true) model tree. For each 
character trimming method and each level of divergence, we computed the 
distribution of the so-obtained bootstrap-based confidence values, as 
well as its average value (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig2">2</a>).
 Similarly, we also computed the distributions of aLRT branch support 
estimated on the (true) branches of the model trees from the different 
multiple sequence alignments, as well as their average values (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig3">3</a>).
 For each way to estimate confidence values (i.e. BioNJ bootstrap-based 
and aLRT-based ones) and each level of divergence (i.e. ×1, ×2, ×3), a <i>χ</i><sup>2</sup> test was performed to assess whether distributions are significantly different. In Figures <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig2">2</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig3">3</a>,
 a distribution is considered as non-significantly different to the best
 one (i.e. those corresponding to the highest average confidence value) 
if the <i>p</i>-value returned by the <i>χ</i><sup>2</sup> test is &gt; 5%.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2"><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 2</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig2_HTML.jpg?as=webp"><img aria-describedby="figure-2-desc" src="12862_2010_Article_1426_Fig2_HTML.jpg" alt="figure2"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>Distributions
 of the BioNJ bootstrap-based confidence values on the true branches of 
model trees estimated from initial (non-trimmed) multiple sequence 
alignments and from alignments returned by seven character trimming 
methods</b>. Average confidence values are written under each 
corresponding histogram. For each level of divergence, the best (i.e. 
higher) average confidence value is written in boldface characters. 
Average confidence values associated to distributions that are not 
significantly different to this best distribution (as assessed by a <i>χ</i><sup><i>2</i></sup> test) are underscored.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/2" data-track-dest="link:Figure2 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3"><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 3</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig3_HTML.jpg?as=webp"><img aria-describedby="figure-3-desc" src="12862_2010_Article_1426_Fig3_HTML.jpg" alt="figure3"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>Distributions
 of the aLRT confidence values on the true branches of model trees 
estimated from initial (non-trimmed) multiple sequence alignments and 
from alignments returned by seven character trimming methods</b>. See Figure 2.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/3" data-track-dest="link:Figure3 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>For
 each of the three levels of divergence and from each multiple sequence 
alignment, BioNJ-based bootstrap proportions were also estimated on the 
branches of the phylogenetic trees inferred by BioNJ (i.e. those used to
 compute quartet distances in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab3">3</a>).
 A bootstrap-based confidence value is then expected to be high when the
 corresponding branch is true (i.e. present in the model tree), and is 
expected to be low for false branches. Given a threshold <i>τ</i>, all 
branches of an inferred tree can be partitioned into four subsets: true 
positives or false negatives (i.e. true branches with confidence values ≥
 <i>τ</i> or &lt; <i>τ</i>, respectively), false positives or true negatives (i.e. false branches with confidence values ≥ <i>τ</i> or &lt; <i>τ</i>, respectively). As a consequence, there exists a point in the ROC space (see above) that is associated to a threshold value <i>τ</i>, and plotting these points for a large number of possible threshold values <i>τ</i> results in a so-called ROC curve (see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Metz CE: Basic principles of ROC analysis. Semin Nucl Med. 1978, 8: 283-298. 10.1016/S0001-2998(78)80014-2." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR52" id="ref-link-section-d16311e4652">52</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Fawcett T: An introduction to ROC analysis. Pattern Recogn Lett. 2006, 27: 861-874. 10.1016/j.patrec.2005.10.010." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR54" id="ref-link-section-d16311e4655">54</a>] for more details). ROC curves obtained by varying τ from 0 to 1 with 0.02-increment are displayed in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig4">4</a>, where up-right head of each ROC curve corresponds to lowest <i>τ</i> values, whereas down-left tail corresponds to highest <i>τ</i> values. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig5">5</a>
 represents ROC curves built in the same way from aLRT-based confidence 
values on the branches of the inferred ML trees (i.e. those used to 
compute quartet distances in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab2">2</a>). Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig4">4</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig5">5</a>
 also display the area under each ROC curve (AUC) that corresponds to 
the probability that a confidence value will be higher for a true branch
 than for a false branch (see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Hanley JA, McNeil BJ: The meaning and the use of the area under a receiver operating characteristic (ROC) curve. Radiology. 1982, 143: 29-36." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR65" id="ref-link-section-d16311e4681">65</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Fawcett T: An introduction to ROC analysis. Pattern Recogn Lett. 2006, 27: 861-874. 10.1016/j.patrec.2005.10.010." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR54" id="ref-link-section-d16311e4684">54</a>] for more details). For each of the three levels of divergence, a Z test (as described in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Fogarty J, Baker RS, Hudson SE: Case studies in the use of ROC curve analysis for sensor-based estimates in human computer interaction. Proceedings of Graphics Interface (GI 2005): 09-11 May 2005; Victoria, British Columbia. Edited by: Michael McCool. 2005, University of Waterloo, 129-136." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR66" id="ref-link-section-d16311e4687">66</a>]) was performed to assess the statistical significance between the best AUC (i.e. the highest) and the other ones. In Figures <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig4">4</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig5">5</a>, an AUC is considered as non-significantly different to the best one if the <i>p</i>-value returned by the Z test is &gt; 5%.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4"><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 4</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig4_HTML.jpg?as=webp"><img aria-describedby="figure-4-desc" src="12862_2010_Article_1426_Fig4_HTML.jpg" alt="figure4"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>ROC curves constructed by thresholding BioNJ bootstrap-based confidence values</b>.
 Best tree branch classifiers (i.e., that are able to associate higher 
confidence values to true branches than to false branches) are those 
that maximize the area under the ROC curve (AUC). For each simulation 
case, the AUC is given under the corresponding ROC space representation.
 For each column, the best (i.e. higher) AUC is written in boldface 
characters. AUCs that are not significantly different to this best value
 (as assessed by a Z test) are underscored.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/4" data-track-dest="link:Figure4 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5"><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 5</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig5_HTML.jpg?as=webp"><img aria-describedby="figure-5-desc" src="12862_2010_Article_1426_Fig5_HTML.jpg" alt="figure5"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>ROC curves constructed by thresholding aLRT-based confidence values</b>. See Figure 4.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/5" data-track-dest="link:Figure5 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>Broadly, Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig2">2</a>
 shows that the estimate of BioNJ bootstrap proportions from the initial
 (non-trimmed) multiple sequence alignments leads to very biased values,
 with a proportion of true branches with bootstrap-based confidence 
values ≤ 0.1 varying from 31% to 47%, and those &gt; 0.9 varying from 
18% to 20%. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig2">2</a>
 also shows that the highest average bootstrap-based confidence values 
are obtained from the character-trimming methods that best optimize both
 <i>tpr</i> and <i>fpr</i> criteria (see above and Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig1">1</a>). The same conclusions hold for aLRT with level of divergence ×1 (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig3">3</a>),
 with (significantly) best average aLRT values observed with BMGE and 
Gblocks. Surprisingly, for level of divergence ×3, the best average aLRT
 values are obtained from the characters selected by the most liberal 
character trimming methods, i.e. initial alignments and trimAl 
(strictplus and automated1).</p><p>AUC values in Figures <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig4">4</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig5">5</a>
 show that initial (non-trimmed) multiple sequence alignments lead to 
more incorrect confidence values than those estimated from characters 
selected by trimming methods. Interestingly, Figures <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig4">4</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig5">5</a>
 show that aLRT-based ROC curves induce highest AUC values on average 
that bootstrap-based ones (with the slight exception of BMGE with 
BLOSUM95 for level of divergence ×3). This shows that aLRT-based 
confidence values are more able to discreminate true and false branches 
than BioNJ bootstrap-based ones. ROC curve shapes also show that 
trimming methods lead to more precise confidence values. Indeed, in 
Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig4">4</a>, the down-left tail point of each ROC curve (i.e. corresponding to threshold <i>τ</i> = 0.98) always has highest <i>tpr</i>
 (y-axis) for trimmed alignments than for initial ones; this shows that a
 larger proportion of true branches with BioNJ bootstrap-based 
confidence values &gt;0.98 is observed with trimmed alignments than with
 initial alignments. For initial multiple sequence alignments and for 
level of divergence ×2 and ×3 in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig4">4</a>, the up-right head points of these two ROC curves (i.e. each corresponding to threshold <i>τ</i> = 0.02) have <i>tpr</i>
 &lt; 1; this means that there exists some true branches with BioNJ 
bootstrap-based confidence value &lt; 0.02. Notably, this is not the 
case when using character trimming methods. These two tendencies on <i>tpr</i> values for both extremities of the ROC curves are in agreement with the distributions in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig2">2</a>. Nevertheless, when observing the <i>fpr</i> ranges (x-axis) of the ROC curves in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig4">4</a>, trimmed multiple sequence alignments all induce larger <i>fpr</i>
 values than initial ones; this shows that when using character trimming
 methods instead of initial multiple sequence alignments, there is an 
increase in the proportion of false branches in BioNJ trees with high 
confidence value (e.g. down-left tail of ROC curves corresponding to <i>τ</i>
 = 0.98) and a decrease of the proportion of false branches with low 
confidence values (e.g. up-right tail of ROC curves corresponding to <i>τ</i> = 0.02). This last tendency is clearly obvious with level of divergence ×3 (see Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig4">4</a>).
 However, shapes of ROC curves constructed by thresholding aLRT-based 
confidence values on ML trees do not seems to be strongly modified by 
trimming methods (see Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig5">5</a>).
 More precisely, for levels of divergence ×1 and ×2, Gblocks and BMGE 
(with BLOSUM62) present always among the best results. For level of 
divergence ×3, Noisy and BMGE with BLOSUM30 present the significantly 
best AUC values for aLRT-based confidence values.</p><p>To sum up, these
 simulation results show that, by selecting among variable characters 
those with biologically-relevant expected variability thanks to the use 
of similarity matrices, BMGE often presents results that are among the 
(significantly) best (e.g. phylogenetic accuracy, better confidence 
values for true branches) and leads to a less biased phylogenetic 
signal.</p><h3 class="c-article__sub-heading u-h3" id="Sec11">Entropy-based character trimming in a phylogenomics context</h3><p>In
 order to illustrate the benefit of using character trimming approaches,
 we have re-analysed the multi-gene dataset used by Castresana (2000) to
 describe the usefulness of Gblocks [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e4837">7</a>] in selecting suited characters. This amino-acid dataset is composed by ten genes (i.e. three subsunits of cytochrome <i>c</i> oxidase CO1-CO3, one subunit of cytochrome <i>c</i>-ubiquinol
 oxidoreductase CYTb, and six subunits of NADH desydrogenase ND1-ND5 and
 ND4L) gathered from the complete mitochondrial genome of 16 eukaryotes 
(11 unikonts, 4 archaeplastida, 1 excavate), and from <i>Paracoccus denitrificans</i>, an α-proteobacterium used as outgroup (see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e4849">7</a>] for more details).</p><p>Protein
 sequences were aligned with MUSCLE, and each of the 10 multiple 
sequence alignments were trimmed by BMGE, Gblocks, trimAl and Noisy with
 the same parameters used in the previous simulations. For each of these
 seven character trimming approaches as well as the initial 
(non-trimmed) multiple sequence alignment, the ten so-obtained character
 matrices were concatenated into a single character supermatrix with the
 software Concatenate [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Concatenate: a software to build supermatrices of characters. [
                    http://www.supertriplets.univ-montp2.fr/PhyloTools.php
                    
                  ]" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR67" id="ref-link-section-d16311e4855">67</a>]. Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab5">5</a>
 shows the different number of characters of these eight supermatrices. 
ML trees were inferred with PhyML from each character supermatrix with 
the mtREV model of amino acid substitution [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Adachi J, Hasegawa M: Model of amino acid substitution in proteins encoded by mitochondrial DNA. J Mol Evol. 1996, 42: 459-468. 10.1007/BF02498640." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR68" id="ref-link-section-d16311e4861">68</a>].
 Parameters defining the shape of the Γ distribution (8 categories) and 
the proportion of invariable characters were both left as free. The 
different log-likelihood (log-<i>lk</i>) estimated for each inferred ML 
trees were normalized by the total number of characters for better 
comparison and are shown in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab5">5</a>. ML bootstrap-based (100 replicates) and aLRT confidence values were assessed on the branches of the different inferred trees.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-5"><figure><figcaption class="c-article-table__figcaption"><b id="Tab5" data-test="table-caption">Table
 5 Lengths of the different character supermatrices, and average 
log-likelihood per character of the corresponding ML trees.</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-category="article body" data-track-label="button" rel="nofollow" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/tables/5"><span>Full size table</span></a></div></figure></div><p>It should be stressed that our initial character supermatrix (i.e. 4,530 characters; see Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab5">5</a>) is slightly larger than the number of characters inside the original character supermatrix in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e5107">7</a>]
 (i.e. 4,453); this is due to the use of different multiple sequence 
alignment softwares. It should also be stressed that the log-<i>lk</i> per character estimated from our initial character supermatrix (i.e. -21.07; see Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab5">5</a>) is higher than those provided in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e5116">7</a>] (i.e. -22.71; see Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab4">4</a>, page 546 in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e5123">7</a>]); this can be explained by the different number of characters but also by our use of the amino acid model mtREV+Γ<sub>8</sub>+I instead of just mtREV in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e5128">7</a>]. However, the ML tree inferred from our initial character supermatrix has the same topology (left-hand tree in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig6">6</a>) as the ML tree inferred by Castresana (2000; see Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig5">5A</a>, page 545 in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e5138">7</a>]).
 The use of BMGE (with default BLOSUM62 and liberal BLOSUM30 matrices), 
Gblocks (relaxed), trimAl (strictplus and automated1) and Noisy lead to 
the same phylogenetic tree (left-hand tree in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig6">6</a>).
 However, BMGE with the stringent BLOSUM95 similarity matrix leads to a 
different ML phylogenetic tree (right-hand tree in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig6">6</a>). These two trees agree on the monophyly of Unikonts (nodes 1 and 1' in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig6">6</a>), but differ in the placement of the jacobid <i>Reclinomonas americana</i>, which is inside the Archaeplastida subtree in the left-hand tree of Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig6">6</a>.
 Knowing that Archaeplastida and Unikonts are each likely monophyletic, 
and that jacobids are phylogenetically distinct from Archaeplastida 
(e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Rodríguez-Ezpeleta N, Brinkmann H, Burey SC, Roure B, Burger G, Löffelhardt W, Bohnert HJ, Philippe H: Monophyly of Primary Photosynthetic Eukaryotes: Green Plants, Red Algae, and Glaucophytes. Curr Biol. 2005, 15: 1325-1330. 10.1016/j.cub.2005.06.040." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR69" id="ref-link-section-d16311e5157">69</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Hampl V, Hug L, Leigh JW, Dacks JB, Lang BF, Simpson AGB, Roger AJ: Phylogenomic analyses support the monophyly of Excavata and resolve relationships among eukaryotic &quot;supergroups&quot;. Proc Natl Acad Sci USA. 2009, 106: 3859-3864. 10.1073/pnas.0807880106." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR72" id="ref-link-section-d16311e5160">72</a>]), the BMGE (with BLOSUM95) tree at the right-hand side in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig6">6</a>
 seems more accurate. Moreover, BLOSUM95-based character trimming in 
BMGE gives among the best confidence values for the monophyly of 
Unikonts whereas Gblocks and trimAl both weakly support this subtree 
(see confidence values for nodes 1 and 1' in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Tab6">6</a>).
 The monophyly of Archaeplastida is weakly supported by all approaches, 
suggesting that this dataset does not induces sufficient phylogenetic 
signal for this particular node. However, BMGE with BLOSUM95 seems to 
provide a character trimming that leads to both accurate phylogenetic 
tree and confidence values, probably due to its ability to best minimize
 the number of false positive characters (see above and Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig1">1</a>).</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-6"><figure><figcaption class="c-article-table__figcaption"><b id="Tab6" data-test="table-caption">Table 6 ML bootstrap-based (boot.) and aLRT-based confidence values derived from different character supermatrices.</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-category="article body" data-track-label="button" rel="nofollow" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/tables/6"><span>Full size table</span></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6"><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 6</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig6_HTML.jpg?as=webp"><img aria-describedby="figure-6-desc" src="12862_2010_Article_1426_Fig6_HTML.jpg" alt="figure6"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>Phylogenetic
 trees obtained from a non-trimmed character supermatrix (left) and from
 the concatenation of the multiple sequence alignments trimmed by BMGE 
with BLOSUM95 (right)</b>. These ML trees were inferred by PhyML with the model mtREV+Γ<sub>8</sub>+I.
 Note that the left topology was also inferred from character 
supermatrices built by concatenating multiple sequence alignments 
trimmed by BMGE (BLOSUM30 and BLOSUM62), Gblocks (relaxed), trimAl 
(strictplus and automated1), and Noisy. Bootstap-based and aLRT-based 
confidence values at nodes (1), (1'), (2) and (3) are given in Table 6.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/6" data-track-dest="link:Figure6 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>This
 re-analysis of a known phylogenomic dataset shows that it is quite 
difficult to choose an appropriate similarity matrix with BMGE, knowing 
that, in the one hand, these mitochondrial amino acid sequences are 
distantly related (see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e5745">7</a>]),
 and that, in the other hand, a less unrealistic tree is inferred when 
BMGE is used with the stringent BLOSUM95 similarity matrix. To deal with
 amino acid sequences and BLOSUM matrices, a trivial approach would be 
to use the sequence clustering rule defined by Henikoff and Henikoff [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Henikoff S, Henikoff JG: Amino acid substitution matrices from protein blocks. Proc Natl Acad Sci. 1992, 89: 10915-10919. 10.1073/pnas.89.22.10915." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR39" id="ref-link-section-d16311e5748">39</a>]: to compute the BLOSUM<i>η</i>
 similarity matrix, they grouped amino acid sequences into clusters in 
such a way that each sequence in any cluster has at least <i>η</i>% identity to at least one other sequence in this cluster. Given an alignment with <i>n</i> amino acid sequences, a systematic procedure to choose the BLOSUM <i>η</i> similarity matrix is then to apply the following formula:</p><div id="Equ5" class="c-article-equation"><div class="c-article-equation__content"><span class="MathJax_Preview" style=""></span><div class="MathJax_Display" style="text-align: left;"><span class="MathJax" id="MathJax-Element-11-Frame" tabindex="0" style="text-align: left; position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; display=&quot;block&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B7;&lt;/mi&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;munder&gt;&lt;mrow&gt;&lt;mi&gt;min&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow /&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mo&gt;&amp;#x2026;&lt;/mo&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mi&gt;n&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/munder&gt;&lt;mspace width=&quot;0.25em&quot; /&gt;&lt;munder&gt;&lt;mrow&gt;&lt;mi&gt;max&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow /&gt;&lt;mrow&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mn&gt;&amp;#x2026;&lt;/mn&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo&gt;&amp;#x2212;&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo&gt;+&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mn&gt;&amp;#x2026;&lt;/mn&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mi&gt;n&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/munder&gt;&lt;mo&gt;{&lt;/mo&gt;&lt;mi&gt;%&lt;/mi&gt;&lt;mspace width=&quot;0.25em&quot; /&gt;&lt;mspace width=&quot;0.25em&quot; /&gt;&lt;mtext&gt;identity&lt;/mtext&gt;&lt;mspace width=&quot;0.25em&quot; /&gt;&lt;mtext&gt;between&lt;/mtext&gt;&lt;mspace width=&quot;0.25em&quot; /&gt;&lt;mtext&gt;sequences&lt;/mtext&gt;&lt;mspace width=&quot;0.25em&quot; /&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mspace width=&quot;0.25em&quot; /&gt;&lt;mi&gt;a&lt;/mi&gt;&lt;mi&gt;n&lt;/mi&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;mspace width=&quot;0.25em&quot; /&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;mo&gt;}&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-249" style="width: 34.972em; display: inline-block;"><span style="display: inline-block; position: relative; width: 27.089em; height: 0px; font-size: 129%;"><span style="position: absolute; clip: rect(1.867em, 1026.96em, 4.524em, -1000em); top: -3.445em; left: 0em;"><span class="mrow" id="MathJax-Span-250"><span class="mrow" id="MathJax-Span-251"><span class="mi" id="MathJax-Span-252" style="font-family: MathJax_Math; font-style: italic;">η<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.006em;"></span></span><span class="mo" id="MathJax-Span-253" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="munder" id="MathJax-Span-254" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.418em; height: 0px;"><span style="position: absolute; clip: rect(3.207em, 1001.65em, 4.134em, -1000em); top: -4.005em; left: 0.375em;"><span class="mrow" id="MathJax-Span-255"><span class="mi" id="MathJax-Span-256" style="font-family: MathJax_Main;">min</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.649em, 1002.42em, 4.437em, -1000em); top: -3.405em; left: 0em;"><span class="mrow" id="MathJax-Span-257"><span class="msub" id="MathJax-Span-258"><span style="display: inline-block; position: relative; width: 2.418em; height: 0px;"><span style="position: absolute; clip: rect(3.876em, 1000em, 4.134em, -1000em); top: -4.005em; left: 0em;"><span class="mrow" id="MathJax-Span-259"></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; top: -3.899em; left: 0em;"><span class="mrow" id="MathJax-Span-260"><span class="mi" id="MathJax-Span-261" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-262" style="font-size: 50%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-263" style="font-size: 50%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-264" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mn" id="MathJax-Span-265" style="font-size: 50%; font-family: MathJax_Main;">2</span><span class="mo" id="MathJax-Span-266" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mo" id="MathJax-Span-267" style="font-size: 50%; font-family: MathJax_Main;">…</span><span class="mo" id="MathJax-Span-268" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-269" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">n</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mspace" id="MathJax-Span-270" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="munder" id="MathJax-Span-271"><span style="display: inline-block; position: relative; width: 5.077em; height: 0px;"><span style="position: absolute; clip: rect(3.428em, 1001.85em, 4.145em, -1000em); top: -4.005em; left: 1.608em;"><span class="mrow" id="MathJax-Span-272"><span class="mi" id="MathJax-Span-273" style="font-family: MathJax_Main;">max</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; clip: rect(3.649em, 1005.08em, 4.442em, -1000em); top: -3.394em; left: 0em;"><span class="mrow" id="MathJax-Span-274"><span class="msub" id="MathJax-Span-275"><span style="display: inline-block; position: relative; width: 5.077em; height: 0px;"><span style="position: absolute; clip: rect(3.876em, 1000em, 4.134em, -1000em); top: -4.005em; left: 0em;"><span class="mrow" id="MathJax-Span-276"></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span><span style="position: absolute; top: -3.899em; left: 0em;"><span class="mrow" id="MathJax-Span-277"><span class="mi" id="MathJax-Span-278" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">j</span><span class="mo" id="MathJax-Span-279" style="font-size: 50%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-280" style="font-size: 50%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-281" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mn" id="MathJax-Span-282" style="font-size: 50%; font-family: MathJax_Main;">2</span><span class="mo" id="MathJax-Span-283" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mn" id="MathJax-Span-284" style="font-size: 50%; font-family: MathJax_Main;">…</span><span class="mo" id="MathJax-Span-285" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-286" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-287" style="font-size: 50%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-288" style="font-size: 50%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-289" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-290" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo" id="MathJax-Span-291" style="font-size: 50%; font-family: MathJax_Main;">+</span><span class="mn" id="MathJax-Span-292" style="font-size: 50%; font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-293" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mn" id="MathJax-Span-294" style="font-size: 50%; font-family: MathJax_Main;">…</span><span class="mo" id="MathJax-Span-295" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-296" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">n</span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.005em;"></span></span></span></span><span class="mo" id="MathJax-Span-297" style="vertical-align: 0em;"><span style="font-family: MathJax_Size3;">{</span></span><span class="mi" id="MathJax-Span-298" style="font-family: MathJax_Main; font-style: italic;">%<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.029em;"></span></span><span class="mspace" id="MathJax-Span-299" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mspace" id="MathJax-Span-300" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mtext" id="MathJax-Span-301" style="font-family: MathJax_Main;">identity</span><span class="mspace" id="MathJax-Span-302" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mtext" id="MathJax-Span-303" style="font-family: MathJax_Main;">between</span><span class="mspace" id="MathJax-Span-304" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mtext" id="MathJax-Span-305" style="font-family: MathJax_Main;">sequences</span><span class="mspace" id="MathJax-Span-306" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mi" id="MathJax-Span-307" style="font-family: MathJax_Math; font-style: italic;">i</span><span class="mspace" id="MathJax-Span-308" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mi" id="MathJax-Span-309" style="font-family: MathJax_Math; font-style: italic;">a</span><span class="mi" id="MathJax-Span-310" style="font-family: MathJax_Math; font-style: italic;">n</span><span class="mi" id="MathJax-Span-311" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mspace" id="MathJax-Span-312" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mi" id="MathJax-Span-313" style="font-family: MathJax_Math; font-style: italic;">j</span><span class="mo" id="MathJax-Span-314" style="vertical-align: 0em;"><span style="font-family: MathJax_Size3;">}</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.445em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.28em; border-left: 0px solid; width: 0px; height: 3.206em;"></span></span></nobr><span class="MJX_Assistive_MathML MJX_Assistive_MathML_Block" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><mrow><mi>η</mi><mo>=</mo><munder><mrow><mi>min</mi></mrow><mrow><msub><mrow></mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>n</mi></mrow></msub></mrow></munder><mspace width="0.25em"></mspace><munder><mrow><mi>max</mi></mrow><mrow><msub><mrow></mrow><mrow><mi>j</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mn>…</mn><mo>,</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi>i</mi><mo>+</mo><mn>1</mn><mo>,</mo><mn>…</mn><mo>,</mo><mi>n</mi></mrow></msub></mrow></munder><mo>{</mo><mi>%</mi><mspace width="0.25em"></mspace><mspace width="0.25em"></mspace><mtext>identity</mtext><mspace width="0.25em"></mspace><mtext>between</mtext><mspace width="0.25em"></mspace><mtext>sequences</mtext><mspace width="0.25em"></mspace><mi>i</mi><mspace width="0.25em"></mspace><mi>a</mi><mi>n</mi><mi>d</mi><mspace width="0.25em"></mspace><mi>j</mi><mo>}</mo></mrow></math></span></span></div><script type="math/mml" id="MathJax-Element-11"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block"><mml:mrow>
                          <mml:mi>η</mml:mi>
                          <mml:mo>=</mml:mo>
                          <mml:munder>
                            <mml:mrow>
                              <mml:mi>min</mml:mi>
                            </mml:mrow>
                            <mml:mrow>
                              <mml:msub>
                                <mml:mrow></mml:mrow>
                                <mml:mrow>
                                  <mml:mi>i</mml:mi>
                                  <mml:mo>=</mml:mo>
                                  <mml:mn>1</mml:mn>
                                  <mml:mo>,</mml:mo>
                                  <mml:mn>2</mml:mn>
                                  <mml:mo>,</mml:mo>
                                  <mml:mo>…</mml:mo>
                                  <mml:mo>,</mml:mo>
                                  <mml:mi>n</mml:mi>
                                </mml:mrow>
                              </mml:msub>
                            </mml:mrow>
                          </mml:munder>
                          <mml:mspace width="0.25em"></mml:mspace>
                          <mml:munder>
                            <mml:mrow>
                              <mml:mi>max</mml:mi>
                            </mml:mrow>
                            <mml:mrow>
                              <mml:msub>
                                <mml:mrow></mml:mrow>
                                <mml:mrow>
                                  <mml:mi>j</mml:mi>
                                  <mml:mo>=</mml:mo>
                                  <mml:mn>1</mml:mn>
                                  <mml:mo>,</mml:mo>
                                  <mml:mn>2</mml:mn>
                                  <mml:mo>,</mml:mo>
                                  <mml:mn>…</mml:mn>
                                  <mml:mo>,</mml:mo>
                                  <mml:mi>i</mml:mi>
                                  <mml:mo>−</mml:mo>
                                  <mml:mn>1</mml:mn>
                                  <mml:mo>,</mml:mo>
                                  <mml:mi>i</mml:mi>
                                  <mml:mo>+</mml:mo>
                                  <mml:mn>1</mml:mn>
                                  <mml:mo>,</mml:mo>
                                  <mml:mn>…</mml:mn>
                                  <mml:mo>,</mml:mo>
                                  <mml:mi>n</mml:mi>
                                </mml:mrow>
                              </mml:msub>
                            </mml:mrow>
                          </mml:munder>
                          <mml:mo>{</mml:mo>
                          <mml:mi>%</mml:mi>
                          <mml:mspace width="0.25em"></mml:mspace>
                          <mml:mspace width="0.25em"></mml:mspace>
                          <mml:mtext>identity</mml:mtext>
                          <mml:mspace width="0.25em"></mml:mspace>
                          <mml:mtext>between</mml:mtext>
                          <mml:mspace width="0.25em"></mml:mspace>
                          <mml:mtext>sequences</mml:mtext>
                          <mml:mspace width="0.25em"></mml:mspace>
                          <mml:mi>i</mml:mi>
                          <mml:mspace width="0.25em"></mml:mspace>
                          <mml:mi>a</mml:mi>
                          <mml:mi>n</mml:mi>
                          <mml:mi>d</mml:mi>
                          <mml:mspace width="0.25em"></mml:mspace>
                          <mml:mi>j</mml:mi>
                          <mml:mo>}</mml:mo>
                        </mml:mrow></math></script></div><div class="c-article-equation__number">
                    (4)
                </div></div><p>Unfortunately, this formula underestimates the value of <i>η</i> that best minimizes both the number of false positive and negative characters in our simulated datasets, e.g. <i>η</i>,
 as estimated by (4), varies from 16 to 71 with an average of 33 with 
level of divergence ×1, and varies from 10 to 62 with an average of 22 
with level of divergence ×3, whereas simulation shows that best results 
are obtained with <i>η</i> = 95 and 30 for levels of divergence ×1 and 
×3, respectively. Indeed, formula (4) on the Castresana (2000) 
phylogenomic dataset gives <i>η</i> values varying from 34 (ND3) to 64 
(CO1), which shows that the amino acid sequences constituting these ten 
alignments are not closely related, whereas we have shown that the 
conservative BLOSUM95 similarity matrix is best adapted than BLOSUM62 
and BLOSUM30 ones. Moreover, when estimated by (4) from the eight 
considered character supermatrices, <i>η</i> varies from 52 
(concatenated initial multiple sequence alignments) to 60 (concatenated 
alignments trimmed by Noisy). A similar underestimation of <i>η</i> was also observed by averaging the <i>n</i>
 max values in formula (4), or by considering only characters with no 
gaps. Finally, even if formula (4) or related could give a suited 
approximate of the <i>η</i> value by using the building rules of BLOSUM matrices, it will be even more difficult to provide a similar formula for the PAM-<i>η</i> similarity matrices when dealing with DNA sequences.</p><p>Therefore, as maintained by Ewens and Grant (2005) for BLAST queries, we believe that "<i>one
 often has prior knowledge about the evolutionary distance between the 
sequences of interest that helps one choose which BLOSUM matrix to use</i>" (p. 244 in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Ewens W, Grant G: Statistical methods in bioinformatics: an introduction. 2005, New York: Springer, Second" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR73" id="ref-link-section-d16311e5963">73</a>]).
 In practice, when inferring a particular gene tree, our opinion is to 
carefully examine the original multiple sequence alignment and those 
obtained by BMGE trimming with several similarity matrices (i.e. BLOSUM 
or PAM). On the contrary, when building a phylogenomic dataset, we 
believe, as Talavera and Castresana (2007), that "<i>there is enough information from the concatenation of several genes</i>" and then that "<i>stringent conditions tend to give rise to the best phylogenetic trees</i>" (p. 575 in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Castresana J: Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR7" id="ref-link-section-d16311e5972">7</a>]):
 the use of BMGE with stringent similarity matrices (e.g. BLOSUM95 or 
PAM-1) can strongly increase the number of false negatives (i.e. too 
many characters suited for phylogenetic inference are removed; see 
Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig1">1</a>)
 but systematic biases due to this conservative approach are often 
compensated by a sufficiently large number of genes used. Finally and in
 every case, we think that it is more relevant to deal with well-defined
 similarity matrices in order to choose among stringent to relaxed 
character trimming as in BMGE, rather than having to set several (and 
subjective) numerical parameters.</p><h3 class="c-article__sub-heading u-h3" id="Sec12">Simulation results with stationary-based character trimming</h3><p>We
 illustrate the impact of compositional heterogeneity and 
stationary-based character trimming on the accuracy of phylogenetic tree
 inference with a simple computer simulation. Given the quartet tree in 
Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig7">7</a>,
 the evolution of a DNA sequence of length 10,000 with 25%-proportion of
 each nucleotide was simulated by Seq-Gen from the root to the four 
leaves <i>u</i>, <i>v</i>, <i>x</i> and <i>y</i>. To infer the evolution of this DNA sequence, the evolutionary model F81 [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Felsenstein J: Evolutionary tree from DNA sequences: a maximum likelihood approach. J Mol Evol. 1981, 17: 368-376. 10.1007/BF01734359." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR74" id="ref-link-section-d16311e6003">74</a>]
 was chosen. On the one hand, Seq-Gen was used with equal relative 
character state frequencies to generate a compositionally homogeneous 
region (i.e. from 100% to 50% of the total length, with 10% increments).
 On the other hand, to generate a compositionally heterogeneous region 
(i.e. from 0% to 50% of the total length, respectively), DNA evolution 
was simulated with 80% GC-content for the external branches 
corresponding to the taxa <i>u</i> and <i>x</i>, and 20% GC-content for the two other external branches (i.e. taxa <i>v</i> and <i>y</i>).
 For each proportion of characters with heterogeneous composition (i.e. 
0%, 10%, ..., 50% of the length of the alignment), 500 four-sequence 
alignments were generated following this procedure.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-7"><figure><figcaption><b id="Fig7" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 7</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/7" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig7_HTML.jpg?as=webp"><img aria-describedby="figure-7-desc" src="12862_2010_Article_1426_Fig7_HTML.jpg" alt="figure7"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p><b>Phylogenetic tree used to simulate the non-stationary evolution of a DNA sequence</b>. This tree and the different branch lengths are closely related to [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Jermiin LS, Ho SYW, Ababneh F, Robinson J, Larkum AWD: The biasing effect of compositional heterogeneity on phylogenetic estimates may be underestimated. Syst Biol. 2004, 53: 638-643. 10.1080/10635150490468648." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR24" id="ref-link-section-d16311e6030">24</a>].</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/7" data-track-dest="link:Figure7 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>ML
 trees were inferred with PhyML (model F81) from all these simulated 
alignments. Stationary-based character trimming was applied with BMGE, 
and ML trees were inferred from the resulting compositionally 
homogeneous alignments. For each of the five proportions (i.e. 0%, 10%, 
..., 50%) of unequal GC-content characters, the number of times that the
 model tree (Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig1">1</a>) was correctly inferred from both initial and trimmed alignments is graphically represented in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig8">8</a>. The average proportions of characters removed by the stationary-based trimming are also reported. As expected [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Jermiin LS, Ho SYW, Ababneh F, Robinson J, Larkum AWD: The biasing effect of compositional heterogeneity on phylogenetic estimates may be underestimated. Syst Biol. 2004, 53: 638-643. 10.1080/10635150490468648." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR24" id="ref-link-section-d16311e6050">24</a>],
 the model tree is often recovered with the initial alignments when 
these contain no or a small proportion of compositionally heterogeneous 
characters (e.g. &lt; 20% of the total length of the initial alignment; 
see Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig8">8</a>), whereas the model tree is much less or not recovered when there is a high overall GC-content in taxa <i>u</i> and <i>x</i>, causing a biased attraction between them [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Phillips MJ, Delsuc F, Penny D: Genome-scale phylogeny and the detection of systematic biases. Mol Biol Evol. 2004, 21: 1455-1458. 10.1093/molbev/msh137." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR25" id="ref-link-section-d16311e6063">25</a>].
 Thanks to the stationary-based character trimming, BMGE detects and 
removes regions that are compositionally heterogeneous; then, as shown 
in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig8">8</a>,
 the model tree is very often recovered (e.g. the model tree is 
correctly inferred from more than 90% of the trimmed alignments), even 
when the proportion of unequal GC-content characters across sequences is
 high (e.g. &gt; 30% of the total length of the initial alignment).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-8"><figure><figcaption><b id="Fig8" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 8</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/8" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig8_HTML.jpg?as=webp"><img aria-describedby="figure-8-desc" src="12862_2010_Article_1426_Fig8_HTML.jpg" alt="figure8"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-8-desc"><p><b>Frequency
 of recovered model tree in function of the proportion of regions with 
heterogeneous composition inside an alignment of four DNA sequences</b>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/8" data-track-dest="link:Figure8 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>There
 exist many alternative statistical solutions to compare the character 
state composition of two (or more) sequences. Each of these has its own 
strengths and weaknesses (see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Jermiin LS, Ho SYW, Ababneh F, Robinson J, Larkum AWD: The biasing effect of compositional heterogeneity on phylogenetic estimates may be underestimated. Syst Biol. 2004, 53: 638-643. 10.1080/10635150490468648." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR24" id="ref-link-section-d16311e6092">24</a>] for instructive survey and discussion). However, matched-pairs tests (such as Stuart's test [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Stuart A: A test for homogeneity of the marginal distributions in a two-way classification. Biometrika. 1955, 42: 412-416." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR30" id="ref-link-section-d16311e6095">30</a>]) are comparatively efficient, particularly because of their ability to consider aligned sequences on a site-by-site basis [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Jermiin LS, Ho SYW, Ababneh F, Robinson J, Larkum AWD: The biasing effect of compositional heterogeneity on phylogenetic estimates may be underestimated. Syst Biol. 2004, 53: 638-643. 10.1080/10635150490468648." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR24" id="ref-link-section-d16311e6098">24</a>].
 Albeit the stationary-based character trimming may be extended by the 
use of overall tests for marginal symmetry (i.e. assessing the 
compositional homogeneity in the complete multiple sequence alignment [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Rzhetsky A, Nei M: Tests of applicability of several substitution models for DNA sequence data. Mol Biol Evol. 1995, 12: 131-151." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR75" id="ref-link-section-d16311e6101">75</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Ababneh F, Jermiin LS, Ma C, Robinson J: Matched-pairs tests of homogeneity with applications to homologous nucleotide sequences. Bioinformatics. 2006, 22: 1225-1231. 10.1093/bioinformatics/btl064." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR31" id="ref-link-section-d16311e6104">31</a>]), we think that using pairwise <i>p</i>-values allows more precise sorting of the characters <i>c</i> according to their <i>σ</i>(<i>c</i>) score value (see above). Moreover, such overall tests are more time consuming. It should also be stressed that the Bowker's [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Bowker AH: A test for symmetry in contingency tables. J Am Stat Assoc. 1948, 43: 572-574. 10.2307/2280710." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR76" id="ref-link-section-d16311e6120">76</a>] test (i.e. another matched-pairs test assessing the complete symmetry inside <b>F</b>)
 was tried instead of the Stuart's test in the stationary-based 
trimming, but it led to worse results in the simulation analysis (not 
shown). Finally, as stressed in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Uesaka H: Validity and applicability of several tests for comparing marginal distributions of a square table with ordered categories. Behaviormetrika. 1991, 30: 65-78. 10.2333/bhmk.18.30_65." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR43" id="ref-link-section-d16311e6127">43</a>], Stuart's test is based on ordinary <i>χ</i><sup><i>2</i></sup> approximation and is not appropriate for small samples, particularly when the number of categories (i.e. the row number in <b>F</b>)
 is not small. It is then strongly recommended to use the 
stationary-based character trimming on a large number of characters 
(e.g. ≥ 1, 000, such as in supermatrices of characters), especially when
 dealing with amino acid sequences.</p><h3 class="c-article__sub-heading u-h3" id="Sec13">Real case study with stationary-based character trimming</h3><p>In
 order to illustrate the performance of the stationary-based character 
trimming, we applied it on the multi-gene dataset described in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Rokas A, Williams BL, King N, Carroll SB: Genome-scale approaches to resolving incongruence in molecular phylogenies. Nature. 2003, 425: 798-804. 10.1038/nature02053." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR77" id="ref-link-section-d16311e6147">77</a>] (available at [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Yeast multi-gene dataset. [
                    http://systbio.org/files/Ren_etal_Rokas2003data.txt
                    
                  ]" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR78" id="ref-link-section-d16311e6150">78</a>]),
 which is known to suffer from a GC-content bias. This phylogenomic 
dataset was built by concatenating 106 alignments of DNA sequences 
gathered from 7 <i>Saccharomyces</i> species and <i>Candida albicans</i> as outgroup (see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Rokas A, Williams BL, King N, Carroll SB: Genome-scale approaches to resolving incongruence in molecular phylogenies. Nature. 2003, 425: 798-804. 10.1038/nature02053." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR77" id="ref-link-section-d16311e6159">77</a>] for more details). The so-obtained supermatrix contains 127,026 nucleotide characters.</p><p>By using on this character supermatrix the same methods and software as Phillips, Delsuc and Penny (2004; see [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Phillips MJ, Delsuc F, Penny D: Genome-scale phylogeny and the detection of systematic biases. Mol Biol Evol. 2004, 21: 1455-1458. 10.1093/molbev/msh137." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR25" id="ref-link-section-d16311e6165">25</a>] for more details), we retrieved the same phylogenetic tree (left-hand tree in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig9">9</a>) by minimizing the Minimum Evolution (ME) criterion with both GTR [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Lanave C, Preparata G, Saccone C, Serio G: A new method for calculating evolutionary substitution rates. J Mol Evol. 1984, 20: 86-93. 10.1007/BF02101990." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR79" id="ref-link-section-d16311e6171">79</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Yang Z: Estimating the pattern of nucleotide substitution. J Mol Evol. 1994, 39: 105-111." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR81" id="ref-link-section-d16311e6174">81</a>] and LogDet [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Lake JA: Reconstructing evolutionary trees from DNA and protein sequences: paralinear distances. Proc Natl Acad Sci USA. 1994, 91: 1455-1459. 10.1073/pnas.91.4.1455." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR82" id="ref-link-section-d16311e6177">82</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Steel MA: Recovering a tree from the leaf colourations it generates under a Markov model. Appl Math Lett. 1994, 7: 19-23. 10.1016/0893-9659(94)90024-8." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR84" id="ref-link-section-d16311e6181">84</a>] distance estimates. We also retrieved the same ME bootstrap-based confidence values as in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Phillips MJ, Delsuc F, Penny D: Genome-scale phylogeny and the detection of systematic biases. Mol Biol Evol. 2004, 21: 1455-1458. 10.1093/molbev/msh137." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR25" id="ref-link-section-d16311e6184">25</a>],
 i.e. 100% bootstrap proportion at all branches. Nevertheless, as shown 
by numerous phylogenetic analyses of the same dataset (e.g. [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Rokas A, Williams BL, King N, Carroll SB: Genome-scale approaches to resolving incongruence in molecular phylogenies. Nature. 2003, 425: 798-804. 10.1038/nature02053." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR77" id="ref-link-section-d16311e6187">77</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Phillips MJ, Delsuc F, Penny D: Genome-scale phylogeny and the detection of systematic biases. Mol Biol Evol. 2004, 21: 1455-1458. 10.1093/molbev/msh137." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR25" id="ref-link-section-d16311e6190">25</a>, <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Taylor DJ, Piel WH: An assessment of accuracy, error, and conflict with support values from genome-scale phylogenetic data. Mol Biol Evol. 2004, 21: 1534-1537. 10.1093/molbev/msh156." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR85" id="ref-link-section-d16311e6193">85</a>–<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Criscuolo A, Michel CJ: Phylogenetic inference with weighted codon evolutionary distances. J Mol Evol. 2009, 68: 377-392. 10.1007/s00239-009-9212-y." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR89" id="ref-link-section-d16311e6196">89</a>]), this tree is incorrect: the real evolutionary history of these 8 yeast taxa is in fact the right-hand tree in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig9">9</a>, with no monophyletic relationship between <i>S. kudriavzevii</i> and <i>S. bayanus</i>. It was shown in [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Phillips MJ, Delsuc F, Penny D: Genome-scale phylogeny and the detection of systematic biases. Mol Biol Evol. 2004, 21: 1455-1458. 10.1093/molbev/msh137." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR25" id="ref-link-section-d16311e6209">25</a>]
 that this systematic bias is due to a GC-content compositional 
heterogeneity across sequences that is sufficiently important to mislead
 the ME criterion.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-9"><figure><figcaption><b id="Fig9" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 9</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/9" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig9_HTML.jpg?as=webp"><img aria-describedby="figure-9-desc" src="12862_2010_Article_1426_Fig9_HTML.jpg" alt="figure9"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-9-desc"><p><b>Phylogenetic trees inferred before (left) and after (right) stationary-based character trimming</b>.
 Both trees are inferred by minimizing the Minimum Evolution criterion 
with GTR and LogDet distance estimations. ME bootstrap-based confidence 
values at branches are all 100%.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/9" data-track-dest="link:Figure9 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>A
 subset of 114,105 characters was selected by stationary-based character
 trimming implemented in BMGE. These so-selected characters were used to
 infer ME trees following the same methods as described previously. As 
expected, the right-hand tree in Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#Fig9">9</a>
 was inferred from both GTR and LogDet distance estimates with 100% 
bootstrap-based confidence value at each branch. More precisely, 
pairwise Stuart's test <i>p</i>-values are all ≈ 0 in the initial character supermatrix (with the slight exception of the two sequence pairs <i>S. cerevisiae - S. paradoxus</i> and <i>S. cerevisiae - S. mikatae</i> with <i>p</i>-values of 0.0146 and 0.0868, respectively). After the stationary-based trimming performed by BMGE, all <i>Saccharomyces</i> sequences induce pairwise Stuart's test <i>p</i>-values &gt; 0.85, the lowest <i>p</i>-values being induced by the outgroup species (i.e. varying from 0.1000 to 0.3675 for the sequence pairs <i>C. albicans - S. kluyveri</i> and <i>C. albicans - S. castellii</i>, respectively). This shows that by removing <b>≈</b>10%
 characters, the stationary-based trimming is able to select a subset of
 compositionally homogeneous characters (as assessed by the pairwise 
Stuart's tests) that leads to unbiased ME phylogenetic inference.</p></div></div></section><section aria-labelledby="Sec14"><div class="c-article-section js-article-section" id="Sec14-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec14">Conclusions</h2><div class="c-article-section__content js-collapsible-section" id="Sec14-content"><p>There
 exists a real need for accurate bioinformatic tools to extract at best 
the phylogenetic information contained in the ever-growing amount of 
available genomic sequence data. BMGE allows accurate character trimming
 of multiple alignments of DNA, codon, or amino acid sequences based on 
entropy-like scores weighted with BLOSUM or PAM matrices. Thus, BMGE is 
able to identify and extract unambiguously aligned blocks of characters 
that contain biologically expected variability and are therefore 
suitable for phylogenetic analysis. Simulation studies show that the 
trimmed datasets returned by BMGE lead to inference of accurate trees, 
in particular when in presence of multiple alignments including 
distantly-related sequences. BMGE also allows a number of useful 
recoding and trimming aimed at minimizing compositional heterogeneity in
 the alignment dataset and therefore the risk of phylogenetic artefacts.</p><p>In
 conclusion, BMGE is an accurate tool that can have several applications
 in phylogenomics analyses. The software BMGE is freely available (see 
below), and can also be used online through the Mobyle Web Portal [<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Néron B, Ménager H, Maufrais C, Joly N, Maupetit J, Letort S, Carrere S, Tuffery P, Letondal C: Mobyle: a new full web bioinformatics framework. Bioinformatics. 2009, 25: 3005-3011. 10.1093/bioinformatics/btp493." href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210#ref-CR90" id="ref-link-section-d16311e6280">90</a>] at <a href="http://mobyle.pasteur.fr/cgi-bin/portal.py">http://mobyle.pasteur.fr/cgi-bin/portal.py</a>.</p></div></div></section><section aria-labelledby="Sec15"><div class="c-article-section js-article-section" id="Sec15-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec15">Availability and Requirements</h2><div class="c-article-section__content js-collapsible-section" id="Sec15-content"><p><b>• Project name</b>: Block l Entropy (BMGE)</p><ul class="u-list-style-none">
                  <li>
                    <p><b>• Project home page</b>: <a href="ftp://ftp.pasteur.fr/pub/GenSoft/projects/BMGE/">ftp://ftp.pasteur.fr/pub/GenSoft/projects/BMGE/</a></p>
                  </li>
                  <li>
                    <p><b>• Operating systems</b>: Platform independent</p>
                  </li>
                  <li>
                    <p><b>• Programming language</b>: Java</p>
                  </li>
                  <li>
                    <p><b>• Other requirements</b>: Java 1.6 or higher</p>
                  </li>
                  <li>
                    <p><b>• License</b>: GNU General Public License (version 2)</p>
                  </li>
                  <li>
                    <p><b>• Any restrictions to use by non-academics</b>: None</p>
                  </li>
                </ul></div></div></section>
                        <section aria-labelledby="Bib1"><div class="c-article-section js-article-section" id="Bib1-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content js-collapsible-section" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references"><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">1.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR1">Lake JA: The order of sequence alignment can bias the selection of tree topology. Mol Biol Evol. 1991, 8: 378-385.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3MXit1Cjs7o%253D" aria-label="View reference 1 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2072863" aria-label="View reference 1 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 1 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20order%20of%20sequence%20alignment%20can%20bias%20the%20selection%20of%20tree%20topology&amp;journal=Mol%20Biol%20Evol&amp;volume=8&amp;pages=378-385&amp;publication_year=1991&amp;author=Lake%2CJA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">2.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR2">Morrison
 DA, Ellis JT: Effects of nucleotide sequence alignment on phylogeny 
estimation: a case study on 18 S rDNAs of apicomplexa. Mol Biol Evol. 
1997, 14: 428-441.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Foxfordjournals.molbev.a025779" aria-label="View reference 2">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2sXitlOhtLY%253D" aria-label="View reference 2 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9100373" aria-label="View reference 2 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 2 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20nucleotide%20sequence%20alignment%20on%20phylogeny%20estimation%3A%20a%20case%20study%20on%2018%20S%20rDNAs%20of%20apicomplexa&amp;journal=Mol%20Biol%20Evol&amp;volume=14&amp;pages=428-441&amp;publication_year=1997&amp;author=Morrison%2CDA&amp;author=Ellis%2CJT">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">3.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR3">Ogden
 TH, Rosenberg MS: Multiple sequence alignment accuracy and phylogenetic
 inference. Syst Biol. 2006, 55: 314-328. 10.1080/10635150500541730.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F10635150500541730" aria-label="View reference 3">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16611602" aria-label="View reference 3 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 3 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiple%20sequence%20alignment%20accuracy%20and%20phylogenetic%20inference&amp;journal=Syst%20Biol&amp;volume=55&amp;pages=314-328&amp;publication_year=2006&amp;author=Ogden%2CTH&amp;author=Rosenberg%2CMS">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><span class="c-article-references__counter">4.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR4">Wang
 L-S, Leebens-Mack J, Wall PK, Beckmann K, dePamphilis CW, Warnow T: The
 impact of multiple protein sequence alignment on phylogenetic 
estimation. IEEE/ACM Trans Comput Biol Bioinf. 2009.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 4 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=IEEE%2FACM%20Trans%20Comput%20Biol%20Bioinf&amp;publication_year=2009&amp;author=Wang%2CL-S&amp;author=Leebens-Mack%2CJ&amp;author=Wall%2CPK&amp;author=Beckmann%2CK&amp;author=dePamphilis%2CCW&amp;author=Warnow%2CT">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">5.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR5">Edgar RC, Batzoglou S: Multiple sequence alignment. Curr Opin Struct Biol. 2006, 16: 368-373. 10.1016/j.sbi.2006.04.004.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.sbi.2006.04.004" aria-label="View reference 5">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD28Xls1aksLo%253D" aria-label="View reference 5 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16679011" aria-label="View reference 5 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 5 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiple%20sequence%20alignment&amp;journal=Curr%20Opin%20Struct%20Biol&amp;volume=16&amp;pages=368-373&amp;publication_year=2006&amp;author=Edgar%2CRC&amp;author=Batzoglou%2CS">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">6.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR6">Notredame
 C: Recent evolutions of multiple sequence alignment algorithms. PLoS 
Comput Biol. 2007, 3: e123-10.1371/journal.pcbi.0030123.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1963500" aria-label="View reference 6 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pcbi.0030123" aria-label="View reference 6">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17784778" aria-label="View reference 6 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 6 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Recent%20evolutions%20of%20multiple%20sequence%20alignment%20algorithms&amp;journal=PLoS%20Comput%20Biol&amp;volume=3&amp;publication_year=2007&amp;author=Notredame%2CC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">7.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR7">Castresana
 J: Selection of conserved blocks from multiple alignments for their use
 in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Foxfordjournals.molbev.a026334" aria-label="View reference 7">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3cXisVSgt7g%253D" aria-label="View reference 7 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10742046" aria-label="View reference 7 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 7 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Selection%20of%20conserved%20blocks%20from%20multiple%20alignments%20for%20their%20use%20in%20phylogenetic%20analysis&amp;journal=Mol%20Biol%20Evol&amp;volume=17&amp;pages=540-552&amp;publication_year=2000&amp;author=Castresana%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">8.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR8">Talavera
 G, Castresana J: Improvement of phylogenies after removing divergent 
and ambiguously aligned blocks from protein sequence alignments. Syst 
Biol. 2007, 56: 564-577. 10.1080/10635150701472164.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F10635150701472164" aria-label="View reference 8">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXhtFKrs7%252FP" aria-label="View reference 8 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17654362" aria-label="View reference 8 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 8 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Improvement%20of%20phylogenies%20after%20removing%20divergent%20and%20ambiguously%20aligned%20blocks%20from%20protein%20sequence%20alignments&amp;journal=Syst%20Biol&amp;volume=56&amp;pages=564-577&amp;publication_year=2007&amp;author=Talavera%2CG&amp;author=Castresana%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">9.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR9">Olsen GJ, Woese CR: Ribosomal RNA: a key to phylogeny. FASEB J. 1993, 7: 113-123.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3sXhtF2jtro%253D" aria-label="View reference 9 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8422957" aria-label="View reference 9 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 9 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Ribosomal%20RNA%3A%20a%20key%20to%20phylogeny&amp;journal=FASEB%20J&amp;volume=7&amp;pages=113-123&amp;publication_year=1993&amp;author=Olsen%2CGJ&amp;author=Woese%2CCR">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">10.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR10">Rodrigo
 AG, Bergquist PR, Bergquist PL: Inadequate support for an evolutionary 
link between the Metazoa and the Fungi. Syst Biol. 1994, 43: 578-584. 
10.1093/sysbio/43.4.578.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fsysbio%2F43.4.578" aria-label="View reference 10">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 10 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Inadequate%20support%20for%20an%20evolutionary%20link%20between%20the%20Metazoa%20and%20the%20Fungi&amp;journal=Syst%20Biol&amp;volume=43&amp;pages=578-584&amp;publication_year=1994&amp;author=Rodrigo%2CAG&amp;author=Bergquist%2CPR&amp;author=Bergquist%2CPL">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><span class="c-article-references__counter">11.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR11">Swofford
 DL, Olsen GJ, Waddell PJ, Hillis DM: Phylogenetic inference. Molecular 
Systematics. Edited by: Hillis DM, Moritz C, Mable BK. 1996, Sunderland:
 Sinauer Associates, 407-514.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 11 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20Systematics&amp;pages=407-514&amp;publication_year=1996&amp;author=Swofford%2CDL&amp;author=Olsen%2CGJ&amp;author=Waddell%2CPJ&amp;author=Hillis%2CDM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">12.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR12">Rodríguez-Ezpeleta
 N, Brinkmann H, Burey SC, Roure B, Burger G, Löffelhardt W, Philippe H,
 Lang BF: Monophyly of primary photosynthetic eukaryotes: green plants, 
red algae, and glaucophytes. Curr Biol. 2005, 15: 1325-1330. 
10.1016/j.cub.2005.06.040.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.cub.2005.06.040" aria-label="View reference 12">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16051178" aria-label="View reference 12 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 12 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Monophyly%20of%20primary%20photosynthetic%20eukaryotes%3A%20green%20plants%2C%20red%20algae%2C%20and%20glaucophytes&amp;journal=Curr%20Biol&amp;volume=15&amp;pages=1325-1330&amp;publication_year=2005&amp;author=Rodr%C3%ADguez-Ezpeleta%2CN&amp;author=Brinkmann%2CH&amp;author=Burey%2CSC&amp;author=Roure%2CB&amp;author=Burger%2CG&amp;author=L%C3%B6ffelhardt%2CW&amp;author=Philippe%2CH&amp;author=Lang%2CBF">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">13.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR13">Huerta-Cepas
 J, Bueno A, Dopazo J, Gabaldón T: PhylomeDB: a database for genome-wide
 collections of gene phylogenies. Nucleic Acids Res. 2008, 36: 
D491-D496. 10.1093/nar/gkm899.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2238872" aria-label="View reference 13 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2Fgkm899" aria-label="View reference 13">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXhtVSktbg%253D" aria-label="View reference 13 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17962297" aria-label="View reference 13 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 13 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=PhylomeDB%3A%20a%20database%20for%20genome-wide%20collections%20of%20gene%20phylogenies&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=D491-D496&amp;publication_year=2008&amp;author=Huerta-Cepas%2CJ&amp;author=Bueno%2CA&amp;author=Dopazo%2CJ&amp;author=Gabald%C3%B3n%2CT">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">14.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR14">Dress
 AWM, Flamm C, Fritzsch G, Grünewald S, Kruspe M, Prohaska SJ, Stadler 
PF: Noisy: Identification of problematic columns in multiple sequence 
alignments. Algorithms for Molecular Biology. 2008, 3: 
7-10.1186/1748-7188-3-7.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2464588" aria-label="View reference 14 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1186%2F1748-7188-3-7" aria-label="View reference 14">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18577231" aria-label="View reference 14 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 14 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Noisy%3A%20Identification%20of%20problematic%20columns%20in%20multiple%20sequence%20alignments&amp;journal=Algorithms%20for%20Molecular%20Biology&amp;volume=3&amp;publication_year=2008&amp;author=Dress%2CAWM&amp;author=Flamm%2CC&amp;author=Fritzsch%2CG&amp;author=Gr%C3%BCnewald%2CS&amp;author=Kruspe%2CM&amp;author=Prohaska%2CSJ&amp;author=Stadler%2CPF">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">15.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR15">Swingley
 WD, Blankenship RE, Raymond J: Integrating Markov clustering and 
molecular phylogenetics to reconstruct the cyanobacterial species tree 
from conserved protein families. Mol Biol Evol. 2008, 25: 643-654. 
10.1093/molbev/msn034.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fmolbev%2Fmsn034" aria-label="View reference 15">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXksFWksbc%253D" aria-label="View reference 15 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18296704" aria-label="View reference 15 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 15 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrating%20Markov%20clustering%20and%20molecular%20phylogenetics%20to%20reconstruct%20the%20cyanobacterial%20species%20tree%20from%20conserved%20protein%20families&amp;journal=Mol%20Biol%20Evol&amp;volume=25&amp;pages=643-654&amp;publication_year=2008&amp;author=Swingley%2CWD&amp;author=Blankenship%2CRE&amp;author=Raymond%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">16.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR16">Capella-Gutiérez
 S, Silla-Martínez JM, Gabaldón T: trimAl: a tool for automated 
alignment triming in large-scale phylogenetic analyses. Bioinformatics. 
2009, 25: 1972-1973. 10.1093/bioinformatics/btp348.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp348" aria-label="View reference 16">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 16 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=trimAl%3A%20a%20tool%20for%20automated%20alignment%20triming%20in%20large-scale%20phylogenetic%20analyses&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1972-1973&amp;publication_year=2009&amp;author=Capella-Guti%C3%A9rez%2CS&amp;author=Silla-Mart%C3%ADnez%2CJM&amp;author=Gabald%C3%B3n%2CT">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">17.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR17">Daubin
 V, Gouy M, Perrière G: A phylogenomic approach to bacterial phylogeny: 
evidence of a core of genes sharing a common history. Genome Res. 2002, 
12: 1080-1090. 10.1101/gr.187002.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC186629" aria-label="View reference 17 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1101%2Fgr.187002" aria-label="View reference 17">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD38XlsVWgsro%253D" aria-label="View reference 17 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12097345" aria-label="View reference 17 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 17 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20phylogenomic%20approach%20to%20bacterial%20phylogeny%3A%20evidence%20of%20a%20core%20of%20genes%20sharing%20a%20common%20history&amp;journal=Genome%20Res&amp;volume=12&amp;pages=1080-1090&amp;publication_year=2002&amp;author=Daubin%2CV&amp;author=Gouy%2CM&amp;author=Perri%C3%A8re%2CG">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">18.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR18">Ciccarelli
 FD, Doerks T, von Mering C, Creevey CJ, Snel B, Bork P: Toward 
automatic reconstruction of a highly resolved tree of life. Science. 
2006, 311: 1283-1287. 10.1126/science.1123061.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1126%2Fscience.1123061" aria-label="View reference 18">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD28XhvVeku7s%253D" aria-label="View reference 18 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16513982" aria-label="View reference 18 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 18 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Toward%20automatic%20reconstruction%20of%20a%20highly%20resolved%20tree%20of%20life&amp;journal=Science&amp;volume=311&amp;pages=1283-1287&amp;publication_year=2006&amp;author=Ciccarelli%2CFD&amp;author=Doerks%2CT&amp;author=von%20Mering%2CC&amp;author=Creevey%2CCJ&amp;author=Snel%2CB&amp;author=Bork%2CP">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">19.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR19">Adachi
 J, Waddell PJ, Martin W, Hasegawa M: Plastid genome phylogeny and a 
model of amino acid substitution for protein encoded by chloroplast DNA.
 J Mol Evol. 2000, 50: 348-358.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3cXivFyjur8%253D" aria-label="View reference 19 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10795826" aria-label="View reference 19 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 19 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Plastid%20genome%20phylogeny%20and%20a%20model%20of%20amino%20acid%20substitution%20for%20protein%20encoded%20by%20chloroplast%20DNA&amp;journal=J%20Mol%20Evol&amp;volume=50&amp;pages=348-358&amp;publication_year=2000&amp;author=Adachi%2CJ&amp;author=Waddell%2CPJ&amp;author=Martin%2CW&amp;author=Hasegawa%2CM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">20.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR20">McMahon
 MM, Sanderson MJ: Phylogenetic supermatrix analysis of GenBank 
sequences from 2228 Papilionoid legumes. Syst Biol. 2006, 55: 818-836. 
10.1080/10635150600999150.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F10635150600999150" aria-label="View reference 20">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17060202" aria-label="View reference 20 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 20 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenetic%20supermatrix%20analysis%20of%20GenBank%20sequences%20from%202228%20Papilionoid%20legumes&amp;journal=Syst%20Biol&amp;volume=55&amp;pages=818-836&amp;publication_year=2006&amp;author=McMahon%2CMM&amp;author=Sanderson%2CMJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">21.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR21">Schneider
 TD, Stephens RM: Sequence logos: a new way to display consensus 
sequences. Nucl Acids Res. 1990, 18: 6097-6100. 10.1093/nar/18.20.6097.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC332411" aria-label="View reference 21 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2F18.20.6097" aria-label="View reference 21">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3MXitVGmuw%253D%253D" aria-label="View reference 21 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2172928" aria-label="View reference 21 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 21 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Sequence%20logos%3A%20a%20new%20way%20to%20display%20consensus%20sequences&amp;journal=Nucl%20Acids%20Res&amp;volume=18&amp;pages=6097-6100&amp;publication_year=1990&amp;author=Schneider%2CTD&amp;author=Stephens%2CRM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">22.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR22">Jayaswal V, Jermiin LS, Robinson J: Estimation of phylogeny using a general Markov model. Evol Bioinf Online. 2005, 1: 62-80.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 22 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimation%20of%20phylogeny%20using%20a%20general%20Markov%20model&amp;journal=Evol%20Bioinf%20Online&amp;volume=1&amp;pages=62-80&amp;publication_year=2005&amp;author=Jayaswal%2CV&amp;author=Jermiin%2CLS&amp;author=Robinson%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">23.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR23">Galtier
 N, Gouy M: Inferring pattern and process: maximum-likelihood 
implementation of a nonhomogeneous model of DNA sequence evolution for 
phylogenetic analysis. Mol Biol Evol. 1998, 15: 871-879.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Foxfordjournals.molbev.a025991" aria-label="View reference 23">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK1cXksFCisbY%253D" aria-label="View reference 23 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9656487" aria-label="View reference 23 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 23 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Inferring%20pattern%20and%20process%3A%20maximum-likelihood%20implementation%20of%20a%20nonhomogeneous%20model%20of%20DNA%20sequence%20evolution%20for%20phylogenetic%20analysis&amp;journal=Mol%20Biol%20Evol&amp;volume=15&amp;pages=871-879&amp;publication_year=1998&amp;author=Galtier%2CN&amp;author=Gouy%2CM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">24.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR24">Jermiin
 LS, Ho SYW, Ababneh F, Robinson J, Larkum AWD: The biasing effect of 
compositional heterogeneity on phylogenetic estimates may be 
underestimated. Syst Biol. 2004, 53: 638-643. 10.1080/10635150490468648.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F10635150490468648" aria-label="View reference 24">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15371251" aria-label="View reference 24 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 24 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20biasing%20effect%20of%20compositional%20heterogeneity%20on%20phylogenetic%20estimates%20may%20be%20underestimated&amp;journal=Syst%20Biol&amp;volume=53&amp;pages=638-643&amp;publication_year=2004&amp;author=Jermiin%2CLS&amp;author=Ho%2CSYW&amp;author=Ababneh%2CF&amp;author=Robinson%2CJ&amp;author=Larkum%2CAWD">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">25.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR25">Phillips
 MJ, Delsuc F, Penny D: Genome-scale phylogeny and the detection of 
systematic biases. Mol Biol Evol. 2004, 21: 1455-1458. 
10.1093/molbev/msh137.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fmolbev%2Fmsh137" aria-label="View reference 25">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXltlKlsbw%253D" aria-label="View reference 25 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15084674" aria-label="View reference 25 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 25 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-scale%20phylogeny%20and%20the%20detection%20of%20systematic%20biases&amp;journal=Mol%20Biol%20Evol&amp;volume=21&amp;pages=1455-1458&amp;publication_year=2004&amp;author=Phillips%2CMJ&amp;author=Delsuc%2CF&amp;author=Penny%2CD">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">26.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR26">International
 Union of Pure and Applied Chemistery and International Union of 
Biochemistery (IUPAC-IUB) Commission on Biochemical Nomenclature: 
Abbreviations and symbols for nucleic acids, polynucleotides and their 
constituents. Biochem J. 1970, 120: 449-454.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1042%2Fbj1200449" aria-label="View reference 26">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 26 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Abbreviations%20and%20symbols%20for%20nucleic%20acids%2C%20polynucleotides%20and%20their%20constituents&amp;journal=Biochem%20J&amp;volume=120&amp;pages=449-454&amp;publication_year=1970">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">27.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR27">Dayhoff
 MO, Schwartz RM, Orcutt BD: A model of evolutionary change in proteins.
 Atlas of Protein Sequence and Structure. Edited by: Dayhoff MO. 1978, 
Washington: National Biomedical Research Foundation, 5 (Suppl 3): 
345-352.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">28.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR28">Embley
 TM, van der Giezen M, Horner DS, Dyal PL, Foster P: Mitochondria and 
hydrogenosomes are two forms of the same fundamental organelle. Philos 
Trans R Soc Lond B Biol Sci. 2003, 358: 191-203. 10.1098/rstb.2002.1190.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1693103" aria-label="View reference 28 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1098%2Frstb.2002.1190" aria-label="View reference 28">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXktVWhs74%253D" aria-label="View reference 28 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12594927" aria-label="View reference 28 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 28 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Mitochondria%20and%20hydrogenosomes%20are%20two%20forms%20of%20the%20same%20fundamental%20organelle&amp;journal=Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci&amp;volume=358&amp;pages=191-203&amp;publication_year=2003&amp;author=Embley%2CTM&amp;author=van%20der%20Giezen%2CM&amp;author=Horner%2CDS&amp;author=Dyal%2CPL&amp;author=Foster%2CP">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">29.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR29">Susko
 E, Roger AJ: On reduced amino acid alphabets for phylogenetic 
inference. Mol Biol Evol. 2007, 24: 2139-2150. 10.1093/molbev/msm144.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fmolbev%2Fmsm144" aria-label="View reference 29">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXhtFSjsb7P" aria-label="View reference 29 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17652333" aria-label="View reference 29 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 29 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20reduced%20amino%20acid%20alphabets%20for%20phylogenetic%20inference&amp;journal=Mol%20Biol%20Evol&amp;volume=24&amp;pages=2139-2150&amp;publication_year=2007&amp;author=Susko%2CE&amp;author=Roger%2CAJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">30.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR30">Stuart A: A test for homogeneity of the marginal distributions in a two-way classification. Biometrika. 1955, 42: 412-416.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fbiomet%2F42.3-4.412" aria-label="View reference 30">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 30 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20test%20for%20homogeneity%20of%20the%20marginal%20distributions%20in%20a%20two-way%20classification&amp;journal=Biometrika&amp;volume=42&amp;pages=412-416&amp;publication_year=1955&amp;author=Stuart%2CA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">31.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR31">Ababneh
 F, Jermiin LS, Ma C, Robinson J: Matched-pairs tests of homogeneity 
with applications to homologous nucleotide sequences. Bioinformatics. 
2006, 22: 1225-1231. 10.1093/bioinformatics/btl064.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl064" aria-label="View reference 31">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD28Xktlaku7Y%253D" aria-label="View reference 31 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16492684" aria-label="View reference 31 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 31 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Matched-pairs%20tests%20of%20homogeneity%20with%20applications%20to%20homologous%20nucleotide%20sequences&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1225-1231&amp;publication_year=2006&amp;author=Ababneh%2CF&amp;author=Jermiin%2CLS&amp;author=Ma%2CC&amp;author=Robinson%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">32.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR32">International
 Union of Pure and Applied Chemistery and International Union of 
Biochemistery (IUPAC-IUB) Commission on Biochemical Nomenclature: A 
one-letter notation for amino acid sequences (definitive rules). Pure 
Appl Chem. 1972, 31: 639-645. 10.1351/pac197231040639.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 32 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20one-letter%20notation%20for%20amino%20acid%20sequences%20%28definitive%20rules%29&amp;journal=Pure%20Appl%20Chem&amp;volume=31&amp;pages=639-645&amp;publication_year=1972">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><span class="c-article-references__counter">33.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR33">Von Neumann J: Mathematische Grundlagen der Quantenmechanik. 1932, Berlin: Springer</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 33 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Mathematische%20Grundlagen%20der%20Quantenmechanik&amp;publication_year=1932&amp;author=Von%20Neumann%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">34.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR34">Caffrey
 DR, Somaroo S, Hughes JD, Mintseris J, Huang ES: Are protein-protein 
interfaces more conserved in sequence than the rest of the protein 
surface?. Protein Sci. 2004, 13: 190-202. 10.1110/ps.03323604.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2286531" aria-label="View reference 34 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1110%2Fps.03323604" aria-label="View reference 34">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXhsFWiug%253D%253D" aria-label="View reference 34 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14691234" aria-label="View reference 34 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 34 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Are%20protein-protein%20interfaces%20more%20conserved%20in%20sequence%20than%20the%20rest%20of%20the%20protein%20surface%3F&amp;journal=Protein%20Sci&amp;volume=13&amp;pages=190-202&amp;publication_year=2004&amp;author=Caffrey%2CDR&amp;author=Somaroo%2CS&amp;author=Hughes%2CJD&amp;author=Mintseris%2CJ&amp;author=Huang%2CES">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">35.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR35">JAMA: A Java Matrix Package. [<a href="http://math.nist.gov/javanumerics/jama/">http://math.nist.gov/javanumerics/jama/</a>]</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">36.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR36">Shannon C: A mathematical theory of communication. Bell System Tech J. 1948, 27: 379-423. 623-656</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1002%2Fj.1538-7305.1948.tb01338.x" aria-label="View reference 36">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 36 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20mathematical%20theory%20of%20communication&amp;journal=Bell%20System%20Tech%20J&amp;volume=27&amp;pages=379-423&amp;publication_year=1948&amp;author=Shannon%2CC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">37.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR37">Taylor WR: The classification of amino acid conservation. J Theor Biol. 1986, 119: 205-218. 10.1016/S0022-5193(86)80075-3.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2FS0022-5193%2886%2980075-3" aria-label="View reference 37">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaL28XitVensbc%253D" aria-label="View reference 37 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3461222" aria-label="View reference 37 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 37 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20classification%20of%20amino%20acid%20conservation&amp;journal=J%20Theor%20Biol&amp;volume=119&amp;pages=205-218&amp;publication_year=1986&amp;author=Taylor%2CWR">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">38.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR38">Bordo
 D, Argos P: Suggestion for "safe" residue substitutions in 
site-directed mutagenesis. J mol Biol. 1991, 217: 721-729. 
10.1016/0022-2836(91)90528-E.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2F0022-2836%2891%2990528-E" aria-label="View reference 38">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3MXitVejtL0%253D" aria-label="View reference 38 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2005621" aria-label="View reference 38 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 38 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Suggestion%20for%20%22safe%22%20residue%20substitutions%20in%20site-directed%20mutagenesis&amp;journal=J%20mol%20Biol&amp;volume=217&amp;pages=721-729&amp;publication_year=1991&amp;author=Bordo%2CD&amp;author=Argos%2CP">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">39.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR39">Henikoff
 S, Henikoff JG: Amino acid substitution matrices from protein blocks. 
Proc Natl Acad Sci. 1992, 89: 10915-10919. 10.1073/pnas.89.22.10915.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC50453" aria-label="View reference 39 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.89.22.10915" aria-label="View reference 39">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3sXjsFCgsQ%253D%253D" aria-label="View reference 39 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1438297" aria-label="View reference 39 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 39 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Amino%20acid%20substitution%20matrices%20from%20protein%20blocks&amp;journal=Proc%20Natl%20Acad%20Sci&amp;volume=89&amp;pages=10915-10919&amp;publication_year=1992&amp;author=Henikoff%2CS&amp;author=Henikoff%2CJG">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">40.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR40">BLOSUM matrices. [<a href="http://blocks.fhcrc.org/blocks/uploads/blosum/">http://blocks.fhcrc.org/blocks/uploads/blosum/</a>]</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">41.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR41">Eddy SR: Where did the BLOSUM62 alignment score matrix come from?. Nat Biotechnol. 2004, 22: 1035-1036. 10.1038/nbt0804-1035.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnbt0804-1035" aria-label="View reference 41">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXmtFWrtL8%253D" aria-label="View reference 41 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15286655" aria-label="View reference 41 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 41 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Where%20did%20the%20BLOSUM62%20alignment%20score%20matrix%20come%20from%3F&amp;journal=Nat%20Biotechnol&amp;volume=22&amp;pages=1035-1036&amp;publication_year=2004&amp;author=Eddy%2CSR">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">42.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR42">States
 DJ, Gish W, Altschul SF: Improved sensitivity of nucleic acid database 
searches using application-specific scoring matrices. Methods: A 
Companion to Methods Enzymol. 1991, 3: 66-70. 
10.1016/S1046-2023(05)80165-3.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2FS1046-2023%2805%2980165-3" aria-label="View reference 42">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK38Xhs1Ohu70%253D" aria-label="View reference 42 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 42 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20sensitivity%20of%20nucleic%20acid%20database%20searches%20using%20application-specific%20scoring%20matrices&amp;journal=Methods%3A%20A%20Companion%20to%20Methods%20Enzymol&amp;volume=3&amp;pages=66-70&amp;publication_year=1991&amp;author=States%2CDJ&amp;author=Gish%2CW&amp;author=Altschul%2CSF">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">43.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR43">Uesaka
 H: Validity and applicability of several tests for comparing marginal 
distributions of a square table with ordered categories. 
Behaviormetrika. 1991, 30: 65-78. 10.2333/bhmk.18.30_65.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.2333%2Fbhmk.18.30_65" aria-label="View reference 43">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 43 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Validity%20and%20applicability%20of%20several%20tests%20for%20comparing%20marginal%20distributions%20of%20a%20square%20table%20with%20ordered%20categories&amp;journal=Behaviormetrika&amp;volume=30&amp;pages=65-78&amp;publication_year=1991&amp;author=Uesaka%2CH">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><span class="c-article-references__counter">44.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR44">Press
 WH, Teukolsky SA, Vetterling WT, Flannery BP: Numerical Recipes in C --
 The Art of Scientific Computing. 1992, Cambridge: Cambridge University 
Press, 2</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 44 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Numerical%20Recipes%20in%20C%20--%20The%20Art%20of%20Scientific%20Computing&amp;publication_year=1992&amp;author=Press%2CWH&amp;author=Teukolsky%2CSA&amp;author=Vetterling%2CWT&amp;author=Flannery%2CBP">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">45.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR45">Artificial datasets used in [48]. [<a href="http://www.atgc-montpellier.fr/phyml/datasets.php">http://www.atgc-montpellier.fr/phyml/datasets.php</a>]</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">46.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR46">Rambaut
 A, Grassly NC: Seq-Gen: an application for the Monte Carlo simulation 
of DNA sequence evolution along phylogenetic trees. Comput Appl Biosci. 
1997, 13: 235-238.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2sXksVGntL4%253D" aria-label="View reference 46 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9183526" aria-label="View reference 46 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 46 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Seq-Gen%3A%20an%20application%20for%20the%20Monte%20Carlo%20simulation%20of%20DNA%20sequence%20evolution%20along%20phylogenetic%20trees&amp;journal=Comput%20Appl%20Biosci&amp;volume=13&amp;pages=235-238&amp;publication_year=1997&amp;author=Rambaut%2CA&amp;author=Grassly%2CNC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">47.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR47">Jones
 D, Taylor W, Thornton J: The rapid generation of mutation data matrices
 from protein sequences. Comput Appl Biosci. 1992, 8: 275-282.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK38Xlt1Okt7w%253D" aria-label="View reference 47 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1633570" aria-label="View reference 47 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 47 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20rapid%20generation%20of%20mutation%20data%20matrices%20from%20protein%20sequences&amp;journal=Comput%20Appl%20Biosci&amp;volume=8&amp;pages=275-282&amp;publication_year=1992&amp;author=Jones%2CD&amp;author=Taylor%2CW&amp;author=Thornton%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">48.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR48">Guindon
 S, Gascuel O: A simple, fast, and accurate algorithm to estimate large 
phylogenies by maximum likelihood. Syst Biol. 2003, 52: 696-704. 
10.1080/10635150390235520.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F10635150390235520" aria-label="View reference 48">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14530136" aria-label="View reference 48 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 48 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;journal=Syst%20Biol&amp;volume=52&amp;pages=696-704&amp;publication_year=2003&amp;author=Guindon%2CS&amp;author=Gascuel%2CO">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">49.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR49">Edgar
 RC: MUSCLE: multiple sequence alignment with high accuracy and high 
throughput. Nucleic Acids Res. 2004, 32: 1792-1797. 10.1093/nar/gkh340.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC390337" aria-label="View reference 49 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2Fgkh340" aria-label="View reference 49">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXisF2ks7w%253D" aria-label="View reference 49 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15034147" aria-label="View reference 49 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 49 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=1792-1797&amp;publication_year=2004&amp;author=Edgar%2CRC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">50.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR50">Edgar
 RC: MUSCLE: a multiple sequence alignment method with reduced time and 
space complexity. BMC Bioinformatics. 2004, 5: 
113-10.1186/1471-2105-5-113.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC517706" aria-label="View reference 50 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1186%2F1471-2105-5-113" aria-label="View reference 50">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15318951" aria-label="View reference 50 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 50 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=MUSCLE%3A%20a%20multiple%20sequence%20alignment%20method%20with%20reduced%20time%20and%20space%20complexity&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;publication_year=2004&amp;author=Edgar%2CRC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><span class="c-article-references__counter">51.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR51">Egan JP: Signal detection theory and ROC analysis, series in cognition and perception. 1975, New York: Academic Press</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 51 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Signal%20detection%20theory%20and%20ROC%20analysis%2C%20series%20in%20cognition%20and%20perception&amp;publication_year=1975&amp;author=Egan%2CJP">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">52.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR52">Metz CE: Basic principles of ROC analysis. Semin Nucl Med. 1978, 8: 283-298. 10.1016/S0001-2998(78)80014-2.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2FS0001-2998%2878%2980014-2" aria-label="View reference 52">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ASTN%3A280%3ADyaE1M3kvVyltQ%253D%253D" aria-label="View reference 52 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=112681" aria-label="View reference 52 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 52 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Basic%20principles%20of%20ROC%20analysis&amp;journal=Semin%20Nucl%20Med&amp;volume=8&amp;pages=283-298&amp;publication_year=1978&amp;author=Metz%2CCE">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">53.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR53">Swets J: Measuring the accuracy of diagnostic systems. Science. 1988, 240: 1285-1293. 10.1126/science.3287615.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1126%2Fscience.3287615" aria-label="View reference 53">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ASTN%3A280%3ADyaL1c3jsF2jtQ%253D%253D" aria-label="View reference 53 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3287615" aria-label="View reference 53 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 53 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Measuring%20the%20accuracy%20of%20diagnostic%20systems&amp;journal=Science&amp;volume=240&amp;pages=1285-1293&amp;publication_year=1988&amp;author=Swets%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">54.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR54">Fawcett T: An introduction to ROC analysis. Pattern Recogn Lett. 2006, 27: 861-874. 10.1016/j.patrec.2005.10.010.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.patrec.2005.10.010" aria-label="View reference 54">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 54 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20introduction%20to%20ROC%20analysis&amp;journal=Pattern%20Recogn%20Lett&amp;volume=27&amp;pages=861-874&amp;publication_year=2006&amp;author=Fawcett%2CT">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">55.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR55">McStewart W: A note on the power of the sign test. Ann Math Stat. 1941, 12: 279-303. 10.1214/aoms/1177731710.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1214%2Faoms%2F1177731710" aria-label="View reference 55">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 55 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20note%20on%20the%20power%20of%20the%20sign%20test&amp;journal=Ann%20Math%20Stat&amp;volume=12&amp;pages=279-303&amp;publication_year=1941&amp;author=McStewart%2CW">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">56.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR56">Dixon WJ, Mood AM: The statistical sign test. J Am Statist Assoc. 1946, 41: 557-566. 10.2307/2280577.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F01621459.1946.10501898" aria-label="View reference 56">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ASTN%3A280%3ADyaH2s%252FktVyltw%253D%253D" aria-label="View reference 56 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 56 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20statistical%20sign%20test&amp;journal=J%20Am%20Statist%20Assoc&amp;volume=41&amp;pages=557-566&amp;publication_year=1946&amp;author=Dixon%2CWJ&amp;author=Mood%2CAM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">57.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR57">Hemelrijk J: A theorem on the sign test when ties are present. Proc Nederl Akad Weten Ser A. 1952, 55: 322-</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 57 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20theorem%20on%20the%20sign%20test%20when%20ties%20are%20present&amp;journal=Proc%20Nederl%20Akad%20Weten%20Ser%20A&amp;volume=55&amp;publication_year=1952&amp;author=Hemelrijk%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">58.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR58">Gascuel
 O: BIONJ: an improved version of the NJ algorithm based on a simple 
model of sequence data. Mol Biol Evol. 1997, 14: 685-695.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Foxfordjournals.molbev.a025808" aria-label="View reference 58">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2sXksVahs7o%253D" aria-label="View reference 58 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9254330" aria-label="View reference 58 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 58 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=BIONJ%3A%20an%20improved%20version%20of%20the%20NJ%20algorithm%20based%20on%20a%20simple%20model%20of%20sequence%20data&amp;journal=Mol%20Biol%20Evol&amp;volume=14&amp;pages=685-695&amp;publication_year=1997&amp;author=Gascuel%2CO">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><span class="c-article-references__counter">59.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR59">Goloboff
 P, Farris S, Nixon KC: TNT (Tree analysis using New Technology) ver. 
1.0. 2000, Published by the authors, Tucumán, Argentina</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 59 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=TNT%20%28Tree%20analysis%20using%20New%20Technology%29%20ver.%201.0&amp;publication_year=2000&amp;author=Goloboff%2CP&amp;author=Farris%2CS&amp;author=Nixon%2CKC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">60.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR60">Nixon
 KC: The parsimony ratchet, a new method for rapid parsimony analysis. 
Cladistics. 1999, 15: 407-414. 10.1111/j.1096-0031.1999.tb00277.x.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2Fj.1096-0031.1999.tb00277.x" aria-label="View reference 60">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 60 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20parsimony%20ratchet%2C%20a%20new%20method%20for%20rapid%20parsimony%20analysis&amp;journal=Cladistics&amp;volume=15&amp;pages=407-414&amp;publication_year=1999&amp;author=Nixon%2CKC">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">61.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR61">Estabrook
 GF, McMorris FR, Meacham CA: Comparison of undirected phylogenetic 
trees based on subtrees of four evolutionary units. Syst Zool. 1985, 34:
 193-200. 10.2307/2413326.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.2307%2F2413326" aria-label="View reference 61">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 61 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20undirected%20phylogenetic%20trees%20based%20on%20subtrees%20of%20four%20evolutionary%20units&amp;journal=Syst%20Zool&amp;volume=34&amp;pages=193-200&amp;publication_year=1985&amp;author=Estabrook%2CGF&amp;author=McMorris%2CFR&amp;author=Meacham%2CCA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">62.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR62">Felsenstein J: Confidence limits on phylogenies: an approach using the bootstrap. Evolution. 1985, 39: 783-791. 10.2307/2408678.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.2307%2F2408678" aria-label="View reference 62">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 62 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Confidence%20limits%20on%20phylogenies%3A%20an%20approach%20using%20the%20bootstrap&amp;journal=Evolution&amp;volume=39&amp;pages=783-791&amp;publication_year=1985&amp;author=Felsenstein%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">63.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR63">Anisimova
 M, Gascuel O: Approximate likelihood ratio test for branches: a fast, 
accurate and powerful alternative. Syst Biol. 2006, 55: 539-552. 
10.1080/10635150600755453.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F10635150600755453" aria-label="View reference 63">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16785212" aria-label="View reference 63 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 63 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Approximate%20likelihood%20ratio%20test%20for%20branches%3A%20a%20fast%2C%20accurate%20and%20powerful%20alternative&amp;journal=Syst%20Biol&amp;volume=55&amp;pages=539-552&amp;publication_year=2006&amp;author=Anisimova%2CM&amp;author=Gascuel%2CO">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">64.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR64">Guindon
 S, Dufayard JF, Lefort V, Anisimova M, Hordijk W, Gascuel O: New 
algorithms and methods to estimate maximum-likelihood phylogenies: 
assessing the performance of PhyML 3.0. Syst Biol. 2010, 59: 307-321. 
10.1093/sysbio/syq010.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fsysbio%2Fsyq010" aria-label="View reference 64">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXks1Kms7s%253D" aria-label="View reference 64 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20525638" aria-label="View reference 64 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 64 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;journal=Syst%20Biol&amp;volume=59&amp;pages=307-321&amp;publication_year=2010&amp;author=Guindon%2CS&amp;author=Dufayard%2CJF&amp;author=Lefort%2CV&amp;author=Anisimova%2CM&amp;author=Hordijk%2CW&amp;author=Gascuel%2CO">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">65.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR65">Hanley
 JA, McNeil BJ: The meaning and the use of the area under a receiver 
operating characteristic (ROC) curve. Radiology. 1982, 143: 29-36.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1148%2Fradiology.143.1.7063747" aria-label="View reference 65">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ASTN%3A280%3ADyaL387ltFyksQ%253D%253D" aria-label="View reference 65 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7063747" aria-label="View reference 65 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 65 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20meaning%20and%20the%20use%20of%20the%20area%20under%20a%20receiver%20operating%20characteristic%20%28ROC%29%20curve&amp;journal=Radiology&amp;volume=143&amp;pages=29-36&amp;publication_year=1982&amp;author=Hanley%2CJA&amp;author=McNeil%2CBJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><span class="c-article-references__counter">66.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR66">Fogarty
 J, Baker RS, Hudson SE: Case studies in the use of ROC curve analysis 
for sensor-based estimates in human computer interaction. Proceedings of
 Graphics Interface (GI 2005): 09-11 May 2005; Victoria, British 
Columbia. Edited by: Michael McCool. 2005, University of Waterloo, 
129-136.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 66 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Proceedings%20of%20Graphics%20Interface%20%28GI%202005%29%3A%2009-11%20May%202005%3B%20Victoria%2C%20British%20Columbia&amp;pages=129-136&amp;publication_year=2005&amp;author=Fogarty%2CJ&amp;author=Baker%2CRS&amp;author=Hudson%2CSE">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">67.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR67">Concatenate: a software to build supermatrices of characters. [<a href="http://www.supertriplets.univ-montp2.fr/PhyloTools.php">http://www.supertriplets.univ-montp2.fr/PhyloTools.php</a>]</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">68.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR68">Adachi
 J, Hasegawa M: Model of amino acid substitution in proteins encoded by 
mitochondrial DNA. J Mol Evol. 1996, 42: 459-468. 10.1007/BF02498640.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1007%2FBF02498640" aria-label="View reference 68">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK28XivFKnurw%253D" aria-label="View reference 68 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8642615" aria-label="View reference 68 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 68 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Model%20of%20amino%20acid%20substitution%20in%20proteins%20encoded%20by%20mitochondrial%20DNA&amp;journal=J%20Mol%20Evol&amp;volume=42&amp;pages=459-468&amp;publication_year=1996&amp;author=Adachi%2CJ&amp;author=Hasegawa%2CM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">69.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR69">Rodríguez-Ezpeleta
 N, Brinkmann H, Burey SC, Roure B, Burger G, Löffelhardt W, Bohnert HJ,
 Philippe H: Monophyly of Primary Photosynthetic Eukaryotes: Green 
Plants, Red Algae, and Glaucophytes. Curr Biol. 2005, 15: 1325-1330. 
10.1016/j.cub.2005.06.040.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.cub.2005.06.040" aria-label="View reference 69">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16051178" aria-label="View reference 69 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 69 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Monophyly%20of%20Primary%20Photosynthetic%20Eukaryotes%3A%20Green%20Plants%2C%20Red%20Algae%2C%20and%20Glaucophytes&amp;journal=Curr%20Biol&amp;volume=15&amp;pages=1325-1330&amp;publication_year=2005&amp;author=Rodr%C3%ADguez-Ezpeleta%2CN&amp;author=Brinkmann%2CH&amp;author=Burey%2CSC&amp;author=Roure%2CB&amp;author=Burger%2CG&amp;author=L%C3%B6ffelhardt%2CW&amp;author=Bohnert%2CHJ&amp;author=Philippe%2CH">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">70.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR70">Hackett
 JD, Yoon HS, Li S, Reyes-Prieto A, Rümmele SE, Bhattacharya D: 
Phylogenomic analysis supports the monophyly of Cryptophytes and 
Haptophytes and the association of Rhizaria with Chromalveolates. Mol 
Biol Evol. 2007, 24: 1702-1713. 10.1093/molbev/msm089.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fmolbev%2Fmsm089" aria-label="View reference 70">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXpsVGrsb0%253D" aria-label="View reference 70 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17488740" aria-label="View reference 70 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 70 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenomic%20analysis%20supports%20the%20monophyly%20of%20Cryptophytes%20and%20Haptophytes%20and%20the%20association%20of%20Rhizaria%20with%20Chromalveolates&amp;journal=Mol%20Biol%20Evol&amp;volume=24&amp;pages=1702-1713&amp;publication_year=2007&amp;author=Hackett%2CJD&amp;author=Yoon%2CHS&amp;author=Li%2CS&amp;author=Reyes-Prieto%2CA&amp;author=R%C3%BCmmele%2CSE&amp;author=Bhattacharya%2CD">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">71.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR71">Burki
 F, Shalchian-Tabrizi K, Pawlowski J: Phylogenomics reveals a new 
'metagroup' including most photosynthetic eukaryotes. Biol Lett. 2008, 
4: 366-369. 10.1098/rsbl.2008.0224.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2610160" aria-label="View reference 71 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1098%2Frsbl.2008.0224" aria-label="View reference 71">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18522922" aria-label="View reference 71 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 71 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenomics%20reveals%20a%20new%20%27metagroup%27%20including%20most%20photosynthetic%20eukaryotes&amp;journal=Biol%20Lett&amp;volume=4&amp;pages=366-369&amp;publication_year=2008&amp;author=Burki%2CF&amp;author=Shalchian-Tabrizi%2CK&amp;author=Pawlowski%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">72.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR72">Hampl
 V, Hug L, Leigh JW, Dacks JB, Lang BF, Simpson AGB, Roger AJ: 
Phylogenomic analyses support the monophyly of Excavata and resolve 
relationships among eukaryotic "supergroups". Proc Natl Acad Sci USA. 
2009, 106: 3859-3864. 10.1073/pnas.0807880106.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2656170" aria-label="View reference 72 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.0807880106" aria-label="View reference 72">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXjt1Gksb0%253D" aria-label="View reference 72 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19237557" aria-label="View reference 72 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 72 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenomic%20analyses%20support%20the%20monophyly%20of%20Excavata%20and%20resolve%20relationships%20among%20eukaryotic%20%22supergroups%22&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=106&amp;pages=3859-3864&amp;publication_year=2009&amp;author=Hampl%2CV&amp;author=Hug%2CL&amp;author=Leigh%2CJW&amp;author=Dacks%2CJB&amp;author=Lang%2CBF&amp;author=Simpson%2CAGB&amp;author=Roger%2CAJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/Book"><span class="c-article-references__counter">73.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR73">Ewens W, Grant G: Statistical methods in bioinformatics: an introduction. 2005, New York: Springer, Second</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 73 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Statistical%20methods%20in%20bioinformatics%3A%20an%20introduction&amp;publication_year=2005&amp;author=Ewens%2CW&amp;author=Grant%2CG">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">74.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR74">Felsenstein
 J: Evolutionary tree from DNA sequences: a maximum likelihood approach.
 J Mol Evol. 1981, 17: 368-376. 10.1007/BF01734359.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1007%2FBF01734359" aria-label="View reference 74">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaL3MXls1Cisr8%253D" aria-label="View reference 74 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7288891" aria-label="View reference 74 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 74 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Evolutionary%20tree%20from%20DNA%20sequences%3A%20a%20maximum%20likelihood%20approach&amp;journal=J%20Mol%20Evol&amp;volume=17&amp;pages=368-376&amp;publication_year=1981&amp;author=Felsenstein%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">75.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR75">Rzhetsky
 A, Nei M: Tests of applicability of several substitution models for DNA
 sequence data. Mol Biol Evol. 1995, 12: 131-151.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Foxfordjournals.molbev.a040182" aria-label="View reference 75">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2MXivVahur0%253D" aria-label="View reference 75 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7877488" aria-label="View reference 75 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 75 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Tests%20of%20applicability%20of%20several%20substitution%20models%20for%20DNA%20sequence%20data&amp;journal=Mol%20Biol%20Evol&amp;volume=12&amp;pages=131-151&amp;publication_year=1995&amp;author=Rzhetsky%2CA&amp;author=Nei%2CM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">76.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR76">Bowker AH: A test for symmetry in contingency tables. J Am Stat Assoc. 1948, 43: 572-574. 10.2307/2280710.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F01621459.1948.10483284" aria-label="View reference 76">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ASTN%3A280%3ADyaH1M%252FisVKluw%253D%253D" aria-label="View reference 76 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18123073" aria-label="View reference 76 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 76 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20test%20for%20symmetry%20in%20contingency%20tables&amp;journal=J%20Am%20Stat%20Assoc&amp;volume=43&amp;pages=572-574&amp;publication_year=1948&amp;author=Bowker%2CAH">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">77.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR77">Rokas
 A, Williams BL, King N, Carroll SB: Genome-scale approaches to 
resolving incongruence in molecular phylogenies. Nature. 2003, 425: 
798-804. 10.1038/nature02053.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1038%2Fnature02053" aria-label="View reference 77">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXotlWqsbc%253D" aria-label="View reference 77 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14574403" aria-label="View reference 77 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 77 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-scale%20approaches%20to%20resolving%20incongruence%20in%20molecular%20phylogenies&amp;journal=Nature&amp;volume=425&amp;pages=798-804&amp;publication_year=2003&amp;author=Rokas%2CA&amp;author=Williams%2CBL&amp;author=King%2CN&amp;author=Carroll%2CSB">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">78.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR78">Yeast multi-gene dataset. [<a href="http://systbio.org/files/Ren_etal_Rokas2003data.txt">http://systbio.org/files/Ren_etal_Rokas2003data.txt</a>]</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">79.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR79">Lanave
 C, Preparata G, Saccone C, Serio G: A new method for calculating 
evolutionary substitution rates. J Mol Evol. 1984, 20: 86-93. 
10.1007/BF02101990.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1007%2FBF02101990" aria-label="View reference 79">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaL2cXktF2hsLY%253D" aria-label="View reference 79 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6429346" aria-label="View reference 79 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 79 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20new%20method%20for%20calculating%20evolutionary%20substitution%20rates&amp;journal=J%20Mol%20Evol&amp;volume=20&amp;pages=86-93&amp;publication_year=1984&amp;author=Lanave%2CC&amp;author=Preparata%2CG&amp;author=Saccone%2CC&amp;author=Serio%2CG">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">80.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR80">Rodriguez
 R, Oliver JL, Marin A, Medina JR: The general stochastic model of 
nucleotide substitution. J Theor Biol. 1990, 142: 485-501. 
10.1016/S0022-5193(05)80104-3.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2FS0022-5193%2805%2980104-3" aria-label="View reference 80">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3cXktlKksbY%253D" aria-label="View reference 80 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2338834" aria-label="View reference 80 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 80 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20general%20stochastic%20model%20of%20nucleotide%20substitution&amp;journal=J%20Theor%20Biol&amp;volume=142&amp;pages=485-501&amp;publication_year=1990&amp;author=Rodriguez%2CR&amp;author=Oliver%2CJL&amp;author=Marin%2CA&amp;author=Medina%2CJR">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">81.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR81">Yang Z: Estimating the pattern of nucleotide substitution. J Mol Evol. 1994, 39: 105-111.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8064867" aria-label="View reference 81 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 81 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimating%20the%20pattern%20of%20nucleotide%20substitution&amp;journal=J%20Mol%20Evol&amp;volume=39&amp;pages=105-111&amp;publication_year=1994&amp;author=Yang%2CZ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">82.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR82">Lake
 JA: Reconstructing evolutionary trees from DNA and protein sequences: 
paralinear distances. Proc Natl Acad Sci USA. 1994, 91: 1455-1459. 
10.1073/pnas.91.4.1455.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC43178" aria-label="View reference 82 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.91.4.1455" aria-label="View reference 82">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2cXitFWqt7Y%253D" aria-label="View reference 82 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8108430" aria-label="View reference 82 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 82 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Reconstructing%20evolutionary%20trees%20from%20DNA%20and%20protein%20sequences%3A%20paralinear%20distances&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=91&amp;pages=1455-1459&amp;publication_year=1994&amp;author=Lake%2CJA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">83.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR83">Lockhart
 PJ, Steel MA, Hendy MD, Penny D: Recovering evolutionary trees under a 
more realistic model of sequence evolution. Mol Biol Evol. 1994, 11: 
605-612.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2cXlsFaks7c%253D" aria-label="View reference 83 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19391266" aria-label="View reference 83 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 83 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Recovering%20evolutionary%20trees%20under%20a%20more%20realistic%20model%20of%20sequence%20evolution&amp;journal=Mol%20Biol%20Evol&amp;volume=11&amp;pages=605-612&amp;publication_year=1994&amp;author=Lockhart%2CPJ&amp;author=Steel%2CMA&amp;author=Hendy%2CMD&amp;author=Penny%2CD">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">84.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR84">Steel
 MA: Recovering a tree from the leaf colourations it generates under a 
Markov model. Appl Math Lett. 1994, 7: 19-23. 
10.1016/0893-9659(94)90024-8.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2F0893-9659%2894%2990024-8" aria-label="View reference 84">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 84 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Recovering%20a%20tree%20from%20the%20leaf%20colourations%20it%20generates%20under%20a%20Markov%20model&amp;journal=Appl%20Math%20Lett&amp;volume=7&amp;pages=19-23&amp;publication_year=1994&amp;author=Steel%2CMA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">85.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR85">Taylor
 DJ, Piel WH: An assessment of accuracy, error, and conflict with 
support values from genome-scale phylogenetic data. Mol Biol Evol. 2004,
 21: 1534-1537. 10.1093/molbev/msh156.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fmolbev%2Fmsh156" aria-label="View reference 85">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXmtlKgt7w%253D" aria-label="View reference 85 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15140947" aria-label="View reference 85 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 85 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20assessment%20of%20accuracy%2C%20error%2C%20and%20conflict%20with%20support%20values%20from%20genome-scale%20phylogenetic%20data&amp;journal=Mol%20Biol%20Evol&amp;volume=21&amp;pages=1534-1537&amp;publication_year=2004&amp;author=Taylor%2CDJ&amp;author=Piel%2CWH">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">86.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR86">Ren
 F, Tanaka H, Yang Z: An empirical examination of the utility of 
codon-substitution models in phylogeny reconstruction. Syst Biol. 2005, 
54: 808-818. 10.1080/10635150500354688.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F10635150500354688" aria-label="View reference 86">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16243764" aria-label="View reference 86 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 86 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20empirical%20examination%20of%20the%20utility%20of%20codon-substitution%20models%20in%20phylogeny%20reconstruction&amp;journal=Syst%20Biol&amp;volume=54&amp;pages=808-818&amp;publication_year=2005&amp;author=Ren%2CF&amp;author=Tanaka%2CH&amp;author=Yang%2CZ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">87.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR87">Burleigh
 JG, Driskell AC, Sanderson MJ: Supertree bootstrapping methods for 
assessing phylogenetic variation among genes in genome-scale data sets. 
Syst Biol. 2006, 55: 426-440. 10.1080/10635150500541722.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F10635150500541722" aria-label="View reference 87">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16861207" aria-label="View reference 87 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 87 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Supertree%20bootstrapping%20methods%20for%20assessing%20phylogenetic%20variation%20among%20genes%20in%20genome-scale%20data%20sets&amp;journal=Syst%20Biol&amp;volume=55&amp;pages=426-440&amp;publication_year=2006&amp;author=Burleigh%2CJG&amp;author=Driskell%2CAC&amp;author=Sanderson%2CMJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">88.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR88">Ané
 C, Larget B, Baum DA, Smith SD, Rokas A: Bayesian estimation of 
concordance among gene trees. Mol Biol Evol. 2007, 24: 412-426. 
10.1093/molbev/msl170.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fmolbev%2Fmsl170" aria-label="View reference 88">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17095535" aria-label="View reference 88 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 88 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Bayesian%20estimation%20of%20concordance%20among%20gene%20trees&amp;journal=Mol%20Biol%20Evol&amp;volume=24&amp;pages=412-426&amp;publication_year=2007&amp;author=An%C3%A9%2CC&amp;author=Larget%2CB&amp;author=Baum%2CDA&amp;author=Smith%2CSD&amp;author=Rokas%2CA">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">89.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR89">Criscuolo
 A, Michel CJ: Phylogenetic inference with weighted codon evolutionary 
distances. J Mol Evol. 2009, 68: 377-392. 10.1007/s00239-009-9212-y.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1007%2Fs00239-009-9212-y" aria-label="View reference 89">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXksVCgu7k%253D" aria-label="View reference 89 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19308635" aria-label="View reference 89 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 89 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenetic%20inference%20with%20weighted%20codon%20evolutionary%20distances&amp;journal=J%20Mol%20Evol&amp;volume=68&amp;pages=377-392&amp;publication_year=2009&amp;author=Criscuolo%2CA&amp;author=Michel%2CCJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">90.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR90">Néron
 B, Ménager H, Maufrais C, Joly N, Maupetit J, Letort S, Carrere S, 
Tuffery P, Letondal C: Mobyle: a new full web bioinformatics framework. 
Bioinformatics. 2009, 25: 3005-3011. 10.1093/bioinformatics/btp493.</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2773253" aria-label="View reference 90 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp493" aria-label="View reference 90">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19689959" aria-label="View reference 90 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 90 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Mobyle%3A%20a%20new%20full%20web%20bioinformatics%20framework&amp;journal=Bioinformatics&amp;volume=25&amp;pages=3005-3011&amp;publication_year=2009&amp;author=N%C3%A9ron%2CB&amp;author=M%C3%A9nager%2CH&amp;author=Maufrais%2CC&amp;author=Joly%2CN&amp;author=Maupetit%2CJ&amp;author=Letort%2CS&amp;author=Carrere%2CS&amp;author=Tuffery%2CP&amp;author=Letondal%2CC">
                        Google Scholar</a></li></ul></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-category="article body" data-track-label="link" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210-references.ris">
                    Download references
                </a></p></div></div></div></section><section aria-labelledby="Ack1"><div class="c-article-section js-article-section" id="Ack1-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content js-collapsible-section" id="Ack1-content"><p>We
 thank Céline Petitjean, Elie Desmond, Céline Brochier and two anonymous
 reviewers for their comments and for testing the program, as well as 
the BMGE team for support. Many thanks to Corrine Maufrais for providing
 us with an ftp home page, and for placing BMGE in the Mobyle portal. 
This research was supported by the PhyloCyano project of the Agence 
Nationale de la Recherche (ANR; 07-JCJC-0094-01).</p></div></div></section><section aria-labelledby="author-information"><div class="c-article-section js-article-section" id="author-information-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content js-collapsible-section" id="author-information-content"><h3 class="c-article-author-information__subtitle u-h3" id="affiliations">Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><h4 class="c-article-author-affiliation__address u-h3">Institut
 Pasteur, Unité de Biologie Moléculaire du Gène chez les Extrêmophiles, 
Département de Microbiologie, 25 rue du Dr Roux, 75015, Paris, France</h4><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Alexis Criscuolo</li><li class="c-article-author-affiliation__authors-item">&nbsp;&amp;&nbsp;Simonetta Gribaldo</li></ul></li></ol><div class="js-hide u-hide-print" data-test="author-info"><h3 class="c-article-author-information__subtitle u-h3">Authors</h3><ol class="c-article-author-authors-search"><li id="auth-1"><h3 class="c-article-author-authors-search__title u-h3 js-search-name">Search for Alexis Criscuolo in:</h3><ul class="c-article-author-authors-search__list"><li class="c-article-author-authors-search__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Alexis+Criscuolo">PubMed</a><span class="bullet"> • </span></li><li class="c-article-author-authors-search__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alexis+Criscuolo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">
                        Google Scholar
                    </a></li></ul></li><li id="auth-2"><h3 class="c-article-author-authors-search__title u-h3 js-search-name">Search for Simonetta Gribaldo in:</h3><ul class="c-article-author-authors-search__list"><li class="c-article-author-authors-search__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Simonetta+Gribaldo">PubMed</a><span class="bullet"> • </span></li><li class="c-article-author-authors-search__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Simonetta+Gribaldo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">
                        Google Scholar
                    </a></li></ul></li></ol></div><h3 class="c-article-author-information__subtitle u-h3" id="corresponding-author">Corresponding author</h3><p>Correspondence to
                <a id="corresp-c1" rel="nofollow" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/email/correspondent/c1/new">Alexis Criscuolo</a>.</p></div></div></section><section aria-labelledby="additional-information"><div class="c-article-section js-article-section" id="additional-information-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content js-collapsible-section" id="additional-information-content"><h3 class="c-article__sub-heading u-h3">Authors' contributions</h3><p>AC
 initiated the project, designed and implemented the different methods 
inside the software, performed the computer simulations, and wrote the 
manuscript. SG supervised the project, and participated in designing the
 entropy-based character trimming method and in writing the manuscript. 
All authors read and approved the final manuscript.</p></div></div></section><section aria-labelledby="Sec16"><div class="c-article-section js-article-section" id="Sec16-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec16">Authors’ original submitted files for images</h2><div class="c-article-section__content js-collapsible-section" id="Sec16-content"><p>Below are the links to the authors’ original submitted files for images.</p><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_1426_MOESM1_ESM.pdf" data-supp-info-image="">Authors’ original file for figure 1</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_1426_MOESM2_ESM.pdf" data-supp-info-image="">Authors’ original file for figure 2</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_1426_MOESM3_ESM.pdf" data-supp-info-image="">Authors’ original file for figure 3</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_1426_MOESM4_ESM.pdf" data-supp-info-image="">Authors’ original file for figure 4</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_1426_MOESM5_ESM.pdf" data-supp-info-image="">Authors’ original file for figure 5</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_1426_MOESM6_ESM.pdf" data-supp-info-image="">Authors’ original file for figure 6</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_1426_MOESM7_ESM.pdf" data-supp-info-image="">Authors’ original file for figure 7</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_1426_MOESM8_ESM.pdf" data-supp-info-image="">Authors’ original file for figure 8</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_1426_MOESM9_ESM.pdf" data-supp-info-image="">Authors’ original file for figure 9</a></h3></div></div></div></section><section aria-labelledby="rightslink"><div class="c-article-section js-article-section" id="rightslink-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content js-collapsible-section" id="rightslink-content"><div class="c-article-license">
                <p>This article is published under license to BioMed 
Central Ltd. This is an Open Access article distributed under the terms 
of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0" rel="license" itemprop="license">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
              </div><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-category="article body" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?imprint=Nature&amp;oa=CC%20BY&amp;title=BMGE%20%28Block%20Mapping%20and%20Gathering%20with%20Entropy%29%3A%20a%20new%20software%20for%20selection%20of%20phylogenetic%20informative%20regions%20from%20multiple%20sequence%20alignments&amp;author=Alexis%20Criscuolo%20et%20al&amp;contentID=10.1186%2F1471-2148-10-210&amp;publication=BMC%20Evolutionary%20Biology&amp;publicationDate=2010-07-13&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info"><div class="c-article-section js-article-section" id="article-info-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content js-collapsible-section" id="article-info-content"><div class="c-bibliographic-information"><div class="c-bibliographic-information__column"><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><h4 class="u-h4">Received</h4><p class="c-bibliographic-information__value"><time datetime="2010-02-10">10 February 2010</time></p></li><li class="c-bibliographic-information__list-item"><h4 class="u-h4">Accepted</h4><p class="c-bibliographic-information__value"><time datetime="2010-07-13">13 July 2010</time></p></li><li class="c-bibliographic-information__list-item"><h4 class="u-h4">Published</h4><p class="c-bibliographic-information__value"><time datetime="2010-07-13">13 July 2010</time></p></li><li class="c-bibliographic-information__list-item"><h4 class="u-h4"><abbr title="Digital Object Identifier">DOI</abbr></h4><p class="c-bibliographic-information__value"><a href="https://doi.org/10.1186/1471-2148-10-210" data-track="click" data-track-action="view doi" data-track-category="article body" data-track-label="link" itemprop="sameAs">https://doi.org/10.1186/1471-2148-10-210</a></p></li></ul><div data-component="share-box"><div class="c-article-share-box"><h3 class="c-article-share-box__title">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" id="get-share-url" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="get shareable link">Get shareable link</button>			<div class="js-no-share-url-container u-display-none" aria-hidden="true"><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div> 			<div class="js-share-url-container u-display-none" aria-hidden="true"><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="select share url"> 		</p><button class="js-copy-share-url c-article-share-box__button--link-like" id="copy-share-url" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="copy share url">Copy to clipboard</button><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">Provided by the Springer Nature SharedIt content-sharing initiative</p></div> 		</div></div><h3 class="c-article__sub-heading u-h3">Keywords</h3><ul class="c-article-subject-list"><li class="c-article-subject-list__subject"><span itemprop="about">Multiple Sequence Alignment</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Phylogenetic Inference</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Initial Alignment</span></li><li class="c-article-subject-list__subject"><span itemprop="about">Universal Genetic Code</span></li><li class="c-article-subject-list__subject"><span itemprop="about">False Branch</span></li></ul><div data-component="article-info-list"></div></div></div></div></div></section>

                        


                    </article>
                </main>

                <div class="c-page-layout__side u-text-sm">
                    <aside>
                        
    
    <div class="c-pdf-download u-clear-both">
        <a href="https://bmcevolbiol.biomedcentral.com/track/pdf/10.1186/1471-2148-10-210" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="pdf-link" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-category="article body" data-track-label="link">
            <span>Download PDF</span>
        </a>
    </div>
    

                        
                        
    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <div class="c-article-metrics-bar">
                <h3 class="u-visually-hidden">Article metrics</h3>
                
                    <div class="c-article-metrics-bar__item">
                        <h4 class="c-article-metrics-bar__label">Accesses</h4>
                        <p class="c-article-metrics-bar__count">13k</p>
                    </div>
                
                
                    <div class="c-article-metrics-bar__item">
                        <h4 class="c-article-metrics-bar__label">Citations</h4>
                        <p class="c-article-metrics-bar__count">322</p>
                    </div>
                
                
                    
                        <div class="c-article-metrics-bar__item">
                            <h4 class="c-article-metrics-bar__label">Altmetric</h4>
                            <p class="c-article-metrics-bar__count">0</p>
                        </div>
                    
                    
                        <div class="c-article-metrics-bar__item">
                            <h4 class="u-visually-hidden">More details</h4>
                            <p class="c-article-metrics-bar__details"><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/metrics" data-track="click" data-track-action="view metrics" data-track-category="article body" data-track-label="link" rel="nofollow">Metrics</a></p>
                        </div>
                    
                
            </div>
        </div>
    


                        
    


                        <div class="c-reading-companion">
                            <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" style="width: 276px;"><ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab">References</button></li></ul>
                                <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" tabindex="0"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Background">Background</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Implementation">Implementation</a></li><li id="rc-sec-Sec7" class="c-reading-companion__section-item"><a href="#Sec7" data-track="click" data-track-action="section anchor" data-track-label="link:Results and Discussion">Results and Discussion</a></li><li id="rc-sec-Sec14" class="c-reading-companion__section-item"><a href="#Sec14" data-track="click" data-track-action="section anchor" data-track-label="link:Conclusions">Conclusions</a></li><li id="rc-sec-Sec15" class="c-reading-companion__section-item"><a href="#Sec15" data-track="click" data-track-action="section anchor" data-track-label="link:Availability and Requirements">Availability and Requirements</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec16" class="c-reading-companion__section-item"><a href="#Sec16" data-track="click" data-track-action="section anchor" data-track-label="link:Authors original submitted files for images">Authors’ original submitted files for images</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li></ul></div>
                                    <div class="js-ad">
    <div class="adsbox c-ad c-ad--MPU1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-MPU1" data-gpt-unitpath="/270604982/bmc/bmcevolbiol/articles" data-gpt-sizes="300x250" data-gpt-targeting="pos=MPU1;doi=10.1186/1471-2148-10-210;kwrd=Multiple Sequence Alignment,Phylogenetic Inference,Initial Alignment,Universal Genetic Code,False Branch;pmc=L21001,L2504X,L25090,M31010,L00004;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/bmcevolbiol/articles&amp;sz=300x250&amp;pos=MPU1&amp;doi=10.1186/1471-2148-10-210&amp;kwrd=Multiple Sequence Alignment,Phylogenetic Inference,Initial Alignment,Universal Genetic Code,False Branch&amp;pmc=L21001,L2504X,L25090,M31010,L00004&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/bmcevolbiol/articles&amp;sz=300x250&amp;pos=MPU1&amp;doi=10.1186/1471-2148-10-210&amp;kwrd=Multiple Sequence Alignment,Phylogenetic Inference,Initial Alignment,Universal Genetic Code,False Branch&amp;pmc=L21001,L2504X,L25090,M31010,L00004&amp;"
                             alt="Advertisement"
                             width="300"
                             height="250">
                    </a>
                </noscript>
            </div>
        </div>
    </div>
</div>
                                </div>
                                <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig1">Figure 1</b></figcaption><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig1_HTML.jpg?as=webp"><img aria-describedby="figure-1-desc" src="12862_2010_Article_1426_Fig1_HTML.jpg" alt="figure1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig2">Figure 2</b></figcaption><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig2_HTML.jpg?as=webp"><img aria-describedby="figure-2-desc" src="12862_2010_Article_1426_Fig2_HTML.jpg" alt="figure2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig3">Figure 3</b></figcaption><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig3_HTML.jpg?as=webp"><img aria-describedby="figure-3-desc" src="12862_2010_Article_1426_Fig3_HTML.jpg" alt="figure3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig4">Figure 4</b></figcaption><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig4_HTML.jpg?as=webp"><img aria-describedby="figure-4-desc" src="12862_2010_Article_1426_Fig4_HTML.jpg" alt="figure4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig5">Figure 5</b></figcaption><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig5_HTML.jpg?as=webp"><img aria-describedby="figure-5-desc" src="12862_2010_Article_1426_Fig5_HTML.jpg" alt="figure5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig6">Figure 6</b></figcaption><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig6_HTML.jpg?as=webp"><img aria-describedby="figure-6-desc" src="12862_2010_Article_1426_Fig6_HTML.jpg" alt="figure6"></picture><p class="c-reading-companion__figure-links"><a href="#Fig6" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/6" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig7">Figure 7</b></figcaption><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig7_HTML.jpg?as=webp"><img aria-describedby="figure-7-desc" src="12862_2010_Article_1426_Fig7_HTML.jpg" alt="figure7"></picture><p class="c-reading-companion__figure-links"><a href="#Fig7" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/7" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig8">Figure 8</b></figcaption><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig8_HTML.jpg?as=webp"><img aria-describedby="figure-8-desc" src="12862_2010_Article_1426_Fig8_HTML.jpg" alt="figure8"></picture><p class="c-reading-companion__figure-links"><a href="#Fig8" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/8" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig9">Figure 9</b></figcaption><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1186%2F1471-2148-10-210/MediaObjects/12862_2010_Article_1426_Fig9_HTML.jpg?as=webp"><img aria-describedby="figure-9-desc" src="12862_2010_Article_1426_Fig9_HTML.jpg" alt="figure9"></picture><p class="c-reading-companion__figure-links"><a href="#Fig9" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/1471-2148-10-210/figures/9" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li></ul></div></div>
                                <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR1">Lake JA: The order of sequence alignment can bias the selection of tree topology. Mol Biol Evol. 1991, 8: 378-385.</p><ul class="c-reading-companion__reference-links"><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3MXit1Cjs7o%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2072863" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20order%20of%20sequence%20alignment%20can%20bias%20the%20selection%20of%20tree%20topology&amp;journal=Mol%20Biol%20Evol&amp;volume=8&amp;pages=378-385&amp;publication_year=1991&amp;author=Lake%2CJA" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR2">Morrison
 DA, Ellis JT: Effects of nucleotide sequence alignment on phylogeny 
estimation: a case study on 18 S rDNAs of apicomplexa. Mol Biol Evol. 
1997, 14: 428-441.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Foxfordjournals.molbev.a025779" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2sXitlOhtLY%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9100373" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20nucleotide%20sequence%20alignment%20on%20phylogeny%20estimation%3A%20a%20case%20study%20on%2018%20S%20rDNAs%20of%20apicomplexa&amp;journal=Mol%20Biol%20Evol&amp;volume=14&amp;pages=428-441&amp;publication_year=1997&amp;author=Morrison%2CDA&amp;author=Ellis%2CJT" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR3">Ogden
 TH, Rosenberg MS: Multiple sequence alignment accuracy and phylogenetic
 inference. Syst Biol. 2006, 55: 314-328. 10.1080/10635150500541730.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1080%2F10635150500541730" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16611602" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Multiple%20sequence%20alignment%20accuracy%20and%20phylogenetic%20inference&amp;journal=Syst%20Biol&amp;volume=55&amp;pages=314-328&amp;publication_year=2006&amp;author=Ogden%2CTH&amp;author=Rosenberg%2CMS" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR4">Wang
 L-S, Leebens-Mack J, Wall PK, Beckmann K, dePamphilis CW, Warnow T: The
 impact of multiple protein sequence alignment on phylogenetic 
estimation. IEEE/ACM Trans Comput Biol Bioinf. 2009.</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=IEEE%2FACM%20Trans%20Comput%20Biol%20Bioinf&amp;publication_year=2009&amp;author=Wang%2CL-S&amp;author=Leebens-Mack%2CJ&amp;author=Wall%2CPK&amp;author=Beckmann%2CK&amp;author=dePamphilis%2CCW&amp;author=Warnow%2CT" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR5">Edgar RC, Batzoglou S: Multiple sequence alignment. Curr Opin Struct Biol. 2006, 16: 368-373. 10.1016/j.sbi.2006.04.004.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1016%2Fj.sbi.2006.04.004" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD28Xls1aksLo%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16679011" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Multiple%20sequence%20alignment&amp;journal=Curr%20Opin%20Struct%20Biol&amp;volume=16&amp;pages=368-373&amp;publication_year=2006&amp;author=Edgar%2CRC&amp;author=Batzoglou%2CS" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR6">Notredame
 C: Recent evolutions of multiple sequence alignment algorithms. PLoS 
Comput Biol. 2007, 3: e123-10.1371/journal.pcbi.0030123.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1963500" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1371%2Fjournal.pcbi.0030123" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17784778" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Recent%20evolutions%20of%20multiple%20sequence%20alignment%20algorithms&amp;journal=PLoS%20Comput%20Biol&amp;volume=3&amp;publication_year=2007&amp;author=Notredame%2CC" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR7">Castresana
 J: Selection of conserved blocks from multiple alignments for their use
 in phylogenetic analysis. Mol Biol Evol. 2000, 17: 540-552.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Foxfordjournals.molbev.a026334" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3cXisVSgt7g%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10742046" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Selection%20of%20conserved%20blocks%20from%20multiple%20alignments%20for%20their%20use%20in%20phylogenetic%20analysis&amp;journal=Mol%20Biol%20Evol&amp;volume=17&amp;pages=540-552&amp;publication_year=2000&amp;author=Castresana%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR8">Talavera
 G, Castresana J: Improvement of phylogenies after removing divergent 
and ambiguously aligned blocks from protein sequence alignments. Syst 
Biol. 2007, 56: 564-577. 10.1080/10635150701472164.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1080%2F10635150701472164" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXhtFKrs7%252FP" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17654362" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Improvement%20of%20phylogenies%20after%20removing%20divergent%20and%20ambiguously%20aligned%20blocks%20from%20protein%20sequence%20alignments&amp;journal=Syst%20Biol&amp;volume=56&amp;pages=564-577&amp;publication_year=2007&amp;author=Talavera%2CG&amp;author=Castresana%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR9">Olsen GJ, Woese CR: Ribosomal RNA: a key to phylogeny. FASEB J. 1993, 7: 113-123.</p><ul class="c-reading-companion__reference-links"><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3sXhtF2jtro%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8422957" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Ribosomal%20RNA%3A%20a%20key%20to%20phylogeny&amp;journal=FASEB%20J&amp;volume=7&amp;pages=113-123&amp;publication_year=1993&amp;author=Olsen%2CGJ&amp;author=Woese%2CCR" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR10">Rodrigo
 AG, Bergquist PR, Bergquist PL: Inadequate support for an evolutionary 
link between the Metazoa and the Fungi. Syst Biol. 1994, 43: 578-584. 
10.1093/sysbio/43.4.578.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fsysbio%2F43.4.578" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Inadequate%20support%20for%20an%20evolutionary%20link%20between%20the%20Metazoa%20and%20the%20Fungi&amp;journal=Syst%20Biol&amp;volume=43&amp;pages=578-584&amp;publication_year=1994&amp;author=Rodrigo%2CAG&amp;author=Bergquist%2CPR&amp;author=Bergquist%2CPL" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR11">Swofford
 DL, Olsen GJ, Waddell PJ, Hillis DM: Phylogenetic inference. Molecular 
Systematics. Edited by: Hillis DM, Moritz C, Mable BK. 1996, Sunderland:
 Sinauer Associates, 407-514.</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20Systematics&amp;pages=407-514&amp;publication_year=1996&amp;author=Swofford%2CDL&amp;author=Olsen%2CGJ&amp;author=Waddell%2CPJ&amp;author=Hillis%2CDM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR12">Rodríguez-Ezpeleta
 N, Brinkmann H, Burey SC, Roure B, Burger G, Löffelhardt W, Philippe H,
 Lang BF: Monophyly of primary photosynthetic eukaryotes: green plants, 
red algae, and glaucophytes. Curr Biol. 2005, 15: 1325-1330. 
10.1016/j.cub.2005.06.040.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1016%2Fj.cub.2005.06.040" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16051178" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Monophyly%20of%20primary%20photosynthetic%20eukaryotes%3A%20green%20plants%2C%20red%20algae%2C%20and%20glaucophytes&amp;journal=Curr%20Biol&amp;volume=15&amp;pages=1325-1330&amp;publication_year=2005&amp;author=Rodr%C3%ADguez-Ezpeleta%2CN&amp;author=Brinkmann%2CH&amp;author=Burey%2CSC&amp;author=Roure%2CB&amp;author=Burger%2CG&amp;author=L%C3%B6ffelhardt%2CW&amp;author=Philippe%2CH&amp;author=Lang%2CBF" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR13">Huerta-Cepas
 J, Bueno A, Dopazo J, Gabaldón T: PhylomeDB: a database for genome-wide
 collections of gene phylogenies. Nucleic Acids Res. 2008, 36: 
D491-D496. 10.1093/nar/gkm899.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2238872" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1093%2Fnar%2Fgkm899" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXhtVSktbg%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17962297" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=PhylomeDB%3A%20a%20database%20for%20genome-wide%20collections%20of%20gene%20phylogenies&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=D491-D496&amp;publication_year=2008&amp;author=Huerta-Cepas%2CJ&amp;author=Bueno%2CA&amp;author=Dopazo%2CJ&amp;author=Gabald%C3%B3n%2CT" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR14">Dress
 AWM, Flamm C, Fritzsch G, Grünewald S, Kruspe M, Prohaska SJ, Stadler 
PF: Noisy: Identification of problematic columns in multiple sequence 
alignments. Algorithms for Molecular Biology. 2008, 3: 
7-10.1186/1748-7188-3-7.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2464588" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1186%2F1748-7188-3-7" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18577231" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Noisy%3A%20Identification%20of%20problematic%20columns%20in%20multiple%20sequence%20alignments&amp;journal=Algorithms%20for%20Molecular%20Biology&amp;volume=3&amp;publication_year=2008&amp;author=Dress%2CAWM&amp;author=Flamm%2CC&amp;author=Fritzsch%2CG&amp;author=Gr%C3%BCnewald%2CS&amp;author=Kruspe%2CM&amp;author=Prohaska%2CSJ&amp;author=Stadler%2CPF" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR15">Swingley
 WD, Blankenship RE, Raymond J: Integrating Markov clustering and 
molecular phylogenetics to reconstruct the cyanobacterial species tree 
from conserved protein families. Mol Biol Evol. 2008, 25: 643-654. 
10.1093/molbev/msn034.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fmolbev%2Fmsn034" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXksFWksbc%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18296704" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Integrating%20Markov%20clustering%20and%20molecular%20phylogenetics%20to%20reconstruct%20the%20cyanobacterial%20species%20tree%20from%20conserved%20protein%20families&amp;journal=Mol%20Biol%20Evol&amp;volume=25&amp;pages=643-654&amp;publication_year=2008&amp;author=Swingley%2CWD&amp;author=Blankenship%2CRE&amp;author=Raymond%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR16">Capella-Gutiérez
 S, Silla-Martínez JM, Gabaldón T: trimAl: a tool for automated 
alignment triming in large-scale phylogenetic analyses. Bioinformatics. 
2009, 25: 1972-1973. 10.1093/bioinformatics/btp348.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp348" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=trimAl%3A%20a%20tool%20for%20automated%20alignment%20triming%20in%20large-scale%20phylogenetic%20analyses&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1972-1973&amp;publication_year=2009&amp;author=Capella-Guti%C3%A9rez%2CS&amp;author=Silla-Mart%C3%ADnez%2CJM&amp;author=Gabald%C3%B3n%2CT" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR17">Daubin
 V, Gouy M, Perrière G: A phylogenomic approach to bacterial phylogeny: 
evidence of a core of genes sharing a common history. Genome Res. 2002, 
12: 1080-1090. 10.1101/gr.187002.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC186629" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1101%2Fgr.187002" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD38XlsVWgsro%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12097345" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20phylogenomic%20approach%20to%20bacterial%20phylogeny%3A%20evidence%20of%20a%20core%20of%20genes%20sharing%20a%20common%20history&amp;journal=Genome%20Res&amp;volume=12&amp;pages=1080-1090&amp;publication_year=2002&amp;author=Daubin%2CV&amp;author=Gouy%2CM&amp;author=Perri%C3%A8re%2CG" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR18">Ciccarelli
 FD, Doerks T, von Mering C, Creevey CJ, Snel B, Bork P: Toward 
automatic reconstruction of a highly resolved tree of life. Science. 
2006, 311: 1283-1287. 10.1126/science.1123061.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1126%2Fscience.1123061" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD28XhvVeku7s%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16513982" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Toward%20automatic%20reconstruction%20of%20a%20highly%20resolved%20tree%20of%20life&amp;journal=Science&amp;volume=311&amp;pages=1283-1287&amp;publication_year=2006&amp;author=Ciccarelli%2CFD&amp;author=Doerks%2CT&amp;author=von%20Mering%2CC&amp;author=Creevey%2CCJ&amp;author=Snel%2CB&amp;author=Bork%2CP" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR19">Adachi
 J, Waddell PJ, Martin W, Hasegawa M: Plastid genome phylogeny and a 
model of amino acid substitution for protein encoded by chloroplast DNA.
 J Mol Evol. 2000, 50: 348-358.</p><ul class="c-reading-companion__reference-links"><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3cXivFyjur8%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10795826" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Plastid%20genome%20phylogeny%20and%20a%20model%20of%20amino%20acid%20substitution%20for%20protein%20encoded%20by%20chloroplast%20DNA&amp;journal=J%20Mol%20Evol&amp;volume=50&amp;pages=348-358&amp;publication_year=2000&amp;author=Adachi%2CJ&amp;author=Waddell%2CPJ&amp;author=Martin%2CW&amp;author=Hasegawa%2CM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR20">McMahon
 MM, Sanderson MJ: Phylogenetic supermatrix analysis of GenBank 
sequences from 2228 Papilionoid legumes. Syst Biol. 2006, 55: 818-836. 
10.1080/10635150600999150.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1080%2F10635150600999150" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17060202" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenetic%20supermatrix%20analysis%20of%20GenBank%20sequences%20from%202228%20Papilionoid%20legumes&amp;journal=Syst%20Biol&amp;volume=55&amp;pages=818-836&amp;publication_year=2006&amp;author=McMahon%2CMM&amp;author=Sanderson%2CMJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR21">Schneider
 TD, Stephens RM: Sequence logos: a new way to display consensus 
sequences. Nucl Acids Res. 1990, 18: 6097-6100. 10.1093/nar/18.20.6097.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC332411" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1093%2Fnar%2F18.20.6097" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3MXitVGmuw%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2172928" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Sequence%20logos%3A%20a%20new%20way%20to%20display%20consensus%20sequences&amp;journal=Nucl%20Acids%20Res&amp;volume=18&amp;pages=6097-6100&amp;publication_year=1990&amp;author=Schneider%2CTD&amp;author=Stephens%2CRM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR22">Jayaswal V, Jermiin LS, Robinson J: Estimation of phylogeny using a general Markov model. Evol Bioinf Online. 2005, 1: 62-80.</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Estimation%20of%20phylogeny%20using%20a%20general%20Markov%20model&amp;journal=Evol%20Bioinf%20Online&amp;volume=1&amp;pages=62-80&amp;publication_year=2005&amp;author=Jayaswal%2CV&amp;author=Jermiin%2CLS&amp;author=Robinson%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR23">Galtier
 N, Gouy M: Inferring pattern and process: maximum-likelihood 
implementation of a nonhomogeneous model of DNA sequence evolution for 
phylogenetic analysis. Mol Biol Evol. 1998, 15: 871-879.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Foxfordjournals.molbev.a025991" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK1cXksFCisbY%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9656487" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Inferring%20pattern%20and%20process%3A%20maximum-likelihood%20implementation%20of%20a%20nonhomogeneous%20model%20of%20DNA%20sequence%20evolution%20for%20phylogenetic%20analysis&amp;journal=Mol%20Biol%20Evol&amp;volume=15&amp;pages=871-879&amp;publication_year=1998&amp;author=Galtier%2CN&amp;author=Gouy%2CM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR24">Jermiin
 LS, Ho SYW, Ababneh F, Robinson J, Larkum AWD: The biasing effect of 
compositional heterogeneity on phylogenetic estimates may be 
underestimated. Syst Biol. 2004, 53: 638-643. 10.1080/10635150490468648.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1080%2F10635150490468648" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15371251" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20biasing%20effect%20of%20compositional%20heterogeneity%20on%20phylogenetic%20estimates%20may%20be%20underestimated&amp;journal=Syst%20Biol&amp;volume=53&amp;pages=638-643&amp;publication_year=2004&amp;author=Jermiin%2CLS&amp;author=Ho%2CSYW&amp;author=Ababneh%2CF&amp;author=Robinson%2CJ&amp;author=Larkum%2CAWD" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR25">Phillips
 MJ, Delsuc F, Penny D: Genome-scale phylogeny and the detection of 
systematic biases. Mol Biol Evol. 2004, 21: 1455-1458. 
10.1093/molbev/msh137.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fmolbev%2Fmsh137" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXltlKlsbw%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15084674" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-scale%20phylogeny%20and%20the%20detection%20of%20systematic%20biases&amp;journal=Mol%20Biol%20Evol&amp;volume=21&amp;pages=1455-1458&amp;publication_year=2004&amp;author=Phillips%2CMJ&amp;author=Delsuc%2CF&amp;author=Penny%2CD" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR26">International
 Union of Pure and Applied Chemistery and International Union of 
Biochemistery (IUPAC-IUB) Commission on Biochemical Nomenclature: 
Abbreviations and symbols for nucleic acids, polynucleotides and their 
constituents. Biochem J. 1970, 120: 449-454.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1042%2Fbj1200449" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Abbreviations%20and%20symbols%20for%20nucleic%20acids%2C%20polynucleotides%20and%20their%20constituents&amp;journal=Biochem%20J&amp;volume=120&amp;pages=449-454&amp;publication_year=1970" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR27">Dayhoff
 MO, Schwartz RM, Orcutt BD: A model of evolutionary change in proteins.
 Atlas of Protein Sequence and Structure. Edited by: Dayhoff MO. 1978, 
Washington: National Biomedical Research Foundation, 5 (Suppl 3): 
345-352.</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR28">Embley
 TM, van der Giezen M, Horner DS, Dyal PL, Foster P: Mitochondria and 
hydrogenosomes are two forms of the same fundamental organelle. Philos 
Trans R Soc Lond B Biol Sci. 2003, 358: 191-203. 10.1098/rstb.2002.1190.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1693103" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1098%2Frstb.2002.1190" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXktVWhs74%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12594927" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Mitochondria%20and%20hydrogenosomes%20are%20two%20forms%20of%20the%20same%20fundamental%20organelle&amp;journal=Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci&amp;volume=358&amp;pages=191-203&amp;publication_year=2003&amp;author=Embley%2CTM&amp;author=van%20der%20Giezen%2CM&amp;author=Horner%2CDS&amp;author=Dyal%2CPL&amp;author=Foster%2CP" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR29">Susko
 E, Roger AJ: On reduced amino acid alphabets for phylogenetic 
inference. Mol Biol Evol. 2007, 24: 2139-2150. 10.1093/molbev/msm144.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fmolbev%2Fmsm144" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXhtFSjsb7P" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17652333" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=On%20reduced%20amino%20acid%20alphabets%20for%20phylogenetic%20inference&amp;journal=Mol%20Biol%20Evol&amp;volume=24&amp;pages=2139-2150&amp;publication_year=2007&amp;author=Susko%2CE&amp;author=Roger%2CAJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR30">Stuart A: A test for homogeneity of the marginal distributions in a two-way classification. Biometrika. 1955, 42: 412-416.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fbiomet%2F42.3-4.412" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20test%20for%20homogeneity%20of%20the%20marginal%20distributions%20in%20a%20two-way%20classification&amp;journal=Biometrika&amp;volume=42&amp;pages=412-416&amp;publication_year=1955&amp;author=Stuart%2CA" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR31">Ababneh
 F, Jermiin LS, Ma C, Robinson J: Matched-pairs tests of homogeneity 
with applications to homologous nucleotide sequences. Bioinformatics. 
2006, 22: 1225-1231. 10.1093/bioinformatics/btl064.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtl064" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD28Xktlaku7Y%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16492684" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Matched-pairs%20tests%20of%20homogeneity%20with%20applications%20to%20homologous%20nucleotide%20sequences&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1225-1231&amp;publication_year=2006&amp;author=Ababneh%2CF&amp;author=Jermiin%2CLS&amp;author=Ma%2CC&amp;author=Robinson%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR32">International
 Union of Pure and Applied Chemistery and International Union of 
Biochemistery (IUPAC-IUB) Commission on Biochemical Nomenclature: A 
one-letter notation for amino acid sequences (definitive rules). Pure 
Appl Chem. 1972, 31: 639-645. 10.1351/pac197231040639.</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20one-letter%20notation%20for%20amino%20acid%20sequences%20%28definitive%20rules%29&amp;journal=Pure%20Appl%20Chem&amp;volume=31&amp;pages=639-645&amp;publication_year=1972" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR33">Von Neumann J: Mathematische Grundlagen der Quantenmechanik. 1932, Berlin: Springer</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Mathematische%20Grundlagen%20der%20Quantenmechanik&amp;publication_year=1932&amp;author=Von%20Neumann%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR34">Caffrey
 DR, Somaroo S, Hughes JD, Mintseris J, Huang ES: Are protein-protein 
interfaces more conserved in sequence than the rest of the protein 
surface?. Protein Sci. 2004, 13: 190-202. 10.1110/ps.03323604.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2286531" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1110%2Fps.03323604" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXhsFWiug%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14691234" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Are%20protein-protein%20interfaces%20more%20conserved%20in%20sequence%20than%20the%20rest%20of%20the%20protein%20surface%3F&amp;journal=Protein%20Sci&amp;volume=13&amp;pages=190-202&amp;publication_year=2004&amp;author=Caffrey%2CDR&amp;author=Somaroo%2CS&amp;author=Hughes%2CJD&amp;author=Mintseris%2CJ&amp;author=Huang%2CES" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR35">JAMA: A Java Matrix Package. [<a href="http://math.nist.gov/javanumerics/jama/">http://math.nist.gov/javanumerics/jama/</a>]</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR36">Shannon C: A mathematical theory of communication. Bell System Tech J. 1948, 27: 379-423. 623-656</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1002%2Fj.1538-7305.1948.tb01338.x" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20mathematical%20theory%20of%20communication&amp;journal=Bell%20System%20Tech%20J&amp;volume=27&amp;pages=379-423&amp;publication_year=1948&amp;author=Shannon%2CC" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR37">Taylor WR: The classification of amino acid conservation. J Theor Biol. 1986, 119: 205-218. 10.1016/S0022-5193(86)80075-3.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1016%2FS0022-5193%2886%2980075-3" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaL28XitVensbc%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3461222" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20classification%20of%20amino%20acid%20conservation&amp;journal=J%20Theor%20Biol&amp;volume=119&amp;pages=205-218&amp;publication_year=1986&amp;author=Taylor%2CWR" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR38">Bordo
 D, Argos P: Suggestion for "safe" residue substitutions in 
site-directed mutagenesis. J mol Biol. 1991, 217: 721-729. 
10.1016/0022-2836(91)90528-E.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1016%2F0022-2836%2891%2990528-E" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3MXitVejtL0%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2005621" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Suggestion%20for%20%22safe%22%20residue%20substitutions%20in%20site-directed%20mutagenesis&amp;journal=J%20mol%20Biol&amp;volume=217&amp;pages=721-729&amp;publication_year=1991&amp;author=Bordo%2CD&amp;author=Argos%2CP" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR39">Henikoff
 S, Henikoff JG: Amino acid substitution matrices from protein blocks. 
Proc Natl Acad Sci. 1992, 89: 10915-10919. 10.1073/pnas.89.22.10915.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC50453" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1073%2Fpnas.89.22.10915" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3sXjsFCgsQ%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1438297" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Amino%20acid%20substitution%20matrices%20from%20protein%20blocks&amp;journal=Proc%20Natl%20Acad%20Sci&amp;volume=89&amp;pages=10915-10919&amp;publication_year=1992&amp;author=Henikoff%2CS&amp;author=Henikoff%2CJG" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR40">BLOSUM matrices. [<a href="http://blocks.fhcrc.org/blocks/uploads/blosum/">http://blocks.fhcrc.org/blocks/uploads/blosum/</a>]</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR41">Eddy SR: Where did the BLOSUM62 alignment score matrix come from?. Nat Biotechnol. 2004, 22: 1035-1036. 10.1038/nbt0804-1035.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1038%2Fnbt0804-1035" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXmtFWrtL8%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15286655" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Where%20did%20the%20BLOSUM62%20alignment%20score%20matrix%20come%20from%3F&amp;journal=Nat%20Biotechnol&amp;volume=22&amp;pages=1035-1036&amp;publication_year=2004&amp;author=Eddy%2CSR" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR42">States
 DJ, Gish W, Altschul SF: Improved sensitivity of nucleic acid database 
searches using application-specific scoring matrices. Methods: A 
Companion to Methods Enzymol. 1991, 3: 66-70. 
10.1016/S1046-2023(05)80165-3.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1016%2FS1046-2023%2805%2980165-3" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK38Xhs1Ohu70%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20sensitivity%20of%20nucleic%20acid%20database%20searches%20using%20application-specific%20scoring%20matrices&amp;journal=Methods%3A%20A%20Companion%20to%20Methods%20Enzymol&amp;volume=3&amp;pages=66-70&amp;publication_year=1991&amp;author=States%2CDJ&amp;author=Gish%2CW&amp;author=Altschul%2CSF" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR43">Uesaka
 H: Validity and applicability of several tests for comparing marginal 
distributions of a square table with ordered categories. 
Behaviormetrika. 1991, 30: 65-78. 10.2333/bhmk.18.30_65.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.2333%2Fbhmk.18.30_65" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Validity%20and%20applicability%20of%20several%20tests%20for%20comparing%20marginal%20distributions%20of%20a%20square%20table%20with%20ordered%20categories&amp;journal=Behaviormetrika&amp;volume=30&amp;pages=65-78&amp;publication_year=1991&amp;author=Uesaka%2CH" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR44">Press
 WH, Teukolsky SA, Vetterling WT, Flannery BP: Numerical Recipes in C --
 The Art of Scientific Computing. 1992, Cambridge: Cambridge University 
Press, 2</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Numerical%20Recipes%20in%20C%20--%20The%20Art%20of%20Scientific%20Computing&amp;publication_year=1992&amp;author=Press%2CWH&amp;author=Teukolsky%2CSA&amp;author=Vetterling%2CWT&amp;author=Flannery%2CBP" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR45">Artificial datasets used in [48]. [<a href="http://www.atgc-montpellier.fr/phyml/datasets.php">http://www.atgc-montpellier.fr/phyml/datasets.php</a>]</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR46">Rambaut
 A, Grassly NC: Seq-Gen: an application for the Monte Carlo simulation 
of DNA sequence evolution along phylogenetic trees. Comput Appl Biosci. 
1997, 13: 235-238.</p><ul class="c-reading-companion__reference-links"><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2sXksVGntL4%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9183526" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Seq-Gen%3A%20an%20application%20for%20the%20Monte%20Carlo%20simulation%20of%20DNA%20sequence%20evolution%20along%20phylogenetic%20trees&amp;journal=Comput%20Appl%20Biosci&amp;volume=13&amp;pages=235-238&amp;publication_year=1997&amp;author=Rambaut%2CA&amp;author=Grassly%2CNC" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR47">Jones
 D, Taylor W, Thornton J: The rapid generation of mutation data matrices
 from protein sequences. Comput Appl Biosci. 1992, 8: 275-282.</p><ul class="c-reading-companion__reference-links"><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK38Xlt1Okt7w%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1633570" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20rapid%20generation%20of%20mutation%20data%20matrices%20from%20protein%20sequences&amp;journal=Comput%20Appl%20Biosci&amp;volume=8&amp;pages=275-282&amp;publication_year=1992&amp;author=Jones%2CD&amp;author=Taylor%2CW&amp;author=Thornton%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR48">Guindon
 S, Gascuel O: A simple, fast, and accurate algorithm to estimate large 
phylogenies by maximum likelihood. Syst Biol. 2003, 52: 696-704. 
10.1080/10635150390235520.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1080%2F10635150390235520" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14530136" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;journal=Syst%20Biol&amp;volume=52&amp;pages=696-704&amp;publication_year=2003&amp;author=Guindon%2CS&amp;author=Gascuel%2CO" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR49">Edgar
 RC: MUSCLE: multiple sequence alignment with high accuracy and high 
throughput. Nucleic Acids Res. 2004, 32: 1792-1797. 10.1093/nar/gkh340.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC390337" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1093%2Fnar%2Fgkh340" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXisF2ks7w%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15034147" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=1792-1797&amp;publication_year=2004&amp;author=Edgar%2CRC" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR50">Edgar
 RC: MUSCLE: a multiple sequence alignment method with reduced time and 
space complexity. BMC Bioinformatics. 2004, 5: 
113-10.1186/1471-2105-5-113.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC517706" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1186%2F1471-2105-5-113" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15318951" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=MUSCLE%3A%20a%20multiple%20sequence%20alignment%20method%20with%20reduced%20time%20and%20space%20complexity&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;publication_year=2004&amp;author=Edgar%2CRC" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR51">Egan JP: Signal detection theory and ROC analysis, series in cognition and perception. 1975, New York: Academic Press</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Signal%20detection%20theory%20and%20ROC%20analysis%2C%20series%20in%20cognition%20and%20perception&amp;publication_year=1975&amp;author=Egan%2CJP" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR52">Metz CE: Basic principles of ROC analysis. Semin Nucl Med. 1978, 8: 283-298. 10.1016/S0001-2998(78)80014-2.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1016%2FS0001-2998%2878%2980014-2" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ASTN%3A280%3ADyaE1M3kvVyltQ%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=112681" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Basic%20principles%20of%20ROC%20analysis&amp;journal=Semin%20Nucl%20Med&amp;volume=8&amp;pages=283-298&amp;publication_year=1978&amp;author=Metz%2CCE" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR53">Swets J: Measuring the accuracy of diagnostic systems. Science. 1988, 240: 1285-1293. 10.1126/science.3287615.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1126%2Fscience.3287615" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ASTN%3A280%3ADyaL1c3jsF2jtQ%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3287615" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Measuring%20the%20accuracy%20of%20diagnostic%20systems&amp;journal=Science&amp;volume=240&amp;pages=1285-1293&amp;publication_year=1988&amp;author=Swets%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR54">Fawcett T: An introduction to ROC analysis. Pattern Recogn Lett. 2006, 27: 861-874. 10.1016/j.patrec.2005.10.010.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1016%2Fj.patrec.2005.10.010" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20introduction%20to%20ROC%20analysis&amp;journal=Pattern%20Recogn%20Lett&amp;volume=27&amp;pages=861-874&amp;publication_year=2006&amp;author=Fawcett%2CT" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR55">McStewart W: A note on the power of the sign test. Ann Math Stat. 1941, 12: 279-303. 10.1214/aoms/1177731710.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1214%2Faoms%2F1177731710" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20note%20on%20the%20power%20of%20the%20sign%20test&amp;journal=Ann%20Math%20Stat&amp;volume=12&amp;pages=279-303&amp;publication_year=1941&amp;author=McStewart%2CW" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR56">Dixon WJ, Mood AM: The statistical sign test. J Am Statist Assoc. 1946, 41: 557-566. 10.2307/2280577.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1080%2F01621459.1946.10501898" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ASTN%3A280%3ADyaH2s%252FktVyltw%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20statistical%20sign%20test&amp;journal=J%20Am%20Statist%20Assoc&amp;volume=41&amp;pages=557-566&amp;publication_year=1946&amp;author=Dixon%2CWJ&amp;author=Mood%2CAM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR57">Hemelrijk J: A theorem on the sign test when ties are present. Proc Nederl Akad Weten Ser A. 1952, 55: 322-</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20theorem%20on%20the%20sign%20test%20when%20ties%20are%20present&amp;journal=Proc%20Nederl%20Akad%20Weten%20Ser%20A&amp;volume=55&amp;publication_year=1952&amp;author=Hemelrijk%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR58">Gascuel
 O: BIONJ: an improved version of the NJ algorithm based on a simple 
model of sequence data. Mol Biol Evol. 1997, 14: 685-695.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Foxfordjournals.molbev.a025808" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2sXksVahs7o%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9254330" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=BIONJ%3A%20an%20improved%20version%20of%20the%20NJ%20algorithm%20based%20on%20a%20simple%20model%20of%20sequence%20data&amp;journal=Mol%20Biol%20Evol&amp;volume=14&amp;pages=685-695&amp;publication_year=1997&amp;author=Gascuel%2CO" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR59">Goloboff
 P, Farris S, Nixon KC: TNT (Tree analysis using New Technology) ver. 
1.0. 2000, Published by the authors, Tucumán, Argentina</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=TNT%20%28Tree%20analysis%20using%20New%20Technology%29%20ver.%201.0&amp;publication_year=2000&amp;author=Goloboff%2CP&amp;author=Farris%2CS&amp;author=Nixon%2CKC" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR60">Nixon
 KC: The parsimony ratchet, a new method for rapid parsimony analysis. 
Cladistics. 1999, 15: 407-414. 10.1111/j.1096-0031.1999.tb00277.x.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1111%2Fj.1096-0031.1999.tb00277.x" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20parsimony%20ratchet%2C%20a%20new%20method%20for%20rapid%20parsimony%20analysis&amp;journal=Cladistics&amp;volume=15&amp;pages=407-414&amp;publication_year=1999&amp;author=Nixon%2CKC" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR61">Estabrook
 GF, McMorris FR, Meacham CA: Comparison of undirected phylogenetic 
trees based on subtrees of four evolutionary units. Syst Zool. 1985, 34:
 193-200. 10.2307/2413326.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.2307%2F2413326" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20undirected%20phylogenetic%20trees%20based%20on%20subtrees%20of%20four%20evolutionary%20units&amp;journal=Syst%20Zool&amp;volume=34&amp;pages=193-200&amp;publication_year=1985&amp;author=Estabrook%2CGF&amp;author=McMorris%2CFR&amp;author=Meacham%2CCA" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR62">Felsenstein J: Confidence limits on phylogenies: an approach using the bootstrap. Evolution. 1985, 39: 783-791. 10.2307/2408678.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.2307%2F2408678" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Confidence%20limits%20on%20phylogenies%3A%20an%20approach%20using%20the%20bootstrap&amp;journal=Evolution&amp;volume=39&amp;pages=783-791&amp;publication_year=1985&amp;author=Felsenstein%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR63">Anisimova
 M, Gascuel O: Approximate likelihood ratio test for branches: a fast, 
accurate and powerful alternative. Syst Biol. 2006, 55: 539-552. 
10.1080/10635150600755453.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1080%2F10635150600755453" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16785212" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Approximate%20likelihood%20ratio%20test%20for%20branches%3A%20a%20fast%2C%20accurate%20and%20powerful%20alternative&amp;journal=Syst%20Biol&amp;volume=55&amp;pages=539-552&amp;publication_year=2006&amp;author=Anisimova%2CM&amp;author=Gascuel%2CO" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR64">Guindon
 S, Dufayard JF, Lefort V, Anisimova M, Hordijk W, Gascuel O: New 
algorithms and methods to estimate maximum-likelihood phylogenies: 
assessing the performance of PhyML 3.0. Syst Biol. 2010, 59: 307-321. 
10.1093/sysbio/syq010.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fsysbio%2Fsyq010" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXks1Kms7s%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20525638" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;journal=Syst%20Biol&amp;volume=59&amp;pages=307-321&amp;publication_year=2010&amp;author=Guindon%2CS&amp;author=Dufayard%2CJF&amp;author=Lefort%2CV&amp;author=Anisimova%2CM&amp;author=Hordijk%2CW&amp;author=Gascuel%2CO" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR65">Hanley
 JA, McNeil BJ: The meaning and the use of the area under a receiver 
operating characteristic (ROC) curve. Radiology. 1982, 143: 29-36.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1148%2Fradiology.143.1.7063747" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ASTN%3A280%3ADyaL387ltFyksQ%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7063747" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20meaning%20and%20the%20use%20of%20the%20area%20under%20a%20receiver%20operating%20characteristic%20%28ROC%29%20curve&amp;journal=Radiology&amp;volume=143&amp;pages=29-36&amp;publication_year=1982&amp;author=Hanley%2CJA&amp;author=McNeil%2CBJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR66">Fogarty
 J, Baker RS, Hudson SE: Case studies in the use of ROC curve analysis 
for sensor-based estimates in human computer interaction. Proceedings of
 Graphics Interface (GI 2005): 09-11 May 2005; Victoria, British 
Columbia. Edited by: Michael McCool. 2005, University of Waterloo, 
129-136.</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Proceedings%20of%20Graphics%20Interface%20%28GI%202005%29%3A%2009-11%20May%202005%3B%20Victoria%2C%20British%20Columbia&amp;pages=129-136&amp;publication_year=2005&amp;author=Fogarty%2CJ&amp;author=Baker%2CRS&amp;author=Hudson%2CSE" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR67">Concatenate: a software to build supermatrices of characters. [<a href="http://www.supertriplets.univ-montp2.fr/PhyloTools.php">http://www.supertriplets.univ-montp2.fr/PhyloTools.php</a>]</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR68">Adachi
 J, Hasegawa M: Model of amino acid substitution in proteins encoded by 
mitochondrial DNA. J Mol Evol. 1996, 42: 459-468. 10.1007/BF02498640.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1007%2FBF02498640" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK28XivFKnurw%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8642615" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Model%20of%20amino%20acid%20substitution%20in%20proteins%20encoded%20by%20mitochondrial%20DNA&amp;journal=J%20Mol%20Evol&amp;volume=42&amp;pages=459-468&amp;publication_year=1996&amp;author=Adachi%2CJ&amp;author=Hasegawa%2CM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR69">Rodríguez-Ezpeleta
 N, Brinkmann H, Burey SC, Roure B, Burger G, Löffelhardt W, Bohnert HJ,
 Philippe H: Monophyly of Primary Photosynthetic Eukaryotes: Green 
Plants, Red Algae, and Glaucophytes. Curr Biol. 2005, 15: 1325-1330. 
10.1016/j.cub.2005.06.040.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1016%2Fj.cub.2005.06.040" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16051178" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Monophyly%20of%20Primary%20Photosynthetic%20Eukaryotes%3A%20Green%20Plants%2C%20Red%20Algae%2C%20and%20Glaucophytes&amp;journal=Curr%20Biol&amp;volume=15&amp;pages=1325-1330&amp;publication_year=2005&amp;author=Rodr%C3%ADguez-Ezpeleta%2CN&amp;author=Brinkmann%2CH&amp;author=Burey%2CSC&amp;author=Roure%2CB&amp;author=Burger%2CG&amp;author=L%C3%B6ffelhardt%2CW&amp;author=Bohnert%2CHJ&amp;author=Philippe%2CH" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR70">Hackett
 JD, Yoon HS, Li S, Reyes-Prieto A, Rümmele SE, Bhattacharya D: 
Phylogenomic analysis supports the monophyly of Cryptophytes and 
Haptophytes and the association of Rhizaria with Chromalveolates. Mol 
Biol Evol. 2007, 24: 1702-1713. 10.1093/molbev/msm089.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fmolbev%2Fmsm089" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXpsVGrsb0%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17488740" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenomic%20analysis%20supports%20the%20monophyly%20of%20Cryptophytes%20and%20Haptophytes%20and%20the%20association%20of%20Rhizaria%20with%20Chromalveolates&amp;journal=Mol%20Biol%20Evol&amp;volume=24&amp;pages=1702-1713&amp;publication_year=2007&amp;author=Hackett%2CJD&amp;author=Yoon%2CHS&amp;author=Li%2CS&amp;author=Reyes-Prieto%2CA&amp;author=R%C3%BCmmele%2CSE&amp;author=Bhattacharya%2CD" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR71">Burki
 F, Shalchian-Tabrizi K, Pawlowski J: Phylogenomics reveals a new 
'metagroup' including most photosynthetic eukaryotes. Biol Lett. 2008, 
4: 366-369. 10.1098/rsbl.2008.0224.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2610160" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1098%2Frsbl.2008.0224" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18522922" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenomics%20reveals%20a%20new%20%27metagroup%27%20including%20most%20photosynthetic%20eukaryotes&amp;journal=Biol%20Lett&amp;volume=4&amp;pages=366-369&amp;publication_year=2008&amp;author=Burki%2CF&amp;author=Shalchian-Tabrizi%2CK&amp;author=Pawlowski%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR72">Hampl
 V, Hug L, Leigh JW, Dacks JB, Lang BF, Simpson AGB, Roger AJ: 
Phylogenomic analyses support the monophyly of Excavata and resolve 
relationships among eukaryotic "supergroups". Proc Natl Acad Sci USA. 
2009, 106: 3859-3864. 10.1073/pnas.0807880106.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2656170" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1073%2Fpnas.0807880106" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXjt1Gksb0%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19237557" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenomic%20analyses%20support%20the%20monophyly%20of%20Excavata%20and%20resolve%20relationships%20among%20eukaryotic%20%22supergroups%22&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=106&amp;pages=3859-3864&amp;publication_year=2009&amp;author=Hampl%2CV&amp;author=Hug%2CL&amp;author=Leigh%2CJW&amp;author=Dacks%2CJB&amp;author=Lang%2CBF&amp;author=Simpson%2CAGB&amp;author=Roger%2CAJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR73">Ewens W, Grant G: Statistical methods in bioinformatics: an introduction. 2005, New York: Springer, Second</p><ul class="c-reading-companion__reference-links"><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Statistical%20methods%20in%20bioinformatics%3A%20an%20introduction&amp;publication_year=2005&amp;author=Ewens%2CW&amp;author=Grant%2CG" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR74">Felsenstein
 J: Evolutionary tree from DNA sequences: a maximum likelihood approach.
 J Mol Evol. 1981, 17: 368-376. 10.1007/BF01734359.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1007%2FBF01734359" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaL3MXls1Cisr8%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7288891" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Evolutionary%20tree%20from%20DNA%20sequences%3A%20a%20maximum%20likelihood%20approach&amp;journal=J%20Mol%20Evol&amp;volume=17&amp;pages=368-376&amp;publication_year=1981&amp;author=Felsenstein%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR75">Rzhetsky
 A, Nei M: Tests of applicability of several substitution models for DNA
 sequence data. Mol Biol Evol. 1995, 12: 131-151.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Foxfordjournals.molbev.a040182" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2MXivVahur0%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7877488" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Tests%20of%20applicability%20of%20several%20substitution%20models%20for%20DNA%20sequence%20data&amp;journal=Mol%20Biol%20Evol&amp;volume=12&amp;pages=131-151&amp;publication_year=1995&amp;author=Rzhetsky%2CA&amp;author=Nei%2CM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR76">Bowker AH: A test for symmetry in contingency tables. J Am Stat Assoc. 1948, 43: 572-574. 10.2307/2280710.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1080%2F01621459.1948.10483284" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ASTN%3A280%3ADyaH1M%252FisVKluw%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18123073" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20test%20for%20symmetry%20in%20contingency%20tables&amp;journal=J%20Am%20Stat%20Assoc&amp;volume=43&amp;pages=572-574&amp;publication_year=1948&amp;author=Bowker%2CAH" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR77">Rokas
 A, Williams BL, King N, Carroll SB: Genome-scale approaches to 
resolving incongruence in molecular phylogenies. Nature. 2003, 425: 
798-804. 10.1038/nature02053.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1038%2Fnature02053" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXotlWqsbc%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14574403" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-scale%20approaches%20to%20resolving%20incongruence%20in%20molecular%20phylogenies&amp;journal=Nature&amp;volume=425&amp;pages=798-804&amp;publication_year=2003&amp;author=Rokas%2CA&amp;author=Williams%2CBL&amp;author=King%2CN&amp;author=Carroll%2CSB" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR78">Yeast multi-gene dataset. [<a href="http://systbio.org/files/Ren_etal_Rokas2003data.txt">http://systbio.org/files/Ren_etal_Rokas2003data.txt</a>]</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR79">Lanave
 C, Preparata G, Saccone C, Serio G: A new method for calculating 
evolutionary substitution rates. J Mol Evol. 1984, 20: 86-93. 
10.1007/BF02101990.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1007%2FBF02101990" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaL2cXktF2hsLY%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6429346" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20new%20method%20for%20calculating%20evolutionary%20substitution%20rates&amp;journal=J%20Mol%20Evol&amp;volume=20&amp;pages=86-93&amp;publication_year=1984&amp;author=Lanave%2CC&amp;author=Preparata%2CG&amp;author=Saccone%2CC&amp;author=Serio%2CG" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR80">Rodriguez
 R, Oliver JL, Marin A, Medina JR: The general stochastic model of 
nucleotide substitution. J Theor Biol. 1990, 142: 485-501. 
10.1016/S0022-5193(05)80104-3.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1016%2FS0022-5193%2805%2980104-3" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK3cXktlKksbY%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2338834" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20general%20stochastic%20model%20of%20nucleotide%20substitution&amp;journal=J%20Theor%20Biol&amp;volume=142&amp;pages=485-501&amp;publication_year=1990&amp;author=Rodriguez%2CR&amp;author=Oliver%2CJL&amp;author=Marin%2CA&amp;author=Medina%2CJR" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR81">Yang Z: Estimating the pattern of nucleotide substitution. J Mol Evol. 1994, 39: 105-111.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8064867" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Estimating%20the%20pattern%20of%20nucleotide%20substitution&amp;journal=J%20Mol%20Evol&amp;volume=39&amp;pages=105-111&amp;publication_year=1994&amp;author=Yang%2CZ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR82">Lake
 JA: Reconstructing evolutionary trees from DNA and protein sequences: 
paralinear distances. Proc Natl Acad Sci USA. 1994, 91: 1455-1459. 
10.1073/pnas.91.4.1455.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC43178" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1073%2Fpnas.91.4.1455" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2cXitFWqt7Y%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8108430" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Reconstructing%20evolutionary%20trees%20from%20DNA%20and%20protein%20sequences%3A%20paralinear%20distances&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=91&amp;pages=1455-1459&amp;publication_year=1994&amp;author=Lake%2CJA" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR83">Lockhart
 PJ, Steel MA, Hendy MD, Penny D: Recovering evolutionary trees under a 
more realistic model of sequence evolution. Mol Biol Evol. 1994, 11: 
605-612.</p><ul class="c-reading-companion__reference-links"><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADyaK2cXlsFaks7c%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19391266" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Recovering%20evolutionary%20trees%20under%20a%20more%20realistic%20model%20of%20sequence%20evolution&amp;journal=Mol%20Biol%20Evol&amp;volume=11&amp;pages=605-612&amp;publication_year=1994&amp;author=Lockhart%2CPJ&amp;author=Steel%2CMA&amp;author=Hendy%2CMD&amp;author=Penny%2CD" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR84">Steel
 MA: Recovering a tree from the leaf colourations it generates under a 
Markov model. Appl Math Lett. 1994, 7: 19-23. 
10.1016/0893-9659(94)90024-8.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1016%2F0893-9659%2894%2990024-8" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Recovering%20a%20tree%20from%20the%20leaf%20colourations%20it%20generates%20under%20a%20Markov%20model&amp;journal=Appl%20Math%20Lett&amp;volume=7&amp;pages=19-23&amp;publication_year=1994&amp;author=Steel%2CMA" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR85">Taylor
 DJ, Piel WH: An assessment of accuracy, error, and conflict with 
support values from genome-scale phylogenetic data. Mol Biol Evol. 2004,
 21: 1534-1537. 10.1093/molbev/msh156.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fmolbev%2Fmsh156" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXmtlKgt7w%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15140947" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20assessment%20of%20accuracy%2C%20error%2C%20and%20conflict%20with%20support%20values%20from%20genome-scale%20phylogenetic%20data&amp;journal=Mol%20Biol%20Evol&amp;volume=21&amp;pages=1534-1537&amp;publication_year=2004&amp;author=Taylor%2CDJ&amp;author=Piel%2CWH" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR86">Ren
 F, Tanaka H, Yang Z: An empirical examination of the utility of 
codon-substitution models in phylogeny reconstruction. Syst Biol. 2005, 
54: 808-818. 10.1080/10635150500354688.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1080%2F10635150500354688" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16243764" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20empirical%20examination%20of%20the%20utility%20of%20codon-substitution%20models%20in%20phylogeny%20reconstruction&amp;journal=Syst%20Biol&amp;volume=54&amp;pages=808-818&amp;publication_year=2005&amp;author=Ren%2CF&amp;author=Tanaka%2CH&amp;author=Yang%2CZ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR87">Burleigh
 JG, Driskell AC, Sanderson MJ: Supertree bootstrapping methods for 
assessing phylogenetic variation among genes in genome-scale data sets. 
Syst Biol. 2006, 55: 426-440. 10.1080/10635150500541722.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1080%2F10635150500541722" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16861207" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Supertree%20bootstrapping%20methods%20for%20assessing%20phylogenetic%20variation%20among%20genes%20in%20genome-scale%20data%20sets&amp;journal=Syst%20Biol&amp;volume=55&amp;pages=426-440&amp;publication_year=2006&amp;author=Burleigh%2CJG&amp;author=Driskell%2CAC&amp;author=Sanderson%2CMJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR88">Ané
 C, Larget B, Baum DA, Smith SD, Rokas A: Bayesian estimation of 
concordance among gene trees. Mol Biol Evol. 2007, 24: 412-426. 
10.1093/molbev/msl170.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fmolbev%2Fmsl170" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17095535" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Bayesian%20estimation%20of%20concordance%20among%20gene%20trees&amp;journal=Mol%20Biol%20Evol&amp;volume=24&amp;pages=412-426&amp;publication_year=2007&amp;author=An%C3%A9%2CC&amp;author=Larget%2CB&amp;author=Baum%2CDA&amp;author=Smith%2CSD&amp;author=Rokas%2CA" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR89">Criscuolo
 A, Michel CJ: Phylogenetic inference with weighted codon evolutionary 
distances. J Mol Evol. 2009, 68: 377-392. 10.1007/s00239-009-9212-y.</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1007%2Fs00239-009-9212-y" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="https://bmcevolbiol.biomedcentral.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1MXksVCgu7k%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19308635" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenetic%20inference%20with%20weighted%20codon%20evolutionary%20distances&amp;journal=J%20Mol%20Evol&amp;volume=68&amp;pages=377-392&amp;publication_year=2009&amp;author=Criscuolo%2CA&amp;author=Michel%2CCJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR90">Néron
 B, Ménager H, Maufrais C, Joly N, Maupetit J, Letort S, Carrere S, 
Tuffery P, Letondal C: Mobyle: a new full web bioinformatics framework. 
Bioinformatics. 2009, 25: 3005-3011. 10.1093/bioinformatics/btp493.</p><ul class="c-reading-companion__reference-links"><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2773253" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp493" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19689959" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Mobyle%3A%20a%20new%20full%20web%20bioinformatics%20framework&amp;journal=Bioinformatics&amp;volume=25&amp;pages=3005-3011&amp;publication_year=2009&amp;author=N%C3%A9ron%2CB&amp;author=M%C3%A9nager%2CH&amp;author=Maufrais%2CC&amp;author=Joly%2CN&amp;author=Maupetit%2CJ&amp;author=Letort%2CS&amp;author=Carrere%2CS&amp;author=Tuffery%2CP&amp;author=Letondal%2CC" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li></ol></div></div>
                            </div>
                        </div>
                    </aside>
                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">BMC Evolutionary Biology</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1471-2148</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="https://www.editorialmanager.com/evob" target="_blank">Access here and click Contact Us</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="1471-2148-10-210.png" alt="">

         
    <footer>
        
            <div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="u-flex u-flex-wrap u-flex-justify-space-between" data-test="publisher-footer-menu">
            <div class="u-flex">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://support.biomedcentral.com/support/home">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
                    
                
            </div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="u-flex u-list-reset" data-test="footer-social-links">
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-twitter-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-facebook-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-weibo-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="https://www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="https://www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="https://www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        
        <div class="c-corporate-footer">
    <div class="u-container">
        <img src="logo-springernature-8bd4e8d82f.svg" class="c-corporate-footer__logo" alt="Springer Nature" itemprop="logo" role="img">
        <p class="c-corporate-footer__legal" data-test="copyright"> © 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a class="c-corporate-footer__link" href="https://www.springernature.com/" itemscope="" itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

        
    </footer>

    </div>

    
    <script type="text/javascript" src="jquery-2.js"></script>
    <script>
        window.Component = {};
    </script>

    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

    <script>
    var appendExtraFlagsToLinks = function(params) {
        $('a').filter(function(_, x) {
            var href = $(x).attr('href');
            return href && href.startsWith('/');
        }).each(function(_, x) {
            var $x = $(x);
            var url = new URL('http://temp.com' + $x.attr('href'));
            params.forEach(function(param) {
                url.searchParams.set(param[0], param[1]);
            });
            $x.attr('href', url.pathname + url.search + url.hash);
        });
    };

    if (window.URLSearchParams) {
        var params = new URLSearchParams(location.search.slice(1));
        var flag = 'shared-article-renderer';
        if (params.has('shared-article-renderer')) {
            appendExtraFlagsToLinks([[flag, true]]);
        }
    }
</script>


        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            
            appScript.src = '/static/js/app-bundle-7470bff19c.js';
            
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script><script src="app-bundle-7470bff19c.js"></script>




<script src="global-article-bundle-d8d941b1d0.js"></script>

    




<script type="text/javascript" id="polyfill-matches">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="gpt-control" src="gpt.js"></script><script type="text/javascript" id="krux-control-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){var a=document.createElement("script");a.type="text/javascript";a.async=!0;a.src=("https:"===location.protocol?"https:":"http:")+"//cdn.krxd.net/controltag/KDqylSLE.js";var b=document.getElementsByTagName("script")[0];b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="krux-interchange-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){function c(a){a="kxmacmillan_"+a;try{var b=window.localStorage}catch(e){b=null}return b?b[a]||"":navigator.cookieEnabled?(b=document.cookie.match(a+"\x3d([^;]*)"))&&unescape(b[1])||"":""}Krux.user=c("user");Krux.segments=c("segs")?c("segs").split(","):[];var a=[];Krux.user&&a.push("kuid\x3d"+Krux.user);for(var d=0;d<Krux.segments.length;d++)a.push("ksg\x3d"+Krux.segments[d]);Krux.dfppKeyValues=a.length?a.join(";")+";":""})();</script><script type="text/javascript" id="krux-consent">var allowed=google_tag_manager["GTM-TDGJHK"].macro(27);allowed=!0===allowed?1:0;Krux("consent:set",{dc:allowed,al:allowed,tg:allowed,cd:!1,sh:!1,re:!1},function(a,b){});</script><script type="text/javascript" id="gpt-retrieve-ads">function splitKeys(a){var f=[],g="",h=[],k=a.split(";");for(a=0;a<k.length;++a){void 0!==l&&(g=l);var d=k[a].split("\x3d");var l=d[0];l!==g&&(0<g.length&&0<f.length&&h.push([g,f]),f=[]);if(2===d.length&&""!==d[0]&&""!==d[1]){var m=d[1].split(",");for(d=0;d<m.length;++d)f.push(m[d])}}0<l.length&&0<f.length&&h.push([l,f]);return h}
function splitSizes(a){var f=[];if(null!==a){var g=0<=a.indexOf("|")?a.split("|"):a.split(",");for(a=0;a<g.length;++a){var h=g[a].split("x");var k=parseInt(h[0],10);var d=parseInt(h[1],10);2===h.length&&!isNaN(k)&&!isNaN(d)&&0<=k&&0<=d&&f.push([k,d])}}return f}function debounce(a,f){var g=null,h=null;return function(){var k=this,d=Number(new Date),l=arguments;g&&d<g+f?(clearTimeout(h),h=setTimeout(function(){g=d;a.apply(k,l)},f)):(g=d,a.apply(k,l))}}
function addScrollEvent(a){window.addEventListener?window.addEventListener("scroll",a,!1):window.attachEvent("onscroll",a)}function removeScrollEvent(a){window.removeEventListener?window.removeEventListener("scroll",a,!1):window.detachEvent("scroll",a)}function getAdContainers(){return document.querySelectorAll("div[data-gpt-unitpath]")}function isLoggedIn(){return 0<=document.cookie.indexOf("OSCAR_SESSION_COOKIE")?"logged\x3dy;":"logged\x3dn;"}
(function(a,f){function g(e){var b="test";b=b.replace(/[\[\]]/g,"\\$\x26");b=new RegExp("[?\x26]"+b+"(\x3d([^\x26#]*)|\x26|#|$)");b=(b=b.exec(window.location.href))?b[2]?decodeURIComponent(b[2].replace(/\+/g," ")):"":null;b=b&&"ads"===b?"adtype\x3dtest;":"";e=e.getAttribute("data-gpt-targeting");var a=document.querySelector("[data-ad-targeting-search-terms]")?"search\x3d"+document.querySelector("[data-ad-targeting-search-terms]").innerText+";":!1;var c=isLoggedIn();b&&-1===e.indexOf(b)&&(e+=b);a&&
(e+=a);return e+c+q}function h(e,b){for(var a=e.length,c=[];a--;)b(e[a],a)&&(c.push(e[a].slot),e.splice(a,1));c.length&&googletag.pubads().refresh(c);return e}function k(e){var b=Math.max(document.documentElement.clientHeight,a.innerHeight||0);return h(e,function(a){a=document.getElementById(a.divId);a=a.getBoundingClientRect();a=a.top-300;return b>a})}function d(){googletag.cmd.push(function(){for(var a=0;c[a];++a)googletag.display(c[a].divId)});googletag.cmd.push(function(){c=k(c)});var a=debounce(function(){googletag.cmd.push(function(){c=
k(c);c.length||removeScrollEvent(a)})},250);addScrollEvent(a)}function l(a){var e=document.createElement("script");e.onload=function(){a(window.PubGrade)};e.async=!0;e.src="https://cdn.pbgrd.com/core-bmc.js";document.head.appendChild(e)}function m(a){!a||a&&a.fired?setTimeout(d,1):document.addEventListener("pbgrdFinished",d)}var q,r=getAdContainers(),c=[];f&&(q=f.dfppKeyValues?f.dfppKeyValues:"");a.googletag=a.googletag||{};a.googletag.cmd=a.googletag.cmd||[];for(var p=0;r[p];++p){var n=r[p];splitKeys(g(n));
c.push({divId:n.getAttribute("id"),adUnitPath:n.getAttribute("data-gpt-unitpath"),sizeArray:splitSizes(n.getAttribute("data-gpt-sizes")),targeting:splitKeys(g(n))})}googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["GTM-TDGJHK"].macro(31)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices()});googletag.cmd.push(function(){for(var a=0;c[a];++a)try{c[a].slot=googletag.defineSlot(c[a].adUnitPath,c[a].sizeArray,c[a].divId).addService(googletag.pubads());for(var b=
0,d=c[a].targeting.length;b<d;++b)2===c[a].targeting[b].length&&""!==c[a].targeting[b][0]&&""!==c[a].targeting[b][1]&&c[a].slot.setTargeting(c[a].targeting[b][0],c[a].targeting[b][1])}catch(t){console.log("failed to create slot for",c[a])}});-1!==window.location.href.indexOf("/articles/")?l(m):m(null)})(window,window.Krux);</script><script type="text/javascript" id="ga-data-track-event-listeners">var setupHandler=google_tag_manager["GTM-TDGJHK"].macro(55);setupHandler("click");setupHandler("submit");</script><script type="text/javascript" id="crossmark-script" src="widget.js"></script><script type="text/javascript" id="recommended-script" src="entry-point"></script><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; min-width: 0px; max-width: none; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size4,sans-serif;"></div></div><iframe style="width: 0px; height: 0px; bottom: 0px; right: 0px; z-index: 100000; position: fixed;" scrolling="no" src="https://recommended.springernature.com/v2308/generated/client.html?source=https://bmcevolbiol.biomedcentral.com" name="uptodate-client" id="uptodate-client" title="Personalised recommendations" allowtransparency="true" frameborder="0"></iframe><img class="pub_300x250 pub_300x250m pub_728x90 text-ad textAd text_ad text_ads text-ads text-ad-links" style="width: 1px !important; height: 1px !important; position: absolute !important; left: -10000px !important; top: -1000px !important;"><div class="kxhead" data-id="KDqylSLE" style="display:none !important;"></div></body><iframe id="kx-proxy-KDqylSLE" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=KDqylSLE&amp;kxt=https%3A%2F%2Fbmcevolbiol.biomedcentral.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>