<!DOCTYPE html>
<html lang="en"><head><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.json"></script><script type="text/javascript" src="_ate.config_resp"></script><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace">
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    <!-- Meta -->
    <meta charset="utf-8"><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><script async="" src="analytics.js"></script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":1300,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="https://academic.oup.com/UI/app/img/apple-touch-icon.png"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-32x32.png" sizes="32x32"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-16x16.png" sizes="16x16"><link rel="manifest" href="https://academic.oup.com/UI/app/img/manifest.json"><link rel="mask-icon" href="https://academic.oup.com/UI/app/img/safari-pinned-tab.svg" color="#5bbad5"><link rel="shortcut icon" href="https://academic.oup.com/UI/app/img/favicon.ico"><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml"><meta name="theme-color" content="#002f65">

    <!-- Stylesheets -->
    
<link href="css.css" type="text/css" rel="stylesheet">
<link href="globalCSS.css" type="text/css" rel="stylesheet">
<link href="cookiePolicyCSS.css" type="text/css" rel="stylesheet">
<link href="chartist.css" type="text/css" rel="stylesheet">

        <link href="global.css" rel="stylesheet" type="text/css">
    
        <link href="Site-304756074.css" rel="stylesheet" type="text/css">
    

    <!-- Scripts -->
    
<script src="jquery-2.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [0] 
        };

        App.CurrentSubdomain = 'mbe';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/mbe/article/30/4/772/1073398","@type":"ScholarlyArticle","name":"MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability","datePublished":"2013-01-16","isPartOf":{"@id":"https://academic.oup.com/mbe/issue/30/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2013-04-01","isPartOf":{"@id":"https://academic.oup.com/mbe","@type":"Periodical","name":"Molecular Biology and Evolution","issn":["1537-1719"]}},"url":"http://dx.doi.org/10.1093/molbev/mst010","keywords":["multiple sequence alignment","metagemone","protein structure","progressive alignment","parallel processing"],"inLanguage":"en","copyrightHolder":"Society for Molecular Biology and Evolution","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/mbe/article/30/4/772/1073398","author":[{"name":"Katoh, Kazutaka","affiliation":"Immunology Frontier Research Center, Osaka University, Suita, Osaka, Japan  Computational Biology Research Center, The National Institute of Advanced Industrial Science and Technology (AIST), Tokyo, Japan","@type":"Person"},{"name":"Standley, Daron M.","affiliation":"Immunology Frontier Research Center, Osaka University, Suita, Osaka, Japan","@type":"Person"}],"description":"Abstract.  We report a major update of the MAFFT multiple sequence alignment program. This version has several new features, including options for adding unalig","pageStart":"772","pageEnd":"780","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/30/4/0/m_mbe30_4.cover.gif?Expires=2147483647&Signature=IrB4CIhfIZrNmoiMy0TpeYYw7fXHmxR9m3s1febMkpFig15Zmgt~mVU2-Aje7iFubZdM3rirD1AyU3wmMayudigxhHCZtFkuGeZvDc0mCQetw7nVNDQ-IUJBsfcOQwt4TkBL8rGfnQ2XjvL~evAvwS1OOMVJyeMcFIqDnqwKoac~y20EfN9Qf2d83nS4plrtbb2tx0u1Sn6W5t9mBuWy1bL5rcFDoPjCdURwi6I7o5C9E~QAZNzuglRKKcfCD4zJinRM7A5cnkcdq42nHk9pk2RfvlH2kRkiPg6ojmV4296gfUFjyDApZSNFLTmztyiFdLSy8RSwb0WFknFM1ZgQ5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/30/4/0/m_mbe30_4.cover.gif?Expires=2147483647&Signature=IrB4CIhfIZrNmoiMy0TpeYYw7fXHmxR9m3s1febMkpFig15Zmgt~mVU2-Aje7iFubZdM3rirD1AyU3wmMayudigxhHCZtFkuGeZvDc0mCQetw7nVNDQ-IUJBsfcOQwt4TkBL8rGfnQ2XjvL~evAvwS1OOMVJyeMcFIqDnqwKoac~y20EfN9Qf2d83nS4plrtbb2tx0u1Sn6W5t9mBuWy1bL5rcFDoPjCdURwi6I7o5C9E~QAZNzuglRKKcfCD4zJinRM7A5cnkcdq42nHk9pk2RfvlH2kRkiPg6ojmV4296gfUFjyDApZSNFLTmztyiFdLSy8RSwb0WFknFM1ZgQ5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article">
    <meta property="og:title" content="MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability | Molecular Biology and Evolution | Oxford Academic">
    <meta property="og:description" content="Abstract.  We report a major update of the MAFFT multiple sequence alignment program. This version has several new features, including options for adding unalig">
    <meta property="og:url" content="https://academic.oup.com/mbe/article/30/4/772/1073398">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/35/10/1/m_molbiolevol_35_10cover.png?Expires=2147483647&amp;Signature=ovhLH5Jp5vurDX37AM2aOxITo5poFVa~zb57nrpwFmttgcVe93Exxjj4ZF~tt2AnUF~vhgA4tuIjLHodp6u7CRcYU9DiYvuX8yaNNDfH3NKXJcek1TR3at1pkNptLqnJlJvurJmnktOKbWUqhOokkanvi7Qy-1buL-NzhVcvx~dURkFVqsw97R9fZw~SzhsYh1vRGgsj5Tv330CV9N1so2zjwsJPE2X~8UUPLVMTnpJ85Ht1JJLXIsmoqaLQm9fSw5T3DqEtf~QbgLD2tO0Zf76-XnQuAfD4aSMWSUeb6YCkQzJjtpOwi-gOkq~lpWRUStdsbX~6XJN0HvJfpyGRbA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2013-01-16">
    <meta property="og:site_name" content="OUP Academic">
<meta name="citation_author" content="Katoh, Kazutaka"><meta name="citation_author_institution" content="Immunology Frontier Research Center, Osaka University, Suita, Osaka, Japan"><meta name="citation_author_institution" content="Computational Biology Research Center, The National Institute of Advanced Industrial Science and Technology (AIST), Tokyo, Japan"><meta name="citation_author" content="Standley, Daron M."><meta name="citation_author_institution" content="Immunology Frontier Research Center, Osaka University, Suita, Osaka, Japan"><meta name="citation_title" content="MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability"><meta name="citation_firstpage" content="772"><meta name="citation_lastpage" content="780"><meta name="citation_doi" content="10.1093/molbev/mst010"><meta name="citation_journal_title" content="Molecular Biology and Evolution"><meta name="citation_journal_abbrev" content="Mol Biol Evol"><meta name="citation_volume" content="30"><meta name="citation_issue" content="4"><meta name="citation_publication_date" content="2013/04/01"><meta name="citation_issn" content="0737-4038"><meta name="citation_publisher" content="Oxford University Press"><meta name="citation_reference" content="citation_title=Generalized affine gap costs for protein sequence alignment; Proteins;  citation_year=1998;  citation_volume=32;  citation_pages=88-96; "><meta name="citation_reference" content="citation_title=A strategy for the rapid multiple alignment of protein sequences. Confidence levels from tertiary structure comparisons; J Mol Biol.;  citation_year=1987;  citation_volume=198;  citation_pages=327-337; "><meta name="citation_reference" content="citation_title=A novel randomized iterative strategy for aligning multiple protein sequences; Comput Appl Biosci.;  citation_year=1991;  citation_volume=7;  citation_pages=479-484; "><meta name="citation_reference" content="citation_title=Aligning short reads to reference alignments and trees; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2068-2075; "><meta name="citation_reference" content="citation_title=Class of multiple sequence alignment algorithm affects genomic analysis; citation_text=Advance access published December 4, 2012, doi:10.1093/molbev/mss256; Mol Biol Evol.;  citation_year=2012; "><meta name="citation_reference" content="citation_title=Measuring the distance between multiple sequence alignments; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=495-502; "><meta name="citation_reference" content="citation_title=The comparative RNA web (CRW) site: an online database of comparative sequence and structure information for ribosomal, intron, and other RNAs; BMC Bioinformatics;  citation_year=2002;  citation_volume=3;  citation_pages=2"><meta name="citation_reference" content="citation_title=The ribosomal database project: improved alignments and new tools for rRNA analysis; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D141-D145; "><meta name="citation_reference" content="citation_title=Phylogenetic assessment of alignments reveals neglected tree signal in gaps; Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=R37"><meta name="citation_reference" content="citation_title=Progressive sequence alignment as a prerequisite to correct phylogenetic trees; J Mol Evol.;  citation_year=1987;  citation_volume=25;  citation_pages=351-360; "><meta name="citation_reference" content="citation_title=INDELible: a flexible simulator of biological sequence evolution; Mol Biol Evol.;  citation_year=2009;  citation_volume=26;  citation_pages=1879-1888; "><meta name="citation_reference" content="citation_title=Mind the gaps: evidence of bias in estimates of multiple sequence alignments; Mol Biol Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=2433-2442; "><meta name="citation_reference" content="citation_title=Optimal alignment between groups of sequences and its application to multiple sequence alignment; Comput Appl Biosci.;  citation_year=1993;  citation_volume=9;  citation_pages=361-370; "><meta name="citation_reference" content="citation_title=A weighting system and algorithm for aligning many phylogenetically related sequences; Comput Appl Biosci.;  citation_year=1995;  citation_volume=11;  citation_pages=543-551; "><meta name="citation_reference" content="citation_title=Amino acid substitution matrices from protein blocks; Proc Natl Acad Sci U S A.;  citation_year=1992;  citation_volume=89;  citation_pages=10915-10919; "><meta name="citation_reference" content="citation_title=CLUSTAL: a package for performing multiple sequence alignment on a microcomputer; Gene;  citation_year=1988;  citation_volume=73;  citation_pages=237-244; "><meta name="citation_reference" content="citation_title=The rapid generation of mutation data matrices from protein sequences; Comput Appl Biosci.;  citation_year=1992;  citation_volume=8;  citation_pages=275-282; "><meta name="citation_reference" content="citation_title=A mutation data matrix for transmembrane proteins; FEBS Lett.;  citation_year=1994;  citation_volume=339;  citation_pages=269-275; "><meta name="citation_reference" content="citation_title=Multiple alignment of DNA sequences with MAFFT; Methods Mol Biol.;  citation_year=2009;  citation_volume=537;  citation_pages=39-64; "><meta name="citation_reference" content="citation_title=Adding unaligned sequences into an existing alignment using MAFFT and LAST; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=3144-3146; "><meta name="citation_reference" content="citation_title=MAFFT version 5: improvement in accuracy of multiple sequence alignment; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=511-518; "><meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=3059-3066; "><meta name="citation_reference" content="citation_title=MAFFT: iterative refinement and additional methods; Methods Mol Biol.;  citation_year=2013; "><meta name="citation_reference" content="citation_title=PartTree: An algorithm to build an approximate tree from a large number of unaligned sequences; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=372-374; "><meta name="citation_reference" content="citation_title=Improved accuracy of multiple ncRNA alignment by incorporating structural information into a MAFFT-based framework; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=212"><meta name="citation_reference" content="citation_title=Recent developments in the MAFFT multiple sequence alignment program; Brief Bioinform.;  citation_year=2008;  citation_volume=9;  citation_pages=286-298; "><meta name="citation_reference" content="citation_title=Parallelization of the MAFFT multiple sequence alignment program; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=1899-1900; "><meta name="citation_reference" content="citation_title=Adaptive seeds tame genomic sequence comparison; Genome Res.;  citation_year=2011;  citation_volume=21;  citation_pages=487-493; "><meta name="citation_reference" content="citation_title=The impact of rRNA secondary structure consideration in alignment and tree reconstruction: simulated data and a case study on the phylogeny of hexapods; Mol Biol Evol.;  citation_year=2010;  citation_volume=27;  citation_pages=2507-2521; "><meta name="citation_reference" content="citation_title=Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1684-1691; "><meta name="citation_reference" content="citation_title=pplacer: linear time maximum-likelihood and Bayesian phylogenetic placement of sequences onto a fixed reference tree; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=538"><meta name="citation_reference" content="citation_title=SEPP: SATé-enabled phylogenetic placement; Pac Symp Biocomput.;  citation_year=2012;  citation_volume=17;  citation_pages=247-258; "><meta name="citation_reference" content="citation_title=An open source software package for automated extraction of ITS1 and ITS2 from fungal ITS sequences for use in high-throughput community assays and molecular ecology; Fungal Ecology;  citation_year=2010;  citation_volume=3;  citation_pages=284-287; "><meta name="citation_reference" content="citation_title=COFFEE: an objective function for multiple sequence alignments; Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=407-422; "><meta name="citation_reference" content="citation_title=The accuracy of several multiple sequence alignment programs for proteins; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=471"><meta name="citation_reference" content="citation_title=3DCoffee: combining protein sequences and structures within multiple sequence alignments; J Mol Biol.;  citation_year=2004;  citation_volume=340;  citation_pages=385-395; "><meta name="citation_reference" content="citation_title=PROMALS3D: a tool for multiple protein sequence and structure alignments; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=2295-2300; "><meta name="citation_reference" content="citation_title=The Pfam protein families database; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D290-D301; "><meta name="citation_reference" content="citation_title=PicXAA-R: efficient structural alignment of multiple RNA sequences using a greedy approach; BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=S38"><meta name="citation_reference" content="citation_title=The PyMOL Molecular Graphics System, Version 1.3r1. Portland; citation_text=Oregon: Schrödinger, LLC;  citation_year=2010; "><meta name="citation_reference" content="citation_title=Fast scalable generation of high-quality protein multiple sequence alignments using Clustal Omega; Mol Syst Biol.;  citation_year=2011;  citation_volume=7;  citation_pages=539"><meta name="citation_reference" content="citation_title=PROSITE, a protein domain database for functional characterization and annotation; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=D161-D166; "><meta name="citation_reference" content="citation_title=Ash structure alignment package: sensitivity and selectivity in domain classification; BMC Bioinformatics;  citation_year=2007;  citation_volume=8;  citation_pages=116"><meta name="citation_reference" content="citation_title=Detecting local structural similarity in proteins by maximizing number of equivalent residues; Proteins;  citation_year=2004;  citation_volume=57;  citation_pages=381-391; "><meta name="citation_reference" content="citation_title=PhyLAT: a phylogenetic local alignment tool; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1336-1344; "><meta name="citation_reference" content="citation_title=A fast structural multiple alignment method for long RNA sequences; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=33"><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-4680; "><meta name="citation_reference" content="citation_title=Jalview version 2—a multiple sequence alignment editor and analysis workbench; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1189-1191; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/mbe/article-pdf/30/4/772/6420419/mst010.pdf"><meta name="description" content="Abstract.  We report a major update of the MAFFT multiple sequence alignment program. This version has several new features, including options for adding unalig">
        <meta name="citation_xml_url" content="https://academic.oup.com/mbe/article-xml/30/4/772/1073398">



    <link rel="canonical" href="https://academic.oup.com/mbe/article/30/4/772/1073398">


        <!-- Start: GPT Async -->

<script async="async" src="gpt.js"></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type="text/javascript">
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "mbe/article/30/4/772/1073398");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=molbev&authzrequired=false&account_id=&doi=10.1093/molbev/mst010';
    </script>
<title>
	MAFFT Multiple Sequence Alignment Software Version 7: Improvements in 
Performance and Usability | Molecular Biology and Evolution | Oxford 
Academic
</title><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_265.js" async=""></script><script id="altmetric-embed-js" src="altmetric_badges-6b74dfff5c7f701b334f3e44e411f88f.js"></script><style></style><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_google_plusone,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.at-share-tbx-element .google_plusone_iframe_widget>span>div{vertical-align:top!important}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_google_plusone.wsb-social-share-button{padding-top:0}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-2c47105b6381604898bbf8ae8a680350.css"></head>

<body data-sitename="molecularbiologyandevolution" class="off-canvas pg_article" theme-mbe="">
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 100});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1073398" id="webform">
<div class="aspNetHidden">
<input name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZVhPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXRvaCwgS2F6dXRha2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbW11bm9sb2d5IEZyb250aWVyIFJlc2VhcmNoIENlbnRlciwgT3Nha2EgVW5pdmVyc2l0eSwgU3VpdGEsIE9zYWthLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNvbXB1dGF0aW9uYWwgQmlvbG9neSBSZXNlYXJjaCBDZW50ZXIsIFRoZSBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWR2YW5jZWQgSW5kdXN0cmlhbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5IChBSVNUKSwgVG9reW8sIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3RhbmRsZXksIERhcm9uIE0uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW1tdW5vbG9neSBGcm9udGllciBSZXNlYXJjaCBDZW50ZXIsIE9zYWthIFVuaXZlcnNpdHksIFN1aXRhLCBPc2FrYSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik1BRkZUIE11bHRpcGxlIFNlcXVlbmNlIEFsaWdubWVudCBTb2Z0d2FyZSBWZXJzaW9uIDc6IEltcHJvdmVtZW50cyBpbiBQZXJmb3JtYW5jZSBhbmQgVXNhYmlsaXR5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNzcyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI3ODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL21vbGJldi9tc3QwMTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iTW9sZWN1bGFyIEJpb2xvZ3kgYW5kIEV2b2x1dGlvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iTW9sIEJpb2wgRXZvbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTMvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMDczNy00MDM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcmFsaXplZCBhZmZpbmUgZ2FwIGNvc3RzIGZvciBwcm90ZWluIHNlcXVlbmNlIGFsaWdubWVudDsgUHJvdGVpbnM7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz04OC05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdHJhdGVneSBmb3IgdGhlIHJhcGlkIG11bHRpcGxlIGFsaWdubWVudCBvZiBwcm90ZWluIHNlcXVlbmNlcy4gQ29uZmlkZW5jZSBsZXZlbHMgZnJvbSB0ZXJ0aWFyeSBzdHJ1Y3R1cmUgY29tcGFyaXNvbnM7IEogTW9sIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE5ODsgIGNpdGF0aW9uX3BhZ2VzPTMyNy0zMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgcmFuZG9taXplZCBpdGVyYXRpdmUgc3RyYXRlZ3kgZm9yIGFsaWduaW5nIG11bHRpcGxlIHByb3RlaW4gc2VxdWVuY2VzOyBDb21wdXQgQXBwbCBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00NzktNDg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbGlnbmluZyBzaG9ydCByZWFkcyB0byByZWZlcmVuY2UgYWxpZ25tZW50cyBhbmQgdHJlZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjA2OC0yMDc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbGFzcyBvZiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgYWxnb3JpdGhtIGFmZmVjdHMgZ2Vub21pYyBhbmFseXNpczsgY2l0YXRpb25fdGV4dD1BZHZhbmNlIGFjY2VzcyBwdWJsaXNoZWQgRGVjZW1iZXIgNCwgMjAxMiwgZG9pOjEwLjEwOTMvbW9sYmV2L21zczI1NjsgTW9sIEJpb2wgRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lYXN1cmluZyB0aGUgZGlzdGFuY2UgYmV0d2VlbiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnRzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTQ5NS01MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wYXJhdGl2ZSBSTkEgd2ViIChDUlcpIHNpdGU6IGFuIG9ubGluZSBkYXRhYmFzZSBvZiBjb21wYXJhdGl2ZSBzZXF1ZW5jZSBhbmQgc3RydWN0dXJlIGluZm9ybWF0aW9uIGZvciByaWJvc29tYWwsIGludHJvbiwgYW5kIG90aGVyIFJOQXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByaWJvc29tYWwgZGF0YWJhc2UgcHJvamVjdDogaW1wcm92ZWQgYWxpZ25tZW50cyBhbmQgbmV3IHRvb2xzIGZvciByUk5BIGFuYWx5c2lzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1EMTQxLUQxNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyBhc3Nlc3NtZW50IG9mIGFsaWdubWVudHMgcmV2ZWFscyBuZWdsZWN0ZWQgdHJlZSBzaWduYWwgaW4gZ2FwczsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9UjM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZ3Jlc3NpdmUgc2VxdWVuY2UgYWxpZ25tZW50IGFzIGEgcHJlcmVxdWlzaXRlIHRvIGNvcnJlY3QgcGh5bG9nZW5ldGljIHRyZWVzOyBKIE1vbCBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTM1MS0zNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlOREVMaWJsZTogYSBmbGV4aWJsZSBzaW11bGF0b3Igb2YgYmlvbG9naWNhbCBzZXF1ZW5jZSBldm9sdXRpb247IE1vbCBCaW9sIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTg3OS0xODg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaW5kIHRoZSBnYXBzOiBldmlkZW5jZSBvZiBiaWFzIGluIGVzdGltYXRlcyBvZiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnRzOyBNb2wgQmlvbCBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI0MzMtMjQ0MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3B0aW1hbCBhbGlnbm1lbnQgYmV0d2VlbiBncm91cHMgb2Ygc2VxdWVuY2VzIGFuZCBpdHMgYXBwbGljYXRpb24gdG8gbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50OyBDb21wdXQgQXBwbCBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNjEtMzcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHdlaWdodGluZyBzeXN0ZW0gYW5kIGFsZ29yaXRobSBmb3IgYWxpZ25pbmcgbWFueSBwaHlsb2dlbmV0aWNhbGx5IHJlbGF0ZWQgc2VxdWVuY2VzOyBDb21wdXQgQXBwbCBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTQzLTU1MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1pbm8gYWNpZCBzdWJzdGl0dXRpb24gbWF0cmljZXMgZnJvbSBwcm90ZWluIGJsb2NrczsgUHJvYyBOYXRsIEFjYWQgU2NpIFUgUyBBLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTEwOTE1LTEwOTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DTFVTVEFMOiBhIHBhY2thZ2UgZm9yIHBlcmZvcm1pbmcgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IG9uIGEgbWljcm9jb21wdXRlcjsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT03MzsgIGNpdGF0aW9uX3BhZ2VzPTIzNy0yNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByYXBpZCBnZW5lcmF0aW9uIG9mIG11dGF0aW9uIGRhdGEgbWF0cmljZXMgZnJvbSBwcm90ZWluIHNlcXVlbmNlczsgQ29tcHV0IEFwcGwgQmlvc2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9Mjc1LTI4MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtdXRhdGlvbiBkYXRhIG1hdHJpeCBmb3IgdHJhbnNtZW1icmFuZSBwcm90ZWluczsgRkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0zMzk7ICBjaXRhdGlvbl9wYWdlcz0yNjktMjc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSBhbGlnbm1lbnQgb2YgRE5BIHNlcXVlbmNlcyB3aXRoIE1BRkZUOyBNZXRob2RzIE1vbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01Mzc7ICBjaXRhdGlvbl9wYWdlcz0zOS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWRkaW5nIHVuYWxpZ25lZCBzZXF1ZW5jZXMgaW50byBhbiBleGlzdGluZyBhbGlnbm1lbnQgdXNpbmcgTUFGRlQgYW5kIExBU1Q7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MzE0NC0zMTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQUZGVCB2ZXJzaW9uIDU6IGltcHJvdmVtZW50IGluIGFjY3VyYWN5IG9mIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NTExLTUxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUFGRlQ6IGEgbm92ZWwgbWV0aG9kIGZvciByYXBpZCBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgYmFzZWQgb24gZmFzdCBGb3VyaWVyIHRyYW5zZm9ybTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MzA1OS0zMDY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQUZGVDogaXRlcmF0aXZlIHJlZmluZW1lbnQgYW5kIGFkZGl0aW9uYWwgbWV0aG9kczsgTWV0aG9kcyBNb2wgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhcnRUcmVlOiBBbiBhbGdvcml0aG0gdG8gYnVpbGQgYW4gYXBwcm94aW1hdGUgdHJlZSBmcm9tIGEgbGFyZ2UgbnVtYmVyIG9mIHVuYWxpZ25lZCBzZXF1ZW5jZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MzcyLTM3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZWQgYWNjdXJhY3kgb2YgbXVsdGlwbGUgbmNSTkEgYWxpZ25tZW50IGJ5IGluY29ycG9yYXRpbmcgc3RydWN0dXJhbCBpbmZvcm1hdGlvbiBpbnRvIGEgTUFGRlQtYmFzZWQgZnJhbWV3b3JrOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTIxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY2VudCBkZXZlbG9wbWVudHMgaW4gdGhlIE1BRkZUIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBwcm9ncmFtOyBCcmllZiBCaW9pbmZvcm0uOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yODYtMjk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJhbGxlbGl6YXRpb24gb2YgdGhlIE1BRkZUIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBwcm9ncmFtOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTE4OTktMTkwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWRhcHRpdmUgc2VlZHMgdGFtZSBnZW5vbWljIHNlcXVlbmNlIGNvbXBhcmlzb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NDg3LTQ5MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGltcGFjdCBvZiByUk5BIHNlY29uZGFyeSBzdHJ1Y3R1cmUgY29uc2lkZXJhdGlvbiBpbiBhbGlnbm1lbnQgYW5kIHRyZWUgcmVjb25zdHJ1Y3Rpb246IHNpbXVsYXRlZCBkYXRhIGFuZCBhIGNhc2Ugc3R1ZHkgb24gdGhlIHBoeWxvZ2VueSBvZiBoZXhhcG9kczsgTW9sIEJpb2wgRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yNTA3LTI1MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY3VyYXRlIGV4dGVuc2lvbiBvZiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnRzIHVzaW5nIGEgcGh5bG9nZW55LWF3YXJlIGdyYXBoIGFsZ29yaXRobTsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xNjg0LTE2OTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXBwbGFjZXI6IGxpbmVhciB0aW1lIG1heGltdW0tbGlrZWxpaG9vZCBhbmQgQmF5ZXNpYW4gcGh5bG9nZW5ldGljIHBsYWNlbWVudCBvZiBzZXF1ZW5jZXMgb250byBhIGZpeGVkIHJlZmVyZW5jZSB0cmVlOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz01MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TRVBQOiBTQVTDqS1lbmFibGVkIHBoeWxvZ2VuZXRpYyBwbGFjZW1lbnQ7IFBhYyBTeW1wIEJpb2NvbXB1dC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0yNDctMjU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBvcGVuIHNvdXJjZSBzb2Z0d2FyZSBwYWNrYWdlIGZvciBhdXRvbWF0ZWQgZXh0cmFjdGlvbiBvZiBJVFMxIGFuZCBJVFMyIGZyb20gZnVuZ2FsIElUUyBzZXF1ZW5jZXMgZm9yIHVzZSBpbiBoaWdoLXRocm91Z2hwdXQgY29tbXVuaXR5IGFzc2F5cyBhbmQgbW9sZWN1bGFyIGVjb2xvZ3k7IEZ1bmdhbCBFY29sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0yODQtMjg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DT0ZGRUU6IGFuIG9iamVjdGl2ZSBmdW5jdGlvbiBmb3IgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50czsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz00MDctNDIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYWNjdXJhY3kgb2Ygc2V2ZXJhbCBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgcHJvZ3JhbXMgZm9yIHByb3RlaW5zOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTQ3MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPTNEQ29mZmVlOiBjb21iaW5pbmcgcHJvdGVpbiBzZXF1ZW5jZXMgYW5kIHN0cnVjdHVyZXMgd2l0aGluIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudHM7IEogTW9sIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM0MDsgIGNpdGF0aW9uX3BhZ2VzPTM4NS0zOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBST01BTFMzRDogYSB0b29sIGZvciBtdWx0aXBsZSBwcm90ZWluIHNlcXVlbmNlIGFuZCBzdHJ1Y3R1cmUgYWxpZ25tZW50czsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9MjI5NS0yMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUGZhbSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1EMjkwLUQzMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBpY1hBQS1SOiBlZmZpY2llbnQgc3RydWN0dXJhbCBhbGlnbm1lbnQgb2YgbXVsdGlwbGUgUk5BIHNlcXVlbmNlcyB1c2luZyBhIGdyZWVkeSBhcHByb2FjaDsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9UzM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFB5TU9MIE1vbGVjdWxhciBHcmFwaGljcyBTeXN0ZW0sIFZlcnNpb24gMS4zcjEuIFBvcnRsYW5kOyBjaXRhdGlvbl90ZXh0PU9yZWdvbjogU2NocsO2ZGluZ2VyLCBMTEM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3Qgc2NhbGFibGUgZ2VuZXJhdGlvbiBvZiBoaWdoLXF1YWxpdHkgcHJvdGVpbiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnRzIHVzaW5nIENsdXN0YWwgT21lZ2E7IE1vbCBTeXN0IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz01MzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QUk9TSVRFLCBhIHByb3RlaW4gZG9tYWluIGRhdGFiYXNlIGZvciBmdW5jdGlvbmFsIGNoYXJhY3Rlcml6YXRpb24gYW5kIGFubm90YXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPUQxNjEtRDE2NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNoIHN0cnVjdHVyZSBhbGlnbm1lbnQgcGFja2FnZTogc2Vuc2l0aXZpdHkgYW5kIHNlbGVjdGl2aXR5IGluIGRvbWFpbiBjbGFzc2lmaWNhdGlvbjsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3RpbmcgbG9jYWwgc3RydWN0dXJhbCBzaW1pbGFyaXR5IGluIHByb3RlaW5zIGJ5IG1heGltaXppbmcgbnVtYmVyIG9mIGVxdWl2YWxlbnQgcmVzaWR1ZXM7IFByb3RlaW5zOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9MzgxLTM5MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5TEFUOiBhIHBoeWxvZ2VuZXRpYyBsb2NhbCBhbGlnbm1lbnQgdG9vbDsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xMzM2LTEzNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmFzdCBzdHJ1Y3R1cmFsIG11bHRpcGxlIGFsaWdubWVudCBtZXRob2QgZm9yIGxvbmcgUk5BIHNlcXVlbmNlczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNMVVNUQUwgVzogaW1wcm92aW5nIHRoZSBzZW5zaXRpdml0eSBvZiBwcm9ncmVzc2l2ZSBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgdGhyb3VnaCBzZXF1ZW5jZSB3ZWlnaHRpbmcsIHBvc2l0aW9uLXNwZWNpZmljIGdhcCBwZW5hbHRpZXMgYW5kIHdlaWdodCBtYXRyaXggY2hvaWNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz00NjczLTQ2ODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUphbHZpZXcgdmVyc2lvbiAy4oCUYSBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgZWRpdG9yIGFuZCBhbmFseXNpcyB3b3JrYmVuY2g7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTE4OS0xMTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vbWJlL2FydGljbGUtcGRmLzMwLzQvNzcyLzY0MjA0MTkvbXN0MDEwLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSByZXBvcnQgYSBtYWpvciB1cGRhdGUgb2YgdGhlIE1BRkZUIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBwcm9ncmFtLiBUaGlzIHZlcnNpb24gaGFzIHNldmVyYWwgbmV3IGZlYXR1cmVzLCBpbmNsdWRpbmcgb3B0aW9ucyBmb3IgYWRkaW5nIHVuYWxpZyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MTA3MzM5OCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRPz1fBOF2rjWEy7HfE/4nxyusUwA==" type="hidden">
</div>

<div class="aspNetHidden">

	<input name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" type="hidden">
</div>
        </form>
        
        <input id="hdnSiteID" value="5325" type="hidden">
        <input id="hdnAdDelaySeconds" value="5000" type="hidden">
        <input id="hdnAdConfigurationTop" value="default" type="hidden">
        <input id="hdnAdConfigurationRightRail" value="sticky" type="hidden">

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Behind_Ad1_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/mbe_Behind_Ad1_0" title="3rd party ad content" name="google_ads_iframe_/116097782/mbe_Behind_Ad1_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="728" frameborder="0" height="90"></iframe></div></div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="https://academic.oup.com/"><img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f30%2f4%2f772%2f1073398" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f30%2f4%2f772%2f1073398" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132764">
            <a href="https://academic.oup.com/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132765">
            <a href="https://academic.oup.com/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132762">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132766">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132767">
            <a href="https://academic.oup.com/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132768">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132769">
            <a href="https://academic.oup.com/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132770">
            <a href="https://academic.oup.com/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132771">
            <a href="https://academic.oup.com/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132772">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132763">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132773">
            <a href="https://academic.oup.com/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132774">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132775">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132776">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132777">
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132778">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132779">
            <a href="https://academic.oup.com/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132780">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132781">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="https://academic.oup.com/mbe" class=" journal-logo-container">

            <img id="logo-MolecularBiologyandEvolution" class="journal-logo" src="mbe_title188616415.svg" alt="Molecular Biology and Evolution">

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.smbe.org/smbe/" target="" class="society-logo-container">
                                <img id="logo-SocietyforMolecularBiologyandEvolution" class="society-logo" src="mbe_h1-1277340080.svg" alt="Society for Molecular Biology and Evolution">
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132764">
            <a href="https://academic.oup.com/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132765">
            <a href="https://academic.oup.com/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132762">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132766">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132767">
            <a href="https://academic.oup.com/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132768">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132769">
            <a href="https://academic.oup.com/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132770">
            <a href="https://academic.oup.com/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132771">
            <a href="https://academic.oup.com/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132772">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1132763">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132773">
            <a href="https://academic.oup.com/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132774">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132775">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132776">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132777">
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132778">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132779">
            <a href="https://academic.oup.com/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132780">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1132781">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/mbe/advanced-search?page=1&amp;fl_SiteID=5325&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="SMBE Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/smbe" type="hidden">
    <input name="siteID" id="hfsiteID" value="5325" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5538" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=46264" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/mbe" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="mbe" type="hidden">
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left mobile">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/mbe/issue/30/4"><img id="issueImage" class="fb-featured-image" src="m_mbe30_4.gif" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/mbe/issue/30/4">
                        <div class="volume trailing-comma">Volume 30</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="33680819">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680819">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="33680821">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680821">Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="33680823">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680823">Basic Concepts and Usage</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="33680827">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680827">Profile Alignments</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="33680831">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680831">Adding Unaligned Sequences into an MSA</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="33680858">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680858">Parallelization</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="33680861">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680861">Utility Options</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="33680873">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680873">Parameters</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="33680878">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680878">Use of Structural Information</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="33680886">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680886">Acknowledgments</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="33680888">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#33680888">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/mbe/article/30/4/761/1071846">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/mbe/article/30/4/781/1067373">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazutaka Katoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Immunology Frontier Research Center, Osaka University, Suita, Osaka, Japan</div><div class="aff"><div class="label -label"><sup>2</sup></div>Computational Biology Research Center, The National Institute of Advanced Industrial Science and Technology (AIST), Tokyo, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="mst010-COR1" data-open="mst010-COR1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <strong>*Corresponding author:</strong> E-mail: <a href="mailto:kazutaka.katoh@aist.go.jp" target="">kazutaka.katoh@aist.go.jp</a>.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Kazutaka+Katoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katoh%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Katoh%20K%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazutaka Katoh<i class="icon-general-mail"></i></a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daron M. Standley</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Immunology Frontier Research Center, Osaka University, Suita, Osaka, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Daron+M.+Standley">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Standley%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Standley%20D%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Daron M. Standley</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content"><strong>Associate editor:</strong> Sudhir Kumar</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>Molecular Biology and Evolution</em>, Volume 30, Issue 4, 1 April 2013, Pages 772–780, <a href="https://doi.org/10.1093/molbev/mst010">https://doi.org/10.1093/molbev/mst010</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">16 January 2013</div>
                        </div>
                    </div>
                            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views" style="display: none;">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up" style="display: block;"></div>
                                    <li class="article-content-filter" data-content-filter="article-content" style="display: none;"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    
                                    
                                    
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1073398" href="https://academic.oup.com/mbe/article-pdf/30/4/772/6420419/mst010.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Kazutaka Katoh, Daron M. Standley;  MAFFT Multiple Sequence 
Alignment Software Version 7: Improvements in Performance and Usability,
 <em>Molecular Biology and Evolution</em>, Volume 30, Issue 4, 1 April 2013, Pages 772–780, <a href="https://doi.org/10.1093/molbev/mst010">https://doi.org/10.1093/molbev/mst010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/mbe/downloadcitation/1073398?format=ris">RIS (Zotero)</a></li>
        <li><a href="https://academic.oup.com/mbe/downloadcitation/1073398?format=ris">EndNote</a></li>
        <li><a href="https://academic.oup.com/mbe/downloadcitation/1073398?format=bibtex">BibTex</a></li>
        <li><a href="https://academic.oup.com/mbe/downloadcitation/1073398?format=txt">Medlars</a></li>
        <li><a href="https://academic.oup.com/mbe/downloadcitation/1073398?format=ris">ProCite</a></li>
        <li><a href="https://academic.oup.com/mbe/downloadcitation/1073398?format=txt">RefWorks</a></li>
        <li><a href="https://academic.oup.com/mbe/downloadcitation/1073398?format=ris">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1537-1719&amp;title=MAFFT%20Multiple%20Sequence%20Alignment%20Software%20Version%207%3A%20Improvements%20in%20Performance%20and%20Usability&amp;publicationDate=2013-01-16&amp;volumeNum=30&amp;issueNum=4&amp;author=Katoh%2C%20Kazutaka%3B%20Standley%2C%20Daron%20M.&amp;startPage=772&amp;endPage=780&amp;contentId=10.1093%2Fmolbev%2Fmst010&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email at300b" target="_blank" title="Email" href="#"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <div class="atclear"></div></ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f30%2f4%2f772%2f1073398" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/mbe/advanced-search?page=1&amp;fl_SiteID=5325&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" value="mbe" type="hidden">
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination="33680819" id="33680819" class="abstract-title">Abstract</h2>
<section class="abstract"><p>We report a major update of the MAFFT 
multiple sequence alignment program. This version has several new 
features, including options for adding unaligned sequences into an 
existing alignment, adjustment of direction in nucleotide alignment, 
constrained alignment and parallel processing, which were implemented 
after the previous major update. This report shows actual examples to 
explain how these features work, alone and in combination. Some examples
 incorrectly aligned by MAFFT are also shown to clarify its limitations.
 We discuss how to avoid misalignments, and our ongoing efforts to 
overcome such limitations.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;multiple sequence alignment&quot;">multiple sequence alignment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="metagemone">metagemone</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;protein structure&quot;">protein structure</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;progressive alignment&quot;">progressive alignment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;parallel processing&quot;">parallel processing</a></div>                            <h2 scrollto-destination="33680821" id="33680821" class="section-title">Introduction</h2>
<p>Multiple sequence alignment (MSA) plays an important role in 
evolutionary analyses of biological sequences. MAFFT is an MSA program, 
first released in 2002 (<a reveal-id="mst010-B22" data-open="mst010-B22" class="link link-ref link-reveal xref-bibr">Katoh et al. 2002</a>). Because of its high performance (<a reveal-id="mst010-B34" data-open="mst010-B34" class="link link-ref link-reveal xref-bibr">Nuin et al. 2006</a>; <a reveal-id="mst010-B12" data-open="mst010-B12" class="link link-ref link-reveal xref-bibr">Golubchik et al. 2007</a>; <a reveal-id="mst010-B9" data-open="mst010-B9" class="link link-ref link-reveal xref-bibr">Dessimoz and Gil 2010</a>; <a reveal-id="mst010-B29" data-open="mst010-B29" class="link link-ref link-reveal xref-bibr">Letsch et al. 2010</a>; <a reveal-id="mst010-B38" data-open="mst010-B38" class="link link-ref link-reveal xref-bibr">Sahraeian and Yoon 2011</a>; <a reveal-id="mst010-B40" data-open="mst010-B40" class="link link-ref link-reveal xref-bibr">Sievers et al. 2011</a>), MAFFT is becoming popular in recent years. After reviewing the previous version (version 6) in <a reveal-id="mst010-B26" data-open="mst010-B26" class="link link-ref link-reveal xref-bibr">Katoh and Toh (2008b)</a>,
 we have been continuously improving its accuracy, speed, and utility in
 practical situations. These improvements and techniques were mostly 
reported in individual papers (<a reveal-id="mst010-B19" data-open="mst010-B19" class="link link-ref link-reveal xref-bibr">Katoh et al. 2009</a>; <a reveal-id="mst010-B27" data-open="mst010-B27" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2010</a>; <a reveal-id="mst010-B20" data-open="mst010-B20" class="link link-ref link-reveal xref-bibr">Katoh and Frith 2012</a>; <a reveal-id="mst010-B23" data-open="mst010-B23" class="link link-ref link-reveal xref-bibr">Katoh and Standley 2013</a>).
 In this report, we demonstrate the different kinds of analyses that can
 be achieved with the new features, alone and in combination, using 
realistic examples. We also discuss limitations of current version by 
giving examples of sequences incorrectly aligned by MAFFT, and describe 
our ongoing efforts to overcome these limitations.</p>                            <h2 scrollto-destination="33680823" id="33680823" class="section-title">Basic Concepts and Usage</h2>
<p>As listed in <a reveal-id="mst010-T1" data-open="mst010-T1" class="link link-reveal link-table xref-fig">table 1</a>,
 MAFFT version 7 has options for various alignment strategies, including
 progressive methods (PartTree, FFT-NS-1, and L-INS-1) (<a reveal-id="mst010-B10" data-open="mst010-B10" class="link link-ref link-reveal xref-bibr">Feng and Doolittle 1987</a>; <a reveal-id="mst010-B16" data-open="mst010-B16" class="link link-ref link-reveal xref-bibr">Higgins and Sharp 1988</a>; <a reveal-id="mst010-B24" data-open="mst010-B24" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2007</a>), iterative refinement methods (FFT-NS-i, L-INS-i, E-INS-i, and G-INS-i) (<a reveal-id="mst010-B2" data-open="mst010-B2" class="link link-ref link-reveal xref-bibr">Barton and Sternberg 1987</a>; <a reveal-id="mst010-B3" data-open="mst010-B3" class="link link-ref link-reveal xref-bibr">Berger and Munson 1991</a>; <a reveal-id="mst010-B13" data-open="mst010-B13" class="link link-ref link-reveal xref-bibr">Gotoh 1993</a>; <a reveal-id="mst010-B21" data-open="mst010-B21" class="link link-ref link-reveal xref-bibr">Katoh et al. 2005</a>), and structural alignment methods for RNAs (Q-INS-i and X-INS-i; <a reveal-id="mst010-B25" data-open="mst010-B25" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2008a</a>). See <a reveal-id="mst010-B26" data-open="mst010-B26" class="link link-ref link-reveal xref-bibr">Katoh and Toh (2008b)</a> for details of these strategies. According to a recent comparative study based on the MetAl metric (<a reveal-id="mst010-B5" data-open="mst010-B5" class="link link-ref link-reveal xref-bibr">Blackburne and Whelan 2012a</a>, <a reveal-id="mst010-B6" data-open="mst010-B6" class="link link-ref link-reveal xref-bibr">2012b</a>),
 there are two significantly different classes of MSA methods, 
similarity-based methods and evolution-based methods. MAFFT is 
classified as a similarity-based method. However, evolutionary 
information is useful even for similarity-based methods, because the 
sequences to be aligned are generated from a common ancestor in the 
course of evolution. In this respect, MAFFT takes evolutionary 
information into account. </p>                        <a id="33680825" scrollto-destination="33680825"></a>
<div content-id="mst010-T1" class="table-modal" style="display: block;"><div class="table-wrap"><div id="mst010-T1" class="table-wrap-title" data-id="mst010-T1"><div class="label -label">Table 1.</div><div class="caption"><p>Options of MAFFT Version 7.</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Option name&nbsp;</th><th>Command&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td colspan="3">For a large-scale alignment: progressive methods with the PartTree algorithm<hr>&nbsp;</td></tr><tr><td>    NW-NS-PartTree1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">parttree </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Distance is by the 6mer method.&nbsp;</td></tr><tr><td>    NW-NS-PartTree2&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">parttree </span><span class="monospace">––</span><span class="monospace">retree 2 input</span>&nbsp;</td><td>Distance is by the 6mer method. Guide tree is re-built.&nbsp;</td></tr><tr><td>    NW-NS-DPPartTree1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">dpparttree </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Distance is estimated based on DP.&nbsp;</td></tr><tr><td>    NW-NS-DPPartTree2&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">dpparttree </span><span class="monospace">––</span><span class="monospace">retree 2 input</span>&nbsp;</td><td>Distance is estimated based on DP. Guide tree is re-built.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a medium-scale alignment: progressive methods&nbsp;</td></tr><tr><td>    FFT-NS-1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Approximately two times faster than the default.&nbsp;</td></tr><tr><td>    FFT-NS-2&nbsp;</td><td><span class="monospace">mafft input</span>&nbsp;</td><td>Default.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a small-scale alignment: iterative refinement methods&nbsp;</td></tr><tr><td>    FFT-NS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 input</span>&nbsp;</td><td>Fastest of the four in this category. Uses WSP score (<a reveal-id="mst010-B14" data-open="mst010-B14" class="link link-ref link-reveal xref-bibr">Gotoh 1995</a>) only.&nbsp;</td></tr><tr><td>    G-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">globalpair input</span>&nbsp;</td><td>Uses WSP score and consistency (<a reveal-id="mst010-B33" data-open="mst010-B33" class="link link-ref link-reveal xref-bibr">Notredame et al. 1998</a>) score from global alignments.&nbsp;</td></tr><tr><td>    L-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">localpair input</span>&nbsp;</td><td>Uses WSP score and consistency score from local alignments.&nbsp;</td></tr><tr><td>    E-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">genafpair input</span>&nbsp;</td><td>Uses WSP score and consistency score from local alignments with a generalized affine gap cost (<a reveal-id="mst010-B1" data-open="mst010-B1" class="link link-ref link-reveal xref-bibr">Altschul 1998</a>).&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">If not sure which option to use&nbsp;</td></tr><tr><td>    Automatic&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">auto</span>&nbsp;</td><td>Selects an appropriate option from FFT-NS-2, FFT-NS-i and L-INS-i, according to the size of input data.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a small-scale RNA alignment: structural alignment methods&nbsp;</td></tr><tr><td>    Q-INS-i&nbsp;</td><td><span class="monospace">mafft-qinsi input</span>&nbsp;</td><td>Structure information is included in iterative refinement step.&nbsp;</td></tr><tr><td>    X-INS-i-scarnapair&nbsp;</td><td><span class="monospace">mafft-xinsi </span><span class="monospace">––</span><span class="monospace">scarnapair input</span>&nbsp;</td><td>Uses pairwise structural alignment by MXSCARNA (<a reveal-id="mst010-B45" data-open="mst010-B45" class="link link-ref link-reveal xref-bibr">Tabei et al. 2008</a>).&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">To add new sequences into an existing MSA&nbsp;</td></tr><tr><td>    Add&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">add mew msa</span>&nbsp;</td><td>The simplest option for alignment extension.&nbsp;</td></tr><tr><td>    Addprofile&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addprofile msa1 </span><span class="monospace">msa2</span>&nbsp;</td><td>msa1 must form a monophyletic cluster.&nbsp;</td></tr><tr><td>    Addfragments&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addfragments new msa</span>&nbsp;</td><td>Suitable for short new sequences.&nbsp;</td></tr><tr><td>    Addfragments, LAST&nbsp;</td><td><span class="monospace">mafft ––addfragments new </span><span class="monospace">––</span><span class="monospace">lastmultipair msa</span>&nbsp;</td><td>Faster option, LAST (<a reveal-id="mst010-B28" data-open="mst010-B28" class="link link-ref link-reveal xref-bibr">Kiełbasa et al. 2011</a>) is required.&nbsp;</td></tr><tr><td>    Addfragments, 6mer&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addfragments new </span><span class="monospace">––</span><span class="monospace">6merpair msa</span>&nbsp;</td><td>Faster option for conserved data.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">Parameters&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">bl #, </span><span class="monospace">––</span><span class="monospace">jtt #, </span><span class="monospace">––</span><span class="monospace">tm #</span>&nbsp;</td><td>Score matrices for protein alignment.&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">kimura #</span>&nbsp;</td><td>Score matrix for nucleotide alignment.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">Utility options&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">anysymbol</span>&nbsp;</td><td rowspan="11">See main text.&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">reorder</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">clustalout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">phylipout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">namelength #</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">adjustdirection</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">adjustdirectionaccurately</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">seed msa1 –seed msa2</span><span class="monospace"> …</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">treein treefile</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">treeout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">thread #</span>&nbsp;</td></tr></tbody></table></div><div content-id="mst010-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>Option name&nbsp;</th><th>Command&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td colspan="3">For a large-scale alignment: progressive methods with the PartTree algorithm<hr>&nbsp;</td></tr><tr><td>    NW-NS-PartTree1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">parttree </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Distance is by the 6mer method.&nbsp;</td></tr><tr><td>    NW-NS-PartTree2&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">parttree </span><span class="monospace">––</span><span class="monospace">retree 2 input</span>&nbsp;</td><td>Distance is by the 6mer method. Guide tree is re-built.&nbsp;</td></tr><tr><td>    NW-NS-DPPartTree1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">dpparttree </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Distance is estimated based on DP.&nbsp;</td></tr><tr><td>    NW-NS-DPPartTree2&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">dpparttree </span><span class="monospace">––</span><span class="monospace">retree 2 input</span>&nbsp;</td><td>Distance is estimated based on DP. Guide tree is re-built.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a medium-scale alignment: progressive methods&nbsp;</td></tr><tr><td>    FFT-NS-1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Approximately two times faster than the default.&nbsp;</td></tr><tr><td>    FFT-NS-2&nbsp;</td><td><span class="monospace">mafft input</span>&nbsp;</td><td>Default.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a small-scale alignment: iterative refinement methods&nbsp;</td></tr><tr><td>    FFT-NS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 input</span>&nbsp;</td><td>Fastest of the four in this category. Uses WSP score (<a reveal-id="mst010-B14" data-open="mst010-B14" class="link link-ref link-reveal xref-bibr">Gotoh 1995</a>) only.&nbsp;</td></tr><tr><td>    G-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">globalpair input</span>&nbsp;</td><td>Uses WSP score and consistency (<a reveal-id="mst010-B33" data-open="mst010-B33" class="link link-ref link-reveal xref-bibr">Notredame et al. 1998</a>) score from global alignments.&nbsp;</td></tr><tr><td>    L-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">localpair input</span>&nbsp;</td><td>Uses WSP score and consistency score from local alignments.&nbsp;</td></tr><tr><td>    E-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">genafpair input</span>&nbsp;</td><td>Uses WSP score and consistency score from local alignments with a generalized affine gap cost (<a reveal-id="mst010-B1" data-open="mst010-B1" class="link link-ref link-reveal xref-bibr">Altschul 1998</a>).&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">If not sure which option to use&nbsp;</td></tr><tr><td>    Automatic&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">auto</span>&nbsp;</td><td>Selects an appropriate option from FFT-NS-2, FFT-NS-i and L-INS-i, according to the size of input data.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a small-scale RNA alignment: structural alignment methods&nbsp;</td></tr><tr><td>    Q-INS-i&nbsp;</td><td><span class="monospace">mafft-qinsi input</span>&nbsp;</td><td>Structure information is included in iterative refinement step.&nbsp;</td></tr><tr><td>    X-INS-i-scarnapair&nbsp;</td><td><span class="monospace">mafft-xinsi </span><span class="monospace">––</span><span class="monospace">scarnapair input</span>&nbsp;</td><td>Uses pairwise structural alignment by MXSCARNA (<a reveal-id="mst010-B45" data-open="mst010-B45" class="link link-ref link-reveal xref-bibr">Tabei et al. 2008</a>).&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">To add new sequences into an existing MSA&nbsp;</td></tr><tr><td>    Add&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">add mew msa</span>&nbsp;</td><td>The simplest option for alignment extension.&nbsp;</td></tr><tr><td>    Addprofile&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addprofile msa1 </span><span class="monospace">msa2</span>&nbsp;</td><td>msa1 must form a monophyletic cluster.&nbsp;</td></tr><tr><td>    Addfragments&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addfragments new msa</span>&nbsp;</td><td>Suitable for short new sequences.&nbsp;</td></tr><tr><td>    Addfragments, LAST&nbsp;</td><td><span class="monospace">mafft ––addfragments new </span><span class="monospace">––</span><span class="monospace">lastmultipair msa</span>&nbsp;</td><td>Faster option, LAST (<a reveal-id="mst010-B28" data-open="mst010-B28" class="link link-ref link-reveal xref-bibr">Kiełbasa et al. 2011</a>) is required.&nbsp;</td></tr><tr><td>    Addfragments, 6mer&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addfragments new </span><span class="monospace">––</span><span class="monospace">6merpair msa</span>&nbsp;</td><td>Faster option for conserved data.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">Parameters&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">bl #, </span><span class="monospace">––</span><span class="monospace">jtt #, </span><span class="monospace">––</span><span class="monospace">tm #</span>&nbsp;</td><td>Score matrices for protein alignment.&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">kimura #</span>&nbsp;</td><td>Score matrix for nucleotide alignment.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">Utility options&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">anysymbol</span>&nbsp;</td><td rowspan="11">See main text.&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">reorder</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">clustalout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">phylipout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">namelength #</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">adjustdirection</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">adjustdirectionaccurately</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">seed msa1 –seed msa2</span><span class="monospace"> …</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">treein treefile</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">treeout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">thread #</span>&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="mst010-TF1"><div class="fn"><p>N<span class="small-caps">ote</span>.—<strong>N, the number of sequences; L, the sequence length; <span class="monospace"><strong>input, new</strong></span>, unaligned sequences in the multi-fasta format; <span class="monospace"><strong>msa, msa1, msa2</strong></span>, aligned sequences in the multi-fasta format; <span class="monospace"><strong>treefile</strong></span>, input guide tree file.</strong></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/33680825" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="mst010-T1" class="table-wrap-title" data-id="mst010-T1"><div class="label -label">Table 1.</div><div class="caption"><p>Options of MAFFT Version 7.</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Option name&nbsp;</th><th>Command&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td colspan="3">For a large-scale alignment: progressive methods with the PartTree algorithm<hr>&nbsp;</td></tr><tr><td>    NW-NS-PartTree1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">parttree </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Distance is by the 6mer method.&nbsp;</td></tr><tr><td>    NW-NS-PartTree2&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">parttree </span><span class="monospace">––</span><span class="monospace">retree 2 input</span>&nbsp;</td><td>Distance is by the 6mer method. Guide tree is re-built.&nbsp;</td></tr><tr><td>    NW-NS-DPPartTree1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">dpparttree </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Distance is estimated based on DP.&nbsp;</td></tr><tr><td>    NW-NS-DPPartTree2&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">dpparttree </span><span class="monospace">––</span><span class="monospace">retree 2 input</span>&nbsp;</td><td>Distance is estimated based on DP. Guide tree is re-built.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a medium-scale alignment: progressive methods&nbsp;</td></tr><tr><td>    FFT-NS-1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Approximately two times faster than the default.&nbsp;</td></tr><tr><td>    FFT-NS-2&nbsp;</td><td><span class="monospace">mafft input</span>&nbsp;</td><td>Default.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a small-scale alignment: iterative refinement methods&nbsp;</td></tr><tr><td>    FFT-NS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 input</span>&nbsp;</td><td>Fastest of the four in this category. Uses WSP score (<a reveal-id="mst010-B14" data-open="mst010-B14" class="link link-ref link-reveal xref-bibr">Gotoh 1995</a>) only.&nbsp;</td></tr><tr><td>    G-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">globalpair input</span>&nbsp;</td><td>Uses WSP score and consistency (<a reveal-id="mst010-B33" data-open="mst010-B33" class="link link-ref link-reveal xref-bibr">Notredame et al. 1998</a>) score from global alignments.&nbsp;</td></tr><tr><td>    L-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">localpair input</span>&nbsp;</td><td>Uses WSP score and consistency score from local alignments.&nbsp;</td></tr><tr><td>    E-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">genafpair input</span>&nbsp;</td><td>Uses WSP score and consistency score from local alignments with a generalized affine gap cost (<a reveal-id="mst010-B1" data-open="mst010-B1" class="link link-ref link-reveal xref-bibr">Altschul 1998</a>).&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">If not sure which option to use&nbsp;</td></tr><tr><td>    Automatic&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">auto</span>&nbsp;</td><td>Selects an appropriate option from FFT-NS-2, FFT-NS-i and L-INS-i, according to the size of input data.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a small-scale RNA alignment: structural alignment methods&nbsp;</td></tr><tr><td>    Q-INS-i&nbsp;</td><td><span class="monospace">mafft-qinsi input</span>&nbsp;</td><td>Structure information is included in iterative refinement step.&nbsp;</td></tr><tr><td>    X-INS-i-scarnapair&nbsp;</td><td><span class="monospace">mafft-xinsi </span><span class="monospace">––</span><span class="monospace">scarnapair input</span>&nbsp;</td><td>Uses pairwise structural alignment by MXSCARNA (<a reveal-id="mst010-B45" data-open="mst010-B45" class="link link-ref link-reveal xref-bibr">Tabei et al. 2008</a>).&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">To add new sequences into an existing MSA&nbsp;</td></tr><tr><td>    Add&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">add mew msa</span>&nbsp;</td><td>The simplest option for alignment extension.&nbsp;</td></tr><tr><td>    Addprofile&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addprofile msa1 </span><span class="monospace">msa2</span>&nbsp;</td><td>msa1 must form a monophyletic cluster.&nbsp;</td></tr><tr><td>    Addfragments&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addfragments new msa</span>&nbsp;</td><td>Suitable for short new sequences.&nbsp;</td></tr><tr><td>    Addfragments, LAST&nbsp;</td><td><span class="monospace">mafft ––addfragments new </span><span class="monospace">––</span><span class="monospace">lastmultipair msa</span>&nbsp;</td><td>Faster option, LAST (<a reveal-id="mst010-B28" data-open="mst010-B28" class="link link-ref link-reveal xref-bibr">Kiełbasa et al. 2011</a>) is required.&nbsp;</td></tr><tr><td>    Addfragments, 6mer&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addfragments new </span><span class="monospace">––</span><span class="monospace">6merpair msa</span>&nbsp;</td><td>Faster option for conserved data.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">Parameters&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">bl #, </span><span class="monospace">––</span><span class="monospace">jtt #, </span><span class="monospace">––</span><span class="monospace">tm #</span>&nbsp;</td><td>Score matrices for protein alignment.&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">kimura #</span>&nbsp;</td><td>Score matrix for nucleotide alignment.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">Utility options&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">anysymbol</span>&nbsp;</td><td rowspan="11">See main text.&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">reorder</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">clustalout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">phylipout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">namelength #</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">adjustdirection</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">adjustdirectionaccurately</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">seed msa1 –seed msa2</span><span class="monospace"> …</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">treein treefile</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">treeout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">thread #</span>&nbsp;</td></tr></tbody></table></div><div content-id="mst010-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>Option name&nbsp;</th><th>Command&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td colspan="3">For a large-scale alignment: progressive methods with the PartTree algorithm<hr>&nbsp;</td></tr><tr><td>    NW-NS-PartTree1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">parttree </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Distance is by the 6mer method.&nbsp;</td></tr><tr><td>    NW-NS-PartTree2&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">parttree </span><span class="monospace">––</span><span class="monospace">retree 2 input</span>&nbsp;</td><td>Distance is by the 6mer method. Guide tree is re-built.&nbsp;</td></tr><tr><td>    NW-NS-DPPartTree1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">dpparttree </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Distance is estimated based on DP.&nbsp;</td></tr><tr><td>    NW-NS-DPPartTree2&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">dpparttree </span><span class="monospace">––</span><span class="monospace">retree 2 input</span>&nbsp;</td><td>Distance is estimated based on DP. Guide tree is re-built.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a medium-scale alignment: progressive methods&nbsp;</td></tr><tr><td>    FFT-NS-1&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">retree 1 input</span>&nbsp;</td><td>Approximately two times faster than the default.&nbsp;</td></tr><tr><td>    FFT-NS-2&nbsp;</td><td><span class="monospace">mafft input</span>&nbsp;</td><td>Default.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a small-scale alignment: iterative refinement methods&nbsp;</td></tr><tr><td>    FFT-NS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 input</span>&nbsp;</td><td>Fastest of the four in this category. Uses WSP score (<a reveal-id="mst010-B14" data-open="mst010-B14" class="link link-ref link-reveal xref-bibr">Gotoh 1995</a>) only.&nbsp;</td></tr><tr><td>    G-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">globalpair input</span>&nbsp;</td><td>Uses WSP score and consistency (<a reveal-id="mst010-B33" data-open="mst010-B33" class="link link-ref link-reveal xref-bibr">Notredame et al. 1998</a>) score from global alignments.&nbsp;</td></tr><tr><td>    L-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">localpair input</span>&nbsp;</td><td>Uses WSP score and consistency score from local alignments.&nbsp;</td></tr><tr><td>    E-INS-i&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">maxiterate 16 </span><span class="monospace">––</span><span class="monospace">genafpair input</span>&nbsp;</td><td>Uses WSP score and consistency score from local alignments with a generalized affine gap cost (<a reveal-id="mst010-B1" data-open="mst010-B1" class="link link-ref link-reveal xref-bibr">Altschul 1998</a>).&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">If not sure which option to use&nbsp;</td></tr><tr><td>    Automatic&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">auto</span>&nbsp;</td><td>Selects an appropriate option from FFT-NS-2, FFT-NS-i and L-INS-i, according to the size of input data.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">For a small-scale RNA alignment: structural alignment methods&nbsp;</td></tr><tr><td>    Q-INS-i&nbsp;</td><td><span class="monospace">mafft-qinsi input</span>&nbsp;</td><td>Structure information is included in iterative refinement step.&nbsp;</td></tr><tr><td>    X-INS-i-scarnapair&nbsp;</td><td><span class="monospace">mafft-xinsi </span><span class="monospace">––</span><span class="monospace">scarnapair input</span>&nbsp;</td><td>Uses pairwise structural alignment by MXSCARNA (<a reveal-id="mst010-B45" data-open="mst010-B45" class="link link-ref link-reveal xref-bibr">Tabei et al. 2008</a>).&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">To add new sequences into an existing MSA&nbsp;</td></tr><tr><td>    Add&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">add mew msa</span>&nbsp;</td><td>The simplest option for alignment extension.&nbsp;</td></tr><tr><td>    Addprofile&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addprofile msa1 </span><span class="monospace">msa2</span>&nbsp;</td><td>msa1 must form a monophyletic cluster.&nbsp;</td></tr><tr><td>    Addfragments&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addfragments new msa</span>&nbsp;</td><td>Suitable for short new sequences.&nbsp;</td></tr><tr><td>    Addfragments, LAST&nbsp;</td><td><span class="monospace">mafft ––addfragments new </span><span class="monospace">––</span><span class="monospace">lastmultipair msa</span>&nbsp;</td><td>Faster option, LAST (<a reveal-id="mst010-B28" data-open="mst010-B28" class="link link-ref link-reveal xref-bibr">Kiełbasa et al. 2011</a>) is required.&nbsp;</td></tr><tr><td>    Addfragments, 6mer&nbsp;</td><td><span class="monospace">mafft </span><span class="monospace">––</span><span class="monospace">addfragments new </span><span class="monospace">––</span><span class="monospace">6merpair msa</span>&nbsp;</td><td>Faster option for conserved data.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">Parameters&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">bl #, </span><span class="monospace">––</span><span class="monospace">jtt #, </span><span class="monospace">––</span><span class="monospace">tm #</span>&nbsp;</td><td>Score matrices for protein alignment.&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">kimura #</span>&nbsp;</td><td>Score matrix for nucleotide alignment.&nbsp;</td></tr><tr><td colspan="3"><hr>&nbsp;</td></tr><tr><td colspan="3">Utility options&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">anysymbol</span>&nbsp;</td><td rowspan="11">See main text.&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">reorder</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">clustalout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">phylipout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">namelength #</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">adjustdirection</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">adjustdirectionaccurately</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">seed msa1 –seed msa2</span><span class="monospace"> …</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">treein treefile</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">treeout</span>&nbsp;</td></tr><tr><td>&nbsp;</td><td><span class="monospace">––</span><span class="monospace">thread #</span>&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="mst010-TF1"><div class="fn"><p>N<span class="small-caps">ote</span>.—<strong>N, the number of sequences; L, the sequence length; <span class="monospace"><strong>input, new</strong></span>, unaligned sequences in the multi-fasta format; <span class="monospace"><strong>msa, msa1, msa2</strong></span>, aligned sequences in the multi-fasta format; <span class="monospace"><strong>treefile</strong></span>, input guide tree file.</strong></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/33680825" target="_blank">View Large</a></div></div><p>All
 the options of MAFFT assume that the input sequences are all 
homologous, that is, descended from a common ancestor. Thus, all the 
letters in the input data are aligned. Genomic rearrangement or domain 
shuffling is not assumed, and thus the order of the letters in each 
sequence is always preserved, although the sequences can be reordered 
according to similarity. Most options in MAFFT assume that almost all 
the pairs in the input sequences can be aligned, locally or globally. In
 such a situation, there is a tradeoff between accuracy and speed. For 
example, the PartTree option (<a reveal-id="mst010-B24" data-open="mst010-B24" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2007</a>)
 is a fast and rough method, whereas L-INS-i and G-INS-i are slower and 
more accurate. RNA structural alignment methods are generally more 
accurate and computationally more expensive because they need additional
 calculations (<a reveal-id="mst010-B25" data-open="mst010-B25" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2008a</a>). However, this tradeoff does not always hold. In particular, the new options to add sequences into an existing alignment (<a reveal-id="mst010-B20" data-open="mst010-B20" class="link link-ref link-reveal xref-bibr">Katoh and Frith 2012</a>), requires careful consideration of this tradeoff, as discussed later.</p>                            <h2 scrollto-destination="33680827" id="33680827" class="section-title">Profile Alignments</h2>
<p>MAFFT has a subprogram, <span class="monospace">mafft-profile</span>, to align two existing alignments. </p><blockquote><p><span class="monospace">mafft-profile alignment1 alignment2 </span><span class="monospace">&gt;</span><span class="monospace"> output</span></p></blockquote> This method separately converts <span class="monospace">alignment1</span> and <span class="monospace">alignment2</span>
 to profiles and then aligns the two profiles. It means that the two 
input alignments are assumed to be phylogenetically isolated from each 
other, like <a reveal-id="mst010-F1" data-open="mst010-F1" class="link link-reveal link-table xref-fig">figure 1</a><em>A</em>. Careless application of this method results in serious misalignments, as discussed in later section. <p></p>                        <a id="33680829" scrollto-destination="33680829"></a>
<div id="mst010-F1" data-id="mst010-F1" class="fig fig-section"><div class="label fig-label"><strong>F<span class="small-caps">ig</span>. 1</strong>.</div><div class="graphic-wrap"><img class="content-image" src="" alt="Assumptions on the phylogenetic relationship in different options of MAFFT. (A) mafft-profile, (B) ––addprofile, (C), misuse of mafft-profile, and (D) ––add or ––addprofile." path-from-xml="mst010f1p"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/33680829/mst010f1p" path-from-xml="mst010f1p" target="_blank">View large</a><a section="33680829" class="download-slide" path-from-xml="mst010f1p" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Assumptions on the phylogenetic relationship in different options of MAFFT. (<em>A</em>) <span class="monospace">mafft-profile</span>, (<em>B</em>) <span class="monospace">––</span><span class="monospace">addprofile</span>, (<em>C</em>), misuse of <span class="monospace">mafft-profile</span>, and (<em>D</em>) <span class="monospace">––</span><span class="monospace">add</span> or <span class="monospace">––</span><span class="monospace">addprofile</span>.</p></div></div><div content-id="mst010-F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label"><strong>F<span class="small-caps">ig</span>. 1</strong>.</div><div class="graphic-wrap"><img class="content-image" src="" alt="Assumptions on the phylogenetic relationship in different options of MAFFT. (A) mafft-profile, (B) ––addprofile, (C), misuse of mafft-profile, and (D) ––add or ––addprofile." path-from-xml="mst010f1p"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/33680829/mst010f1p" path-from-xml="mst010f1p" target="_blank">View large</a><a section="33680829" class="download-slide" path-from-xml="mst010f1p" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Assumptions on the phylogenetic relationship in different options of MAFFT. (<em>A</em>) <span class="monospace">mafft-profile</span>, (<em>B</em>) <span class="monospace">––</span><span class="monospace">addprofile</span>, (<em>C</em>), misuse of <span class="monospace">mafft-profile</span>, and (<em>D</em>) <span class="monospace">––</span><span class="monospace">add</span> or <span class="monospace">––</span><span class="monospace">addprofile</span>.</p></div></div><p>MAFFT version 7 has an alternative option, <span class="monospace">––</span><span class="monospace">addprofile</span>, which is safer against misuses. </p><blockquote><p><span class="monospace">mafft <span class="monospace">--</span></span><span class="monospace">addprofile alignment1 alignment2 </span><span class="monospace">&gt;</span><span class="monospace"> output</span></p></blockquote> This option accepts two existing alignments, <span class="monospace">alignment1</span> and <span class="monospace">alignment2</span>, and assumes a phylogenetic relationship shown in <a reveal-id="mst010-F1" data-open="mst010-F1" class="link link-reveal link-table xref-fig">figure 1</a><em>B</em>. That is, <span class="monospace">alignment1</span> is assumed to form a monophyletic cluster, but <span class="monospace">alignment2</span> is not assumed to form a monophyletic cluster. The cluster of <span class="monospace">alignment1</span> can be placed in any phylogenetic position in the tree of <span class="monospace">alignment2</span>. Moreover, this option checks whether <span class="monospace">alignment1</span> forms a monophyletic cluster. If not, it returns an error message and asks user to use the <span class="monospace">––</span><span class="monospace">add</span> option (see the following section).<p></p>                            <h2 scrollto-destination="33680831" id="33680831" class="section-title">Adding Unaligned Sequences into an MSA</h2>
<p>As a result of advances in sequencing technologies, we increasingly 
need MSAs consisting of a larger number of sequences. There are several 
different approaches to enable construction of large MSAs, such as rapid
 algorithms and parallelization. Here, we describe an alternate 
approach: use of an existing alignment. There already exist databases of
 carefully aligned and annotated sequences (<a reveal-id="mst010-B8" data-open="mst010-B8" class="link link-ref link-reveal xref-bibr">Cole et al. 2009</a>; <a reveal-id="mst010-B41" data-open="mst010-B41" class="link link-ref link-reveal xref-bibr">Sigrist et al. 2010</a>; <a reveal-id="mst010-B37" data-open="mst010-B37" class="link link-ref link-reveal xref-bibr">Punta et al. 2012</a>),
 in which each MSA consists of a small number (typically up to ∼1,000) 
of sequences. We can use such MSAs as a backbone to build a larger MSA 
containing newly sequenced data. This is more efficient than rebuilding 
the entire MSA from a set of ungapped sequences. Moreover, this approach
 is relatively robust to low-quality sequences resulting from sequencing
 errors, misassemblies, and other factors. Such noise usually has a 
negative effect on the quality of an MSA, but there are situations where
 biologically important information is contained in low-quality 
sequences. In such a case, we first select highly reliable sequences to 
build a backbone MSA, and then add the other sequences, including 
low-quality ones, into the MSA. As a result, the quality of the final 
MSA is less affected by the low-quality sequences.</p>                            <h3 scrollto-destination="33680833" id="33680833" class="section-title">Inappropriate Applications of Profile Alignment</h3>
<p>The <span class="monospace">mafft-profile</span> program is not 
useful for this purpose. There are two types of misapplications. One is 
as follows: 1) convert an existing alignment to a profile, 2) align new 
sequences and convert them to a profile, and 3) align the two profiles. 
This procedure is inappropriate for adding new sequences because it 
assumes a phylogenetic relationship as illustrated in <a reveal-id="mst010-F1" data-open="mst010-F1" class="link link-reveal link-table xref-fig">figure 1</a><em>A</em>.</p><p>Another misapplication is as follows: 1) convert the existing alignment to a profile, <em>2</em>)
 separately align each new sequence to the profile of the existing 
alignment, and 3) construct a full alignment from the individual 
alignments computed in the previous step. This approach is more 
reasonable than the first one but still problematic, because the 
phylogenetic positions of new sequences are assumed at the root of the 
tree, as illustrated in <a reveal-id="mst010-F1" data-open="mst010-F1" class="link link-reveal link-table xref-fig">figure 1</a><em>C</em>. Results of this procedure for two cases are shown in <a reveal-id="mst010-T2" data-open="mst010-T2" class="link link-reveal link-table xref-fig">table 2</a> and <a reveal-id="mst010-F2" data-open="mst010-F2" class="link link-reveal link-table xref-fig">figure 2</a>. </p>                        <a id="33680836" scrollto-destination="33680836"></a>
<div id="mst010-F2" data-id="mst010-F2" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 2.</div><div class="graphic-wrap"><img class="content-image" src="" alt="ITS alignments by different options of MAFFT, displayed on Jalview (Waterhouse et al. 2009). (A, B) Incorrect alignments by the FFT-NS-2 and L-INS-i algorithms, respectively. (C) An incorrect alignment by mafft-profile. The full-length sequences were aligned with the L-INS-i algorithm and then each new sequence was separately added to the full-length alignment, using mafft-profile. (D) Reasonable alignment by a two-step strategy. The ––6merpair ––addfragments option was used at the second step. (E) Reordered version of D; sequences are ordered such that similar sequences are placed closely. All calculations were performed using 16 cores on a Linux PC with 2.67 GHz Intel Xeon E7-8837/256 GB RAM." path-from-xml="mst010f2p"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/33680836/mst010f2p" path-from-xml="mst010f2p" target="_blank">View large</a><a section="33680836" class="download-slide" path-from-xml="mst010f2p" target="">Download slide</a></div></div><div class="caption fig-caption"><p>ITS alignments by different options of MAFFT, displayed on Jalview (<a reveal-id="mst010-B47" data-open="mst010-B47" class="link link-ref link-reveal xref-bibr">Waterhouse et al. 2009</a>). (<em>A</em>, <em>B</em>) Incorrect alignments by the FFT-NS-2 and L-INS-i algorithms, respectively. (<em>C</em>) An incorrect alignment by <span class="monospace">mafft-profile</span>.
 The full-length sequences were aligned with the L-INS-i algorithm and 
then each new sequence was separately added to the full-length 
alignment, using <span class="monospace">mafft-profile</span>. (<em>D</em>) Reasonable alignment by a two-step strategy. The <span class="monospace">––</span><span class="monospace">6merpair </span><span class="monospace">––</span><span class="monospace">addfragments</span> option was used at the second step. (<em>E</em>)
 Reordered version of D; sequences are ordered such that similar 
sequences are placed closely. All calculations were performed using 16 
cores on a Linux PC with 2.67 GHz Intel Xeon E7-8837/256 GB RAM.</p></div></div><div content-id="mst010-F2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">ig</span>. 2.</div><div class="graphic-wrap"><img class="content-image" src="" alt="ITS alignments by different options of MAFFT, displayed on Jalview (Waterhouse et al. 2009). (A, B) Incorrect alignments by the FFT-NS-2 and L-INS-i algorithms, respectively. (C) An incorrect alignment by mafft-profile. The full-length sequences were aligned with the L-INS-i algorithm and then each new sequence was separately added to the full-length alignment, using mafft-profile. (D) Reasonable alignment by a two-step strategy. The ––6merpair ––addfragments option was used at the second step. (E) Reordered version of D; sequences are ordered such that similar sequences are placed closely. All calculations were performed using 16 cores on a Linux PC with 2.67 GHz Intel Xeon E7-8837/256 GB RAM." path-from-xml="mst010f2p"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/33680836/mst010f2p" path-from-xml="mst010f2p" target="_blank">View large</a><a section="33680836" class="download-slide" path-from-xml="mst010f2p" target="">Download slide</a></div></div><div class="caption fig-caption"><p>ITS alignments by different options of MAFFT, displayed on Jalview (<a reveal-id="mst010-B47" data-open="mst010-B47" class="link link-ref link-reveal xref-bibr">Waterhouse et al. 2009</a>). (<em>A</em>, <em>B</em>) Incorrect alignments by the FFT-NS-2 and L-INS-i algorithms, respectively. (<em>C</em>) An incorrect alignment by <span class="monospace">mafft-profile</span>.
 The full-length sequences were aligned with the L-INS-i algorithm and 
then each new sequence was separately added to the full-length 
alignment, using <span class="monospace">mafft-profile</span>. (<em>D</em>) Reasonable alignment by a two-step strategy. The <span class="monospace">––</span><span class="monospace">6merpair </span><span class="monospace">––</span><span class="monospace">addfragments</span> option was used at the second step. (<em>E</em>)
 Reordered version of D; sequences are ordered such that similar 
sequences are placed closely. All calculations were performed using 16 
cores on a Linux PC with 2.67 GHz Intel Xeon E7-8837/256 GB RAM.</p></div></div>                        <a id="33680837" scrollto-destination="33680837"></a>
<div content-id="mst010-T2" class="table-modal" style="display: block;"><div class="table-wrap"><div id="mst010-T2" class="table-wrap-title" data-id="mst010-T2"><div class="label -label"><strong>Table 2</strong>.</div><div class="caption"><p>Comparison of Different Options Using the 16S.B.ALL Data Set (<a reveal-id="mst010-B32" data-open="mst010-B32" class="link link-ref link-reveal xref-bibr">Mirarab et al. 2012</a>).</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Data&nbsp;</th><th>Method&nbsp;</th><th>Accuracy&nbsp;</th><th>CPU Time&nbsp;</th><th>Actual Time<a reveal-id="mst010-TF1" data-open="mst010-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td rowspan="5">Case 1&nbsp;</td><td><span class="monospace">mafft ––multipair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9969&nbsp;</td><td>6.67 days&nbsp;</td><td>18.3 h&nbsp;</td></tr><tr><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9949&nbsp;</td><td>3.76 h&nbsp;</td><td>36.2 min&nbsp;</td></tr><tr><td><span class="monospace">mafft ––localpair ––add  frags existingmsa</span>&nbsp;</td><td>0.9707&nbsp;</td><td>23.4 days<a reveal-id="mst010-TF2" data-open="mst010-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td><td>2.43 days<a reveal-id="mst010-TF2" data-open="mst010-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td></tr><tr><td><span class="monospace">mafft ––6merpair ––add frags existingmsa</span>&nbsp;</td><td>0.9604&nbsp;</td><td>1.32 h&nbsp;</td><td>1.44 h&nbsp;</td></tr><tr><td>profile alignment&nbsp;</td><td>0.2779&nbsp;</td><td>15.5 h&nbsp;</td><td>1.60 h&nbsp;</td></tr><tr><td colspan="5"><hr>&nbsp;</td></tr><tr><td>Case 2&nbsp;</td><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9969&nbsp;</td><td>4.54 h&nbsp;</td><td>33.8 min&nbsp;</td></tr><tr><td colspan="5"><hr>&nbsp;</td></tr><tr><td>Case 3&nbsp;</td><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9949&nbsp;</td><td>1.79 days&nbsp;</td><td>5.91 h&nbsp;</td></tr></tbody></table></div><div content-id="mst010-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>Data&nbsp;</th><th>Method&nbsp;</th><th>Accuracy&nbsp;</th><th>CPU Time&nbsp;</th><th>Actual Time<a reveal-id="mst010-TF1" data-open="mst010-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td rowspan="5">Case 1&nbsp;</td><td><span class="monospace">mafft ––multipair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9969&nbsp;</td><td>6.67 days&nbsp;</td><td>18.3 h&nbsp;</td></tr><tr><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9949&nbsp;</td><td>3.76 h&nbsp;</td><td>36.2 min&nbsp;</td></tr><tr><td><span class="monospace">mafft ––localpair ––add  frags existingmsa</span>&nbsp;</td><td>0.9707&nbsp;</td><td>23.4 days<a reveal-id="mst010-TF2" data-open="mst010-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td><td>2.43 days<a reveal-id="mst010-TF2" data-open="mst010-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td></tr><tr><td><span class="monospace">mafft ––6merpair ––add frags existingmsa</span>&nbsp;</td><td>0.9604&nbsp;</td><td>1.32 h&nbsp;</td><td>1.44 h&nbsp;</td></tr><tr><td>profile alignment&nbsp;</td><td>0.2779&nbsp;</td><td>15.5 h&nbsp;</td><td>1.60 h&nbsp;</td></tr><tr><td colspan="5"><hr>&nbsp;</td></tr><tr><td>Case 2&nbsp;</td><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9969&nbsp;</td><td>4.54 h&nbsp;</td><td>33.8 min&nbsp;</td></tr><tr><td colspan="5"><hr>&nbsp;</td></tr><tr><td>Case 3&nbsp;</td><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9949&nbsp;</td><td>1.79 days&nbsp;</td><td>5.91 h&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="mst010-TF2"><div class="fn"><p>N<span class="small-caps">ote</span>.—The
 estimated alignments were compared with the CRW alignment to measure 
the accuracy (the number of correctly aligned letters/the number of 
aligned letters in the CRW alignment). Calculations were performed on a 
Linux PC with 2.67 GHz Intel Xeon E7-8837/256 GB RAM (for the case 
marked with superscript alphabet “b”), or on a Linux PC with 3.47 GHz 
Intel Xeon X5690/48 GB RAM (for the other cases). Case 1: 13,822 
sequences in the existing alignment × 13,821 fragments; Case 2: 1,000 
sequences in the existing alignment × 138,210 fragments; Case 3: 13,822 
sequences in the existing alignment × 138,210 fragments.</p></div></div><div content-id="mst010-TF3"><div class="fn"><p><sup>a</sup>Wall-clock time with 10 cores. Command-line argument for parallel processing is <span class="monospace">––thread 10</span>.</p></div></div><div content-id="mst010-TF4"><div class="fn"><p><sup>b</sup>Full command-line options are as follows: <span class="monospace">mafft ––localpair ––weighti 0 ––add frags existingmsa</span>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/33680837" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="mst010-T2" class="table-wrap-title" data-id="mst010-T2"><div class="label -label"><strong>Table 2</strong>.</div><div class="caption"><p>Comparison of Different Options Using the 16S.B.ALL Data Set (<a reveal-id="mst010-B32" data-open="mst010-B32" class="link link-ref link-reveal xref-bibr">Mirarab et al. 2012</a>).</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Data&nbsp;</th><th>Method&nbsp;</th><th>Accuracy&nbsp;</th><th>CPU Time&nbsp;</th><th>Actual Time<a reveal-id="mst010-TF1" data-open="mst010-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td rowspan="5">Case 1&nbsp;</td><td><span class="monospace">mafft ––multipair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9969&nbsp;</td><td>6.67 days&nbsp;</td><td>18.3 h&nbsp;</td></tr><tr><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9949&nbsp;</td><td>3.76 h&nbsp;</td><td>36.2 min&nbsp;</td></tr><tr><td><span class="monospace">mafft ––localpair ––add  frags existingmsa</span>&nbsp;</td><td>0.9707&nbsp;</td><td>23.4 days<a reveal-id="mst010-TF2" data-open="mst010-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td><td>2.43 days<a reveal-id="mst010-TF2" data-open="mst010-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td></tr><tr><td><span class="monospace">mafft ––6merpair ––add frags existingmsa</span>&nbsp;</td><td>0.9604&nbsp;</td><td>1.32 h&nbsp;</td><td>1.44 h&nbsp;</td></tr><tr><td>profile alignment&nbsp;</td><td>0.2779&nbsp;</td><td>15.5 h&nbsp;</td><td>1.60 h&nbsp;</td></tr><tr><td colspan="5"><hr>&nbsp;</td></tr><tr><td>Case 2&nbsp;</td><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9969&nbsp;</td><td>4.54 h&nbsp;</td><td>33.8 min&nbsp;</td></tr><tr><td colspan="5"><hr>&nbsp;</td></tr><tr><td>Case 3&nbsp;</td><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9949&nbsp;</td><td>1.79 days&nbsp;</td><td>5.91 h&nbsp;</td></tr></tbody></table></div><div content-id="mst010-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>Data&nbsp;</th><th>Method&nbsp;</th><th>Accuracy&nbsp;</th><th>CPU Time&nbsp;</th><th>Actual Time<a reveal-id="mst010-TF1" data-open="mst010-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td rowspan="5">Case 1&nbsp;</td><td><span class="monospace">mafft ––multipair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9969&nbsp;</td><td>6.67 days&nbsp;</td><td>18.3 h&nbsp;</td></tr><tr><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9949&nbsp;</td><td>3.76 h&nbsp;</td><td>36.2 min&nbsp;</td></tr><tr><td><span class="monospace">mafft ––localpair ––add  frags existingmsa</span>&nbsp;</td><td>0.9707&nbsp;</td><td>23.4 days<a reveal-id="mst010-TF2" data-open="mst010-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td><td>2.43 days<a reveal-id="mst010-TF2" data-open="mst010-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td></tr><tr><td><span class="monospace">mafft ––6merpair ––add frags existingmsa</span>&nbsp;</td><td>0.9604&nbsp;</td><td>1.32 h&nbsp;</td><td>1.44 h&nbsp;</td></tr><tr><td>profile alignment&nbsp;</td><td>0.2779&nbsp;</td><td>15.5 h&nbsp;</td><td>1.60 h&nbsp;</td></tr><tr><td colspan="5"><hr>&nbsp;</td></tr><tr><td>Case 2&nbsp;</td><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9969&nbsp;</td><td>4.54 h&nbsp;</td><td>33.8 min&nbsp;</td></tr><tr><td colspan="5"><hr>&nbsp;</td></tr><tr><td>Case 3&nbsp;</td><td><span class="monospace">mafft ––6merpair ––addfragments frags existingmsa</span>&nbsp;</td><td>0.9949&nbsp;</td><td>1.79 days&nbsp;</td><td>5.91 h&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="mst010-TF2"><div class="fn"><p>N<span class="small-caps">ote</span>.—The
 estimated alignments were compared with the CRW alignment to measure 
the accuracy (the number of correctly aligned letters/the number of 
aligned letters in the CRW alignment). Calculations were performed on a 
Linux PC with 2.67 GHz Intel Xeon E7-8837/256 GB RAM (for the case 
marked with superscript alphabet “b”), or on a Linux PC with 3.47 GHz 
Intel Xeon X5690/48 GB RAM (for the other cases). Case 1: 13,822 
sequences in the existing alignment × 13,821 fragments; Case 2: 1,000 
sequences in the existing alignment × 138,210 fragments; Case 3: 13,822 
sequences in the existing alignment × 138,210 fragments.</p></div></div><div content-id="mst010-TF3"><div class="fn"><p><sup>a</sup>Wall-clock time with 10 cores. Command-line argument for parallel processing is <span class="monospace">––thread 10</span>.</p></div></div><div content-id="mst010-TF4"><div class="fn"><p><sup>b</sup>Full command-line options are as follows: <span class="monospace">mafft ––localpair ––weighti 0 ––add frags existingmsa</span>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/33680837" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination="33680838" id="33680838" class="section-title">The <span class="monospace">––</span><span class="monospace">a</span><span class="monospace">dd</span> and <span class="monospace">––</span><span class="monospace">a</span><span class="monospace">ddfragments</span> Options</h3>
<p>To overcome this limitation of profile alignment, in 2010, we implemented an option, <span class="monospace">––</span><span class="monospace">add</span>,
 to add unaligned sequences to an existing MSA. This option assumes that
 each new sequence was derived from a branch in the tree of an existing 
alignment, as illustrated in <a reveal-id="mst010-F1" data-open="mst010-F1" class="link link-reveal link-table xref-fig">figure 1</a><em>D</em>.
 This option works almost identically to the standard progressive 
method, except that the alignment calculation is skipped at the nodes 
whose children are all in the existing alignment.</p><p>Along with 
popularization of second-generation sequencers, we sometimes need to 
align short reads to an existing alignment. Several tools (<a reveal-id="mst010-B4" data-open="mst010-B4" class="link link-ref link-reveal xref-bibr">Berger and Stamatakis 2011</a>; <a reveal-id="mst010-B30" data-open="mst010-B30" class="link link-ref link-reveal xref-bibr">Löytynoja et al. 2012</a>; <a reveal-id="mst010-B44" data-open="mst010-B44" class="link link-ref link-reveal xref-bibr">Sun and Buhler 2012</a>) for this purpose were developed between 2011 and 2012. A limitation of the <span class="monospace">––</span><span class="monospace">add</span> option in MAFFT for this purpose was pointed out in <a reveal-id="mst010-B30" data-open="mst010-B30" class="link link-ref link-reveal xref-bibr">Löytynoja et al. (2012)</a>. Thus, we implemented a new option, <span class="monospace">––</span><span class="monospace">addfragments</span>, which does not consider the relationship among the sequences to be added. Details of the <span class="monospace">––</span><span class="monospace">add</span> and <span class="monospace">––</span><span class="monospace">addfragments</span> options are described in <a reveal-id="mst010-B20" data-open="mst010-B20" class="link link-ref link-reveal xref-bibr">Katoh and Frith (2012)</a>.</p>                            <h3 scrollto-destination="33680841" id="33680841" class="section-title">Test Case 1: Fungal Internal Transcribed Spacers Sequences</h3>
<p>Here, we discuss how the <span class="monospace">––</span><span class="monospace">addfragments</span>
 option works, using an actual case. Internal transcribed spacers (ITSs)
 are spacer regions located between structural ribosomal RNAs. The 
structure of the rDNA region in a eukaryotic genome is 18S − ITS1 − 5.8S
 − ITS2 − 28S. Here, we use a data set consisting of ITS1 and ITS2 
sequences obtained from environmental samples (Chen W, personal 
communication). Each sequence has either ITS1 or ITS2 region only, 
extracted from 454 pyrosequencing data using FungalITSextractor (<a reveal-id="mst010-B48" data-open="mst010-B48" class="link link-ref link-reveal xref-bibr">Nilsson et al. 2010</a>). In addition, several fungal genomic sequences that fully cover ITS1 + 5.8S rRNA + ITS2 are available from public databases.</p><p>Suppose
 a situation where we need an MSA of approximately 300 full-length 
sequences and approximately 5,000 ITS1 or ITS2 sequences. One possible 
solution is to build an entire MSA at once. The result of the default 
option (FFT-NS-2) of MAFFT is obviously incorrect, as shown in <a reveal-id="mst010-F2" data-open="mst010-F2" class="link link-reveal link-table xref-fig">figure 2</a><em>A</em>.
 ITS1 and ITS2 regions are forced to be aligned to each other. Even if a
 more computationally expensive (and usually more accurate) method, 
L-INS-i, is applied (CPU time = 98 h), the alignment is still obviously 
incorrect (<a reveal-id="mst010-F2" data-open="mst010-F2" class="link link-reveal link-table xref-fig">fig. 2</a><em>B</em>).</p><p>Two-step
 strategies can solve this type of problem. That is, a set of 
full-length sequences taken from databases are first aligned to build a 
backbone MSA, and then the new ITS1 and ITS2 sequences are added into 
this backbone MSA, using the <span class="monospace">––</span><span class="monospace">addfragments</span> option. </p><blockquote><p>Step 1: <span class="monospace">mafft <span class="monospace">--</span></span><span class="monospace">auto full_length_sequences &gt;\ backbone_msa</span></p><p>Step 2: <span class="monospace">mafft <span class="monospace">--</span></span><span class="monospace">addfragments \ new_sequences backbone_msa &gt; output</span></p></blockquote> The second command is equivalent to <blockquote><p><span class="monospace">mafft <span class="monospace">--</span></span><span class="monospace">multipair <span class="monospace">--</span></span><span class="monospace">addfragments \ new_sequences backbone_msa &gt; output</span></p></blockquote>
 in which Dynamic Programming (DP) is used to compare the distances 
between every new sequence and every sequence in the backbone MSA (<span class="monospace">––</span><span class="monospace">multipair</span> is selected by default). <blockquote><p><span class="monospace">mafft <span class="monospace">--</span></span><span class="monospace">6merpair <span class="monospace">--</span></span><span class="monospace">addfragments \ new_sequences backbone_msa &gt; output</span></p></blockquote> where distances are rapidly estimated using the number of shared 6mers, instead of DP.<p></p><p>The result of the latter option (<span class="monospace">––</span><span class="monospace">6merpair </span><span class="monospace">––</span><span class="monospace">addfragments</span>) is shown in <a reveal-id="mst010-F2" data-open="mst010-F2" class="link link-reveal link-table xref-fig">fig. 2</a><em>D</em> and <em>E</em>. The difference between <em>D</em> and <em>E</em> is just in the order of sequences; the sequences were reordered according to similarity using the <span class="monospace">––</span><span class="monospace">reorder</span> option in <em>E</em>.
 In this alignment, ITS1 and ITS2 are clearly separated and aligned to 
appropriate positions in the full-length alignment. Moreover, this 
strategy is computationally much less expensive (CPU time = 15 min 
[first step] + 1.5 min [second step]) than the full application of 
L-INS-i (CPU time = 98 h). The former option (<span class="monospace">––multipair </span><span class="monospace">––</span><span class="monospace">addfragments</span>) also returns a similar result to the latter (<span class="monospace">––</span><span class="monospace">6merpair)</span> but is slower (CPU time = 48.6 min [second step]).</p><p>This
 case suggests that it is crucial to select a strategy appropriate to 
the problem of interest. The most time-consuming method, L-INS-i, is not
 always the most accurate one. The difficulty of this problem for 
standard approaches comes from the fact that ITS1 sequences and ITS2 
sequences are not homologous to each other and most pairwise alignments 
are impossible. Because of these nonhomologous pairs, the distance 
matrix used for the guide tree calculation is not additive; the 
distances between ITS1 and full-length sequences and those between ITS2 
and full-length sequences are close to zero, whereas the distances 
between ITS1 and ITS2 are quite large. In this situation, it is 
difficult for normal distance-based tree-building methods to give a 
reasonable tree. Moreover, in the alignment step, the objective function
 of the L-INS-i is affected by inappropriate pairwise alignment scores 
between ITS1 and ITS2. Such problems can be avoided by just ignoring the
 relationship between ITS1 and ITS2, as done in the <span class="monospace">––</span><span class="monospace">addfragments</span> option.</p><p>In addition, a result of the second type of misuse of <span class="monospace">mafft-profile</span> (discussed earlier) is shown in <a reveal-id="mst010-F2" data-open="mst010-F2" class="link link-reveal link-table xref-fig">figure 2</a><em>C</em>. Some new sequences are correctly aligned but others are obviously incorrectly aligned (note that the order of sequences in <a reveal-id="mst010-F2" data-open="mst010-F2" class="link link-reveal link-table xref-fig">fig. 2</a><em>C</em> is identical that in <a reveal-id="mst010-F2" data-open="mst010-F2" class="link link-reveal link-table xref-fig">fig. 2</a><em>D</em>). These misalignments are due to an incorrect assumption on phylogenetic placement of new sequences shown in <a reveal-id="mst010-F1" data-open="mst010-F1" class="link link-reveal link-table xref-fig">figure 1</a><em>C</em>.</p>                            <h3 scrollto-destination="33680848" id="33680848" class="section-title">Test Case 2: Bacterial SSU rRNA</h3>
<p>Another case is the 16S.B.ALL data set by <a reveal-id="mst010-B32" data-open="mst010-B32" class="link link-ref link-reveal xref-bibr">Mirarab et al. (2012)</a>. It consists of an MSA of 13,822 bacterial SSU rRNA sequences, taken from the Gutell Comparative RNA Website (CRW) (<a reveal-id="mst010-B7" data-open="mst010-B7" class="link link-ref link-reveal xref-bibr">Cannone et al. 2002</a>)
 and 138,210 fragmentary sequences, which are originally included in the
 CRW alignment but ungapped and artificially truncated. In <a reveal-id="mst010-B23" data-open="mst010-B23" class="link link-ref link-reveal xref-bibr">Katoh and Standley (2013)</a>, we used a subset (13,821 fragmentary sequences) prepared by <a reveal-id="mst010-B32" data-open="mst010-B32" class="link link-ref link-reveal xref-bibr">Mirarab et al. (2012)</a>.
 In addition to this subset, here we use the full data set (138,210 
fragmentary sequences), to examine the scalability. Suppose a situation 
where we already have a manually curated (or backbone) MSA and a newly 
determined set of many fragmentary sequences in a metagenomics project, 
and we need an entire MSA of them.</p><p>The first four lines in <a reveal-id="mst010-T2" data-open="mst010-T2" class="link link-reveal link-table xref-fig">table 2</a>
 (case 1) show the performances of various options for such an analysis,
 with a relatively small data set (13,822 sequences in the existing 
alignment × 13,821 fragments). The accuracy of each resulting MSA was 
evaluated by comparing the MSA with the original CRW alignment. CPU time
 and wall-clock time for each method are also listed. As the sequences 
in this data set are highly conserved, the difference in accuracy 
between the default (<span class="monospace">––</span><span class="monospace">multipair </span><span class="monospace">––</span><span class="monospace">addfragments</span>) and the faster option (<span class="monospace">––</span><span class="monospace">6merpair </span><span class="monospace">––</span><span class="monospace">addfragments</span>) is small.</p><p>Again,
 the tradeoff between accuracy and speed does not hold. The application 
of a computationally expensive method based on L-INS-1 (<span class="monospace">––</span><span class="monospace">localpair </span><span class="monospace">––</span><span class="monospace">add</span>)
 has no advantage, because the extra computational time is spent on the 
comparison of nonoverlapping fragmentary sequences, which have no 
reasonable solutions.</p><p>The “profile alignment” line in <a reveal-id="mst010-T2" data-open="mst010-T2" class="link link-reveal link-table xref-fig">table 2</a>
 shows results of the second type of misuse of profile alignment 
(discussed earlier), in which the given alignment is converted to a 
profile and each new sequence is separately aligned to the profile. This
 result clearly indicates that the application of profile alignment must
 be avoided in this case, too. Users do not need to be too worried about
 this misuse, because this calculation is disabled in MAFFT unless the 
user modifies the code or writes a wrapper script.</p><p>The last two lines in <a reveal-id="mst010-T2" data-open="mst010-T2" class="link link-reveal link-table xref-fig">table 2</a> (Cases 2 and 3) show the performance of the fast option (<span class="monospace">––</span><span class="monospace">6merpair </span><span class="monospace">––</span><span class="monospace">addfragments</span>)
 for a larger number (138,210) of fragmentary sequences. The number of 
sequences in the existing alignment is 1,000 and 13,822 in cases 2 and 
3, respectively. This fast option gives a reasonable quality of result 
in a reasonable computing time. At present, the default option (<span class="monospace">––</span><span class="monospace">multipair </span><span class="monospace">––</span><span class="monospace">addfragments</span>) cannot handle cases 2 and 3. Simulation-based benchmarks in <a reveal-id="mst010-B20" data-open="mst010-B20" class="link link-ref link-reveal xref-bibr">Katoh and Frith (2012)</a>
 suggested that, for cases with more divergent sequences, the accuracy 
of the default option is higher than that of the fast option. We are now
 trying to improve the scalability of the default option.</p>                            <h2 scrollto-destination="33680858" id="33680858" class="section-title">Parallelization</h2>
<p>MAFFT version 7 has an option for parallel processing, <span class="monospace">––</span><span class="monospace">thread</span> (<a reveal-id="mst010-B27" data-open="mst010-B27" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2010</a>).
 This feature is currently supported on Mac OS X in addition to Linux, 
but not yet supported on Windows for technical reasons. With the <span class="monospace">––</span><span class="monospace">thread</span><em>n</em> option, it runs in parallel with <em>n</em> threads. The number of threads can be automatically determined by <span class="monospace">––</span><span class="monospace">thread </span><span class="monospace">–</span><span class="monospace">1</span>.
 This option sets the number of threads as the number of physical cores,
 not the number of logical cores in Intel’s hyperthreaded CPUs.</p><p>For
 progressive methods, the result with the multithread version is 
identical to that of the serial processing version. However, for 
iterative refinement methods, the results are not always identical. We 
confirmed that the accuracy of the parallel version in this case is 
comparable with that of the serial version (<a reveal-id="mst010-B27" data-open="mst010-B27" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2010</a>). The efficiency of parallelization depends on the alignment strategy. In the case of the <span class="monospace">––</span><span class="monospace">addfragments</span> option, the efficiency is acceptably high as shown in <a reveal-id="mst010-T2" data-open="mst010-T2" class="link link-reveal link-table xref-fig">table 2</a>.</p>                            <h2 scrollto-destination="33680861" id="33680861" class="section-title">Utility Options</h2>
<p>MAFFT version 7 also has several enhanced options for peripheral functions.</p>                            <h3 scrollto-destination="33680863" id="33680863" class="section-title">Estimating the Direction of DNA Sequences</h3>
<p>In the case of nucleotide alignments, if some of input sequences have
 an incorrect direction relative to the other sequences, the directions 
can be automatically adjusted by the <span class="monospace">––</span><span class="monospace">adjustdirection</span> option. We use an algorithm with a time complexity of <span class="inline-formula no-formula-id"><img class="content-image" src="" alt="forumla" path-from-xml="mst010i1"></span>, where <em>n</em> is the number of sequences (<a reveal-id="mst010-B23" data-open="mst010-B23" class="link link-ref link-reveal xref-bibr">Katoh and Standley 2013</a>).
 It is slow when the distances are calculated with DP. However, when the
 distance is rapidly calculated based on the number of shared 6mers, the
 speed is reasonable. This option is also available on the web version, 
with the “Adjust direction” button.</p><p>MAFFT cannot handle more 
complicated sequences with genomic rearrangements (translocations, 
duplications, or inversions). The web version of MAFFT displays dot 
plots between the first sequence and the remaining sequences, using the 
LAST local alignment program (<a reveal-id="mst010-B28" data-open="mst010-B28" class="link link-ref link-reveal xref-bibr">Kiełbasa et al. 2011</a>),
 for every nucleotide alignment run. By viewing the dot plots, a user 
can easily check for genomic rearrangements and the directions of input 
sequences. See <a reveal-id="mst010-B23" data-open="mst010-B23" class="link link-ref link-reveal xref-bibr">Katoh and Standley (2013)</a> for details and an example.</p>                            <h3 scrollto-destination="33680866" id="33680866" class="section-title">Input/Output</h3>
<p>MAFFT version 7 has several enhancements in the flexibility of 
input/output. The following options related to input/output are 
available and can be combined with other options. </p><ul class="list-simple"><li><p><span class="monospace">––</span><span class="monospace">anysymbol</span> If the input data include unusual letters, like U, J, etc., (in the case of protein data), MAFFT stops by default. The <span class="monospace">––</span><span class="monospace">anysymbol</span> option allows these letters and nonalphabetical letters.</p></li><li><p><span class="monospace">––</span><span class="monospace">preservecase</span><span class="monospace"> </span>By
 default, amino acid sequences are converted to upper case and 
nucleotide sequences are converted to lower case. This behavior can be 
changed by using the <span class="monospace">––</span><span class="monospace">preservecase</span> option.</p></li><li><p><span class="monospace">––</span><span class="monospace">reorder</span><span class="monospace"> </span>The
 order of sequences is the same as the input sequences by default, but 
the sequences can be sorted according to similarity to each other by the
 <span class="monospace">––</span><span class="monospace">reorder</span> option.</p></li><li><p><span class="monospace">––</span><span class="monospace">phylipout</span><span class="monospace">a</span>nd <span class="monospace">––<span class="monospace">clustalout</span></span><span class="monospace"> </span>The output format is multi-fasta by default, but the phylip (interleaved) format and the clustal format can be selected.</p></li></ul>                            <h3 scrollto-destination="33680869" id="33680869" class="section-title">Guide Tree and Phylogenetic Positions of New Sequences</h3>
<p>Users can check the guide tree by using the <span class="monospace">––</span><span class="monospace">treeout</span> option. In the case of <span class="monospace">––</span><span class="monospace">addfragments</span>,
 the estimated phylogenetic positions of new sequences are shown 
together with the estimated tree of the existing alignment. The 
alignment calculation is performed based on this phylogenetic 
estimation. It is also possible to compute such phylogenetic information
 only, without alignment, by the <span class="monospace">––</span><span class="monospace">retree 0</span> option. An example of output is shown in <a reveal-id="mst010-F3" data-open="mst010-F3" class="link link-reveal link-table xref-fig">Figure 3</a><em>A</em>. </p>                        <a id="33680871" scrollto-destination="33680871"></a>
<div id="mst010-F3" data-id="mst010-F3" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 3.</div><div class="graphic-wrap"><img class="content-image" src="" alt="(A) A part of output of the ––treeout option showing the phylogenetic positions of new sequences (new#) in the tree of the existing alignment (backbone#), estimated before the alignment calculation. This file also shows a Newick format tree of the existing alignment (not shown in this figure). For each new sequence, the nearest sequence in the existing alignment (nearest sequence), approximate distance to the nearest sequence (approximate distance), and the members of the sister group (sister group) are shown. (B) Graphical representation of (A)." path-from-xml="mst010f3p"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/33680871/mst010f3p" path-from-xml="mst010f3p" target="_blank">View large</a><a section="33680871" class="download-slide" path-from-xml="mst010f3p" target="">Download slide</a></div></div><div class="caption fig-caption"><p>(<em>A</em>) A part of output of the <span class="monospace">––</span><span class="monospace">treeout</span> option showing the phylogenetic positions of new sequences (<span class="monospace">new#</span>) in the tree of the existing alignment (<span class="monospace">backbone#</span>),
 estimated before the alignment calculation. This file also shows a 
Newick format tree of the existing alignment (not shown in this figure).
 For each new sequence, the nearest sequence in the existing alignment (<span class="monospace">nearest sequence</span>), approximate distance to the nearest sequence (<span class="monospace">approximate distance</span>), and the members of the sister group (<span class="monospace">sister group</span>) are shown. (<em>B</em>) Graphical representation of (<em>A</em>).</p></div></div><div content-id="mst010-F3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">ig</span>. 3.</div><div class="graphic-wrap"><img class="content-image" src="" alt="(A) A part of output of the ––treeout option showing the phylogenetic positions of new sequences (new#) in the tree of the existing alignment (backbone#), estimated before the alignment calculation. This file also shows a Newick format tree of the existing alignment (not shown in this figure). For each new sequence, the nearest sequence in the existing alignment (nearest sequence), approximate distance to the nearest sequence (approximate distance), and the members of the sister group (sister group) are shown. (B) Graphical representation of (A)." path-from-xml="mst010f3p"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/33680871/mst010f3p" path-from-xml="mst010f3p" target="_blank">View large</a><a section="33680871" class="download-slide" path-from-xml="mst010f3p" target="">Download slide</a></div></div><div class="caption fig-caption"><p>(<em>A</em>) A part of output of the <span class="monospace">––</span><span class="monospace">treeout</span> option showing the phylogenetic positions of new sequences (<span class="monospace">new#</span>) in the tree of the existing alignment (<span class="monospace">backbone#</span>),
 estimated before the alignment calculation. This file also shows a 
Newick format tree of the existing alignment (not shown in this figure).
 For each new sequence, the nearest sequence in the existing alignment (<span class="monospace">nearest sequence</span>), approximate distance to the nearest sequence (<span class="monospace">approximate distance</span>), and the members of the sister group (<span class="monospace">sister group</span>) are shown. (<em>B</em>) Graphical representation of (<em>A</em>).</p></div></div><p>Note that this phylogenetic information is roughly estimated <em>before</em> the MSA calculation, not based on the MSA. Especially, with the fast option, <span class="monospace">––</span><span class="monospace">6merpair</span>, the estimation is very rough. With the <span class="monospace">––</span><span class="monospace">multipair</span>
 option (default), the estimation is expected to be better, but it needs
 a relatively long computational time. For more rigorous estimation of 
phylogenetic positions of new sequences, specially designed tools, such 
as pplacer (<a reveal-id="mst010-B31" data-open="mst010-B31" class="link link-ref link-reveal xref-bibr">Matsen et al. 2010</a>), PaPaRa (<a reveal-id="mst010-B4" data-open="mst010-B4" class="link link-ref link-reveal xref-bibr">Berger and Stamatakis 2011</a>), PAGAN (<a reveal-id="mst010-B30" data-open="mst010-B30" class="link link-ref link-reveal xref-bibr">Löytynoja et al. 2012</a>), SEPP (<a reveal-id="mst010-B32" data-open="mst010-B32" class="link link-ref link-reveal xref-bibr">Mirarab et al. 2012</a>), or combinations of them including MAFFT, should be tried.</p>                            <h2 scrollto-destination="33680873" id="33680873" class="section-title">Parameters</h2>
<p>For amino acid alignment, MAFFT uses the BLOSUM62 matrix by default. 
For nucleotide alignment, a 200PAM log-odds scoring matrix is generated 
assuming that the transition rate is twice the transversion rate. These 
matrices are suitable for aligning distantly related sequences. We 
selected these default parameters based on an expectation that, if the 
program works well for difficult (distantly related) cases, it should 
also work well for easy cases.</p><p>It is unclear whether this expectation is always correct. For example, in a benchmark using simulated protein sequences (<a reveal-id="mst010-B30" data-open="mst010-B30" class="link link-ref link-reveal xref-bibr">Löytynoja et al. 2012</a>) generated by INDELiBLE (<a reveal-id="mst010-B11" data-open="mst010-B11" class="link link-ref link-reveal xref-bibr">Fletcher and Yang 2009</a>), when we tested a more stringent scoring matrix, JTT 1PAM (<a reveal-id="mst010-B17" data-open="mst010-B17" class="link link-ref link-reveal xref-bibr">Jones et al. 1992</a>)
 with weaker gap penalties than the default, the benchmark scores were 
considerably improved. Despite this observation, we consistently used 
the default parameters in the benchmark in <a reveal-id="mst010-B20" data-open="mst010-B20" class="link link-ref link-reveal xref-bibr">Katoh and Frith (2012)</a>,
 because it does not make sense to arbitrarily adjust parameters to a 
simulation setting. This observation suggests that the current default 
parameters of MAFFT might not be very suitable for aligning closely 
related sequences. However, this idea must be checked using actual 
biological sequences.</p><p>User can select different scoring matrices other than the default. For amino acid alignment, <span class="monospace">––</span><span class="monospace">bl 45</span>, <span class="monospace">––</span><span class="monospace">bl 62</span>, <span class="monospace">––</span><span class="monospace">bl 80</span>, <span class="monospace">––</span><span class="monospace">jtt N</span>, and <span class="monospace">––</span><span class="monospace">tm N</span> are accepted, where <span class="monospace">N</span> is an expected evolutionary distance among input sequences. The <span class="monospace">––</span><span class="monospace">bl</span>, <span class="monospace">––</span><span class="monospace">jtt</span>, and <span class="monospace">––</span><span class="monospace">tm</span> options mean BLOSUM (Henikoff S and Henikoff JG <a reveal-id="mst010-B15" data-open="mst010-B15" class="link link-ref link-reveal xref-bibr">1992</a>), JTT (<a reveal-id="mst010-B17" data-open="mst010-B17" class="link link-ref link-reveal xref-bibr">Jones et al. 1992</a>), and a transmembrane model (<a reveal-id="mst010-B18" data-open="mst010-B18" class="link link-ref link-reveal xref-bibr">Jones et al. 1994</a>), respectively. A user-defined scoring matrix can also be accepted, by <span class="monospace">––</span><span class="monospace">aamatrix</span>. For nucleotide alignments, <span class="monospace">––</span><span class="monospace">kimura N</span> is accepted, where <span class="monospace">N</span> is an expected evolutionary distance among input sequences. Gap penalties can be adjusted by <span class="monospace">––</span><span class="monospace">op</span>, <span class="monospace">––</span><span class="monospace">exp</span>, <span class="monospace">––</span><span class="monospace">lop</span>, and <span class="monospace">––</span><span class="monospace">lexp</span> options.</p><p>One
 possible extension is to use different scoring matrices and gap 
penalties for different sequence pairs according to the divergence 
level, like ClustalW (<a reveal-id="mst010-B46" data-open="mst010-B46" class="link link-ref link-reveal xref-bibr">Thompson et al. 1994</a>).
 More studies using actual sequence data will be necessary before 
implementing this extension. It will also be necessary to adjust gap 
penalties, preferably based on a realistic evolutionary model of 
insertions and deletions.</p>                            <h2 scrollto-destination="33680878" id="33680878" class="section-title">Use of Structural Information</h2>
<p>We have discussed possible improvements in MSAs of closely related 
sequences in the previous section. MSA of distantly related sequences is
 still a challenging problem.</p>                            <h3 scrollto-destination="33680880" id="33680880" class="section-title">Test Case 3: PIN Domain</h3>
<p><a reveal-id="mst010-F4" data-open="mst010-F4" class="link link-reveal link-table xref-fig">Figure 4</a>
 shows a typical limitation of sequence level alignment for a highly 
divergent set of three PIN-domain containing proteins: human regnase-1, 
VPA0982 from <em>Vibrio parahaemolyticus</em>, nuclease domain of taq polymerase from <em>Thermus aquaticus</em>. These three proteins share a magnesium-binding site composed of three conserved aspartic acids. <a reveal-id="mst010-F4" data-open="mst010-F4" class="link link-reveal link-table xref-fig">Figure 4</a><em>A</em>
 shows a superposition of the three structures (Protein Databank 
identifiers 3v33, 2qip, and 1taq, respectively). The middle aspartic 
acid is indicated by sphere-representation, colored red. In <a reveal-id="mst010-F4" data-open="mst010-F4" class="link link-reveal link-table xref-fig">Figure 4</a><em>B</em>, a typical MSA (by MAFFT-L-INS-i) is shown wherein the middle aspartic acid position is misaligned. In <a reveal-id="mst010-F4" data-open="mst010-F4" class="link link-reveal link-table xref-fig">Figure 4</a><em>C</em>, a structure-informed MSA (described below), with the middle aspartic acid correctly aligned, is shown. </p>                        <a id="33680882" scrollto-destination="33680882"></a>
<div id="mst010-F4" data-id="mst010-F4" class="fig fig-section"><div class="label fig-label"><strong>F<span class="small-caps">ig</span>. 4</strong>.</div><div class="graphic-wrap"><img class="content-image" src="" alt="(A) Superposition of 3v33, 2qip, and 1taq structures visualized by PyMOL (Schrödinger LLC 2010). (B) MAFFT-L-INS-i sequence alignment displayed on jalview (Waterhouse et al. 2009). Misaligned Ds are highlighed in red. (C) Structure-informed MSA with correctly aligned Ds; Alpha helices and beta sheets are shown in blue and yellow, respectively, in (A–C)." path-from-xml="mst010f4p"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/33680882/mst010f4p" path-from-xml="mst010f4p" target="_blank">View large</a><a section="33680882" class="download-slide" path-from-xml="mst010f4p" target="">Download slide</a></div></div><div class="caption fig-caption"><p>(<em>A</em>) Superposition of 3v33, 2qip, and 1taq structures visualized by PyMOL (<a reveal-id="mst010-B39" data-open="mst010-B39" class="link link-ref link-reveal xref-bibr">Schrödinger LLC 2010</a>). (<em>B</em>) MAFFT-L-INS-i sequence alignment displayed on jalview (<a reveal-id="mst010-B47" data-open="mst010-B47" class="link link-ref link-reveal xref-bibr">Waterhouse et al. 2009</a>). Misaligned Ds are highlighed in red. (<em>C</em>)
 Structure-informed MSA with correctly aligned Ds; Alpha helices and 
beta sheets are shown in blue and yellow, respectively, in (<em>A–C</em>).</p></div></div><div content-id="mst010-F4" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label"><strong>F<span class="small-caps">ig</span>. 4</strong>.</div><div class="graphic-wrap"><img class="content-image" src="" alt="(A) Superposition of 3v33, 2qip, and 1taq structures visualized by PyMOL (Schrödinger LLC 2010). (B) MAFFT-L-INS-i sequence alignment displayed on jalview (Waterhouse et al. 2009). Misaligned Ds are highlighed in red. (C) Structure-informed MSA with correctly aligned Ds; Alpha helices and beta sheets are shown in blue and yellow, respectively, in (A–C)." path-from-xml="mst010f4p"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/33680882/mst010f4p" path-from-xml="mst010f4p" target="_blank">View large</a><a section="33680882" class="download-slide" path-from-xml="mst010f4p" target="">Download slide</a></div></div><div class="caption fig-caption"><p>(<em>A</em>) Superposition of 3v33, 2qip, and 1taq structures visualized by PyMOL (<a reveal-id="mst010-B39" data-open="mst010-B39" class="link link-ref link-reveal xref-bibr">Schrödinger LLC 2010</a>). (<em>B</em>) MAFFT-L-INS-i sequence alignment displayed on jalview (<a reveal-id="mst010-B47" data-open="mst010-B47" class="link link-ref link-reveal xref-bibr">Waterhouse et al. 2009</a>). Misaligned Ds are highlighed in red. (<em>C</em>)
 Structure-informed MSA with correctly aligned Ds; Alpha helices and 
beta sheets are shown in blue and yellow, respectively, in (<em>A–C</em>).</p></div></div>                            <h3 scrollto-destination="33680883" id="33680883" class="section-title">Strategy for Integrating Structural Alignments and MAFFT</h3>
<p>It has long been known that structural information can be used to 
improve MSA calculations. This was the basis of the 3D Coffee program (<a reveal-id="mst010-B35" data-open="mst010-B35" class="link link-ref link-reveal xref-bibr">O’Sullivan et al. 2004</a>), and later the PROMALS3D package (<a reveal-id="mst010-B36" data-open="mst010-B36" class="link link-ref link-reveal xref-bibr">Pei et al. 2008</a>).
 Here, we address incorporation of protein structural information in 
MAFFT-based MSA construction. There are both conceptual issues and 
technical issues that complicate the process. Conceptually, we have to 
define structural similarity in such a way that it can easily be used in
 sequence alignments. We discuss our approach to this problem below in 
the context of integrating MAFFT with the structural alignment program 
ASH (<a reveal-id="mst010-B43" data-open="mst010-B43" class="link link-ref link-reveal xref-bibr">Standley et al. 2004</a>, <a reveal-id="mst010-B42" data-open="mst010-B42" class="link link-ref link-reveal xref-bibr">2007</a>).
 On the technical level, structural information complicates matters 
simply because protein structures contain more information and more 
noise than sequence information.</p><div class="block-child-p">Here, we 
focus on one essential feature of ASH: the equivalence score that is 
used to define structural similarity. A particular element in the 
structural similarity matrix takes the form of a Gaussian-shaped 
function of the inter-residue distance <a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" id="mst010um1"><img class="content-image" src="" alt="formula" path-from-xml="mst010um1"></a></div></div></div> where <em>d<sub>ij</sub></em> is the distance between two alpha carbons <em>i</em> and <em>j</em> in the two input structures and <em>d</em><sub>0</sub> is a parameter that defines tolerance in the score. The default behavior is to set <em>d</em><sub>0</sub> to 4 Å. The goal of ASH is to maximize the sum of e<em><sub>ij</sub></em>
 over aligned residues. The residue-level equivalences, which form the 
basis of all ASH alignments, provide a convenient route for combining 
MAFFT and ASH. We can, for example, set a threshold value of e<em><sub>ij</sub></em>
 and incorporate highly confident parts of the alignment into MAFFT to 
“seed” the MSA calculation. If we consider the case of the three PIN 
domain-containing structures in <a reveal-id="mst010-F4" data-open="mst010-F4" class="link link-reveal link-table xref-fig">Figure 4</a>,
 we can first compute structural alignments for the three unique pairs 
using ASH (ash_3v33A-2qipA, ash_3v33A-1taqA, and ash_2qipA-1taqA). If we
 set a threshold for residue equivalence at <span class="inline-formula no-formula-id"><img class="content-image" src="" alt="forumla" path-from-xml="mst010i2"></span>0.5, we can then combine the equivalence-filtered alignments into MAFFT using the seed option (<a reveal-id="mst010-B19" data-open="mst010-B19" class="link link-ref link-reveal xref-bibr">Katoh et al. 2009</a>): <blockquote><p><span class="monospace">mafft-linsi <span class="monospace">--</span></span><span class="monospace">seed ash_3v33A-2qipA <span class="monospace">\ --</span></span><span class="monospace">seed ash_3v33A-1taqA</span><span class="monospace">\</span></p><p><span class="monospace">--</span><span class="monospace">seed ash_2qipA-1taqA \ sequences <span class="monospace">&gt;</span></span><span class="monospace"> output</span></p></blockquote>
 Because the sequence identities between the aligned structures are low,
 we see an improvement in the resulting MSA relative to conventional 
MAFFT (<a reveal-id="mst010-F4" data-open="mst010-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Based on this approach, we are developing an integrative service for protein structure-informed MSA construction.</div>                            <h2 scrollto-destination="33680886" id="33680886" class="backacknowledgements-title">Acknowledgments</h2>
<p>The authors thank Drs. Wen Chen, C. André Lévesque, and Christopher 
Lewis, Agriculture and Agri-Food Canada, for permitting the use of the 
ITS data in this article and providing other challenging problems. This 
work was supported by Platform for Drug Discovery, Informatics, and 
Structural Life Science from the Ministry of Education, Culture, Sports,
 Science and Technology, Japan, and the Computational Biology Research 
Center, National Institute of Advanced Industrial Science and Technology
 (AIST), Japan.</p>                            <h2 scrollto-destination="33680888" id="33680888" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="mst010-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B1" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">SF</div></div>.&nbsp;</div><div class="article-title">Generalized affine gap costs for protein sequence alignment</div>,&nbsp;<div class="source">Proteins</div>&nbsp;,&nbsp;<div class="year">1998</div>,&nbsp;vol.&nbsp;<div class="volume">32</div>&nbsp;(pg.&nbsp;<div class="fpage">88</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generalized%20affine%20gap%20costs%20for%20protein%20sequence%20alignment&amp;author=SF%20Altschul&amp;publication_year=1998&amp;journal=Proteins&amp;volume=32&amp;pages=88-96" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/%28ISSN%291097-0134" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F%28ISSN%291097-0134" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2F(ISSN)1097-0134"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9672045" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B2" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barton</div> <div class="given-names">GJ</div></div>,&nbsp; <div class="name"><div class="surname">Sternberg</div> <div class="given-names">MJ</div></div>.&nbsp;</div><div class="article-title">A strategy for the rapid multiple alignment of protein sequences. Confidence levels from tertiary structure comparisons</div>,&nbsp;<div class="source">J Mol Biol.</div>&nbsp;,&nbsp;<div class="year">1987</div>,&nbsp;vol.&nbsp;<div class="volume">198</div>&nbsp;(pg.&nbsp;<div class="fpage">327</div>-<div class="lpage">337</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20strategy%20for%20the%20rapid%20multiple%20alignment%20of%20protein%20sequences.%20Confidence%20levels%20from%20tertiary%20structure%20comparisons&amp;author=GJ%20Barton&amp;author=MJ%20Sternberg&amp;publication_year=1987&amp;journal=J%20Mol%20Biol.&amp;volume=198&amp;pages=327-337" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836%2887%2990316-0" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836%2887%2990316-0" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0022-2836(87)90316-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3430611" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B3" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berger</div> <div class="given-names">MP</div></div>,&nbsp; <div class="name"><div class="surname">Munson</div> <div class="given-names">PJ</div></div>.&nbsp;</div><div class="article-title">A novel randomized iterative strategy for aligning multiple protein sequences</div>,&nbsp;<div class="source">Comput Appl Biosci.</div>&nbsp;,&nbsp;<div class="year">1991</div>,&nbsp;vol.&nbsp;<div class="volume">7</div>&nbsp;(pg.&nbsp;<div class="fpage">479</div>-<div class="lpage">484</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20randomized%20iterative%20strategy%20for%20aligning%20multiple%20protein%20sequences&amp;author=MP%20Berger&amp;author=PJ%20Munson&amp;publication_year=1991&amp;journal=Comput%20Appl%20Biosci.&amp;volume=7&amp;pages=479-484" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1747779" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B4" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berger</div> <div class="given-names">SA</div></div>,&nbsp; <div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">Aligning short reads to reference alignments and trees</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">2068</div>-<div class="lpage">2075</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aligning%20short%20reads%20to%20reference%20alignments%20and%20trees&amp;author=SA%20Berger&amp;author=A%20Stamatakis&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2068-2075" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr320" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr320" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr320"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21636595" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B5" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blackburne</div> <div class="given-names">BP</div></div>,&nbsp; <div class="name"><div class="surname">Whelan</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">Class of multiple sequence alignment algorithm affects genomic analysis</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2012</div>&nbsp;<div class="comment">Advance access published December 4, 2012, doi:10.1093/molbev/mss256</div></div></div></div></div><div content-id="mst010-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B6" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blackburne</div> <div class="given-names">BP</div></div>,&nbsp; <div class="name"><div class="surname">Whelan</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">Measuring the distance between multiple sequence alignments</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">28</div>&nbsp;(pg.&nbsp;<div class="fpage">495</div>-<div class="lpage">502</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Measuring%20the%20distance%20between%20multiple%20sequence%20alignments&amp;author=BP%20Blackburne&amp;author=S%20Whelan&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=495-502" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr701" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr701" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr701"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22199391" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B7" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cannone</div> <div class="given-names">JJ</div></div>,&nbsp; <div class="name"><div class="surname">Subramanian</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Schnare</div> <div class="given-names">MN</div></div>,&nbsp;et al. &nbsp;,&nbsp;<div class="collab">(14 co-authors)</div>.&nbsp;</div><div class="article-title">The
 comparative RNA web (CRW) site: an online database of comparative 
sequence and structure information for ribosomal, intron, and other RNAs</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">3</div>&nbsp;pg.&nbsp;<div class="fpage">2</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20comparative%20RNA%20web%20%28CRW%29%20site%3A%20an%20online%20database%20of%20comparative%20sequence%20and%20structure%20information%20for%20ribosomal%2C%20intron%2C%20and%20other%20RNAs&amp;author=JJ%20Cannone&amp;author=S%20Subramanian&amp;author=MN%20Schnare&amp;publication_year=2002&amp;journal=BMC%20Bioinformatics&amp;volume=3&amp;pages=2" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-3-2" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-3-2" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-3-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11869452" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B8" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cole</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">Wang</div> <div class="given-names">Q</div></div>,&nbsp; <div class="name"><div class="surname">Cardenas</div> <div class="given-names">E</div></div>,&nbsp;et al. &nbsp;,&nbsp;<div class="collab">(11 co-authors)</div>.&nbsp;</div><div class="article-title">The ribosomal database project: improved alignments and new tools for rRNA analysis</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">37</div>&nbsp;(pg.&nbsp;<div class="fpage">D141</div>-<div class="lpage">D145</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ribosomal%20database%20project%3A%20improved%20alignments%20and%20new%20tools%20for%20rRNA%20analysis&amp;author=JR%20Cole&amp;author=Q%20Wang&amp;author=E%20Cardenas&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D141-D145" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn879" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn879" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkn879"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19004872" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B9" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dessimoz</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Gil</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Phylogenetic assessment of alignments reveals neglected tree signal in gaps</div>,&nbsp;<div class="source">Genome Biol.</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;pg.&nbsp;<div class="fpage">R37</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20assessment%20of%20alignments%20reveals%20neglected%20tree%20signal%20in%20gaps&amp;author=C%20Dessimoz&amp;author=M%20Gil&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=R37" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-4-r37" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-4-r37" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fgb-2010-11-4-r37"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20370897" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B10" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feng</div> <div class="given-names">DF</div></div>,&nbsp; <div class="name"><div class="surname">Doolittle</div> <div class="given-names">RF</div></div>.&nbsp;</div><div class="article-title">Progressive sequence alignment as a prerequisite to correct phylogenetic trees</div>,&nbsp;<div class="source">J Mol Evol.</div>&nbsp;,&nbsp;<div class="year">1987</div>,&nbsp;vol.&nbsp;<div class="volume">25</div>&nbsp;(pg.&nbsp;<div class="fpage">351</div>-<div class="lpage">360</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Progressive%20sequence%20alignment%20as%20a%20prerequisite%20to%20correct%20phylogenetic%20trees&amp;author=DF%20Feng&amp;author=RF%20Doolittle&amp;publication_year=1987&amp;journal=J%20Mol%20Evol.&amp;volume=25&amp;pages=351-360" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02603120" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02603120" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02603120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3118049" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B11" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fletcher</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Yang</div> <div class="given-names">Z</div></div>.&nbsp;</div><div class="article-title">INDELible: a flexible simulator of biological sequence evolution</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">26</div>&nbsp;(pg.&nbsp;<div class="fpage">1879</div>-<div class="lpage">1888</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=INDELible%3A%20a%20flexible%20simulator%20of%20biological%20sequence%20evolution&amp;author=W%20Fletcher&amp;author=Z%20Yang&amp;publication_year=2009&amp;journal=Mol%20Biol%20Evol.&amp;volume=26&amp;pages=1879-1888" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp098" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp098" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsp098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19423664" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B12" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Golubchik</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Wise</div> <div class="given-names">MJ</div></div>,&nbsp; <div class="name"><div class="surname">Easteal</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Jermiin</div> <div class="given-names">LS</div></div>.&nbsp;</div><div class="article-title">Mind the gaps: evidence of bias in estimates of multiple sequence alignments</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">24</div>&nbsp;(pg.&nbsp;<div class="fpage">2433</div>-<div class="lpage">2442</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mind%20the%20gaps%3A%20evidence%20of%20bias%20in%20estimates%20of%20multiple%20sequence%20alignments&amp;author=T%20Golubchik&amp;author=MJ%20Wise&amp;author=S%20Easteal&amp;author=LS%20Jermiin&amp;publication_year=2007&amp;journal=Mol%20Biol%20Evol.&amp;volume=24&amp;pages=2433-2442" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm176" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm176" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsm176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17709332" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B13" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gotoh</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">Optimal alignment between groups of sequences and its application to multiple sequence alignment</div>,&nbsp;<div class="source">Comput Appl Biosci.</div>&nbsp;,&nbsp;<div class="year">1993</div>,&nbsp;vol.&nbsp;<div class="volume">9</div>&nbsp;(pg.&nbsp;<div class="fpage">361</div>-<div class="lpage">370</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimal%20alignment%20between%20groups%20of%20sequences%20and%20its%20application%20to%20multiple%20sequence%20alignment&amp;author=O%20Gotoh&amp;publication_year=1993&amp;journal=Comput%20Appl%20Biosci.&amp;volume=9&amp;pages=361-370" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8324637" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B14" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gotoh</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">A weighting system and algorithm for aligning many phylogenetically related sequences</div>,&nbsp;<div class="source">Comput Appl Biosci.</div>&nbsp;,&nbsp;<div class="year">1995</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;(pg.&nbsp;<div class="fpage">543</div>-<div class="lpage">551</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20weighting%20system%20and%20algorithm%20for%20aligning%20many%20phylogenetically%20related%20sequences&amp;author=O%20Gotoh&amp;publication_year=1995&amp;journal=Comput%20Appl%20Biosci.&amp;volume=11&amp;pages=543-551" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8590178" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B15" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Henikoff</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Henikoff</div> <div class="given-names">JG</div></div>.&nbsp;</div><div class="article-title">Amino acid substitution matrices from protein blocks</div>,&nbsp;<div class="source">Proc Natl Acad Sci U S A.</div>&nbsp;,&nbsp;<div class="year">1992</div>,&nbsp;vol.&nbsp;<div class="volume">89</div>&nbsp;(pg.&nbsp;<div class="fpage">10915</div>-<div class="lpage">10919</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amino%20acid%20substitution%20matrices%20from%20protein%20blocks&amp;author=S%20Henikoff&amp;author=JG%20Henikoff&amp;publication_year=1992&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A.&amp;volume=89&amp;pages=10915-10919" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.22.10915" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.22.10915" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.89.22.10915"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1438297" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B16" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Higgins</div> <div class="given-names">DG</div></div>,&nbsp; <div class="name"><div class="surname">Sharp</div> <div class="given-names">PM</div></div>.&nbsp;</div><div class="article-title">CLUSTAL: a package for performing multiple sequence alignment on a microcomputer</div>,&nbsp;<div class="source">Gene</div>&nbsp;,&nbsp;<div class="year">1988</div>,&nbsp;vol.&nbsp;<div class="volume">73</div>&nbsp;(pg.&nbsp;<div class="fpage">237</div>-<div class="lpage">244</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%3A%20a%20package%20for%20performing%20multiple%20sequence%20alignment%20on%20a%20microcomputer&amp;author=DG%20Higgins&amp;author=PM%20Sharp&amp;publication_year=1988&amp;journal=Gene&amp;volume=73&amp;pages=237-244" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119%2888%2990330-7" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119%2888%2990330-7" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0378-1119(88)90330-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3243435" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B17" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">DT</div></div>,&nbsp; <div class="name"><div class="surname">Taylor</div> <div class="given-names">WR</div></div>,&nbsp; <div class="name"><div class="surname">Thornton</div> <div class="given-names">JM</div></div>.&nbsp;</div><div class="article-title">The rapid generation of mutation data matrices from protein sequences</div>,&nbsp;<div class="source">Comput Appl Biosci.</div>&nbsp;,&nbsp;<div class="year">1992</div>,&nbsp;vol.&nbsp;<div class="volume">8</div>&nbsp;(pg.&nbsp;<div class="fpage">275</div>-<div class="lpage">282</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rapid%20generation%20of%20mutation%20data%20matrices%20from%20protein%20sequences&amp;author=DT%20Jones&amp;author=WR%20Taylor&amp;author=JM%20Thornton&amp;publication_year=1992&amp;journal=Comput%20Appl%20Biosci.&amp;volume=8&amp;pages=275-282" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1633570" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B18" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">DT</div></div>,&nbsp; <div class="name"><div class="surname">Taylor</div> <div class="given-names">WR</div></div>,&nbsp; <div class="name"><div class="surname">Thornton</div> <div class="given-names">JM</div></div>.&nbsp;</div><div class="article-title">A mutation data matrix for transmembrane proteins</div>,&nbsp;<div class="source">FEBS Lett.</div>&nbsp;,&nbsp;<div class="year">1994</div>,&nbsp;vol.&nbsp;<div class="volume">339</div>&nbsp;(pg.&nbsp;<div class="fpage">269</div>-<div class="lpage">275</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20mutation%20data%20matrix%20for%20transmembrane%20proteins&amp;author=DT%20Jones&amp;author=WR%20Taylor&amp;author=JM%20Thornton&amp;publication_year=1994&amp;journal=FEBS%20Lett.&amp;volume=339&amp;pages=269-275" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793%2894%2980429-X" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793%2894%2980429-X" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0014-5793(94)80429-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8112466" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B19" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Asimenos</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">Multiple alignment of DNA sequences with MAFFT</div>,&nbsp;<div class="source">Methods Mol Biol.</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">537</div>&nbsp;(pg.&nbsp;<div class="fpage">39</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20alignment%20of%20DNA%20sequences%20with%20MAFFT&amp;author=K%20Katoh&amp;author=G%20Asimenos&amp;author=H%20Toh&amp;publication_year=2009&amp;journal=Methods%20Mol%20Biol.&amp;volume=537&amp;pages=39-64" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19378139" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B20" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Frith</div> <div class="given-names">MC</div></div>.&nbsp;</div><div class="article-title">Adding unaligned sequences into an existing alignment using MAFFT and LAST</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">28</div>&nbsp;(pg.&nbsp;<div class="fpage">3144</div>-<div class="lpage">3146</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adding%20unaligned%20sequences%20into%20an%20existing%20alignment%20using%20MAFFT%20and%20LAST&amp;author=K%20Katoh&amp;author=MC%20Frith&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=3144-3146" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts578" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts578" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbts578"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23023983" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B21" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Kuma</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Miyata</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">MAFFT version 5: improvement in accuracy of multiple sequence alignment</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">33</div>&nbsp;(pg.&nbsp;<div class="fpage">511</div>-<div class="lpage">518</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%20version%205%3A%20improvement%20in%20accuracy%20of%20multiple%20sequence%20alignment&amp;author=K%20Katoh&amp;author=K%20Kuma&amp;author=H%20Toh&amp;author=T%20Miyata&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=511-518" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki198" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki198" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgki198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15661851" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B22" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Misawa</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Kuma</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Miyata</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">30</div>&nbsp;(pg.&nbsp;<div class="fpage">3059</div>-<div class="lpage">3066</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform&amp;author=K%20Katoh&amp;author=K%20Misawa&amp;author=K%20Kuma&amp;author=T%20Miyata&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=3059-3066" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf436" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf436" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkf436"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B23" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Standley</div> <div class="given-names">DM</div></div>.&nbsp;</div><div class="article-title">MAFFT: iterative refinement and additional methods</div>,&nbsp;<div class="source">Methods Mol Biol.</div>&nbsp;,&nbsp;<div class="year">2013</div></div></div></div></div><div content-id="mst010-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B24" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">PartTree: An algorithm to build an approximate tree from a large number of unaligned sequences</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">23</div>&nbsp;(pg.&nbsp;<div class="fpage">372</div>-<div class="lpage">374</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PartTree%3A%20An%20algorithm%20to%20build%20an%20approximate%20tree%20from%20a%20large%20number%20of%20unaligned%20sequences&amp;author=K%20Katoh&amp;author=H%20Toh&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=372-374" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl592" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl592" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl592"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17118958" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B25" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">Improved accuracy of multiple ncRNA alignment by incorporating structural information into a MAFFT-based framework</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">9</div>&nbsp;pg.&nbsp;<div class="fpage">212</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20accuracy%20of%20multiple%20ncRNA%20alignment%20by%20incorporating%20structural%20information%20into%20a%20MAFFT-based%20framework&amp;author=K%20Katoh&amp;author=H%20Toh&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=212" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-212" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-212" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-9-212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18439255" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B26" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">Recent developments in the MAFFT multiple sequence alignment program</div>,&nbsp;<div class="source">Brief Bioinform.</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">9</div>&nbsp;(pg.&nbsp;<div class="fpage">286</div>-<div class="lpage">298</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20developments%20in%20the%20MAFFT%20multiple%20sequence%20alignment%20program&amp;author=K%20Katoh&amp;author=H%20Toh&amp;publication_year=2008&amp;journal=Brief%20Bioinform.&amp;volume=9&amp;pages=286-298" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbn013" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbn013" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbib%2Fbbn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18372315" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B27" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">Parallelization of the MAFFT multiple sequence alignment program</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">26</div>&nbsp;(pg.&nbsp;<div class="fpage">1899</div>-<div class="lpage">1900</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parallelization%20of%20the%20MAFFT%20multiple%20sequence%20alignment%20program&amp;author=K%20Katoh&amp;author=H%20Toh&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=1899-1900" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq224" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq224" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq224"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20427515" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B28" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kiełbasa</div> <div class="given-names">SM</div></div>,&nbsp; <div class="name"><div class="surname">Wan</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Sato</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Horton</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Frith</div> <div class="given-names">MC</div></div>.&nbsp;</div><div class="article-title">Adaptive seeds tame genomic sequence comparison</div>,&nbsp;<div class="source">Genome Res.</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">21</div>&nbsp;(pg.&nbsp;<div class="fpage">487</div>-<div class="lpage">493</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20seeds%20tame%20genomic%20sequence%20comparison&amp;author=SM%20Kie%C5%82basa&amp;author=R%20Wan&amp;author=K%20Sato&amp;author=P%20Horton&amp;author=MC%20Frith&amp;publication_year=2011&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=487-493" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.113985.110" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.113985.110" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.113985.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21209072" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B29" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Letsch</div> <div class="given-names">HO</div></div>,&nbsp; <div class="name"><div class="surname">Kuck</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Stocsits</div> <div class="given-names">RR</div></div>,&nbsp; <div class="name"><div class="surname">Misof</div> <div class="given-names">B</div></div>.&nbsp;</div><div class="article-title">The
 impact of rRNA secondary structure consideration in alignment and tree 
reconstruction: simulated data and a case study on the phylogeny of 
hexapods</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">2507</div>-<div class="lpage">2521</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20rRNA%20secondary%20structure%20consideration%20in%20alignment%20and%20tree%20reconstruction%3A%20simulated%20data%20and%20a%20case%20study%20on%20the%20phylogeny%20of%20hexapods&amp;author=HO%20Letsch&amp;author=P%20Kuck&amp;author=RR%20Stocsits&amp;author=B%20Misof&amp;publication_year=2010&amp;journal=Mol%20Biol%20Evol.&amp;volume=27&amp;pages=2507-2521" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msq140" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq140" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsq140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20530152" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B30" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Löytynoja</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Vilella</div> <div class="given-names">AJ</div></div>,&nbsp; <div class="name"><div class="surname">Goldman</div> <div class="given-names">N</div></div>.&nbsp;</div><div class="article-title">Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">28</div>&nbsp;(pg.&nbsp;<div class="fpage">1684</div>-<div class="lpage">1691</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20extension%20of%20multiple%20sequence%20alignments%20using%20a%20phylogeny-aware%20graph%20algorithm&amp;author=A%20L%C3%B6ytynoja&amp;author=AJ%20Vilella&amp;author=N%20Goldman&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1684-1691" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts198" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts198" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbts198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22531217" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B31" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsen</div> <div class="given-names">FA</div></div>,&nbsp; <div class="name"><div class="surname">Kodner</div> <div class="given-names">RB</div></div>,&nbsp; <div class="name"><div class="surname">Armbrust</div> <div class="given-names">EV</div></div>.&nbsp;</div><div class="article-title">pplacer: linear time maximum-likelihood and Bayesian phylogenetic placement of sequences onto a fixed reference tree</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;pg.&nbsp;<div class="fpage">538</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=pplacer%3A%20linear%20time%20maximum-likelihood%20and%20Bayesian%20phylogenetic%20placement%20of%20sequences%20onto%20a%20fixed%20reference%20tree&amp;author=FA%20Matsen&amp;author=RB%20Kodner&amp;author=EV%20Armbrust&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=538" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-538" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-538" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-11-538"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21034504" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B32" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mirarab</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Nguyen</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Warnow</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">SEPP: SATé-enabled phylogenetic placement</div>,&nbsp;<div class="source">Pac Symp Biocomput.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">17</div>&nbsp;(pg.&nbsp;<div class="fpage">247</div>-<div class="lpage">258</div>)</div></div></div></div><div content-id="mst010-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B48" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nilsson</div> <div class="given-names">RH</div></div>,&nbsp; <div class="name"><div class="surname">Veldre</div> <div class="given-names">V</div></div>,&nbsp; <div class="name"><div class="surname">Hartmann</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Unterseher</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Amend</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Bergsten</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Kristiansson</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Ryberg</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Jumpponen</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Abarenkov</div> <div class="given-names">K</div></div>.&nbsp;</div><div class="article-title">An
 open source software package for automated extraction of ITS1 and ITS2 
from fungal ITS sequences for use in high-throughput community assays 
and molecular ecology</div>,&nbsp;<div class="source">Fungal Ecology</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">3</div>&nbsp;(pg.&nbsp;<div class="fpage">284</div>-<div class="lpage">287</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20open%20source%20software%20package%20for%20automated%20extraction%20of%20ITS1%20and%20ITS2%20from%20fungal%20ITS%20sequences%20for%20use%20in%20high-throughput%20community%20assays%20and%20molecular%20ecology&amp;author=RH%20Nilsson&amp;author=V%20Veldre&amp;author=M%20Hartmann&amp;author=M%20Unterseher&amp;author=A%20Amend&amp;author=J%20Bergsten&amp;author=E%20Kristiansson&amp;author=M%20Ryberg&amp;author=A%20Jumpponen&amp;author=K%20Abarenkov&amp;publication_year=2010&amp;journal=Fungal%20Ecology&amp;volume=3&amp;pages=284-287" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.funeco.2010.05.002" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.funeco.2010.05.002" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.funeco.2010.05.002"> </span></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B33" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Notredame</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Holm</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Higgins</div> <div class="given-names">DG</div></div>.&nbsp;</div><div class="article-title">COFFEE: an objective function for multiple sequence alignments</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">1998</div>,&nbsp;vol.&nbsp;<div class="volume">14</div>&nbsp;(pg.&nbsp;<div class="fpage">407</div>-<div class="lpage">422</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=COFFEE%3A%20an%20objective%20function%20for%20multiple%20sequence%20alignments&amp;author=C%20Notredame&amp;author=L%20Holm&amp;author=DG%20Higgins&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=407-422" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.5.407" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.5.407" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F14.5.407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9682054" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B34" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nuin</div> <div class="given-names">PA</div></div>,&nbsp; <div class="name"><div class="surname">Wang</div> <div class="given-names">Z</div></div>,&nbsp; <div class="name"><div class="surname">Tillier</div> <div class="given-names">ER</div></div>.&nbsp;</div><div class="article-title">The accuracy of several multiple sequence alignment programs for proteins</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">7</div>&nbsp;pg.&nbsp;<div class="fpage">471</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20accuracy%20of%20several%20multiple%20sequence%20alignment%20programs%20for%20proteins&amp;author=PA%20Nuin&amp;author=Z%20Wang&amp;author=ER%20Tillier&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=471" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-471" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-471" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-7-471"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17062146" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B35" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">O’Sullivan</div> <div class="given-names">O</div></div>,&nbsp; <div class="name"><div class="surname">Suhre</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Abergel</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Higgins</div> <div class="given-names">DG</div></div>,&nbsp; <div class="name"><div class="surname">Notredame</div> <div class="given-names">C</div></div>.&nbsp;</div><div class="article-title">3DCoffee: combining protein sequences and structures within multiple sequence alignments</div>,&nbsp;<div class="source">J Mol Biol.</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">340</div>&nbsp;(pg.&nbsp;<div class="fpage">385</div>-<div class="lpage">395</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=3DCoffee%3A%20combining%20protein%20sequences%20and%20structures%20within%20multiple%20sequence%20alignments&amp;author=O%20O%E2%80%99Sullivan&amp;author=K%20Suhre&amp;author=C%20Abergel&amp;author=DG%20Higgins&amp;author=C%20Notredame&amp;publication_year=2004&amp;journal=J%20Mol%20Biol.&amp;volume=340&amp;pages=385-395" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2004.04.058" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2004.04.058" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jmb.2004.04.058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15201059" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B36" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pei</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Kim</div> <div class="given-names">BH</div></div>,&nbsp; <div class="name"><div class="surname">Grishin</div> <div class="given-names">NV</div></div>.&nbsp;</div><div class="article-title">PROMALS3D: a tool for multiple protein sequence and structure alignments</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">36</div>&nbsp;(pg.&nbsp;<div class="fpage">2295</div>-<div class="lpage">2300</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PROMALS3D%3A%20a%20tool%20for%20multiple%20protein%20sequence%20and%20structure%20alignments&amp;author=J%20Pei&amp;author=BH%20Kim&amp;author=NV%20Grishin&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=2295-2300" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn072" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn072" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkn072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18287115" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B37" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Punta</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Coggill</div> <div class="given-names">PC</div></div>,&nbsp; <div class="name"><div class="surname">Eberhardt</div> <div class="given-names">RY</div></div>,&nbsp;et al. &nbsp;,&nbsp;<div class="collab">(16 co-authors)</div>.&nbsp;</div><div class="article-title">The Pfam protein families database</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">40</div>&nbsp;(pg.&nbsp;<div class="fpage">D290</div>-<div class="lpage">D301</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=M%20Punta&amp;author=PC%20Coggill&amp;author=RY%20Eberhardt&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D290-D301" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1065" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1065" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr1065"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127870" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B38" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sahraeian</div> <div class="given-names">SM</div></div>,&nbsp; <div class="name"><div class="surname">Yoon</div> <div class="given-names">BJ</div></div>.&nbsp;</div><div class="article-title">PicXAA-R: efficient structural alignment of multiple RNA sequences using a greedy approach</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">12</div>&nbsp;pg.&nbsp;<div class="fpage">S38</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PicXAA-R%3A%20efficient%20structural%20alignment%20of%20multiple%20RNA%20sequences%20using%20a%20greedy%20approach&amp;author=SM%20Sahraeian&amp;author=BJ%20Yoon&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=S38" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-S1-S38" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-S1-S38" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-12-S1-S38"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21342569" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B39" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schrödinger</div> <div class="given-names">LLC</div></div>.&nbsp;</div><div class="article-title">The PyMOL Molecular Graphics System, Version 1.3r1. Portland</div>,&nbsp;<div class="year">2010</div>&nbsp;<div class="comment">Oregon: Schrödinger, LLC</div></div></div></div></div><div content-id="mst010-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B40" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sievers</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Wilm</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Dineen</div> <div class="given-names">D</div></div>,&nbsp;et al. &nbsp;,&nbsp;<div class="collab">(12 co-authors)</div>.&nbsp;</div><div class="article-title">Fast scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</div>,&nbsp;<div class="source">Mol Syst Biol.</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">7</div>&nbsp;pg.&nbsp;<div class="fpage">539</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20scalable%20generation%20of%20high-quality%20protein%20multiple%20sequence%20alignments%20using%20Clustal%20Omega&amp;author=F%20Sievers&amp;author=A%20Wilm&amp;author=D%20Dineen&amp;publication_year=2011&amp;journal=Mol%20Syst%20Biol.&amp;volume=7&amp;pages=539" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2011.75" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2011.75" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fmsb.2011.75"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21988835" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B41" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sigrist</div> <div class="given-names">CJ</div></div>,&nbsp; <div class="name"><div class="surname">Cerutti</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">de Castro</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Langendijk-Genevaux</div> <div class="given-names">PS</div></div>,&nbsp; <div class="name"><div class="surname">Bulliard</div> <div class="given-names">V</div></div>,&nbsp; <div class="name"><div class="surname">Bairoch</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Hulo</div> <div class="given-names">N</div></div>.&nbsp;</div><div class="article-title">PROSITE, a protein domain database for functional characterization and annotation</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">38</div>&nbsp;(pg.&nbsp;<div class="fpage">D161</div>-<div class="lpage">D166</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PROSITE%2C%20a%20protein%20domain%20database%20for%20functional%20characterization%20and%20annotation&amp;author=CJ%20Sigrist&amp;author=L%20Cerutti&amp;author=E%20de%20Castro&amp;author=PS%20Langendijk-Genevaux&amp;author=V%20Bulliard&amp;author=A%20Bairoch&amp;author=N%20Hulo&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=D161-D166" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp885" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp885" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkp885"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19858104" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B42" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Standley</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Nakamura</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">Ash structure alignment package: sensitivity and selectivity in domain classification</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">8</div>&nbsp;pg.&nbsp;<div class="fpage">116</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ash%20structure%20alignment%20package%3A%20sensitivity%20and%20selectivity%20in%20domain%20classification&amp;author=D%20Standley&amp;author=H%20Toh&amp;author=H%20Nakamura&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics&amp;volume=8&amp;pages=116" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-116" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-116" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-8-116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17407606" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B43" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Standley</div> <div class="given-names">DM</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Nakamura</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">Detecting local structural similarity in proteins by maximizing number of equivalent residues</div>,&nbsp;<div class="source">Proteins</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">57</div>&nbsp;(pg.&nbsp;<div class="fpage">381</div>-<div class="lpage">391</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20local%20structural%20similarity%20in%20proteins%20by%20maximizing%20number%20of%20equivalent%20residues&amp;author=DM%20Standley&amp;author=H%20Toh&amp;author=H%20Nakamura&amp;publication_year=2004&amp;journal=Proteins&amp;volume=57&amp;pages=381-391" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/prot.20211" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fprot.20211" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2Fprot.20211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15340925" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B44" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Buhler</div> <div class="given-names">JD</div></div>.&nbsp;</div><div class="article-title">PhyLAT: a phylogenetic local alignment tool</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">28</div>&nbsp;(pg.&nbsp;<div class="fpage">1336</div>-<div class="lpage">1344</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PhyLAT%3A%20a%20phylogenetic%20local%20alignment%20tool&amp;author=H%20Sun&amp;author=JD%20Buhler&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1336-1344" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts158" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts158" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbts158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22492645" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B45" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tabei</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Kiryu</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Kin</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Asai</div> <div class="given-names">K</div></div>.&nbsp;</div><div class="article-title">A fast structural multiple alignment method for long RNA sequences</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">9</div>&nbsp;pg.&nbsp;<div class="fpage">33</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%20structural%20multiple%20alignment%20method%20for%20long%20RNA%20sequences&amp;author=Y%20Tabei&amp;author=H%20Kiryu&amp;author=T%20Kin&amp;author=K%20Asai&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=33" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-33" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-33" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-9-33"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18215258" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B46" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">JD</div></div>,&nbsp; <div class="name"><div class="surname">Higgins</div> <div class="given-names">DG</div></div>,&nbsp; <div class="name"><div class="surname">Gibson</div> <div class="given-names">TJ</div></div>.&nbsp;</div><div class="article-title">CLUSTAL
 W: improving the sensitivity of progressive multiple sequence alignment
 through sequence weighting, position-specific gap penalties and weight 
matrix choice</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">1994</div>,&nbsp;vol.&nbsp;<div class="volume">22</div>&nbsp;(pg.&nbsp;<div class="fpage">4673</div>-<div class="lpage">4680</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=JD%20Thompson&amp;author=DG%20Higgins&amp;author=TJ%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4673-4680" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="mst010-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="mst010-B47" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waterhouse</div> <div class="given-names">AM</div></div>,&nbsp; <div class="name"><div class="surname">Procter</div> <div class="given-names">JB</div></div>,&nbsp; <div class="name"><div class="surname">Martin</div> <div class="given-names">DM</div></div>,&nbsp; <div class="name"><div class="surname">Clamp</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Barton</div> <div class="given-names">GJ</div></div>.&nbsp;</div><div class="article-title">Jalview version 2—a multiple sequence alignment editor and analysis workbench</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">25</div>&nbsp;(pg.&nbsp;<div class="fpage">1189</div>-<div class="lpage">1191</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Jalview%20version%202%E2%80%94a%20multiple%20sequence%20alignment%20editor%20and%20analysis%20workbench&amp;author=AM%20Waterhouse&amp;author=JB%20Procter&amp;author=DM%20Martin&amp;author=M%20Clamp&amp;author=GJ%20Barton&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1189-1191" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp033" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp033" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19151095" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix" style="display: none;"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution.</div><div class="license"><div class="license-p">This
 is an Open Access article distributed under the terms of the Creative 
Commons Attribution Non-Commercial License 
(http://creativecommons.org/licenses/by-nc/3.0/), which permits 
unrestricted non-commercial use, distribution, and reproduction in any 
medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-RelatedTags widget-instance-OUP_Article_RelatedTags_Widget">
        
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=Fast%20Tracks">Fast Tracks</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt" style="display: none;">
                            <a href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=1073398&amp;xsltPath=%7E/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Behind_Ad2_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/mbe_Behind_Ad2_0" title="3rd party ad content" name="google_ads_iframe_/116097782/mbe_Behind_Ad2_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div></div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">15,467</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000317002300004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">5,781</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/molbev/mst010" data-uuid="cdbb3c7f-1ecf-99b5-b1d6-3a2fb71caac1"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=1187623" style="display:inline-block;">
        <img alt="Article has an altmetric score of 18" src="https://badges.altmetric.com/?size=128&amp;score=18&amp;types=matttttg" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1073398"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000317002300004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fmbe%2farticle%2f30%2f4%2f772%2f1073398">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000317002300004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (5781)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F30%2F4%2F772%2F1073398" target="">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href="https://academic.oup.com/mbe/CrossRef-CitedBy/1073398">CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/mbe/advance-article/doi/10.1093/molbev/msy193/5133304">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy193/5133304">Secreted bacterial adenosine deaminase is an evolutionary precursor of adenosine deaminase growth factor</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy193/5133304">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/mbe/advance-article/doi/10.1093/molbev/msy189/5126238">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy189/5126238">New insight into parrots’ mitogenomes indicates that their ancestor contained a duplicated region</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy189/5126238">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/mbe/advance-article/doi/10.1093/molbev/msy188/5123518">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy188/5123518">Understanding the factors that shape patterns of nucleotide diversity in the house mouse genome</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy188/5123518">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/mbe/advance-article/doi/10.1093/molbev/msy191/5123519">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy191/5123519">Functional conservation of a developmental switch in mammals since the Jurassic age</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy191/5123519">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/mbe/advance-article/doi/10.1093/molbev/msy190/5115937">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy190/5115937">Genomic
 analyses of human European diversity at the southwestern edge: 
isolation, African influence and disease associations in the Canary 
Islands</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msy190/5115937">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Behind_Ad3_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/mbe_Behind_Ad3_0" title="3rd party ad content" name="google_ads_iframe_/116097782/mbe_Behind_Ad3_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div></div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" id="hfArticleTitle" value="MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability" type="hidden">

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/molbev/mst010");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Behind_Ad4_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/mbe_Behind_Ad4_0" title="3rd party ad content" name="google_ads_iframe_/116097782/mbe_Behind_Ad4_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="728" frameborder="0" height="90"></iframe></div></div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="https://academic.oup.com/mbe/pages/About">About Molecular Biology and Evolution</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/mbe/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/mbe/pages/Policies">Policies</a>
                            </li>
                            <li class="link-4">
                                <a href="https://academic.oup.com/mbe/pages/General_Instructions">Author Guidelines</a>
                            </li>
                            <li class="link-5">
                                <a href="http://www.smbe.org/smbe/">Contact Us</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
                            <li class="link-2">
                                <a href="https://twitter.com/OfficialSMBE">Twitter</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/mbe/subscribe">Purchase</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                            </li>
                            <li class="link-5">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/mbe" target="">
                <img id="footer-logo-MolecularBiologyandEvolution" class="journal-footer-affiliations-logo" src="mbe_f1-2062292105.svg" alt="Molecular Biology and Evolution">
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1537-1719</li>
                                            <li>Print ISSN 0737-4038</li>
                    <li>Copyright © 2018 Society for Molecular Biology and Evolution</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal="">
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f30%2f4%2f772%2f1073398">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="footerJS" type="text/javascript"></script>
<script src="OupCookiePolicyJS" type="text/javascript"></script>
<script src="MathJax.js" type="text/javascript"></script>
<script src="chartist.js" type="text/javascript"></script>
<script src="ArticlePageJS" type="text/javascript"></script>
<script src="addthis_widget.js" type="text/javascript"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh64" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.8296555419636504&amp;iit=1539788365432&amp;tmr=load%3D1539788365370%26core%3D1539788365426%26main%3D1539788365428%26ifr%3D1539788365448&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F30%2F4%2F772%2F1073398&amp;href=https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F30%2F4%2F772%2F1073398&amp;dt=MAFFT%20Multiple%20Sequence%20Alignment%20Software%20Version%207%3A%20Improvements%20in%20Performance%20and%20Usability%20%7C%20Molecular%20Biology%20and%20Evolution%20%7C%20Oxford%20Academic&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5bc74e4d02b44ca7&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DMAFFT%2520Multiple%2520Sequence%2520Alignment%2520Software%2520Version%25207%253A%2520Improvements%2520in%2520Performance%2520and%2520Usability%2520%257C%2520Molecular%2520Biology%2520and%2520Evolution%2520%257C%2520Oxford%2520Academic%26description%3DAbstract.%2520%2520We%2520report%2520a%2520major%2520update%2520of%2520the%2520MAFFT%2520multiple%2520sequence%2520alignment%2520program.%2520This%2520version%2520has%2520several%2520new%2520features%252C%2520including%2520options%2520for%2520adding%2520unalig%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fmbe%252Farticle%252F30%252F4%252F772%252F1073398%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fmbe%252FIssue%252F35%252F10%252F1%252Fm_molbiolevol_35_10cover.png%253FExpires%253D2147483647%2526Signature%253DovhLH5Jp5vurDX37AM2aOxITo5poFVa%7Ezb57nrpwFmttgcVe93Exxjj4ZF%7Ett2AnUF%7EvhgA4tuIjLHodp6u7CRcYU9DiYvuX8yaNNDfH3NKXJcek1TR3at1pkNptLqnJlJvurJmnktOKbWUqhOokkanvi7Qy-1buL-NzhVcvx%7EdURkFVqsw97R9fZw%7ESzhsYh1vRGgsj5Tv330CV9N1so2zjwsJPE2X%7E8UUPLVMTnpJ85Ht1JJLXIsmoqaLQm9fSw5T3DqEtf%7EQbgLD2tO0Zf76-XnQuAfD4aSMWSUeb6YCkQzJjtpOwi-gOkq%7ElpWRUStdsbX%7E6XJN0HvJfpyGRbA__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2013-01-16%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.3.31-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>
<script src="api.js" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="mst010"></script></body></html>