<!DOCTYPE html>
<html class="pb-page readcubeEpdfEnabled" data-request-id="1a47c4df-57b4-4226-919c-df13edb04959" lang="en"><head data-pb-dropzone="head">
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="pbContext" content=";page:string:Article/Chapter View;website:website:pericles;issue:issue:doi\:10.1111/mec.2018.27.issue-2;journal:journal:1365294x;article:article:doi\:10.1111/mec.14478;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;subPage:string:Full Text"><meta data-pb-head="head-widgets-start">
<iframe style="position: absolute; top: -10000px; left: -1000px;"></iframe><script async="" src="prum.js"></script><script src="iframe_api"></script><script type="text/javascript" src="get"></script><script type="text/javascript" src="optout_check"></script><script async="" src="controltag.js"></script><script async="" src="gpt.js"></script><script type="text/javascript" async="" src="controltag"></script><script async="" src="fbevents.js"></script><script type="text/javascript">var $DoubleClickZone = "molec-eco_mec";var $DoubleClickSite =  "wly.ecology_000046";</script><script>digitalData = {"site":{"ip":"197.248.44.39","environment":"LIVE","website":"onlinelibrary.wiley.com","serverDate":"2019-04-04"},"page":{"pageGroup":"Publication Pages","page":"Article/Chapter View","subPage":"Full Text","title":"title","errorCode":200},"identities":[{"type":"BasicGroup","uuid":"be20949b-b372-471e-96a5-9963c7266a9d","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000054870"}]},{"type":"BasicGroup","uuid":"98ba5e39-8ca0-4d0a-addd-395777fbf515","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000034166"}]},{"type":"InstitutionUser","uuid":"6e97ed1f-543d-493d-b458-2e63cb8fbde5","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#KLISC"}]},{"type":"BasicGroup","uuid":"e623deab-c83b-41e3-84a2-d9f1d46c5f17"},{"type":"RegionUser","uuid":"53f1676b-6f75-446d-8831-5915e1edce59"},{"type":"InstitutionUser","uuid":"dab9a169-ec74-4502-8066-0f76387b0f4e","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASP"}]},{"type":"BasicGroup","uuid":"40e3f768-07c0-45e5-aa81-97aef6eb686c","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000005407"}]},{"type":"BasicGroup","uuid":"c3199058-0f44-4f36-b385-3a7c0023d85c","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"KENYA HINARI NPL"}]},{"type":"InstitutionUser","uuid":"1f5548c7-7894-487c-82f4-7a9f5c335106","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"ALFC00012"}]},{"type":"BasicGroup","uuid":"197fe835-5b5f-4f4e-b8e9-a74d68491a84"},{"type":"InstitutionUser","uuid":"5196cf05-b385-4de3-91a2-2d5ffd18abc5","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASPKEN"}]},{"type":"InstitutionUser","uuid":"480e4dba-ca42-475f-a7ca-5b6fbfad49f7","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL00000014668"}]}],"publication":{"item":{"title":"Scaling up: A guide to high‐throughput genomic approaches for biodiversity analysis","doi":"10.1111/mec.14478","access":"open","earliestDate":"2018-01-02","formats":["Abstract","Cited by","First page image","Full text","PDF","References","AM pdf"],"topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"}]},"group":{"doi":"10.1111/mec.2018.27.issue-2","access":"full","volume":"27","issue":"2","coverDate":"2018-01-01","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"}]},"series":{"title":"Molecular Ecology","doi":"10.1111/(ISSN)1365-294X","access":"no","code":"1365294x","issn":"0962-1083","eissn":"1365-294X","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"},{"taxonomyUri":"gpt-sitename","topicLabel":"wly.ecology_000046","topicUri":"wly.ecology_000046"},{"taxonomyUri":"gpt-zonename","topicLabel":"molec-eco_MEC","topicUri":"molec-eco_mec"}]}}}</script>






<meta charset="UTF-8">




<meta name="robots" content="noarchive">

















    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=1">







    
        <meta name="citation_fulltext_world_readable" content="">
    









    

    <meta name="citation_doi" content="10.1111/mec.14478">

    <meta name="dc.identifier" content="10.1111/mec.14478">




<meta name="citation_journal_title" content="Molecular Ecology">
<meta name="citation_title" content="Scaling up: A guide to high‐throughput genomic approaches for biodiversity analysis">
<meta name="citation_keywords" content="biodiversity">
<meta name="citation_keywords" content="bioinformatics">
<meta name="citation_keywords" content="biomonitoring">
<meta name="citation_keywords" content="DNA barcode">
<meta name="citation_keywords" content="environment">
<meta name="citation_keywords" content="high‐throughput sequencing">
<meta name="citation_keywords" content="metabarcoding">
<meta name="citation_keywords" content="metagenomics">
<meta name="citation_volume" content="27">
<meta name="citation_issue" content="2">
<meta name="citation_firstpage" content="313">
<meta name="citation_lastpage" content="338">
<meta name="citation_publication_date" content="2018/01/01">
<meta name="citation_online_date" content="2018/03/03">
<meta name="citation_issn" content="1365-294X">
<meta name="citation_language" content="en">
<meta name="citation_author" content="Teresita M. Porter">
<meta name="citation_author_institution" content="Centre for Biodiversity Genomics Biodiversity Institute of Ontario and Department of Integrative Biology University of Guelph Guelph ON Canada">
<meta name="citation_author_institution" content="Natural Resources Canada Great Lakes Forestry Centre Sault Ste. Marie ON Canada">
<meta name="citation_author" content="Mehrdad Hajibabaei">
<meta name="citation_author_institution" content="Centre for Biodiversity Genomics Biodiversity Institute of Ontario and Department of Integrative Biology University of Guelph Guelph ON Canada">
<meta name="citation_abstract_html_url" content="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.14478">
<meta name="citation_fulltext_html_url" content="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.14478">
<meta name="citation_pdf_url" content="https://onlinelibrary.wiley.com/doi/pdf/10.1111/mec.14478">
<meta name="citation_publisher" content="John Wiley &amp; Sons, Ltd (10.1111)"><meta name="citation_fulltext_world_readable" content="">
<meta name="article_references" content="
 Porter, TM,  Hajibabaei, M.  Scaling up: A guide to high‐throughput genomic approaches for biodiversity analysis. Mol Ecol.  2018; 27: 313– 338. https://doi.org/10.1111/mec.14478
">
<meta name="epdf_available" content="true">


<title>Scaling up: A guide to high‐throughput genomic approaches for 
biodiversity analysis - Porter - 2018 - Molecular Ecology - Wiley Online
 Library</title>




<meta data-pb-head="head-widgets-end">

<link rel="stylesheet" type="text/css" href="gj6xproduct.css"><link rel="stylesheet" type="text/css" href="build.css">
<link rel="stylesheet" type="text/css" href="print.css" media="print">
<link rel="stylesheet" type="text/css" href="default.css" id="pb-css" data-pb-css-id="t-1-v0/default.css">




<script async="" src="catchpoint-1549680351760.js"></script>
<meta name="referrer" content="origin-when-cross-origin">

<link rel="stylesheet" type="text/css" href="hub-1534777306460.css">
<link rel="stylesheet" type="text/css" href="iucr-1510734465460.css">
<link rel="stylesheet" type="text/css" href="readcube-1510565373110.css">
<link rel="stylesheet" type="text/css" href="dr-overrides-1552321492860.css">
<!-- link rel="stylesheet" type="text/css" href="/pb-assets/css/temporary-1525702334993.css" -->

<style>
/* PERICLES-7186: distorted cover images */
.journal-info-container .cover-image__image img {
    min-height: unset;
}

/* TEMPORARY CSS HACK TO FIX LIT-220590 */
.icon-icon-Rss::before {
    content: "\e629";
}

/* TEMPORARY CSS TO FIX ALIGNMENT OF TOP AD IN RIGHT RAIL */
.publication-cover-info {
    margin-bottom: 20px;
}

.article-row-right > .pb-ad {
    margin-left: 0;
    margin-bottom: 10px;
}

/* TEMPORARY HACK FOR PERICLES-6763 */
.table-of-content .issue-item h2 {
    font-size: 16px;
}

.table-of-content .issue-item h2 a.issue-item__title {
    display: inline;
}

/* TEMPORARY CSS FOR PERICLES-7445 */

input.saml__consent__yes {
    background: #005274;
    font-size: 12px;
    font-size: .75rem;
    padding: 12px 20px;
    border-radius: 0;
    margin-top: 20px;
    margin-right: 10px;
    width: 100px;
}

input.saml__consent__yes:hover {
    background: #003951;
}

/* HACK TO CHANGE LABEL IN LOGIN FORM */
.login-popup .login-form label[for="login"]:before {
    content: "Email or Customer ID";
    color: black;
}

.login-popup .login-form label[for="login"] {
    color: white;
}

/* PERICLES-2166: Site-wide maintenance message */

.admin-message {
    background-color: #005274;
    color: white;
}

/* TEMPORARY CSS FOR STAY-CONNECTED SOCIAL MEDIA ICONS */
.stay-connected ul  {
    list-style: none;
    margin: 0;
    padding: 0;
}

.stay-connected i {
    margin-right: 10px;
    font-size: 25px;
    line-height: 55px;
}

.stay-connected i.icon-hubs-rg-icon,
.stay-connected i.icon-hubs-soundcloud-icon {
    font-size: 41px;
}

.stay-connected .icon-hubs-pinterest-icon:before {
    color: white;
}

.stay-connected .icon-hubs-pinterest-icon {
    background-color: #BD081C;
    border: solid #BD081C;
    border-width: 8px;
}

/* PERICLES-5833: TEMPORARY CSS TO ADJUST POSITION OF ADS */
.article-mpu-bottom .pb-ad {
    margin: 0 0 0 195px;
}


.article-mpu-top .pb-ad {
    padding-top: 20px;
    margin: 0 0 -35px 0;
}

.banner-wrapper .pb-ad {
    padding: 0;
}

/* TEMPORARY CSS FOR THE SOCIETY MEMBER LOGIN PAGE */
.society-member-login .widget__header,
.society-member-login .show-registration,
.society-member-login .show-request-reset-password {
    display: none;
}

.society-member-login .login-form {
    margin-top: 20px;
}

.society-member-login .widget__body .input-group+.input-group {
    margin-top: 15px;
}


/* TEMPORARY CSS TO HIDE FPI IN TOP CONTENT ACTION WIDGET */
.top-content .firstPageImage {
	display: none;
}

/* PERICLES-5518: TEMPORARY CSS TO FIX ANCHOR TAG OFFSET ON CUSTOM PAGES */
.custom-page-hub .main-content :target:before {
	content: "";
	display: block;
	height: 205px;
	margin: -205px 0 0;
}

.custom-page-pericles .main-content :target:before {
	content: "";
	display: block;
	height: 140px;
	margin: -140px 0 0;
}

/* PERICLES-4777: TEMPORARY CSS TO HIDE OPTIONS IN THE SIDEBAR ON MRW PAGES */
.journal-resources li#TableofContents,
.journal-resources li#Topics,
.journal-resources li#ArticleTitlesAZ {
	display: none;
}

/* TEMPORARY FIX FOR UNWANTED MARGIN BELOW COVER IMAGES ON BOOK SERIES PAGES */
.remove-margin .cover-image {
	margin: 0;
}

/* TEMPORARY STYLE TO HIDE NUMBER OF DOWNLOADS ON ARTICLE PAGES */
.info-tab-wrapper .metrics .section__body div:first-child {
	display: none;
}

/* TEMPORARY FIX TO MAKE IT POSSIBLE TO EDIT RICH TEXT WIDGETS IN SLIDESHOWS */
.bs .modal-backdrop {
	display: none;
}

/* TEMPORARY FIX FOR MENU POSITION */
.hubpage-menu nav {
	padding-bottom: 0;
	position: relative;
}

/* TEMPORARY FOR MARGIN AROUND ADS */
.advert.advert-leaderboard {
	margin: 15px 0;
}

.banner-wrapper {
	box-shadow: none;
}

/* TEMPORARY TO FIX OVERFLOW FOR P IN JOURNAL-NEWS SLIDESHOWS */
.journal-news .slide-item p {
	overflow: unset;
}

/* EDITOR HIGHLIGHTS CSS */
div.ancillary-content.relateditem div.content {
	height: 10em;
}
div.ancillary-content.listitem div.content {
	height: 4em;
}

/* TEMPORARY Fix heading on Topical Index widget for MRWs */
.topical-index h2 {
    margin: 15px 0;
    font-size: 1.75rem;
    line-height: 28px;
    color: #1c1d1e;
    font-family: Source Serif Pro,serif;
    font-weight: normal;
}

/* PERICLES-7288 */
.issue-supplementary-files, .cover-image__virtual-issue {
    margin-bottom: 15px;
}

/* PERICLES-8875 */
.journal-news .slide-item img {
    float: left;
    margin-right: 10px;
}

</style>

<!-- The Adobe DTM script. Must come after the digitalData object -->
<script src="satelliteLib-c1ac050bbdc647f13a62578e71f0d817f8a6cbb5.js"></script><script src="satellite-5c87b4fe64746d78ad006ae3.js"></script><script src="satellite-5aac819d64746d762f005d2f.js"></script><script src="satellite-59cd501964746d5527006272.js"></script><script src="satellite-5aaec36164746d79c1001d14.js"></script>

<!-- Google Search Console verification -->
<meta name="google-site-verification" content="ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM">

<!-- KRUX Implementation-->
<script class="kxct" data-id="JMOONjNh" data-timing="async" data-version="1.9" type="text/javascript" xml:space="preserve">

	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments )
	}).q = []);

	( function () {
		var k , m , s , src;
		k = document.createElement ( 'script' );
		k.type 	= 'text/javascript';
		k.async = true;
		src = ( m = location.href.match ( /\bkxsrc=([^&]+)/ )) && decodeURIComponent ( m [ 1 ]);
		k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test ( src ) ? src : src === 'disable' ? '' : ( location.protocol === 'https:'?'https:':'http:' ) + '//cdn.krxd.net/controltag?confid=JMOONjNh';
		s = document.getElementsByTagName ( 'script' ) [ 0 ];
		s.parentNode.insertBefore ( k , s );
	} ());
</script>

<!-- KRUX Implementation cont'd -->
<script type="text/javascript">
	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments );
	}).q = []);

	( function () {
		function retrieve ( n ) {
			var m , k = 'kx' + n;
			if ( window.localStorage ) {
				return window.localStorage [ k ] || '';
			} else if ( navigator.cookieEnabled ) {
				m = document.cookie.match ( k + '=([^;]*)' );
				return ( m && unescape ( m [ 1 ])) || '';
			} else {
				return '';
			}
		}

		Krux.user = retrieve ( 'user' );
		Krux.segments = retrieve ( 'segs' ) && retrieve ( 'segs' ).split ( ',' ) || [];
	}) ();
</script>

<!-- Grapeshot Implementation -->
<script>
	var gs_channels = 'default';
</script>

<script src="channels.cgi" async=""></script>

<!-- DoubleClick Implementation -->
<script type="text/javascript">
	var $WileyGooglePublisherID = 2215;


  if (!$DoubleClickSite) {
		var $DoubleClickSite = "undefined-site";
	};

	if (!$DoubleClickZone) {
		var $DoubleClickZone = "undefined-zone";
	};

	var $GooglePublisherCategory = "/" + $WileyGooglePublisherID + "/" + $DoubleClickSite + "/" + $DoubleClickZone;
</script>

<script>
	var googletag = googletag || {};

	googletag.cmd = googletag.cmd || [];

	( function () {

		var gads = document.createElement ( 'script' ) ,
		node = document.getElementsByTagName ( 'script' ) [ 0 ] ;

		gads.async = true;
		gads.src = document.location.protocol + '//www.googletagservices.com/tag/js/gpt.js';

		node.parentNode.insertBefore ( gads , node );

	}) ();
</script>

<script>
	googletag.cmd.push ( function () {

		var sizes = {
			leaderboard : googletag.sizeMapping ()
				.addSize ([ 300 , 100 ] , [ 300 , 50 ])
				.addSize ([ 740 , 100 ] , [ 728 , 90 ])
				.addSize ([ 0 , 0 ] , 'fluid')
				.build () ,
			rail : googletag.sizeMapping ()
				.addSize ([ 0 , 0 ] , [ 300 , 250 ])
				.build ()
		};

		googletag.pubads ()
			.setTargeting ( 'property' 	, 'value-of-property' 	) 	// Change and duplicate this line for as many additional keywords to send
			.setTargeting ( 'doi' , '10.1111/mec.14478') // Publication DOI
			.setTargeting ( 'gs_cat' 	, gs_channels 		) 	// Grapeshot
			.setTargeting ( 'ksg' 		, Krux.segments 	) 	// Krux
			.setTargeting ( 'kuid' 		, Krux.user 		); 	// Krux

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-leaderboard' )
			.defineSizeMapping ( sizes.leaderboard )
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-1' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-1')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-2' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-2')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.pubads ().enableSingleRequest ();
		googletag.enableServices ();
	});
	//]]>
</script>








    
        
    








    


<script>
    var digitalData = window.digitalData ? window.digitalData : {};
    digitalData.page.title = "Scaling up: A guide to high‐throughput genomic approaches for biodiversity analysis - Porter - 2018 - Molecular Ecology - Wiley Online Library";
</script>

<script>
    tracking = {};

    // events which don't generate server side events!
    tracking.adtmOnlyEvents = ["readcube.share.createlink", "readcube.ecommerce.load", "readcube.ecommerce.new",
        "readcube.ecommerce.userinfo", "readcube.ecommerce.register", "readcube.ecommerce.billing", "readcube.ecommerce.final",
        "readcube.ecommerce.purchasefailed"];

    tracking.ajaxData = function (eventName) {
        return {
            url: "/readcube/log",
            data: {
                "doi": digitalData.publication.item.doi,
                event: eventName
            }
        }
    };

    tracking.trackEvent = function (eventName, eventData, callback) {

        if (!tracking.adtmOnlyEvents.indexOf(eventName) >= 0) {
            tracking.trackUsage(eventName, callback);
        }
        tracking.trackAnalytics(eventName, eventData);
    };

    tracking.trackUsage = function (eventName, callback) {
        pr = $.ajax(tracking.ajaxData(eventName));

        if (callback) {
            pr.always(callback);
        }

    };

    tracking.trackAnalytics = function (eventName, eventData) {
        digitalData = window.digitalData ? digitalData : {};
        digitalData.event = digitalData.event || [];
        digitalData.event.push({
            "eventName": eventName,
            "eventData": eventData
        });
    }
</script>


    <script src="epdf_linker.js" async=""></script>
<script async="" type="text/javascript" src="epdf_linker_002.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019032901.js" async=""></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><script src="satellite-5b4e5b3364746d482f001123.js"></script><script src="satellite-5c58af5e64746d5111000925.js"></script><script src="satellite-5c87cfcf64746d7044002f7b.js"></script><script src="analytics.js"></script><script src="s-code-contents-7d04accfbc0deee918eb104cc0a954ffa836b5dd.js"></script><script src="satellite-5ad74ae364746d5a68016a67.js"></script><script src="satellite-5c87b56e64746d4e33003a7c.js"></script><link href="style.css" type="text/css" rel="stylesheet"><style type="text/css">.fancybox-margin{margin-right:0px;}</style></head>
<body class="pb-ui">

<div class="skip-links">
    <ul class="rlist">
        
                <li><a href="#article__content">Skip to Article Content</a></li>
                <li><a href="#pane-pcw-detailscon">Skip to Article Information</a></li>
            
    </ul>
</div>




    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="pageHeader" data-widget-id="62175478-b98a-43d5-90f2-1f2045ac1a7e" class="pageHeader fixed-element">
        



        
        <header class="page-header">
    <div data-pb-dropzone="main">
    









        <div data-widget-def="ux3-layout-widget" data-widget-id="bb5b5e2c-d3d6-4015-b586-eeade38f59d4" class="admin-message">
        



        
        <div class="container"><div class="row"><div>



        
        


        
    

</div></div></div>

        </div>
    




        
        




        
        <div class="container"><div class="row"><div class="contents col-sm-12 col-md-12">









        <div data-widget-def="UX3InstitutionBanner" data-widget-id="0a735273-4141-44d8-801a-2b89143bed55" class="institution-wrapper-mobile hidden-lg padded-content">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access By</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    




        
        <div class="pull-left">









        <div data-widget-def="ux3-general-image" data-widget-id="ff262197-0fb8-4973-aaa2-2fceba4b5a61" class="logo-container hidden-xs hidden-sm hidden-md">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library" tabindex="1"><img alt="Wiley Online Library" src="logo-header-1526603583437.png"></a>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="9a75b789-224c-4745-833a-930cf6c36f35" class="logo-container-mobile hidden-lg hidden-md hidden-sm">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img alt="Wiley Online Library" src="mobilehublogo-1518781861020.png"></a>

        </div>
    










        <div data-widget-def="UX3InstitutionBanner" data-widget-id="14d0e2f8-27d3-42d0-a477-72015e14e818" class="institution-wrapper hidden-xs hidden-sm hidden-md institution--block">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access by</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    
</div><div class="pull-right">









        <div data-widget-def="UX3QuickSearchWidget" data-widget-id="48175bb0-0f70-46e6-bb89-7e2781fdb8e4" class="header-quicksearch">
        



        
        <div class="quick-search-wrapper"><a href="#" title="search" aria-label="Search" class="quick-search__item"><i aria-hidden="true" class="icon-search"></i><span class="quick-search__item__arrow"></span></a><div data-linkto-page="true" class="quick-search quickSearchFormContainer"><div class="main-search-field-wrapper" style="overflow: hidden;"><div class="tab clearfix"><input placeholder="Enter Your Search Term" tabindex="13" class="fakeQSInput" type="search"><label for="search-within" class="sr-only">Search within</label><select name="search-within" id="search-within" role="tablist" tabindex="10"><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c0" selected="selected">This Journal</option><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c1">Anywhere</option></select><ul role="tablist" class="rlist tab__content"><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c0" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c0-con" role="tab" class="tab__pane quick-search-pane__anywhere active"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" title="Quick Search"><div class="input-group option-0"><label for="searchField0" class="hiddenLabel">Search term</label><input name="AllField" id="searchField0" placeholder="Enter Your Search Term" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><input name="SeriesKey" value="1365294x" type="hidden"><div class="search-options-wrapper" style="overflow: hidden; height: 0px; padding: 0px; margin: 15px 0px 0px; width: 0px; opacity: 1;"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c1" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c1-con" role="tab" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" title="Quick Search"><div class="input-group option-1"><label for="searchField1" class="hiddenLabel">Search term</label><input name="AllField" id="searchField1" placeholder="Enter Your Search Term" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><div class="search-options-wrapper" style="overflow: hidden; height: 0px; padding: 0px; margin: 15px 0px 0px; width: 0px; opacity: 1;"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li></ul></div></div></div></div>

        </div>
    










        <div data-widget-def="literatumNavigationLoginBar" data-widget-id="d384ca92-7137-45d6-b3c2-78c9c0a98c2b" class="login-bar-container">
        



        
        




    









    <div class="loginBar login-open">
        <a href="https://onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2Ffull%2F10.1111%2Fmec.14478&amp;aria-label=Log+in+or+Register" class="show-login" role="button" uri="/doi/full/10.1111/mec.14478" aria-label="Log in or Register" tabindex="30">
            <span class="icon-profile_empty"></span>
            <span class="sign-in-label">Login / Register</span>
        </a>
            
    </div>



        </div>
    
</div>
</div></div></div>
    </div>
</header>


        </div>
    




        
        




        
        




        
        <!-- [Tag{id&#x3D;320, uri&#x3D;ls20, name&#x3D;Ecology &amp; Organismal Biology}, Tag{id&#x3D;688, uri&#x3D;ch00, name&#x3D;General Chemistry}, Tag{id&#x3D;1184, uri&#x3D;ls2f, name&#x3D;Ecological Genetics}, Tag{id&#x3D;323, uri&#x3D;ls40, name&#x3D;Evolution}, Tag{id&#x3D;1206, uri&#x3D;ls43, name&#x3D;Molecular Evolution}, Tag{id&#x3D;1207, uri&#x3D;ls41, name&#x3D;Evolutionary Biology}, Tag{id&#x3D;60, uri&#x3D;ls, name&#x3D;LIFE SCIENCES}, Tag{id&#x3D;1180, uri&#x3D;ls2g, name&#x3D;Molecular Ecology}, Tag{id&#x3D;93, uri&#x3D;ch, name&#x3D;CHEMISTRY}, Tag{id&#x3D;317, uri&#x3D;ls60, name&#x3D;Genetics}, Tag{id&#x3D;101738, uri&#x3D;wly.ecology_000046, name&#x3D;wly.ecology_000046}, Tag{id&#x3D;103071, uri&#x3D;molec-eco_mec, name&#x3D;molec-eco_MEC}] -->                          <div id="iucr-failure"> <p><i class="icon-notice"></i>The full text of this article hosted at iucr.org is unavailable due to technical difficulties.<a href="#" class="close-iucr-failure"><i class="icon-x"></i></a> </p> </div> 












        <div data-widget-def="pageBody" data-widget-id="72100436-7a82-49fc-933b-c6c9d8c42914" class="pageBody article-page-tablet body-text">
        



        
        <main id="main-content" tabindex="-1" style="">
    <!--begin pagefulltext-->
    <div class="page-body pagefulltext">
        <div data-pb-dropzone="main">
        



        
        <div>









        <div data-widget-def="literatumAd" data-widget-id="0220a30b-90c0-4b51-8637-141edf12f86a" class="banner-wrapper">
        



        
        


        <div class="pb-ad">
            <aside role="complementary">
		<div id="advert-leaderboard" class="advert advert-leaderboard">
			<script>
				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-leaderboard' );
				});
			</script>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1013&url=http%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fmec.14478&pubId=62117837&placeholderId=1015&productId=101018" target="_blank">
					<img src="http://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
				</a>
			</noscript>
		<div id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_0__container__" style="border: 0pt none; width: 100%; height: 0%;"></div></div>
	</aside>
        </div>
    


        </div>
    




        
        
</div>










        <section data-widget-def="ux3-layout-widget" data-widget-id="0f7bccba-8328-4dbd-a1fa-e84c526dd9ab">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">



        
        <div>



        
        <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fmec.14478&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2018.27.issue-2%3Bjournal%3Ajournal%3A1365294x%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fmec.14478%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fmec.14478&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2018.27.issue-2%3Bjournal%3Ajournal%3A1365294x%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fmec.14478%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fmec.14478&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2018.27.issue-2%3Bjournal%3Ajournal%3A1365294x%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fmec.14478%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fmec.14478&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2018.27.issue-2%3Bjournal%3Ajournal%3A1365294x%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fmec.14478%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><div class="row article-row"><div id="article__content" tabindex="-1" class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone="mrwBannerDropZone" class="mrw-banner-dropzone">



        
        
</div><div class="article-citation"><div class="citation">
   <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-series-title">
      <div data-pb-dropzone="publicaitonContent-series-title" class="pb-dropzone">
  <h1><a href="https://onlinelibrary.wiley.com/journal/1365294x" title="Molecular Ecology homepage">Molecular Ecology</a></h1>
  <a href="https://onlinelibrary.wiley.com/toc/1365294x/27/2" class="volume-issue">Volume 27, Issue 2</a>
  <a href="https://onlinelibrary.wiley.com/journal/1365294x" title="Molecular Ecology homepage" class="citation--logo"><img class="journal-banner-image" test="test-value" alt="Molecular Ecology banner" src="1365294x-1525974682817.jpg"></a>
</div>
   </div>
   <div class="doi-access-container clearfix"><span class="primary-heading">INVITED REVIEWS AND SYNTHESES</span>
      <div class="doi-access-wrapper"><div class="open-access access-type"> <i aria-hidden="true" class="icon-icon-lock_open"></i> <div class="doi-access"> <div class="doi-access" tabindex="0">Open Access</div></div></div></div><div class="doi-access-wrapper"><a target="_blank" class="access-icon-wrapper" aria-label="Creative Commons license information" href="http://creativecommons.org/licenses/by/4.0/"><span aria-hidden="true" class="access-icon icon-cc"></span><span aria-hidden="true" class="access-icon icon-by"></span></a></div>
   </div>
   <h2 class="citation__title">Scaling up: A guide to high‐throughput genomic approaches for biodiversity analysis</h2>
   <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc">
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Porter%2C+Teresita+M" class="author-name accordion-tabbed__control" data-id="am1" data-db-target-for="am1" aria-controls="am1" aria-haspopup="true" id="am1_Ctrl"><span>Teresita M. Porter</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am1" aria-labelledby="am1_Ctrl" id="am1">
            <p class="orcid-account"><i aria-hidden="true" class="icon-orcid"></i><a target="_blank" href="http://orcid.org/0000-0002-0227-6874">http://orcid.org/0000-0002-0227-6874</a></p>
            <p>Centre for Biodiversity Genomics, Biodiversity Institute 
of Ontario and Department of Integrative Biology, University of Guelph, 
Guelph, ON, Canada</p>
            <p>Natural Resources Canada, Great Lakes Forestry Centre, Sault Ste. Marie, ON, Canada</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Porter%2C+Teresita+M">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Hajibabaei%2C+Mehrdad" class="author-name accordion-tabbed__control" data-id="am2" data-db-target-for="am2" aria-controls="am2" aria-haspopup="true" id="am2_Ctrl"><span>Mehrdad Hajibabaei<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am2" aria-labelledby="am2_Ctrl" id="am2">
            <p class="author-type">Corresponding Author</p>
            <p>
               </p><p><span>E-mail address: </span><a title="
                            Link to email address
                        " href="mailto:mhajibab@uoguelph.ca">mhajibab@uoguelph.ca</a></p>
            <p></p>
            <p class="orcid-account"><i aria-hidden="true" class="icon-orcid"></i><a target="_blank" href="http://orcid.org/0000-0002-8859-7977">http://orcid.org/0000-0002-8859-7977</a></p>
            <p>Centre for Biodiversity Genomics, Biodiversity Institute 
of Ontario and Department of Integrative Biology, University of Guelph, 
Guelph, ON, Canada</p>
            <div class="bottom-info">
               <p><b>Correspondence</b></p>
               <p>Mehrdad Hajibabaei, Centre for Biodiversity Genomics 
&amp; Department of Integrative Biology, University of Guelph, Guelph, 
ON, Canada.</p>
               <p>Email: <a title="
                        Link to email address
                    " href="mailto:mhajibab@uoguelph.ca" class="corr-email">mhajibab@uoguelph.ca</a></p>
            </div><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Hajibabaei%2C+Mehrdad">Search for more papers by this author</a></div>
      </div>
   </div>
   <div class="loa-wrapper loa-authors hidden-xs">
      <div id="sb-1" class="accordion">
         <div class="accordion-tabbed">
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Porter%2C+Teresita+M" class="author-name accordion-tabbed__control" data-id="a1" data-db-target-for="a1" aria-controls="a1" aria-haspopup="true" id="a1_Ctrl"><span>Teresita M. Porter</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a1" aria-labelledby="a1_Ctrl" id="a1">
                  <p class="orcid-account"><i aria-hidden="true" class="icon-orcid"></i><a target="_blank" href="http://orcid.org/0000-0002-0227-6874">http://orcid.org/0000-0002-0227-6874</a></p>
                  <p>Centre for Biodiversity Genomics, Biodiversity 
Institute of Ontario and Department of Integrative Biology, University 
of Guelph, Guelph, ON, Canada</p>
                  <p>Natural Resources Canada, Great Lakes Forestry Centre, Sault Ste. Marie, ON, Canada</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Porter%2C+Teresita+M">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Hajibabaei%2C+Mehrdad" class="author-name accordion-tabbed__control" data-id="a2" data-db-target-for="a2" aria-controls="a2" aria-haspopup="true" id="a2_Ctrl"><span>Mehrdad Hajibabaei<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a2" aria-labelledby="a2_Ctrl" id="a2">
                  <p class="author-type">Corresponding Author</p>
                  <p>
                     </p><p><span>E-mail address: </span><a title="
                            Link to email address
                        " href="mailto:mhajibab@uoguelph.ca">mhajibab@uoguelph.ca</a></p>
                  <p></p>
                  <p class="orcid-account"><i aria-hidden="true" class="icon-orcid"></i><a target="_blank" href="http://orcid.org/0000-0002-8859-7977">http://orcid.org/0000-0002-8859-7977</a></p>
                  <p>Centre for Biodiversity Genomics, Biodiversity 
Institute of Ontario and Department of Integrative Biology, University 
of Guelph, Guelph, ON, Canada</p>
                  <div class="bottom-info">
                     <p><b>Correspondence</b></p>
                     <p>Mehrdad Hajibabaei, Centre for Biodiversity 
Genomics &amp; Department of Integrative Biology, University of Guelph, 
Guelph, ON, Canada.</p>
                     <p>Email: <a title="
                        Link to email address
                    " href="mailto:mhajibab@uoguelph.ca" class="corr-email">mhajibab@uoguelph.ca</a></p>
                  </div><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Hajibabaei%2C+Mehrdad">Search for more papers by this author</a></div>
            </div>
         </div>
      </div>
   </div>
   <div class="epub-sections">
      <div class="epub-section"><span class="epub-state">First published:&nbsp;</span><span class="epub-date">02 January 2018</span></div>
      <div class="epub-section"><a class="epub-doi" aria-label="Digital Object Identifier" href="https://doi.org/10.1111/mec.14478">https://doi.org/10.1111/mec.14478</a></div>
      <div class="epub-section cited-by-count"><span>Cited by: <a href="#citedby-section">17</a></span></div>
   </div>
   <div class="epub-sections"></div>
</div></div><nav class="stickybar coolBar trans"><div class="stickybar__wrapper coolBar__wrapper clearfix"><div class="rlist coolBar__zone"><div class="coolBar__section"><a href="#" data-db-target-for="article" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="article_Pop" role="button" id="article_Ctrl" class="coolBar__ctrl hidden-lg"><i class="icon-abstract"></i><span>About</span></a><div data-db-target-of="article" aria-labelledby="article_Ctrl" role="menu" id="article_Pop" class="coolBar__drop fixed rlist dropBlock__holder"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove="false" data-targetclass="hidden-xs hidden-sm" data-toggle="transplant" data-direction="from" data-transplant="self" class="transplant showit"><div class="transplanted-clone"><ul data-mobile-toggle="slide" class="rlist tab__nav w-slide--list"><li role="presentation" class=""><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" aria-expanded="false"><i aria-hidden="true" class="icon-multi-figure"></i><span>Figures</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty"><i aria-hidden="true" class="icon-chain"></i><span>References</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><span>Related</span></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" aria-expanded="true"><i aria-hidden="true" class="icon-Icon_Information"></i><span>Information</span></a></li></ul><div class="coolBar__section coolBar--download readcubeEPdfLink"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/mec.14478" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink scroll-to-target pdfLink--is-hidden"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/mec.14478" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div></div></div></div></div><div class="coolBar__section coolBar--sections"><a href="#" data-db-target-for="sections" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="sections_Pop" role="button" id="sections_Ctrl" class="coolBar__ctrl"><i class="icon-list-unordered"></i><span>Sections</span></a><div data-db-target-of="sections" aria-labelledby="sections_Ctrl" role="menu" id="sections_Pop" class="coolBar__drop dropBlock__holder rlist fixed"><ul class="rlist w-slide--list"><li role="menuitem"><a class="w-slide__hide" href="#d9996652"><span>Abstract</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0001-title"><span>1 INTRODUCTION</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0002-title"><span>2 MICROARRAYS</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0003-title"><span>3 QUANTITATIVE POLYMERASE CHAIN REACTION (QPCR)</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0004-title"><span>4 DIGITAL PCR</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0005-title"><span>5 DNA METABARCODING</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0032-title"><span> METADATA</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0033-title"><span> REFERENCE DATABASES</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0034-title"><span> SEQUENCE READ ERRORS</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0035-title"><span> DETERMINING THE BASIC UNIT FOR DNA‐BASED BIODIVERSITY ANALYSES</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0036-title"><span> CHOOSING A SEQUENCE CLUSTERING METHOD</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0037-title"><span> TAXONOMIC ASSIGNMENT</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0038-title"><span> DATA NORMALIZATION</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0011-title"><span>6 ORGANELLE SEQUENCING</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0012-title"><span>7 GENOME SKIMMING</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0013-title"><span>8 WHOLE‐GENOME SEQUENCING</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0026-title"><span> OTU DELIMITATION</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0027-title"><span> TAXONOMIC ASSIGNMENT</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0028-title"><span> PHYLOGENOMICS</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0029-title"><span> PHYLOGENETIC DIVERSITY</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0015-title"><span>9 METAGENOMICS</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0016-title"><span>10 METATRANSCRIPTOMICS</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0017-title"><span>11 TARGET ENRICHMENT</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0018-title"><span>12 FUTURE OUTLOOK AND CHALLENGES</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec14478-sec-0023-title"><span> AUTHOR CONTRIBUTIONS</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#references-section-1"><span>REFERENCES</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#citedby-section"><span>Citing Literature</span></a></li></ul></div></div></div><ul data-cb-group="Article" data-cb-group-icon="icon-toc" class="rlist coolBar__first"></ul><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download readcubeEPdfLink cloned"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/mec.14478" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink scroll-to-target pdfLink--is-hidden cloned"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/mec.14478" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div>



        
        <div class="relative"><a href="#" data-db-target-for="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" role="button" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" class="article-tools__ctrl"><i aria-hidden="true" class="icon-Icon_Tools"></i><span>Tools</span></a><div data-db-target-of="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" aria-labelledby="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" role="menu" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" class="article-tools__block fixed dropBlock__holder"><ul class="rlist w-slide--list"><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/rightsLink?doi=10.1111%2Fmec.14478&amp;mode=" role="menuitem" target="_blank"><i aria-hidden="true" class="icon-permissions-key"></i><span>Request permission</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/showCitFormats?doi=10.1111%2Fmec.14478" role="menuitem" target="_self"><i aria-hidden="true" class="icon-export-icon"></i><span>Export citation</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/personalize/addFavoritePublication?doi=10.1111%2Fmec.14478" role="menuitem" target="_self"><i aria-hidden="true" class="icon-favorites"></i><span>Add to favorites</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/addCitationAlert?doi=10.1111%2Fmec.14478" role="menuitem" target="_self"><i aria-hidden="true" class="icon-Icon_Track-citations"></i><span>Track citation</span></a></li></ul></div></div>




        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" src="addthis_widget.js"></script><div class="relative"><a id="share__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="share_Pop" role="button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><div data-db-target-of="dff42abe-4b17-4fe9-a9c5-625405cd1007" aria-labelledby="share__ctrl" role="menu" id="share_Pop" class="share__block dropBlock__holder fixed">



        
        <div data-readcube="/pb/widgets/readCubeP2PUrl/10.1111/mec.14478/" class="readCube-sharing hidden"><p>Give access</p><a href="#" data-toggle="modal" data-target="#readCube" class="readCube-sharing__link"><i class="icon-users"></i>Share full text access</a></div><div class="ux-modal-container readCube-sharing__modal"><div id="readCube" class="modal"><div class="modal__dialog"><div class="modal__header"><button type="button" data-dismiss="modal" class="close"><i aria-hidden="true" class="icon-close_thin"></i></button><p>Share full text access</p></div><div class="modal__body"><form method="post"><fieldset><legend>Please review our <a href="https://onlinelibrary.wiley.com/termsAndConditions" target="_blank">Terms and Conditions of Use</a> and check box below to share full-text version of article.</legend><div class="input-group"><label for="terms-and-conditions" class="checkbox--primary"><input id="terms-and-conditions" value="yes" required="" name="terms-and-conditions" type="checkbox"><span class="label-txt">I have read and accept the Wiley Online Library Terms and Conditions of Use.</span></label></div></fieldset><hr class="separator"><fieldset><label>Shareable Link</label><p>Use the link below to share a full-text version of this article with your friends and colleagues. <a href="https://onlinelibrary.wiley.com/researchers/tools-resources/sharing" target="_blank" class="emphasis">Learn more.</a></p><div class="shareable-link"><div class="shareable-link__field"><span id="shareable-link__text"></span></div></div><button type="submit" disabled="disabled" class="btn shareable-link__btn">Copy URL</button></fieldset></form></div></div></div></div>
<p>Share a link</p><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a class="addthis_button_email" target="_blank" title="Email" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i>Email</a></li><li><a class="addthis_button_facebook" title="Facebook" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a class="addthis_button_twitter" title="Twitter" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-twitter-t"></i>Twitter</a></li><li><a class="addthis_button_linkedin" target="_blank" title="LinkedIn" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a class="addthis_button_reddit" target="_blank" title="Reddit" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-reddit-alien"></i>Reddit</a></li><div class="atclear"></div></ul></div></div>
</div></div></nav><div id="accessDenialslot" class="accessDenialslot"></div><div class="article__body"><style>
            .inline-equation {
            overflow-x:auto;
            }
        </style><article>
   
   <div class="abstract-group">
      <section class="article-section article-section__abstract" data-lang="en" id="section-1-en" style="" lang="en">
         <h2 id="d9996652" class="article-section__header section__title main abstractlang_en main">Abstract</h2>
         <div class="article-section__content en main">
            
            <p>The purpose of this review is to present the most common 
and emerging DNA‐based methods used to generate data for biodiversity 
and biomonitoring studies. As environmental assessment and monitoring 
programmes may require biodiversity information at multiple levels, we 
pay particular attention to the DNA metabarcoding method and discuss a 
number of bioinformatic tools and considerations for producing DNA‐based
 indicators using operational taxonomic units (OTUs), taxa at a variety 
of ranks and community composition. By developing the capacity to 
harness the advantages provided by the newest technologies, 
investigators can “scale up” by increasing the number of samples and 
replicates processed, the frequency of sampling over time and space, and
 even the depth of sampling such as by sequencing more reads per sample 
or more markers per sample. The ability to scale up is made possible by 
the reduced hands‐on time and cost per sample provided by the newest 
kits, platforms and software tools. Results gleaned from broad‐scale 
monitoring will provide opportunities to address key scientific 
questions linked to biodiversity and its dynamics across time and space 
as well as being more relevant for policymakers, enabling science‐based 
decision‐making, and provide a greater socio‐economic impact. As genomic
 approaches are continually evolving, we provide this guide to methods 
used in biodiversity genomics.</p>
            
         </div>
      </section>
   </div>
   <div class="pb-dropzone" data-pb-dropzone="below-abstract-group">
      <!-- Empty dropzone -->
   </div>
   
   <section class="article-section article-section__full">
      
      <div class="article-section__content" id="mec14478-sec-0001">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0001-title">1 INTRODUCTION</h2>
         
         <p>Biodiversity encompasses the diversity of organisms, their 
relationships and their functions within ecosystems. Biodiversity 
assessment using traditional methods mainly involves identifying 
morphological characters whose states can be compared with taxonomic 
keys for species identification. It is not uncommon that characters 
needed for taxonomic assignment are not present or are difficult to 
discern even for highly experienced taxonomists. The process of 
identifying smaller taxa such as insects or microscopic organisms from 
environmental samples often continues well beyond the field collection 
season, and the results are taxonomic assignments with varying degrees 
of resolution that is dependent on the availability of taxonomic keys, 
expertise of the taxonomist and condition of the sample. The issues vary
 by organism: for insects, damaged or juvenile specimens may not contain
 the characters needed for identification (Sweeney, Battle, Jackson, 
&amp; Dapkey, <span><a href="#mec14478-bib-0167" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>);
 for fungi, bacteria and other microscopic organisms, it may be the 
difficulty in isolating and culturing individuals or the collection of 
samples in life stages that lack the characters needed for 
identification (Bridge &amp; Spooner, <span><a href="#mec14478-bib-0021" class="bibLink tab-link" data-tab="pane-pcw-references">2001</a></span>).
 These are just a few of the impediments to the morphology‐based 
identification process that also affect downstream users of taxonomic 
data for biodiversity research (Ebach, Valdecasas, &amp; Wheeler, <span><a href="#mec14478-bib-3000" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 In biomonitoring studies, sampling needs to be repeated across time and
 space. Essentially, the pace that samples are collected in large‐scale 
biomonitoring programmes can quickly exceed the capacity for taxonomists
 to identify them in a timely manner and this is where DNA‐based methods
 can help investigators identify more taxa using methods capable of 
processing large data sets. In this review, we specifically address how 
DNA‐based methods with high‐throughput potential are scalable, that is, 
methods that can be easily adapted to process larger numbers of samples 
collected across time and space as well as automatically identify larger
 numbers of taxa from large‐scale studies.
         </p>
         
         <p>Biodiversity genomics integrates different data types such 
as biological indicators from traditional specimen collection, biomass 
estimates and biological activity assessments; environmental indicators 
that describe site characteristics; as well as DNA‐based indicators 
which are the focus of this review (Figure&nbsp;<a href="#mec14478-fig-0001">1</a>).
 The use of DNA‐based methods to both detect organisms and assign 
putative functions without having to isolate or identify individuals 
from environmental samples has been revolutionary in many different 
fields (Aylagas, Borja, Irigoien, &amp; Rodríguez‐Ezpeleta, <span><a href="#mec14478-bib-0007" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>; Elbrecht &amp; Leese, <span><a href="#mec14478-bib-0053" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>; Gibson et&nbsp;al., <span><a href="#mec14478-bib-0062" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>; Hajibabaei, Shokralla, Zhou, Singer, &amp; Baird, <span><a href="#mec14478-bib-0073" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>; Horton &amp; Bruns, <span><a href="#mec14478-bib-0083" class="bibLink tab-link" data-tab="pane-pcw-references">2001</a></span>; Langille et&nbsp;al., <span><a href="#mec14478-bib-0097" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>; Nguyen et&nbsp;al., <span><a href="#mec14478-bib-0122" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>; O'Brien, Parrent, Jackson, Moncalvo, &amp; Vilgalys, <span><a href="#mec14478-bib-0124" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>; Taberlet, Coissac, Hajibabaei, &amp; Rieseberg, <span><a href="#mec14478-bib-0168" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; Torsvik &amp; Ovreas, <span><a href="#mec14478-bib-0179" class="bibLink tab-link" data-tab="pane-pcw-references">2002</a></span>).
 In particular, high‐throughput sequencing (HTS) of environmental DNA 
(eDNA) for biomonitoring applications has been referred to as <span class="smallCaps">biomonitoring</span> 2.0 (Baird &amp; Hajibabaei, <span><a href="#mec14478-bib-0009" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 We use the term eDNA loosely to include the free degraded DNAs in the 
environment; DNA emitted from living organisms through their DNA 
secretions, faeces and shed cells; DNA contained within dead or dormant 
cells such as seeds, spores and sclerotia; as well as the DNA from whole
 organisms that are also recovered during the extraction process 
(Bohmann et&nbsp;al., <span><a href="#mec14478-bib-0018" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>; Pietramellara et&nbsp;al., <span><a href="#mec14478-bib-0132" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>; Taberlet, Prud'homme, et&nbsp;al., <span><a href="#mec14478-bib-0170" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 Biodiversity genomic methods are analogous in many ways to microbiome 
studies in agricultural or biomedical studies. Methods that start with 
total DNA extraction from a bulk sample such as soil, water or sediments
 can detect what organisms are present or the genomic potential of a 
community (Bik et&nbsp;al., <span><a href="#mec14478-bib-0016" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 The natural persistence and turnover of DNA in the environment result 
in a snapshot of community diversity within a certain window of time 
that depends on specimen biomass and density, temperature and trophic 
status of the system (Dejean et&nbsp;al., <span><a href="#mec14478-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 Biological indicators such as presence, abundance, or relative 
abundance of OTUs, taxa or communities can be derived from eDNA using 
the methods described below (Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>, Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec14478-fea-0001">1</a>).
         </p>
         
         <section class="article-section__inline-figure">
            <figure class="figure" id="mec14478-fig-0001"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/2eeb1422-5547-482a-9335-ee330c47762d/mec14478-fig-0001-m.jpg">
                  <picture>
                     <source srcset="https://wol-prod-cdn.literatumonline.com/cms/attachment/2eeb1422-5547-482a-9335-ee330c47762d/mec14478-fig-0001-m.jpg" media="(min-width: 1650px)"><img class="figure__image" src="mec14478-fig-0001-m.png" data-lg-src="https://wol-prod-cdn.literatumonline.com/cms/attachment/2eeb1422-5547-482a-9335-ee330c47762d/mec14478-fig-0001-m.jpg" alt="image" title="image"></picture></a><figcaption class="figure__caption">
                  <div class="figure__caption__header"><strong class="figure-title">Figure 1</strong> 
                     <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=mec14478-fig-0001&amp;doi=10.1111%2Fmec.14478" class="ppt-figure-link">PowerPoint</a></div>
                  </div>
                  <div>Integration of data types in biodiversity 
genomics. Boxes outline the various ways biodiversity can be sampled 
using DNA‐based or traditional methods that use biological and 
environmental ecological indicators</div>
               </figcaption>
            </figure>
         </section>
         
         <div class="article-table-content" id="mec14478-tbl-0001">
            <header class="article-table-caption"><span class="table-caption__label">Table 1.
            </span>Summary of biodiversity genomic methods
            </header>
            
            
            
            <div class="article-table-content-wrapper">
               <table class="table article-section__table">
                  
                  
                  
                  
                  
                  
                  
                  <thead>
                     
                     <tr>
                        
                        <th class="bottom-bordered-cell right-bordered-cell left-aligned">Method</th>
                        
                        <th class="bottom-bordered-cell left-aligned">Uses</th>
                        
                        <th class="bottom-bordered-cell left-aligned">Pros</th>
                        
                        <th class="bottom-bordered-cell left-aligned">Cons</th>
                        
                        <th class="bottom-bordered-cell left-aligned">Scalability</th>
                        
                        <th class="bottom-bordered-cell left-aligned">Future outlook</th>
                        
                     </tr>
                     
                  </thead>
                  
                  <tbody>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Microarrays (F;I/C)</td>
                        
                        <td class="left-aligned">Detects the presence or
 relative abundance of suites of markers or expressed transcripts from 
an individual specimen or a community sample</td>
                        
                        <td class="left-aligned">Simplified data analysis compared with the bioinformatics processing required with HTS methods</td>
                        
                        <td class="left-aligned">Not an appropriate method for taxon discovery. Data produced are fluorescence signals not sequences</td>
                        
                        <td class="left-aligned">Requires one single‐use chip per sample. Not easily scalable</td>
                        
                        <td class="left-aligned">Likely to be replaced 
by sequencing‐based methods below as cost per sequence continues to 
decrease and bioinformatic pipelines become easier to use</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Quantitative PCR (qPCR) (F;I/C)</td>
                        
                        <td class="left-aligned">Detects the presence, abundance or relative abundance of a single marker from an individual specimen or a community sample</td>
                        
                        <td class="left-aligned">More sensitive than PCR. Can be used to quantify starting template amounts</td>
                        
                        <td class="left-aligned">Can be complicated by 
the background mixture of organismal DNAs co‐isolated from eDNA samples.
 Data produced are fluorescence signals not sequences</td>
                        
                        <td class="left-aligned">Highly scalable as many samples can be run in batches on plates</td>
                        
                        <td class="left-aligned">Likely to remain a 
useful tool where rapid monitoring is needed to detect target taxa until
 integrated microfluidic devices are further developed and become a more
 cost‐effective option</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Digital PCR (dPCR) (F;I/C)</td>
                        
                        <td class="left-aligned">See <i>qPCR</i> above
                        </td>
                        
                        <td class="left-aligned">See <i>qPCR</i> above. 
Digital PCR is more sensitive than PCR or qPCR. As only a 
single‐template molecule is amplified per reaction, this avoids the 
problem of PCR bias with eDNA samples. Can be used as a target 
enrichment method prior to HTS
                        </td>
                        
                        <td class="left-aligned">Requires the purchase of specialized equipment. Data produced are fluorescence signals not sequences</td>
                        
                        <td class="left-aligned">Current platforms are 
scalable in terms of the number of parallel PCRs reactions that occur, 
but typically, only a single sample can be run at a time</td>
                        
                        <td class="left-aligned">This platform is ideal 
for analysing eDNA samples, but the need to purchase specialized 
equipment may be a barrier to widespread adoption among laboratories 
that already have established qPCR protocols</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Metabarcoding (S;C)</td>
                        
                        <td class="left-aligned">Gene marker surveys are currently the most popular method for biodiversity and DNA‐based biomonitoring from a community sample</td>
                        
                        <td class="left-aligned">Reference databases and
 taxonomic assignment tools are available for the most widely used DNA 
barcoding regions. Phylogenetic and statistical analysis of ESVs or OTUs
 can be conducted even without the identification of metabarcodes</td>
                        
                        <td class="left-aligned">Identification of 
metabarcodes from eDNA samples relies on comparisons to potentially 
incomplete or inaccurate reference sequence databases. A single DNA 
marker, a short sequence or a lack of similar reference sequences may 
hinder fine‐level taxonomic assignments</td>
                        
                        <td class="left-aligned">Highly scalable when combined with HTS and bioinformatic analysis in a HPC environment, see Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec14478-fea-0003">3</a></td>
                        
                        <td class="left-aligned">This method is likely 
to be superseded by whole‐genome assembly from metagenomic sequencing 
as: (i) single‐molecule sequencing accuracy is improved, and (ii) as 
whole‐genome sequence reference databases become more representative of 
environmental diversity</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Organelle sequencing (S;I)</td>
                        
                        <td class="left-aligned">Sequencing of organelle DNA, for example, mtDNA or cpDNA from an individual specimen</td>
                        
                        <td class="left-aligned">Can provide additional sequence information for improved taxonomic assignment</td>
                        
                        <td class="left-aligned">All the markers are linked and may only reflect the evolution of the organelle as opposed to the organism as a whole</td>
                        
                        <td class="left-aligned">This method is not as scalable as methods that simply isolate eDNA without the need for prior isolation of individuals</td>
                        
                        <td class="left-aligned">As single‐molecule 
sequencing technologies develop, whole genome sequencing will likely 
become more routinely applied in biodiversity studies. This technology 
shift may replace reduced representation methods such as organelle 
sequencing and genome skimming</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Genome skimming (S;I)</td>
                        
                        <td class="left-aligned">Low‐coverage genome sequencing of an individual specimen</td>
                        
                        <td class="left-aligned">Can provide high 
sequencing coverage of high copy number or highly repetitive regions 
such as mtDNA, rDNA or cpDNA to improve taxonomic assignments. Can also 
sample additional markers that may be useful for phylogenomics</td>
                        
                        <td class="left-aligned">The isolation of a 
sufficient amount of high‐quality DNA from taxa that are not readily 
cultured using standard methods or for those with small body size may be
 difficult. Single‐copy markers of interest may not be detected</td>
                        
                        <td class="left-aligned">See <i>Organelle sequencing</i> above
                        </td>
                        
                        <td class="left-aligned">See <i>Organelle sequencing</i> above
                        </td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Whole‐genome sequencing (WGS) (S;I)</td>
                        
                        <td class="left-aligned">Genome sequencing of an individual specimen</td>
                        
                        <td class="left-aligned">Provides a reference 
sequence for the further development of molecular markers and to aid 
taxonomic assignment of sequences derived from a community sample</td>
                        
                        <td class="left-aligned">The isolation of a 
sufficient amount of high‐quality DNA from taxa that are not readily 
cultured using standard methods or for those with small body size may be
 difficult</td>
                        
                        <td class="left-aligned">See <i>Organelle sequencing</i> above
                        </td>
                        
                        <td class="left-aligned">See <i>Organelle sequencing</i> above
                        </td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Metagenomics (S;C)</td>
                        
                        <td class="left-aligned">Shotgun DNA sequencing of a community sample</td>
                        
                        <td class="left-aligned">Provides an overview of
 the metabolic potential and taxonomic diversity of a community. For 
simple communities comprised of taxa with small genomes, it is possible 
to reconstruct WGSs</td>
                        
                        <td class="left-aligned">Pathway genes or 
markers of interest may only be recovered at low frequency in a 
metagenomic data set. Taxonomic assignments based on markers that are 
not commonly used may not be possible</td>
                        
                        <td class="left-aligned">Highly scalable when combined with HTS and bioinformatic analysis in a HPC environment</td>
                        
                        <td class="left-aligned">As HTS methods develop 
and reference databases grow, this method may become more useful in 
biodiversity studies and biomonitoring applications</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Metatranscriptomics (S;C)</td>
                        
                        <td class="left-aligned">RNA‐seq (cDNA sequencing) of a community sample</td>
                        
                        <td class="left-aligned">Provides an overview of the actual metabolic activity and taxonomic diversity of a community</td>
                        
                        <td class="left-aligned">It can be challenging to extract high‐quality mRNA from environmental samples</td>
                        
                        <td class="left-aligned">See <i>Metagenomics</i> above
                        </td>
                        
                        <td class="left-aligned">Likely to see more widespread use as WGS reference databases become more representative of environmental diversity</td>
                        
                     </tr>
                     
                  </tbody>
                  
               </table>
            </div>
            
            
            
            <div class="article-section__table-footnotes">
               <ul>
                  <li id="mec14478-note-0001">
                     F&nbsp; = &nbsp;Fluorescence‐based 
detection/S&nbsp;=&nbsp;Sequencing‐based detection; I&nbsp;=&nbsp;Used 
on an individual specimen/C&nbsp;=&nbsp;Used on mixed‐community samples.
                     
                  </li>
               </ul>
            </div>
            <div class="article-section__table-source"></div>
         </div>
         
         <div class="feature" id="mec14478-fea-0001">
            <h3><span class="feature-label">Box 1. </span>Glossary
            </h3>
            <div class="article-section__content Box " id="mec14478-sec-0024">
               
               <p><i>Amplicon</i>: The short DNA sequence products of 
polymerase chain reaction (PCR) amplification using taxon‐ or 
gene‐specific primers to target a particular region of the genome.
               </p>
               
               <p><i>Biodiversity</i>: The diversity of life, their relationships and their functions within ecosystems.
               </p>
               
               <p><i>Biodiversity genomics</i>: Biodiversity assessed 
using high‐throughput DNA‐based methods or data from whole genomes 
integrated with a broad array of metadata describing biological and 
environmental indicators.
               </p>
               
               <p><i>Biomonitoring</i>: Biodiversity analysis that is 
repeated across space and time that may focus on a target organism such 
as invasive or at‐risk species, an assemblage such as the bioindicator 
groups (amphibians, birds, macroinvertebrates) as an indicator of 
ecosystem status.
               </p>
               
               <p><i>cDNA</i>: Messenger RNA reverse‐transcribed into its complementary DNA sequence.
               </p>
               
               <p><i>DNA Barcoding</i>: A minimal standardized signature
 DNA sequence is used for species identification, for example, a 658‐bp 
region of CO1 mtDNA is used for identification of animals. Other DNA 
barcode markers have been proposed for fungi, plants and protists. 16S 
rDNA has been used for the identification of bacteria.
               </p>
               
               <p><i>eDNA</i>: Environmental DNA comprised of free 
degraded DNAs in the environment as well as DNA co‐extracted from whole 
organisms such as microscopic organisms, arthropods, nematodes; shed 
cells; faeces; as well as the DNA contained within dead or dormant cells
 such as seeds or spores.
               </p>
               
               <p><i>ESV</i>: Exact sequence variant. Also known as an 
amplicon sequence variant (ASV), zero‐radius OTU (ZOTU) or simply an OTU
 defined by 100% sequence similarity.
               </p>
               
               <p><i>Genome</i>: The complete set of genetic data contained in an organism including organellar DNA.
               </p>
               
               <p><i>Genomics</i>: The sequencing and analysis of the genetic material of an organism.
               </p>
               
               <p><i>HPC</i>: High‐performance computing, computer 
clusters can be used to run the same analysis for many samples in 
parallel, or splitting large jobs into many smaller ones for a quicker 
overall runtime. Available through private clusters or third‐party cloud
 computing services.
               </p>
               
               <p><i>HTS</i>: High‐throughput sequencing, sometimes 
referred to as next‐generation sequencing or second‐generation 
sequencing. Distinguished by the high number of sequencing reactions 
that occur in parallel.
               </p>
               
               <p><i>mRNA</i>: Messenger RNA that encodes for a gene product.
               </p>
               
               <p><i>Marker</i>: A gene or signature region of DNA with a known location in the genome and can be used to identify individuals or species.
               </p>
               
               <p><i>Metadata</i>: Supplementary data linked to DNA 
sequences that provide information in a standard and searchable way such
 as organismal or bulk environmental sample description.
               </p>
               
               <p><i>Metagenomics</i>: The study of genetic material 
isolated directly from environmental samples, such as water, soil or 
sediments, may also be referred to as environmental genomics, 
ecogenomics or community genomics.
               </p>
               
               <p><i>Metatranscriptomics</i>: The study of the expressed
 portion of genomes, mRNAs, isolated directly from an environmental 
sample that may be transcribed into cDNAs for high‐throughput 
sequencing.
               </p>
               
               <p><i>Mito‐metagenomics</i>: The assembly of whole mitochondrial DNA sequences from eDNA samples.
               </p>
               
               <p><i>MIP</i>: Molecular inversion probe used for target enrichment.
               </p>
               
               <p><i>Multiplex sequencing</i>: The addition of a unique 
DNA sequence tag to each sample, such as when multiple samples are 
pooled and sequenced at the same time, allows sequences from different 
samples to be distinguished from each other during data analysis.
               </p>
               
               <p><i>Oligonucleotides</i>: Relatively short nucleotide molecules used as primers for PCR, as probes on microarrays, or baits during target enrichment.
               </p>
               
               <p><i>OTU</i>: Operational taxonomic unit, a group of similar DNA sequences sometimes used as a proxy for “species” in diversity measures.
               </p>
               
               <p><i>Primers</i>: Short oligonucleotides that are 
complementary to a particular region of the genome and are a starting 
point for DNA replication by DNA polymerase during PCR.
               </p>
               
               <p><i>rDNA</i>: Ribosomal DNA that codes for the ribosomal RNA subunits that form ribosomes.
               </p>
               
               <p><i>Super‐barcoding</i>: The use of whole‐organelle DNA sequences for species identification.
               </p>
               
               <p><i>Taxon</i>: An organism identified to any taxonomic rank (e.g., species to kingdom); plural taxa.
               </p>
               
               <p><i>TE</i>: Target enrichment.
               </p>
               
               <p><i>WGS</i>: Whole‐genome sequencing involves 
determining the complete DNA sequence of an organism's genome, also 
known as complete genome sequencing, full‐genome sequencing, entire 
genome sequencing.
               </p>
               
            </div>
         </div>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0002">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0002-title">2 MICROARRAYS</h2>
         
         <p>As shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this is a fluorescence‐based method that can be used to detect 
individuals or a community of taxa or genes. This method enables highly 
parallel monitoring by miniaturizing traditional tube‐ or plate‐based 
assays. Microscopic spots of oligonucleotide probes are attached to a 
solid surface called a microarray “chip” (Sauer et&nbsp;al., <span><a href="#mec14478-bib-0151" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>; Schena et&nbsp;al., <span><a href="#mec14478-bib-0152" class="bibLink tab-link" data-tab="pane-pcw-references">1998</a></span>).
 A fluorescently labelled sample containing sequences complementary to 
the probes is added to the chip. When hybridization occurs, fluorescence
 is measured. The use of microarrays in biodiversity research 
transformed the field by allowing pan‐community assays to occur in a 
single reaction without the complication of preparing gels for 
visualization (Gardner, Jaing, McLoughlin, &amp; Slezak, <span><a href="#mec14478-bib-0061" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Metfies &amp; Medlin, <span><a href="#mec14478-bib-0115" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>; Schena et&nbsp;al., <span><a href="#mec14478-bib-0152" class="bibLink tab-link" data-tab="pane-pcw-references">1998</a></span>).
 Microarrays also enable samples to be evaluated with replication, an 
important consideration in microbial ecology (DeSantis et&nbsp;al., <span><a href="#mec14478-bib-0039" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>).
 DNA microarrays can be used to answer questions about the presence and 
relative abundance of a known set of taxa using rDNA sequences or genes 
from mixed‐community samples. For example, various chips (e.g., 
Virochip, GreeneChipPm, PhyloChip, Lawrence Livermore Microbial 
Detection Array) have been designed to obtain a snapshot of viral and 
microbial (bacteria, protozoa, fungi) diversity without the cost 
associated with the older generation of HTS platforms and without the 
time‐consuming bioinformatics needed to process raw sequence data (Chou 
et&nbsp;al., <span><a href="#mec14478-bib-0028" class="bibLink tab-link" data-tab="pane-pcw-references">2006</a></span>; DeSantis et&nbsp;al., <span><a href="#mec14478-bib-0040" class="bibLink tab-link" data-tab="pane-pcw-references">2006</a></span>, <span><a href="#mec14478-bib-0039" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>; Gardner et&nbsp;al., <span><a href="#mec14478-bib-0061" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Palacios et&nbsp;al., <span><a href="#mec14478-bib-0127" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>). Another way to assess biodiversity is to view it through a phylogenetic lens, which can be performed with <span class="smallCaps">phylochip</span> results using the <span class="smallCaps">fast unifrac</span> program for phylogeny‐based large‐scale community analyses (Hamady, Lozupone, &amp; Knight, <span><a href="#mec14478-bib-0075" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>).
 To detect environmental processes, the GeoChip has been designed to 
detect the genes involved in nutrient cycling, metal reduction, 
resistance and degradation (He et&nbsp;al., <span><a href="#mec14478-bib-0078" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>).
 Custom oligonucleotides can also be designed and spotted on arrays. In 
an environmental monitoring example, a custom‐designed microarray was 
developed to identify genes potentially involved in environmental stress
 responses in a widely cultivated marine clam (Milan et&nbsp;al., <span><a href="#mec14478-bib-0118" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 This method was shown to be reproducible and allowed investigators to 
identify a range of genes potentially involved in environmental stress 
responses. Microarrays are also useful for detecting short degraded 
sequences such as those extracted from eDNA and have even been 
successfully applied to the analysis of highly degraded ancient DNAs 
(Devault et&nbsp;al., <span><a href="#mec14478-bib-0041" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>).
         </p>
         
         <p>The most attractive feature of DNA microarrays for 
biodiversity genomics, the ability to compile pan‐community profiles, 
has been surpassed by other DNA‐based HTS methods described below. A 
microarray assay may cost hundreds of dollars per sample depending on 
the chip and provider. For comparison, the cost of an Illumina MiSeq run
 can be divided by 96 or more samples depending on the desired 
sequencing coverage bringing the cost per sample down to $100 or less if
 many samples are run together, notwithstanding the time and cost 
associated with HTS bioinformatics. The resolution of a microarray assay
 depends on the specificity of the oligonucleotides on the chip and may 
be as specific as the species level using “detection” probes or more 
general using “discovery” probes that only bind to highly conserved 
regions (Gardner et&nbsp;al., <span><a href="#mec14478-bib-0061" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>). Unfortunately, microarrays are not suitable for detecting novel taxa or genes on their own (DeSantis et&nbsp;al., <span><a href="#mec14478-bib-0039" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>). This is a major limitation as the extent of environmental biodiversity has yet to be fully described (Hawksworth, <span><a href="#mec14478-bib-0077" class="bibLink tab-link" data-tab="pane-pcw-references">1991</a></span>; Torsvik &amp; Ovreas, <span><a href="#mec14478-bib-0179" class="bibLink tab-link" data-tab="pane-pcw-references">2002</a></span>).
 Another limitation of this method is that only hybridization patterns 
are recorded not the DNA sequences, which could be otherwise used for 
sequence‐based inference methods for biodiversity analysis or the 
development of new probes. As microarrays are normally designed for a 
single use, the number of chips that are purchased as well as access to 
specialized equipment to analyse the chips may limit the number of 
samples and replicates that can be processed. Overall, the scalability 
of this method is poor compared with methods below that can be run in a 
multiwell format for parallel batch sample processing. As shown in 
Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 we think microarrays will eventually be phased out in favour of DNA 
sequence‐based methods (below) as bioinformatics tools become easier to 
use.
         </p>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0003">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0003-title">3 QUANTITATIVE POLYMERASE CHAIN REACTION (QPCR)</h2>
         
         <p>As shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this is a fluorescence‐based method that can be used to detect 
individual species or genes in biodiversity studies. This is a PCR‐based
 method similar to standard endpoint PCR except that light is emitted 
and measured during every cycle as new DNA is synthesized (Arya 
et&nbsp;al., <span><a href="#mec14478-bib-0006" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>).
 Also called real‐time PCR, this method can be used in a quantitative 
manner with reference to a standard curve or a semi‐quantitative manner 
among samples. This method can also be used with reverse‐transcription 
qPCR to measure gene expression levels. Quantitative PCR to detect and 
quantify taxa and functional genes such as those involved in nutrient 
cycling or biodegradation from environmental samples is attractive for 
monitoring applications as well as more general biodiversity analysis 
(Smith &amp; Osborn, <span><a href="#mec14478-bib-0160" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>).
 This method is appealing for biodiversity studies because of the 
enhanced sensitivity of qPCR, compared with standard PCR, making this 
method more suitable for detecting rare species of interest for 
biosecurity (e.g., alien invasive species) or conservation efforts 
(e.g., endangered species) (Wilcox et&nbsp;al., <span><a href="#mec14478-bib-0194" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>).
 The genes in pathways activated upon exposure to toxins or involved 
directly in nutrient cycling are obvious targets for qPCR. In a 
biomonitoring application, reverse‐transcription qPCR was used to detect
 the expression of two sets of biomarker genes in response to heavy 
metal exposure in a marine mussel (Banni et&nbsp;al., <span><a href="#mec14478-bib-0010" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>).
 In an ecotoxicological study, qPCR was adapted to target very long 
amplicons to assess DNA damage in different parts of the genome (Meyer, <span><a href="#mec14478-bib-0116" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>).
 The premise behind this assay is that DNA damage caused by genotoxins 
may inhibit DNA polymerase progression along the template and results in
 reduced amplification. To address the question of how long DNA persists
 in the environment and the window of time captured in eDNA samples, a 
freshwater mesocosm experiment found that DNA became undetectable by 
qPCR 2&nbsp;weeks after removal of animals (Thomsen et&nbsp;al., <span><a href="#mec14478-bib-0178" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 This method is more sensitive and more expensive than standard PCR and 
ideally suited for tracking single or small suites of target taxa. 
Taxonomic resolution as well as prevalence of false‐positive and 
false‐negative results depend on the specificity of the primers designed
 for qPCR but can be used to detect species even in the presence of 
congeneric species (Wilcox et&nbsp;al., <span><a href="#mec14478-bib-0194" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>).
 However, caution must be exercised when closely related species might 
occur in the same sample at different concentrations as the abundant 
templates may be preferentially amplified (Wilcox et&nbsp;al., <span><a href="#mec14478-bib-0194" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>).
 An advantage of this method for large‐scale biodiversity studies is 
that the equipment needed to run qPCRs can accommodate 96‐well or 
384‐well plate formats and this allows for easy scalability to 
parallelize the analysis of many samples at once. As a 
fluorescence‐based detection method, however, it does not provide the 
sequence information that could be used for sequence‐based biodiversity 
analyses or for the development of new molecular probes and primers.
         </p>
         
         <p>As qPCR is a sensitive method, there are a number of 
considerations that need to be accounted for specifically when working 
with mixed‐community samples. First, the limits of detection would need 
to be determined for different types of environmental samples. In fact, 
any mixed‐template PCR‐based method is susceptible to similar challenges
 as different types of eDNA samples will contain a different background 
of DNAs from a community of organisms in addition to PCR inhibitors such
 as polysaccharides, humic acids, tannins and heavy metals (Braid, 
Daniels, &amp; Kitts, <span><a href="#mec14478-bib-0020" class="bibLink tab-link" data-tab="pane-pcw-references">2003</a></span>; Schrader, Schielke, Ellerbroek, &amp; Johne, <span><a href="#mec14478-bib-0154" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 Additionally, this method on its own is not suitable for discovering 
novel genes or species because the primers used for qPCR are 
specifically designed to target only known species or genes. The primary
 advantages of using qPCR are sensitivity, scalability, cost and speed 
for diagnostic screening of target taxa; however, issues caused by the 
complex eDNA background may be better circumvented using digital PCR 
(below). As noted in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 qPCR is likely to remain the method of choice wherever rapid monitoring
 of target taxa is needed, but this method could be supplanted by 
integrated microfluidic devices if they become a more cost‐effective 
option in future (see Section&nbsp;<a class="sectionLink scrollableLink" href="#mec14478-sec-0003">3</a>).
         </p>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0004">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0004-title">4 DIGITAL PCR</h2>
         
         <p>As shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this is another fluorescence‐based detection method that can be used to
 quantify individuals or be used as a target enrichment method prior to 
HTS. Digital PCR is an alternative to traditional qPCR where a sample is
 separated into thousands of parallel PCRs each with a single‐ or 
no‐template molecule (Vogelstein &amp; Kinzler, <span><a href="#mec14478-bib-0188" class="bibLink tab-link" data-tab="pane-pcw-references">1999</a></span>).
 This method can be used to determine starting copy number without the 
use of reference standards. Depending on the platform, reactions are 
carried out on a microfluidic device or in their own individual micelle 
droplets. Chamber digital PCR (cdPCR) is when a microfluidic device is 
used for digital PCR and can be used for real‐time DNA quantification. 
Digital droplet PCR (ddPCR) is when digital PCR is carried out in 
micelle droplets, and this can be used for endpoint or real‐time DNA 
quantification depending on the platform. In digital PCR, a single 
template is amplified on its own and can avoid problems from 
mixed‐template PCR such as the generation of chimeric sequences, 
primer‐template bias and template competition (Boers, Hays, &amp; 
Jansen, <span><a href="#mec14478-bib-0017" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>; Williams et&nbsp;al., <span><a href="#mec14478-bib-0195" class="bibLink tab-link" data-tab="pane-pcw-references">2006</a></span>). ddPCR has been used to estimate eDNA concentration, fish abundance and biomass (Doi et&nbsp;al., <span><a href="#mec14478-bib-0043" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>). ddPCR can also be used to target multiple markers in a single run for enrichment prior to HTS (see Section&nbsp;<a class="sectionLink scrollableLink" href="#mec14478-sec-0002">2</a>
 below). In a variation of the above techniques, microfluidic, multiplex
 digital PCR was used to co‐amplify 16S rDNA and a metabolic gene from 
single bacterial cells (Ottesen, Hong, Quake, &amp; Leadbetter, <span><a href="#mec14478-bib-0126" class="bibLink tab-link" data-tab="pane-pcw-references">2006</a></span>).
 In this example, termite gut endosymbionts previously known from a 
metabolic gene survey were linked with their 16S rDNA sequence for the 
first time. An advantage of this method in place of traditional qPCR is 
that the complexity of background DNAs is reduced to a single‐template 
strand per reaction. For example, in a study that directly compared 
ddPCR with qPCR, ddPCR was found to quantify eDNA, fish abundance and 
fish biomass more accurately than qPCR (Doi et&nbsp;al., <span><a href="#mec14478-bib-0043" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>).
 For laboratories that already have qPCR protocols, conditions would 
need to be re‐optimized for digital PCR. The sensitive nature of the 
method makes the problem of contaminants in laboratory products more 
pernicious, highlighting the importance of running negative controls 
(Salter et&nbsp;al., <span><a href="#mec14478-bib-0150" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>). As shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this method is similar to traditional qPCR but is more sensitive and 
ideal for mixed templates derived from environmental samples. Due to the
 requirement for specialized equipment, this method may not be readily 
adopted by laboratories that already have qPCR equipment and protocols.
         </p>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0005">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0005-title">5 DNA METABARCODING</h2>
         
         <p>The marker gene DNA sequencing technique used for the 
original prokaryote 16S ribosomal gene phylogenies and community surveys
 were quickly adapted to other markers to target fungi and then 
eukaryotes where the approach was rebranded as DNA metabarcoding with 
the defining goal of “species identification” from bulk environmental 
samples (Bik et&nbsp;al., <span><a href="#mec14478-bib-0016" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; O'Brien et&nbsp;al., <span><a href="#mec14478-bib-0124" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>; Taberlet, Coissac, Pompanon, Brochmann, &amp; Willerslev, <span><a href="#mec14478-bib-0169" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; Torsvik &amp; Ovreas, <span><a href="#mec14478-bib-0179" class="bibLink tab-link" data-tab="pane-pcw-references">2002</a></span>).
 DNA metabarcoding is rooted in efforts associated with DNA barcoding, 
where a standard species‐specific marker gene such as mitochondrial 
cytochrome c oxidase 1 (CO1) is used for identifying single specimens of
 animals (Hebert, Cywinska, Ball, &amp; deWaard, <span><a href="#mec14478-bib-0079" class="bibLink tab-link" data-tab="pane-pcw-references">2003</a></span>).
 DNA metabarcoding involves PCR‐coupled HTS of one or more DNA barcode 
markers (or other biodiversity markers) directly from mixed‐community 
samples without the need to isolate individuals. The term “DNA 
metabarcoding” (Taberlet, Coissac, Pompanon, et&nbsp;al., <span><a href="#mec14478-bib-0169" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; Yu et&nbsp;al., <span><a href="#mec14478-bib-0200" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>) has also been referred to as “DNA metagenetics” (Creer et&nbsp;al., <span><a href="#mec14478-bib-0033" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>), “environmental barcoding” (Hajibabaei et&nbsp;al., <span><a href="#mec14478-bib-0073" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>), “DNA metasystematics” (Hajibabaei, <span><a href="#mec14478-bib-0072" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>), metagenomic amplicon sequencing or simply “marker gene surveys” (Bik et&nbsp;al., <span><a href="#mec14478-bib-0016" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 Essentially, these methods transformed the fields of microbial 
molecular ecology, biodiversity and biomonitoring by allowing whole 
communities of organisms to be targeted, at the same time, without the 
need to isolate individuals. Unlike the fluorescence or PCR‐based 
methods discussed in the sections above, the DNA sequences produced by 
DNA metabarcoding provided greater resolution to distinguish among taxa 
and sparked discussions concerning the significance of the “rare 
biosphere” (Huse, Welch, Morrison, &amp; Sogin, <span><a href="#mec14478-bib-0084" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Reeder &amp; Knight, <span><a href="#mec14478-bib-0145" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>; Sogin et&nbsp;al., <span><a href="#mec14478-bib-0161" class="bibLink tab-link" data-tab="pane-pcw-references">2006</a></span>).
 DNA metabarcodes are amenable to phylogenetic analysis and introduced a
 new way to analyse biodiversity using a phylogenetic diversity method 
that could be scaled up to keep pace with the newest HTS methods (Faith,
 Lozupone, Nipperess, &amp; Knight, <span><a href="#mec14478-bib-0058" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>; Hamady et&nbsp;al., <span><a href="#mec14478-bib-0075" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>).
 Given the widespread application of DNA metabarcoding, we provide more 
details on key aspects of this approach as it is commonly used in 
biodiversity and biomonitoring studies.
         </p>
         
         <div class="article-section__sub-content" id="mec14478-sec-0006">
            
            <h3 class="article-section__sub-title section2">5.1 Mixed‐template PCR</h3>
            
            <p>Target enrichment/amplification from mixed communities 
using PCR has been referred to as mixed‐template or multitemplate PCR 
(Kalle, Kubista, &amp; Rensing, <span><a href="#mec14478-bib-0090" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>).
 PCR‐coupled DNA metabarcoding is sensitive to the initial 
mixed‐template PCR, including PCR cocktail composition, primers and 
cycling conditions. PCR bias caused by differential binding of PCR 
primers to template eDNA, the generation of artefacts (heteroduplexes, 
chimeric sequences, PCR duplicates), has been discussed at length in the
 literature (Bik et&nbsp;al., <span><a href="#mec14478-bib-0016" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; Shokralla, Spall, Gibson, &amp; Hajibabaei, <span><a href="#mec14478-bib-0159" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; Tedersoo et&nbsp;al., <span><a href="#mec14478-bib-0172" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>).
 Mixed‐template PCR optimization often involves steps such as reducing 
the number of PCR cycles and increasing extension time (Gohl 
et&nbsp;al., <span><a href="#mec14478-bib-0068" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>; Haas et&nbsp;al., <span><a href="#mec14478-bib-0071" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>; Ishii &amp; Fukui, <span><a href="#mec14478-bib-0086" class="bibLink tab-link" data-tab="pane-pcw-references">2001</a></span>; Kurata et&nbsp;al., <span><a href="#mec14478-bib-0094" class="bibLink tab-link" data-tab="pane-pcw-references">2004</a></span>; O'Donnell, Kelly, Lowell, &amp; Port, <span><a href="#mec14478-bib-0125" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>; Suzuki &amp; Giovannoni, <span><a href="#mec14478-bib-0166" class="bibLink tab-link" data-tab="pane-pcw-references">1996</a></span>; Wang &amp; Wang, <span><a href="#mec14478-bib-0190" class="bibLink tab-link" data-tab="pane-pcw-references">1997</a></span>).
 The resulting amplicon sequences can be analysed as is or identified by
 comparison with a reference sequence database. PCR is not the only 
method that can be used for target enrichment prior to DNA 
metabarcoding, however, and these are described below, see Section&nbsp;<a class="sectionLink scrollableLink" href="#mec14478-sec-0002">2</a>.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="mec14478-sec-0007">
            
            <h3 class="article-section__sub-title section2">5.2 Marker selection</h3>
            
            <p>Research communities focusing on a variety of taxonomic groups have identified their own signature DNA regions (Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0002">2</a>) suitable for high‐throughput taxonomic identification using a variety of methods (Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0003">3</a>).
 The metabarcoding approach gained much momentum after the advancement 
of HTS technologies, and researchers focused on differing taxonomic 
groups have established their own markers and standard methods 
(Hajibabaei, <span><a href="#mec14478-bib-0072" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 Selection and standardization of marker genes used for DNA 
metabarcoding remain an active area of research as it can significantly 
influence results obtained in metabarcoding studies (Deagle, Jarman, 
Coissac, Pompanon, &amp; Taberlet, <span><a href="#mec14478-bib-0036" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>; Fahner, Shokralla, Baird, &amp; Hajibabaei, <span><a href="#mec14478-bib-0055" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>; Porter, Shokralla, Baird, Golding, &amp; Hajibabaei, <span><a href="#mec14478-bib-0139" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>).
 Marker choice is known to affect the subset of the community resolved 
due to differing levels of variability in sequences of different 
lengths, across taxonomic groups and primer bias (Bellemain et&nbsp;al.,
 <span><a href="#mec14478-bib-0012" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Claesson et&nbsp;al., <span><a href="#mec14478-bib-0029" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Hollingsworth, Graham, &amp; Little, <span><a href="#mec14478-bib-0081" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>; Porter &amp; Hajibabaei, <span><a href="#mec14478-bib-0138" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>).
            </p>
            
            <div class="article-table-content" id="mec14478-tbl-0002">
               <header class="article-table-caption"><span class="table-caption__label">Table 2.
            </span>A list of the commonly used markers for DNA 
metabarcoding, databases, and tools for various taxonomic groups. This 
is not an exhaustive list, for generic tools we focus on those that seem
 to be most popular or are best suited for high‐throughput preprocessing
 of amplicon reads
               </header>
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell right-bordered-cell left-aligned">Taxa</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Marker</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Reference databases</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Software tools</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Prokaryotes</td>
                           
                           <td class="left-aligned">16S rDNA</td>
                           
                           <td class="left-aligned">
                              <p>GreenGenes (DeSantis et&nbsp;al., <span><a href="#mec14478-bib-0040" class="bibLink tab-link" data-tab="pane-pcw-references">2006</a></span>)</p>
                              <p>Ribosomal Database Project (RDP) (Cole et&nbsp;al., <span><a href="#mec14478-bib-0031" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>)</p>
                              <p><span class="smallCaps">silva</span> (Pruesse et&nbsp;al., <span><a href="#mec14478-bib-0141" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>)</p>
                              
                           </td>
                           
                           <td class="left-aligned">
                              <p><span class="smallCaps">picrust</span> (Langille et&nbsp;al., <span><a href="#mec14478-bib-0097" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>)</p>
                              <p>RDP classifier (Wang, Garrity, Tiedje, &amp; Cole, <span><a href="#mec14478-bib-0189" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>)</p>
                              
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Fungi</td>
                           
                           <td class="left-aligned">ITS rDNA</td>
                           
                           <td class="left-aligned">
                              <p><span class="smallCaps">its2</span> dbase (Ankenbrand, Keller, Wolf, Schultz, &amp; Förster, <span><a href="#mec14478-bib-0005" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>)</p>
                              <p><span class="smallCaps">unite</span> (Abarenkov et&nbsp;al., <span><a href="#mec14478-bib-0001" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>)</p>
                              
                           </td>
                           
                           <td class="left-aligned">
                              <p><span class="smallCaps">emerencia</span> (Ryberg, Kristiansson, Sjökvist, &amp; Nilsson, <span><a href="#mec14478-bib-0149" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>)</p>
                              <p>ITSx (Bengtsson‐Palme et&nbsp;al., <span><a href="#mec14478-bib-0013" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>)</p>
                              <p>RDP classifier</p>
                              
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Animals</td>
                           
                           <td class="left-aligned">CO1 mtDNA</td>
                           
                           <td class="left-aligned">
                              <p><span class="smallCaps">bold</span> (Ratnasingham &amp; Hebert, <span><a href="#mec14478-bib-0144" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>)</p>
                              <p>CO1 Arthropod classifier (Porter &amp; Hajibabaei, <span><a href="#mec14478-bib-0138" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>)</p>
                              <p>CO1 Insect classifier (Porter et&nbsp;al., <span><a href="#mec14478-bib-0134" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>)</p>
                              
                           </td>
                           
                           <td class="left-aligned">
                              <p><span class="smallCaps">bold</span> (Ratnasingham &amp; Hebert, <span><a href="#mec14478-bib-0144" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>)</p>
                              <p>RDP classifier</p>
                              
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Plants</td>
                           
                           <td class="left-aligned">rbcL + matK</td>
                           
                           <td class="left-aligned">See <i>Generic databases</i> below
                           </td>
                           
                           <td class="left-aligned">See <i>Generic pipelines</i> below
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Other Eukaryotes</td>
                           
                           <td class="left-aligned">18S rDNA</td>
                           
                           <td class="left-aligned">
                              <p>GreenGenes</p>
                              <p>SILVA</p>
                              
                           </td>
                           
                           <td class="left-aligned">See <i>Generic pipelines</i> below
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Any</td>
                           
                           <td class="left-aligned">Any</td>
                           
                           <td class="left-aligned"><i>Generic databases</i>: International Nucleotide Sequence Database (INSD) Collaboration<a class="noteLink scrollableLink" title="Link to note" id="mec14478-note-0002_120-controller" href="#mec14478-note-0002_120" aria-controls="mec14478-note-0002_120" aria-haspopup="false" aria-expanded="true" aria-label="Link to Note">a</a><div class="footNotePopup" id="mec14478-note-0002_120" aria-labelledby="mec14478-note-0002_120-controller"><button class="footNoteClose"><i class="icon-tools_close" aria-hidden="true"></i></button><span><span><sup><i>a</i></sup> </span>
                                    The INSD is an international 
initiative between the National Centre for Biotechnology Information 
(NCBI), the DNA Data Bank of Japan (DDBJ) and the European Nucleotide 
Archive (ENA).
                                    <br></span><span class="chevronDown"></span></div>
                           </td>
                           
                           <td class="left-aligned">
                              <p><i>Generic pipelines</i>:</p>
                              <p><span class="smallCaps">dada2</span> (Callahan et&nbsp;al., <span><a href="#mec14478-bib-0025" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>)</p>
                              <p>Galaxy (Goecks, Nekrutenko, &amp; Taylor, <span><a href="#mec14478-bib-0067" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>)</p>
                              <p><span class="smallCaps">mothur</span> (Schloss et&nbsp;al., <span><a href="#mec14478-bib-0153" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>)</p>
                              <p><span class="smallCaps">qiime</span> (Caporaso et&nbsp;al., <span><a href="#mec14478-bib-0026" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>)</p>
                              <p>RDP pipeline (Cole et&nbsp;al., <span><a href="#mec14478-bib-0031" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>)</p>
                              <p><span class="smallCaps">usearch</span> package (Edgar, <span><a href="#mec14478-bib-0047" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>, <span><a href="#mec14478-bib-0048" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>)</p>
                              <p><span class="smallCaps">vsearch</span> (Rognes, Flouri, Nichols, Quince, &amp; Mahé, <span><a href="#mec14478-bib-0147" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>)</p>
                              
                           </td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               
               <div class="article-section__table-footnotes">
                  <ul>
                     <li id="mec14478-note-0002"><span><sup><i>a</i></sup> </span>
                        The INSD is an international initiative between 
the National Centre for Biotechnology Information (NCBI), the DNA Data 
Bank of Japan (DDBJ) and the European Nucleotide Archive (ENA).
                        
                     </li>
                  </ul>
               </div>
               <div class="article-section__table-source"></div>
            </div>
            
            <div class="article-table-content" id="mec14478-tbl-0003">
               <header class="article-table-caption"><span class="table-caption__label">Table 3.
            </span>Commonly used methods for taxonomic assignment of 
signature DNA sequences from DNA metabarcoding studies. In this table, 
we have specifically omitted species delineation methods that should not
 be conflated with taxonomic assignment methods. Additionally, some of 
these methods were originally developed for the taxonomic assignment of 
metagenomic reads but can be applied to amplicon sequences
               </header>
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell right-bordered-cell left-aligned">Taxonomic assignment method</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Description</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Programs</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Similarity‐based</td>
                           
                           <td class="left-aligned">Includes methods 
that use a score calculated from pairwise sequence alignments or a 
comparison between a sequence and a profile hidden Markov model (HMM) 
(generated from a multiple sequence alignment)</td>
                           
                           <td class="left-aligned">
                              <p><span class="smallCaps">blast</span> (Altschul, Gish, Miller, Myers, &amp; Lipman, <span><a href="#mec14478-bib-0003" class="bibLink tab-link" data-tab="pane-pcw-references">1990</a></span>)</p>
                              <p><span class="smallCaps">bold</span> identification engine (Ratnasingham &amp; Hebert, <span><a href="#mec14478-bib-0144" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>)</p>
                              <p><span class="smallCaps">metaphyler</span> (Liu et&nbsp;al., <span><a href="#mec14478-bib-0100" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>)</p>
                              <p><span class="smallCaps">mg‐rast</span> (Meyer et&nbsp;al., <span><a href="#mec14478-bib-0117" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>)</p>
                              
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Phylogeny‐based</td>
                           
                           <td class="left-aligned">Based on the concept
 of orthology and phylogenetic theory, this method requires a multiple 
sequence alignment and a model of sequence evolution. The results are a 
phylogenetic hypothesis of evolutionary relatedness and a measure of 
statistical support for branch points</td>
                           
                           <td class="left-aligned">
                              <p><span class="smallCaps">nj k2p</span> analysis (Hebert et&nbsp;al., <span><a href="#mec14478-bib-0079" class="bibLink tab-link" data-tab="pane-pcw-references">2003</a></span>)</p>
                              <p><span class="smallCaps">sap</span> (Munch, Boomsma, Huelsenbeck, Willerslev, &amp; Nielsen, <span><a href="#mec14478-bib-0120" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>; Munch, Boomsma, Willerslev, &amp; Nielsen, <span><a href="#mec14478-bib-0120" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>)</p>
                              <p><span class="smallCaps">pplacer</span> (Matsen, Kodner, &amp; Armbrust, <span><a href="#mec14478-bib-0110" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>)</p>
                              <p><span class="smallCaps">epa</span> (Berger, Krompass, &amp; Stamatakis, <span><a href="#mec14478-bib-0014" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>)</p>
                              
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Composition‐based</td>
                           
                           <td class="left-aligned">Query and reference 
sequences are broken down into libraries of shorter words of size “k.” 
Taxonomic assignment is based on k‐mer frequencies in the query and 
reference library sequences</td>
                           
                           <td class="left-aligned">
                              <p>KNN Classify.seqs method (Schloss et al., 2009) <span class="smallCaps">qiime otu</span>‐picking uses <span class="smallCaps">uclust</span> by default (Caporaso et&nbsp;al., <span><a href="#mec14478-bib-0026" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>)</p>
                              <p><span class="smallCaps">rdp</span> Classifier (Wang et&nbsp;al., <span><a href="#mec14478-bib-0189" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>)</p>
                              <p><span class="smallCaps">uclust</span> (Edgar, <span><a href="#mec14478-bib-0046" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>)</p>
                              
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Hybrid methods</td>
                           
                           <td class="left-aligned">Combines different approaches from above, sometimes with other methods where indicated, to make taxonomic assignments</td>
                           
                           <td class="left-aligned">
                              <p><span class="smallCaps">fuzzyid</span>2 (Shi et&nbsp;al., <span><a href="#mec14478-bib-0156" class="bibLink tab-link" data-tab="pane-pcw-references">in press</a></span>)</p>
                              <p><span class="smallCaps">megan</span> (Huson, Mitra, Ruscheweyh, Weber, &amp; Schuster, <span><a href="#mec14478-bib-0085" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>)</p>
                              <p>PhymmBL (Brady &amp; Salzberg, <span><a href="#mec14478-bib-0019" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>)</p>
                              
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Bayesian tree‐less methods</td>
                           
                           <td class="left-aligned">Based on the 
coalescent theory of speciation, uses Bayesian tree‐less methods that 
use either the coalescent or a proxy for the coalescent, the number of 
segregating sites</td>
                           
                           <td class="left-aligned">
                              <p>Coalescent Assigner (Abdo &amp; Golding, <span><a href="#mec14478-bib-0002" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>)</p>
                              <p>Segregating Sites Assigner (Lou &amp; Golding, <span><a href="#mec14478-bib-0104" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>).</p>
                              
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="right-bordered-cell left-aligned">Machine learning</td>
                           
                           <td class="left-aligned">Uses theories from machine learning such as support vector machines and artificial neural networks</td>
                           
                           <td class="left-aligned">
                              <p><span class="smallCaps">bpsi</span> (Zhang, Sikes, Muster, &amp; Li, <span><a href="#mec14478-bib-0201" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>)</p>
                              <p><span class="smallCaps">svm</span> classifier (Seo, <span><a href="#mec14478-bib-0155" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>)</p>
                              
                           </td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               <div class="article-section__table-source"></div>
            </div>
            
         </div>
         
         <div class="article-section__sub-content" id="mec14478-sec-0008">
            
            <h3 class="article-section__sub-title section2">5.3 Metabarcoding versus traditional biomonitoring</h3>
            
            <p>In a study that directly compared traditional 
morphology‐based and metabarcoding methods for surveying 
macroinvertebrates from river benthos, all species that comprised 
greater than 1% of the individuals in the sample mixture were detected 
(Hajibabaei et&nbsp;al., <span><a href="#mec14478-bib-0073" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 In fact, the metabarcoding approach has already become a key tool in 
some large‐scale biomonitoring programmes looking to incorporate 
DNA‐based methods into their existing regional or national programmes 
(Baird &amp; Hajibabaei, <span><a href="#mec14478-bib-0009" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; Gilbert, Jansson, &amp; Knight, <span><a href="#mec14478-bib-0065" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>; GRDI‐EcoBiomics, <span><a href="#mec14478-bib-0069" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>).
 A key question for biodiversity analyses is how metabarcoding compares 
with traditional methods for community profiling. Despite differences in
 the exact taxa recovered using traditional methods and DNA 
metabarcoding (Hajibabaei et&nbsp;al., <span><a href="#mec14478-bib-0073" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>),
 recent studies have found that metabarcoding of insects, birds, diatoms
 and zooplankton tends to recover more taxa than traditional methods, 
provide a finer level of resolution and can similarly be used as a 
DNA‐based biological indicator (Ji et&nbsp;al., <span><a href="#mec14478-bib-0088" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>; Pawlowski, Esling, Lejzerowicz, Cedhagen, &amp; Wilding, <span><a href="#mec14478-bib-0131" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>; Sweeney et&nbsp;al., <span><a href="#mec14478-bib-0167" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>; Yang et&nbsp;al., <span><a href="#mec14478-bib-0198" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>). Results from studies focusing on plants and animals have been reviewed in Deiner et&nbsp;al. (<span><a href="#mec14478-bib-0037" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>)
 and also found that DNA metabarcoding provided either complimentary or 
increased richness compared with traditional methods. Although many 
studies have successfully used metabarcoding to find differences in 
sites that are known to be distinct, a recent analysis used 
metabarcoding to assess similar sites subject to natural variation and 
low‐intensity management (Emilson et&nbsp;al., <span><a href="#mec14478-bib-0054" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>).
 This study showed that freshwater invertebrate biodiversity obtained 
from metabarcoding the CO1 BR5 region was positively correlated with 
stream condition gradients (Emilson et&nbsp;al., <span><a href="#mec14478-bib-0054" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>).
 Additionally, DNA metabarcoding compares favourably to traditional 
methods for zooplankton and diatom community profiling by alleviating 
the taxonomic impediment—the bottleneck implicated in traditional 
morphology‐based specimen identification (Ji et&nbsp;al., <span><a href="#mec14478-bib-0088" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>; Yang et&nbsp;al., <span><a href="#mec14478-bib-0198" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>; Zimmermann, Glöckner, Jahn, Enke, &amp; Gemeinholzer, <span><a href="#mec14478-bib-0202" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>).
 Although fieldwork is still a costly and time‐consuming endeavour, once
 samples are subsampled into tubes or plates, this method becomes highly
 scalable and amenable to parallelization and automation (Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec14478-fea-0002">2</a>).

            </p>
            
            <div class="feature" id="mec14478-fea-0002">
               <h3><span class="feature-label">Box 2. </span>Scaling up
               </h3>
               <div class="article-section__content Box " id="mec14478-sec-0030">
                  
                  <p>DNA metabarcoding is a highly scalable method that 
can be used to increase both the breadth and depth of sampling in 
biodiversity and biomonitoring studies as shown in 
Box&nbsp;Figure&nbsp;1. The barcoding marker is enriched from eDNA 
samples in the metabarcoding step. Using kits for substrate‐specific DNA
 extraction can increase efficiency and reproducibility, as well as 
reduce laboratory‐to‐laboratory variability across large projects. Using
 automation, liquid‐handling robots, can reduce pipetting errors. 
Working with samples in multi‐well plates can improve throughput by 
conducting many reactions in parallel. When samples are prepared for 
high‐throughput sequencing (HTS), sequencing breadth and depth can be 
adjusted by varying the number of pooled (multiplexed) samples loaded on
 an Illumina MiSeq lane. For instance, samples collected from different 
field sites or multiple markers from the same sample can be tagged, 
pooled, and run together. To adjust the depth of sequencing, the total 
number of samples run in a lane can be varied up or down. The large 
amount of data generated by HTS often necessitates analysis via 
high‐performance computing (HPC). Cloud computing, in particular, can be
 used to store and analyze data in an environment with more computing 
resources than typically available to most individual researchers. 
Bioinformatic pipelines can be run on batches of samples across multiple
 runs in parallel to reduce computational time. As data analysis methods
 develop, data flows can be developed, documented, and versioned to 
ensure analysis reproducibility (Falster, FitzJohn, Pennell, &amp; 
Cornwell, 2017).</p>
                  
                  <section class="article-section__inline-figure">
                     <figure class="figure" id="mec14478-fig-0002"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/cbddea1d-d521-43f7-be24-b1aafff2f3b9/mec14478-fig-0002-m.jpg">
                           <picture>
                              <source srcset="https://wol-prod-cdn.literatumonline.com/cms/attachment/cbddea1d-d521-43f7-be24-b1aafff2f3b9/mec14478-fig-0002-m.jpg" media="(min-width: 1650px)"><img class="figure__image" src="mec14478-fig-0002-m.png" data-lg-src="https://wol-prod-cdn.literatumonline.com/cms/attachment/cbddea1d-d521-43f7-be24-b1aafff2f3b9/mec14478-fig-0002-m.jpg" alt="image" title="image"></picture></a><figcaption class="figure__caption">
                           <div class="figure__caption__header"><strong class="figure-title">Figure 2</strong> 
                              <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=mec14478-fig-0002&amp;doi=10.1111%2Fmec.14478" class="ppt-figure-link">PowerPoint</a></div>
                           </div>
                           <div>DNA metabarcoding is a scalable DNA‐based biodiversity analysis method. At each step, methods can be easily
scaled upwards to accommodate more samples if needed. HPC, high‐performance computing; HTS, high‐throughput sequencing</div>
                        </figcaption>
                     </figure>
                  </section>
                  
               </div>
            </div>
            
         </div>
         
         <div class="article-section__sub-content" id="mec14478-sec-0009">
            
            <h3 class="article-section__sub-title section2">5.4 Quantitative or not?</h3>
            
            <p>Because PCR is commonly used for amplification of target 
genes in DNA metabarcoding, read abundance does not necessarily reflect 
organismal abundance, biomass, or activity (Amend, Seifert, &amp; Bruns,
 <span><a href="#mec14478-bib-0004" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Elbrecht &amp; Leese, <span><a href="#mec14478-bib-0052" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>; Klappenbach, Saxman, Cole, &amp; Schidt, <span><a href="#mec14478-bib-0091" class="bibLink tab-link" data-tab="pane-pcw-references">2001</a></span>; Polz &amp; Cavanaugh, <span><a href="#mec14478-bib-0133" class="bibLink tab-link" data-tab="pane-pcw-references">1998</a></span>; Tedersoo et&nbsp;al., <span><a href="#mec14478-bib-0174" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Větrovský &amp; Baldrian, <span><a href="#mec14478-bib-0185" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>).
 There have been reports, however, that read abundance does scale with 
estimated biomass for fungi and zooplankton under certain conditions 
(Amend et&nbsp;al., <span><a href="#mec14478-bib-0004" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Yang et&nbsp;al., <span><a href="#mec14478-bib-0198" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>).
 Generally speaking, because of issues with primer bias and 
mixed‐template PCR bias, natural variation in copy number, as well as 
variation in biomass and density among organisms, it has been suggested 
that a conservative approach is to treat DNA metabarcoding data as 
presence–absence data only (Elbrecht &amp; Leese, <span><a href="#mec14478-bib-0052" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>; Hajibabaei, Spall, Shokralla, &amp; van Konynenburg, <span><a href="#mec14478-bib-0074" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; Hajibabaei et&nbsp;al., <span><a href="#mec14478-bib-0073" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 In a study of marine benthic fauna, ecological indices using abundance 
or presence‐only data performed similarly (Ranasinghe, Stein, Miller, 
&amp; Weisberg, <span><a href="#mec14478-bib-0143" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="mec14478-sec-0010">
            
            <h3 class="article-section__sub-title section2">5.5 Bioinformatics</h3>
            
            <p>With advances from new HTS platforms and a growing need 
for more efficient data analysis and interpretation, the bioinformatics 
considerations are varied and constantly evolving (Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec14478-fea-0003">3</a>).
 Bioinformatic challenges related to plant and animal metabarcoding are 
different than those faced by DNA barcoding methodologies (Coissac, 
Riaz, &amp; Puillandre, <span><a href="#mec14478-bib-0030" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 The taxonomic resolution gained by this method depends on the 
resolution of the signature DNA sequence targeted (e.g., species level 
for DNA barcodes) and availability of sequences in reference databases. 
There are a variety of platforms available for biodiversity analysis, 
some particularly well‐suited for beginners in the field because they 
can provide smooth wrappers around commonly used command‐line tools as 
well as well‐documented usage examples in online forums (Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0004">4</a>).
            </p>
            
            
            <div class="feature" id="mec14478-fea-0003">
               <h3><span class="feature-label">Box 3. </span>Bioinformatics considerations for using DNA metabarcoding for biodiversity analysis
               </h3>
               <div class="article-section__content Box " id="mec14478-sec-0031">
                  
                  <p>Two key resources for making DNA metabarcoding a 
suitable tool for biodiversity analysis are as follows: (i) 
comprehensive metadata and (ii) comprehensive reference databases.</p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0032">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0032-title"> METADATA</h2>
                  
                  <p>The deposition of rich metadata ensures the 
long‐term utility of published amplicon sequences and encourages 
comparative studies (Bik et&nbsp;al., <span><a href="#mec14478-bib-0016" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 As a result, sequence annotation standards can facilitate the 
comparison of sequence data across studies (Yilmaz et&nbsp;al., <span><a href="#mec14478-bib-0199" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
                  </p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0033">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0033-title"> REFERENCE DATABASES</h2>
                  
                  <p>High quality reference databases for taxonomic 
assignment are essential for successful DNA metabarcoding. There are 
several issues that affect current reference database quality:</p>
                  
                  <p>
                     
                     </p><ul class="plain-list">
                        
                        <li><i>Insufficiently identified records</i>. 
This is a problem that affects all metabarcode markers and public 
records can have differing levels of taxonomic annotation from strain or
 BIN or species to a variety of more inclusive taxonomic ranks. It has 
been shown for fungal ITS sequences that with the increasing use of 
high‐throughput DNA metabarcoding from eDNA, the number of unnamed 
anonymous DNA sequences has been accumulating in GenBank and vastly 
exceeds the number of known taxonomically identified reference sequences
 (Hibbett et&nbsp;al., 2011; Nilsson, Kristiansson, Ryberg, &amp; 
Larsson, <span><a href="#mec14478-bib-0006" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>).
                        </li>
                        
                        
                        <li><i>Incorrectly annotated records</i>. This 
is an issue that affects all metabarcode markers. Unexpected annotation 
errors can arise due to contamination or misidentification by 
collectors. For fungal ITS sequences, the reliability of taxonomic 
records in public databases have been questioned and there has even been
 a call for third‐party annotation of GenBank records (Bidartondo, <span><a href="#mec14478-bib-0001" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Nilsson et&nbsp;al., <span><a href="#mec14478-bib-0007" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>).
                        </li>
                        
                        
                        <li><i>Biased record collection</i>. For any 
metabarcode marker, there can be variability in taxon representation 
across different geographic locations, habitats, and taxonomic groups. 
For instance, CO1 sequences in the BOLD database are dominated by 
Diptera (Insecta) and Lepidoptera (Insecta) sequences from Canada 
(Porter et&nbsp;al., 2014). This is particularly problematic for 
metabarcoding studies in areas where endemic diversity is expected to be
 high, such as in the tropics or for microbes—particularly in forest 
soils (Basset et&nbsp;al., 2012; Hawksworth, 1991; Tedersoo et&nbsp;al.,
 2014).
                        </li>
                        
                        
                        <li><i>Incomplete databases</i>. It is known 
that sequence‐based identification of samples can be hindered by 
incomplete reference databases (Porter &amp; Golding, 2011, 2012; Porter
 et&nbsp;al., 2014; Sundquist et&nbsp;al., 2007; Taberlet, Coissac, 
Pompanon, et&nbsp;al., 2012). Most studies assume that their taxa will 
be identified through metabarcoding, however, the extent of database 
incompleteness is not known for every group. For example, despite 
Insecta being one of the largest groups of CO1 sequences in GenBank, it 
has been estimated that only about 12% of extant insect genera are 
represented by a CO1 sequence (Porter et&nbsp;al., 2014).
                        </li>
                        
                     </ul>
                     
                  <p></p>
                  
                  <p>The significance of these issues lies in the level 
of undetected false‐positive taxonomic assignments in metabarcoding 
studies. Type II error, incorrectly rejecting a true null hypothesis, is
 also referred to as a false‐positive assignment and has been discussed 
in the literature (Virgilio, Backeljau, Nevado, &amp; De Meyer, 2010; 
Porter et&nbsp;al., 2014). See <i>Taxonomic assignment</i> below.
                  </p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0034">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0034-title"> SEQUENCE READ ERRORS</h2>
                  
                  <p>PCR and sequencing errors can produce artefactual 
sequences. The removal of these sequences is essential to avoid 
inflating richness counts and including sequence artefacts in community 
analyses (Huse et&nbsp;al., 2010; Reeder &amp; Knight, 2009). There are 
several methods for handling sequence errors in metabarcoding studies, 
especially those that use PCR for enrichment.</p>
                  
                  <p>The use of paired‐end sequencing, especially for 
short amplicons, can provide sequence overlap between the forward and 
reverse reads and provide redundancy in the regions where sequence error
 rates start to increase (i.e., at the ends of each read). Chimeric 
sequences can be generated during PCR. A chimera removal step is 
included in many of the widely used bioinformatic pipelines 
(Table&nbsp;4). Clustering reads into OTUs can absorb highly similar 
reads with sequence errors. Clustering reads into OTUs defined by some 
distance threshold (or sequence similarity cutoff) is often the default 
approach in most metabarcode bioinformatic pipelines (Table&nbsp;4). 
Additionally, a generic method to remove sequence artefacts is to simply
 remove low frequency OTUs such as singletons and doubletons (Huse 
et&nbsp;al., 2010; Tedersoo et&nbsp;al., 2010). A step including the 
removal of low frequency OTUs are integrated into many of the most 
widely used bioinformatic pipelines (Table&nbsp;4).</p>
                  
                  <p>The term denoising was first introduced as a 
sequencing platform‐specific method to remove sequences with distinctive
 errors produced during the sequencing step (Quince, Lanzen, Davenport, 
&amp; Turnbaugh, 2011). Since then, it has become a more general term to
 describe the process of removing reads with predicted errors from any 
source. The denoising method can be as simple as&nbsp;removing low 
frequency OTUs. For example, ‘rare’ OTUs such as singletons and 
doubletons or OTUs containing a particularly low</p>
                  
                  <p>frequency of total reads. This step is implemented 
in most bioinformatic pipelines (Table&nbsp;4). The USEARCH UNOISE 
algorithm attempts to predict and remove low read frequency amplicons as
 containing possible sequence errors if there is a high similarity high 
frequency amplicon present (Edgar, 2016). Part of the denoising process 
can also include the removal of contaminant sequences. The USEARCH 
unoise algorithm will remove PhiX contaminant sequences automatically, 
however, the removal human, host, or common lab contaminants requires 
extra steps outside of this pipeline (Edgar, 2016). Finally, the removal
 of nonspecific amplification products (e.g., nuclear pseudogenes of 
mitochondrial genes; NUMTs) can be challenging to implement in 
high‐throughput pipelines, but for protein coding genes, such as CO1, 
these can include removing reads that contain frameshifts or indels that
 disrupt the open reading frame (Song, Buhay, Whiting, &amp; Crandall, 
2008).</p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0035">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0035-title"> DETERMINING THE BASIC UNIT FOR DNA‐BASED BIODIVERSITY ANALYSES</h2>
                  
                  <p>There are three main types of DNA‐based indicators 
generated by DNA metabarcoding: (i) taxa (at one or several taxonomic 
ranks), (ii) operational taxonomic units (OTUs), or (iii) exact sequence
 variants (ESVs). For simple biological inventories, taxonomy‐based 
lists are probably the most relevant output (see Table&nbsp;3). For a 
simple calculation of richness, one could count the number of unique 
taxa, ESVs, or OTUs. Moving from taxon lists to community analyses, 
however, involves the creation of data matrices. These matrices may be 
based simply on taxonomy (at any rank or variable ranks), or they can be
 based on OTUs or ESVs. OTUs (or ESVs) in turn can be global OTUs 
(clustered from reads from all samples at once) or OTUs created for a 
single sample at a time. The advantage of using global OTUs is that 
these are directly comparable across all samples. The advantage of 
OTU‐based analyses is that all the data can be analysed together whether
 or not they can be taxonomically assigned with confidence. The 
advantage of taxonomy‐based analyses is that known taxonomy can be used 
to help narrow down a complex data set into groups of indicator taxa 
that are known be ecologically relevant such as members of the insect 
orders Ephemeroptera, Plecoptera and Trichoptera (EPT) that have been 
shown to be sensitive to water pollution as they require clean water 
with a high level of dissolved oxygen or members of the Chironomidae 
that can be an indicator of poor water quality as they are rapid 
colonizers and have been previously shown to be tolerant of high water 
pollution (Buss, Baptista, Silveira, Nessimian, &amp; Dorvillé, 2002; 
Emilson et&nbsp;al., 2017).</p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0036">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0036-title"> CHOOSING A SEQUENCE CLUSTERING METHOD</h2>
                  
                  <p>Metabarcode clustering can be performed using a 
variety of methods: phylogenetic clustering (Box&nbsp;4); 
single‐linkage, average‐linkage, or furthest‐linkage clustering, or 
other hybrid methods. Reads can be clustered against a reference in 
open‐ or closed‐reference methods. Closed‐reference clustering is when 
reads are clustered against a reference database directly. 
Open‐reference clustering is much like closed‐reference clustering 
except that reads that do not cluster with the references are then 
clustered de novo into their own new clusters using a distance threshold
 as a cut‐off. For example, the Galaxy, MOTHUR, QIIME, and USEARCH 
pipelines all provide methods to perform reference‐based clustering 
using the 16S rDNA Green Genes or SILVA databases, or the ITS reference 
databases. Reference‐based clustering is popular with markers where 
extensive reference databases exist, but a recent study has shown that 
OTU richness and beta‐diversity are often greatly exaggerated with these
 methods (Edgar, 2017). De novo clustering, on the other hand, may 
outperform closed‐ and open‐reference clustering with 16S sequences by 
better representing actual distances between sequences (Westcott &amp; 
Schloss, 2015). De novo clustering is when all reads are clustered among
 themselves using any of a variety of algorithms, then subsequently 
taxonomically assigned (Table&nbsp;3). A consideration with this 
approach is the cut‐off that is used to delineate the OTUs. A range of 
1–3% sequence dissimilarity is popular in the literature and was once 
used to approximate species units for measuring or estimating diversity,
 but it is now recognized to be somewhat arbitrary as sequence variation
 within and among species varies across taxa. To avoid the ‘lumping’ of 
similar reads from different species into a single OTU, the use of exact
 sequence variants (ESVs) has been proposed (Callahan, McMurdie, &amp; 
Holmes, 2017; Edgar, 2017b). The DADA2 and USEARCH unoise method both 
specifically produce ESVs as output (Callahan et&nbsp;al., 2016; Edgar, 
2016). Another hybrid clustering method is SWARM and it can be run on 
its own or as a part of the QIIME pipeline (Mahé, Rognes, Quince, de 
Vargas, &amp; Dunthorn, <span><a href="#mec14478-bib-0004" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>).
 A comparison of OTU clustering methods with 16S rDNA data showed that 
the most stable and accurate OTUs could be produced using different 
algorithms and may vary according to data set (Westcott &amp; Schloss, 
2015).
                  </p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0037">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0037-title"> TAXONOMIC ASSIGNMENT</h2>
                  
                  <p>One of the simplest taxonomic assignment methods is
 incorporated into the reference‐based clustering approaches described 
above. Alternatively, there are a variety of methods for taxonomic 
assignment that use different parameters for optimization such as 
similarity, phylogeny, or composition among others (Table&nbsp;3). The 
most popular method for metabarcode taxonomic assignment is the 
similarity‐based top BLAST hit approach. Though the method is relatively
 easy to implement and run in parallel, studies that have shown that 
type II error (false‐positive rates) are high with this method and that 
the top hit is not always the closest phylogenetic neighbour (Koski 
&amp; Golding, 2001; Virgilio et&nbsp;al. 2010). One way to reduce false
 positive rates is to use a method that produces a measure of 
statistical confidence to filter out uncertain taxonomic assignments 
(Munch, Boomsma, Willerslev, &amp; Nielsen, <span><a href="#mec14478-bib-0005" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>;
 Munch, Boomsma, Willerslev, et&nbsp;al., 2008; Porter &amp; Hajibabaei,
 2017; Wang et&nbsp;al., 2007). Additionally, the top BLAST hit method 
is slow compared to other widely used methods. In large‐scale 
biodiversity and biomonitoring studies, taxonomic assignment methods 
need to be fast and provide a statistical measure of confidence for 
taxonomic assignments at all ranks. A popular method is the Ribosomal 
Database Project naïve Bayesian classifier which is implemented in most 
popular bioinformatics pipelines (Table&nbsp;4) or can be run on it's 
own. This method can be trained for any metabarcode marker and reference
 sets already exist for the prokaryote 16S, fungal ITS and LSU rDNA 
regions, as well as for the CO1 animal barcode marker (Cole et&nbsp;al.,
 <span><a href="#mec14478-bib-0002" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>;
 Liu, Porras‐Alfaro, Kuske, Eichorst, &amp; Xie, 2012; Porter 
et&nbsp;al., 2014; Porter &amp; Hajibabaei, 2017; Wang et&nbsp;al., 
2007).
                  </p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0038">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0038-title"> DATA NORMALIZATION</h2>
                  
                  <p>It has been shown that uneven sequencing effort can
 skew community comparisons because false negatives (undersampling) can 
emphasize differences between communities (Gihring, Green, &amp; Schadt,
 <span><a href="#mec14478-bib-0064" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 There are methods to normalize or accommodate uneven sequence numbers 
among samples to avoid library size bias. Most recently, it has been 
pointed out that metabarcoding data are inherently compositional. As 
such, the most appropriate normalization technique would be a log ratio 
transformation (Gloor, Macklaim, Pawlowsky‐Glahn, &amp; Egozcue, <span><a href="#mec14478-bib-0066" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>).
 Traditionally, however, randomly subsampling all libraries down to the 
smallest library size prior to OTU creation has been a simple solution 
to avoid diversity estimator sample size bias; however, this involves 
throwing away sequence information. Alternatively, rarefaction estimates
 of samples can be compared at some common level of sequencing depth. 
Both these methods also work when comparing obviously different sample 
types using presence–absence data. For finding species showing different
 abundances among samples, however, it has also been shown that 
rarefaction, as well as the common approach where sequence numbers are 
converted to simple proportions for each library, results in a high rate
 of false positives (McMurdie &amp; Holmes, <span><a href="#mec14478-bib-0111" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>). Instead, the use of alternative methods such as <span class="smallCaps">ancom</span> (a compositional approach) or <span class="smallCaps">deseq2</span> may be able to more accurately identify differential taxon abundances among samples (Mandal et&nbsp;al., <span><a href="#mec14478-bib-0107" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>; McMurdie &amp; Holmes, <span><a href="#mec14478-bib-0111" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>).
 A comparison of these methods using 16S rDNA has shown that rarefaction
 is an effective method for normalizing data and that for comparing 
abundances the best method to use may depend on data characteristics 
such as the total number of samples and how uneven the library sizes 
are, as well as composition of microbial communities among samples 
(Weiss et&nbsp;al., <span><a href="#mec14478-bib-0191" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>).
                  </p>
                  
                  <div class="article-table-content" id="mec14478-tbl-0004">
                     <header class="article-table-caption"><span class="table-caption__label">Table 4.
            </span>Commonly used generic bioinformatics pipelines and packages to analyse signature DNA regions from metabarcoding studies
                     </header>
                     
                     
                     
                     <div class="article-table-content-wrapper">
                        <table class="table article-section__table">
                           
                           
                           
                           
                           <thead>
                              
                              <tr>
                                 
                                 <th class="bottom-bordered-cell right-bordered-cell left-aligned">Pipeline</th>
                                 
                                 <th class="bottom-bordered-cell left-aligned">Description</th>
                                 
                                 <th class="bottom-bordered-cell left-aligned">Features</th>
                                 
                              </tr>
                              
                           </thead>
                           
                           <tbody>
                              
                              <tr>
                                 
                                 <td class="right-bordered-cell left-aligned">DADA2</td>
                                 
                                 <td class="left-aligned">Runs in the R environment. Processes data from <span class="smallCaps">fastq</span> files, removes errors and chimeras, and produces sample abundances and taxonomic assignments
                                 </td>
                                 
                                 <td class="left-aligned">Produces exact sequence variants (ESVs) instead of OTUs for greater resolution than OTU‐based methods</td>
                                 
                              </tr>
                              
                              <tr>
                                 
                                 <td class="right-bordered-cell left-aligned">Galaxy</td>
                                 
                                 <td class="left-aligned">Provides an environment where scripts can be assembled into pipelines to assist with raw data processing</td>
                                 
                                 <td class="left-aligned">Graphical user interface</td>
                                 
                              </tr>
                              
                              <tr>
                                 
                                 <td class="right-bordered-cell left-aligned">MOTHUR</td>
                                 
                                 <td class="left-aligned">Command‐line 
driven. Semi‐automated pipeline allows raw sequence data to be processed
 through to community analysis using OTU‐ or taxonomy‐based methods</td>
                                 
                                 <td class="left-aligned">Initially 
offered a way to create OTUs, remove putatively chimeric sequences using
 a variety of methods, calculate ecological indices and create Venn 
diagrams. Now also offers a variety of pipelines to process raw reads 
and make taxonomic assignments</td>
                                 
                              </tr>
                              
                              <tr>
                                 
                                 <td class="right-bordered-cell left-aligned">QIIME</td>
                                 
                                 <td class="left-aligned">Command‐line driven. Semi‐automated pipeline allows raw sequence data to be processed through to community analysis</td>
                                 
                                 <td class="left-aligned">Wrapper for 
many commonly used programs for analysing DNA metabarcoding reads, 
particularly 16S sequences. Pipeline automatically formats the input and
 output files to work with a variety of programs to allow easy 
comparison of results using the most popular methods. QIIME2 also comes 
with an easy to use graphical user interface and an application 
programmer interface for data scientists</td>
                                 
                              </tr>
                              
                              <tr>
                                 
                                 <td class="right-bordered-cell left-aligned">RDP pipeline</td>
                                 
                                 <td class="left-aligned">Provides a 
graphical user interface to process amplicon sequences from raw reads, 
performs 16S and 28S rDNA taxonomic assignments, as well as provides 16S
 and 28S secondary structure, and diversity analysis tools</td>
                                 
                                 <td class="left-aligned">Provides access to the RDP classifier for classifying SSU rDNA for bacteria and archaea, as well as ITS and LSU rDNA for fungi</td>
                                 
                              </tr>
                              
                              <tr>
                                 
                                 <td class="right-bordered-cell left-aligned">USEARCH</td>
                                 
                                 <td class="left-aligned">Command‐line 
driven, normally used for sequence clustering into operational taxonomic
 units, but can also be used for sequence similarity searches, denoising
 and newer versions can handle raw sequence data</td>
                                 
                                 <td class="left-aligned">Initially 
offered a way to cluster reads into operational taxonomic units (OTUs), a
 method to search for similar sequences and identify putatively chimeric
 sequences. This package now offers pipelines to process raw reads, 
denoise reads, and cluster reads while automatically removing chimeric 
sequences, sequence errors and PhiX reads. 32‐bit version is available 
for all users free of charge, but is limited to 4‐Gb memory at most. 
64‐bit version available and allows users to use all the memory 
available on a 64‐bit computer</td>
                                 
                              </tr>
                              
                              <tr>
                                 
                                 <td class="right-bordered-cell left-aligned">VSEARCH</td>
                                 
                                 <td class="left-aligned">Performs many of the functions available in USEARCH except denoising</td>
                                 
                                 <td class="left-aligned">Open‐source software available free of charge and allows users to use all the memory available on a 64‐bit computer</td>
                                 
                              </tr>
                              
                           </tbody>
                           
                        </table>
                     </div>
                     
                     
                     <div class="article-section__table-source"></div>
                  </div>
                  
               </div>
            </div>
            
            <p>Often, results from DNA metabarcoding are used to inform 
researchers as to which samples are worth a deeper look using 
metagenomic, metatranscriptomic or target‐enriched sequencing. The data 
generated using DNA metabarcoding are more tractable in amount and 
computational resources required for analysis than those generated from 
whole metagenome or metatranscriptome sequencing, and the most popular 
platform is currently the Illumina MiSeq (Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec14478-fea-0005">5</a>).
 For current biomonitoring applications, DNA metabarcoding leads the way
 over other methods of biodiversity analysis in terms of scalability, 
cost and tractability. As summarized in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this method is likely to remain the preferred choice for routine 
biodiversity and biomonitoring applications until the cost of sequencing
 whole genomes is decreased to the point that it can become routine so 
that genome databases become more representative of the biodiversity 
found in nature.
            </p>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0011">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0011-title">6 ORGANELLE SEQUENCING</h2>
         
         <p>As shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this is a sequencing‐based method suitable for characterizing 
individual organelle genomes. In this method, organelle DNA is either 
isolated from an individual or the organelle genome can be 
bioinformatically assembled from an individual's shotgun genomic 
sequences with or without prior enrichment (Cronn et&nbsp;al., <span><a href="#mec14478-bib-0034" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>; McPherson et&nbsp;al., <span><a href="#mec14478-bib-0112" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>; Nock et&nbsp;al., <span><a href="#mec14478-bib-0123" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>; Parks, Cronn, &amp; Liston, <span><a href="#mec14478-bib-0128" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>).
 Individual or combinations of organelle genes from mitochondria and 
plastids have a long history of use in biosystematics as a basis for 
classification (Hollingsworth et&nbsp;al., <span><a href="#mec14478-bib-0081" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 The term “super‐barcoding” has been used when whole‐organelle genomes 
are used specifically for taxonomic assignment (Li et&nbsp;al., <span><a href="#mec14478-bib-0099" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>).
 For plants in particular, this has been shown to circumvent the lack of
 variation seen in some groups when single barcode markers are used for 
taxonomic assignment (Li et&nbsp;al., <span><a href="#mec14478-bib-0099" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>).
 Recent advances in sequencing technologies now enable whole chloroplast
 or mitochondrial genomes to be sequenced and used for a wide range of 
population genetic to phylogenetic studies not only from individuals but
 also from eDNA samples. Specifically, researchers have proposed a 
mito‐metagenomics approach for eDNA (Tang et&nbsp;al., <span><a href="#mec14478-bib-0171" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>).
 This involves computationally assembling all mitochondrial genes 
sequenced from eDNA into whole mitochondria. With respect to the amount 
of DNA sequence data generated, organelle sequencing sits between DNA 
metabarcoding and whole‐genome sequencing. It provides a way to obtain 
more genetic information at a much lower cost compared with whole‐genome
 analysis. Unfortunately, even if a whole mitochondrial genome is 
sequenced and used for comparative analysis, it essentially behaves as a
 single marker because all the mitochondrial genes are linked. The 
utility of organellar genomes for addressing challenging biosystematics 
questions is limited because the bulk of genetic information comes from 
unlinked nuclear markers whose evolution can follow a different 
evolutionary trajectory (Hollingsworth, Li, van der Bank, &amp; Twyford,
 <span><a href="#mec14478-bib-0082" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>).
 Also, in an eDNA framework, mito‐metagenomics has not been applied to 
large sample sizes and varied sample types presumably because the 
sequencing depth needed for whole mitochondrial genome reconstruction is
 high and reference databases for mitochondrial genomes are very small 
compared with reference databases for the commonly used DNA 
metabarcoding markers (Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0002">2</a>). As summarized in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this method will likely be phased out in favour of whole‐genome 
sequencing as single‐molecule methods develop, accuracy improves and 
costs decrease.
         </p>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0012">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0012-title">7 GENOME SKIMMING</h2>
         
         <p>As shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this is a DNA sequencing‐based detection method applied to individual 
specimens. This method differs from organelle sequencing because there 
is no need for prior isolation or targeting of organelle DNA prior to 
sequencing individuals. Genome skimming involves shallow‐ or 
low‐coverage sequencing of an organism's genome (including organelle 
genomes) to obtain sequence data that can be used to address 
biodiversity‐related questions. Experimental work has shown that even 
shallow sequencing (e.g., 1–2&nbsp;Gb) can provide surprisingly deep 
coverage of high copy number organelle DNA (plastids, mitochondria) and 
other repetitive sequences such as the full ribosomal cistron (Straub 
et&nbsp;al., <span><a href="#mec14478-bib-0164" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>) often used in biosystematics (Hollingsworth et&nbsp;al., <span><a href="#mec14478-bib-0082" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>; Li et&nbsp;al., <span><a href="#mec14478-bib-0099" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>).
 This method has primarily been employed in plants due to the difficulty
 in obtaining species‐level resolution using DNA barcodes (Hollingsworth
 et&nbsp;al., <span><a href="#mec14478-bib-0082" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>).
 Genome skimming can provide sequence data suitable for biodiversity 
analysis without the need to pick and choose marker genes or optimizing 
PCR protocols for individual genes. With museum or eDNA samples with 
degraded DNAs, focusing on high copy regions by genome skimming may be 
more successful than targeting low copy regions of the genome 
(Dodsworth, <span><a href="#mec14478-bib-0042" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>).
 In plants where the variation in plastid DNA can be especially valuable
 for taxonomic assignment, the natural variation of cpDNA per cell in 
different life stages of a leaf can result in more or less coverage of 
cpDNA versus other repetitive regions such as rDNA (Dodsworth, <span><a href="#mec14478-bib-0042" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>).
 Additionally, the genome skimming approach may not be easily applicable
 to smaller organisms (e.g., small insects) and difficult to cultivate 
organisms where it could be difficult to extract enough genomic 
material. These issues, combined with the need to isolate individuals 
before sequencing, could make scalability a problem for large‐scale 
ecological investigations. As summarized in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 as single‐molecule sequencing methods advance further, accuracy 
improves and cost reduces, this method is likely to be phased out in 
favour of WGS.
         </p>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0013">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0013-title">8 WHOLE‐GENOME SEQUENCING</h2>
         
         <p>As shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this is another DNA sequence‐based method applied to individual 
specimens. This involves obtaining a tissue sample or pure culture of an
 individual organism, DNA extraction and sequencing of the entire 
nuclear genome as well as any mitochondrial and plastid genomes. 
Initially, genome sequencing was such an expensive and time‐consuming 
process (Lander et&nbsp;al., <span><a href="#mec14478-bib-0096" class="bibLink tab-link" data-tab="pane-pcw-references">2001</a></span>; Venter et&nbsp;al., <span><a href="#mec14478-bib-0183" class="bibLink tab-link" data-tab="pane-pcw-references">2001</a></span>)
 that the application of this method for biodiversity research was not 
yet feasible. With the continued development of HTS and now 
third‐generation nanopore single‐strand sequencing, the associated 
increase in sequence throughput and reduced cost per base pair, it is 
now possible to sequence whole nuclear genomes that can be used as a 
resource for further biodiversity and evolutionary analyses.
         </p>
         
         <div class="article-section__sub-content" id="mec14478-sec-0014">
            
            <h3 class="article-section__sub-title section2">8.1 Whole‐genome sequencing projects</h3>
            
            <p>Numerous projects are contributing to the population of 
whole‐genome sequences in databases such as the i5K initiative that aims
 to sequence 5000 arthropod genomes (i5K Consortium, <span><a href="#mec14478-bib-0089" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>),
 the 1000 fungal genome project (approximately 800 fungal genomes are 
currently available through the U.S. Department of Energy's Joint Genome
 Institute MycoCosm portal) (Grigoriev et&nbsp;al., <span><a href="#mec14478-bib-0070" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>),
 the GIGA project targeting 7000 noninsect and non‐nematode 
invertebrates (mostly marine taxa) for sequencing (GIGA Community of 
Scientists, <span><a href="#mec14478-bib-0063" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>), the Genome 10K project that aims to sequence one individual from every vertebrate genus (Koepfli, Paten, &amp; O'Brien, <span><a href="#mec14478-bib-0092" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>),
 the Genomic Encyclopedia of Bacteria and Archaea (GEBA) initiative that
 sequenced and released 1000 bacterial and archaeal genomes (Mukherjee 
et&nbsp;al., <span><a href="#mec14478-bib-0119" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>), as well as other projects targeting plant and crop genomes (Li, Wang, &amp; Zeigler, <span><a href="#mec14478-bib-0098" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>).
 All of these data are essential resources for the further development 
of molecular primers, probes and, in some cases, identification of eDNA 
sequences generated by other genomic methods discussed in this review.
            </p>
            
            <p>In a biodiversity or biomonitoring context, WGS data from
 single organisms are useful for both taxonomic or functional 
assignments as well as marker and primer development for qPCR, digital 
PCR or phylogenetics (Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec14478-fea-0004">4</a>). As summarized in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 as single‐molecule sequencing methods become more available and 
accurate, this method may become as routine as single gene sequencing is
 today in biodiversity studies and biomonitoring applications. At 
present, WGS of organisms from eDNA is only feasible for microbes with 
the smallest genomes and simplest organization (a single or few circular
 chromosomes) (see Sections <a class="sectionLink scrollableLink" href="#mec14478-sec-0010">5.5</a> and <a class="sectionLink scrollableLink" href="#mec14478-sec-0015">9</a>).
            </p>
            
            <div class="feature" id="mec14478-fea-0004">
               <h3><span class="feature-label">Box 4. </span>Phylogenetics in biodiversity analysis
               </h3>
               <div class="article-section__content Box " id="mec14478-sec-0025">
                  
                  <p>Phylogenetic methods can be used to analyse 
biodiversity data in several ways: (i) for OTU delimitation, (ii) for 
single‐marker taxonomic assignments, (iii) for phylogenomic taxonomic 
assignments and (iv) for comparing beta‐diversity across communities. 
These methods are included here because phylogenetics can complement the
 use of traditional biodiversity metrics by taking into consideration 
the evolutionary history of the sampled lineages across sites (Faith, <span><a href="#mec14478-bib-0056" class="bibLink tab-link" data-tab="pane-pcw-references">1996</a></span>, <span><a href="#mec14478-bib-0057" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>; Hamady et&nbsp;al., <span><a href="#mec14478-bib-0075" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Parks, Porter, et&nbsp;al., <span><a href="#mec14478-bib-0129" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>).
                  </p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0026">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0026-title"> OTU DELIMITATION</h2>
                  
                  <p>There are instances when defining an OTU based on 
sequence similarity alone can be problematic. For example, choosing a 
sequence similarity cut‐off to define an OTU is arbitrary, often based 
on community consensus, or chosen to approximate “species” even though 
it is known that Linnaean taxa do not necessary coincide with DNA‐based 
OTUs across lineages. It is possible instead to use a phylogeny‐based 
method to define OTUs, like a single gene application of the 
phylogenetic species concept where the members of a terminal clade may 
comprise one OTU or taxon, although this is usually a manual process, 
difficult to apply in large‐scale studies as this method can be 
computationally time consuming as the number of sequences analysed 
grows. This approach is best used to refine OTU membership for target 
taxa of interest as opposed to clustering whole communities of 
organisms. Once OTUs are delimited, they can be analysed as is or they 
can be taxonomically assigned (Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0003">3</a>).
                  </p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0027">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0027-title"> TAXONOMIC ASSIGNMENT</h2>
                  
                  <p>Phylogenetic methods can be used to make taxonomic 
assignments. For example, the sequence variability in the fungal barcode
 marker, the ITS rDNA region, is an advantage for making fine‐level 
taxonomic assignments but is problematic for phylogeny‐based taxonomic 
assignments across diverse groups of taxa. It is common that only 
congeneric taxa can be analysed in the same multiple sequence alignment 
and thus to analyse a community of diverse taxa would require many 
independent alignments and phylogenetic trees to identify their closest 
relatives for taxonomic assignment (Porter &amp; Golding, <span><a href="#mec14478-bib-0135" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 To overcome this difficulty, phylogeny‐based programs are available 
that can automate this process for any marker (Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0003">3</a>).
 These methods can help investigators apply consistent cut‐offs based on
 statistical support values to identify taxonomic assignments that they 
can be confident in. A major drawback of these methods is that they can 
be relatively slow and computationally intensive compared with 
alternative methods.
                  </p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0028">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0028-title"> PHYLOGENOMICS</h2>
                  
                  <p>When markers are sampled by genome mining from 
whole‐genome sequences, genome skimming or transcriptome sequencing, 
combined to make phylogenies, this is referred to as phylogenomics 
(Eisen, <span><a href="#mec14478-bib-0051" class="bibLink tab-link" data-tab="pane-pcw-references">1998</a></span>).
 With respect to biodiversity studies, phylogenomics may be most useful 
for refining species identifications of target taxa as well as to 
understand their evolutionary histories at multiple taxonomic levels 
(Steele &amp; Pires, <span><a href="#mec14478-bib-0163" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 For example, highly resolved phylogenies of yeast species have been 
produced by concatenating up to 106 genes (Rokas, Williams, King, &amp; 
Carroll, <span><a href="#mec14478-bib-0148" class="bibLink tab-link" data-tab="pane-pcw-references">2003</a></span>).
 Although most studies aim to identify taxa using single signature DNA 
markers, some groups can be problematic and may require multiple markers
 for correct species assignments. For example, to correctly identify 
plant taxa, multiple regions can be used (Straub et&nbsp;al., <span><a href="#mec14478-bib-0164" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>). Comparative analyses can identify key genes and pathways that can be used as markers in future work (Riley et&nbsp;al., <span><a href="#mec14478-bib-0146" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>).
 Comparisons among taxa in this way can help predict clades representing
 feature diversity worthy of special consideration for conservation 
efforts and can be used to guide topological restrictions on trees based
 on less data (see <i>Phylogenetic diversity</i> below).
                  </p>
                  
               </div>
               <div class="article-section__content Box " id="mec14478-sec-0029">
                  
                  <h2 class="article-section__title section__title section3" id="mec14478-sec-0029-title"> PHYLOGENETIC DIVERSITY</h2>
                  
                  <p>Perhaps the most practical use for phylogenetic 
methods in biodiversity analysis is to use this as a way to compare 
phylogenetic diversity across samples providing another window on 
beta‐diversity. The concept of phylogenetic diversity is based on the 
assumption that branch lengths among taxa from different samples 
represent the underlying feature diversity of these taxa (Faith, <span><a href="#mec14478-bib-0057" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>).
 Unlike the traditional richness measure where each species is given an 
equal weight of 1, using a phylogenetic diversity metric, communities 
can be compared according to the amount of unique branch lengths they 
represent (Tucker &amp; Cadotte, <span><a href="#mec14478-bib-0181" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>).
 In such a scenario, it is possible that a site with high richness may 
reflect low phylogenetic diversity if the species are closely related. 
This may have an impact in conservation studies where decisions on which
 sites to protect are driven by how diversity is measured. To facilitate
 the processing of large data sets, software tools need to be chosen 
carefully. For example, heuristics exist to construct very large trees 
such as with FastTree (Hamady et&nbsp;al., <span><a href="#mec14478-bib-0075" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Price, Dehal, &amp; Arkin, <span><a href="#mec14478-bib-0140" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>).
 Branch lengths can then be used to calculate phylogenetic 
beta‐diversity in large data sets using Fast UniFrac (Faith et&nbsp;al.,
 <span><a href="#mec14478-bib-0058" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>).
 A drawback of this method is that is may not be possible to produce a 
good phylogeny based on a single marker. Topological restrictions based 
on previous multi‐marker work, however, can be used to structure 
single‐gene trees. Fortunately, the calculation of phylogenetic 
diversity using the UniFrac method has been shown to be robust to 
sequencing effort and phylogenetic method (Lozupone, Hamady, Kelley, 
&amp; Knight, <span><a href="#mec14478-bib-0105" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>).
                  </p>
                  
               </div>
            </div>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0015">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0015-title">9 METAGENOMICS</h2>
         
         <p>As shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this is a DNA sequencing‐based method that can used to profile mixed 
communities. Metagenomics involves sequencing all the genomic material 
from many different taxa whose bulk DNA was extracted directly from 
environmental samples such as soil, biofilms, water, sediments, benthos 
and air (Venter et&nbsp;al., <span><a href="#mec14478-bib-0184" class="bibLink tab-link" data-tab="pane-pcw-references">2004</a></span>).
 This approach is also known as “shotgun sequencing,” “environmental 
genomics,” “ecogenomics” or “community genomics.” Although this approach
 was first introduced based on Sanger sequencing technology, the 
advances of HTS approaches have generated much momentum in metagenomics 
research and applications. Typically, this technique samples genes from 
across the genome, not just DNA barcode regions, so the functional 
potential of a sample can be explored. Comparative metagenomics can be 
used to compare the metabolic potential of organisms from different 
environmental samples and even determine taxonomic and functional 
profiles from the thousands of gene markers from communities of 
organisms (Tringe et&nbsp;al., <span><a href="#mec14478-bib-0180" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>).
 This method has been proposed as a way to detect uncultured organisms 
that are difficult to identify by traditional means (Handelsman, <span><a href="#mec14478-bib-0076" class="bibLink tab-link" data-tab="pane-pcw-references">2004</a></span>).
 Genes and gene families can be identified from metagenomic sequences. 
Identifying the taxa that these genes belong to, however, can be 
challenging. As signature DNA regions suitable for taxonomic analysis 
will also be present in the sample, these can be used to identify 
individual taxa (Liu, Gibbons, Ghodsi, &amp; Pop, <span><a href="#mec14478-bib-0100" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Manichanh et&nbsp;al., <span><a href="#mec14478-bib-0108" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>).
 Reconstruction of individual genomes is also possible depending on the 
sequencing depth, taxonomic complexity and size of organismal genomes in
 the sample. In a recent study, nearly 8,000 metagenome‐derived 
prokaryote genomes were assembled from 1,500 public metagenomes (Parks 
et&nbsp;al., <span><a href="#mec14478-bib-0130" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>).
 This type of achievement is not yet possible for eukaryotes due to the 
size and complexity of their genomes, but it may be in future as 
sequencing technologies and bioinformatics methods progress. 
Metagenomics has found application in ancient DNA studies looking at the
 evolution of antibiotic resistance, studies of the microbes involved in
 honey bee colony collapse disorder (Cox‐Foster et&nbsp;al., <span><a href="#mec14478-bib-0032" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>; D'Costa et&nbsp;al., <span><a href="#mec14478-bib-0035" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 Metagenomics is a widely used technique to explore microbiomes on a 
small scale and can be scaled upwards for broad‐scale ecological surveys
 (The Human Microbiome Project Consortium, <span><a href="#mec14478-bib-0176" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; Venter et&nbsp;al., <span><a href="#mec14478-bib-0184" class="bibLink tab-link" data-tab="pane-pcw-references">2004</a></span>).
 An advantage of this method is that amplification‐free metagenomic 
sample preparation avoids the PCR bias that other methods may otherwise 
be subject to. A challenge with this method is that with sequencing 
effort spread over all genomic regions, not just the signature DNA 
regions suitable for taxonomic assignment, there may be a reduced set of
 taxa that can be identified with confidence. Unfortunately, taxonomic 
assignment of nonsignature DNA regions may be biased towards organisms 
whose whole genomes are present in databases (false positives). The 
sequencing depth required to capture a community would be much higher 
than the sequencing depth required to saturate taxon sampling using DNA 
metabarcoding. As summarized in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>, as HTS and single‐molecule sequencing technologies advance, output grows, and costs decrease (Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec14478-fea-0005">5</a>),
 this method is likely to be even more widely used as amplification‐free
 methods are very appealing to many investigators hoping to circumvent 
the many known issues with mixed template PCR in PCR‐coupled DNA 
metabarcoding. As with many other methods, as the number of annotated 
genomes in public databases grows, the ability to annotate metagenomic 
samples should continue to improve.
         </p>
         
         <div class="feature" id="mec14478-fea-0005">
            <h3><span class="feature-label">Box 5. </span>HIGH‐THROUGHPUT SEQUENCING PLATFORMS
            </h3>
            <div class="article-section__content Box " id="mec14478-sec-0039">
               
               <p>First‐generation sequencing, also known as dideoxy 
sequencing or Sanger sequencing, generally produces read lengths of 
600–800&nbsp;bp in batches of 96 to 384 samples at a time and up to 16 
plates can be loaded in the queue on the most advanced instruments. The 
most popular platforms are the Applied Biosystems capillary sequencers. 
Second‐generation sequencing became popular in the mid‐2000s and was 
initially referred to as “next‐generation sequencing” in the literature,
 but with the development of third‐generation single‐molecule sequencing
 platforms, it is often now simply referred to as high‐throughput 
sequencing. The most commonly used method for DNA metabarcoding is 
currently the Illumina MiSeq platform a second‐generation method that 
uses clonal amplification on a plate followed by sequencing by synthesis
 (SBS) technology (BOX TABLE 1). Third‐generation sequencing, also known
 as single‐molecule sequencing, is not yet widespread but is 
characterized by not needing PCR before sequencing and producing a 
signal that is captured in real time (Liu, Li, et&nbsp;al., <span><a href="#mec14478-bib-0101" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
               </p>
               
            </div>
         </div>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0016">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0016-title">10 METATRANSCRIPTOMICS</h2>
         
         <p>As shown in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 this is a sequencing‐based detection tool suitable for identifying 
genes from individuals in a community. Whereas metagenomics can provide 
information on taxonomic composition and metabolic potential, 
metatranscriptomics can be used to provide a snapshot of the metabolic 
activity in a community. Metatranscriptomics involves HTS of 
reverse‐transcribed complementary DNA (cDNA) from messenger RNA (mRNA) 
isolated directly from environmental samples (Carvalhais, Dennis, Tyson,
 &amp; Schenk, <span><a href="#mec14478-bib-0027" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>; Mason et&nbsp;al., <span><a href="#mec14478-bib-0109" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>). This method has already been used to look at functional diversity of microbes and eukaryotes in soil (Bailly et&nbsp;al., <span><a href="#mec14478-bib-0008" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>; Urich et&nbsp;al., <span><a href="#mec14478-bib-0182" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>).
 It has been shown that while metagenomics can show metabolic potential 
(e.g., of deep‐sea microbial communities), the results from 
metatranscriptomics may yield very different insights as to which genes 
are actually being expressed (Mason et&nbsp;al., <span><a href="#mec14478-bib-0109" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 Whereas reverse transcriptase PCRs can only detect the expression of a 
single gene at a time, metatranscriptomics is a high‐throughput method 
that can survey thousands of genes at a time. Unfortunately, the 
proportion of an RNA extraction that contains mRNA is very low (2–3%) 
and may need to be amplified to obtain enough material for sequencing. 
If a PCR‐based amplification step is used, then the diversity in 
downstream steps may not reflect initial relative abundances. Obtaining 
high‐quality samples with intact mRNA may be challenging for many types 
of environmental samples. As the number of organisms with sequenced 
genomes increases, so too should the ability of investigators to 
annotate their metatranscriptomes. As this method provides a snapshot of
 the genes and pathways that are expressed in an environmental sample, 
this is a very attractive method of generating a very large set of 
functional gene information from across a community of organisms playing
 a variety of functional roles while using rather generic methods. When 
coupled with HTS, this method is highly scalable. As summarized in 
Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec14478-tbl-0001">1</a>,
 as WGS reference databases become more representative of environmental 
diversity, this method is likely to become a more reliable source of 
functional community profiling.
         </p>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0017">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0017-title">11 TARGET ENRICHMENT</h2>
         
         <p>The terms “target” or “targeted” enrichment refers to a 
general technique that can be used in combination with many of the 
above‐mentioned methods. Target enrichment resides between single gene 
metabarcoding and whole‐genome sequencing approaches because it allows a
 suite of markers to be targeted and enriched prior to sequencing. 
Commonly used enrichment methods include the following: (i) hybrid 
capture, (ii) selective circularization and (iii) PCR amplification. The
 use of target enrichment is to increase the efficiency of HTS for 
biodiversity analyses (Mamanova et&nbsp;al., <span><a href="#mec14478-bib-0106" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Mertes et&nbsp;al., <span><a href="#mec14478-bib-0114" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>).
 Generally, these methods are used to enrich for taxa/genes present at 
low abundance in a sample (e.g., parasites/pathogens), or to reduce the 
detection of taxa/genes present at high abundance in a sample (e.g., the
 host). Because this method relies on designing oligonucleotides to 
capture target sequences, this method may limit the detection of new 
taxa not currently represented in public databases.
         </p>
         
         <p><i>Hybrid capture</i> uses long oligonucleotides, either 
bound to a microarray or to beads in solution, to capture target 
sequences (Mamanova et&nbsp;al., <span><a href="#mec14478-bib-0106" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>).
 This is sometimes referred to as “PCR‐free” enrichment. For 
biodiversity analyses where the objective is to detect as many different
 taxa as possible, hybridization capture has been shown to recover a 
greater diversity of arthropod and insect orders compared with 
traditional morphological taxonomic assignment methods and PCR‐coupled 
metabarcoding (Shokralla et&nbsp;al., <span><a href="#mec14478-bib-0157" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>).
 Hybrid capture is a reproducible method, produces relatively uniform 
coverage of target sequences and has good capture rates (Tewhey 
et&nbsp;al., <span><a href="#mec14478-bib-0175" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>).
 This method may be able to generate sufficient template for library 
preparation that the initial mixed‐template PCR step in DNA 
metabarcoding can be avoided (Shokralla et&nbsp;al., <span><a href="#mec14478-bib-0157" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>).
 Additionally, hybrid capture tends to select for short fragments with 
higher specificity than longer fragments. This is because longer 
fragments will have a higher proportion of off‐target sequence compared 
with the probe and because of possible cross‐hybridization within longer
 fragments (Mamanova et&nbsp;al., <span><a href="#mec14478-bib-0106" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>).
 This bias towards short fragments makes hybrid capture suitable for 
processing potentially degraded samples from eDNA and producing the 
short sequence libraries typically prepared for Illumina MiSeq DNA 
metabarcoding (Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec14478-fea-0005">5</a>).
 Bead‐based hybridization is highly scalable across many samples and can
 be used to detect thousands of targets at a time. Baits targeting 
highly repetitive elements are known to work especially well compared 
with baits targeting low copy regions, but this may not be a problem for
 studies targeting rDNA or mtDNA across many taxa. This method is likely
 to see increased use in biomonitoring and ecological studies, 
particularly those studies targeting relatively low abundance taxa such 
as arthropods from soil samples where bacteria and fungi tend to 
dominate DNA metabarcoding libraries. Hybridization‐based capture is 
easy to implement and is a low‐cost approach to improve the efficiency 
of high‐throughput DNA metabarcoding. It has been suggested that the 
integration of hybridization enrichment in biodiversity analyses of 
signature DNA regions could mean a shift to a more meaningful 
interpretation of read numbers for CO1 metabarcoding studies, that is, 
reflecting biomass, but this needs further study as data on 
mitochondrial number variation and body size variation can be quite 
different even across a single taxonomic group such as the Insecta 
(Shokralla et&nbsp;al., <span><a href="#mec14478-bib-0157" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>).
 Bead‐based hybridization in solution can be conducted in 96‐well plates
 and is more scalable than on‐array enrichment, which also requires 
special equipment (Mamanova et&nbsp;al., <span><a href="#mec14478-bib-0106" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>).
         </p>
         
         <p><i>Selective circularization</i> using molecular inversion 
probes (MIPs) works much like hybridization capture except that a 
universal sequence is flanked by target‐specific sequences, such as 
restriction sites, and these constructs hybridize to sheared or digested
 DNA‐forming loops. Once the MIPs have hybridized to their targets, 
nucleotides are added to fill the gap and ligation closes the circles. 
This method is highly specific. It has been shown that a large portion 
of sequences, however, map to the universal sequence and target‐specific
 sequences (Tewhey et&nbsp;al., <span><a href="#mec14478-bib-0175" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>).
 This method has the potential to detect fewer taxa for biodiversity 
analyses compared with hybridization capture so is less likely to be 
adopted by the molecular ecology community for biodiversity studies.
         </p>
         
         <p><i>Target enrichment using PCR</i> is often the first step 
in PCR‐coupled DNA metabarcoding. Digital PCR (discussed above) can also
 be used for target enrichment prior to HTS (Tewhey et&nbsp;al., <span><a href="#mec14478-bib-0175" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>).
 The main advantage of using PCR is its low cost, ease of implementation
 and the production of large volumes of template for library prior to 
HTS. However, there are many issues regarding amplification bias and 
subsequent changes from the original template ratios in mixed‐template 
reactions and have already been discussed (<i>Metabarcoding</i>, above).
 Additionally, even digital PCR has its own biases and requires careful 
optimization. Because of the biases associated with mixed‐template PCR, 
any method that avoids this is an attractive option for investigators 
who want to see a less‐biased view of biodiversity in their samples.
         </p>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0018">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0018-title">12 FUTURE OUTLOOK AND CHALLENGES</h2>
         
         <p>The purpose of this review was to provide a guide to 
commonly used as well as newer and lesser‐known methods for genomics 
analysis of biodiversity data. Along with this, we also presented 
databases, tools and methods used with the widely popular and highly 
scalable DNA metabarcoding method for conducting biodiversity and 
biomonitoring studies. Despite widespread use of many of the techniques 
we review here, there remain challenges to DNA‐based biodiversity 
analyses that need to be addressed before the field can move from 
descriptive works to a form that can be used to inform policy and 
management decisions or be utilized in long‐term large‐scale studies: 
(i) continued development of highly scalable laboratory methods, (ii) 
improving bioinformatic algorithms and their accessibility through 
robust software tools, (iii) large‐scale integration of different data 
types and (iv) growth of reference databases.</p>
         
         <div class="article-section__sub-content" id="mec14478-sec-0019">
            
            <h3 class="article-section__sub-title section2">12.1 Scalable laboratory methods</h3>
            
            <p>The most popular data generation methods for 
high‐throughput biodiversity and biomonitoring studies are scalable; 
that is, they can accommodate increases in number of samples to be 
processed because they are amenable to automation and parallelization. 
Kits are currently available to process samples from DNA extraction 
through to sequencing in plate‐formatted batches. Microfluidics, 
however, can further miniaturize a reaction's footprint to microscopic 
lengths and to microlitre or picolitre volumes. Microfluidics, or 
lab‐on‐a‐chip solutions, could play a role in biodiversity studies by 
reducing sample sizes, decreasing reaction times, increasing automation 
and eventually reducing cost (Dutse &amp; Yusof, <span><a href="#mec14478-bib-0044" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>; Liu &amp; Zhu, <span><a href="#mec14478-bib-0103" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>; Wu, Kodzius, Cao, &amp; Wen, <span><a href="#mec14478-bib-0197" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>).
 An integrated microfluidic solution that allows for DNA extraction, PCR
 and DNA fragment size detection on a single chip already exists (Easley
 et&nbsp;al., <span><a href="#mec14478-bib-0045" class="bibLink tab-link" data-tab="pane-pcw-references">2006</a></span>).
 In the future, we could see how an integrated microfluidic solution 
that manages nucleic acid extraction through to sequencing could become 
the “sample‐in‐answer‐out” holy grail for truly high‐throughput 
biomonitoring that is rapid, reproducible, and eventually portable and 
easy to use by nonspecialists.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="mec14478-sec-0020">
            
            <h3 class="article-section__sub-title section2">12.2 Bioinformatics</h3>
            
            <p>We use the term bioinformatics to include not just raw 
sequence processing, but the implementation of algorithms for the 
analysis of large‐scale data sets. Current bioinformatic methods are a 
moving target, continually striving to keep up with the increasingly 
large data sets being generated by HTS platforms. Current challenges 
include improving the existing taxonomic and functional assignment 
tools, generally moving away from similarity‐ and phylogeny‐based 
assignments in large‐scale studies and moving towards composition‐based,
 machine learning, and other hybrid methods that are faster and produce 
meaningful confidence values for assignments. We predict the next 
generation of algorithms will not only classify sequences, but attempt 
to predict which ones represent new species (Lan, Wang, Cole, &amp; 
Rosen, <span><a href="#mec14478-bib-0095" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 The newest trends are random forest classifiers that can be used, for 
example, to predict sample origin based on community composition, that 
is, classification of whole communities as opposed to single taxa. 
Additionally, Bayesian classifiers can be used not only for taxonomic 
assignment but also for determining source/sink environmental 
interactions. For example, the Earth Microbiome Project analysed 2.2 
billion 16S rDNA sequence reads from more than 23,000 samples, and they 
used a portion of this extensive microbial catalog to train a random 
forest sample classifier to predict the origin of the remaining samples 
(Thompson et&nbsp;al., <span><a href="#mec14478-bib-0177" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>).
 They also used a leave‐one‐out cross‐validated model with all source 
environments to determine which other environments were most similar. 
Another bioinformatic bottleneck is the production of reports and 
visualizations in an intuitive manner without the need for extensive 
programming skills. For example, a drag‐and‐drop type platform that 
allows users to explore different data visualizations, such as from 
microbiome studies, is already being developed (Bik, <span><a href="#mec14478-bib-0015" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>).
 The ability to reduce large amounts of data into usable results, a 
process that can take just as long or even longer than the sampling 
process, will go a long way towards understanding complicated systems, 
and informing management decisions in a more timely manner.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="mec14478-sec-0021">
            
            <h3 class="article-section__sub-title section2">12.3 Integration of different data types</h3>
            
            <p>Biodiversity studies greatly benefit from databases 
containing DNA sequences (National Center for Biotechnology Information 
(NCBI), <span><a href="#mec14478-bib-0121" class="bibLink tab-link" data-tab="pane-pcw-references">1988</a></span>; Ratnasingham &amp; Hebert, <span><a href="#mec14478-bib-0144" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>; Cole et&nbsp;al., <span><a href="#mec14478-bib-0031" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>).
 Sequence data are not particularly meaningful on its own, however, 
without their metadata. A future challenge will involve strengthening 
linkages among the usual biodiversity metadata such as taxonomy, 
geographic information, local biotic and abiotic measurements, as well 
as incorporate earth observation data such as numerical weather data as 
well as photograph, radar and sonar imagery. For instance, addressing 
management impacts on a large scale to inform science‐based 
decision‐making will require marrying environmental data from Earth 
observation with biodiversity information for comprehensive modelling 
(Bush et&nbsp;al., <span><a href="#mec14478-bib-0022" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>).
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="mec14478-sec-0022">
            
            <h3 class="article-section__sub-title section2">12.4 Growth of reference databases</h3>
            
            <p>In the future, the ability to concurrently sample large 
numbers of unlinked markers from individuals as well as from eDNA 
samples in large‐scale biodiversity studies will likely come from 
PCR‐free techniques such as target enrichment, metatranscriptome, and 
metagenome sequencing (Hollingsworth et&nbsp;al., <span><a href="#mec14478-bib-0082" class="bibLink tab-link" data-tab="pane-pcw-references">2016</a></span>).
 Each of these methods allows multiple regions of the genome to be 
captured, increasing the DNA sequence information per taxon and 
increasing the chances of detecting the greatest number of taxa. This 
information can only be fully leveraged when comprehensive reference 
sequence databases are richly annotated as well as designed to allow for
 efficient data mining and report generation.
            </p>
            
            <p>Focusing on individual specimens and alpha taxonomy has 
been the tradition in biodiversity surveys of macroscopic organisms. 
Although specimens are necessary for assembling vouchers and reference 
sequence libraries, biomonitoring projects have gained momentum by 
including genomic analysis of environmental samples. It has already been
 shown that techniques such as DNA barcoding and metabarcoding can make 
significant contributions to biodiversity and biomonitoring studies 
(Janzen et&nbsp;al., 2005; Meier, Wong, Srivathsan, &amp; Foo, 2016; 
Shokralla, Hellberg, Handy, King, &amp; Hajibabaei, 2015). For better or
 worse, DNA‐based methods are supplementing and, in some cases, even 
supplanting individual specimen‐based collection for large‐scale 
biomonitoring (Baird &amp; Hajibabaei,<span><a href="#mec14478-bib-0009" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>).
 Although multi‐omics are often considered the future of community 
studies, in the microbial world, the thinking has come full circle. 
There has been a call for more work on isolating and cultivating 
specimens together with ecological observations to improve their 
understanding of microbial communities (Vilanova &amp; Porcar, 2016). To
 provide some perspective, we borrow the analogy used by E.O. Wilson 
(Wilson, <span><a href="#mec14478-bib-0196" class="bibLink tab-link" data-tab="pane-pcw-references">2017</a></span>),
 that DNA‐based biodiversity and biomonitoring studies are like an 
aerial‐survey; what we need are more “boots‐on‐the‐ground”. Ultimately, 
the continued growth of high‐quality reference sequences will only be 
possible in collaboration with taxonomists who have the expertise to 
find, collect, culture, and identify new specimens for DNA barcoding and
 WGS. If every “metabarcoder” reached out to include such experts in 
their projects, this could help to build a stronger foundation for the 
community as a whole. We hope this review provides some insight on how 
scalable DNA‐based methods are currently becoming the leading source for
 acquiring biodiversity information.
            </p>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="mec14478-sec-0023">
         
         <h2 class="article-section__title section__title section1" id="mec14478-sec-0023-title"> AUTHOR CONTRIBUTIONS</h2>
         
         <p>T.M.P. and M.H. wrote the manuscript.</p>
         
      </div>
      
      <section class="article-section article-section__references" id="references-section">
         <div class="accordion article-accordion"><span href="#" class="accordion__control" aria-controls="idmec14478-bibl-0001" aria-expanded="false" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fmec.14478&amp;pbContext=%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2018.27.issue-2%3Bjournal%3Ajournal%3A1365294x%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fmec.14478%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;onlyLog=true">
               <h2 class="section__title" id="references-section-1">REFERENCES</h2><i aria-hidden="true" class="icon-section_arrow_d"></i></span><div class="accordion__content" style="display: none;" id="mec14478-bibl-0001">
               <ul class="rlist separator">
                  
                  
                  <li data-bib-id="mec14478-bib-0001"> <span class="author">Abarenkov, K.</span>,  <span class="author">Nilsson, R. H.</span>,  <span class="author">Larsson, K.‐H.</span>,  <span class="author">Alexander, I. J.</span>,  <span class="author">Eberhardt, U.</span>,  <span class="author">Erland, S.</span>, …  <span class="author">Koljalg, U.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">The UNITE database for molecular identification of fungi – recent updates and future perspectives</span>. <i>New Phytologist</i>,  <span class="vol">186</span>,  <span class="pageFirst">281</span>– <span class="pageLast">285</span>. <a href="https://doi.org/10.1111/j.1469-8137.2009.03160.x" class="linkBehavior">https://doi.org/10.1111/j.1469-8137.2009.03160.x</a><div class="extra-links" id="e_1_2_14_2_1"><span>
                           <a href="https://nph.onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2009.03160.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20409185" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000276024900006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=186&amp;publication_year=2010&amp;pages=281-285&amp;journal=New+Phytologist&amp;author=K.+Abarenkov&amp;author=R.+H.+Nilsson&amp;author=K.%E2%80%90H.+Larsson&amp;author=I.+J.+Alexander&amp;author=U.+Eberhardt&amp;author=S.+Erland&amp;author=U.+Koljalg&amp;title=The+UNITE+database+for+molecular+identification+of+fungi+%E2%80%93+recent+updates+and+future+perspectives" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0002"> <span class="author">Abdo, Z.</span>, &amp;  <span class="author">Golding, G. B.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">A step toward barcoding life: a model‐based, decision‐theoretic method to assign genes to preexisting species groups</span>. <i>Systematic Biology</i>,  <span class="vol">56</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">44</span>– <span class="pageLast">56</span>. <a href="https://doi.org/10.1080/10635150601167005" class="linkBehavior">https://doi.org/10.1080/10635150601167005</a><div class="extra-links" id="e_1_2_14_3_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1080%2F10635150601167005" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVenurjL" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=17366136" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000245054600004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=56&amp;publication_year=2007&amp;pages=44-56&amp;journal=Systematic+Biology&amp;issue=1&amp;author=Z.+Abdo&amp;author=G.+B.+Golding&amp;title=A+step+toward+barcoding+life%3A+a+model%E2%80%90based%2C+decision%E2%80%90theoretic+method+to+assign+genes+to+preexisting+species+groups" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0003"> <span class="author">Altschul, S. F.</span>,  <span class="author">Gish, W.</span>,  <span class="author">Miller, W.</span>,  <span class="author">Myers, E. W.</span>, &amp;  <span class="author">Lipman, D. J.</span> ( <span class="pubYear">1990</span>).  <span class="articleTitle">Basic local alignment search tool</span>. <i>Journal of Molecular Biology</i>,  <span class="vol">215</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">403</span>– <span class="pageLast">410</span>. <a href="https://doi.org/10.1016/S0022-2836%2805%2980360-2" class="linkBehavior">https://doi.org/10.1016/S0022-2836(05)80360-2</a><div class="extra-links" id="e_1_2_14_4_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2FS0022-2836%2805%2980360-2" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADyaK3MXitVGmsA%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=2231712" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=A1990ED16700008" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=215&amp;publication_year=1990&amp;pages=403-410&amp;journal=Journal+of+Molecular+Biology&amp;issue=3&amp;author=S.+F.+Altschul&amp;author=W.+Gish&amp;author=W.+Miller&amp;author=E.+W.+Myers&amp;author=D.+J.+Lipman&amp;title=Basic+local+alignment+search+tool" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0004"> <span class="author">Amend, A. S.</span>,  <span class="author">Seifert, K. A.</span>, &amp;  <span class="author">Bruns, T. D.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Quantifying microbial communities with 454 pyrosequencing: Does read abundance count?</span> <i>Molecular Ecology</i>,  <span class="vol">19</span>( <span class="citedIssue">24</span>),  <span class="pageFirst">5555</span>– <span class="pageLast">5565</span>. <a href="https://doi.org/10.1111/j.1365-294X.2010.04898.x" class="linkBehavior">https://doi.org/10.1111/j.1365-294X.2010.04898.x</a><div class="extra-links" id="e_1_2_14_5_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2010.04898.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlagsb0%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21050295" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000285012300020" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2010&amp;pages=5555-5565&amp;journal=Molecular+Ecology&amp;issue=24&amp;author=A.+S.+Amend&amp;author=K.+A.+Seifert&amp;author=T.+D.+Bruns&amp;title=Quantifying+microbial+communities+with+454+pyrosequencing%3A+Does+read+abundance+count%3F" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0005"> <span class="author">Ankenbrand, M. J.</span>,  <span class="author">Keller, A.</span>,  <span class="author">Wolf, M.</span>,  <span class="author">Schultz, J.</span>, &amp;  <span class="author">Förster, F.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">ITS2 database V: Twice as much</span>. <i>Molecular Biology and Evolution</i>,  <span class="vol">32</span>( <span class="citedIssue">11</span>),  <span class="pageFirst">3030</span>– <span class="pageLast">3032</span>. <a href="https://doi.org/10.1093/molbev/msv174" class="linkBehavior">https://doi.org/10.1093/molbev/msv174</a><div class="extra-links" id="e_1_2_14_6_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fmolbev%2Fmsv174" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KqtrjM" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=26248563" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000363033100019" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=32&amp;publication_year=2015&amp;pages=3030-3032&amp;journal=Molecular+Biology+and+Evolution&amp;issue=11&amp;author=M.+J.+Ankenbrand&amp;author=A.+Keller&amp;author=M.+Wolf&amp;author=J.+Schultz&amp;author=F.+F%C3%B6rster&amp;title=ITS2+database+V%3A+Twice+as+much" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0006"> <span class="author">Arya, M.</span>,  <span class="author">Shergill, I. S.</span>,  <span class="author">Williamson, M.</span>,  <span class="author">Gommersall, L.</span>,  <span class="author">Arya, N.</span>, &amp;  <span class="author">Patel, H. R.</span> ( <span class="pubYear">2005</span>).  <span class="articleTitle">Basic principles of real‐time quantitative PCR</span>. <i>Expert Review of Molecular Diagnostics</i>,  <span class="vol">5</span>( <span class="citedIssue">2</span>),  <span class="pageFirst">209</span>– <span class="pageLast">219</span>. <a href="https://doi.org/10.1586/14737159.5.2.209" class="linkBehavior">https://doi.org/10.1586/14737159.5.2.209</a><div class="extra-links" id="e_1_2_14_7_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1586%2F14737159.5.2.209" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjt1Kmu7c%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=15833050" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000230163700008" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2005&amp;pages=209-219&amp;journal=Expert+Review+of+Molecular+Diagnostics&amp;issue=2&amp;author=M.+Arya&amp;author=I.+S.+Shergill&amp;author=M.+Williamson&amp;author=L.+Gommersall&amp;author=N.+Arya&amp;author=H.+R.+Patel&amp;title=Basic+principles+of+real%E2%80%90time+quantitative+PCR" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0007"> <span class="author">Aylagas, E.</span>,  <span class="author">Borja, Á.</span>,  <span class="author">Irigoien, X.</span>, &amp;  <span class="author">Rodríguez‐Ezpeleta, N.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">Benchmarking DNA metabarcoding for biodiversity‐based monitoring and assessment</span>. <i>Frontiers in Marine Science</i>,  <span class="vol">3</span>,  <span class="pageFirst">96</span> <a href="https://doi.org/10.3389/fmars.2016.00096" class="linkBehavior">https://doi.org/10.3389/fmars.2016.00096</a><div class="extra-links" id="e_1_2_14_8_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.3389%2Ffmars.2016.00096" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2016&amp;pages=96&amp;journal=Frontiers+in+Marine+Science&amp;author=E.+Aylagas&amp;author=%C3%81.+Borja&amp;author=X.+Irigoien&amp;author=N.+Rodr%C3%ADguez%E2%80%90Ezpeleta&amp;title=Benchmarking+DNA+metabarcoding+for+biodiversity%E2%80%90based+monitoring+and+assessment" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0008"> <span class="author">Bailly, J.</span>,  <span class="author">Fraissinet‐Tachet, L.</span>,  <span class="author">Verner, M.‐C.</span>,  <span class="author">Debaud, J.‐C.</span>,  <span class="author">Lemaire, M.</span>,  <span class="author">Wésolowski‐Louvel, M.</span>, &amp;  <span class="author">Marmeisse, R.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">Soil eukaryotic functional diversity, a metatranscriptomic approach</span>. <i>ISME Journal</i>,  <span class="vol">1</span>( <span class="citedIssue">7</span>),  <span class="pageFirst">632</span>. <a href="https://doi.org/10.1038/ismej.2007.68" class="linkBehavior">https://doi.org/10.1038/ismej.2007.68</a><div class="extra-links" id="e_1_2_14_9_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fismej.2007.68" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVSku77M" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18043670" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000250894300006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2007&amp;pages=632&amp;journal=ISME+Journal&amp;issue=7&amp;author=J.+Bailly&amp;author=L.+Fraissinet%E2%80%90Tachet&amp;author=M.%E2%80%90C.+Verner&amp;author=J.%E2%80%90C.+Debaud&amp;author=M.+Lemaire&amp;author=M.+W%C3%A9solowski%E2%80%90Louvel&amp;author=R.+Marmeisse&amp;title=Soil+eukaryotic+functional+diversity%2C+a+metatranscriptomic+approach" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0009"> <span class="author">Baird, D. J.</span>, &amp;  <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Biomonitoring 2.0: A new paradigm in ecosystem assessment made possible by next‐generation DNA sequencing</span>. <i>Molecular Ecology</i>,  <span class="vol">21</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">2039</span>– <span class="pageLast">2044</span>. <a href="https://doi.org/10.1111/j.1365-294X.2012.05519.x" class="linkBehavior">https://doi.org/10.1111/j.1365-294X.2012.05519.x</a><div class="extra-links" id="e_1_2_14_10_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2012.05519.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22590728" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000302616200021" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=2039-2044&amp;journal=Molecular+Ecology&amp;issue=8&amp;author=D.+J.+Baird&amp;author=M.+Hajibabaei&amp;title=Biomonitoring+2.0%3A+A+new+paradigm+in+ecosystem+assessment+made+possible+by+next%E2%80%90generation+DNA+sequencing" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0010"> <span class="author">Banni, M.</span>,  <span class="author">Dondero, F.</span>,  <span class="author">Jebali, J.</span>,  <span class="author">Guerbej, H.</span>,  <span class="author">Boussetta, H.</span>, &amp;  <span class="author">Viarengo, A.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">Assessment
 of heavy metal contamination using real‐time PCR analysis of mussel 
metallothionein mt10 and mt20 expression: A validation along the 
Tunisian coast</span>. <i>Biomarkers</i>,  <span class="vol">12</span>( <span class="citedIssue">4</span>),  <span class="pageFirst">369</span>– <span class="pageLast">383</span>. <a href="https://doi.org/10.1080/13547500701217061" class="linkBehavior">https://doi.org/10.1080/13547500701217061</a><div class="extra-links" id="e_1_2_14_11_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1080%2F13547500701217061" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtlGrtrc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=17564843" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000247730100003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2007&amp;pages=369-383&amp;journal=Biomarkers&amp;issue=4&amp;author=M.+Banni&amp;author=F.+Dondero&amp;author=J.+Jebali&amp;author=H.+Guerbej&amp;author=H.+Boussetta&amp;author=A.+Viarengo&amp;title=Assessment+of+heavy+metal+contamination+using+real%E2%80%90time+PCR+analysis+of+mussel+metallothionein+mt10+and+mt20+expression%3A+A+validation+along+the+Tunisian+coast" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0011"> <span class="author">Basset, Y.</span>,  <span class="author">Cizek, L.</span>,  <span class="author">Cuénoud, P.</span>,  <span class="author">Didham, R. K.</span>,  <span class="author">Guilhaumon, F.</span>,  <span class="author">Missa, O.</span>, …  <span class="author">Leponce, M.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Arthropod diversity in a tropical forest</span>. <i>Science</i>,  <span class="vol">338</span>( <span class="citedIssue">6113</span>),  <span class="pageFirst">1481</span>– <span class="pageLast">1484</span>. <a href="https://doi.org/10.1126/science.1226727" class="linkBehavior">https://doi.org/10.1126/science.1226727</a><div class="extra-links" id="e_1_2_14_12_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1126%2Fscience.1226727" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSktb%252FO" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=23239740" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000312250800051" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=338&amp;publication_year=2012&amp;pages=1481-1484&amp;journal=Science&amp;issue=6113&amp;author=Y.+Basset&amp;author=L.+Cizek&amp;author=P.+Cu%C3%A9noud&amp;author=R.+K.+Didham&amp;author=F.+Guilhaumon&amp;author=O.+Missa&amp;author=M.+Leponce&amp;title=Arthropod+diversity+in+a+tropical+forest" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0012"> <span class="author">Bellemain, E.</span>,  <span class="author">Carlsen, T.</span>,  <span class="author">Brochmann, C.</span>,  <span class="author">Coissac, E.</span>,  <span class="author">Taberlet, P.</span>, &amp;  <span class="author">Kauserud, H.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">ITS as an environmental DNA barcode for fungi: An in silico approach reveals potential PCR biases</span>. <i>BMC Microbiology</i>,  <span class="vol">10</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">189</span>. <a href="https://doi.org/10.1186/1471-2180-10-189" class="linkBehavior">https://doi.org/10.1186/1471-2180-10-189</a><div class="extra-links" id="e_1_2_14_13_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1471-2180-10-189" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20618939" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000280805400003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2010&amp;pages=189&amp;journal=BMC+Microbiology&amp;issue=1&amp;author=E.+Bellemain&amp;author=T.+Carlsen&amp;author=C.+Brochmann&amp;author=E.+Coissac&amp;author=P.+Taberlet&amp;author=H.+Kauserud&amp;title=ITS+as+an+environmental+DNA+barcode+for+fungi%3A+An+in+silico+approach+reveals+potential+PCR+biases" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0013"> <span class="author">Bengtsson‐Palme, J.</span>,  <span class="author">Ryberg, M.</span>,  <span class="author">Hartmann, M.</span>,  <span class="author">Branco, S.</span>,  <span class="author">Wang, Z.</span>,  <span class="author">Godhe, A.</span>, …  <span class="author">Nilsson, R. H.</span> ( <span class="pubYear">2013</span>).  <span class="articleTitle">Improved
 software detection and extraction of ITS1 and ITS2 from ribosomal ITS 
sequences of fungi and other eukaryotes for analysis of environmental 
sequencing data</span>. <i>Methods in Ecology and Evolution</i>,  <span class="vol">4</span>,  <span class="pageFirst">914</span>– <span class="pageLast">919</span>. <a href="https://doi.org/10.1111/2041-210X.12073" class="linkBehavior">https://doi.org/10.1111/2041-210X.12073</a><div class="extra-links" id="e_1_2_14_14_1"><span>
                           <a href="https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.12073" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000325459600003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2013&amp;pages=914-919&amp;journal=Methods+in+Ecology+and+Evolution&amp;author=J.+Bengtsson%E2%80%90Palme&amp;author=M.+Ryberg&amp;author=M.+Hartmann&amp;author=S.+Branco&amp;author=Z.+Wang&amp;author=A.+Godhe&amp;author=R.+H.+Nilsson&amp;title=Improved+software+detection+and+extraction+of+ITS1+and+ITS2+from+ribosomal+ITS+sequences+of+fungi+and+other+eukaryotes+for+analysis+of+environmental+sequencing+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0014"> <span class="author">Berger, S. A.</span>,  <span class="author">Krompass, D.</span>, &amp;  <span class="author">Stamatakis, A.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Performance, accuracy, and web server for evolutionary placement of short sequence reads under maximum likelihood</span>. <i>Systematic Biology</i>,  <span class="vol">60</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">291</span>– <span class="pageLast">302</span>. <a href="https://doi.org/10.1093/sysbio/syr010" class="linkBehavior">https://doi.org/10.1093/sysbio/syr010</a><div class="extra-links" id="e_1_2_14_15_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fsysbio%2Fsyr010" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21436105" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000289805600004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=60&amp;publication_year=2011&amp;pages=291-302&amp;journal=Systematic+Biology&amp;issue=3&amp;author=S.+A.+Berger&amp;author=D.+Krompass&amp;author=A.+Stamatakis&amp;title=Performance%2C+accuracy%2C+and+web+server+for+evolutionary+placement+of+short+sequence+reads+under+maximum+likelihood" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-1111"> <span class="author">Bidartondo, M. I.</span> ( <span class="pubYear">2008</span>).  <span class="articleTitle">Preserving accuracy in GenBank</span>. <i>Science</i>,  <span class="vol">319</span>( <span class="citedIssue">5870</span>),  <span class="pageFirst">1616</span>. <a href="https://doi.org/10.1126/science.319.5870.1616a" class="linkBehavior">https://doi.org/10.1126/science.319.5870.1616a</a><div class="extra-links" id="e_1_2_14_16_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1126%2Fscience.319.5870.1616a" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktFWlu70%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18356505" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000254172300015" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=319&amp;publication_year=2008&amp;pages=1616&amp;journal=Science&amp;issue=5870&amp;author=M.+I.+Bidartondo&amp;title=Preserving+accuracy+in+GenBank" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0015"> <span class="author">Bik, H. M.</span> &amp;  <span class="groupName">Pitch Interactive Inc.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Phinch: An interactive, exploratory data visualization framework for–Omic datasets</span>. <i>bioRxiv</i>,  <span class="pageFirst">9944</span>.
                     <div class="extra-links" id="e_1_2_14_17_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2014&amp;pages=9944&amp;journal=bioRxiv&amp;author=H.+M.+Bik&amp;title=Phinch%3A+An+interactive%2C+exploratory+data+visualization+framework+for%E2%80%93Omic+datasets" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0016"> <span class="author">Bik, H. M.</span>,  <span class="author">Porazinska, D. L.</span>,  <span class="author">Creer, S.</span>,  <span class="author">Caporaso, J. G.</span>,  <span class="author">Knight, R.</span>, &amp;  <span class="author">Thomas, W. K.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Sequencing our way towards understanding global eukaryotic biodiversity</span>. <i>Trends in Ecology &amp; Evolution</i>,  <span class="vol">27</span>( <span class="citedIssue">4</span>),  <span class="pageFirst">233</span>– <span class="pageLast">243</span>. <a href="https://doi.org/10.1016/j.tree.2011.11.010" class="linkBehavior">https://doi.org/10.1016/j.tree.2011.11.010</a><div class="extra-links" id="e_1_2_14_18_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2Fj.tree.2011.11.010" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22244672" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000302976000011" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=2012&amp;pages=233-243&amp;journal=Trends+in+Ecology+%26+Evolution&amp;issue=4&amp;author=H.+M.+Bik&amp;author=D.+L.+Porazinska&amp;author=S.+Creer&amp;author=J.+G.+Caporaso&amp;author=R.+Knight&amp;author=W.+K.+Thomas&amp;title=Sequencing+our+way+towards+understanding+global+eukaryotic+biodiversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0017"> <span class="author">Boers, S. A.</span>,  <span class="author">Hays, J. P.</span>, &amp;  <span class="author">Jansen, R.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">Micelle PCR reduces chimera formation in 16S rRNA profiling of complex microbial DNA mixtures</span>. <i>Scientific Reports</i>,  <span class="vol">5</span>( <span class="citedIssue">1</span>), <a href="https://doi.org/10.1038/srep14181" class="linkBehavior">https://doi.org/10.1038/srep14181</a><div class="extra-links" id="e_1_2_14_19_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fsrep14181" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=26373611" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000361292200001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2015&amp;journal=Scientific+Reports&amp;issue=1&amp;author=S.+A.+Boers&amp;author=J.+P.+Hays&amp;author=R.+Jansen&amp;title=Micelle+PCR+reduces+chimera+formation+in+16S+rRNA+profiling+of+complex+microbial+DNA+mixtures" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0018"> <span class="author">Bohmann, K.</span>,  <span class="author">Evans, A.</span>,  <span class="author">Gilbert, M. T. P.</span>,  <span class="author">Carvalho, G. R.</span>,  <span class="author">Creer, S.</span>,  <span class="author">Knapp, M.</span>, …  <span class="author">Bruyn, M.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Environmental DNA for wildlife biology and biodiversity monitoring</span>. <i>Trends in Ecology &amp; Evolution</i>,  <span class="vol">29</span>( <span class="citedIssue">6</span>),  <span class="pageFirst">358</span>– <span class="pageLast">367</span>. <a href="https://doi.org/10.1016/j.tree.2014.04.003" class="linkBehavior">https://doi.org/10.1016/j.tree.2014.04.003</a><div class="extra-links" id="e_1_2_14_20_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2Fj.tree.2014.04.003" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=24821515" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000336873800011" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2014&amp;pages=358-367&amp;journal=Trends+in+Ecology+%26+Evolution&amp;issue=6&amp;author=K.+Bohmann&amp;author=A.+Evans&amp;author=M.+T.+P.+Gilbert&amp;author=G.+R.+Carvalho&amp;author=S.+Creer&amp;author=M.+Knapp&amp;author=M.+Bruyn&amp;title=Environmental+DNA+for+wildlife+biology+and+biodiversity+monitoring" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0019"> <span class="author">Brady, A.</span>, &amp;  <span class="author">Salzberg, S. L.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">Phymm and PhymmBL: Metagenomic phylogenetic classification with interpolated Markov models</span>. <i>Nature Methods</i>,  <span class="vol">6</span>( <span class="citedIssue">9</span>),  <span class="pageFirst">673</span>– <span class="pageLast">676</span>. <a href="https://doi.org/10.1038/nmeth.1358" class="linkBehavior">https://doi.org/10.1038/nmeth.1358</a><div class="extra-links" id="e_1_2_14_21_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnmeth.1358" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1OgtLo%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19648916" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000269382900018" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2009&amp;pages=673-676&amp;journal=Nature+Methods&amp;issue=9&amp;author=A.+Brady&amp;author=S.+L.+Salzberg&amp;title=Phymm+and+PhymmBL%3A+Metagenomic+phylogenetic+classification+with+interpolated+Markov+models" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0020"> <span class="author">Braid, M. D.</span>,  <span class="author">Daniels, L. M.</span>, &amp;  <span class="author">Kitts, C. L.</span> ( <span class="pubYear">2003</span>).  <span class="articleTitle">Removal of PCR inhibitors from soil DNA by chemical flocculation</span>. <i>Journal of Microbiological Methods</i>,  <span class="vol">52</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">389</span>– <span class="pageLast">393</span>. <a href="https://doi.org/10.1016/S0167-7012%2802%2900210-5" class="linkBehavior">https://doi.org/10.1016/S0167-7012(02)00210-5</a><div class="extra-links" id="e_1_2_14_22_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2FS0167-7012%2802%2900210-5" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtF2jsQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=12531508" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000180678300013" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2003&amp;pages=389-393&amp;journal=Journal+of+Microbiological+Methods&amp;issue=3&amp;author=M.+D.+Braid&amp;author=L.+M.+Daniels&amp;author=C.+L.+Kitts&amp;title=Removal+of+PCR+inhibitors+from+soil+DNA+by+chemical+flocculation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0021"> <span class="author">Bridge, P.</span>, &amp;  <span class="author">Spooner, B.</span> ( <span class="pubYear">2001</span>).  <span class="articleTitle">Soil fungi: Diversity and detection</span>. <i>Plant and Soil</i>,  <span class="vol">232</span>( <span class="citedIssue">1–2</span>),  <span class="pageFirst">147</span>– <span class="pageLast">154</span>. <a href="https://doi.org/10.1023/A:1010346305799" class="linkBehavior">https://doi.org/10.1023/A:1010346305799</a><div class="extra-links" id="e_1_2_14_23_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1023%2FA%3A1010346305799" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsVCksL0%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000169218200016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=232&amp;publication_year=2001&amp;pages=147-154&amp;journal=Plant+and+Soil&amp;issue=1%E2%80%932&amp;author=P.+Bridge&amp;author=B.+Spooner&amp;title=Soil+fungi%3A+Diversity+and+detection" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0022"> <span class="author">Bush, A.</span>,  <span class="author">Sollmann, R.</span>,  <span class="author">Wilting, A.</span>,  <span class="author">Bohmann, K.</span>,  <span class="author">Cole, B.</span>,  <span class="author">Balzter, H.</span>, …  <span class="author">Yu, D. W.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">Connecting Earth observation to high‐throughput biodiversity data</span>. <i>Nature Ecology &amp; Evolution</i>,  <span class="vol">1</span>( <span class="citedIssue">7</span>),  <span class="pageFirst">176</span>. <a href="https://doi.org/10.1038/s41559-017-0176" class="linkBehavior">https://doi.org/10.1038/s41559-017-0176</a><div class="extra-links" id="e_1_2_14_24_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fs41559-017-0176" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=28812589" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000417179000006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2017&amp;pages=176&amp;journal=Nature+Ecology+%26+Evolution&amp;issue=7&amp;author=A.+Bush&amp;author=R.+Sollmann&amp;author=A.+Wilting&amp;author=K.+Bohmann&amp;author=B.+Cole&amp;author=H.+Balzter&amp;author=D.+W.+Yu&amp;title=Connecting+Earth+observation+to+high%E2%80%90throughput+biodiversity+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0023"> <span class="author">Buss, D. F.</span>,  <span class="author">Baptista, D. F.</span>,  <span class="author">Silveira, M. P.</span>,  <span class="author">Nessimian, J. L.</span>, &amp;  <span class="author">Dorvillé, L. F.</span> ( <span class="pubYear">2002</span>).  <span class="articleTitle">Influence
 of water chemistry and environmental degradation on macroinvertebrate 
assemblages in a river basin in south‐east Brazil</span>. <i>Hydrobiologia</i>,  <span class="vol">481</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">125</span>– <span class="pageLast">136</span>. <a href="https://doi.org/10.1023/A:1021281508709" class="linkBehavior">https://doi.org/10.1023/A:1021281508709</a><div class="extra-links" id="e_1_2_14_25_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1023%2FA%3A1021281508709" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD38XovFShurs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000179387400010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=481&amp;publication_year=2002&amp;pages=125-136&amp;journal=Hydrobiologia&amp;issue=1&amp;author=D.+F.+Buss&amp;author=D.+F.+Baptista&amp;author=M.+P.+Silveira&amp;author=J.+L.+Nessimian&amp;author=L.+F.+Dorvill%C3%A9&amp;title=Influence+of+water+chemistry+and+environmental+degradation+on+macroinvertebrate+assemblages+in+a+river+basin+in+south%E2%80%90east+Brazil" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0024"> <span class="author">Callahan, B. J.</span>,  <span class="author">McMurdie, P. J.</span>, &amp;  <span class="author">Holmes, S. P.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">Exact sequence variants should replace operational taxonomic units in marker‐gene data analysis</span>. <i>ISME Journal</i>,  <span class="vol">11</span>,  <span class="pageFirst">2639</span>– <span class="pageLast">2643</span>. <a href="https://doi.org/10.1038/ismej.2017.119" class="linkBehavior">https://doi.org/10.1038/ismej.2017.119</a><div class="extra-links" id="e_1_2_14_26_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fismej.2017.119" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=28731476" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000415947900001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2017&amp;pages=2639-2643&amp;journal=ISME+Journal&amp;author=B.+J.+Callahan&amp;author=P.+J.+McMurdie&amp;author=S.+P.+Holmes&amp;title=Exact+sequence+variants+should+replace+operational+taxonomic+units+in+marker%E2%80%90gene+data+analysis" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0025"> <span class="author">Callahan, B. J.</span>,  <span class="author">McMurdie, P. J.</span>,  <span class="author">Rosen, M. J.</span>,  <span class="author">Han, A. W.</span>,  <span class="author">Johnson, A. J. A.</span>, &amp;  <span class="author">Holmes, S. P.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">DADA2: High‐resolution sample inference from Illumina amplicon data</span>. <i>Nature Methods</i>,  <span class="vol">13</span>( <span class="citedIssue">7</span>),  <span class="pageFirst">581</span>– <span class="pageLast">583</span>. <a href="https://doi.org/10.1038/nmeth.3869" class="linkBehavior">https://doi.org/10.1038/nmeth.3869</a><div class="extra-links" id="e_1_2_14_27_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnmeth.3869" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC28XosVWitb4%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=27214047" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000383794500017" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2016&amp;pages=581-583&amp;journal=Nature+Methods&amp;issue=7&amp;author=B.+J.+Callahan&amp;author=P.+J.+McMurdie&amp;author=M.+J.+Rosen&amp;author=A.+W.+Han&amp;author=A.+J.+A.+Johnson&amp;author=S.+P.+Holmes&amp;title=DADA2%3A+High%E2%80%90resolution+sample+inference+from+Illumina+amplicon+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0026"> <span class="author">Caporaso, J. G.</span>,  <span class="author">Kuczynski, J.</span>,  <span class="author">Stombaugh, J.</span>,  <span class="author">Bittinger, K.</span>,  <span class="author">Bushman, F. D.</span>,  <span class="author">Costello, E. K.</span>, …  <span class="author">Knight, R.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">QIIME allows analysis of high‐throughput community sequencing data</span>. <i>Nature Methods</i>,  <span class="vol">7</span>( <span class="citedIssue">5</span>),  <span class="pageFirst">335</span>– <span class="pageLast">336</span>. <a href="https://doi.org/10.1038/nmeth.f.303" class="linkBehavior">https://doi.org/10.1038/nmeth.f.303</a><div class="extra-links" id="e_1_2_14_28_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnmeth.f.303" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFalurg%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20383131" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000277175100003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2010&amp;pages=335-336&amp;journal=Nature+Methods&amp;issue=5&amp;author=J.+G.+Caporaso&amp;author=J.+Kuczynski&amp;author=J.+Stombaugh&amp;author=K.+Bittinger&amp;author=F.+D.+Bushman&amp;author=E.+K.+Costello&amp;author=R.+Knight&amp;title=QIIME+allows+analysis+of+high%E2%80%90throughput+community+sequencing+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0027"> <span class="author">Carvalhais, L. C.</span>,  <span class="author">Dennis, P. G.</span>,  <span class="author">Tyson, G. W.</span>, &amp;  <span class="author">Schenk, P. M.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Application of metatranscriptomics to soil environments</span>. <i>Journal of Microbiological Methods</i>,  <span class="vol">91</span>( <span class="citedIssue">2</span>),  <span class="pageFirst">246</span>– <span class="pageLast">251</span>. <a href="https://doi.org/10.1016/j.mimet.2012.08.011" class="linkBehavior">https://doi.org/10.1016/j.mimet.2012.08.011</a><div class="extra-links" id="e_1_2_14_29_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2Fj.mimet.2012.08.011" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKksLfF" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22963791" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000310863800005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=91&amp;publication_year=2012&amp;pages=246-251&amp;journal=Journal+of+Microbiological+Methods&amp;issue=2&amp;author=L.+C.+Carvalhais&amp;author=P.+G.+Dennis&amp;author=G.+W.+Tyson&amp;author=P.+M.+Schenk&amp;title=Application+of+metatranscriptomics+to+soil+environments" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0028"> <span class="author">Chou, C.‐C.</span>,  <span class="author">Lee, T.‐T.</span>,  <span class="author">Chen, C.‐H.</span>,  <span class="author">Hsiao, H.‐Y.</span>,  <span class="author">Lin, Y.‐L.</span>,  <span class="author">Ho, M.‐S.</span>, …  <span class="author">Peck, K.</span> ( <span class="pubYear">2006</span>).  <span class="articleTitle">Design of microarray probes for virus identification and detection of emerging viruses at the genus level</span>. <i>BMC Bioinformatics</i>,  <span class="vol">7</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">232</span>. <a href="https://doi.org/10.1186/1471-2105-7-232" class="linkBehavior">https://doi.org/10.1186/1471-2105-7-232</a><div class="extra-links" id="e_1_2_14_30_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1471-2105-7-232" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=16643672" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000239303200001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2006&amp;pages=232&amp;journal=BMC+Bioinformatics&amp;issue=1&amp;author=C.%E2%80%90C.+Chou&amp;author=T.%E2%80%90T.+Lee&amp;author=C.%E2%80%90H.+Chen&amp;author=H.%E2%80%90Y.+Hsiao&amp;author=Y.%E2%80%90L.+Lin&amp;author=M.%E2%80%90S.+Ho&amp;author=K.+Peck&amp;title=Design+of+microarray+probes+for+virus+identification+and+detection+of+emerging+viruses+at+the+genus+level" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0029"> <span class="author">Claesson, M. J.</span>,  <span class="author">Wang, Q.</span>,  <span class="author">O'Sullivan, O.</span>,  <span class="author">Greene‐Diniz, R.</span>,  <span class="author">Cole, J. R.</span>,  <span class="author">Ross, R. P.</span>, &amp;  <span class="author">O'Toole, P. W.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Comparison
 of two next‐generation sequencing technologies for resolving highly 
complex microbiota composition using tandem variable 16S rRNA gene 
regions</span>. <i>Nucleic Acids Research</i>,  <span class="vol">38</span>( <span class="citedIssue">22</span>),  <span class="pageFirst">e200</span>. <a href="https://doi.org/10.1093/nar/gkq873" class="linkBehavior">https://doi.org/10.1093/nar/gkq873</a><div class="extra-links" id="e_1_2_14_31_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fnar%2Fgkq873" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20880993" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000285420300004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=2010&amp;pages=e200&amp;journal=Nucleic+Acids+Research&amp;issue=22&amp;author=M.+J.+Claesson&amp;author=Q.+Wang&amp;author=O.+O%27Sullivan&amp;author=R.+Greene%E2%80%90Diniz&amp;author=J.+R.+Cole&amp;author=R.+P.+Ross&amp;author=P.+W.+O%27Toole&amp;title=Comparison+of+two+next%E2%80%90generation+sequencing+technologies+for+resolving+highly+complex+microbiota+composition+using+tandem+variable+16S+rRNA+gene+regions" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0030"> <span class="author">Coissac, E.</span>,  <span class="author">Riaz, T.</span>, &amp;  <span class="author">Puillandre, N.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Bioinformatic challenges for DNA metabarcoding of plants and animals: Bioinformatic for DNA metabarcoding</span>. <i>Molecular Ecology</i>,  <span class="vol">21</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">1834</span>– <span class="pageLast">1847</span>. <a href="https://doi.org/10.1111/j.1365-294X.2012.05550.x" class="linkBehavior">https://doi.org/10.1111/j.1365-294X.2012.05550.x</a><div class="extra-links" id="e_1_2_14_32_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2012.05550.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVGksbw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22486822" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000302616200006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=1834-1847&amp;journal=Molecular+Ecology&amp;issue=8&amp;author=E.+Coissac&amp;author=T.+Riaz&amp;author=N.+Puillandre&amp;title=Bioinformatic+challenges+for+DNA+metabarcoding+of+plants+and+animals%3A+Bioinformatic+for+DNA+metabarcoding" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-2222"> <span class="author">Cole, J. R.</span>,  <span class="author">Wang, Q.</span>,  <span class="author">Cardenas, E.</span>,  <span class="author">Fish, J.</span>,  <span class="author">Chai, B.</span>,  <span class="author">Farris, R. J.</span>, …  <span class="author">Tiedje, J. M.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">The ribosomal database project: Improved alignments and new tools for rRNA analysis</span>. <i>Nucleic Acids Research</i>,  <span class="vol">37</span>( <span class="citedIssue">Database</span>),  <span class="pageFirst">D141</span>– <span class="pageLast">D145</span>. <a href="https://doi.org/10.1093/nar/gkn879" class="linkBehavior">https://doi.org/10.1093/nar/gkn879</a><div class="extra-links" id="e_1_2_14_33_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fnar%2Fgkn879" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFejtLbF" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19004872" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000261906200025" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=37&amp;publication_year=2009&amp;pages=D141-D145&amp;journal=Nucleic+Acids+Research&amp;issue=Database&amp;author=J.+R.+Cole&amp;author=Q.+Wang&amp;author=E.+Cardenas&amp;author=J.+Fish&amp;author=B.+Chai&amp;author=R.+J.+Farris&amp;author=J.+M.+Tiedje&amp;title=The+ribosomal+database+project%3A+Improved+alignments+and+new+tools+for+rRNA+analysis" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0031"> <span class="author">Cole, J. R.</span>,  <span class="author">Wang, Q.</span>,  <span class="author">Fish, J. A.</span>,  <span class="author">Chai, B.</span>,  <span class="author">McGarrell, D. M.</span>,  <span class="author">Sun, Y.</span>, …  <span class="author">Tiedje, J. M.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Ribosomal Database Project: Data and tools for high throughput rRNA analysis</span>. <i>Nucleic Acids Research</i>,  <span class="vol">42</span>( <span class="citedIssue">D1</span>),  <span class="pageFirst">D633</span>– <span class="pageLast">D642</span>. <a href="https://doi.org/10.1093/nar/gkt1244" class="linkBehavior">https://doi.org/10.1093/nar/gkt1244</a><div class="extra-links" id="e_1_2_14_34_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fnar%2Fgkt1244" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslGk" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=24288368" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000331139800093" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=42&amp;publication_year=2014&amp;pages=D633-D642&amp;journal=Nucleic+Acids+Research&amp;issue=D1&amp;author=J.+R.+Cole&amp;author=Q.+Wang&amp;author=J.+A.+Fish&amp;author=B.+Chai&amp;author=D.+M.+McGarrell&amp;author=Y.+Sun&amp;author=J.+M.+Tiedje&amp;title=Ribosomal+Database+Project%3A+Data+and+tools+for+high+throughput+rRNA+analysis" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0032"> <span class="author">Cox‐Foster, D. L.</span>,  <span class="author">Conlan, S.</span>,  <span class="author">Holmes, E. C.</span>,  <span class="author">Palacios, G.</span>,  <span class="author">Evans, J. D.</span>,  <span class="author">Moran, N. A.</span>, …  <span class="author">Lipkin, W. I.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">A metagenomic survey of microbes in honey bee colony collapse disorder</span>. <i>Science</i>,  <span class="vol">318</span>,  <span class="pageFirst">283</span>– <span class="pageLast">287</span>. <a href="https://doi.org/10.1126/science.1146498" class="linkBehavior">https://doi.org/10.1126/science.1146498</a><div class="extra-links" id="e_1_2_14_35_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1126%2Fscience.1146498" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFCjt7vM" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=17823314" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000250086100050" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=318&amp;publication_year=2007&amp;pages=283-287&amp;journal=Science&amp;author=D.+L.+Cox%E2%80%90Foster&amp;author=S.+Conlan&amp;author=E.+C.+Holmes&amp;author=G.+Palacios&amp;author=J.+D.+Evans&amp;author=N.+A.+Moran&amp;author=W.+I.+Lipkin&amp;title=A+metagenomic+survey+of+microbes+in+honey+bee+colony+collapse+disorder" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0033"> <span class="author">Creer, S.</span>,  <span class="author">Fonseca, V. G.</span>,  <span class="author">Porazinska, D. L.</span>,  <span class="author">Giblin‐Davis, R. M.</span>,  <span class="author">Sung, W.</span>,  <span class="author">Power, D. M.</span>, …  <span class="author">Thomas, W. K.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Ultrasequencing of the meiofaunal biosphere: Practice, pitfalls and promises</span>. <i>Molecular Ecology</i>,  <span class="vol">19</span>,  <span class="pageFirst">4</span>– <span class="pageLast">20</span>. <a href="https://doi.org/10.1111/j.1365-294X.2009.04473.x" class="linkBehavior">https://doi.org/10.1111/j.1365-294X.2009.04473.x</a><div class="extra-links" id="e_1_2_14_36_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2009.04473.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20331766" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000275645700002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2010&amp;pages=4-20&amp;journal=Molecular+Ecology&amp;author=S.+Creer&amp;author=V.+G.+Fonseca&amp;author=D.+L.+Porazinska&amp;author=R.+M.+Giblin%E2%80%90Davis&amp;author=W.+Sung&amp;author=D.+M.+Power&amp;author=W.+K.+Thomas&amp;title=Ultrasequencing+of+the+meiofaunal+biosphere%3A+Practice%2C+pitfalls+and+promises" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0034"> <span class="author">Cronn, R.</span>,  <span class="author">Liston, A.</span>,  <span class="author">Parks, M.</span>,  <span class="author">Gernandt, D. S.</span>,  <span class="author">Shen, R.</span>, &amp;  <span class="author">Mockler, T.</span> ( <span class="pubYear">2008</span>).  <span class="articleTitle">Multiplex sequencing of plant chloroplast genomes using Solexa sequencing‐by‐synthesis technology</span>. <i>Nucleic Acids Research</i>,  <span class="vol">36</span>( <span class="citedIssue">19</span>),  <span class="pageFirst">e122</span>. <a href="https://doi.org/10.1093/nar/gkn502" class="linkBehavior">https://doi.org/10.1093/nar/gkn502</a><div class="extra-links" id="e_1_2_14_37_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fnar%2Fgkn502" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlGgs73M" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18753151" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000260983000001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2008&amp;pages=e122&amp;journal=Nucleic+Acids+Research&amp;issue=19&amp;author=R.+Cronn&amp;author=A.+Liston&amp;author=M.+Parks&amp;author=D.+S.+Gernandt&amp;author=R.+Shen&amp;author=T.+Mockler&amp;title=Multiplex+sequencing+of+plant+chloroplast+genomes+using+Solexa+sequencing%E2%80%90by%E2%80%90synthesis+technology" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0035"> <span class="author">D'Costa, V. M.</span>,  <span class="author">King, C. E.</span>,  <span class="author">Kalan, L.</span>,  <span class="author">Morar, M.</span>,  <span class="author">Sung, W. W. L.</span>,  <span class="author">Schwarz, C.</span>, …  <span class="author">Wright, G. D.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Antibiotic resistance is ancient</span>. <i>Nature</i>,  <span class="vol">477</span>( <span class="citedIssue">7365</span>),  <span class="pageFirst">457</span>– <span class="pageLast">461</span>. <a href="https://doi.org/10.1038/nature10388" class="linkBehavior">https://doi.org/10.1038/nature10388</a><div class="extra-links" id="e_1_2_14_38_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnature10388" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFers7rJ" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21881561" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000295080500039" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=477&amp;publication_year=2011&amp;pages=457-461&amp;journal=Nature&amp;issue=7365&amp;author=V.+M.+D%27Costa&amp;author=C.+E.+King&amp;author=L.+Kalan&amp;author=M.+Morar&amp;author=W.+W.+L.+Sung&amp;author=C.+Schwarz&amp;author=G.+D.+Wright&amp;title=Antibiotic+resistance+is+ancient" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0036"> <span class="author">Deagle, B. E.</span>,  <span class="author">Jarman, S. N.</span>,  <span class="author">Coissac, E.</span>,  <span class="author">Pompanon, F.</span>, &amp;  <span class="author">Taberlet, P.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">DNA metabarcoding and the cytochrome c oxidase subunit I marker: Not a perfect match</span>. <i>Biology Letters</i>,  <span class="vol">10</span>( <span class="citedIssue">9</span>),  <span class="pageFirst">20140562</span>. <a href="https://doi.org/10.1098/rsbl.2014.0562" class="linkBehavior">https://doi.org/10.1098/rsbl.2014.0562</a><div class="extra-links" id="e_1_2_14_39_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1098%2Frsbl.2014.0562" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=25209199" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000341935400005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2014&amp;pages=20140562&amp;journal=Biology+Letters&amp;issue=9&amp;author=B.+E.+Deagle&amp;author=S.+N.+Jarman&amp;author=E.+Coissac&amp;author=F.+Pompanon&amp;author=P.+Taberlet&amp;title=DNA+metabarcoding+and+the+cytochrome+c+oxidase+subunit+I+marker%3A+Not+a+perfect+match" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0037"> <span class="author">Deiner, K.</span>,  <span class="author">Bik, H. M.</span>,  <span class="author">Mächler, E.</span>,  <span class="author">Seymour, M.</span>,  <span class="author">Lacoursière‐Roussel, A.</span>,  <span class="author">Altermatt, F.</span>, …  <span class="author">Bernatchez, L.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">Environmental DNA metabarcoding: Transforming how we survey animal and plant communities</span>. <i>Molecular Ecology</i>,  <span class="vol">26</span>,  <span class="pageFirst">5872</span>– <span class="pageLast">5895</span>.
                     <div class="extra-links" id="e_1_2_14_40_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.14350" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=28921802" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000415362500002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2017&amp;pages=5872-5895&amp;journal=Molecular+Ecology&amp;author=K.+Deiner&amp;author=H.+M.+Bik&amp;author=E.+M%C3%A4chler&amp;author=M.+Seymour&amp;author=A.+Lacoursi%C3%A8re%E2%80%90Roussel&amp;author=F.+Altermatt&amp;author=L.+Bernatchez&amp;title=Environmental+DNA+metabarcoding%3A+Transforming+how+we+survey+animal+and+plant+communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0038"> <span class="author">Dejean, T.</span>,  <span class="author">Valentini, A.</span>,  <span class="author">Duparc, A.</span>,  <span class="author">Pellier‐Cuit, S.</span>,  <span class="author">Pompanon, F.</span>,  <span class="author">Taberlet, P.</span>, &amp;  <span class="author">Miaud, C.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Persistence of environmental DNA in freshwater ecosystems</span>. <i>PLoS ONE</i>,  <span class="vol">6</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">e23398</span>. <a href="https://doi.org/10.1371/journal.pone.0023398" class="linkBehavior">https://doi.org/10.1371/journal.pone.0023398</a><div class="extra-links" id="e_1_2_14_41_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0023398" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFanur3E" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21858099" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000293773300026" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2011&amp;pages=e23398&amp;journal=PLoS+ONE&amp;issue=8&amp;author=T.+Dejean&amp;author=A.+Valentini&amp;author=A.+Duparc&amp;author=S.+Pellier%E2%80%90Cuit&amp;author=F.+Pompanon&amp;author=P.+Taberlet&amp;author=C.+Miaud&amp;title=Persistence+of+environmental+DNA+in+freshwater+ecosystems" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0039"> <span class="author">DeSantis, T. Z.</span>,  <span class="author">Brodie, E. L.</span>,  <span class="author">Moberg, J. P.</span>,  <span class="author">Zubieta, I. X.</span>,  <span class="author">Piceno, Y. M.</span>, &amp;  <span class="author">Andersen, G. L.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">High‐density
 universal 16S rRNA microarray analysis reveals broader diversity than 
typical clone library when sampling the environment</span>. <i>Microbial Ecology</i>,  <span class="vol">53</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">371</span>– <span class="pageLast">383</span>. <a href="https://doi.org/10.1007/s00248-006-9134-9" class="linkBehavior">https://doi.org/10.1007/s00248-006-9134-9</a><div class="extra-links" id="e_1_2_14_42_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1007%2Fs00248-006-9134-9" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXntVSgtrY%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=17334858" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000246232000002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=53&amp;publication_year=2007&amp;pages=371-383&amp;journal=Microbial+Ecology&amp;issue=3&amp;author=T.+Z.+DeSantis&amp;author=E.+L.+Brodie&amp;author=J.+P.+Moberg&amp;author=I.+X.+Zubieta&amp;author=Y.+M.+Piceno&amp;author=G.+L.+Andersen&amp;title=High%E2%80%90density+universal+16S+rRNA+microarray+analysis+reveals+broader+diversity+than+typical+clone+library+when+sampling+the+environment" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0040"> <span class="author">DeSantis, T. Z.</span>,  <span class="author">Hugenholtz, P.</span>,  <span class="author">Larsen, N.</span>,  <span class="author">Rojas, M.</span>,  <span class="author">Brodie, E. L.</span>,  <span class="author">Keller, K.</span>, …  <span class="author">Andersen, G. L.</span> ( <span class="pubYear">2006</span>).  <span class="articleTitle">Greengenes, a Chimera‐checked 16S rRNA gene database and workbench compatible with ARB</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">72</span>( <span class="citedIssue">7</span>),  <span class="pageFirst">5069</span>– <span class="pageLast">5072</span>. <a href="https://doi.org/10.1128/AEM.03006-05" class="linkBehavior">https://doi.org/10.1128/AEM.03006-05</a><div class="extra-links" id="e_1_2_14_43_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1128%2FAEM.03006-05" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsVaqtLg%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=16820507" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000238961000071" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=72&amp;publication_year=2006&amp;pages=5069-5072&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=7&amp;author=T.+Z.+DeSantis&amp;author=P.+Hugenholtz&amp;author=N.+Larsen&amp;author=M.+Rojas&amp;author=E.+L.+Brodie&amp;author=K.+Keller&amp;author=G.+L.+Andersen&amp;title=Greengenes%2C+a+Chimera%E2%80%90checked+16S+rRNA+gene+database+and+workbench+compatible+with+ARB" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0041"> <span class="author">Devault, A. M.</span>,  <span class="author">McLoughlin, K.</span>,  <span class="author">Jaing, C.</span>,  <span class="author">Gardner, S.</span>,  <span class="author">Porter, T. M.</span>,  <span class="author">Enk, J. M.</span>, …  <span class="author">Poinar, H. N.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Ancient pathogen DNA in archaeological samples detected with a Microbial Detection Array</span>. <i>Scientific Reports</i>,  <span class="vol">4</span>,  <span class="pageFirst">4245</span>. <a href="https://doi.org/10.1038/srep04245" class="linkBehavior">https://doi.org/10.1038/srep04245</a><div class="extra-links" id="e_1_2_14_44_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fsrep04245" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=24603850" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000332325900001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2014&amp;pages=4245&amp;journal=Scientific+Reports&amp;author=A.+M.+Devault&amp;author=K.+McLoughlin&amp;author=C.+Jaing&amp;author=S.+Gardner&amp;author=T.+M.+Porter&amp;author=J.+M.+Enk&amp;author=H.+N.+Poinar&amp;title=Ancient+pathogen+DNA+in+archaeological+samples+detected+with+a+Microbial+Detection+Array" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0042"> <span class="author">Dodsworth, S.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">Genome skimming for next‐generation biodiversity analysis</span>. <i>Trends in Plant Science</i>,  <span class="vol">20</span>( <span class="citedIssue">9</span>),  <span class="pageFirst">525</span>– <span class="pageLast">527</span>. <a href="https://doi.org/10.1016/j.tplants.2015.06.012" class="linkBehavior">https://doi.org/10.1016/j.tplants.2015.06.012</a><div class="extra-links" id="e_1_2_14_45_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2Fj.tplants.2015.06.012" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOksr%252FN" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=26205170" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000361920700001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2015&amp;pages=525-527&amp;journal=Trends+in+Plant+Science&amp;issue=9&amp;author=S.+Dodsworth&amp;title=Genome+skimming+for+next%E2%80%90generation+biodiversity+analysis" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0043"> <span class="author">Doi, H.</span>,  <span class="author">Uchii, K.</span>,  <span class="author">Takahara, T.</span>,  <span class="author">Matsuhashi, S.</span>,  <span class="author">Yamanaka, H.</span>, &amp;  <span class="author">Minamoto, T.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">Use of droplet digital PCR for estimation of fish abundance and biomass in environmental DNA surveys</span>. <i>PLoS ONE</i>,  <span class="vol">10</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">e0122763</span>. <a href="https://doi.org/10.1371/journal.pone.0122763" class="linkBehavior">https://doi.org/10.1371/journal.pone.0122763</a><div class="extra-links" id="e_1_2_14_46_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0122763" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=25799582" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000351987300342" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2015&amp;pages=e0122763&amp;journal=PLoS+ONE&amp;issue=3&amp;author=H.+Doi&amp;author=K.+Uchii&amp;author=T.+Takahara&amp;author=S.+Matsuhashi&amp;author=H.+Yamanaka&amp;author=T.+Minamoto&amp;title=Use+of+droplet+digital+PCR+for+estimation+of+fish+abundance+and+biomass+in+environmental+DNA+surveys" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0044"> <span class="author">Dutse, S. W.</span>, &amp;  <span class="author">Yusof, N. A.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Microfluidics‐based lab‐on‐chip systems in DNA‐based biosensing: an overview</span>. <i>Sensors</i>,  <span class="vol">11</span>( <span class="citedIssue">12</span>),  <span class="pageFirst">5754</span>– <span class="pageLast">5768</span>. <a href="https://doi.org/10.3390/s110605754" class="linkBehavior">https://doi.org/10.3390/s110605754</a><div class="extra-links" id="e_1_2_14_47_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.3390%2Fs110605754" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFCrtLY%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000292026400014" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2011&amp;pages=5754-5768&amp;journal=Sensors&amp;issue=12&amp;author=S.+W.+Dutse&amp;author=N.+A.+Yusof&amp;title=Microfluidics%E2%80%90based+lab%E2%80%90on%E2%80%90chip+systems+in+DNA%E2%80%90based+biosensing%3A+an+overview" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0045"> <span class="author">Easley, C. J.</span>,  <span class="author">Karlinsey, J. M.</span>,  <span class="author">Bienvenue, J. M.</span>,  <span class="author">Legendre, L. A.</span>,  <span class="author">Roper, M. G.</span>,  <span class="author">Feldman, S. H.</span>, …  <span class="author">Ferrance, J. P.</span> ( <span class="pubYear">2006</span>).  <span class="articleTitle">A fully integrated microfluidic genetic analysis system with sample‐in–answer‐out capability</span>. <i>Proceedings of the National Academy of Sciences</i>,  <span class="vol">103</span>( <span class="citedIssue">51</span>),  <span class="pageFirst">19272</span>– <span class="pageLast">19277</span>. <a href="https://doi.org/10.1073/pnas.0604663103" class="linkBehavior">https://doi.org/10.1073/pnas.0604663103</a><div class="extra-links" id="e_1_2_14_48_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1073%2Fpnas.0604663103" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhs1ymtg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=17159153" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000243166600012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=103&amp;publication_year=2006&amp;pages=19272-19277&amp;journal=Proceedings+of+the+National+Academy+of+Sciences&amp;issue=51&amp;author=C.+J.+Easley&amp;author=J.+M.+Karlinsey&amp;author=J.+M.+Bienvenue&amp;author=L.+A.+Legendre&amp;author=M.+G.+Roper&amp;author=S.+H.+Feldman&amp;author=J.+P.+Ferrance&amp;title=A+fully+integrated+microfluidic+genetic+analysis+system+with+sample%E2%80%90in%E2%80%93answer%E2%80%90out+capability" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-3000"> <span class="author">Ebach, M. C.</span>,  <span class="author">Valdecasas, A. G.</span>, &amp;  <span class="author">Wheeler, Q. D.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Impediments to taxonomy and users of taxonomy: Accessibility and impact evaluation</span>. <i>Cladistics</i>,  <span class="vol">27</span>( <span class="citedIssue">5</span>),  <span class="pageFirst">550</span>– <span class="pageLast">557</span>. <a href="https://doi.org/10.1111/j.1096-0031.2011.00348.x" class="linkBehavior">https://doi.org/10.1111/j.1096-0031.2011.00348.x</a><div class="extra-links" id="e_1_2_14_49_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1096-0031.2011.00348.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000294911800007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=2011&amp;pages=550-557&amp;journal=Cladistics&amp;issue=5&amp;author=M.+C.+Ebach&amp;author=A.+G.+Valdecasas&amp;author=Q.+D.+Wheeler&amp;title=Impediments+to+taxonomy+and+users+of+taxonomy%3A+Accessibility+and+impact+evaluation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0046"> <span class="author">Edgar, R. C.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Search and clustering orders of magnitude faster than BLAST</span>. <i>Bioinformatics</i>,  <span class="vol">26</span>( <span class="citedIssue">19</span>),  <span class="pageFirst">2460</span>– <span class="pageLast">2461</span>. <a href="https://doi.org/10.1093/bioinformatics/btq461" class="linkBehavior">https://doi.org/10.1093/bioinformatics/btq461</a><div class="extra-links" id="e_1_2_14_50_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fbioinformatics%2Fbtq461" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1WhtbzM" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20709691" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000282170000016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2010&amp;pages=2460-2461&amp;journal=Bioinformatics&amp;issue=19&amp;author=R.+C.+Edgar&amp;title=Search+and+clustering+orders+of+magnitude+faster+than+BLAST" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0047"> <span class="author">Edgar, R. C.</span> ( <span class="pubYear">2013</span>).  <span class="articleTitle">UPARSE: Highly accurate OTU sequences from microbial amplicon reads</span>. <i>Nature Methods</i>,  <span class="vol">10</span>( <span class="citedIssue">10</span>),  <span class="pageFirst">996</span>– <span class="pageLast">998</span>. <a href="https://doi.org/10.1038/nmeth.2604" class="linkBehavior">https://doi.org/10.1038/nmeth.2604</a><div class="extra-links" id="e_1_2_14_51_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnmeth.2604" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1ylsLzE" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=23955772" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000325073800023" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2013&amp;pages=996-998&amp;journal=Nature+Methods&amp;issue=10&amp;author=R.+C.+Edgar&amp;title=UPARSE%3A+Highly+accurate+OTU+sequences+from+microbial+amplicon+reads" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0048"> <span class="author">Edgar, R. C.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">UNOISE2: Improved error‐correction for Illumina 16S and ITS amplicon sequencing</span>. <i>bioRxiv</i>,  <span class="pageFirst">081257</span>.
                     <div class="extra-links" id="e_1_2_14_52_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2016&amp;pages=081257&amp;journal=bioRxiv&amp;author=R.+C.+Edgar&amp;title=UNOISE2%3A+Improved+error%E2%80%90correction+for+Illumina+16S+and+ITS+amplicon+sequencing" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0049"> <span class="author">Edgar, R. C.</span> ( <span class="pubYear">2017a</span>).  <span class="articleTitle">Accuracy of microbial community diversity estimated by closed‐ and open‐reference OTUs</span>. <i>PeerJ</i>,  <span class="vol">5</span>,  <span class="pageFirst">e3889</span>. <a href="https://doi.org/10.7717/peerj.3889" class="linkBehavior">https://doi.org/10.7717/peerj.3889</a><div class="extra-links" id="e_1_2_14_53_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.7717%2Fpeerj.3889" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=29018622" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000412219600002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2017a&amp;pages=e3889&amp;journal=PeerJ&amp;author=R.+C.+Edgar&amp;title=Accuracy+of+microbial+community+diversity+estimated+by+closed%E2%80%90+and+open%E2%80%90reference+OTUs" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0050"> <span class="author">Edgar, R. C.</span> ( <span class="pubYear">2017b</span>).  <span class="articleTitle">Updating the 97% identity threshold for 16S ribosomal RNA OTUs</span>. <i>bioRxiv</i>. <a href="https://doi.org/10.1101/192211" class="linkBehavior">https://doi.org/10.1101/192211</a><div class="extra-links" id="e_1_2_14_54_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2017b&amp;journal=bioRxiv&amp;author=R.+C.+Edgar&amp;title=Updating+the+97%25+identity+threshold+for+16S+ribosomal+RNA+OTUs" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0051"> <span class="author">Eisen, J. A.</span> ( <span class="pubYear">1998</span>).  <span class="articleTitle">Phylogenomics: Improving functional predictions for uncharacterized genes by evolutionary analysis</span>. <i>Genome Research</i>,  <span class="vol">8</span>,  <span class="pageFirst">163</span>– <span class="pageLast">167</span>. <a href="https://doi.org/10.1101/gr.8.3.163" class="linkBehavior">https://doi.org/10.1101/gr.8.3.163</a><div class="extra-links" id="e_1_2_14_55_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1101%2Fgr.8.3.163" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADyaK1cXitlWluro%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=9521918" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000072838200003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=1998&amp;pages=163-167&amp;journal=Genome+Research&amp;author=J.+A.+Eisen&amp;title=Phylogenomics%3A+Improving+functional+predictions+for+uncharacterized+genes+by+evolutionary+analysis" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0052"> <span class="author">Elbrecht, V.</span>, &amp;  <span class="author">Leese, F.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">Can
 DNA‐based ecosystem assessments quantify species abundance? Testing 
primer bias and biomass—sequence relationships with an innovative 
metabarcoding protocol</span>. <i>PLoS ONE</i>,  <span class="vol">10</span>( <span class="citedIssue">7</span>),  <span class="pageFirst">e0130324</span>. <a href="https://doi.org/10.1371/journal.pone.0130324" class="linkBehavior">https://doi.org/10.1371/journal.pone.0130324</a><div class="extra-links" id="e_1_2_14_56_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0130324" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=26154168" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000358159700031" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2015&amp;pages=e0130324&amp;journal=PLoS+ONE&amp;issue=7&amp;author=V.+Elbrecht&amp;author=F.+Leese&amp;title=Can+DNA%E2%80%90based+ecosystem+assessments+quantify+species+abundance%3F+Testing+primer+bias+and+biomass%E2%80%94sequence+relationships+with+an+innovative+metabarcoding+protocol" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0053"> <span class="author">Elbrecht, V.</span>, &amp;  <span class="author">Leese, F.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">Validation and development of COI metabarcoding primers for freshwater macroinvertebrate bioassessment</span>. <i>Frontiers in Environmental Science</i>,  <span class="vol">5</span>,  <span class="pageFirst">11</span>. <a href="https://doi.org/10.3389/fenvs.2017.00011" class="linkBehavior">https://doi.org/10.3389/fenvs.2017.00011</a><div class="extra-links" id="e_1_2_14_57_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000458352900009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2017&amp;pages=11&amp;journal=Frontiers+in+Environmental+Science&amp;author=V.+Elbrecht&amp;author=F.+Leese&amp;title=Validation+and+development+of+COI+metabarcoding+primers+for+freshwater+macroinvertebrate+bioassessment" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0054"> <span class="author">Emilson, C. E.</span>,  <span class="author">Thompson, D. G.</span>,  <span class="author">Venier, L. A.</span>,  <span class="author">Porter, T. M.</span>,  <span class="author">Swystun, T.</span>,  <span class="author">Chartrand, D.</span>, …  <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">DNA
 metabarcoding and morphological macroinvertebrate metrics reveal the 
same changes in boreal watersheds across an environmental gradient</span>. <i>Scientific Reports</i>,  <span class="vol">7</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">12777</span>. <a href="https://doi.org/10.1038/s41598-017-13157-x" class="linkBehavior">https://doi.org/10.1038/s41598-017-13157-x</a><div class="extra-links" id="e_1_2_14_58_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fs41598-017-13157-x" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=28986575" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000412492400003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2017&amp;pages=12777&amp;journal=Scientific+Reports&amp;issue=1&amp;author=C.+E.+Emilson&amp;author=D.+G.+Thompson&amp;author=L.+A.+Venier&amp;author=T.+M.+Porter&amp;author=T.+Swystun&amp;author=D.+Chartrand&amp;author=M.+Hajibabaei&amp;title=DNA+metabarcoding+and+morphological+macroinvertebrate+metrics+reveal+the+same+changes+in+boreal+watersheds+across+an+environmental+gradient" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0055"> <span class="author">Fahner, N. A.</span>,  <span class="author">Shokralla, S.</span>,  <span class="author">Baird, D. J.</span>, &amp;  <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">Large‐scale
 monitoring of plants through environmental DNA metabarcoding of soil: 
recovery, resolution, and annotation of four DNA markers</span>. <i>PLoS ONE</i>,  <span class="vol">11</span>( <span class="citedIssue">6</span>),  <span class="pageFirst">e0157505</span>. <a href="https://doi.org/10.1371/journal.pone.0157505" class="linkBehavior">https://doi.org/10.1371/journal.pone.0157505</a><div class="extra-links" id="e_1_2_14_59_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0157505" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=27310720" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000378029800088" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2016&amp;pages=e0157505&amp;journal=PLoS+ONE&amp;issue=6&amp;author=N.+A.+Fahner&amp;author=S.+Shokralla&amp;author=D.+J.+Baird&amp;author=M.+Hajibabaei&amp;title=Large%E2%80%90scale+monitoring+of+plants+through+environmental+DNA+metabarcoding+of+soil%3A+recovery%2C+resolution%2C+and+annotation+of+four+DNA+markers" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0056"> <span class="author">Faith, D. P.</span> ( <span class="pubYear">1996</span>).  <span class="articleTitle">Conservation priorities and phylogenetic pattern</span>. <i>Conservation Biology</i>,  <span class="vol">10</span>,  <span class="pageFirst">1286</span>– <span class="pageLast">1289</span>. <a href="https://doi.org/10.1046/j.1523-1739.1996.10041286.x" class="linkBehavior">https://doi.org/10.1046/j.1523-1739.1996.10041286.x</a><div class="extra-links" id="e_1_2_14_60_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1523-1739.1996.10041286.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=A1996VC10300045" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=1996&amp;pages=1286-1289&amp;journal=Conservation+Biology&amp;author=D.+P.+Faith&amp;title=Conservation+priorities+and+phylogenetic+pattern" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0057"> <span class="author">Faith, D. P.</span> ( <span class="pubYear">2013</span>).  <span class="articleTitle">Biodiversity
 and evolutionary history: Useful extensions of the PD phylogenetic 
diversity assessment framework: Biodiversity and evolutionary history</span>. <i>Annals of the New York Academy of Sciences</i>,  <span class="vol">1289</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">69</span>– <span class="pageLast">89</span>. <a href="https://doi.org/10.1111/nyas.12186" class="linkBehavior">https://doi.org/10.1111/nyas.12186</a><div class="extra-links" id="e_1_2_14_61_1"><span>
                           <a href="https://nyaspubs.onlinelibrary.wiley.com/doi/10.1111/nyas.12186" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=23773093" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000324894900005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1289&amp;publication_year=2013&amp;pages=69-89&amp;journal=Annals+of+the+New+York+Academy+of+Sciences&amp;issue=1&amp;author=D.+P.+Faith&amp;title=Biodiversity+and+evolutionary+history%3A+Useful+extensions+of+the+PD+phylogenetic+diversity+assessment+framework%3A+Biodiversity+and+evolutionary+history" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0058"> <span class="author">Faith, D. P.</span>,  <span class="author">Lozupone, C. A.</span>,  <span class="author">Nipperess, D.</span>, &amp;  <span class="author">Knight, R.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">The
 cladistic basis for the phylogenetic diversity (PD) measure links 
evolutionary features to environmental gradients and supports broad 
applications of microbial ecology's “phylogenetic beta diversity” 
framework</span>. <i>International Journal of Molecular Sciences</i>,  <span class="vol">10</span>( <span class="citedIssue">11</span>),  <span class="pageFirst">4723</span>– <span class="pageLast">4741</span>. <a href="https://doi.org/10.3390/ijms10114723" class="linkBehavior">https://doi.org/10.3390/ijms10114723</a><div class="extra-links" id="e_1_2_14_62_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.3390%2Fijms10114723" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20087461" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000272191200006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2009&amp;pages=4723-4741&amp;journal=International+Journal+of+Molecular+Sciences&amp;issue=11&amp;author=D.+P.+Faith&amp;author=C.+A.+Lozupone&amp;author=D.+Nipperess&amp;author=R.+Knight&amp;title=The+cladistic+basis+for+the+phylogenetic+diversity+%28PD%29+measure+links+evolutionary+features+to+environmental+gradients+and+supports+broad+applications+of+microbial+ecology%27s+%E2%80%9Cphylogenetic+beta+diversity%E2%80%9D+framework" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0059"> <span class="author">Falster, D.</span>,  <span class="author">FitzJohn, R. G.</span>,  <span class="author">Pennell, M. W.</span>, &amp;  <span class="author">Cornwell, W. K.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">Versioned data: Why it is needed and how it can be achieved (easily and cheaply)</span>. <i>PeerJ PrePrints</i>,  <span class="vol">5</span>,  <span class="pageFirst">e3401v1</span>. <a href="https://doi.org/10.7287/peerj.preprints.3401v1" class="linkBehavior">https://doi.org/10.7287/peerj.preprints.3401v1</a><div class="extra-links" id="e_1_2_14_63_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2017&amp;pages=e3401v1&amp;journal=PeerJ+PrePrints&amp;author=D.+Falster&amp;author=R.+G.+FitzJohn&amp;author=M.+W.+Pennell&amp;author=W.+K.+Cornwell&amp;title=Versioned+data%3A+Why+it+is+needed+and+how+it+can+be+achieved+%28easily+and+cheaply%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0061"> <span class="author">Gardner, S. N.</span>,  <span class="author">Jaing, C. J.</span>,  <span class="author">McLoughlin, K. S.</span>, &amp;  <span class="author">Slezak, T. R.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">A microbial detection array (MDA) for viral and bacterial detection</span>. <i>BMC Genomics</i>,  <span class="vol">11</span>,  <span class="pageFirst">668</span>. <a href="https://doi.org/10.1186/1471-2164-11-668" class="linkBehavior">https://doi.org/10.1186/1471-2164-11-668</a><div class="extra-links" id="e_1_2_14_64_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1471-2164-11-668" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFWksb7M" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21108826" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000285385800001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2010&amp;pages=668&amp;journal=BMC+Genomics&amp;author=S.+N.+Gardner&amp;author=C.+J.+Jaing&amp;author=K.+S.+McLoughlin&amp;author=T.+R.+Slezak&amp;title=A+microbial+detection+array+%28MDA%29+for+viral+and+bacterial+detection" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0062"> <span class="author">Gibson, J. F.</span>,  <span class="author">Shokralla, S.</span>,  <span class="author">Curry, C.</span>,  <span class="author">Baird, D. J.</span>,  <span class="author">Monk, W. A.</span>,  <span class="author">King, I.</span>, &amp;  <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">Large‐scale biomonitoring of remote and threatened ecosystems via high‐throughput sequencing</span>. <i>PLoS ONE</i>,  <span class="vol">10</span>( <span class="citedIssue">10</span>),  <span class="pageFirst">e0138432</span>. <a href="https://doi.org/10.1371/journal.pone.0138432" class="linkBehavior">https://doi.org/10.1371/journal.pone.0138432</a><div class="extra-links" id="e_1_2_14_65_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0138432" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=26488407" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000363248400007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2015&amp;pages=e0138432&amp;journal=PLoS+ONE&amp;issue=10&amp;author=J.+F.+Gibson&amp;author=S.+Shokralla&amp;author=C.+Curry&amp;author=D.+J.+Baird&amp;author=W.+A.+Monk&amp;author=I.+King&amp;author=M.+Hajibabaei&amp;title=Large%E2%80%90scale+biomonitoring+of+remote+and+threatened+ecosystems+via+high%E2%80%90throughput+sequencing" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0063"> <span class="groupName">GIGA Community of Scientists</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">The global invertebrate genomics alliance (GIGA): Developing community resources to study diverse invertebrate genomes</span>. <i>Journal of Heredity</i>,  <span class="vol">105</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">1</span>– <span class="pageLast">18</span>. <a href="https://doi.org/10.1093/jhered/est084" class="linkBehavior">https://doi.org/10.1093/jhered/est084</a><div class="extra-links" id="e_1_2_14_66_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fjhered%2Fest084" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=24336862" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000328427800001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2014&amp;pages=1-18&amp;journal=Journal+of+Heredity&amp;issue=1&amp;title=The+global+invertebrate+genomics+alliance+%28GIGA%29%3A+Developing+community+resources+to+study+diverse+invertebrate+genomes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0064"> <span class="author">Gihring, T. M.</span>,  <span class="author">Green, S. J.</span>, &amp;  <span class="author">Schadt, C. W.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Massively
 parallel rRNA gene sequencing exacerbates the potential for biased 
community diversity comparisons due to variable library sizes: 
Pyrosequencing exacerbates sample size bias</span>. <i>Environmental Microbiology</i>,  <span class="vol">14</span>( <span class="citedIssue">2</span>),  <span class="pageFirst">285</span>– <span class="pageLast">290</span>. <a href="https://doi.org/10.1111/j.1462-2920.2011.02550.x" class="linkBehavior">https://doi.org/10.1111/j.1462-2920.2011.02550.x</a><div class="extra-links" id="e_1_2_14_67_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1462-2920.2011.02550.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XksVanu7g%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21923700" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000302539300001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2012&amp;pages=285-290&amp;journal=Environmental+Microbiology&amp;issue=2&amp;author=T.+M.+Gihring&amp;author=S.+J.+Green&amp;author=C.+W.+Schadt&amp;title=Massively+parallel+rRNA+gene+sequencing+exacerbates+the+potential+for+biased+community+diversity+comparisons+due+to+variable+library+sizes%3A+Pyrosequencing+exacerbates+sample+size+bias" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0065"> <span class="author">Gilbert, J. A.</span>,  <span class="author">Jansson, J. K.</span>, &amp;  <span class="author">Knight, R.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">The Earth Microbiome project: Successes and aspirations</span>. <i>BMC Biology</i>,  <span class="vol">12</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">69</span>. <a href="https://doi.org/10.1186/s12915-014-0069-1" class="linkBehavior">https://doi.org/10.1186/s12915-014-0069-1</a><div class="extra-links" id="e_1_2_14_68_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2Fs12915-014-0069-1" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=10110887" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000340919300001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2014&amp;pages=69&amp;journal=BMC+Biology&amp;issue=1&amp;author=J.+A.+Gilbert&amp;author=J.+K.+Jansson&amp;author=R.+Knight&amp;title=The+Earth+Microbiome+project%3A+Successes+and+aspirations" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0066"> <span class="author">Gloor, G. B.</span>,  <span class="author">Macklaim, J. M.</span>,  <span class="author">Pawlowsky‐Glahn, V.</span>, &amp;  <span class="author">Egozcue, J. J.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">Microbiome datasets are compositional: and this is not optional</span>. <i>Frontiers in Microbiology</i>,  <span class="vol">8</span>,  <span class="pageFirst">2224</span>. <a href="https://doi.org/10.3389/fmicb.2017.02224" class="linkBehavior">https://doi.org/10.3389/fmicb.2017.02224</a><div class="extra-links" id="e_1_2_14_69_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.3389%2Ffmicb.2017.02224" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=29187837" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000415284300001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2017&amp;pages=2224&amp;journal=Frontiers+in+Microbiology&amp;author=G.+B.+Gloor&amp;author=J.+M.+Macklaim&amp;author=V.+Pawlowsky%E2%80%90Glahn&amp;author=J.+J.+Egozcue&amp;title=Microbiome+datasets+are+compositional%3A+and+this+is+not+optional" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0067"> <span class="author">Goecks, J.</span>,  <span class="author">Nekrutenko, A.</span>, &amp;  <span class="author">Taylor, J.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Galaxy:
 A comprehensive approach for supporting accessible, reproducible, and 
transparent computational research in the life sciences</span>. <i>Genome Biology</i>,  <span class="vol">11</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">R86</span>. <a href="https://doi.org/10.1186/gb-2010-11-8-r86" class="linkBehavior">https://doi.org/10.1186/gb-2010-11-8-r86</a><div class="extra-links" id="e_1_2_14_70_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2Fgb-2010-11-8-r86" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20738864" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000283777600007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2010&amp;pages=R86&amp;journal=Genome+Biology&amp;issue=8&amp;author=J.+Goecks&amp;author=A.+Nekrutenko&amp;author=J.+Taylor&amp;title=Galaxy%3A+A+comprehensive+approach+for+supporting+accessible%2C+reproducible%2C+and+transparent+computational+research+in+the+life+sciences" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0068"> <span class="author">Gohl, D. M.</span>,  <span class="author">Vangay, P.</span>,  <span class="author">Garbe, J.</span>,  <span class="author">MacLean, A.</span>,  <span class="author">Hauge, A.</span>,  <span class="author">Becker, A.</span>, …  <span class="author">Beckman, K. B.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">Systematic improvement of amplicon marker gene methods for increased accuracy in microbiome studies</span>. <i>Nature Biotechnology</i>,  <span class="vol">34</span>( <span class="citedIssue">9</span>),  <span class="pageFirst">942</span>– <span class="pageLast">949</span>. <a href="https://doi.org/10.1038/nbt.3601" class="linkBehavior">https://doi.org/10.1038/nbt.3601</a><div class="extra-links" id="e_1_2_14_71_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnbt.3601" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KmurbF" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=27454739" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000383348500019" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2016&amp;pages=942-949&amp;journal=Nature+Biotechnology&amp;issue=9&amp;author=D.+M.+Gohl&amp;author=P.+Vangay&amp;author=J.+Garbe&amp;author=A.+MacLean&amp;author=A.+Hauge&amp;author=A.+Becker&amp;author=K.+B.+Beckman&amp;title=Systematic+improvement+of+amplicon+marker+gene+methods+for+increased+accuracy+in+microbiome+studies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0069"> <span class="groupName">GRDI‐EcoBiomics</span> ( <span class="pubYear">2016</span>).  <span class="otherTitle">Metagenomics
 Based Ecosystem Biomonitoring (GRDI‐EcoBiomics) project, Government of 
Canada, Genomics R&amp;D Initiative, Year‐End Performance Report for 
Shared Priority Projects (2016‐2017)</span>.
                     <div class="extra-links" id="e_1_2_14_72_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=+GRDI%E2%80%90EcoBiomics+%28+2016%29.++Metagenomics+Based+Ecosystem+Biomonitoring+%28GRDI%E2%80%90EcoBiomics%29+project%2C+Government+of+Canada%2C+Genomics+R%26D+Initiative%2C+Year%E2%80%90End+Performance+Report+for+Shared+Priority+Projects+%282016%E2%80%902017%29." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0070"> <span class="author">Grigoriev, I. V.</span>,  <span class="author">Nikitin, R.</span>,  <span class="author">Haridas, S.</span>,  <span class="author">Kuo, A.</span>,  <span class="author">Ohm, R.</span>,  <span class="author">Otillar, R.</span>, …  <span class="author">Shabalov, I.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">MycoCosm portal: Gearing up for 1000 fungal genomes</span>. <i>Nucleic Acids Research</i>,  <span class="vol">42</span>( <span class="citedIssue">D1</span>),  <span class="pageFirst">D699</span>– <span class="pageLast">D704</span>. <a href="https://doi.org/10.1093/nar/gkt1183" class="linkBehavior">https://doi.org/10.1093/nar/gkt1183</a><div class="extra-links" id="e_1_2_14_73_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fnar%2Fgkt1183" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslek" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=24297253" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000331139800103" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=42&amp;publication_year=2014&amp;pages=D699-D704&amp;journal=Nucleic+Acids+Research&amp;issue=D1&amp;author=I.+V.+Grigoriev&amp;author=R.+Nikitin&amp;author=S.+Haridas&amp;author=A.+Kuo&amp;author=R.+Ohm&amp;author=R.+Otillar&amp;author=I.+Shabalov&amp;title=MycoCosm+portal%3A+Gearing+up+for+1000+fungal+genomes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0071"> <span class="author">Haas, B. J.</span>,  <span class="author">Gevers, D.</span>,  <span class="author">Earl, A. M.</span>,  <span class="author">Feldgarden, M.</span>,  <span class="author">Ward, D. V.</span>,  <span class="author">Giannoukos, G.</span>, …  <span class="author">Birren, B. W.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Chimeric 16S rRNA sequence formation and detection in Sanger and 454‐pyrosequenced PCR amplicons</span>. <i>Genome Research</i>,  <span class="vol">21</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">494</span>– <span class="pageLast">504</span>. <a href="https://doi.org/10.1101/gr.112730.110" class="linkBehavior">https://doi.org/10.1101/gr.112730.110</a><div class="extra-links" id="e_1_2_14_74_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1101%2Fgr.112730.110" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjt1aisbc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21212162" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000287841100015" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2011&amp;pages=494-504&amp;journal=Genome+Research&amp;issue=3&amp;author=B.+J.+Haas&amp;author=D.+Gevers&amp;author=A.+M.+Earl&amp;author=M.+Feldgarden&amp;author=D.+V.+Ward&amp;author=G.+Giannoukos&amp;author=B.+W.+Birren&amp;title=Chimeric+16S+rRNA+sequence+formation+and+detection+in+Sanger+and+454%E2%80%90pyrosequenced+PCR+amplicons" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0072"> <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">The golden age of DNA metasystematics</span>. <i>Trends in Genetics</i>,  <span class="vol">28</span>( <span class="citedIssue">11</span>),  <span class="pageFirst">535</span>– <span class="pageLast">537</span>. <a href="https://doi.org/10.1016/j.tig.2012.08.001" class="linkBehavior">https://doi.org/10.1016/j.tig.2012.08.001</a><div class="extra-links" id="e_1_2_14_75_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2Fj.tig.2012.08.001" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1OjsbnF" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22951138" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000310477000001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=2012&amp;pages=535-537&amp;journal=Trends+in+Genetics&amp;issue=11&amp;author=M.+Hajibabaei&amp;title=The+golden+age+of+DNA+metasystematics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0073"> <span class="author">Hajibabaei, M.</span>,  <span class="author">Shokralla, S.</span>,  <span class="author">Zhou, X.</span>,  <span class="author">Singer, G. A. C.</span>, &amp;  <span class="author">Baird, D. J.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Environmental barcoding: a next‐generation sequencing approach for biomonitoring applications using river benthos</span>. <i>PLoS ONE</i>,  <span class="vol">6</span>( <span class="citedIssue">4</span>),  <span class="pageFirst">e17497</span>. <a href="https://doi.org/10.1371/journal.pone.0017497" class="linkBehavior">https://doi.org/10.1371/journal.pone.0017497</a><div class="extra-links" id="e_1_2_14_76_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0017497" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltV2rt7c%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21533287" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000289458800003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2011&amp;pages=e17497&amp;journal=PLoS+ONE&amp;issue=4&amp;author=M.+Hajibabaei&amp;author=S.+Shokralla&amp;author=X.+Zhou&amp;author=G.+A.+C.+Singer&amp;author=D.+J.+Baird&amp;title=Environmental+barcoding%3A+a+next%E2%80%90generation+sequencing+approach+for+biomonitoring+applications+using+river+benthos" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0074"> <span class="author">Hajibabaei, M.</span>,  <span class="author">Spall, J. L.</span>,  <span class="author">Shokralla, S.</span>, &amp;  <span class="author">Konynenburg, S.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Assessing
 biodiversity of a freshwater benthic macroinvertebrate community 
through non‐destructive environmental barcoding of DNA from preservative
 ethanol</span>. <i>BMC Ecology</i>,  <span class="vol">12</span>,  <span class="pageFirst">28</span>. <a href="https://doi.org/10.1186/1472-6785-12-28" class="linkBehavior">https://doi.org/10.1186/1472-6785-12-28</a><div class="extra-links" id="e_1_2_14_77_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1472-6785-12-28" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1yrsb0%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=23259585" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000313504900001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2012&amp;pages=28&amp;journal=BMC+Ecology&amp;author=M.+Hajibabaei&amp;author=J.+L.+Spall&amp;author=S.+Shokralla&amp;author=S.+Konynenburg&amp;title=Assessing+biodiversity+of+a+freshwater+benthic+macroinvertebrate+community+through+non%E2%80%90destructive+environmental+barcoding+of+DNA+from+preservative+ethanol" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0075"> <span class="author">Hamady, M.</span>,  <span class="author">Lozupone, C.</span>, &amp;  <span class="author">Knight, R.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Fast
 UniFrac: Facilitating high‐throughput phylogenetic analyses of 
microbial communities including analysis of pyrosequencing and PhyloChip
 data</span>. <i>ISME Journal</i>,  <span class="vol">4</span>,  <span class="pageFirst">17</span>– <span class="pageLast">27</span>. <a href="https://doi.org/10.1038/ismej.2009.97" class="linkBehavior">https://doi.org/10.1038/ismej.2009.97</a><div class="extra-links" id="e_1_2_14_78_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fismej.2009.97" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhs1SqsL%252FF" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19710709" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000273350200002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2010&amp;pages=17-27&amp;journal=ISME+Journal&amp;author=M.+Hamady&amp;author=C.+Lozupone&amp;author=R.+Knight&amp;title=Fast+UniFrac%3A+Facilitating+high%E2%80%90throughput+phylogenetic+analyses+of+microbial+communities+including+analysis+of+pyrosequencing+and+PhyloChip+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0076"> <span class="author">Handelsman, J.</span> ( <span class="pubYear">2004</span>).  <span class="articleTitle">Metagenomics: Application of genomics to uncultured microorganisms</span>. <i>Microbiology and Molecular Biology Reviews</i>,  <span class="vol">68</span>( <span class="citedIssue">4</span>),  <span class="pageFirst">669</span>– <span class="pageLast">685</span>. <a href="https://doi.org/10.1128/MMBR.68.4.669-685.2004" class="linkBehavior">https://doi.org/10.1128/MMBR.68.4.669-685.2004</a><div class="extra-links" id="e_1_2_14_79_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1128%2FMMBR.68.4.669-685.2004" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvFSgsQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=15590779" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000225854100005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=68&amp;publication_year=2004&amp;pages=669-685&amp;journal=Microbiology+and+Molecular+Biology+Reviews&amp;issue=4&amp;author=J.+Handelsman&amp;title=Metagenomics%3A+Application+of+genomics+to+uncultured+microorganisms" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0077"> <span class="author">Hawksworth, D. L.</span> ( <span class="pubYear">1991</span>).  <span class="articleTitle">The fungal dimension of biodiversity: Magnitude, significance, and conservation</span>. <i>Mycological Research</i>,  <span class="vol">95</span>( <span class="citedIssue">6</span>),  <span class="pageFirst">641</span>– <span class="pageLast">655</span>. <a href="https://doi.org/10.1016/S0953-7562%2809%2980810-1" class="linkBehavior">https://doi.org/10.1016/S0953-7562(09)80810-1</a><div class="extra-links" id="e_1_2_14_80_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2FS0953-7562%2809%2980810-1" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=A1991FV11100001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=95&amp;publication_year=1991&amp;pages=641-655&amp;journal=Mycological+Research&amp;issue=6&amp;author=D.+L.+Hawksworth&amp;title=The+fungal+dimension+of+biodiversity%3A+Magnitude%2C+significance%2C+and+conservation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0078"> <span class="author">He, Z.</span>,  <span class="author">Gentry, T. J.</span>,  <span class="author">Schadt, C. W.</span>,  <span class="author">Wu, L.</span>,  <span class="author">Liebich, J.</span>,  <span class="author">Chong, S. C.</span>, …  <span class="author">Zhou, J.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">GeoChip: A comprehensive microarray for investigating biogeochemical, ecological and environmental processes</span>. <i>ISME Journal</i>,  <span class="vol">1</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">67</span>– <span class="pageLast">77</span>. <a href="https://doi.org/10.1038/ismej.2007.2" class="linkBehavior">https://doi.org/10.1038/ismej.2007.2</a><div class="extra-links" id="e_1_2_14_81_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fismej.2007.2" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXksFSgu7Y%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18043615" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000249215800010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2007&amp;pages=67-77&amp;journal=ISME+Journal&amp;issue=1&amp;author=Z.+He&amp;author=T.+J.+Gentry&amp;author=C.+W.+Schadt&amp;author=L.+Wu&amp;author=J.+Liebich&amp;author=S.+C.+Chong&amp;author=J.+Zhou&amp;title=GeoChip%3A+A+comprehensive+microarray+for+investigating+biogeochemical%2C+ecological+and+environmental+processes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0079"> <span class="author">Hebert, P. D. N.</span>,  <span class="author">Cywinska, A.</span>,  <span class="author">Ball, S. L.</span>, &amp;  <span class="author">deWaard, J. R.</span> ( <span class="pubYear">2003</span>).  <span class="articleTitle">Biological identifications through DNA barcodes</span>. <i>Proceedings of the Royal Society B: Biological Sciences</i>,  <span class="vol">270</span>( <span class="citedIssue">1512</span>),  <span class="pageFirst">313</span>– <span class="pageLast">321</span>. <a href="https://doi.org/10.1098/rspb.2002.2218" class="linkBehavior">https://doi.org/10.1098/rspb.2002.2218</a><div class="extra-links" id="e_1_2_14_82_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1098%2Frspb.2002.2218" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktVWiu7g%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=12614582" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000181064200013" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=270&amp;publication_year=2003&amp;pages=313-321&amp;journal=Proceedings+of+the+Royal+Society+B%3A+Biological+Sciences&amp;issue=1512&amp;author=P.+D.+N.+Hebert&amp;author=A.+Cywinska&amp;author=S.+L.+Ball&amp;author=J.+R.+deWaard&amp;title=Biological+identifications+through+DNA+barcodes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0080"> <span class="author">Hibbett, D. S.</span>,  <span class="author">Ohman, A.</span>,  <span class="author">Glotzer, D.</span>,  <span class="author">Nuhn, M.</span>,  <span class="author">Kirk, P.</span>, &amp;  <span class="author">Nilsson, R. H.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Progress
 in molecular and morphological taxon discovery in Fungi and options for
 formal classification of environmental sequences</span>. <i>Fungal Biology Reviews</i>,  <span class="vol">25</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">38</span>– <span class="pageLast">47</span>. <a href="https://doi.org/10.1016/j.fbr.2011.01.001" class="linkBehavior">https://doi.org/10.1016/j.fbr.2011.01.001</a><div class="extra-links" id="e_1_2_14_83_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2Fj.fbr.2011.01.001" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=25&amp;publication_year=2011&amp;pages=38-47&amp;journal=Fungal+Biology+Reviews&amp;issue=1&amp;author=D.+S.+Hibbett&amp;author=A.+Ohman&amp;author=D.+Glotzer&amp;author=M.+Nuhn&amp;author=P.+Kirk&amp;author=R.+H.+Nilsson&amp;title=Progress+in+molecular+and+morphological+taxon+discovery+in+Fungi+and+options+for+formal+classification+of+environmental+sequences" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0081"> <span class="author">Hollingsworth, P. M.</span>,  <span class="author">Graham, S. W.</span>, &amp;  <span class="author">Little, D. P.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Choosing and using a plant DNA barcode</span>. <i>PLoS ONE</i>,  <span class="vol">6</span>( <span class="citedIssue">5</span>),  <span class="pageFirst">e19254</span>. <a href="https://doi.org/10.1371/journal.pone.0019254" class="linkBehavior">https://doi.org/10.1371/journal.pone.0019254</a><div class="extra-links" id="e_1_2_14_84_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0019254" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntVajt7Y%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21637336" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000291052200009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2011&amp;pages=e19254&amp;journal=PLoS+ONE&amp;issue=5&amp;author=P.+M.+Hollingsworth&amp;author=S.+W.+Graham&amp;author=D.+P.+Little&amp;title=Choosing+and+using+a+plant+DNA+barcode" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0082"> <span class="author">Hollingsworth, P. M.</span>,  <span class="author">Li, D.‐Z.</span>,  <span class="author">Bank, M.</span>, &amp;  <span class="author">Twyford, A. D.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">Telling plant species apart with DNA: From barcodes to genomes</span>. <i>Philosophical Transactions of the Royal Society B: Biological Sciences</i>,  <span class="vol">371</span>( <span class="citedIssue">1702</span>),  <span class="pageFirst">20150338</span>. <a href="https://doi.org/10.1098/rstb.2015.0338" class="linkBehavior">https://doi.org/10.1098/rstb.2015.0338</a><div class="extra-links" id="e_1_2_14_85_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1098%2Frstb.2015.0338" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=27481790" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000381779700007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=371&amp;publication_year=2016&amp;pages=20150338&amp;journal=Philosophical+Transactions+of+the+Royal+Society+B%3A+Biological+Sciences&amp;issue=1702&amp;author=P.+M.+Hollingsworth&amp;author=D.%E2%80%90Z.+Li&amp;author=M.+Bank&amp;author=A.+D.+Twyford&amp;title=Telling+plant+species+apart+with+DNA%3A+From+barcodes+to+genomes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0083"> <span class="author">Horton, T. R.</span>, &amp;  <span class="author">Bruns, T. D.</span> ( <span class="pubYear">2001</span>).  <span class="articleTitle">The molecular revolution in ectomycorrhizal ecology: Peeking into the black‐box</span>. <i>Molecular Ecology</i>,  <span class="vol">10</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">1855</span>– <span class="pageLast">1871</span>. <a href="https://doi.org/10.1046/j.0962-1083.2001.01333.x" class="linkBehavior">https://doi.org/10.1046/j.0962-1083.2001.01333.x</a><div class="extra-links" id="e_1_2_14_86_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.0962-1083.2001.01333.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmslyhtr8%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=11555231" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000170667500001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2001&amp;pages=1855-1871&amp;journal=Molecular+Ecology&amp;issue=8&amp;author=T.+R.+Horton&amp;author=T.+D.+Bruns&amp;title=The+molecular+revolution+in+ectomycorrhizal+ecology%3A+Peeking+into+the+black%E2%80%90box" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0084"> <span class="author">Huse, S. M.</span>,  <span class="author">Welch, D. M.</span>,  <span class="author">Morrison, H. G.</span>, &amp;  <span class="author">Sogin, M. L.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Ironing out the wrinkles in the rare biosphere through improved OTU clustering: Ironing out the wrinkles in the rare biosphere</span>. <i>Environmental Microbiology</i>,  <span class="vol">12</span>( <span class="citedIssue">7</span>),  <span class="pageFirst">1889</span>– <span class="pageLast">1898</span>. <a href="https://doi.org/10.1111/j.1462-2920.2010.02193.x" class="linkBehavior">https://doi.org/10.1111/j.1462-2920.2010.02193.x</a><div class="extra-links" id="e_1_2_14_87_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1462-2920.2010.02193.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVegsLnO" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20236171" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000280101200009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2010&amp;pages=1889-1898&amp;journal=Environmental+Microbiology&amp;issue=7&amp;author=S.+M.+Huse&amp;author=D.+M.+Welch&amp;author=H.+G.+Morrison&amp;author=M.+L.+Sogin&amp;title=Ironing+out+the+wrinkles+in+the+rare+biosphere+through+improved+OTU+clustering%3A+Ironing+out+the+wrinkles+in+the+rare+biosphere" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0085"> <span class="author">Huson, D. H.</span>,  <span class="author">Mitra, S.</span>,  <span class="author">Ruscheweyh, H.‐J.</span>,  <span class="author">Weber, N.</span>, &amp;  <span class="author">Schuster, S. C.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Integrative analysis of environmental sequences using MEGAN4</span>. <i>Genome Research</i>,  <span class="vol">21</span>( <span class="citedIssue">9</span>),  <span class="pageFirst">1552</span>– <span class="pageLast">1560</span>. <a href="https://doi.org/10.1101/gr.120618.111" class="linkBehavior">https://doi.org/10.1101/gr.120618.111</a><div class="extra-links" id="e_1_2_14_88_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1101%2Fgr.120618.111" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFGisLnN" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21690186" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000294477000015" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2011&amp;pages=1552-1560&amp;journal=Genome+Research&amp;issue=9&amp;author=D.+H.+Huson&amp;author=S.+Mitra&amp;author=H.%E2%80%90J.+Ruscheweyh&amp;author=N.+Weber&amp;author=S.+C.+Schuster&amp;title=Integrative+analysis+of+environmental+sequences+using+MEGAN4" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0086"> <span class="author">Ishii, K.</span>, &amp;  <span class="author">Fukui, M.</span> ( <span class="pubYear">2001</span>).  <span class="articleTitle">Optimization of annealing temperature to reduce bias caused by a primer mismatch in multitemplate PCR</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">67</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">3753</span>– <span class="pageLast">3755</span>. <a href="https://doi.org/10.1128/AEM.67.8.3753-3755.2001" class="linkBehavior">https://doi.org/10.1128/AEM.67.8.3753-3755.2001</a><div class="extra-links" id="e_1_2_14_89_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1128%2FAEM.67.8.3753-3755.2001" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlvFSksLw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=11472961" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000170297800061" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=67&amp;publication_year=2001&amp;pages=3753-3755&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=8&amp;author=K.+Ishii&amp;author=M.+Fukui&amp;title=Optimization+of+annealing+temperature+to+reduce+bias+caused+by+a+primer+mismatch+in+multitemplate+PCR" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0087"> <span class="author">Janzen, D. H.</span>,  <span class="author">Hajibabaei, M.</span>,  <span class="author">Burns, J. M.</span>,  <span class="author">Hallwachs, W.</span>,  <span class="author">Remigio, E.</span>, &amp;  <span class="author">Hebert, P. D. N.</span> ( <span class="pubYear">2005</span>).  <span class="articleTitle">Wedding biodiversity inventory of a large and complex Lepidoptera fauna with DNA barcoding</span>. <i>Philosophical Transactions of the Royal Society B: Biological Sciences</i>,  <span class="vol">360</span>( <span class="citedIssue">1462</span>),  <span class="pageFirst">1835</span>– <span class="pageLast">1845</span>. <a href="https://doi.org/10.1098/rstb.2005.1715" class="linkBehavior">https://doi.org/10.1098/rstb.2005.1715</a><div class="extra-links" id="e_1_2_14_90_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1098%2Frstb.2005.1715" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSjsrjI" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=16214742" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000232719300004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=360&amp;publication_year=2005&amp;pages=1835-1845&amp;journal=Philosophical+Transactions+of+the+Royal+Society+B%3A+Biological+Sciences&amp;issue=1462&amp;author=D.+H.+Janzen&amp;author=M.+Hajibabaei&amp;author=J.+M.+Burns&amp;author=W.+Hallwachs&amp;author=E.+Remigio&amp;author=P.+D.+N.+Hebert&amp;title=Wedding+biodiversity+inventory+of+a+large+and+complex+Lepidoptera+fauna+with+DNA+barcoding" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0088"> <span class="author">Ji, Y.</span>,  <span class="author">Ashton, L.</span>,  <span class="author">Pedley, S. M.</span>,  <span class="author">Edwards, D. P.</span>,  <span class="author">Tang, Y.</span>,  <span class="author">Nakamura, A.</span>, …  <span class="author">Yu, D. W.</span> ( <span class="pubYear">2013</span>).  <span class="articleTitle">Reliable, verifiable and efficient monitoring of biodiversity via metabarcoding</span>. <i>Ecology Letters</i>,  <span class="vol">16</span>( <span class="citedIssue">10</span>),  <span class="pageFirst">1245</span>– <span class="pageLast">1257</span>. <a href="https://doi.org/10.1111/ele.12162" class="linkBehavior">https://doi.org/10.1111/ele.12162</a><div class="extra-links" id="e_1_2_14_91_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/ele.12162" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ASTN%3A280%3ADC%252BC3sfmt1Gktg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=23910579" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000324313600003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=2013&amp;pages=1245-1257&amp;journal=Ecology+Letters&amp;issue=10&amp;author=Y.+Ji&amp;author=L.+Ashton&amp;author=S.+M.+Pedley&amp;author=D.+P.+Edwards&amp;author=Y.+Tang&amp;author=A.+Nakamura&amp;author=D.+W.+Yu&amp;title=Reliable%2C+verifiable+and+efficient+monitoring+of+biodiversity+via+metabarcoding" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0089"> <span class="groupName">i5K Consortium</span> ( <span class="pubYear">2013</span>).  <span class="articleTitle">The i5K initiative: advancing arthropod genomics for knowledge, human health, agriculture, and the environment</span>. <i>Journal of Heredity</i>,  <span class="vol">104</span>( <span class="citedIssue">5</span>),  <span class="pageFirst">595</span>– <span class="pageLast">600</span>. <a href="https://doi.org/10.1093/jhered/est050" class="linkBehavior">https://doi.org/10.1093/jhered/est050</a><div class="extra-links" id="e_1_2_14_92_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fjhered%2Fest050" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=23940263" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000323294400001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=104&amp;publication_year=2013&amp;pages=595-600&amp;journal=Journal+of+Heredity&amp;issue=5&amp;title=The+i5K+initiative%3A+advancing+arthropod+genomics+for+knowledge%2C+human+health%2C+agriculture%2C+and+the+environment" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0090"> <span class="author">Kalle, E.</span>,  <span class="author">Kubista, M.</span>, &amp;  <span class="author">Rensing, C.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Multi‐template polymerase chain reaction</span>. <i>Biomolecular Detection and Quantification</i>,  <span class="vol">2</span>,  <span class="pageFirst">11</span>– <span class="pageLast">29</span>. <a href="https://doi.org/10.1016/j.bdq.2014.11.002" class="linkBehavior">https://doi.org/10.1016/j.bdq.2014.11.002</a><div class="extra-links" id="e_1_2_14_93_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2Fj.bdq.2014.11.002" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=27896140" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2014&amp;pages=11-29&amp;journal=Biomolecular+Detection+and+Quantification&amp;author=E.+Kalle&amp;author=M.+Kubista&amp;author=C.+Rensing&amp;title=Multi%E2%80%90template+polymerase+chain+reaction" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0091"> <span class="author">Klappenbach, J. A.</span>,  <span class="author">Saxman, P. R.</span>,  <span class="author">Cole, J. R.</span>, &amp;  <span class="author">Schidt, T. M.</span> ( <span class="pubYear">2001</span>).  <span class="articleTitle">rrndb: The ribosomal RNA operon copy number database</span>. <i>Nucleic Acids Research</i>,  <span class="vol">29</span>,  <span class="pageFirst">181</span>– <span class="pageLast">184</span>. <a href="https://doi.org/10.1093/nar/29.1.181" class="linkBehavior">https://doi.org/10.1093/nar/29.1.181</a><div class="extra-links" id="e_1_2_14_94_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fnar%2F29.1.181" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtlWms7g%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=11125085" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000166360300049" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2001&amp;pages=181-184&amp;journal=Nucleic+Acids+Research&amp;author=J.+A.+Klappenbach&amp;author=P.+R.+Saxman&amp;author=J.+R.+Cole&amp;author=T.+M.+Schidt&amp;title=rrndb%3A+The+ribosomal+RNA+operon+copy+number+database" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0092"> <span class="author">Koepfli, K.‐P.</span>,  <span class="author">Paten, B.</span>,  <span class="groupName">the Genome 10K Community of Scientists</span>,  <span class="author">O'Brien, S. J.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">The genome 10K project: A way forward</span>. <i>Annual Review of Animal Biosciences</i>,  <span class="vol">3</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">57</span>– <span class="pageLast">111</span>. <a href="https://doi.org/10.1146/annurev-animal-090414-014900" class="linkBehavior">https://doi.org/10.1146/annurev-animal-090414-014900</a><div class="extra-links" id="e_1_2_14_95_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1146%2Fannurev-animal-090414-014900" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVWmtL4%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=25689317" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000350991900004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2015&amp;pages=57-111&amp;journal=Annual+Review+of+Animal+Biosciences&amp;issue=1&amp;author=K.%E2%80%90P.+Koepfli&amp;author=B.+Paten&amp;author=S.+J.+O%27Brien&amp;title=The+genome+10K+project%3A+A+way+forward" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0093"> <span class="author">Koski, L. B.</span>, &amp;  <span class="author">Golding, G. B.</span> ( <span class="pubYear">2001</span>).  <span class="articleTitle">The closest BLAST hit is often not the nearest neighbor</span>. <i>Journal of Molecular Evolution</i>,  <span class="vol">52</span>( <span class="citedIssue">6</span>),  <span class="pageFirst">540</span>– <span class="pageLast">542</span>. <a href="https://doi.org/10.1007/s002390010184" class="linkBehavior">https://doi.org/10.1007/s002390010184</a><div class="extra-links" id="e_1_2_14_96_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1007%2Fs002390010184" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksFOiurs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=11443357" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000168991500007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2001&amp;pages=540-542&amp;journal=Journal+of+Molecular+Evolution&amp;issue=6&amp;author=L.+B.+Koski&amp;author=G.+B.+Golding&amp;title=The+closest+BLAST+hit+is+often+not+the+nearest+neighbor" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0094"> <span class="author">Kurata, S.</span>,  <span class="author">Kanagawa, T.</span>,  <span class="author">Magariyama, Y.</span>,  <span class="author">Takatsu, K.</span>,  <span class="author">Yamada, K.</span>,  <span class="author">Yokomaku, T.</span>, &amp;  <span class="author">Kamagata, Y.</span> ( <span class="pubYear">2004</span>).  <span class="articleTitle">Reevaluation and reduction of a PCR bias caused by reannealing of templates</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">70</span>( <span class="citedIssue">12</span>),  <span class="pageFirst">7545</span>– <span class="pageLast">7549</span>. <a href="https://doi.org/10.1128/AEM.70.12.7545-7549.2004" class="linkBehavior">https://doi.org/10.1128/AEM.70.12.7545-7549.2004</a><div class="extra-links" id="e_1_2_14_97_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1128%2FAEM.70.12.7545-7549.2004" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFaitrfL" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=15574958" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000225719300073" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=70&amp;publication_year=2004&amp;pages=7545-7549&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=12&amp;author=S.+Kurata&amp;author=T.+Kanagawa&amp;author=Y.+Magariyama&amp;author=K.+Takatsu&amp;author=K.+Yamada&amp;author=T.+Yokomaku&amp;author=Y.+Kamagata&amp;title=Reevaluation+and+reduction+of+a+PCR+bias+caused+by+reannealing+of+templates" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0095"> <span class="author">Lan, Y.</span>,  <span class="author">Wang, Q.</span>,  <span class="author">Cole, J. R.</span>, &amp;  <span class="author">Rosen, G. L.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Using the RDP classifier to predict taxonomic novelty and reduce the search space for finding novel organisms</span>. <i>PLoS ONE</i>,  <span class="vol">7</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">e32491</span>. <a href="https://doi.org/10.1371/journal.pone.0032491" class="linkBehavior">https://doi.org/10.1371/journal.pone.0032491</a><div class="extra-links" id="e_1_2_14_98_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0032491" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XktVOlsbc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22403664" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000303017700061" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2012&amp;pages=e32491&amp;journal=PLoS+ONE&amp;issue=3&amp;author=Y.+Lan&amp;author=Q.+Wang&amp;author=J.+R.+Cole&amp;author=G.+L.+Rosen&amp;title=Using+the+RDP+classifier+to+predict+taxonomic+novelty+and+reduce+the+search+space+for+finding+novel+organisms" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0096"> <span class="author">Lander, E. S.</span>,  <span class="author">Linton, L. M.</span>,  <span class="author">Birren, B.</span>,  <span class="author">Nusbaum, C.</span>,  <span class="author">Zody, M. C.</span>,  <span class="author">Baldwin, J.</span>, …  <span class="author">Szustakowki, J.</span> ( <span class="pubYear">2001</span>).  <span class="articleTitle">Initial sequencing and analysis of the human genome</span>. <i>Nature</i>,  <span class="vol">409</span>( <span class="citedIssue">6822</span>),  <span class="pageFirst">860</span>– <span class="pageLast">921</span>. <a href="https://doi.org/10.1038/35057062" class="linkBehavior">https://doi.org/10.1038/35057062</a><div class="extra-links" id="e_1_2_14_99_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2F35057062" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhsFCjtLc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=11237011" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000084278100002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=409&amp;publication_year=2001&amp;pages=860-921&amp;journal=Nature&amp;issue=6822&amp;author=E.+S.+Lander&amp;author=L.+M.+Linton&amp;author=B.+Birren&amp;author=C.+Nusbaum&amp;author=M.+C.+Zody&amp;author=J.+Baldwin&amp;author=J.+Szustakowki&amp;title=Initial+sequencing+and+analysis+of+the+human+genome" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0097"> <span class="author">Langille, M. G. I.</span>,  <span class="author">Zaneveld, J.</span>,  <span class="author">Caporaso, J. G.</span>,  <span class="author">McDonald, D.</span>,  <span class="author">Knights, D.</span>,  <span class="author">Reyes, J. A.</span>, …  <span class="author">Huttenhower, C.</span> ( <span class="pubYear">2013</span>).  <span class="articleTitle">Predictive functional profiling of microbial communities using 16S rRNA marker gene sequences</span>. <i>Nature Biotechnology</i>,  <span class="vol">31</span>( <span class="citedIssue">9</span>),  <span class="pageFirst">814</span>– <span class="pageLast">821</span>. <a href="https://doi.org/10.1038/nbt.2676" class="linkBehavior">https://doi.org/10.1038/nbt.2676</a><div class="extra-links" id="e_1_2_14_100_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnbt.2676" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlWku7fJ" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=23975157" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000324306300021" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=31&amp;publication_year=2013&amp;pages=814-821&amp;journal=Nature+Biotechnology&amp;issue=9&amp;author=M.+G.+I.+Langille&amp;author=J.+Zaneveld&amp;author=J.+G.+Caporaso&amp;author=D.+McDonald&amp;author=D.+Knights&amp;author=J.+A.+Reyes&amp;author=C.+Huttenhower&amp;title=Predictive+functional+profiling+of+microbial+communities+using+16S+rRNA+marker+gene+sequences" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0098"> <span class="author">Li, J.‐Y.</span>,  <span class="author">Wang, J.</span>, &amp;  <span class="author">Zeigler, R. S.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">The 3,000 rice genomes project: New opportunities and challenges for future rice research</span>. <i>GigaScience</i>,  <span class="vol">3</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">8</span>. <a href="https://doi.org/10.1186/2047-217X-3-8" class="linkBehavior">https://doi.org/10.1186/2047-217X-3-8</a><div class="extra-links" id="e_1_2_14_101_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F2047-217X-3-8" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=24872878" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000365652900002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2014&amp;pages=8&amp;journal=GigaScience&amp;issue=1&amp;author=J.%E2%80%90Y.+Li&amp;author=J.+Wang&amp;author=R.+S.+Zeigler&amp;title=The+3%2C000+rice+genomes+project%3A+New+opportunities+and+challenges+for+future+rice+research" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0099"> <span class="author">Li, X.</span>,  <span class="author">Yang, Y.</span>,  <span class="author">Henry, R. J.</span>,  <span class="author">Rossetto, M.</span>,  <span class="author">Wang, Y.</span>, &amp;  <span class="author">Chen, S.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">Plant DNA barcoding: From gene to genome: Plant identification using DNA barcodes</span>. <i>Biological Reviews</i>,  <span class="vol">90</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">157</span>– <span class="pageLast">166</span>. <a href="https://doi.org/10.1111/brv.12104" class="linkBehavior">https://doi.org/10.1111/brv.12104</a><div class="extra-links" id="e_1_2_14_102_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/brv.12104" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ASTN%3A280%3ADC%252BC2crnsFygsA%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=24666563" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000348415500010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=90&amp;publication_year=2015&amp;pages=157-166&amp;journal=Biological+Reviews&amp;issue=1&amp;author=X.+Li&amp;author=Y.+Yang&amp;author=R.+J.+Henry&amp;author=M.+Rossetto&amp;author=Y.+Wang&amp;author=S.+Chen&amp;title=Plant+DNA+barcoding%3A+From+gene+to+genome%3A+Plant+identification+using+DNA+barcodes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0100"> <span class="author">Liu, B.</span>,  <span class="author">Gibbons, T.</span>,  <span class="author">Ghodsi, M.</span>, &amp;  <span class="author">Pop, M.</span> ( <span class="pubYear">2010</span>).  <span class="chapterTitle">MetaPhyler: Taxonomic profiling for metagenomic sequences</span>. In <i>Bioinformatics and Biomedicine (BIBM), 2010 IEEE International Conference on</i> (pp.  <span class="pageFirst">95</span>– <span class="pageLast">100</span>). <i>IEEE</i>. Retrieved from <a href="http://ieeexplore.ieee.org/abstract/document/5706544/" class="linkBehavior">http://ieeexplore.ieee.org/abstract/document/5706544/</a><div class="extra-links" id="e_1_2_14_103_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=+Liu%2C+B.%2C++Gibbons%2C+T.%2C++Ghodsi%2C+M.%2C+%26++Pop%2C+M.+%28+2010%29.++MetaPhyler%3A+Taxonomic+profiling+for+metagenomic+sequences.+In+Bioinformatics+and+Biomedicine+%28BIBM%29%2C+2010+IEEE+International+Conference+on+%28pp.++95%E2%80%93+100%29.+IEEE.+Retrieved+from+http%3A%2F%2Fieeexplore.ieee.org%2Fabstract%2Fdocument%2F5706544%2F" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0101"> <span class="author">Liu, L.</span>,  <span class="author">Li, Y.</span>,  <span class="author">Li, S.</span>,  <span class="author">Hu, N.</span>,  <span class="author">He, Y.</span>,  <span class="author">Pong, R.</span>, …  <span class="author">Law, M.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Comparison of next‐generation sequencing systems</span>. <i>Journal of Biomedicine and Biotechnology</i>,  <span class="vol">2012</span>,  <span class="pageFirst">1</span>– <span class="pageLast">11</span>. <a href="https://doi.org/10.1155/2012/251364" class="linkBehavior">https://doi.org/10.1155/2012/251364</a><div class="extra-links" id="e_1_2_14_104_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFajtLfJ" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21836813" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000295640200001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2012&amp;publication_year=2012&amp;pages=1-11&amp;journal=Journal+of+Biomedicine+and+Biotechnology&amp;author=L.+Liu&amp;author=Y.+Li&amp;author=S.+Li&amp;author=N.+Hu&amp;author=Y.+He&amp;author=R.+Pong&amp;author=M.+Law&amp;title=Comparison+of+next%E2%80%90generation+sequencing+systems" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0102"> <span class="author">Liu, K.‐L.</span>,  <span class="author">Porras‐Alfaro, A.</span>,  <span class="author">Kuske, C. R.</span>,  <span class="author">Eichorst, S. A.</span>, &amp;  <span class="author">Xie, G.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Accurate, rapid taxonomic classification of fungal large‐subunit rRNA genes</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">78</span>( <span class="citedIssue">5</span>),  <span class="pageFirst">1523</span>– <span class="pageLast">1533</span>. <a href="https://doi.org/10.1128/AEM.06826-11" class="linkBehavior">https://doi.org/10.1128/AEM.06826-11</a><div class="extra-links" id="e_1_2_14_105_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1128%2FAEM.06826-11" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtVSntLs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22194300" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000300537400024" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=78&amp;publication_year=2012&amp;pages=1523-1533&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=5&amp;author=K.%E2%80%90L.+Liu&amp;author=A.+Porras%E2%80%90Alfaro&amp;author=C.+R.+Kuske&amp;author=S.+A.+Eichorst&amp;author=G.+Xie&amp;title=Accurate%2C+rapid+taxonomic+classification+of+fungal+large%E2%80%90subunit+rRNA+genes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0103"> <span class="author">Liu, W.‐T.</span>, &amp;  <span class="author">Zhu, L.</span> ( <span class="pubYear">2005</span>).  <span class="articleTitle">Environmental microbiology‐on‐a‐chip and its future impacts</span>. <i>Trends in Biotechnology</i>,  <span class="vol">23</span>( <span class="citedIssue">4</span>),  <span class="pageFirst">174</span>– <span class="pageLast">179</span>. <a href="https://doi.org/10.1016/j.tibtech.2005.02.004" class="linkBehavior">https://doi.org/10.1016/j.tibtech.2005.02.004</a><div class="extra-links" id="e_1_2_14_106_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2Fj.tibtech.2005.02.004" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisVWku78%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=15780708" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000228286000003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2005&amp;pages=174-179&amp;journal=Trends+in+Biotechnology&amp;issue=4&amp;author=W.%E2%80%90T.+Liu&amp;author=L.+Zhu&amp;title=Environmental+microbiology%E2%80%90on%E2%80%90a%E2%80%90chip+and+its+future+impacts" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0104"> <span class="author">Lou, M.</span>, &amp;  <span class="author">Golding, G. B.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Assigning sequences to species in the absence of large interspecific differences</span>. <i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">56</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">187</span>– <span class="pageLast">194</span>. <a href="https://doi.org/10.1016/j.ympev.2010.01.002" class="linkBehavior">https://doi.org/10.1016/j.ympev.2010.01.002</a><div class="extra-links" id="e_1_2_14_107_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2Fj.ympev.2010.01.002" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWgtrw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20067837" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000278589500016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=56&amp;publication_year=2010&amp;pages=187-194&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;issue=1&amp;author=M.+Lou&amp;author=G.+B.+Golding&amp;title=Assigning+sequences+to+species+in+the+absence+of+large+interspecific+differences" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0105"> <span class="author">Lozupone, C. A.</span>,  <span class="author">Hamady, M.</span>,  <span class="author">Kelley, S. T.</span>, &amp;  <span class="author">Knight, R.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">Quantitative and qualitative diversity measures lead to different insights into factors that structure microbial communities</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">73</span>( <span class="citedIssue">5</span>),  <span class="pageFirst">1576</span>– <span class="pageLast">1585</span>. <a href="https://doi.org/10.1128/AEM.01996-06" class="linkBehavior">https://doi.org/10.1128/AEM.01996-06</a><div class="extra-links" id="e_1_2_14_108_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1128%2FAEM.01996-06" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVSktr0%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=17220268" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000245068900023" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=73&amp;publication_year=2007&amp;pages=1576-1585&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=5&amp;author=C.+A.+Lozupone&amp;author=M.+Hamady&amp;author=S.+T.+Kelley&amp;author=R.+Knight&amp;title=Quantitative+and+qualitative+diversity+measures+lead+to+different+insights+into+factors+that+structure+microbial+communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-4000"> <span class="author">Mahé, F.</span>,  <span class="author">Rognes, T.</span>,  <span class="author">Quince, C.</span>,  <span class="author">Vargas, C.</span>, &amp;  <span class="author">Dunthorn, M.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Swarm: Robust and fast clustering method for amplicon‐based studies</span>. <i>PeerJ</i>,  <span class="vol">2</span>,  <span class="pageFirst">e593</span>. <a href="https://doi.org/10.7717/peerj.593" class="linkBehavior">https://doi.org/10.7717/peerj.593</a><div class="extra-links" id="e_1_2_14_109_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.7717%2Fpeerj.593" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=25276506" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000347621800005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2014&amp;pages=e593&amp;journal=PeerJ&amp;author=F.+Mah%C3%A9&amp;author=T.+Rognes&amp;author=C.+Quince&amp;author=C.+Vargas&amp;author=M.+Dunthorn&amp;title=Swarm%3A+Robust+and+fast+clustering+method+for+amplicon%E2%80%90based+studies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0106"> <span class="author">Mamanova, L.</span>,  <span class="author">Coffey, A. J.</span>,  <span class="author">Scott, C. E.</span>,  <span class="author">Kozarewa, I.</span>,  <span class="author">Turner, E. H.</span>,  <span class="author">Kumar, A.</span>, …  <span class="author">Turner, D. J.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Target‐enrichment strategies for next‐generation sequencing</span>. <i>Nature Methods</i>,  <span class="vol">7</span>( <span class="citedIssue">2</span>),  <span class="pageFirst">111</span>– <span class="pageLast">118</span>. <a href="https://doi.org/10.1038/nmeth.1419" class="linkBehavior">https://doi.org/10.1038/nmeth.1419</a><div class="extra-links" id="e_1_2_14_110_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnmeth.1419" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Slurg%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20111037" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000274086200015" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2010&amp;pages=111-118&amp;journal=Nature+Methods&amp;issue=2&amp;author=L.+Mamanova&amp;author=A.+J.+Coffey&amp;author=C.+E.+Scott&amp;author=I.+Kozarewa&amp;author=E.+H.+Turner&amp;author=A.+Kumar&amp;author=D.+J.+Turner&amp;title=Target%E2%80%90enrichment+strategies+for+next%E2%80%90generation+sequencing" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0107"> <span class="author">Mandal, S.</span>,  <span class="author">Van Treuren, W.</span>,  <span class="author">White, R. A.</span>,  <span class="author">Eggesbø, M.</span>,  <span class="author">Knight, R.</span>, &amp;  <span class="author">Peddada, S. D.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">Analysis of composition of microbiomes: A novel method for studying microbial composition</span>. <i>Microbial Ecology in Health &amp; Disease</i>,  <span class="vol">26</span>,  <span class="pageFirst">27663</span>. <a href="https://doi.org/10.3402/mehd.v26.27663" class="linkBehavior">https://doi.org/10.3402/mehd.v26.27663</a><div class="extra-links" id="e_1_2_14_111_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.3402%2Fmehd.v26.27663" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=26028277" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2015&amp;pages=27663&amp;journal=Microbial+Ecology+in+Health+%26+Disease&amp;author=S.+Mandal&amp;author=W.+Van+Treuren&amp;author=R.+A.+White&amp;author=M.+Eggesb%C3%B8&amp;author=R.+Knight&amp;author=S.+D.+Peddada&amp;title=Analysis+of+composition+of+microbiomes%3A+A+novel+method+for+studying+microbial+composition" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0108"> <span class="author">Manichanh, C.</span>,  <span class="author">Chapple, C. E.</span>,  <span class="author">Frangeul, L.</span>,  <span class="author">Gloux, K.</span>,  <span class="author">Guigo, R.</span>, &amp;  <span class="author">Dore, J.</span> ( <span class="pubYear">2008</span>).  <span class="articleTitle">A comparison of random sequence reads versus 16S rDNA sequences for estimating the biodiversity of a metagenomic library</span>. <i>Nucleic Acids Research</i>,  <span class="vol">36</span>( <span class="citedIssue">16</span>),  <span class="pageFirst">5180</span>– <span class="pageLast">5188</span>. <a href="https://doi.org/10.1093/nar/gkn496" class="linkBehavior">https://doi.org/10.1093/nar/gkn496</a><div class="extra-links" id="e_1_2_14_112_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fnar%2Fgkn496" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFWhsLzK" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18682527" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000259205600012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2008&amp;pages=5180-5188&amp;journal=Nucleic+Acids+Research&amp;issue=16&amp;author=C.+Manichanh&amp;author=C.+E.+Chapple&amp;author=L.+Frangeul&amp;author=K.+Gloux&amp;author=R.+Guigo&amp;author=J.+Dore&amp;title=A+comparison+of+random+sequence+reads+versus+16S+rDNA+sequences+for+estimating+the+biodiversity+of+a+metagenomic+library" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0109"> <span class="author">Mason, O. U.</span>,  <span class="author">Hazen, T. C.</span>,  <span class="author">Borglin, S.</span>,  <span class="author">Chain, P. S. G.</span>,  <span class="author">Dubinsky, E. A.</span>,  <span class="author">Fortney, J. L.</span>, …  <span class="author">Jansson, J. K.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Metagenome, metatranscriptome and single‐cell sequencing reveal microbial response to Deepwater Horizon oil spill</span>. <i>ISME Journal</i>,  <span class="vol">6</span>,  <span class="pageFirst">1715</span>– <span class="pageLast">1727</span>. <a href="https://doi.org/10.1038/ismej.2012.59" class="linkBehavior">https://doi.org/10.1038/ismej.2012.59</a><div class="extra-links" id="e_1_2_14_113_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fismej.2012.59" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1GjtLbL" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22717885" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000308141900009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2012&amp;pages=1715-1727&amp;journal=ISME+Journal&amp;author=O.+U.+Mason&amp;author=T.+C.+Hazen&amp;author=S.+Borglin&amp;author=P.+S.+G.+Chain&amp;author=E.+A.+Dubinsky&amp;author=J.+L.+Fortney&amp;author=J.+K.+Jansson&amp;title=Metagenome%2C+metatranscriptome+and+single%E2%80%90cell+sequencing+reveal+microbial+response+to+Deepwater+Horizon+oil+spill" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0110"> <span class="author">Matsen, F. A.</span>,  <span class="author">Kodner, R. B.</span>, &amp;  <span class="author">Armbrust, E. V.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">pplacer: Linear time maximum‐likelihood and Bayesian phylogenetic placement of sequences onto a fixed reference tree</span>. <i>BMC Bioinformatics</i>,  <span class="vol">11</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">538</span>. <a href="https://doi.org/10.1186/1471-2105-11-538" class="linkBehavior">https://doi.org/10.1186/1471-2105-11-538</a><div class="extra-links" id="e_1_2_14_114_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1471-2105-11-538" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21034504" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000284533800001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2010&amp;pages=538&amp;journal=BMC+Bioinformatics&amp;issue=1&amp;author=F.+A.+Matsen&amp;author=R.+B.+Kodner&amp;author=E.+V.+Armbrust&amp;title=pplacer%3A+Linear+time+maximum%E2%80%90likelihood+and+Bayesian+phylogenetic+placement+of+sequences+onto+a+fixed+reference+tree" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0111"> <span class="author">McMurdie, P. J.</span>, &amp;  <span class="author">Holmes, S.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Waste not, want not: why rarefying microbiome data is inadmissible</span>. <i>PLoS Computational Biology</i>,  <span class="vol">10</span>( <span class="citedIssue">4</span>),  <span class="pageFirst">e1003531</span>. <a href="https://doi.org/10.1371/journal.pcbi.1003531" class="linkBehavior">https://doi.org/10.1371/journal.pcbi.1003531</a><div class="extra-links" id="e_1_2_14_115_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pcbi.1003531" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=24699258" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000336507500019" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2014&amp;pages=e1003531&amp;journal=PLoS+Computational+Biology&amp;issue=4&amp;author=P.+J.+McMurdie&amp;author=S.+Holmes&amp;title=Waste+not%2C+want+not%3A+why+rarefying+microbiome+data+is+inadmissible" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0112"> <span class="author">McPherson, H.</span>,  <span class="author">Van der Merwe, M.</span>,  <span class="author">Delaney, S. K.</span>,  <span class="author">Edwards, M. A.</span>,  <span class="author">Henry, R. J.</span>,  <span class="author">McIntosh, E.</span>, …  <span class="author">Rossetto, M.</span> ( <span class="pubYear">2013</span>).  <span class="articleTitle">Capturing
 chloroplast variation for molecular ecology studies: A simple next 
generation sequencing approach applied to a rainforest tree</span>. <i>BMC Ecology</i>,  <span class="vol">13</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">8</span>. <a href="https://doi.org/10.1186/1472-6785-13-8" class="linkBehavior">https://doi.org/10.1186/1472-6785-13-8</a><div class="extra-links" id="e_1_2_14_116_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1472-6785-13-8" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslSjsbw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=23497206" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000317102500001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=13&amp;publication_year=2013&amp;pages=8&amp;journal=BMC+Ecology&amp;issue=1&amp;author=H.+McPherson&amp;author=M.+Van+der+Merwe&amp;author=S.+K.+Delaney&amp;author=M.+A.+Edwards&amp;author=R.+J.+Henry&amp;author=E.+McIntosh&amp;author=M.+Rossetto&amp;title=Capturing+chloroplast+variation+for+molecular+ecology+studies%3A+A+simple+next+generation+sequencing+approach+applied+to+a+rainforest+tree" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0113"> <span class="author">Meier, R.</span>,  <span class="author">Wong, W.</span>,  <span class="author">Srivathsan, A.</span>, &amp;  <span class="author">Foo, M.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">$1 DNA barcodes for reconstructing complex phenomes and finding rare species in specimen‐rich samples</span>. <i>Cladistics</i>,  <span class="vol">32</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">100</span>– <span class="pageLast">110</span>. <a href="https://doi.org/10.1111/cla.12115" class="linkBehavior">https://doi.org/10.1111/cla.12115</a><div class="extra-links" id="e_1_2_14_117_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/cla.12115" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000367923600007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=32&amp;publication_year=2016&amp;pages=100-110&amp;journal=Cladistics&amp;issue=1&amp;author=R.+Meier&amp;author=W.+Wong&amp;author=A.+Srivathsan&amp;author=M.+Foo&amp;title=%241+DNA+barcodes+for+reconstructing+complex+phenomes+and+finding+rare+species+in+specimen%E2%80%90rich+samples" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0114"> <span class="author">Mertes, F.</span>,  <span class="author">ElSharawy, A.</span>,  <span class="author">Sauer, S.</span>,  <span class="author">Helvoort, J. M. L. M.</span>,  <span class="author">Zaag, P. J.</span>,  <span class="author">Franke, A.</span>, …  <span class="author">Brookes, A. J.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Targeted enrichment of genomic DNA regions for next‐generation sequencing</span>. <i>Briefings in Functional Genomics</i>,  <span class="vol">10</span>( <span class="citedIssue">6</span>),  <span class="pageFirst">374</span>– <span class="pageLast">386</span>. <a href="https://doi.org/10.1093/bfgp/elr033" class="linkBehavior">https://doi.org/10.1093/bfgp/elr033</a><div class="extra-links" id="e_1_2_14_118_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fbfgp%2Felr033" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1ynu73J" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22121152" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000298387100007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2011&amp;pages=374-386&amp;journal=Briefings+in+Functional+Genomics&amp;issue=6&amp;author=F.+Mertes&amp;author=A.+ElSharawy&amp;author=S.+Sauer&amp;author=J.+M.+L.+M.+Helvoort&amp;author=P.+J.+Zaag&amp;author=A.+Franke&amp;author=A.+J.+Brookes&amp;title=Targeted+enrichment+of+genomic+DNA+regions+for+next%E2%80%90generation+sequencing" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0115"> <span class="author">Metfies, K.</span>, &amp;  <span class="author">Medlin, L.</span> ( <span class="pubYear">2005</span>).  <span class="articleTitle">Ribosomal RNA probes and microarrays: Their potential use in assessing microbial biodiversity</span>. <i>Methods in Enzymology</i>,  <span class="vol">395</span>,  <span class="pageFirst">258</span>– <span class="pageLast">278</span>. <a href="https://doi.org/10.1016/S0076-6879%2805%2995016-7" class="linkBehavior">https://doi.org/10.1016/S0076-6879(05)95016-7</a><div class="extra-links" id="e_1_2_14_119_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2FS0076-6879%2805%2995016-7" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD28XotlGrtbs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=15865972" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000230001100016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=395&amp;publication_year=2005&amp;pages=258-278&amp;journal=Methods+in+Enzymology&amp;author=K.+Metfies&amp;author=L.+Medlin&amp;title=Ribosomal+RNA+probes+and+microarrays%3A+Their+potential+use+in+assessing+microbial+biodiversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0116"> <span class="author">Meyer, J. N.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">QPCR: A tool for analysis of mitochondrial and nuclear DNA damage in ecotoxicology</span>. <i>Ecotoxicology</i>,  <span class="vol">19</span>( <span class="citedIssue">4</span>),  <span class="pageFirst">804</span>– <span class="pageLast">811</span>. <a href="https://doi.org/10.1007/s10646-009-0457-4" class="linkBehavior">https://doi.org/10.1007/s10646-009-0457-4</a><div class="extra-links" id="e_1_2_14_120_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1007%2Fs10646-009-0457-4" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVeis7g%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20049526" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000275963200020" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2010&amp;pages=804-811&amp;journal=Ecotoxicology&amp;issue=4&amp;author=J.+N.+Meyer&amp;title=QPCR%3A+A+tool+for+analysis+of+mitochondrial+and+nuclear+DNA+damage+in+ecotoxicology" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0117"> <span class="author">Meyer, F.</span>,  <span class="author">Paarmann, D.</span>,  <span class="author">D'Souza, M.</span>,  <span class="author">Olson, R.</span>,  <span class="author">Glass, E.</span>,  <span class="author">Kubal, M.</span>, …  <span class="author">Edwards, R.</span> ( <span class="pubYear">2008</span>).  <span class="articleTitle">The metagenomics RAST server – a public resource for the automatic phylogenetic and functional analysis of metagenomes</span>. <i>BMC Bioinformatics</i>,  <span class="vol">9</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">386</span>. <a href="https://doi.org/10.1186/1471-2105-9-386" class="linkBehavior">https://doi.org/10.1186/1471-2105-9-386</a><div class="extra-links" id="e_1_2_14_121_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1471-2105-9-386" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlGjsrbM" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18803844" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000260079800001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2008&amp;pages=386&amp;journal=BMC+Bioinformatics&amp;issue=1&amp;author=F.+Meyer&amp;author=D.+Paarmann&amp;author=M.+D%27Souza&amp;author=R.+Olson&amp;author=E.+Glass&amp;author=M.+Kubal&amp;author=R.+Edwards&amp;title=The+metagenomics+RAST+server+%E2%80%93+a+public+resource+for+the+automatic+phylogenetic+and+functional+analysis+of+metagenomes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0118"> <span class="author">Milan, M.</span>,  <span class="author">Coppe, A.</span>,  <span class="author">Reinhardt, R.</span>,  <span class="author">Cancela, L. M.</span>,  <span class="author">Leite, R. B.</span>,  <span class="author">Saavedra, C.</span>, …  <span class="author">Bargelloni, L.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Transcriptome
 sequencing and microarray development for the Manila clam, Ruditapes 
philippinarum: Genomic tools for environmental monitoring</span>. <i>BMC Genomics</i>,  <span class="vol">12</span>,  <span class="pageFirst">234</span>. <a href="https://doi.org/10.1186/1471-2164-12-234" class="linkBehavior">https://doi.org/10.1186/1471-2164-12-234</a><div class="extra-links" id="e_1_2_14_122_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1471-2164-12-234" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlKlsLk%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21569398" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000291265800001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2011&amp;pages=234&amp;journal=BMC+Genomics&amp;author=M.+Milan&amp;author=A.+Coppe&amp;author=R.+Reinhardt&amp;author=L.+M.+Cancela&amp;author=R.+B.+Leite&amp;author=C.+Saavedra&amp;author=L.+Bargelloni&amp;title=Transcriptome+sequencing+and+microarray+development+for+the+Manila+clam%2C+Ruditapes+philippinarum%3A+Genomic+tools+for+environmental+monitoring" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0119"> <span class="author">Mukherjee, S.</span>,  <span class="author">Seshadri, R.</span>,  <span class="author">Varghese, N. J.</span>,  <span class="author">Eloe‐Fadrosh, E. A.</span>,  <span class="author">Meier‐Kolthoff, J. P.</span>,  <span class="author">Gorker, M.</span>,  <span class="author">Kyrpides, N. C.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">1,003 reference genomes of bacterial and archaeal isolates expand coverage of the tree of life</span>. <i>Nature Biotechnology</i>,  <span class="vol">35</span>,  <span class="pageFirst">676</span>– <span class="pageLast">683</span>. <a href="https://doi.org/10.1038/nbt.3886" class="linkBehavior">https://doi.org/10.1038/nbt.3886</a><div class="extra-links" id="e_1_2_14_123_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnbt.3886" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFCms7s%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=28604660" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000405310300021" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=2017&amp;pages=676-683&amp;journal=Nature+Biotechnology&amp;author=S.+Mukherjee&amp;author=R.+Seshadri&amp;author=N.+J.+Varghese&amp;author=E.+A.+Eloe%E2%80%90Fadrosh&amp;author=J.+P.+Meier%E2%80%90Kolthoff&amp;author=M.+Gorker&amp;author=N.+C.+Kyrpides&amp;title=1%2C003+reference+genomes+of+bacterial+and+archaeal+isolates+expand+coverage+of+the+tree+of+life" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0120"> <span class="author">Munch, K.</span>,  <span class="author">Boomsma, W.</span>,  <span class="author">Huelsenbeck, J.</span>,  <span class="author">Willerslev, E.</span>, &amp;  <span class="author">Nielsen, R.</span> ( <span class="pubYear">2008</span>).  <span class="articleTitle">Statistical assignment of DNA sequences using bayesian phylogenetics</span>. <i>Systematic Biology</i>,  <span class="vol">57</span>( <span class="citedIssue">5</span>),  <span class="pageFirst">750</span>– <span class="pageLast">757</span>. <a href="https://doi.org/10.1080/10635150802422316" class="linkBehavior">https://doi.org/10.1080/10635150802422316</a><div class="extra-links" id="e_1_2_14_124_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1080%2F10635150802422316" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18853361" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000259995600007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=57&amp;publication_year=2008&amp;pages=750-757&amp;journal=Systematic+Biology&amp;issue=5&amp;author=K.+Munch&amp;author=W.+Boomsma&amp;author=J.+Huelsenbeck&amp;author=E.+Willerslev&amp;author=R.+Nielsen&amp;title=Statistical+assignment+of+DNA+sequences+using+bayesian+phylogenetics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-5000"> <span class="author">Munch, K.</span>,  <span class="author">Boomsma, W.</span>,  <span class="author">Willerslev, E.</span>, &amp;  <span class="author">Nielsen, R.</span> ( <span class="pubYear">2008</span>).  <span class="articleTitle">Fast phylogenetic DNA barcoding</span>. <i>Philosophical Transactions of the Royal Society B: Biological Sciences</i>,  <span class="vol">363</span>( <span class="citedIssue">1512</span>),  <span class="pageFirst">3997</span>– <span class="pageLast">4002</span>. <a href="https://doi.org/10.1098/rstb.2008.0169" class="linkBehavior">https://doi.org/10.1098/rstb.2008.0169</a><div class="extra-links" id="e_1_2_14_125_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1098%2Frstb.2008.0169" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjs1CmtQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18852104" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000260886700012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=363&amp;publication_year=2008&amp;pages=3997-4002&amp;journal=Philosophical+Transactions+of+the+Royal+Society+B%3A+Biological+Sciences&amp;issue=1512&amp;author=K.+Munch&amp;author=W.+Boomsma&amp;author=E.+Willerslev&amp;author=R.+Nielsen&amp;title=Fast+phylogenetic+DNA+barcoding" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0121"> <span class="groupName">National Center for Biotechnology Information (NCBI)</span> ( <span class="pubYear">1988</span>).  <span class="otherTitle">Bethesda (MD): National Library of Medicine (US), National Center for Biotechnology Information</span>. Retrieved from <a href="https://www.ncbi.nlm.nih.gov/" class="linkBehavior">https://www.ncbi.nlm.nih.gov/</a><div class="extra-links" id="e_1_2_14_126_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=+National+Center+for+Biotechnology+Information+%28NCBI%29+%28+1988%29.++Bethesda+%28MD%29%3A+National+Library+of+Medicine+%28US%29%2C+National+Center+for+Biotechnology+Information.+Retrieved+from+https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2F" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0122"> <span class="author">Nguyen, N. H.</span>,  <span class="author">Song, Z.</span>,  <span class="author">Bates, S. T.</span>,  <span class="author">Branco, S.</span>,  <span class="author">Tedersoo, L.</span>,  <span class="author">Menke, J.</span>, …  <span class="author">Kennedy, P. G.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">FUNGuild: An open annotation tool for parsing fungal community datasets by ecological guild</span>. <i>Fungal Ecology</i>,  <span class="vol">20</span>,  <span class="pageFirst">241</span>– <span class="pageLast">248</span>. <a href="https://doi.org/10.1016/j.funeco.2015.06.006" class="linkBehavior">https://doi.org/10.1016/j.funeco.2015.06.006</a><div class="extra-links" id="e_1_2_14_127_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2Fj.funeco.2015.06.006" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000373539100029" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2016&amp;pages=241-248&amp;journal=Fungal+Ecology&amp;author=N.+H.+Nguyen&amp;author=Z.+Song&amp;author=S.+T.+Bates&amp;author=S.+Branco&amp;author=L.+Tedersoo&amp;author=J.+Menke&amp;author=P.+G.+Kennedy&amp;title=FUNGuild%3A+An+open+annotation+tool+for+parsing+fungal+community+datasets+by+ecological+guild" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-6000"> <span class="author">Nilsson, R. H.</span>,  <span class="author">Kristiansson, E.</span>,  <span class="author">Ryberg, M.</span>, &amp;  <span class="author">Larsson, K.‐H.</span> ( <span class="pubYear">2005</span>).  <span class="articleTitle">Approaching the taxonomic affiliation of unidentified sequences in public databases–an example from the mycorrhizal fungi</span>. <i>BMC Bioinformatics</i>,  <span class="vol">6</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">178</span>.
                     <div class="extra-links" id="e_1_2_14_128_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1471-2105-6-178" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=16022740" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000231252000001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2005&amp;pages=178&amp;journal=BMC+Bioinformatics&amp;issue=1&amp;author=R.+H.+Nilsson&amp;author=E.+Kristiansson&amp;author=M.+Ryberg&amp;author=K.%E2%80%90H.+Larsson&amp;title=Approaching+the+taxonomic+affiliation+of+unidentified+sequences+in+public+databases%E2%80%93an+example+from+the+mycorrhizal+fungi" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-7000"> <span class="author">Nilsson, R. H.</span>,  <span class="author">Ryberg, M.</span>,  <span class="author">Kristiansson, E.</span>,  <span class="author">Abarenkov, K.</span>,  <span class="author">Larsson, K.‐H.</span>, &amp;  <span class="author">Kõljalg, U.</span> ( <span class="pubYear">2006</span>).  <span class="articleTitle">Taxonomic reliability of DNA sequences in public sequence databases: A fungal perspective</span>. <i>PLoS One</i>,  <span class="vol">1</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">e59</span>. <a href="https://doi.org/10.1371/journal.pone.0000059" class="linkBehavior">https://doi.org/10.1371/journal.pone.0000059</a><div class="extra-links" id="e_1_2_14_129_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0000059" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsFenurc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=17183689" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000207443600059" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2006&amp;pages=e59&amp;journal=PLoS+One&amp;issue=1&amp;author=R.+H.+Nilsson&amp;author=M.+Ryberg&amp;author=E.+Kristiansson&amp;author=K.+Abarenkov&amp;author=K.%E2%80%90H.+Larsson&amp;author=U.+K%C3%B5ljalg&amp;title=Taxonomic+reliability+of+DNA+sequences+in+public+sequence+databases%3A+A+fungal+perspective" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0123"> <span class="author">Nock, C. J.</span>,  <span class="author">Waters, D. L. E.</span>,  <span class="author">Edwards, M. A.</span>,  <span class="author">Bowen, S. G.</span>,  <span class="author">Rice, N.</span>,  <span class="author">Cordeiro, G. M.</span>, &amp;  <span class="author">Henry, R. J.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Chloroplast genome sequences from total DNA for plant identification: Chloroplast genome sequences for plant identification</span>. <i>Plant Biotechnology Journal</i>,  <span class="vol">9</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">328</span>– <span class="pageLast">333</span>. <a href="https://doi.org/10.1111/j.1467-7652.2010.00558.x" class="linkBehavior">https://doi.org/10.1111/j.1467-7652.2010.00558.x</a><div class="extra-links" id="e_1_2_14_130_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1467-7652.2010.00558.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVyitrk%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20796245" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000288630900004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2011&amp;pages=328-333&amp;journal=Plant+Biotechnology+Journal&amp;issue=3&amp;author=C.+J.+Nock&amp;author=D.+L.+E.+Waters&amp;author=M.+A.+Edwards&amp;author=S.+G.+Bowen&amp;author=N.+Rice&amp;author=G.+M.+Cordeiro&amp;author=R.+J.+Henry&amp;title=Chloroplast+genome+sequences+from+total+DNA+for+plant+identification%3A+Chloroplast+genome+sequences+for+plant+identification" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0124"> <span class="author">O'Brien, H. E.</span>,  <span class="author">Parrent, J. L.</span>,  <span class="author">Jackson, J. A.</span>,  <span class="author">Moncalvo, J.‐M.</span>, &amp;  <span class="author">Vilgalys, R.</span> ( <span class="pubYear">2005</span>).  <span class="articleTitle">Fungal community analysis by large‐scale sequencing of&nbsp;environmental samples</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">71</span>( <span class="citedIssue">9</span>),  <span class="pageFirst">5544</span>– <span class="pageLast">5550</span>. <a href="https://doi.org/10.1128/AEM.71.9.5544-5550.2005" class="linkBehavior">https://doi.org/10.1128/AEM.71.9.5544-5550.2005</a><div class="extra-links" id="e_1_2_14_131_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1128%2FAEM.71.9.5544-5550.2005" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVahtr3F" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=16151147" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000231897400076" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=71&amp;publication_year=2005&amp;pages=5544-5550&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=9&amp;author=H.+E.+O%27Brien&amp;author=J.+L.+Parrent&amp;author=J.+A.+Jackson&amp;author=J.%E2%80%90M.+Moncalvo&amp;author=R.+Vilgalys&amp;title=Fungal+community+analysis+by+large%E2%80%90scale+sequencing+of%C2%A0environmental+samples" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0125"> <span class="author">O'Donnell, J. L.</span>,  <span class="author">Kelly, R. P.</span>,  <span class="author">Lowell, N. C.</span>, &amp;  <span class="author">Port, J. A.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">Indexed PCR primers induce template‐specific bias in large‐scale DNA sequencing studies</span>. <i>PLoS ONE</i>,  <span class="vol">11</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">e0148698</span>. <a href="https://doi.org/10.1371/journal.pone.0148698" class="linkBehavior">https://doi.org/10.1371/journal.pone.0148698</a><div class="extra-links" id="e_1_2_14_132_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0148698" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=26950069" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000371990100007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2016&amp;pages=e0148698&amp;journal=PLoS+ONE&amp;issue=3&amp;author=J.+L.+O%27Donnell&amp;author=R.+P.+Kelly&amp;author=N.+C.+Lowell&amp;author=J.+A.+Port&amp;title=Indexed+PCR+primers+induce+template%E2%80%90specific+bias+in+large%E2%80%90scale+DNA+sequencing+studies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0126"> <span class="author">Ottesen, E. A.</span>,  <span class="author">Hong, J. W.</span>,  <span class="author">Quake, S. R.</span>, &amp;  <span class="author">Leadbetter, J. R.</span> ( <span class="pubYear">2006</span>).  <span class="articleTitle">Microfluidic digital PCR enables multigene analysis of individual environmental bacteria</span>. <i>Science</i>,  <span class="vol">314</span>( <span class="citedIssue">5804</span>),  <span class="pageFirst">1464</span>– <span class="pageLast">1467</span>. <a href="https://doi.org/10.1126/science.1131370" class="linkBehavior">https://doi.org/10.1126/science.1131370</a><div class="extra-links" id="e_1_2_14_133_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1126%2Fscience.1131370" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1CntrnM" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=17138901" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000242406100045" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=314&amp;publication_year=2006&amp;pages=1464-1467&amp;journal=Science&amp;issue=5804&amp;author=E.+A.+Ottesen&amp;author=J.+W.+Hong&amp;author=S.+R.+Quake&amp;author=J.+R.+Leadbetter&amp;title=Microfluidic+digital+PCR+enables+multigene+analysis+of+individual+environmental+bacteria" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0127"> <span class="author">Palacios, G.</span>,  <span class="author">Quan, P.‐L.</span>,  <span class="author">Jabado, O. J.</span>,  <span class="author">Conlan, S.</span>,  <span class="author">Hirschberg, D. L.</span>,  <span class="author">Liu, Y.</span>, …  <span class="author">Grolla, A.</span> ( <span class="pubYear">2007</span>).  <span class="chapterTitle">Panmicrobial oligonucleotide array for diagnosis of infectious diseases</span>. Retrieved from <a href="http://edoc.rki.de/docviews/abstract.php?lang=ger&amp;id=349" class="linkBehavior">http://edoc.rki.de/docviews/abstract.php?lang=ger&amp;id=349</a><div class="extra-links" id="e_1_2_14_134_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=+Palacios%2C+G.%2C++Quan%2C+P.%E2%80%90L.%2C++Jabado%2C+O.+J.%2C++Conlan%2C+S.%2C++Hirschberg%2C+D.+L.%2C++Liu%2C+Y.%2C+%E2%80%A6++Grolla%2C+A.+%28+2007%29.++Panmicrobial+oligonucleotide+array+for+diagnosis+of+infectious+diseases.+Retrieved+from+http%3A%2F%2Fedoc.rki.de%2Fdocviews%2Fabstract.php%3Flang%3Dger%26id%3D349" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0128"> <span class="author">Parks, M.</span>,  <span class="author">Cronn, R.</span>, &amp;  <span class="author">Liston, A.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">Increasing phylogenetic resolution at low taxonomic levels using massively parallel sequencing of chloroplast genomes</span>. <i>BMC Biology</i>,  <span class="vol">7</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">84</span>. <a href="https://doi.org/10.1186/1741-7007-7-84" class="linkBehavior">https://doi.org/10.1186/1741-7007-7-84</a><div class="extra-links" id="e_1_2_14_135_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1741-7007-7-84" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19954512" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000272783300001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2009&amp;pages=84&amp;journal=BMC+Biology&amp;issue=1&amp;author=M.+Parks&amp;author=R.+Cronn&amp;author=A.+Liston&amp;title=Increasing+phylogenetic+resolution+at+low+taxonomic+levels+using+massively+parallel+sequencing+of+chloroplast+genomes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0129"> <span class="author">Parks, D. H.</span>,  <span class="author">Porter, M.</span>,  <span class="author">Churcher, S.</span>,  <span class="author">Wang, S.</span>,  <span class="author">Blouin, C.</span>,  <span class="author">Whalley, J.</span>, …  <span class="author">Beiko, R. G.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">GenGIS: A geospatial information system for genomic data</span>. <i>Genome Research</i>,  <span class="vol">19</span>( <span class="citedIssue">10</span>),  <span class="pageFirst">1896</span>– <span class="pageLast">1904</span>. <a href="https://doi.org/10.1101/gr.095612.109" class="linkBehavior">https://doi.org/10.1101/gr.095612.109</a><div class="extra-links" id="e_1_2_14_136_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1101%2Fgr.095612.109" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1GjtbrI" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19635847" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000270389700022" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2009&amp;pages=1896-1904&amp;journal=Genome+Research&amp;issue=10&amp;author=D.+H.+Parks&amp;author=M.+Porter&amp;author=S.+Churcher&amp;author=S.+Wang&amp;author=C.+Blouin&amp;author=J.+Whalley&amp;author=R.+G.+Beiko&amp;title=GenGIS%3A+A+geospatial+information+system+for+genomic+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0130"> <span class="author">Parks, D. H.</span>,  <span class="author">Rinke, C.</span>,  <span class="author">Chuvochina, M.</span>,  <span class="author">Chaumeil, P.‐A.</span>,  <span class="author">Woodcroft, B. J.</span>,  <span class="author">Evans, P. N.</span>, …  <span class="author">Tyson, G. W.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">Recovery of nearly 8,000 metagenome‐assembled genomes substantially expands the tree of life</span>. <i>Nature Microbiology</i>,  <span class="vol">2</span>( <span class="citedIssue">11</span>),  <span class="pageFirst">1533</span>– <span class="pageLast">1542</span>. <a href="https://doi.org/10.1038/s41564-017-0012-7" class="linkBehavior">https://doi.org/10.1038/s41564-017-0012-7</a><div class="extra-links" id="e_1_2_14_137_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fs41564-017-0012-7" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvV2isLnK" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=28894102" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000417975600013" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=2017&amp;pages=1533-1542&amp;journal=Nature+Microbiology&amp;issue=11&amp;author=D.+H.+Parks&amp;author=C.+Rinke&amp;author=M.+Chuvochina&amp;author=P.%E2%80%90A.+Chaumeil&amp;author=B.+J.+Woodcroft&amp;author=P.+N.+Evans&amp;author=G.+W.+Tyson&amp;title=Recovery+of+nearly+8%2C000+metagenome%E2%80%90assembled+genomes+substantially+expands+the+tree+of+life" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0131"> <span class="author">Pawlowski, J.</span>,  <span class="author">Esling, P.</span>,  <span class="author">Lejzerowicz, F.</span>,  <span class="author">Cedhagen, T.</span>, &amp;  <span class="author">Wilding, T. A.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Environmental
 monitoring through protist next‐generation sequencing metabarcoding: 
Assessing the impact of fish farming on benthic foraminifera communities</span>. <i>Molecular Ecology Resources</i>,  <span class="vol">14</span>( <span class="citedIssue">6</span>),  <span class="pageFirst">1129</span>– <span class="pageLast">1140</span>. <a href="https://doi.org/10.1111/1755-0998.12261" class="linkBehavior">https://doi.org/10.1111/1755-0998.12261</a><div class="extra-links" id="e_1_2_14_138_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12261" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslCqsbfL" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=24734911" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000343805400004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2014&amp;pages=1129-1140&amp;journal=Molecular+Ecology+Resources&amp;issue=6&amp;author=J.+Pawlowski&amp;author=P.+Esling&amp;author=F.+Lejzerowicz&amp;author=T.+Cedhagen&amp;author=T.+A.+Wilding&amp;title=Environmental+monitoring+through+protist+next%E2%80%90generation+sequencing+metabarcoding%3A+Assessing+the+impact+of+fish+farming+on+benthic+foraminifera+communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0132"> <span class="author">Pietramellara, G.</span>,  <span class="author">Ascher, J.</span>,  <span class="author">Borgogni, F.</span>,  <span class="author">Ceccherini, M. T.</span>,  <span class="author">Guerri, G.</span>, &amp;  <span class="author">Nannipieri, P.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">Extracellular DNA in soil and sediment: Fate and ecological relevance</span>. <i>Biology and Fertility of Soils</i>,  <span class="vol">45</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">219</span>– <span class="pageLast">235</span>. <a href="https://doi.org/10.1007/s00374-008-0345-8" class="linkBehavior">https://doi.org/10.1007/s00374-008-0345-8</a><div class="extra-links" id="e_1_2_14_139_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1007%2Fs00374-008-0345-8" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFektLs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000262825700001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=2009&amp;pages=219-235&amp;journal=Biology+and+Fertility+of+Soils&amp;issue=3&amp;author=G.+Pietramellara&amp;author=J.+Ascher&amp;author=F.+Borgogni&amp;author=M.+T.+Ceccherini&amp;author=G.+Guerri&amp;author=P.+Nannipieri&amp;title=Extracellular+DNA+in+soil+and+sediment%3A+Fate+and+ecological+relevance" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0133"> <span class="author">Polz, M. F.</span>, &amp;  <span class="author">Cavanaugh, C. M.</span> ( <span class="pubYear">1998</span>).  <span class="articleTitle">Bias in template‐to‐product ratios in multitemplate PCR</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">64</span>( <span class="citedIssue">10</span>),  <span class="pageFirst">3724</span>– <span class="pageLast">3730</span>.
                     <div class="extra-links" id="e_1_2_14_140_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADyaK1cXms1entrc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=9758791" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000076295100026" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=64&amp;publication_year=1998&amp;pages=3724-3730&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=10&amp;author=M.+F.+Polz&amp;author=C.+M.+Cavanaugh&amp;title=Bias+in+template%E2%80%90to%E2%80%90product+ratios+in+multitemplate+PCR" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0134"> <span class="author">Porter, T. M.</span>,  <span class="author">Gibson, J. F.</span>,  <span class="author">Shokralla, S.</span>,  <span class="author">Baird, D. J.</span>,  <span class="author">Golding, G. B.</span>, &amp;  <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Rapid
 and accurate taxonomic classification of insect (class Insecta) 
cytochrome c oxidase subunit 1 (COI) DNA barcode sequences using a naïve
 Bayesian classifier</span>. <i>Molecular Ecology Resources</i>,  <span class="vol">14</span>( <span class="citedIssue">5</span>),  <span class="pageFirst">929</span>– <span class="pageLast">942</span>. <a href="https://doi.org/10.1111/1755-0998.12240" class="linkBehavior">https://doi.org/10.1111/1755-0998.12240</a><div class="extra-links" id="e_1_2_14_141_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12240" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKlu7rL" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000340425100005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=2014&amp;pages=929-942&amp;journal=Molecular+Ecology+Resources&amp;issue=5&amp;author=T.+M.+Porter&amp;author=J.+F.+Gibson&amp;author=S.+Shokralla&amp;author=D.+J.+Baird&amp;author=G.+B.+Golding&amp;author=M.+Hajibabaei&amp;title=Rapid+and+accurate+taxonomic+classification+of+insect+%28class+Insecta%29+cytochrome+c+oxidase+subunit+1+%28COI%29+DNA+barcode+sequences+using+a+na%C3%AFve+Bayesian+classifier" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0135"> <span class="author">Porter, T. M.</span>, &amp;  <span class="author">Golding, G. B.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Are
 similarity‐ or phylogeny‐based methods more appropriate for classifying
 internal transcribed spacer (ITS) metagenomic amplicons?</span> <i>New Phytologist</i>,  <span class="vol">192</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">775</span>– <span class="pageLast">782</span>. <a href="https://doi.org/10.1111/j.1469-8137.2011.03838.x" class="linkBehavior">https://doi.org/10.1111/j.1469-8137.2011.03838.x</a><div class="extra-links" id="e_1_2_14_142_1"><span>
                           <a href="https://nph.onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2011.03838.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFWhsr%252FK" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21806618" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000296850800022" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=192&amp;publication_year=2011&amp;pages=775-782&amp;journal=New+Phytologist&amp;issue=3&amp;author=T.+M.+Porter&amp;author=G.+B.+Golding&amp;title=Are+similarity%E2%80%90+or+phylogeny%E2%80%90based+methods+more+appropriate+for+classifying+internal+transcribed+spacer+%28ITS%29+metagenomic+amplicons%3F" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0136"> <span class="author">Porter, T. M.</span>, &amp;  <span class="author">Golding, G. B.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Factors
 that affect large subunit ribosomal DNA amplicon sequencing studies of 
fungal communities: classification method, primer choice, and error</span>. <i>PLoS ONE</i>,  <span class="vol">7</span>( <span class="citedIssue">4</span>),  <span class="pageFirst">e35749</span>. <a href="https://doi.org/10.1371/journal.pone.0035749" class="linkBehavior">https://doi.org/10.1371/journal.pone.0035749</a><div class="extra-links" id="e_1_2_14_143_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0035749" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFOntL4%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22558215" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000305336000056" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2012&amp;pages=e35749&amp;journal=PLoS+ONE&amp;issue=4&amp;author=T.+M.+Porter&amp;author=G.+B.+Golding&amp;title=Factors+that+affect+large+subunit+ribosomal+DNA+amplicon+sequencing+studies+of+fungal+communities%3A+classification+method%2C+primer+choice%2C+and+error" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0138"> <span class="author">Porter, T. M.</span>, &amp;  <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2017</span>).  <span class="otherTitle">Automated high throughput animal DNA metabarcode classification</span>. <i>bioRxiv</i>, 219675. <a href="https://doi.org/10.1101/219675" class="linkBehavior">https://doi.org/10.1101/219675</a><div class="extra-links" id="e_1_2_14_144_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=+Porter%2C+T.+M.%2C+%26++Hajibabaei%2C+M.+%28+2017%29.++Automated+high+throughput+animal+DNA+metabarcode+classification.+bioRxiv%2C+219675.+https%3A%2F%2Fdoi.org%2F10.1101%2F219675" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0139"> <span class="author">Porter, T. M.</span>,  <span class="author">Shokralla, S.</span>,  <span class="author">Baird, D.</span>,  <span class="author">Golding, G. B.</span>, &amp;  <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">Ribosomal DNA and plastid markers used to sample fungal and plant communities from wetland soils reveals complementary biotas</span>. <i>PLoS ONE</i>,  <span class="vol">11</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">e0142759</span>. <a href="https://doi.org/10.1371/journal.pone.0142759" class="linkBehavior">https://doi.org/10.1371/journal.pone.0142759</a><div class="extra-links" id="e_1_2_14_145_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0142759" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=26731732" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000367801400004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2016&amp;pages=e0142759&amp;journal=PLoS+ONE&amp;issue=1&amp;author=T.+M.+Porter&amp;author=S.+Shokralla&amp;author=D.+Baird&amp;author=G.+B.+Golding&amp;author=M.+Hajibabaei&amp;title=Ribosomal+DNA+and+plastid+markers+used+to+sample+fungal+and+plant+communities+from+wetland+soils+reveals+complementary+biotas" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0140"> <span class="author">Price, M. N.</span>,  <span class="author">Dehal, P. S.</span>, &amp;  <span class="author">Arkin, A. P.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">FastTree: Computing large minimum evolution trees with profiles instead of a distance matrix</span>. <i>Molecular Biology and Evolution</i>,  <span class="vol">26</span>( <span class="citedIssue">7</span>),  <span class="pageFirst">1641</span>– <span class="pageLast">1650</span>. <a href="https://doi.org/10.1093/molbev/msp077" class="linkBehavior">https://doi.org/10.1093/molbev/msp077</a><div class="extra-links" id="e_1_2_14_146_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fmolbev%2Fmsp077" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXntlGjurc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19377059" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000266966200020" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2009&amp;pages=1641-1650&amp;journal=Molecular+Biology+and+Evolution&amp;issue=7&amp;author=M.+N.+Price&amp;author=P.+S.+Dehal&amp;author=A.+P.+Arkin&amp;title=FastTree%3A+Computing+large+minimum+evolution+trees+with+profiles+instead+of+a+distance+matrix" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0141"> <span class="author">Pruesse, E.</span>,  <span class="author">Quast, C.</span>,  <span class="author">Knittel, K.</span>,  <span class="author">Fuchs, B. M.</span>,  <span class="author">Ludwig, W.</span>,  <span class="author">Peplies, J.</span>, &amp;  <span class="author">Glockner, F. O.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">SILVA: A comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB</span>. <i>Nucleic Acids Research</i>,  <span class="vol">35</span>( <span class="citedIssue">21</span>),  <span class="pageFirst">7188</span>– <span class="pageLast">7196</span>. <a href="https://doi.org/10.1093/nar/gkm864" class="linkBehavior">https://doi.org/10.1093/nar/gkm864</a><div class="extra-links" id="e_1_2_14_147_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fnar%2Fgkm864" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFGmtg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=17947321" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000251868800024" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=2007&amp;pages=7188-7196&amp;journal=Nucleic+Acids+Research&amp;issue=21&amp;author=E.+Pruesse&amp;author=C.+Quast&amp;author=K.+Knittel&amp;author=B.+M.+Fuchs&amp;author=W.+Ludwig&amp;author=J.+Peplies&amp;author=F.+O.+Glockner&amp;title=SILVA%3A+A+comprehensive+online+resource+for+quality+checked+and+aligned+ribosomal+RNA+sequence+data+compatible+with+ARB" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0142"> <span class="author">Quince, C.</span>,  <span class="author">Lanzen, A.</span>,  <span class="author">Davenport, R. J.</span>, &amp;  <span class="author">Turnbaugh, P. J.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Removing noise from pyrosequenced amplicons</span>. <i>BMC Bioinformatics</i>,  <span class="vol">12</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">38</span>. <a href="https://doi.org/10.1186/1471-2105-12-38" class="linkBehavior">https://doi.org/10.1186/1471-2105-12-38</a><div class="extra-links" id="e_1_2_14_148_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1471-2105-12-38" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21276213" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000287741700001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2011&amp;pages=38&amp;journal=BMC+Bioinformatics&amp;issue=1&amp;author=C.+Quince&amp;author=A.+Lanzen&amp;author=R.+J.+Davenport&amp;author=P.+J.+Turnbaugh&amp;title=Removing+noise+from+pyrosequenced+amplicons" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0143"> <span class="author">Ranasinghe, J. A.</span>,  <span class="author">Stein, E. D.</span>,  <span class="author">Miller, P. E.</span>, &amp;  <span class="author">Weisberg, S. B.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Performance
 of two southern California benthic community condition indices using 
species abundance and presence‐only data: Relevance to DNA barcoding</span>. <i>PLoS ONE</i>,  <span class="vol">7</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">e40875</span>. <a href="https://doi.org/10.1371/journal.pone.0040875" class="linkBehavior">https://doi.org/10.1371/journal.pone.0040875</a><div class="extra-links" id="e_1_2_14_149_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0040875" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22879881" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000307437900005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2012&amp;pages=e40875&amp;journal=PLoS+ONE&amp;issue=8&amp;author=J.+A.+Ranasinghe&amp;author=E.+D.+Stein&amp;author=P.+E.+Miller&amp;author=S.+B.+Weisberg&amp;title=Performance+of+two+southern+California+benthic+community+condition+indices+using+species+abundance+and+presence%E2%80%90only+data%3A+Relevance+to+DNA+barcoding" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0144"> <span class="author">Ratnasingham, S.</span>, &amp;  <span class="author">Hebert, P. D.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">BOLD: The barcode of life data system</span> (<a href="http://www.barcodinglife.org/" class="linkBehavior">http://www.barcodinglife.org</a>). <i>Molecular Ecology Notes</i>,  <span class="vol">7</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">355</span>– <span class="pageLast">364</span>. <a href="https://doi.org/10.1111/j.1471-8286.2007.01678.x" class="linkBehavior">https://doi.org/10.1111/j.1471-8286.2007.01678.x</a><div class="extra-links" id="e_1_2_14_150_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1471-8286.2007.01678.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXntVyksbc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18784790" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000246244300001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2007&amp;pages=355-364&amp;journal=Molecular+Ecology+Notes&amp;issue=3&amp;author=S.+Ratnasingham&amp;author=P.+D.+Hebert&amp;title=BOLD%3A+The+barcode+of+life+data+system" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0145"> <span class="author">Reeder, J.</span>, &amp;  <span class="author">Knight, R.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">The “rare biosphere”: A reality check</span>. <i>Nature Methods</i>,  <span class="vol">6</span>( <span class="citedIssue">9</span>),  <span class="pageFirst">636</span>– <span class="pageLast">637</span>. <a href="https://doi.org/10.1038/nmeth0909-636" class="linkBehavior">https://doi.org/10.1038/nmeth0909-636</a><div class="extra-links" id="e_1_2_14_151_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnmeth0909-636" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKjsbbE" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19718016" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000269382900009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2009&amp;pages=636-637&amp;journal=Nature+Methods&amp;issue=9&amp;author=J.+Reeder&amp;author=R.+Knight&amp;title=The+%E2%80%9Crare+biosphere%E2%80%9D%3A+A+reality+check" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0146"> <span class="author">Riley, R.</span>,  <span class="author">Salamov, A. A.</span>,  <span class="author">Brown, D. W.</span>,  <span class="author">Nagy, L. G.</span>,  <span class="author">Floudas, D.</span>,  <span class="author">Held, B. W.</span>, …  <span class="author">Grigoriev, I. V.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Extensive sampling of basidiomycete genomes demonstrates inadequacy of the white‐rot/brown‐rot paradigm for wood decay fungi</span>. <i>Proceedings of the National Academy of Sciences of the United States of America</i>,  <span class="vol">111</span>,  <span class="pageFirst">14959</span>.
                     <div class="extra-links" id="e_1_2_14_152_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1073%2Fpnas.1400592111" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000338514800053" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=111&amp;publication_year=2014&amp;pages=14959&amp;journal=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;author=R.+Riley&amp;author=A.+A.+Salamov&amp;author=D.+W.+Brown&amp;author=L.+G.+Nagy&amp;author=D.+Floudas&amp;author=B.+W.+Held&amp;author=I.+V.+Grigoriev&amp;title=Extensive+sampling+of+basidiomycete+genomes+demonstrates+inadequacy+of+the+white%E2%80%90rot%2Fbrown%E2%80%90rot+paradigm+for+wood+decay+fungi" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0147"> <span class="author">Rognes, T.</span>,  <span class="author">Flouri, T.</span>,  <span class="author">Nichols, B.</span>,  <span class="author">Quince, C.</span>, &amp;  <span class="author">Mahé, F.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">VSEARCH: A versatile open source tool for metagenomics</span>. <i>PeerJ</i>,  <span class="vol">4</span>,  <span class="pageFirst">e2584</span>. <a href="https://doi.org/10.7717/peerj.2584" class="linkBehavior">https://doi.org/10.7717/peerj.2584</a><div class="extra-links" id="e_1_2_14_153_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.7717%2Fpeerj.2584" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=27781170" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000385583100005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2016&amp;pages=e2584&amp;journal=PeerJ&amp;author=T.+Rognes&amp;author=T.+Flouri&amp;author=B.+Nichols&amp;author=C.+Quince&amp;author=F.+Mah%C3%A9&amp;title=VSEARCH%3A+A+versatile+open+source+tool+for+metagenomics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0148"> <span class="author">Rokas, A.</span>,  <span class="author">Williams, B. L.</span>,  <span class="author">King, N.</span>, &amp;  <span class="author">Carroll, S. B.</span> ( <span class="pubYear">2003</span>).  <span class="articleTitle">Genome‐scale approaches to resolving incongruence in molecular phylogenies</span>. <i>Nature</i>,  <span class="vol">425</span>( <span class="citedIssue">6960</span>),  <span class="pageFirst">798</span>– <span class="pageLast">804</span>. <a href="https://doi.org/10.1038/nature02053" class="linkBehavior">https://doi.org/10.1038/nature02053</a><div class="extra-links" id="e_1_2_14_154_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnature02053" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotlWqsbc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=14574403" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000186118500036" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=425&amp;publication_year=2003&amp;pages=798-804&amp;journal=Nature&amp;issue=6960&amp;author=A.+Rokas&amp;author=B.+L.+Williams&amp;author=N.+King&amp;author=S.+B.+Carroll&amp;title=Genome%E2%80%90scale+approaches+to+resolving+incongruence+in+molecular+phylogenies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0149"> <span class="author">Ryberg, M.</span>,  <span class="author">Kristiansson, E.</span>,  <span class="author">Sjökvist, E.</span>, &amp;  <span class="author">Nilsson, R. H.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">An
 outlook on the fungal internal transcribed spacer sequences in GenBank 
and the introduction of a web‐based tool for the exploration of fungal 
diversity</span>. <i>New Phytologist</i>,  <span class="vol">181</span>( <span class="citedIssue">2</span>),  <span class="pageFirst">471</span>– <span class="pageLast">477</span>. <a href="https://doi.org/10.1111/j.1469-8137.2008.02667.x" class="linkBehavior">https://doi.org/10.1111/j.1469-8137.2008.02667.x</a><div class="extra-links" id="e_1_2_14_155_1"><span>
                           <a href="https://nph.onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2008.02667.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvV2nsbw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19121041" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000261792900021" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=181&amp;publication_year=2009&amp;pages=471-477&amp;journal=New+Phytologist&amp;issue=2&amp;author=M.+Ryberg&amp;author=E.+Kristiansson&amp;author=E.+Sj%C3%B6kvist&amp;author=R.+H.+Nilsson&amp;title=An+outlook+on+the+fungal+internal+transcribed+spacer+sequences+in+GenBank+and+the+introduction+of+a+web%E2%80%90based+tool+for+the+exploration+of+fungal+diversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0150"> <span class="author">Salter, S. J.</span>,  <span class="author">Cox, M. J.</span>,  <span class="author">Turek, E. M.</span>,  <span class="author">Calus, S. T.</span>,  <span class="author">Cookson, W. O.</span>,  <span class="author">Moffatt, M. F.</span>, …  <span class="author">Walker, A. W.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Reagent and laboratory contamination can critically impact sequence‐based microbiome analyses</span>. <i>BMC Biology</i>,  <span class="vol">12</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">87</span>. <a href="https://doi.org/10.1186/s12915-014-0087-z" class="linkBehavior">https://doi.org/10.1186/s12915-014-0087-z</a><div class="extra-links" id="e_1_2_14_156_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2Fs12915-014-0087-z" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=25387460" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000345876100001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2014&amp;pages=87&amp;journal=BMC+Biology&amp;issue=1&amp;author=S.+J.+Salter&amp;author=M.+J.+Cox&amp;author=E.+M.+Turek&amp;author=S.+T.+Calus&amp;author=W.+O.+Cookson&amp;author=M.+F.+Moffatt&amp;author=A.+W.+Walker&amp;title=Reagent+and+laboratory+contamination+can+critically+impact+sequence%E2%80%90based+microbiome+analyses" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0151"> <span class="author">Sauer, S.</span>,  <span class="author">Lange, B. M. H.</span>,  <span class="author">Gobom, J.</span>,  <span class="author">Nyarsik, L.</span>,  <span class="author">Seitz, H.</span>, &amp;  <span class="author">Lehrach, H.</span> ( <span class="pubYear">2005</span>).  <span class="articleTitle">Miniaturization in functional genomics and proteomics</span>. <i>Nature Reviews Genetics</i>,  <span class="vol">6</span>( <span class="citedIssue">6</span>),  <span class="pageFirst">465</span>– <span class="pageLast">476</span>. <a href="https://doi.org/10.1038/nrg1618" class="linkBehavior">https://doi.org/10.1038/nrg1618</a><div class="extra-links" id="e_1_2_14_157_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnrg1618" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2MXkslSnu7k%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=15931170" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000229490100012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2005&amp;pages=465-476&amp;journal=Nature+Reviews+Genetics&amp;issue=6&amp;author=S.+Sauer&amp;author=B.+M.+H.+Lange&amp;author=J.+Gobom&amp;author=L.+Nyarsik&amp;author=H.+Seitz&amp;author=H.+Lehrach&amp;title=Miniaturization+in+functional+genomics+and+proteomics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0152"> <span class="author">Schena, M.</span>,  <span class="author">Heller, R. A.</span>,  <span class="author">Theriault, T. P.</span>,  <span class="author">Konrad, K.</span>,  <span class="author">Lachenmeier, E.</span>, &amp;  <span class="author">Davis, R. W.</span> ( <span class="pubYear">1998</span>).  <span class="articleTitle">Microarrays: Biotechnology's discovery platform for functional genomics</span>. <i>Trends in Biotechnology</i>,  <span class="vol">16</span>( <span class="citedIssue">7</span>),  <span class="pageFirst">301</span>– <span class="pageLast">306</span>. <a href="https://doi.org/10.1016/S0167-7799%2898%2901219-0" class="linkBehavior">https://doi.org/10.1016/S0167-7799(98)01219-0</a><div class="extra-links" id="e_1_2_14_158_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2FS0167-7799%2898%2901219-0" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADyaK1cXkvVWnt7s%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=9675914" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000074546900005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=1998&amp;pages=301-306&amp;journal=Trends+in+Biotechnology&amp;issue=7&amp;author=M.+Schena&amp;author=R.+A.+Heller&amp;author=T.+P.+Theriault&amp;author=K.+Konrad&amp;author=E.+Lachenmeier&amp;author=R.+W.+Davis&amp;title=Microarrays%3A+Biotechnology%27s+discovery+platform+for+functional+genomics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0153"> <span class="author">Schloss, P. D.</span>,  <span class="author">Westcott, S. L.</span>,  <span class="author">Ryabin, T.</span>,  <span class="author">Hall, J. R.</span>,  <span class="author">Hartmann, M.</span>,  <span class="author">Hollister, E. B.</span>, …  <span class="author">Weber, C. F.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">Introducing
 mothur: Open‐source, platform‐independent, community‐supported software
 for describing and comparing microbial communities</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">75</span>( <span class="citedIssue">23</span>),  <span class="pageFirst">7537</span>– <span class="pageLast">7541</span>. <a href="https://doi.org/10.1128/AEM.01541-09" class="linkBehavior">https://doi.org/10.1128/AEM.01541-09</a><div class="extra-links" id="e_1_2_14_159_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1128%2FAEM.01541-09" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXis1yltw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19801464" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000271944800028" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=75&amp;publication_year=2009&amp;pages=7537-7541&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=23&amp;author=P.+D.+Schloss&amp;author=S.+L.+Westcott&amp;author=T.+Ryabin&amp;author=J.+R.+Hall&amp;author=M.+Hartmann&amp;author=E.+B.+Hollister&amp;author=C.+F.+Weber&amp;title=Introducing+mothur%3A+Open%E2%80%90source%2C+platform%E2%80%90independent%2C+community%E2%80%90supported+software+for+describing+and+comparing+microbial+communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0154"> <span class="author">Schrader, C.</span>,  <span class="author">Schielke, A.</span>,  <span class="author">Ellerbroek, L.</span>, &amp;  <span class="author">Johne, R.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">PCR inhibitors – occurrence, properties and removal</span>. <i>Journal of Applied Microbiology</i>,  <span class="vol">113</span>( <span class="citedIssue">5</span>),  <span class="pageFirst">1014</span>– <span class="pageLast">1026</span>. <a href="https://doi.org/10.1111/j.1365-2672.2012.05384.x" class="linkBehavior">https://doi.org/10.1111/j.1365-2672.2012.05384.x</a><div class="extra-links" id="e_1_2_14_160_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2672.2012.05384.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFWisrjK" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22747964" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000310281100002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=113&amp;publication_year=2012&amp;pages=1014-1026&amp;journal=Journal+of+Applied+Microbiology&amp;issue=5&amp;author=C.+Schrader&amp;author=A.+Schielke&amp;author=L.+Ellerbroek&amp;author=R.+Johne&amp;title=PCR+inhibitors+%E2%80%93+occurrence%2C+properties+and+removal" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0155"> <span class="author">Seo, T.‐K.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Classification of nucleotide sequences using support vector machines</span>. <i>Journal of Molecular Evolution</i>,  <span class="vol">71</span>( <span class="citedIssue">4</span>),  <span class="pageFirst">250</span>– <span class="pageLast">267</span>. <a href="https://doi.org/10.1007/s00239-010-9380-9" class="linkBehavior">https://doi.org/10.1007/s00239-010-9380-9</a><div class="extra-links" id="e_1_2_14_161_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1007%2Fs00239-010-9380-9" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFOhsr7I" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20740280" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000282475700002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=71&amp;publication_year=2010&amp;pages=250-267&amp;journal=Journal+of+Molecular+Evolution&amp;issue=4&amp;author=T.%E2%80%90K.+Seo&amp;title=Classification+of+nucleotide+sequences+using+support+vector+machines" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0156"> <span class="author">Shi, Z. Y.</span>,  <span class="author">Yang, C. Q.</span>,  <span class="author">Hao, M. D.</span>,  <span class="author">Wang, X. Y.</span>,  <span class="author">Ward, R. D.</span>, &amp;  <span class="author">Zhang, A. B.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">FuzzyID2:
 A software package for large dataset species identification via 
barcoding and metabarcoding using Hidden Markov Models and fuzzy set 
methods</span>. <i>Molecular Ecology Resources</i>, <a href="https://doi.org/10.1111/1755-0998.12738" class="linkBehavior">https://doi.org/10.1111/1755-0998.12738</a><div class="extra-links" id="e_1_2_14_162_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12738" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=29154499" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2017&amp;journal=Molecular+Ecology+Resources&amp;author=Z.+Y.+Shi&amp;author=C.+Q.+Yang&amp;author=M.+D.+Hao&amp;author=X.+Y.+Wang&amp;author=R.+D.+Ward&amp;author=A.+B.+Zhang&amp;title=FuzzyID2%3A+A+software+package+for+large+dataset+species+identification+via+barcoding+and+metabarcoding+using+Hidden+Markov+Models+and+fuzzy+set+methods" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0157"> <span class="author">Shokralla, S.</span>,  <span class="author">Gibson, J.</span>,  <span class="author">King, I.</span>,  <span class="author">Baird, D.</span>,  <span class="author">Janzen, D.</span>,  <span class="author">Hallwachs, W.</span>, &amp;  <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">Environmental
 DNA barcode sequence capture: Targeted, PCR‐free sequence capture for 
biodiversity analysis from bulk environmental samples</span>. <i>bioRxiv</i>,  <span class="pageFirst">87437</span>.
                     <div class="extra-links" id="e_1_2_14_163_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2016&amp;pages=87437&amp;journal=bioRxiv&amp;author=S.+Shokralla&amp;author=J.+Gibson&amp;author=I.+King&amp;author=D.+Baird&amp;author=D.+Janzen&amp;author=W.+Hallwachs&amp;author=M.+Hajibabaei&amp;title=Environmental+DNA+barcode+sequence+capture%3A+Targeted%2C+PCR%E2%80%90free+sequence+capture+for+biodiversity+analysis+from+bulk+environmental+samples" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0158"> <span class="author">Shokralla, S.</span>,  <span class="author">Hellberg, R. S.</span>,  <span class="author">Handy, S. M.</span>,  <span class="author">King, I.</span>, &amp;  <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">A DNA mini‐barcoding system for authentication of processed fish products</span>. <i>Scientific Reports</i>,  <span class="vol">5</span>,  <span class="pageFirst">15894</span>. <a href="https://doi.org/10.1038/srep15894" class="linkBehavior">https://doi.org/10.1038/srep15894</a><div class="extra-links" id="e_1_2_14_164_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fsrep15894" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCmtbjJ" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=26516098" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000363778900001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2015&amp;pages=15894&amp;journal=Scientific+Reports&amp;author=S.+Shokralla&amp;author=R.+S.+Hellberg&amp;author=S.+M.+Handy&amp;author=I.+King&amp;author=M.+Hajibabaei&amp;title=A+DNA+mini%E2%80%90barcoding+system+for+authentication+of+processed+fish+products" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0159"> <span class="author">Shokralla, S.</span>,  <span class="author">Spall, J. L.</span>,  <span class="author">Gibson, J. F.</span>, &amp;  <span class="author">Hajibabaei, M.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Next‐generation sequencing technologies for environmental DNA research</span>. <i>Molecular Ecology</i>,  <span class="vol">21</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">1794</span>– <span class="pageLast">1805</span>. <a href="https://doi.org/10.1111/j.1365-294X.2012.05538.x" class="linkBehavior">https://doi.org/10.1111/j.1365-294X.2012.05538.x</a><div class="extra-links" id="e_1_2_14_165_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2012.05538.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVGksLo%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22486820" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000302616200002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=1794-1805&amp;journal=Molecular+Ecology&amp;issue=8&amp;author=S.+Shokralla&amp;author=J.+L.+Spall&amp;author=J.+F.+Gibson&amp;author=M.+Hajibabaei&amp;title=Next%E2%80%90generation+sequencing+technologies+for+environmental+DNA+research" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0160"> <span class="author">Smith, C. J.</span>, &amp;  <span class="author">Osborn, A. M.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">Advantages
 and limitations of quantitative PCR (Q‐PCR)‐based approaches in 
microbial ecology: Application of Q‐PCR in microbial ecology</span>. <i>FEMS Microbiology Ecology</i>,  <span class="vol">67</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">6</span>– <span class="pageLast">20</span>. <a href="https://doi.org/10.1111/j.1574-6941.2008.00629.x" class="linkBehavior">https://doi.org/10.1111/j.1574-6941.2008.00629.x</a><div class="extra-links" id="e_1_2_14_166_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1574-6941.2008.00629.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1Sjsg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19120456" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000261622000002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=67&amp;publication_year=2009&amp;pages=6-20&amp;journal=FEMS+Microbiology+Ecology&amp;issue=1&amp;author=C.+J.+Smith&amp;author=A.+M.+Osborn&amp;title=Advantages+and+limitations+of+quantitative+PCR+%28Q%E2%80%90PCR%29%E2%80%90based+approaches+in+microbial+ecology%3A+Application+of+Q%E2%80%90PCR+in+microbial+ecology" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0161"> <span class="author">Sogin, M. L.</span>,  <span class="author">Morrison, H. G.</span>,  <span class="author">Huber, J. A.</span>,  <span class="author">Welch, D. M.</span>,  <span class="author">Huse, S. M.</span>,  <span class="author">Neal, P. R.</span>, …  <span class="author">Herndl, G. J.</span> ( <span class="pubYear">2006</span>).  <span class="articleTitle">Microbial diversity in the deep sea and the underexplored “rare biosphere”</span>. <i>Proceedings of the National Academy of Sciences</i>,  <span class="vol">103</span>( <span class="citedIssue">32</span>),  <span class="pageFirst">12115</span>– <span class="pageLast">12120</span>. <a href="https://doi.org/10.1073/pnas.0605127103" class="linkBehavior">https://doi.org/10.1073/pnas.0605127103</a><div class="extra-links" id="e_1_2_14_167_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1073%2Fpnas.0605127103" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD28Xotlyisb4%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=16880384" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000239701900053" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=103&amp;publication_year=2006&amp;pages=12115-12120&amp;journal=Proceedings+of+the+National+Academy+of+Sciences&amp;issue=32&amp;author=M.+L.+Sogin&amp;author=H.+G.+Morrison&amp;author=J.+A.+Huber&amp;author=D.+M.+Welch&amp;author=S.+M.+Huse&amp;author=P.+R.+Neal&amp;author=G.+J.+Herndl&amp;title=Microbial+diversity+in+the+deep+sea+and+the+underexplored+%E2%80%9Crare+biosphere%E2%80%9D" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0162"> <span class="author">Song, H.</span>,  <span class="author">Buhay, J. E.</span>,  <span class="author">Whiting, M. F.</span>, &amp;  <span class="author">Crandall, K. A.</span> ( <span class="pubYear">2008</span>).  <span class="articleTitle">Many species in one: DNA barcoding overestimates the number of species when nuclear mitochondrial pseudogenes are coamplified</span>. <i>Proceedings of the National Academy of Sciences</i>,  <span class="vol">105</span>( <span class="citedIssue">36</span>),  <span class="pageFirst">13486</span>– <span class="pageLast">13491</span>. <a href="https://doi.org/10.1073/pnas.0803076105" class="linkBehavior">https://doi.org/10.1073/pnas.0803076105</a><div class="extra-links" id="e_1_2_14_168_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1073%2Fpnas.0803076105" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFChs7vP" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18757756" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000259251700056" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2008&amp;pages=13486-13491&amp;journal=Proceedings+of+the+National+Academy+of+Sciences&amp;issue=36&amp;author=H.+Song&amp;author=J.+E.+Buhay&amp;author=M.+F.+Whiting&amp;author=K.+A.+Crandall&amp;title=Many+species+in+one%3A+DNA+barcoding+overestimates+the+number+of+species+when+nuclear+mitochondrial+pseudogenes+are+coamplified" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0163"> <span class="author">Steele, P. R.</span>, &amp;  <span class="author">Pires, J. C.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Biodiversity assessment: State‐of‐the‐art techniques in phylogenomics and species identification</span>. <i>American Journal of Botany</i>,  <span class="vol">98</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">415</span>– <span class="pageLast">425</span>. <a href="https://doi.org/10.3732/ajb.1000296" class="linkBehavior">https://doi.org/10.3732/ajb.1000296</a><div class="extra-links" id="e_1_2_14_169_1"><span>
                           <a href="https://bsapubs.onlinelibrary.wiley.com/doi/10.3732/ajb.1000296" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21613135" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000288320600015" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=98&amp;publication_year=2011&amp;pages=415-425&amp;journal=American+Journal+of+Botany&amp;issue=3&amp;author=P.+R.+Steele&amp;author=J.+C.+Pires&amp;title=Biodiversity+assessment%3A+State%E2%80%90of%E2%80%90the%E2%80%90art+techniques+in+phylogenomics+and+species+identification" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0164"> <span class="author">Straub, S. C. K.</span>,  <span class="author">Parks, M.</span>,  <span class="author">Weitemier, K.</span>,  <span class="author">Fishbein, M.</span>,  <span class="author">Cronn, R. C.</span>, &amp;  <span class="author">Liston, A.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Navigating the tip of the genomic iceberg: Next‐generation sequencing for plant systematics</span>. <i>American Journal of Botany</i>,  <span class="vol">99</span>( <span class="citedIssue">2</span>),  <span class="pageFirst">349</span>– <span class="pageLast">364</span>. <a href="https://doi.org/10.3732/ajb.1100335" class="linkBehavior">https://doi.org/10.3732/ajb.1100335</a><div class="extra-links" id="e_1_2_14_170_1"><span>
                           <a href="https://bsapubs.onlinelibrary.wiley.com/doi/10.3732/ajb.1100335" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XksValtbo%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22174336" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000300471100015" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=99&amp;publication_year=2012&amp;pages=349-364&amp;journal=American+Journal+of+Botany&amp;issue=2&amp;author=S.+C.+K.+Straub&amp;author=M.+Parks&amp;author=K.+Weitemier&amp;author=M.+Fishbein&amp;author=R.+C.+Cronn&amp;author=A.+Liston&amp;title=Navigating+the+tip+of+the+genomic+iceberg%3A+Next%E2%80%90generation+sequencing+for+plant+systematics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0165"> <span class="author">Sundquist, A.</span>,  <span class="author">Bigdeli, S.</span>,  <span class="author">Jalili, R.</span>,  <span class="author">Druzin, M. L.</span>,  <span class="author">Waller, S.</span>,  <span class="author">Pullen, K. M.</span>, …  <span class="author">Ronaghi, M.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">Bacterial flora‐typing with targeted, chip‐based Pyrosequencing</span>. <i>BMC Microbiology</i>,  <span class="vol">7</span>,  <span class="pageFirst">108</span>. <a href="https://doi.org/10.1186/1471-2180-7-108" class="linkBehavior">https://doi.org/10.1186/1471-2180-7-108</a><div class="extra-links" id="e_1_2_14_171_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1471-2180-7-108" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnslWkug%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18047683" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000253968300001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2007&amp;pages=108&amp;journal=BMC+Microbiology&amp;author=A.+Sundquist&amp;author=S.+Bigdeli&amp;author=R.+Jalili&amp;author=M.+L.+Druzin&amp;author=S.+Waller&amp;author=K.+M.+Pullen&amp;author=M.+Ronaghi&amp;title=Bacterial+flora%E2%80%90typing+with+targeted%2C+chip%E2%80%90based+Pyrosequencing" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0166"> <span class="author">Suzuki, M. T.</span>, &amp;  <span class="author">Giovannoni, S. J.</span> ( <span class="pubYear">1996</span>).  <span class="articleTitle">Bias caused by template annealing in the amplification of mixtures of 16S rRNA genes by PCR</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">62</span>( <span class="citedIssue">2</span>),  <span class="pageFirst">625</span>– <span class="pageLast">630</span>.
                     <div class="extra-links" id="e_1_2_14_172_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADyaK28XovVCrug%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=8593063" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=A1996TT69000049" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=62&amp;publication_year=1996&amp;pages=625-630&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=2&amp;author=M.+T.+Suzuki&amp;author=S.+J.+Giovannoni&amp;title=Bias+caused+by+template+annealing+in+the+amplification+of+mixtures+of+16S+rRNA+genes+by+PCR" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0167"> <span class="author">Sweeney, B. W.</span>,  <span class="author">Battle, J. M.</span>,  <span class="author">Jackson, J. K.</span>, &amp;  <span class="author">Dapkey, T.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Can DNA barcodes of stream macroinvertebrates improve descriptions of community structure and water quality?</span> <i>Journal of the North American Benthological Society</i>,  <span class="vol">30</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">195</span>– <span class="pageLast">216</span>. <a href="https://doi.org/10.1899/10-016.1" class="linkBehavior">https://doi.org/10.1899/10-016.1</a><div class="extra-links" id="e_1_2_14_173_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1899%2F10-016.1" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000287114800016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=30&amp;publication_year=2011&amp;pages=195-216&amp;journal=Journal+of+the+North+American+Benthological+Society&amp;issue=1&amp;author=B.+W.+Sweeney&amp;author=J.+M.+Battle&amp;author=J.+K.+Jackson&amp;author=T.+Dapkey&amp;title=Can+DNA+barcodes+of+stream+macroinvertebrates+improve+descriptions+of+community+structure+and+water+quality%3F" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0168"> <span class="author">Taberlet, P.</span>,  <span class="author">Coissac, E.</span>,  <span class="author">Hajibabaei, M.</span>, &amp;  <span class="author">Rieseberg, L. H.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Environmental DNA</span>. <i>Molecular Ecology</i>,  <span class="vol">21</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">1789</span>– <span class="pageLast">1793</span>. <a href="https://doi.org/10.1111/j.1365-294X.2012.05542.x" class="linkBehavior">https://doi.org/10.1111/j.1365-294X.2012.05542.x</a><div class="extra-links" id="e_1_2_14_174_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2012.05542.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVGksLw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22486819" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000302616200001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=1789-1793&amp;journal=Molecular+Ecology&amp;issue=8&amp;author=P.+Taberlet&amp;author=E.+Coissac&amp;author=M.+Hajibabaei&amp;author=L.+H.+Rieseberg&amp;title=Environmental+DNA" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0169"> <span class="author">Taberlet, P.</span>,  <span class="author">Coissac, E.</span>,  <span class="author">Pompanon, F.</span>,  <span class="author">Brochmann, C.</span>, &amp;  <span class="author">Willerslev, E.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Towards next‐generation biodiversity assessment using DNA metabarcoding</span>. <i>Molecular Ecology</i>,  <span class="vol">21</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">2045</span>– <span class="pageLast">2050</span>. <a href="https://doi.org/10.1111/j.1365-294X.2012.05470.x" class="linkBehavior">https://doi.org/10.1111/j.1365-294X.2012.05470.x</a><div class="extra-links" id="e_1_2_14_175_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2012.05470.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ASTN%3A280%3ADC%252BC38rjsVahuw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22486824" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000302616200022" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=2045-2050&amp;journal=Molecular+Ecology&amp;issue=8&amp;author=P.+Taberlet&amp;author=E.+Coissac&amp;author=F.+Pompanon&amp;author=C.+Brochmann&amp;author=E.+Willerslev&amp;title=Towards+next%E2%80%90generation+biodiversity+assessment+using+DNA+metabarcoding" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0170"> <span class="author">Taberlet, P.</span>,  <span class="author">Prud'Homme, S. M.</span>,  <span class="author">Campione, E.</span>,  <span class="author">Roy, J.</span>,  <span class="author">Miquel, C.</span>,  <span class="author">Shehzad, W.</span>, …  <span class="author">Coissac, E.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Soil
 sampling and isolation of extracellular DNA from large amount of 
starting material suitable for metabarcoding studies: EXTRACTION OF 
EXTRACELLULAR DNA FROM SOIL</span>. <i>Molecular Ecology</i>,  <span class="vol">21</span>( <span class="citedIssue">8</span>),  <span class="pageFirst">1816</span>– <span class="pageLast">1820</span>. <a href="https://doi.org/10.1111/j.1365-294X.2011.05317.x" class="linkBehavior">https://doi.org/10.1111/j.1365-294X.2011.05317.x</a><div class="extra-links" id="e_1_2_14_176_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2011.05317.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVGksLY%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22300434" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000302616200004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=1816-1820&amp;journal=Molecular+Ecology&amp;issue=8&amp;author=P.+Taberlet&amp;author=S.+M.+Prud%27Homme&amp;author=E.+Campione&amp;author=J.+Roy&amp;author=C.+Miquel&amp;author=W.+Shehzad&amp;author=E.+Coissac&amp;title=Soil+sampling+and+isolation+of+extracellular+DNA+from+large+amount+of+starting+material+suitable+for+metabarcoding+studies%3A+EXTRACTION+OF+EXTRACELLULAR+DNA+FROM+SOIL" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0171"> <span class="author">Tang, M.</span>,  <span class="author">Tan, M.</span>,  <span class="author">Meng, G.</span>,  <span class="author">Yang, S.</span>,  <span class="author">Su, X.</span>,  <span class="author">Liu, S.</span>, …  <span class="author">Zhou, X.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Multiplex sequencing of pooled mitochondrial genomes–a crucial step toward biodiversity analysis using mito‐metagenomics</span>. <i>Nucleic Acids Research</i>,  <span class="vol">42</span>( <span class="citedIssue">22</span>),  <span class="pageFirst">e166</span>. <a href="https://doi.org/10.1093/nar/gku917" class="linkBehavior">https://doi.org/10.1093/nar/gku917</a><div class="extra-links" id="e_1_2_14_177_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1093%2Fnar%2Fgku917" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=25294837" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000347916900001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=42&amp;publication_year=2014&amp;pages=e166&amp;journal=Nucleic+Acids+Research&amp;issue=22&amp;author=M.+Tang&amp;author=M.+Tan&amp;author=G.+Meng&amp;author=S.+Yang&amp;author=X.+Su&amp;author=S.+Liu&amp;author=X.+Zhou&amp;title=Multiplex+sequencing+of+pooled+mitochondrial+genomes%E2%80%93a+crucial+step+toward+biodiversity+analysis+using+mito%E2%80%90metagenomics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0172"> <span class="author">Tedersoo, L.</span>,  <span class="author">Anslan, S.</span>,  <span class="author">Bahram, M.</span>,  <span class="author">Põlme, S.</span>,  <span class="author">Riit, T.</span>,  <span class="author">Liiv, I.</span>, …  <span class="author">Abarenkov, K.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">Shotgun metagenomes and multiple primer pair‐barcode combinations of amplicons reveal biases in metabarcoding analyses of fungi</span>. <i>MycoKeys</i>,  <span class="vol">10</span>,  <span class="pageFirst">1</span>– <span class="pageLast">43</span>. <a href="https://doi.org/10.3897/mycokeys.10.4852" class="linkBehavior">https://doi.org/10.3897/mycokeys.10.4852</a><div class="extra-links" id="e_1_2_14_178_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.3897%2Fmycokeys.10.4852" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000358262000001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2015&amp;pages=1-43&amp;journal=MycoKeys&amp;author=L.+Tedersoo&amp;author=S.+Anslan&amp;author=M.+Bahram&amp;author=S.+P%C3%B5lme&amp;author=T.+Riit&amp;author=I.+Liiv&amp;author=K.+Abarenkov&amp;title=Shotgun+metagenomes+and+multiple+primer+pair%E2%80%90barcode+combinations+of+amplicons+reveal+biases+in+metabarcoding+analyses+of+fungi" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0173"> <span class="author">Tedersoo, L.</span>,  <span class="author">Bahram, M.</span>,  <span class="author">Polme, S.</span>,  <span class="author">Koljalg, U.</span>,  <span class="author">Yorou, N. S.</span>,  <span class="author">Wijesundera, R.</span>, …  <span class="author">Abarenkov, K.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Global diversity and geography of soil fungi</span>. <i>Science</i>,  <span class="vol">346</span>( <span class="citedIssue">6213</span>),  <span class="pageFirst">1256688</span>. <a href="https://doi.org/10.1126/science.1256688" class="linkBehavior">https://doi.org/10.1126/science.1256688</a><div class="extra-links" id="e_1_2_14_179_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1126%2Fscience.1256688" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtb7P" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=25430773" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000345763400032" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=346&amp;publication_year=2014&amp;pages=1256688&amp;journal=Science&amp;issue=6213&amp;author=L.+Tedersoo&amp;author=M.+Bahram&amp;author=S.+Polme&amp;author=U.+Koljalg&amp;author=N.+S.+Yorou&amp;author=R.+Wijesundera&amp;author=K.+Abarenkov&amp;title=Global+diversity+and+geography+of+soil+fungi" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0174"> <span class="author">Tedersoo, L.</span>,  <span class="author">Nilsson, R. H.</span>,  <span class="author">Abarenkov, K.</span>,  <span class="author">Jairus, T.</span>,  <span class="author">Sadam, A.</span>,  <span class="author">Saar, I.</span>, …  <span class="author">Kõljalg, U.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">454
 Pyrosequencing and Sanger sequencing of tropical mycorrhizal fungi 
provide similar results but reveal substantial methodological biases</span>. <i>New Phytologist</i>,  <span class="vol">188</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">291</span>– <span class="pageLast">301</span>. <a href="https://doi.org/10.1111/j.1469-8137.2010.03373.x" class="linkBehavior">https://doi.org/10.1111/j.1469-8137.2010.03373.x</a><div class="extra-links" id="e_1_2_14_180_1"><span>
                           <a href="https://nph.onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2010.03373.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1aqsLfK" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20636324" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000281551500028" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=188&amp;publication_year=2010&amp;pages=291-301&amp;journal=New+Phytologist&amp;issue=1&amp;author=L.+Tedersoo&amp;author=R.+H.+Nilsson&amp;author=K.+Abarenkov&amp;author=T.+Jairus&amp;author=A.+Sadam&amp;author=I.+Saar&amp;author=U.+K%C3%B5ljalg&amp;title=454+Pyrosequencing+and+Sanger+sequencing+of+tropical+mycorrhizal+fungi+provide+similar+results+but+reveal+substantial+methodological+biases" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0175"> <span class="author">Tewhey, R.</span>,  <span class="author">Warner, J. B.</span>,  <span class="author">Nakano, M.</span>,  <span class="author">Libby, B.</span>,  <span class="author">Medkova, M.</span>,  <span class="author">David, P. H.</span>, …  <span class="author">Frazer, K. A.</span> ( <span class="pubYear">2009</span>).  <span class="articleTitle">Microdroplet‐based PCR enrichment for large‐scale targeted sequencing</span>. <i>Nature Biotechnology</i>,  <span class="vol">27</span>( <span class="citedIssue">11</span>),  <span class="pageFirst">1025</span>– <span class="pageLast">1031</span>. <a href="https://doi.org/10.1038/nbt.1583" class="linkBehavior">https://doi.org/10.1038/nbt.1583</a><div class="extra-links" id="e_1_2_14_181_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnbt.1583" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlers7nJ" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=19881494" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000271524700023" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=2009&amp;pages=1025-1031&amp;journal=Nature+Biotechnology&amp;issue=11&amp;author=R.+Tewhey&amp;author=J.+B.+Warner&amp;author=M.+Nakano&amp;author=B.+Libby&amp;author=M.+Medkova&amp;author=P.+H.+David&amp;author=K.+A.+Frazer&amp;title=Microdroplet%E2%80%90based+PCR+enrichment+for+large%E2%80%90scale+targeted+sequencing" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0176"> <span class="groupName">The Human Microbiome Project Consortium</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Structure, function and diversity of the healthy human microbiome</span>. <i>Nature</i>,  <span class="vol">486</span>,  <span class="pageFirst">207</span>– <span class="pageLast">214</span>.
                     <div class="extra-links" id="e_1_2_14_182_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnature11234" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38Xos1ejurs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22699609" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000305189000025" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=486&amp;publication_year=2012&amp;pages=207-214&amp;journal=Nature&amp;title=Structure%2C+function+and+diversity+of+the+healthy+human+microbiome" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0177"> <span class="author">Thompson, L. R.</span>,  <span class="author">Sanders, J. G.</span>,  <span class="author">McDonald, D.</span>,  <span class="author">Amir, A.</span>,  <span class="author">Ladau, J.</span>,  <span class="author">Locey, K. J.</span>, …  <span class="author">Zhao, H.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">A communal catalogue reveals Earth's multiscale microbial diversity</span>. <i>Nature</i>,  <span class="vol">551</span>,  <span class="pageFirst">457</span>– <span class="pageLast">463</span>. <a href="https://doi.org/10.1038/nature24621" class="linkBehavior">https://doi.org/10.1038/nature24621</a><div class="extra-links" id="e_1_2_14_183_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnature24621" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslehurzL" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=29088705" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000416043700036" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=551&amp;publication_year=2017&amp;pages=457-463&amp;journal=Nature&amp;author=L.+R.+Thompson&amp;author=J.+G.+Sanders&amp;author=D.+McDonald&amp;author=A.+Amir&amp;author=J.+Ladau&amp;author=K.+J.+Locey&amp;author=H.+Zhao&amp;title=A+communal+catalogue+reveals+Earth%27s+multiscale+microbial+diversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0178"> <span class="author">Thomsen, P. F.</span>,  <span class="author">Kielgast, J.</span>,  <span class="author">Iversen, L. L.</span>,  <span class="author">Wiuf, C.</span>,  <span class="author">Rasmussen, M.</span>,  <span class="author">Gilbert, M. T. P.</span>, …  <span class="author">Willerslev, E.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Monitoring endangered freshwater biodiversity using environmental DNA</span>. <i>Molecular Ecology</i>,  <span class="vol">21</span>( <span class="citedIssue">11</span>),  <span class="pageFirst">2565</span>– <span class="pageLast">2573</span>. <a href="https://doi.org/10.1111/j.1365-294X.2011.05418.x" class="linkBehavior">https://doi.org/10.1111/j.1365-294X.2011.05418.x</a><div class="extra-links" id="e_1_2_14_184_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2011.05418.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Wlu7vO" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=22151771" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000304389500004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=2565-2573&amp;journal=Molecular+Ecology&amp;issue=11&amp;author=P.+F.+Thomsen&amp;author=J.+Kielgast&amp;author=L.+L.+Iversen&amp;author=C.+Wiuf&amp;author=M.+Rasmussen&amp;author=M.+T.+P.+Gilbert&amp;author=E.+Willerslev&amp;title=Monitoring+endangered+freshwater+biodiversity+using+environmental+DNA" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0179"> <span class="author">Torsvik, V.</span>, &amp;  <span class="author">Ovreas, L.</span> ( <span class="pubYear">2002</span>).  <span class="articleTitle">Microbial diversity and function in soil: From genes to ecosystems</span>. <i>Current Opinion in Microbiology</i>,  <span class="vol">5</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">240</span>– <span class="pageLast">245</span>. <a href="https://doi.org/10.1016/S1369-5274%2802%2900324-7" class="linkBehavior">https://doi.org/10.1016/S1369-5274(02)00324-7</a><div class="extra-links" id="e_1_2_14_185_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1016%2FS1369-5274%2802%2900324-7" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD38XktFynsL4%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=12057676" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000176281700002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2002&amp;pages=240-245&amp;journal=Current+Opinion+in+Microbiology&amp;issue=3&amp;author=V.+Torsvik&amp;author=L.+Ovreas&amp;title=Microbial+diversity+and+function+in+soil%3A+From+genes+to+ecosystems" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0180"> <span class="author">Tringe, S. G.</span>,  <span class="author">Mering, C.</span>,  <span class="author">Kobayashi, A.</span>,  <span class="author">Salamov, A. A.</span>,  <span class="author">Chen, K.</span>,  <span class="author">Chang, H. W.</span>, …  <span class="author">Rubin, E. M.</span> ( <span class="pubYear">2005</span>).  <span class="articleTitle">Comparative metagenomics of microbial communities</span>. <i>Science</i>,  <span class="vol">308</span>,  <span class="pageFirst">554</span>– <span class="pageLast">557</span>. <a href="https://doi.org/10.1126/science.1107851" class="linkBehavior">https://doi.org/10.1126/science.1107851</a><div class="extra-links" id="e_1_2_14_186_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1126%2Fscience.1107851" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlOjt7c%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=15845853" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000228810500053" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=308&amp;publication_year=2005&amp;pages=554-557&amp;journal=Science&amp;author=S.+G.+Tringe&amp;author=C.+Mering&amp;author=A.+Kobayashi&amp;author=A.+A.+Salamov&amp;author=K.+Chen&amp;author=H.+W.+Chang&amp;author=E.+M.+Rubin&amp;title=Comparative+metagenomics+of+microbial+communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0181"> <span class="author">Tucker, C. M.</span>, &amp;  <span class="author">Cadotte, M. W.</span> ( <span class="pubYear">2013</span>).  <span class="articleTitle">Unifying measures of biodiversity: Understanding when richness and phylogenetic diversity should be congruent</span>. <i>Diversity and Distributions</i>,  <span class="vol">19</span>( <span class="citedIssue">7</span>),  <span class="pageFirst">845</span>– <span class="pageLast">854</span>. <a href="https://doi.org/10.1111/ddi.12087" class="linkBehavior">https://doi.org/10.1111/ddi.12087</a><div class="extra-links" id="e_1_2_14_187_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/ddi.12087" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000320327500018" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2013&amp;pages=845-854&amp;journal=Diversity+and+Distributions&amp;issue=7&amp;author=C.+M.+Tucker&amp;author=M.+W.+Cadotte&amp;title=Unifying+measures+of+biodiversity%3A+Understanding+when+richness+and+phylogenetic+diversity+should+be+congruent" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0182"> <span class="author">Urich, T.</span>,  <span class="author">Lanzén, A.</span>,  <span class="author">Qi, J.</span>,  <span class="author">Huson, D. H.</span>,  <span class="author">Schleper, C.</span>, &amp;  <span class="author">Schuster, S. C.</span> ( <span class="pubYear">2008</span>).  <span class="articleTitle">Simultaneous assessment of soil microbial community structure and function through analysis of the meta‐transcriptome</span>. <i>PLoS ONE</i>,  <span class="vol">3</span>( <span class="citedIssue">6</span>),  <span class="pageFirst">e2527</span>. <a href="https://doi.org/10.1371/journal.pone.0002527" class="linkBehavior">https://doi.org/10.1371/journal.pone.0002527</a><div class="extra-links" id="e_1_2_14_188_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0002527" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslSrtbs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18575584" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000263288000042" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2008&amp;pages=e2527&amp;journal=PLoS+ONE&amp;issue=6&amp;author=T.+Urich&amp;author=A.+Lanz%C3%A9n&amp;author=J.+Qi&amp;author=D.+H.+Huson&amp;author=C.+Schleper&amp;author=S.+C.+Schuster&amp;title=Simultaneous+assessment+of+soil+microbial+community+structure+and+function+through+analysis+of+the+meta%E2%80%90transcriptome" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0183"> <span class="author">Venter, J. C.</span>,  <span class="author">Adams, M. D.</span>,  <span class="author">Myers, E. W.</span>,  <span class="author">Li, P. W.</span>,  <span class="author">Mural, R. J.</span>,  <span class="author">Sutton, G. G.</span>, …  <span class="author">Gocayne, J. D.</span> ( <span class="pubYear">2001</span>).  <span class="articleTitle">The sequence of the human genome</span>. <i>Science</i>,  <span class="vol">291</span>,  <span class="pageFirst">1304</span>– <span class="pageLast">1351</span>. <a href="https://doi.org/10.1126/science.1058040" class="linkBehavior">https://doi.org/10.1126/science.1058040</a><div class="extra-links" id="e_1_2_14_189_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1126%2Fscience.1058040" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtlSgsbo%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=11181995" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000166993400041" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=291&amp;publication_year=2001&amp;pages=1304-1351&amp;journal=Science&amp;author=J.+C.+Venter&amp;author=M.+D.+Adams&amp;author=E.+W.+Myers&amp;author=P.+W.+Li&amp;author=R.+J.+Mural&amp;author=G.+G.+Sutton&amp;author=J.+D.+Gocayne&amp;title=The+sequence+of+the+human+genome" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0184"> <span class="author">Venter, J. C.</span>,  <span class="author">Remington, K.</span>,  <span class="author">Heidelberg, J. F.</span>,  <span class="author">Halpern, A. L.</span>,  <span class="author">Rusch, D.</span>,  <span class="author">Eisen, J. A.</span>, …  <span class="author">Fouts, D. E.</span> ( <span class="pubYear">2004</span>).  <span class="articleTitle">Environmental genome shotgun sequencing of the Sargasso Sea</span>. <i>Science</i>,  <span class="vol">304</span>( <span class="citedIssue">5667</span>),  <span class="pageFirst">66</span>– <span class="pageLast">74</span>. <a href="https://doi.org/10.1126/science.1093857" class="linkBehavior">https://doi.org/10.1126/science.1093857</a><div class="extra-links" id="e_1_2_14_190_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1126%2Fscience.1093857" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2cXis1ansLw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=15001713" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000220567900037" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=304&amp;publication_year=2004&amp;pages=66-74&amp;journal=Science&amp;issue=5667&amp;author=J.+C.+Venter&amp;author=K.+Remington&amp;author=J.+F.+Heidelberg&amp;author=A.+L.+Halpern&amp;author=D.+Rusch&amp;author=J.+A.+Eisen&amp;author=D.+E.+Fouts&amp;title=Environmental+genome+shotgun+sequencing+of+the+Sargasso+Sea" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0185"> <span class="author">Větrovský, T.</span>, &amp;  <span class="author">Baldrian, P.</span> ( <span class="pubYear">2013</span>).  <span class="articleTitle">The variability of the 16S rRNA gene in bacterial genomes and its consequences for bacterial community analyses</span>. <i>PLoS ONE</i>,  <span class="vol">8</span>( <span class="citedIssue">2</span>),  <span class="pageFirst">e57923</span>. <a href="https://doi.org/10.1371/journal.pone.0057923" class="linkBehavior">https://doi.org/10.1371/journal.pone.0057923</a><div class="extra-links" id="e_1_2_14_191_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0057923" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvFGqtLo%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=23460914" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000315519000169" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2013&amp;pages=e57923&amp;journal=PLoS+ONE&amp;issue=2&amp;author=T.+V%C4%9Btrovsk%C3%BD&amp;author=P.+Baldrian&amp;title=The+variability+of+the+16S+rRNA+gene+in+bacterial+genomes+and+its+consequences+for+bacterial+community+analyses" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0186"> <span class="author">Vilanova, C.</span>, &amp;  <span class="author">Porcar, M.</span> ( <span class="pubYear">2016</span>).  <span class="articleTitle">Are multi‐omics enough?</span> <i>Nature Microbiology</i>,  <span class="vol">1</span>,  <span class="pageFirst">16101</span>. <a href="https://doi.org/10.1038/nmicrobiol.2016.101" class="linkBehavior">https://doi.org/10.1038/nmicrobiol.2016.101</a><div class="extra-links" id="e_1_2_14_192_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnmicrobiol.2016.101" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvF2ju70%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=27573112" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000383605900001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2016&amp;pages=16101&amp;journal=Nature+Microbiology&amp;author=C.+Vilanova&amp;author=M.+Porcar&amp;title=Are+multi%E2%80%90omics+enough%3F" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0187"> <span class="author">Virgilio, M.</span>,  <span class="author">Backeljau, T.</span>,  <span class="author">Nevado, B.</span>, &amp;  <span class="author">De Meyer, M.</span> ( <span class="pubYear">2010</span>).  <span class="articleTitle">Comparative performances of DNA barcoding across insect orders</span>. <i>BMC Bioinformatics</i>,  <span class="vol">11</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">206</span>. <a href="https://doi.org/10.1186/1471-2105-11-206" class="linkBehavior">https://doi.org/10.1186/1471-2105-11-206</a><div class="extra-links" id="e_1_2_14_193_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2F1471-2105-11-206" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=20420717" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000279728400001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2010&amp;pages=206&amp;journal=BMC+Bioinformatics&amp;issue=1&amp;author=M.+Virgilio&amp;author=T.+Backeljau&amp;author=B.+Nevado&amp;author=M.+De+Meyer&amp;title=Comparative+performances+of+DNA+barcoding+across+insect+orders" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0188"> <span class="author">Vogelstein, B.</span>, &amp;  <span class="author">Kinzler, K. W.</span> ( <span class="pubYear">1999</span>).  <span class="articleTitle">Digital PCR</span>. <i>Proceedings of the National Academy of Sciences</i>,  <span class="vol">96</span>( <span class="citedIssue">16</span>),  <span class="pageFirst">9236</span>– <span class="pageLast">9241</span>. <a href="https://doi.org/10.1073/pnas.96.16.9236" class="linkBehavior">https://doi.org/10.1073/pnas.96.16.9236</a><div class="extra-links" id="e_1_2_14_194_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1073%2Fpnas.96.16.9236" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADyaK1MXltVequrY%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=10430926" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000081835500084" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=96&amp;publication_year=1999&amp;pages=9236-9241&amp;journal=Proceedings+of+the+National+Academy+of+Sciences&amp;issue=16&amp;author=B.+Vogelstein&amp;author=K.+W.+Kinzler&amp;title=Digital+PCR" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0189"> <span class="author">Wang, Q.</span>,  <span class="author">Garrity, G. M.</span>,  <span class="author">Tiedje, J. M.</span>, &amp;  <span class="author">Cole, J. R.</span> ( <span class="pubYear">2007</span>).  <span class="articleTitle">Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">73</span>( <span class="citedIssue">16</span>),  <span class="pageFirst">5261</span>– <span class="pageLast">5267</span>. <a href="https://doi.org/10.1128/AEM.00062-07" class="linkBehavior">https://doi.org/10.1128/AEM.00062-07</a><div class="extra-links" id="e_1_2_14_195_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1128%2FAEM.00062-07" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpsleqtrc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=17586664" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000248825900024" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=73&amp;publication_year=2007&amp;pages=5261-5267&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=16&amp;author=Q.+Wang&amp;author=G.+M.+Garrity&amp;author=J.+M.+Tiedje&amp;author=J.+R.+Cole&amp;title=Naive+Bayesian+classifier+for+rapid+assignment+of+rRNA+sequences+into+the+new+bacterial+taxonomy" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0190"> <span class="author">Wang, G. C.</span>, &amp;  <span class="author">Wang, Y.</span> ( <span class="pubYear">1997</span>).  <span class="articleTitle">Frequency
 of formation of chimeric molecules as a consequence of PCR 
coamplification of 16S rRNA genes from mixed bacterial genomes</span>. <i>Applied and Environmental Microbiology</i>,  <span class="vol">63</span>( <span class="citedIssue">12</span>),  <span class="pageFirst">4645</span>– <span class="pageLast">4650</span>.
                     <div class="extra-links" id="e_1_2_14_196_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADyaK2sXnvV2rsbw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=9406382" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=A1997YK70900004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=63&amp;publication_year=1997&amp;pages=4645-4650&amp;journal=Applied+and+Environmental+Microbiology&amp;issue=12&amp;author=G.+C.+Wang&amp;author=Y.+Wang&amp;title=Frequency+of+formation+of+chimeric+molecules+as+a+consequence+of+PCR+coamplification+of+16S+rRNA+genes+from+mixed+bacterial+genomes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0191"> <span class="author">Weiss, S.</span>,  <span class="author">Xu, Z. Z.</span>,  <span class="author">Peddada, S.</span>,  <span class="author">Amir, A.</span>,  <span class="author">Bittinger, K.</span>,  <span class="author">Gonzalez, A.</span>, …  <span class="author">Knight, R.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">Normalization and microbial differential abundance strategies depend upon data characteristics</span>. <i>Microbiome</i>,  <span class="vol">5</span>( <span class="citedIssue">1</span>), <a href="https://doi.org/10.1186/s40168-017-0237-y" class="linkBehavior">https://doi.org/10.1186/s40168-017-0237-y</a><div class="extra-links" id="e_1_2_14_197_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1186%2Fs40168-017-0237-y" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000397308000002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2017&amp;journal=Microbiome&amp;issue=1&amp;author=S.+Weiss&amp;author=Z.+Z.+Xu&amp;author=S.+Peddada&amp;author=A.+Amir&amp;author=K.+Bittinger&amp;author=A.+Gonzalez&amp;author=R.+Knight&amp;title=Normalization+and+microbial+differential+abundance+strategies+depend+upon+data+characteristics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0192"> <span class="author">Westcott, S. L.</span>, &amp;  <span class="author">Schloss, P. D.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">De
 novo clustering methods outperform reference‐based methods for 
assigning 16S rRNA gene sequences to operational taxonomic units</span>. <i>PeerJ</i>,  <span class="vol">3</span>,  <span class="pageFirst">e1487</span>. <a href="https://doi.org/10.7717/peerj.1487" class="linkBehavior">https://doi.org/10.7717/peerj.1487</a><div class="extra-links" id="e_1_2_14_198_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.7717%2Fpeerj.1487" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=26664811" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000366412100003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2015&amp;pages=e1487&amp;journal=PeerJ&amp;author=S.+L.+Westcott&amp;author=P.+D.+Schloss&amp;title=De+novo+clustering+methods+outperform+reference%E2%80%90based+methods+for+assigning+16S+rRNA+gene+sequences+to+operational+taxonomic+units" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0194"> <span class="author">Wilcox, T. M.</span>,  <span class="author">McKelvey, K. S.</span>,  <span class="author">Young, M. K.</span>,  <span class="author">Jane, S. F.</span>,  <span class="author">Lowe, W. H.</span>,  <span class="author">Whiteley, A. R.</span>, &amp;  <span class="author">Schwartz, M. K.</span> ( <span class="pubYear">2013</span>).  <span class="articleTitle">Robust detection of rare species using environmental DNA: The importance of primer specificity</span>. <i>PLoS ONE</i>,  <span class="vol">8</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">e59520</span>. <a href="https://doi.org/10.1371/journal.pone.0059520" class="linkBehavior">https://doi.org/10.1371/journal.pone.0059520</a><div class="extra-links" id="e_1_2_14_199_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1371%2Fjournal.pone.0059520" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlslCgur8%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=23555689" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000317418500042" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2013&amp;pages=e59520&amp;journal=PLoS+ONE&amp;issue=3&amp;author=T.+M.+Wilcox&amp;author=K.+S.+McKelvey&amp;author=M.+K.+Young&amp;author=S.+F.+Jane&amp;author=W.+H.+Lowe&amp;author=A.+R.+Whiteley&amp;author=M.+K.+Schwartz&amp;title=Robust+detection+of+rare+species+using+environmental+DNA%3A+The+importance+of+primer+specificity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0195"> <span class="author">Williams, R.</span>,  <span class="author">Peisajovich, S. G.</span>,  <span class="author">Miller, O. J.</span>,  <span class="author">Magdassi, S.</span>,  <span class="author">Tawfik, D. S.</span>, &amp;  <span class="author">Griffiths, A. D.</span> ( <span class="pubYear">2006</span>).  <span class="articleTitle">Amplification of complex gene libraries by emulsion PCR</span>. <i>Nature Methods</i>,  <span class="vol">3</span>( <span class="citedIssue">7</span>),  <span class="pageFirst">545</span>– <span class="pageLast">550</span>. <a href="https://doi.org/10.1038/nmeth896" class="linkBehavior">https://doi.org/10.1038/nmeth896</a><div class="extra-links" id="e_1_2_14_200_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnmeth896" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtVSntro%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=16791213" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000238669600017" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2006&amp;pages=545-550&amp;journal=Nature+Methods&amp;issue=7&amp;author=R.+Williams&amp;author=S.+G.+Peisajovich&amp;author=O.+J.+Miller&amp;author=S.+Magdassi&amp;author=D.+S.+Tawfik&amp;author=A.+D.+Griffiths&amp;title=Amplification+of+complex+gene+libraries+by+emulsion+PCR" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0196"> <span class="author">Wilson, E. O.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">Biodiversity research requires more boots on the ground</span>. <i>Nature Ecology &amp; Evolution</i>,  <span class="vol">1</span>( <span class="citedIssue">11</span>),  <span class="pageFirst">1590</span>– <span class="pageLast">1591</span>. <a href="https://doi.org/10.1038/s41559-017-0360-y" class="linkBehavior">https://doi.org/10.1038/s41559-017-0360-y</a><div class="extra-links" id="e_1_2_14_201_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fs41559-017-0360-y" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=29066811" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000417193400004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2017&amp;pages=1590-1591&amp;journal=Nature+Ecology+%26+Evolution&amp;issue=11&amp;author=E.+O.+Wilson&amp;title=Biodiversity+research+requires+more+boots+on+the+ground" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0197"> <span class="author">Wu, J.</span>,  <span class="author">Kodzius, R.</span>,  <span class="author">Cao, W.</span>, &amp;  <span class="author">Wen, W.</span> ( <span class="pubYear">2014</span>).  <span class="articleTitle">Extraction, amplification and detection of DNA in microfluidic chip‐based assays</span>. <i>Microchimica Acta</i>,  <span class="vol">181</span>( <span class="citedIssue">13–14</span>),  <span class="pageFirst">1611</span>– <span class="pageLast">1631</span>. <a href="https://doi.org/10.1007/s00604-013-1140-2" class="linkBehavior">https://doi.org/10.1007/s00604-013-1140-2</a><div class="extra-links" id="e_1_2_14_202_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1007%2Fs00604-013-1140-2" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOisbjJ" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000344001000017" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=181&amp;publication_year=2014&amp;pages=1611-1631&amp;journal=Microchimica+Acta&amp;issue=13%E2%80%9314&amp;author=J.+Wu&amp;author=R.+Kodzius&amp;author=W.+Cao&amp;author=W.+Wen&amp;title=Extraction%2C+amplification+and+detection+of+DNA+in+microfluidic+chip%E2%80%90based+assays" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0198"> <span class="author">Yang, J.</span>,  <span class="author">Zhang, X.</span>,  <span class="author">Xie, Y.</span>,  <span class="author">Song, C.</span>,  <span class="author">Zhang, Y.</span>,  <span class="author">Yu, H.</span>, &amp;  <span class="author">Burton, G. A.</span> ( <span class="pubYear">2017</span>).  <span class="articleTitle">Zooplankton community profiling in a eutrophic freshwater ecosystem‐lake Tai Basin by DNA metabarcoding</span>. <i>Scientific Reports</i>,  <span class="vol">7</span>( <span class="citedIssue">1</span>),  <span class="pageFirst">1773</span>. <a href="https://doi.org/10.1038/s41598-017-01808-y" class="linkBehavior">https://doi.org/10.1038/s41598-017-01808-y</a><div class="extra-links" id="e_1_2_14_203_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fs41598-017-01808-y" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=28496194" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000400959000040" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2017&amp;pages=1773&amp;journal=Scientific+Reports&amp;issue=1&amp;author=J.+Yang&amp;author=X.+Zhang&amp;author=Y.+Xie&amp;author=C.+Song&amp;author=Y.+Zhang&amp;author=H.+Yu&amp;author=G.+A.+Burton&amp;title=Zooplankton+community+profiling+in+a+eutrophic+freshwater+ecosystem%E2%80%90lake+Tai+Basin+by+DNA+metabarcoding" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0199"> <span class="author">Yilmaz, P.</span>,  <span class="author">Kottmann, R.</span>,  <span class="author">Field, D.</span>,  <span class="author">Knight, R.</span>,  <span class="author">Cole, J. R.</span>,  <span class="author">Amaral‐Zettler, L.</span>, …  <span class="author">Glöckner, F. O.</span> ( <span class="pubYear">2011</span>).  <span class="articleTitle">Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications</span>. <i>Nature Biotechnology</i>,  <span class="vol">29</span>( <span class="citedIssue">5</span>),  <span class="pageFirst">415</span>– <span class="pageLast">420</span>. <a href="https://doi.org/10.1038/nbt.1823" class="linkBehavior">https://doi.org/10.1038/nbt.1823</a><div class="extra-links" id="e_1_2_14_204_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1038%2Fnbt.1823" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlslykurY%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=21552244" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000290301700021" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=29&amp;publication_year=2011&amp;pages=415-420&amp;journal=Nature+Biotechnology&amp;issue=5&amp;author=P.+Yilmaz&amp;author=R.+Kottmann&amp;author=D.+Field&amp;author=R.+Knight&amp;author=J.+R.+Cole&amp;author=L.+Amaral%E2%80%90Zettler&amp;author=F.+O.+Gl%C3%B6ckner&amp;title=Minimum+information+about+a+marker+gene+sequence+%28MIMARKS%29+and+minimum+information+about+any+%28x%29+sequence+%28MIxS%29+specifications" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0200"> <span class="author">Yu, D. W.</span>,  <span class="author">Ji, Y.</span>,  <span class="author">Emerson, B. C.</span>,  <span class="author">Wang, X.</span>,  <span class="author">Ye, C.</span>,  <span class="author">Yang, C.</span>, &amp;  <span class="author">Ding, Z.</span> ( <span class="pubYear">2012</span>).  <span class="articleTitle">Biodiversity soup: Metabarcoding of arthropods for rapid biodiversity assessment and biomonitoring: Biodiversity soup</span>. <i>Methods in Ecology and Evolution</i>,  <span class="vol">3</span>( <span class="citedIssue">4</span>),  <span class="pageFirst">613</span>– <span class="pageLast">623</span>. <a href="https://doi.org/10.1111/j.2041-210X.2012.00198.x" class="linkBehavior">https://doi.org/10.1111/j.2041-210X.2012.00198.x</a><div class="extra-links" id="e_1_2_14_205_1"><span>
                           <a href="https://besjournals.onlinelibrary.wiley.com/doi/10.1111/j.2041-210X.2012.00198.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000306961700001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2012&amp;pages=613-623&amp;journal=Methods+in+Ecology+and+Evolution&amp;issue=4&amp;author=D.+W.+Yu&amp;author=Y.+Ji&amp;author=B.+C.+Emerson&amp;author=X.+Wang&amp;author=C.+Ye&amp;author=C.+Yang&amp;author=Z.+Ding&amp;title=Biodiversity+soup%3A+Metabarcoding+of+arthropods+for+rapid+biodiversity+assessment+and+biomonitoring%3A+Biodiversity+soup" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0201"> <span class="author">Zhang, A. B.</span>,  <span class="author">Sikes, D. S.</span>,  <span class="author">Muster, C.</span>, &amp;  <span class="author">Li, S. Q.</span> ( <span class="pubYear">2008</span>).  <span class="articleTitle">Inferring species membership using DNA sequences with back‐propagation neural networks</span>. <i>Systematic Biology</i>,  <span class="vol">57</span>( <span class="citedIssue">2</span>),  <span class="pageFirst">202</span>– <span class="pageLast">215</span>. <a href="https://doi.org/10.1080/10635150802032982" class="linkBehavior">https://doi.org/10.1080/10635150802032982</a><div class="extra-links" id="e_1_2_14_206_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.14478&amp;key=10.1080%2F10635150802032982" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVymsr%252FK" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=18398766" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000255690200002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=57&amp;publication_year=2008&amp;pages=202-215&amp;journal=Systematic+Biology&amp;issue=2&amp;author=A.+B.+Zhang&amp;author=D.+S.+Sikes&amp;author=C.+Muster&amp;author=S.+Q.+Li&amp;title=Inferring+species+membership+using+DNA+sequences+with+back%E2%80%90propagation+neural+networks" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec14478-bib-0202"> <span class="author">Zimmermann, J.</span>,  <span class="author">Glöckner, G.</span>,  <span class="author">Jahn, R.</span>,  <span class="author">Enke, N.</span>, &amp;  <span class="author">Gemeinholzer, B.</span> ( <span class="pubYear">2015</span>).  <span class="articleTitle">Metabarcoding vs. morphological identification to assess diatom diversity in environmental studies</span>. <i>Molecular Ecology Resources</i>,  <span class="vol">15</span>( <span class="citedIssue">3</span>),  <span class="pageFirst">526</span>– <span class="pageLast">542</span>. <a href="https://doi.org/10.1111/1755-0998.12336" class="linkBehavior">https://doi.org/10.1111/1755-0998.12336</a><div class="extra-links" id="e_1_2_14_207_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12336" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.14478&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsFamtbc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.14478&amp;key=25270047" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.14478&amp;key=000352653700007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2015&amp;pages=526-542&amp;journal=Molecular+Ecology+Resources&amp;issue=3&amp;author=J.+Zimmermann&amp;author=G.+Gl%C3%B6ckner&amp;author=R.+Jahn&amp;author=N.+Enke&amp;author=B.+Gemeinholzer&amp;title=Metabarcoding+vs.+morphological+identification+to+assess+diatom+diversity+in+environmental+studies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
               </ul>
            </div>
         </div>
      </section>
      
      
      
      
      
      
   </section>
   <div style="display:none">
      <ul class="custom-style-list noteList"></ul>
   </div>
   <section id="citedBy" class="article-section article-section__citedBy">
      <div class="accordion article-accordion"><a href="#" class="accordion__control" aria-controls="idss15" aria-expanded="false"><span class="section__title" id="citedby-section">Citing Literature</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" id="idss15" style="display: none;"><h3 class="border-bottom">Number of times cited according to CrossRef: 17</h3><ul class="rlist separator">
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Amrita
            K. Panda, </span><span class="hlFld-ContribAuthor">Satpal S. Bisht, </span><span class="hlFld-ContribAuthor">Surajit
            De Mandal and </span><span class="hlFld-ContribAuthor">N. Senthil Kumar</span></span>,
            <span>Microbial Diversity of Thermophiles Through the Lens of Next Generation Sequencing</span>,
            <span class="seriesTitle"><span>Microbial Diversity in the Genomic Era</span></span><span class="seperator">, </span><span class="doi">10.1016/B978-0-12-814849-5.00013-7</span><span class="seperator">, </span><span class="page-range"> (217-226)</span><span class="seperator">,
            </span><span class="pub-date"><span class="pub-year">(2019)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1016/B978-0-12-814849-5.00013-7">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Morteza
            Hosseini, </span><span class="hlFld-ContribAuthor">Diogo Pratas and </span><span class="hlFld-ContribAuthor">Armando
            J Pinho</span></span>, <span>Cryfa: a secure encryption tool for genomic data</span><span class="seperator">, </span><span class="seriesTitle"><i>Bioinformatics</i></span><span class="seperator">, </span><span class="doi">10.1093/bioinformatics/bty645</span><span class="seperator">, </span><span class="volume"><b>35</b></span><span class="issue">,
            1</span><span class="seperator">, </span><span class="page-range"> (146-148)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1093/bioinformatics/bty645">Crossref</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Kristina
            Schröter, </span><span class="hlFld-ContribAuthor">Bernd Wemheuer, </span><span class="hlFld-ContribAuthor">Rodica
            Pena, </span><span class="hlFld-ContribAuthor">Ingo Schöning, </span><span class="hlFld-ContribAuthor">Martin
            Ehbrecht, </span><span class="hlFld-ContribAuthor">Peter Schall, </span><span class="hlFld-ContribAuthor">Christian
            Ammer, </span><span class="hlFld-ContribAuthor">Rolf Daniel and </span><span class="hlFld-ContribAuthor">Andrea
            Polle</span></span>, <span>Assembly processes of trophic guilds in the root mycobiome
            of temperate forests</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology</i></span><span class="seperator">, </span><span class="volume"><b>28</b></span><span class="issue">, 2</span><span class="seperator">, </span><span class="page-range">
            (348-364)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.14887">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Karen
            L. Bell, </span><span class="hlFld-ContribAuthor">Kevin S. Burgess, </span><span class="hlFld-ContribAuthor">Jamieson
            C. Botsch, </span><span class="hlFld-ContribAuthor">Emily K. Dobbs, </span><span class="hlFld-ContribAuthor">Timothy
            D. Read and </span><span class="hlFld-ContribAuthor">Berry J. Brosi</span></span>,
            <span>Quantitative and qualitative assessment of pollen DNA metabarcoding using constructed
            species mixtures</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology</i></span><span class="seperator">, </span><span class="volume"><b>28</b></span><span class="issue">, 2</span><span class="seperator">, </span><span class="page-range">
            (431-455)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.14840">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Antton
            Alberdi, </span><span class="hlFld-ContribAuthor">Ostaizka Aizpurua, </span><span class="hlFld-ContribAuthor">Kristine Bohmann, </span><span class="hlFld-ContribAuthor">Shyam
            Gopalakrishnan, </span><span class="hlFld-ContribAuthor">Christina Lynggaard, </span><span class="hlFld-ContribAuthor">Martin Nielsen and </span><span class="hlFld-ContribAuthor">Marcus
            Thomas Pius Gilbert</span></span>, <span>Promises and pitfalls of using high‐throughput
            sequencing for diet analysis</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology Resources</i></span><span class="seperator">, </span><span class="volume"><b>19</b></span><span class="issue">, 2</span><span class="seperator">, </span><span class="page-range">
            (327-348)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12960">Wiley Online Library</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Jihoon
            Jo, </span><span class="hlFld-ContribAuthor">Hyun-Gwan Lee, </span><span class="hlFld-ContribAuthor">Kwang
            Young Kim and </span><span class="hlFld-ContribAuthor">Chungoo Park</span></span>,
            <span>SoEM: a novel PCR-free biodiversity assessment method based on small-organelles
            enriched metagenomics</span><span class="seperator">, </span><span class="seriesTitle"><i>ALGAE</i></span><span class="seperator">, </span><span class="doi">10.4490/algae.2019.34.2.26</span><span class="seperator">, </span><span class="volume"><b>34</b></span><span class="issue">,
            1</span><span class="seperator">, </span><span class="page-range"> (57-70)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2019)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.4490/algae.2019.34.2.26">Crossref</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Wei
            Xiong and </span><span class="hlFld-ContribAuthor">Aibin Zhan</span></span>, <span>Testing
            clustering strategies for metabarcoding‐based investigation of community–environment
            interactions</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology Resources</i></span><span class="seperator">, </span><span class="volume"><b>18</b></span><span class="issue">, 6</span><span class="seperator">, </span><span class="page-range">
            (1326-1338)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12922">Wiley Online Library</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Babett
            Günther, </span><span class="hlFld-ContribAuthor">Thomas Knebelsberger, </span><span class="hlFld-ContribAuthor">Hermann Neumann, </span><span class="hlFld-ContribAuthor">Silke
            Laakmann and </span><span class="hlFld-ContribAuthor">Pedro Martínez Arbizu</span></span>,
            <span>Metabarcoding of marine environmental DNA based on mitochondrial and nuclear
            genes</span><span class="seperator">, </span><span class="seriesTitle"><i>Scientific
            Reports</i></span><span class="seperator">, </span><span class="doi">10.1038/s41598-018-32917-x</span><span class="seperator">, </span><span class="volume"><b>8</b></span><span class="issue">,
            1</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1038/s41598-018-32917-x">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Melania
            E. Cristescu and </span><span class="hlFld-ContribAuthor">Paul D.N. Hebert</span></span>,
            <span>Uses and Misuses of Environmental DNA in Biodiversity Science and Conservation</span><span class="seperator">, </span><span class="seriesTitle"><i>Annual Review of Ecology,
            Evolution, and Systematics</i></span><span class="seperator">, </span><span class="doi">10.1146/annurev-ecolsys-110617-062306</span><span class="seperator">, </span><span class="volume"><b>49</b></span><span class="issue">,
            1</span><span class="seperator">, </span><span class="page-range"> (209-230)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1146/annurev-ecolsys-110617-062306">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Teresita
            M. Porter and </span><span class="hlFld-ContribAuthor">Mehrdad Hajibabaei</span></span>,
            <span>Automated high throughput animal CO1 metabarcode classification</span><span class="seperator">, </span><span class="seriesTitle"><i>Scientific Reports</i></span><span class="seperator">, </span><span class="doi">10.1038/s41598-018-22505-4</span><span class="seperator">, </span><span class="volume"><b>8</b></span><span class="issue">,
            1</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1038/s41598-018-22505-4">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Shaohua
            Gong, </span><span class="hlFld-ContribAuthor">Yanfei Ding, </span><span class="hlFld-ContribAuthor">Yi
            Wang, </span><span class="hlFld-ContribAuthor">Guangze Jiang and </span><span class="hlFld-ContribAuthor">Cheng
            Zhu</span></span>, <span>Advances in DNA Barcoding of Toxic Marine Organisms</span><span class="seperator">, </span><span class="seriesTitle"><i>International Journal of Molecular
            Sciences</i></span><span class="seperator">, </span><span class="doi">10.3390/ijms19102931</span><span class="seperator">, </span><span class="volume"><b>19</b></span><span class="issue">,
            10</span><span class="seperator">, </span><span class="page-range"> (2931)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.3390/ijms19102931">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Samuele
            Bovo, </span><span class="hlFld-ContribAuthor">Anisa Ribani, </span><span class="hlFld-ContribAuthor">Valerio
            Joe Utzeri, </span><span class="hlFld-ContribAuthor">Giuseppina Schiavo, </span><span class="hlFld-ContribAuthor">Francesca Bertolini and </span><span class="hlFld-ContribAuthor">Luca
            Fontanesi</span></span>, <span>Shotgun metagenomics of honey DNA: Evaluation of a
            methodological approach to describe a multi-kingdom honey bee derived environmental
            DNA signature</span><span class="seperator">, </span><span class="seriesTitle"><i>PLOS
            ONE</i></span><span class="seperator">, </span><span class="doi">10.1371/journal.pone.0205575</span><span class="seperator">, </span><span class="volume"><b>13</b></span><span class="issue">,
            10</span><span class="seperator">, </span><span class="page-range"> (e0205575)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1371/journal.pone.0205575">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Franz
            M. Heindler, </span><span class="hlFld-ContribAuthor">Henrik Christiansen, </span><span class="hlFld-ContribAuthor">Bruno Frédérich, </span><span class="hlFld-ContribAuthor">Agnes
            Dettaï, </span><span class="hlFld-ContribAuthor">Gilles Lepoint, </span><span class="hlFld-ContribAuthor">Gregory
            E. Maes, </span><span class="hlFld-ContribAuthor">Anton P. Van de Putte and </span><span class="hlFld-ContribAuthor">Filip A. M. Volckaert</span></span>, <span>Historical
            DNA Metabarcoding of the Prey and Microbiome of Trematomid Fishes Using Museum Samples</span><span class="seperator">, </span><span class="seriesTitle"><i>Frontiers in Ecology and Evolution</i></span><span class="seperator">, </span><span class="doi">10.3389/fevo.2018.00151</span><span class="seperator">,
            </span><span class="volume"><b>6</b></span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.3389/fevo.2018.00151">Crossref</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">M.
            A. Wilkes, </span><span class="hlFld-ContribAuthor">J. A. Webb, </span><span class="hlFld-ContribAuthor">P.
            S. Pompeu, </span><span class="hlFld-ContribAuthor">L. G. M. Silva, </span><span class="hlFld-ContribAuthor">A.
            S. Vowles, </span><span class="hlFld-ContribAuthor">C. F. Baker, </span><span class="hlFld-ContribAuthor">P.
            Franklin, </span><span class="hlFld-ContribAuthor">O. Link, </span><span class="hlFld-ContribAuthor">E.
            Habit and </span><span class="hlFld-ContribAuthor">P. S. Kemp</span></span>, <span>Not
            just a migration problem: Metapopulations, habitat shifts, and gene flow are also
            important for fishway science and management</span><span class="seperator">, </span><span class="seriesTitle"><i>River Research and Applications</i></span><span class="seperator">,
            </span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1002/rra.3320">Wiley Online Library</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Stefano
            Amalfitano, </span><span class="hlFld-ContribAuthor">Caterina Levantesi, </span><span class="hlFld-ContribAuthor">Laurent Garrelly, </span><span class="hlFld-ContribAuthor">Donatella
            Giacosa, </span><span class="hlFld-ContribAuthor">Francesca Bersani and </span><span class="hlFld-ContribAuthor">Simona Rossetti</span></span>, <span>Water Quality and
            Total Microbial Load: A Double-Threshold Identification Procedure Intended for Space
            Applications</span><span class="seperator">, </span><span class="seriesTitle"><i>Frontiers
            in Microbiology</i></span><span class="seperator">, </span><span class="doi">10.3389/fmicb.2018.02903</span><span class="seperator">, </span><span class="volume"><b>9</b></span><span class="seperator">,
            </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.3389/fmicb.2018.02903">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Benjamin
            Linard, </span><span class="hlFld-ContribAuthor">Krister Swenson and </span><span class="hlFld-ContribAuthor">Fabio Pardi</span></span>, <span>Rapid alignment-free
            phylogenetic identification of metagenomic sequences</span><span class="seperator">,
            </span><span class="seriesTitle"><i>Bioinformatics</i></span><span class="seperator">,
            </span><span class="doi">10.1093/bioinformatics/btz068</span><span class="seperator">,
            </span><span class="pub-date"><span class="pub-year">(2019)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1093/bioinformatics/btz068">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Mark
            A. K. Gillespie, </span><span class="hlFld-ContribAuthor">Matthias Alfredsson, </span><span class="hlFld-ContribAuthor">Isabel C. Barrio, </span><span class="hlFld-ContribAuthor">Joseph
            J. Bowden, </span><span class="hlFld-ContribAuthor">Peter Convey, </span><span class="hlFld-ContribAuthor">Lauren
            E. Culler, </span><span class="hlFld-ContribAuthor">Stephen J. Coulson, </span><span class="hlFld-ContribAuthor">Paul Henning Krogh, </span><span class="hlFld-ContribAuthor">Amanda
            M. Koltz, </span><span class="hlFld-ContribAuthor">Seppo Koponen, </span><span class="hlFld-ContribAuthor">Sarah
            Loboda, </span><span class="hlFld-ContribAuthor">Yuri Marusik, </span><span class="hlFld-ContribAuthor">Jonas
            P. Sandström, </span><span class="hlFld-ContribAuthor">Derek S. Sikes and </span><span class="hlFld-ContribAuthor">Toke T. Høye</span></span>, <span>Status and trends of
            terrestrial arthropod abundance and diversity in the North Atlantic region of the
            Arctic</span><span class="seperator">, </span><span class="seriesTitle"><i>Ambio</i></span><span class="seperator">, </span><span class="doi">10.1007/s13280-019-01162-5</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2019)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1007/s13280-019-01162-5">Crossref</a></span></div></li>
            </ul>
         </div>
      </div>
   </section>
   
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md sticko__parent article-row-right"><div class="publication-cover-info clearfix">



        
        <div class="cover-image-wrapper cover-image__image hasDetails"><a href="https://onlinelibrary.wiley.com/toc/1365294x/2018/27/2"><img src="mec.jpg" alt="Publication cover image"></a></div><div class="extra-info-wrapper cover-image__details"><p class="volume-issue"><a href="https://onlinelibrary.wiley.com/toc/1365294x/2018/27/2"><span>Volume</span><span class="val">27</span>, <span>Issue</span><span class="val">2</span></a></p><p>January 2018</p><p class="page-range"><span>Pages</span><span> 313-338</span></p>



        
        
</div>
</div>



        
        


        <div class="pb-ad">
            <div id="advert-rail-1" class="advert advert-rail">
			<script>

				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-rail-1' );
				});

			</script><div id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_1__container__" style="border: 0pt none; width: 300px; height: 250px;"></div>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1011&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fmec.14478&pubId=62117837&placeholderId=1014&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
				</a>
			</noscript>
		</div>
        </div>
    

<div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko tab--flex tabs--xs js--sticko" style="width: 289.983px; top: 117.917px; position: static; height: 0px;"><ul data-mobile-toggle="slide" class="rlist tab__nav w-slide--list cloned"><li role="presentation" class=""><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" aria-expanded="false"><i aria-hidden="true" class="icon-multi-figure"></i><span>Figures</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty"><i aria-hidden="true" class="icon-chain"></i><span>References</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><span>Related</span></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" aria-expanded="true"><i aria-hidden="true" class="icon-Icon_Information"></i><span>Information</span></a></li></ul><ul role="tablist" class="rlist tab__content sticko__child" style="height: 0px; overflow-y: auto;"><li id="pane-pcw-figures" aria-labelledby="pane-pcw-figurescon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-references" aria-labelledby="pane-pcw-referencescon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-related" aria-labelledby="pane-pcw-relatedcon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-details" aria-labelledby="pane-pcw-detailscon" role="tab" class="tab__pane article-row-right__panes active"><div class="contentTab"></div><div class="emptyTab">



        
        


        <div class="pb-ad">
            <aside>
	<div id="advert-rail-2" class="advert advert-rail">
		<script>

			googletag.cmd.push 		( function () {
				googletag.display 	( 'advert-rail-2' );
			});

		</script><div id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_2__container__" style="border: 0pt none; width: 300px; height: 250px;"></div>

		<!-- No JavaScript support is a call to an image -->
		<noscript>
			<a href="/action/clickThrough?id=1012&url=http%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fmec.14478&pubId=62117837&placeholderId=1013&productId=101018" target="_blank">
				<img src="http://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
			</a>
		</noscript>
	</div>
</aside>
        </div>
    

</div></li><li class="tab__spinner"><img src="spinner.gif" id="spinner" style="width: 100%" alt=""></li></ul></div></div></div></article><script>var articleRef = document.querySelector('.article__body:not(.show-references) .article__references');
if (articleRef) { articleRef.style.display = "none"; }

</script><div id="figure-viewer" data-ux3-wrapper="figure-viewer" data-ux3-transformed-by="figureInit" data-ux3-role="parent" role="dialog" class="figure-viewer"><div class="figure-viewer__reg__top clearfix"><div class="figure-viewer__top__right"><a title="Close Figure Viewer" href="#" data-ux3-role="controller" class="figure-viewer__ctrl__close"><span class="icon-close_thin"></span></a></div><div class="figure-viewer__top__left"><a title="Browse All Figures" href="#" class="figure-viewer__ctrl__browse"><span class="icon-allfigures"></span></a><a title="Return to Figure" href="#" class="figure-viewer__ctrl__return is-hidden"><span class="icon-arrow-left"></span></a><button title="zoom out" class="figure-viewer__label__zoom icon-zoom-out zoom-out"></button><span tabindex="1" class="zoomSlider js__zoom-slider ui-slider"><input class="zoom-range" value="50" type="range"></span><button title="zoom in" class="figure-viewer__label__zoom icon-zoom-in zoom-in"></button></div></div><div class="figure-viewer__reg__center" style="height: 100%;"><div class="figure-viewer__cent__left"><a title="Previous Figure" href="#" class="figure-viewer__ctrl__prev"><span class="icon-arrow_l"></span></a><a title="Next Figure" href="#" class="figure-viewer__ctrl__next"><span class="icon-arrow_r"></span></a><div class="figure-viewer__hold__fig"><figure class="holder"></figure></div><div class="figure-viewer__hold__list clearfix container"></div></div><div class="figure-viewer__cent__right"><div class="figure-viewer__title"><a title="Open/Close Caption" href="#" class="figure-viewer__ctrl__caption"><span class="icon-doublearrow"></span><span class="figure-viewer__caption__label">Caption</span></a><span class="figure-viewer__title__text"></span></div><div class="figure-viewer__hold__figcap"></div></div></div></div>
</div>
</div></div></div>

        </section>
    
        </div>
    </div><!--end pagefulltext-->
<div class="w-slide"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div></main>

        </div>
    










        <div data-widget-def="pageFooter" data-widget-id="053994f3-7d80-4bb1-97a4-c67fbc4768cd" class="pageFooter">
        



        
        <footer class="page-footer">
    <div data-pb-dropzone="main">
    



        
        <div class="main-footer">



        
        <style>
    .footer-links {
        text-align: center;
    }
    .footer-links h3 {
        font-weight: 300;
    }
    .footer-links a {
        font-weight: 600;
    }
    @media (min-width: 992px) {
        .footer-links {
            text-align: left;
        }
    }
</style>












        <div data-widget-def="ux3-layout-widget" data-widget-id="2753bec1-a79d-4403-827b-6c788ddd6db3" class="footer-sections">
        



        
            <h3 class="hidden accessibility-enabled">
                Additional links
            </h3>
        
        <div class="container"><div class="row"><div class=" col-md-3">



        
        <div class="footer-links mb-3">
  <h3>About Wiley Online Library</h3>
  <ul class="rlist">
    <li><a href="https://www.wiley.com/privacy">Privacy Policy</a></li>
    <li><a href="https://onlinelibrary.wiley.com/terms-and-conditions">Terms of Use</a></li>
    <li><a href="https://onlinelibrary.wiley.com/cookies">Cookies</a></li>
    <li><a href="https://onlinelibrary.wiley.com/accessibility">Accessibility</a></li>
  </ul>
</div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Help &amp; Support</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/support/onlinelibrary">Contact Us</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Opportunities</h3>
    <ul class="rlist">
      <li><a href="https://onlinelibrary.wiley.com/agents">Subscription Agents</a></li>
      <li><a href="https://onlinelibrary.wiley.com/advertisers">Advertisers &amp; Corporate Partners</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
        <div class="footer-links mb-3">
    <h3>Connect with Wiley</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/exchanges/">The Wiley Network</a></li>
      <li><a href="http://newsroom.wiley.com/">Wiley Press Room</a></li>
    </ul>
  </div>
</div></div></div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="86584e25-e4db-4118-9550-9b4f5341a4ac" class="copyright-container">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="bfbf6c5d-a650-4662-828f-5275fada9120" class="copyright">
        



        
        <div class="copyright pull-left">Copyright © 1999-2019 <a href="http://www.wiley.com/">John Wiley &amp; Sons, Inc</a>. All rights reserved</div>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="063fa3e6-c0a3-4672-a2e6-62aa1b09ed11" class="footer-logo pull-right">
        



        
        <a href="http://www.wiley.com/" title="Wiley Home Page"><img alt="Wiley Home Page" src="footer-logo-wiley-1510029248417.png"></a>

        </div>
    
</div></div></div>

        </div>
    
</div>
    </div>
</footer>

        </div>
    




        
        

















<div class="popup login-popup hidden" role="dialog" tabindex="-1" aria-labelledby="modal__header" aria-hidden="true">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="Cancel Login">
            <i class="icon-tools_close" aria-hidden="true"></i>
        </a>


        <div class="modal__header">
            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>
        </div>

        <div class="modal__body">
            <div class="ssoLogin">
                
            </div>

            <h2>Log in to Wiley Online Library</h2>

            





























<div class="login-form">
    

    

    
    <form action="/action/doLogin?societyURLCode=" class="bordered" method="post"><input name="id" value="67065c09-4a88-49cd-934c-ac707951d35c" type="hidden">
        
            <input name="popup" value="true" type="hidden">
        
        

        
        <input name="loginUri" value="/doi/full/10.1111/mec.14478" type="hidden">
        
            




<input name="redirectUri" value="/doi/full/10.1111/mec.14478" type="hidden">
<input name="loginUri" value="/doi/full/10.1111/mec.14478" type="hidden">
            
        
        
        
                



















    
    
    <div class="input-group">
        <div class="label">

            
                    <label for="login">Email or Customer ID</label>
                
        </div>
        <input id="username" class="login" name="login" size="15" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="text">
        
                    
                        <div class="actions">
                            
                        </div>
                    
                
    </div>


                
                
















    
    <div class="input-group">
        <div class="label">
            <label for="password">Password</label>
        </div>
        <input id="password" class="password" name="password" value="" autocomplete="off" placeholder="Enter your password" autocorrect="off" spellcheck="false" autocapitalize="off" type="password">
        <span class="password-eye-icon icon-eye hidden"></span>
        
                
    </div>

                
                    
                        <div class="actions">
                            
                            <a href="https://onlinelibrary.wiley.com/action/requestResetPassword" class="link show-request-reset-password">
                                Forgot password?
                            </a>
                        </div>
                    
                
            
        
        <div class="loginExtraBeans-dropZone" data-pb-dropzone="loginExtraBeans">
            
        </div>

        
                <div class="align-end">
                    
                    
                            
                            <span class="submit ">
                                <input class="button btn submit primary no-margin-bottom" name="submitButton" value="Log In" type="submit">
                            </span>
                        
                </div></form>
    <div class="societySpecificInstructions">
        
    </div>
</div>

            
                <div class="login-container clearfix form-btn">
                    <a href="https://onlinelibrary.wiley.com/action/registration" class="linkBtn float-left">
                        
                        NEW USER <span aria-hidden="true">&gt;</span>
                    </a>
                    
                        <a href="https://onlinelibrary.wiley.com/action/ssostart?redirectUri=%2Fdoi%2Ffull%2F10.1111%2Fmec.14478" class="linkBtn float-right instituation-login">
                            
                            INSTITUTIONAL LOGIN <span aria-hidden="true">&gt;</span>
                        </a>
                    
                </div>
            

        </div>
    </div>
</div>




        
        






<div class="popup change-password-drawer hidden">
    <div class="content">
        <a href="#" class="close cancel" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
        <div class="modal__header">


            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>
            <h2>Change Password</h2>
        </div>
        <div class="modal__body">

            <div class="form">

                <form action="/action/changePassword" method="post"><div class="message error"></div>
                    <input name="submit" value="submit" type="hidden">
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-old">Old Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-old" class="old" name="old" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                    </div>
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-new">New Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-new" class="pass-hint new" name="new" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                        
                            







    


    


    


<div class="password-strength-indicator" data-min="10" data-max="32" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
                        
                    </div>
                    <input class="button primary submit" value="Submit" disabled="disabled" type="submit"></form>
            </div>
            <div class="success-template hidden">
                <h2>Congrats!</h2>
                <p>Your password has been changed</p>
            </div>
        </div>
    </div>
</div>





        
        <div></div>




        
        





<div class="popup registration-popup hidden">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="logoDropZone" data-pb-dropzone="logoDropZone">
                
            </div>

            <h2>Create a new account</h2>
        </div>

        <div class="modal__body">
            <form action="/action/registration" class="registration-form" method="post"><input name="redirectUri" value="/doi/full/10.1111/mec.14478" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="4e647394-f751-4441-baa4-df426bca4b6e.email">Email or Customer ID</label>
                    </div>
                    <input id="4e647394-f751-4441-baa4-df426bca4b6e.email" class="email" name="email" size="15" type="text">
                </div>
                <div class="submit">
                    <input class="button submit primary" value="Register" disabled="disabled" type="submit">
                </div></form>
            <div class="center">
                <a href="https://onlinelibrary.wiley.com/action/showLogin" class="show-login">
                    
                    Returning user
                </a>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden">
    <div class="content">
        <div class="modal__header">
            <a href="#" class="cancel close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>

            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>

            <h2>Forgot your password?</h2>
            <p class="sub">Enter your email address below. </p>
        </div>

        <div class="modal__body">

            









<div class="error"></div>


<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input name="requestResetPassword" value="true" type="hidden">
    
            <div class="input-group">
                



















    
    
    <div class="input-group">
        <div class="label">

            
                    <label for="email">Email or Customer ID</label>
                
        </div>
        <input id="email" class="email" name="email" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="email">
        
    </div>

            </div>
        


    
            <div class="password-recaptcha-ajax"></div>
        

    <div class="form-btn">
        <input class="button btn primary submit" name="submit" value="RESET PASSWORD" disabled="disabled" type="submit">
    </div></form>
            

            <div class="message error"></div>

            <div class="success-template hidden">
                <p>Please check your email for instructions on resetting your password.
                    If you do not receive an email within 10 minutes, your email address may not be registered,
                    and you may need to create a new Wiley Online Library account.</p>
            </div>
        </div>
    </div>
</div>





        
        




<div class="popup top-drawer request-username-drawer username-popup hidden">
    <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
    <div class="content">
        <h2>Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input name="requestUsername" value="requestUsername" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email">Email or Customer ID</label>
                    </div>
                    <input id="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email" class="email" name="email" size="15" type="text">
                </div>

                <div class="username-recaptcha-ajax">
                </div>

                <input class="button primary submit" name="submit" value="Submit" disabled="disabled" type="submit">
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        <!-- Belongs at the bottom of the page -->

<script type="text/javascript">
if(window._satellite) { 
_satellite.pageBottom(); 
}
</script>
        </div>
    </div><iframe sandbox="allow-scripts allow-same-origin" title="Adobe ID Syncing iFrame" id="destination_publishing_iframe_wileypublishing_0" name="destination_publishing_iframe_wileypublishing_0_name" style="display: none; width: 0px; height: 0px;" src="https://wileypublishing.demdex.net/dest5.html?d_nsid=0#https%3A%2F%2Fonlinelibrary.wiley.com" class="aamIframeLoaded"></iframe><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh335" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.4507315700952257&amp;iit=1554373478928&amp;tmr=load%3D1554373464542%26core%3D1554373464570%26main%3D1554373478918%26ifr%3D1554373478936&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=onlinelibrary.wiley.com&amp;dr=&amp;du=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fmec.14478&amp;href=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fmec.14478&amp;dt=Scaling%20up%3A%20A%20guide%20to%20high%E2%80%90throughput%20genomic%20approaches%20for%20biodiversity%20analysis%20-%20Porter%20-%202018%20-%20Molecular%20Ecology%20-%20Wiley%20Online%20Library&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=5ca5db5837b7b2d1&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=&amp;csi=undefined&amp;rev=v8.4.0-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style><div class="kxhead" data-id="JMOONjNh" style="display:none !important;"><span class="kxtag kxinvisible" data-id="20226"><script>
// this tag is intentionally blank
</script></span><span class="kxtag kxinvisible" data-id="20227"></span><span class="kxtag kxinvisible" data-id="20229"><script>
(function() {
  // adadvisor redirects to http://logger... so it's not https safe
  if (location.protocol == "http:") {
     var u = "https://aa.agkn.com/adscores/g.js?sid=9212244187&_kdpid=2111c0af-fc3a-446f-ab07-63aa74fbde8e";
     (new Image()).src = u;
   }
})();
</script></span><span class="kxtag kxinvisible" data-id="20234"><script type="text/javascript">
var findPh = document.getElementsByTagName("h1")[0]
if(findPh != null) { var pageheader = document.getElementsByTagName("h1")[0].innerHTML }

// Using Omniture prop1 to produce page attribute Page Type
Krux('set', 'page_attr_omniture_prop1', typeof window.s =='object' && window.s['prop1']);

// Using Omniture prop2 to produce page attribute Page Subtype
Krux('set', 'page_attr_omniture_prop2', typeof window.s =='object' && window.s['prop2']);

// Using Omniture prop3 to produce page attribute Title
Krux('set', 'page_attr_omniture_prop3', typeof window.s =='object' && window.s['prop3']);

// Using Omniture prop4 to produce page attribute Subject Codes
Krux('set', 'page_attr_omniture_prop4', typeof window.s =='object' && window.s['prop4']);

// Using Omniture prop7 to produce page attribute Article Title
Krux('set', 'page_attr_omniture_prop7', typeof window.s =='object' && window.s['prop7']);

// Using Omniture prop8 to produce page attribute Journal Volume and Issue
Krux('set', 'page_attr_omniture_prop8', typeof window.s =='object' && window.s['prop8']);

// Using Omniture prop9 to produce page attribute Article Access State
Krux('set', 'page_attr_omniture_prop9', typeof window.s =='object' && window.s['prop9']);

// Using Omniture prop10 to produce page attribute Publication Access Status
Krux('set', 'page_attr_omniture_prop10', typeof window.s =='object' && window.s['prop10']);

// Using Omniture prop13 to produce page attribute Journal DOI
Krux('set', 'page_attr_omniture_prop13', typeof window.s =='object' && window.s['prop13']);

// Using Omniture prop14 to produce page attribute Article / Chapter DOI
Krux('set', 'page_attr_omniture_prop14', typeof window.s =='object' && window.s['prop14']);

// Using Omniture prop15 to produce page attribute Issue DOI
Krux('set', 'page_attr_omniture_prop15', typeof window.s =='object' && window.s['prop15']);

// Using Omniture prop16 to produce page attribute MRW Chapter
Krux('set', 'page_attr_omniture_prop16', typeof window.s =='object' && window.s['prop16']);

// Using Omniture prop20 to produce page attribute Site Search Query
Krux('set', 'page_attr_omniture_prop20', typeof window.s =='object' && window.s['prop20']);

// Using Omniture prop26 to produce page attribute Logged In or Out
Krux('set', 'user_attr_omniture_prop26', typeof window.s =='object' && window.s['prop26']);

// Using Omniture channel to produce page attribute Channel
Krux('set', 'page_attr_omniture_channel', typeof window.s =='object' && window.s['channel']);

// Using Omniture prop33 to produce page attribute Primary Subject Code
Krux('set', 'page_attr_omniture_prop33', typeof window.s =='object' && window.s['prop33']);

// Using pageheader to produce page attribute Page Header
Krux('scrape', { 'page_attr_wileyonlinelib_pageheader': {javascript: 'pageheader'}});

</script></span><span class="kxtag kxinvisible" data-id="20238"></span><span class="kxtag kxinvisible" data-id="20241"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=d7158cb7-a851-4e3c-b7ab-cc9e815b2399&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=gwjn34gwjnh86&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20243"></span><span class="kxtag kxinvisible" data-id="20247"><script type="text/javascript">
(function() {
    (new Image()).src = '//aa.agkn.com/adscores/g.js?sid=9212244187&_kdpid=2111c0af-fc3a-446f-ab07-63aa74fbde8e';
})();
</script></span><span class="kxtag kxinvisible" data-id="20248"><script>
(function(){
        var kuid = Krux('get', 'user');
        var prefix = window.location.protocol == 'https:' ? 'https:' : 'http:';
        if (kuid) {
           var url = prefix + '//pix.btrll.com/partner/868092.png';
           (new Image()).src = url;
        }
})();
</script></span><span class="kxtag kxinvisible" data-id="20251"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=8da8b14d-5569-4bec-bcea-722864ee8d06&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iefs40iefsj26&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20252"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=bef9f122-393d-4c45-8d8d-32d8be7ac433&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iyzu39iyzud95&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20253"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';
 
            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };
         
        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=8bf57916-aac8-4f01-a386-4baf103b3e1f&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=bckw15bckwu20&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20254"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';

            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };

        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=d7158cb7-a851-4e3c-b7ab-cc9e815b2399&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=gwjn34gwjnh86&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20255"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=536f0daa-aaaa-4d9e-9a25-dde40646786a&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iefs40iefsj26&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20256"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=7c6392c9-e878-492c-8b14-bf06e3828ebd&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=rowp70rowpu60&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20257"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';

            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };
        
        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=bb8ae0e2-9cd7-45b2-ad37-7737269627d8&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=rsxs71rsxsk73&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span></div>


<script type="text/javascript" src="main.js"></script>
<script type="text/javascript" src="gj6xproduct.js"></script>
<script>var _prum=[['id','59e8fecb3847311aab7b23c6'],['mark','firstbyte',(new Date()).getTime()]];(function(){var s=document.getElementsByTagName('script')[0],p=document.createElement('script');p.async='async';p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();</script>

<div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><a target="_blank" href="http://www.crossref.org/crossmark"><img class="crossmark-reset crossmark-popup__logo"></a><button class="crossmark-reset crossmark-popup__btn-close"></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content"></iframe></div></div></div></div></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-1" tabindex="0" style="display: none;"></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-2" tabindex="0" style="display: none;"></ul></body><iframe src="https://injections.readcube.com/html/track_epdf.6d337e75.html?action=Link&amp;publisher=wiley&amp;status=OK" style="position: absolute; visibility: hidden; width: 0px; height: 0px;"></iframe><iframe id="kx-proxy-JMOONjNh" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=JMOONjNh&amp;kxt=https%3A%2F%2Fonlinelibrary.wiley.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>